-- MySQL dump 10.13  Distrib 5.6.17, for osx10.7 (x86_64)
--
-- Host: localhost    Database: ucdavis_genetics_lab
-- ------------------------------------------------------
-- Server version	5.6.17

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;

--
-- Table structure for table `wp_commentmeta`
--

DROP TABLE IF EXISTS `wp_commentmeta`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_commentmeta` (
  `meta_id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
  `comment_id` bigint(20) unsigned NOT NULL DEFAULT '0',
  `meta_key` varchar(255) DEFAULT NULL,
  `meta_value` longtext,
  PRIMARY KEY (`meta_id`),
  KEY `comment_id` (`comment_id`),
  KEY `meta_key` (`meta_key`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_commentmeta`
--

LOCK TABLES `wp_commentmeta` WRITE;
/*!40000 ALTER TABLE `wp_commentmeta` DISABLE KEYS */;
/*!40000 ALTER TABLE `wp_commentmeta` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_comments`
--

DROP TABLE IF EXISTS `wp_comments`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_comments` (
  `comment_ID` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
  `comment_post_ID` bigint(20) unsigned NOT NULL DEFAULT '0',
  `comment_author` tinytext NOT NULL,
  `comment_author_email` varchar(100) NOT NULL DEFAULT '',
  `comment_author_url` varchar(200) NOT NULL DEFAULT '',
  `comment_author_IP` varchar(100) NOT NULL DEFAULT '',
  `comment_date` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `comment_date_gmt` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `comment_content` text NOT NULL,
  `comment_karma` int(11) NOT NULL DEFAULT '0',
  `comment_approved` varchar(20) NOT NULL DEFAULT '1',
  `comment_agent` varchar(255) NOT NULL DEFAULT '',
  `comment_type` varchar(20) NOT NULL DEFAULT '',
  `comment_parent` bigint(20) unsigned NOT NULL DEFAULT '0',
  `user_id` bigint(20) unsigned NOT NULL DEFAULT '0',
  PRIMARY KEY (`comment_ID`),
  KEY `comment_post_ID` (`comment_post_ID`),
  KEY `comment_approved_date_gmt` (`comment_approved`,`comment_date_gmt`),
  KEY `comment_date_gmt` (`comment_date_gmt`),
  KEY `comment_parent` (`comment_parent`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_comments`
--

LOCK TABLES `wp_comments` WRITE;
/*!40000 ALTER TABLE `wp_comments` DISABLE KEYS */;
/*!40000 ALTER TABLE `wp_comments` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_gallery_galleries`
--

DROP TABLE IF EXISTS `wp_gallery_galleries`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_gallery_galleries` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `title` varchar(150) NOT NULL DEFAULT '',
  `created` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  PRIMARY KEY (`id`)
) ENGINE=MyISAM DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_gallery_galleries`
--

LOCK TABLES `wp_gallery_galleries` WRITE;
/*!40000 ALTER TABLE `wp_gallery_galleries` DISABLE KEYS */;
/*!40000 ALTER TABLE `wp_gallery_galleries` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_gallery_galleriesslides`
--

DROP TABLE IF EXISTS `wp_gallery_galleriesslides`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_gallery_galleriesslides` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `gallery_id` int(11) NOT NULL DEFAULT '0',
  `slide_id` int(11) NOT NULL DEFAULT '0',
  `order` int(11) NOT NULL DEFAULT '0',
  `created` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  PRIMARY KEY (`id`)
) ENGINE=MyISAM DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_gallery_galleriesslides`
--

LOCK TABLES `wp_gallery_galleriesslides` WRITE;
/*!40000 ALTER TABLE `wp_gallery_galleriesslides` DISABLE KEYS */;
/*!40000 ALTER TABLE `wp_gallery_galleriesslides` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_gallery_slides`
--

DROP TABLE IF EXISTS `wp_gallery_slides`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_gallery_slides` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `title` varchar(150) NOT NULL DEFAULT '',
  `description` text NOT NULL,
  `showinfo` varchar(50) NOT NULL DEFAULT 'both',
  `iopacity` int(11) NOT NULL DEFAULT '70',
  `image` varchar(50) NOT NULL DEFAULT '',
  `type` enum('file','url') NOT NULL DEFAULT 'file',
  `image_url` varchar(200) NOT NULL DEFAULT '',
  `uselink` enum('Y','N') NOT NULL DEFAULT 'N',
  `linktarget` enum('self','blank') NOT NULL DEFAULT 'self',
  `link` varchar(200) NOT NULL DEFAULT '',
  `order` int(11) NOT NULL DEFAULT '0',
  `created` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  PRIMARY KEY (`id`)
) ENGINE=MyISAM AUTO_INCREMENT=4 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_gallery_slides`
--

LOCK TABLES `wp_gallery_slides` WRITE;
/*!40000 ALTER TABLE `wp_gallery_slides` DISABLE KEYS */;
INSERT INTO `wp_gallery_slides` VALUES (1,'Support Us!','Click here to donate!','both',70,'support_header-e1402598171853.png','url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/support_header-e1402598171853.png','Y','self','http://localhost/~psalessi/Genetics_Lab/support-us',2,'2014-06-05 22:31:13','2014-06-12 20:24:15'),(3,'2012 Coastwide Salmon Genetics Meeting at UC Davis','Click here to see our lab participating at the 2012 Coastwide Salmon Genetics Meeting','both',70,'alumni_header.png','url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/alumni_header.png','Y','self','http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html',3,'2014-06-05 22:52:50','2014-06-12 18:21:47'),(2,'Learn About Our Projects!','Click here to view our projects!','both',70,'brackish_water_hydros_header_image.png','url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/brackish_water_hydros_header_image.png','Y','self','http://localhost/~psalessi/Genetics_Lab/projects',1,'2014-06-05 22:31:40','2014-06-12 18:22:08');
/*!40000 ALTER TABLE `wp_gallery_slides` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_huge_itslider_images`
--

DROP TABLE IF EXISTS `wp_huge_itslider_images`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_huge_itslider_images` (
  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `name` varchar(100) DEFAULT NULL,
  `slider_id` varchar(200) DEFAULT NULL,
  `description` text,
  `image_url` text,
  `sl_url` varchar(128) DEFAULT NULL,
  `sl_type` text NOT NULL,
  `link_target` text NOT NULL,
  `ordering` int(11) NOT NULL,
  `published` tinyint(4) unsigned DEFAULT NULL,
  `published_in_sl_width` tinyint(4) unsigned DEFAULT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `id` (`id`)
) ENGINE=MyISAM AUTO_INCREMENT=5 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_huge_itslider_images`
--

LOCK TABLES `wp_huge_itslider_images` WRITE;
/*!40000 ALTER TABLE `wp_huge_itslider_images` DISABLE KEYS */;
INSERT INTO `wp_huge_itslider_images` VALUES (1,'','1','','http://localhost/~psalessi/Genetics_Lab/wp-content/plugins/slider-image/Front_images/slides/slide1.jpg','http://huge-it.com','image','on',1,1,NULL),(2,'Simple Usage','1','','http://localhost/~psalessi/Genetics_Lab/wp-content/plugins/slider-image/Front_images/slides/slide2.jpg','http://huge-it.com','image','on',2,1,NULL),(3,'Huge-IT Slider','1','The slider allows having unlimited amount of images with their titles and descriptions. The slider uses autogenerated shortcodes making it easier for the users to add it to the custom location.','http://localhost/~psalessi/Genetics_Lab/wp-content/plugins/slider-image/Front_images/slides/slide3.jpg','http://huge-it.com','image','on',3,1,NULL);
/*!40000 ALTER TABLE `wp_huge_itslider_images` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_huge_itslider_params`
--

DROP TABLE IF EXISTS `wp_huge_itslider_params`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_huge_itslider_params` (
  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `name` varchar(50) CHARACTER SET utf8 NOT NULL,
  `title` varchar(200) CHARACTER SET utf8 NOT NULL,
  `description` text CHARACTER SET utf8 NOT NULL,
  `value` varchar(200) CHARACTER SET utf8 NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=MyISAM AUTO_INCREMENT=126 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_huge_itslider_params`
--

LOCK TABLES `wp_huge_itslider_params` WRITE;
/*!40000 ALTER TABLE `wp_huge_itslider_params` DISABLE KEYS */;
INSERT INTO `wp_huge_itslider_params` VALUES (89,'slider_crop_image','Slider crop image','Slider crop image','resize'),(90,'slider_title_color','Slider title color','Slider title color','000000'),(91,'slider_title_font_size','Slider title font size','Slider title font size','13'),(92,'slider_description_color','Slider description color','Slider description color','ffffff'),(93,'slider_description_font_size','Slider description font size','Slider description font size','13'),(94,'slider_title_position','Slider title position','Slider title position','right-top'),(95,'slider_description_position','Slider description position','Slider description position','right-bottom'),(96,'slider_title_border_size','Slider Title border size','Slider Title border size','0'),(97,'slider_title_border_color','Slider title border color','Slider title border color','ffffff'),(98,'slider_title_border_radius','Slider title border radius','Slider title border radius','4'),(99,'slider_description_border_size','Slider description border size','Slider description border size','0'),(100,'slider_description_border_color','Slider description border color','Slider description border color','ffffff'),(101,'slider_description_border_radius','Slider description border radius','Slider description border radius','0'),(102,'slider_slideshow_border_size','Slider border size','Slider border size','0'),(103,'slider_slideshow_border_color','Slider border color','Slider border color','ffffff'),(104,'slider_slideshow_border_radius','Slider border radius','Slider border radius','0'),(105,'slider_navigation_type','Slider navigation type','Slider navigation type','1'),(106,'slider_navigation_position','Slider navigation position','Slider navigation position','bottom'),(107,'slider_title_background_color','Slider title background color','Slider title background color','ffffff'),(108,'slider_description_background_color','Slider description background color','Slider description background color','000000'),(109,'slider_title_transparent','Slider title has background','Slider title has background','on'),(110,'slider_description_transparent','Slider description has background','Slider description has background','on'),(111,'slider_slider_background_color','Slider slider background color','Slider slider background color','ffffff'),(112,'slider_dots_position','slider dots position','slider dots position','top'),(113,'slider_active_dot_color','slider active dot color','','ffffff'),(114,'slider_dots_color','slider dots color','','000000'),(115,'slider_description_width','Slider description width','Slider description width','70'),(116,'slider_description_height','Slider description height','Slider description height','50'),(117,'slider_description_background_transparency','slider description background transparency','slider description background transparency','70'),(118,'slider_description_text_align','description text-align','description text-align','justify'),(119,'slider_title_width','slider title width','slider title width','30'),(120,'slider_title_height','slider title height','slider title height','50'),(121,'slider_title_background_transparency','slider title background transparency','slider title background transparency','70'),(122,'slider_title_text_align','title text-align','title text-align','right'),(123,'slider_title_has_margin','title has margin','title has margin','on'),(124,'slider_description_has_margin','description has margin','description has margin','on'),(125,'slider_show_arrows','Slider show left right arrows','Slider show left right arrows','on');
/*!40000 ALTER TABLE `wp_huge_itslider_params` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_huge_itslider_sliders`
--

DROP TABLE IF EXISTS `wp_huge_itslider_sliders`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_huge_itslider_sliders` (
  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `name` varchar(200) NOT NULL,
  `sl_height` int(11) unsigned DEFAULT NULL,
  `sl_width` int(11) unsigned DEFAULT NULL,
  `pause_on_hover` text,
  `slider_list_effects_s` text,
  `description` text,
  `param` text,
  `sl_position` text NOT NULL,
  `ordering` int(11) NOT NULL,
  `published` text,
  PRIMARY KEY (`id`),
  UNIQUE KEY `id` (`id`)
) ENGINE=MyISAM AUTO_INCREMENT=2 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_huge_itslider_sliders`
--

LOCK TABLES `wp_huge_itslider_sliders` WRITE;
/*!40000 ALTER TABLE `wp_huge_itslider_sliders` DISABLE KEYS */;
INSERT INTO `wp_huge_itslider_sliders` VALUES (1,'My First Slider',375,600,'on','random','4000','1000','center',1,'300');
/*!40000 ALTER TABLE `wp_huge_itslider_sliders` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_links`
--

DROP TABLE IF EXISTS `wp_links`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_links` (
  `link_id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
  `link_url` varchar(255) NOT NULL DEFAULT '',
  `link_name` varchar(255) NOT NULL DEFAULT '',
  `link_image` varchar(255) NOT NULL DEFAULT '',
  `link_target` varchar(25) NOT NULL DEFAULT '',
  `link_description` varchar(255) NOT NULL DEFAULT '',
  `link_visible` varchar(20) NOT NULL DEFAULT 'Y',
  `link_owner` bigint(20) unsigned NOT NULL DEFAULT '1',
  `link_rating` int(11) NOT NULL DEFAULT '0',
  `link_updated` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `link_rel` varchar(255) NOT NULL DEFAULT '',
  `link_notes` mediumtext NOT NULL,
  `link_rss` varchar(255) NOT NULL DEFAULT '',
  PRIMARY KEY (`link_id`),
  KEY `link_visible` (`link_visible`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_links`
--

LOCK TABLES `wp_links` WRITE;
/*!40000 ALTER TABLE `wp_links` DISABLE KEYS */;
/*!40000 ALTER TABLE `wp_links` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_nextend_smartslider_layouts`
--

DROP TABLE IF EXISTS `wp_nextend_smartslider_layouts`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_nextend_smartslider_layouts` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `title` varchar(100) NOT NULL,
  `slide` longtext,
  `params` text NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=MyISAM DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_nextend_smartslider_layouts`
--

LOCK TABLES `wp_nextend_smartslider_layouts` WRITE;
/*!40000 ALTER TABLE `wp_nextend_smartslider_layouts` DISABLE KEYS */;
/*!40000 ALTER TABLE `wp_nextend_smartslider_layouts` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_nextend_smartslider_sliders`
--

DROP TABLE IF EXISTS `wp_nextend_smartslider_sliders`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_nextend_smartslider_sliders` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `title` varchar(100) NOT NULL,
  `type` varchar(30) NOT NULL,
  `params` text NOT NULL,
  `generator` text NOT NULL,
  `slide` longtext,
  PRIMARY KEY (`id`)
) ENGINE=MyISAM DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_nextend_smartslider_sliders`
--

LOCK TABLES `wp_nextend_smartslider_sliders` WRITE;
/*!40000 ALTER TABLE `wp_nextend_smartslider_sliders` DISABLE KEYS */;
/*!40000 ALTER TABLE `wp_nextend_smartslider_sliders` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_nextend_smartslider_slides`
--

DROP TABLE IF EXISTS `wp_nextend_smartslider_slides`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_nextend_smartslider_slides` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `title` varchar(200) NOT NULL,
  `slider` int(11) NOT NULL,
  `publish_up` datetime NOT NULL,
  `publish_down` datetime NOT NULL,
  `published` tinyint(1) NOT NULL,
  `first` int(11) NOT NULL,
  `slide` longtext,
  `description` text NOT NULL,
  `thumbnail` varchar(255) NOT NULL,
  `background` varchar(300) NOT NULL,
  `params` text NOT NULL,
  `ordering` int(11) NOT NULL,
  `generator` int(11) NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=MyISAM DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_nextend_smartslider_slides`
--

LOCK TABLES `wp_nextend_smartslider_slides` WRITE;
/*!40000 ALTER TABLE `wp_nextend_smartslider_slides` DISABLE KEYS */;
/*!40000 ALTER TABLE `wp_nextend_smartslider_slides` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_nextend_smartslider_storage`
--

DROP TABLE IF EXISTS `wp_nextend_smartslider_storage`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_nextend_smartslider_storage` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `key` varchar(200) NOT NULL,
  `value` longtext,
  PRIMARY KEY (`id`),
  UNIQUE KEY `key` (`key`)
) ENGINE=MyISAM AUTO_INCREMENT=4 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_nextend_smartslider_storage`
--

LOCK TABLES `wp_nextend_smartslider_storage` WRITE;
/*!40000 ALTER TABLE `wp_nextend_smartslider_storage` DISABLE KEYS */;
INSERT INTO `wp_nextend_smartslider_storage` VALUES (1,'layout','{\"size\":\"1024|*|768\"}'),(2,'settings','{\"debugmessages\":\"1\",\"slideeditoralert\":\"1\",\"translateurl\":\"|*|\",\"jquery\":\"1\",\"placeholder\":\"http:\\/\\/www.nextendweb.com\\/static\\/placeholder.png\",\"tidy-input-encoding\":\"utf8\",\"tidy-output-encoding\":\"utf8\"}'),(3,'font','{\"sliderfont1customlabel\":\"Heading light\",\"sliderfont1\":\"{\\\"firsttab\\\":\\\"Text\\\",\\\"Text\\\":{\\\"color\\\":\\\"ffffffff\\\",\\\"size\\\":\\\"320||%\\\",\\\"tshadow\\\":\\\"0|*|1|*|1|*|000000c7\\\",\\\"afont\\\":\\\"google(@import url(http:\\/\\/fonts.googleapis.com\\/css?family=Open Sans);),Arial\\\",\\\"lineheight\\\":\\\"1.3\\\",\\\"bold\\\":1,\\\"italic\\\":0,\\\"underline\\\":0,\\\"align\\\":\\\"left\\\",\\\"paddingleft\\\":0},\\\"Link\\\":{\\\"paddingleft\\\":0,\\\"size\\\":\\\"100||%\\\"},\\\"Link:Hover\\\":{\\\"paddingleft\\\":0,\\\"size\\\":\\\"100||%\\\"}}\",\"sliderfont2customlabel\":\"Heading dark\",\"sliderfont2\":\"{\\\"firsttab\\\":\\\"Text\\\",\\\"Text\\\":{\\\"color\\\":\\\"000000db\\\",\\\"size\\\":\\\"320||%\\\",\\\"tshadow\\\":\\\"0|*|1|*|0|*|ffffff33\\\",\\\"afont\\\":\\\"google(@import url(http:\\/\\/fonts.googleapis.com\\/css?family=Open Sans);),Arial\\\",\\\"lineheight\\\":\\\"1.3\\\",\\\"bold\\\":1,\\\"italic\\\":0,\\\"underline\\\":0,\\\"align\\\":\\\"left\\\",\\\"paddingleft\\\":0},\\\"Link\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0},\\\"Link:Hover\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0}}\",\"sliderfont3customlabel\":\"Subheading light\",\"sliderfont3\":\"{\\\"firsttab\\\":\\\"Text\\\",\\\"Text\\\":{\\\"color\\\":\\\"ffffffff\\\",\\\"size\\\":\\\"170||%\\\",\\\"tshadow\\\":\\\"0|*|1|*|1|*|000000c7\\\",\\\"afont\\\":\\\"google(@import url(http:\\/\\/fonts.googleapis.com\\/css?family=Open Sans);),Arial\\\",\\\"lineheight\\\":\\\"1.2\\\",\\\"bold\\\":0,\\\"italic\\\":0,\\\"underline\\\":0,\\\"align\\\":\\\"left\\\",\\\"paddingleft\\\":0},\\\"Link\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0},\\\"Link:Hover\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0}}\",\"sliderfont4customlabel\":\"Subheading dark\",\"sliderfont4\":\"{\\\"firsttab\\\":\\\"Text\\\",\\\"Text\\\":{\\\"color\\\":\\\"000000db\\\",\\\"size\\\":\\\"170||%\\\",\\\"tshadow\\\":\\\"0|*|1|*|0|*|ffffff33\\\",\\\"afont\\\":\\\"google(@import url(http:\\/\\/fonts.googleapis.com\\/css?family=Open Sans);),Arial\\\",\\\"lineheight\\\":\\\"1.2\\\",\\\"bold\\\":0,\\\"italic\\\":0,\\\"underline\\\":0,\\\"align\\\":\\\"left\\\",\\\"paddingleft\\\":0},\\\"Link\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0},\\\"Link:Hover\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0}}\",\"sliderfont5customlabel\":\"Paragraph light\",\"sliderfont5\":\"{\\\"firsttab\\\":\\\"Text\\\",\\\"Text\\\":{\\\"color\\\":\\\"ffffffff\\\",\\\"size\\\":\\\"114||%\\\",\\\"tshadow\\\":\\\"0|*|1|*|1|*|000000c7\\\",\\\"afont\\\":\\\"google(@import url(http:\\/\\/fonts.googleapis.com\\/css?family=Open Sans);),Arial\\\",\\\"lineheight\\\":\\\"1.4\\\",\\\"bold\\\":0,\\\"italic\\\":0,\\\"underline\\\":0,\\\"align\\\":\\\"justify\\\",\\\"paddingleft\\\":0},\\\"Link\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0},\\\"Link:Hover\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0}}\",\"sliderfont6customlabel\":\"Paragraph dark\",\"sliderfont6\":\"{\\\"firsttab\\\":\\\"Text\\\",\\\"Text\\\":{\\\"color\\\":\\\"000000db\\\",\\\"size\\\":\\\"114||%\\\",\\\"tshadow\\\":\\\"0|*|1|*|0|*|ffffff33\\\",\\\"afont\\\":\\\"google(@import url(http:\\/\\/fonts.googleapis.com\\/css?family=Open Sans);),Arial\\\",\\\"lineheight\\\":\\\"1.4\\\",\\\"bold\\\":0,\\\"italic\\\":0,\\\"underline\\\":0,\\\"align\\\":\\\"justify\\\",\\\"paddingleft\\\":0},\\\"Link\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0},\\\"Link:Hover\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0}}\",\"sliderfont7customlabel\":\"Small text light\",\"sliderfont7\":\"{\\\"firsttab\\\":\\\"Text\\\",\\\"Text\\\":{\\\"color\\\":\\\"ffffffff\\\",\\\"size\\\":\\\"90||%\\\",\\\"tshadow\\\":\\\"0|*|1|*|1|*|000000c7\\\",\\\"afont\\\":\\\"google(@import url(http:\\/\\/fonts.googleapis.com\\/css?family=Open Sans);),Arial\\\",\\\"lineheight\\\":\\\"1.2\\\",\\\"bold\\\":0,\\\"italic\\\":0,\\\"underline\\\":0,\\\"align\\\":\\\"left\\\",\\\"paddingleft\\\":0},\\\"Link\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0},\\\"Link:Hover\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0}}\",\"sliderfont8customlabel\":\"Small text dark\",\"sliderfont8\":\"{\\\"firsttab\\\":\\\"Text\\\",\\\"Text\\\":{\\\"color\\\":\\\"000000db\\\",\\\"size\\\":\\\"90||%\\\",\\\"tshadow\\\":\\\"0|*|1|*|0|*|ffffff33\\\",\\\"afont\\\":\\\"google(@import url(http:\\/\\/fonts.googleapis.com\\/css?family=Open Sans);),Arial\\\",\\\"lineheight\\\":\\\"1.1\\\",\\\"bold\\\":0,\\\"italic\\\":0,\\\"underline\\\":0,\\\"align\\\":\\\"left\\\",\\\"paddingleft\\\":0},\\\"Link\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0},\\\"Link:Hover\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0}}\",\"sliderfont9customlabel\":\"Handwritten light\",\"sliderfont9\":\"{\\\"firsttab\\\":\\\"Text\\\",\\\"Text\\\":{\\\"color\\\":\\\"ffffffff\\\",\\\"size\\\":\\\"140||%\\\",\\\"tshadow\\\":\\\"0|*|1|*|1|*|000000c7\\\",\\\"afont\\\":\\\"google(@import url(http:\\/\\/fonts.googleapis.com\\/css?family=Pacifico);),Arial\\\",\\\"lineheight\\\":\\\"1.3\\\",\\\"bold\\\":0,\\\"italic\\\":0,\\\"underline\\\":0,\\\"align\\\":\\\"left\\\",\\\"paddingleft\\\":0},\\\"Link\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0},\\\"Link:Hover\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0}}\",\"sliderfont10customlabel\":\"Handwritten dark\",\"sliderfont10\":\"{\\\"firsttab\\\":\\\"Text\\\",\\\"Text\\\":{\\\"color\\\":\\\"000000db\\\",\\\"size\\\":\\\"140||%\\\",\\\"tshadow\\\":\\\"0|*|1|*|0|*|ffffff33\\\",\\\"afont\\\":\\\"google(@import url(http:\\/\\/fonts.googleapis.com\\/css?family=Pacifico);),Arial\\\",\\\"lineheight\\\":\\\"1.3\\\",\\\"bold\\\":0,\\\"italic\\\":0,\\\"underline\\\":0,\\\"align\\\":\\\"left\\\",\\\"paddingleft\\\":0},\\\"Link\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0},\\\"Link:Hover\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0}}\",\"sliderfont11customlabel\":\"Button light\",\"sliderfont11\":\"{\\\"firsttab\\\":\\\"Text\\\",\\\"Text\\\":{\\\"color\\\":\\\"ffffffff\\\",\\\"size\\\":\\\"100||%\\\",\\\"tshadow\\\":\\\"0|*|1|*|1|*|000000c7\\\",\\\"afont\\\":\\\"google(@import url(http:\\/\\/fonts.googleapis.com\\/css?family=Open Sans);),Arial\\\",\\\"lineheight\\\":\\\"1.3\\\",\\\"bold\\\":0,\\\"italic\\\":0,\\\"underline\\\":0,\\\"align\\\":\\\"center\\\",\\\"paddingleft\\\":0},\\\"Link\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0},\\\"Link:Hover\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0}}\",\"sliderfont12customlabel\":\"Button dark\",\"sliderfont12\":\"{\\\"firsttab\\\":\\\"Text\\\",\\\"Text\\\":{\\\"color\\\":\\\"000000db\\\",\\\"size\\\":\\\"100||%\\\",\\\"tshadow\\\":\\\"0|*|1|*|0|*|ffffff33\\\",\\\"afont\\\":\\\"google(@import url(http:\\/\\/fonts.googleapis.com\\/css?family=Open Sans);),Arial\\\",\\\"lineheight\\\":\\\"1.3\\\",\\\"bold\\\":0,\\\"italic\\\":0,\\\"underline\\\":0,\\\"align\\\":\\\"center\\\",\\\"paddingleft\\\":0},\\\"Link\\\":{\\\"paddingleft\\\":0,\\\"size\\\":\\\"100||%\\\"},\\\"Link:Hover\\\":{\\\"paddingleft\\\":0,\\\"size\\\":\\\"100||%\\\"}}\",\"sliderfontcustom1customlabel\":\"My first custom font\",\"sliderfontcustom1\":\"{\\\"firsttab\\\":\\\"Text\\\",\\\"Text\\\":{\\\"color\\\":\\\"1abc9cff\\\",\\\"size\\\":\\\"360||%\\\",\\\"tshadow\\\":\\\"0|*|1|*|1|*|000000c7\\\",\\\"afont\\\":\\\"google(@import url(http:\\/\\/fonts.googleapis.com\\/css?family=Pacifico);),Arial\\\",\\\"lineheight\\\":\\\"1.3\\\",\\\"bold\\\":0,\\\"italic\\\":0,\\\"underline\\\":0,\\\"align\\\":\\\"left\\\",\\\"paddingleft\\\":0},\\\"Link\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0},\\\"Link:Hover\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0}}\",\"sliderfontcustom2customlabel\":\"My second custom font\",\"sliderfontcustom2\":\"{\\\"firsttab\\\":\\\"Text\\\",\\\"Text\\\":{\\\"color\\\":\\\"ffffffff\\\",\\\"size\\\":\\\"140||%\\\",\\\"tshadow\\\":\\\"0|*|1|*|1|*|000000c7\\\",\\\"afont\\\":\\\"google(@import url(http:\\/\\/fonts.googleapis.com\\/css?family=Open Sans);),Arial\\\",\\\"lineheight\\\":\\\"1.2\\\",\\\"bold\\\":0,\\\"italic\\\":0,\\\"underline\\\":0,\\\"align\\\":\\\"center\\\",\\\"paddingleft\\\":0},\\\"Link\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0},\\\"Link:Hover\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0}}\",\"sliderfontcustom3customlabel\":\"My third custom font\",\"sliderfontcustom3\":\"{\\\"firsttab\\\":\\\"Text\\\",\\\"Text\\\":{\\\"color\\\":\\\"1abc9cff\\\",\\\"size\\\":\\\"120||%\\\",\\\"tshadow\\\":\\\"0|*|1|*|1|*|000000c7\\\",\\\"afont\\\":\\\"google(@import url(http:\\/\\/fonts.googleapis.com\\/css?family=Open Sans);),Arial\\\",\\\"lineheight\\\":\\\"1.2\\\",\\\"bold\\\":0,\\\"italic\\\":0,\\\"underline\\\":0,\\\"align\\\":\\\"center\\\",\\\"paddingleft\\\":0},\\\"Link\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0},\\\"Link:Hover\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0}}\",\"sliderfontcustom4customlabel\":\"My fourthcustom font \",\"sliderfontcustom4\":\"{\\\"firsttab\\\":\\\"Text\\\",\\\"Text\\\":{\\\"color\\\":\\\"1abc9cff\\\",\\\"size\\\":\\\"120||%\\\",\\\"tshadow\\\":\\\"0|*|1|*|1|*|000000c7\\\",\\\"afont\\\":\\\"google(@import url(http:\\/\\/fonts.googleapis.com\\/css?family=Open Sans);),Arial\\\",\\\"lineheight\\\":\\\"1.2\\\",\\\"bold\\\":0,\\\"italic\\\":0,\\\"underline\\\":0,\\\"align\\\":\\\"right\\\",\\\"paddingleft\\\":0},\\\"Link\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0},\\\"Link:Hover\\\":{\\\"size\\\":\\\"100||%\\\",\\\"paddingleft\\\":0}}\"}');
/*!40000 ALTER TABLE `wp_nextend_smartslider_storage` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_options`
--

DROP TABLE IF EXISTS `wp_options`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_options` (
  `option_id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
  `option_name` varchar(64) NOT NULL DEFAULT '',
  `option_value` longtext NOT NULL,
  `autoload` varchar(20) NOT NULL DEFAULT 'yes',
  PRIMARY KEY (`option_id`),
  UNIQUE KEY `option_name` (`option_name`)
) ENGINE=InnoDB AUTO_INCREMENT=1441 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_options`
--

LOCK TABLES `wp_options` WRITE;
/*!40000 ALTER TABLE `wp_options` DISABLE KEYS */;
INSERT INTO `wp_options` VALUES (1,'siteurl','http://localhost/~psalessi/Genetics_Lab','yes'),(2,'blogname','Genomic Variation Laboratory','yes'),(3,'blogdescription','Conservation Genetics Fund','yes'),(4,'users_can_register','0','yes'),(5,'admin_email','pasalessi@ucdavis.edu','yes'),(6,'start_of_week','1','yes'),(7,'use_balanceTags','0','yes'),(8,'use_smilies','1','yes'),(9,'require_name_email','1','yes'),(10,'comments_notify','1','yes'),(11,'posts_per_rss','10','yes'),(12,'rss_use_excerpt','0','yes'),(13,'mailserver_url','mail.example.com','yes'),(14,'mailserver_login','login@example.com','yes'),(15,'mailserver_pass','password','yes'),(16,'mailserver_port','110','yes'),(17,'default_category','1','yes'),(18,'default_comment_status','closed','yes'),(19,'default_ping_status','open','yes'),(20,'default_pingback_flag','','yes'),(21,'posts_per_page','10','yes'),(22,'date_format','F j, Y','yes'),(23,'time_format','g:i a','yes'),(24,'links_updated_date_format','F j, Y g:i a','yes'),(28,'comment_moderation','','yes'),(29,'moderation_notify','1','yes'),(30,'permalink_structure','/%postname%/','yes'),(31,'gzipcompression','0','yes'),(32,'hack_file','0','yes'),(33,'blog_charset','UTF-8','yes'),(34,'moderation_keys','','no'),(35,'active_plugins','a:2:{i:0;s:30:\"advanced-custom-fields/acf.php\";i:1;s:39:\"slideshow-gallery/slideshow-gallery.php\";}','yes'),(36,'home','http://localhost/~psalessi/Genetics_Lab','yes'),(37,'category_base','','yes'),(38,'ping_sites','http://rpc.pingomatic.com/','yes'),(39,'advanced_edit','0','yes'),(40,'comment_max_links','2','yes'),(41,'gmt_offset','0','yes'),(42,'default_email_category','1','yes'),(43,'recently_edited','a:5:{i:0;s:89:\"/Users/psalessi/Sites/Genetics_Lab/wp-content/plugins/infinite-slider/infinite-slider.php\";i:1;s:84:\"/Users/psalessi/Sites/Genetics_Lab/wp-content/plugins/advanced-custom-fields/acf.php\";i:2;s:96:\"/Users/psalessi/Sites/Genetics_Lab/wp-content/plugins/wp-content-slideshow/content-slideshow.php\";i:3;s:85:\"/Users/psalessi/Sites/Genetics_Lab/wp-content/plugins/wp-content-slideshow/readme.txt\";i:4;s:99:\"/Users/psalessi/Sites/Genetics_Lab/wp-content/plugins/wp-content-slideshow/wp-content-slideshow.php\";}','no'),(44,'template','twentytwelve','yes'),(45,'stylesheet','child_of_2012','yes'),(46,'comment_whitelist','1','yes'),(47,'blacklist_keys','','no'),(48,'comment_registration','','yes'),(49,'html_type','text/html','yes'),(50,'use_trackback','0','yes'),(51,'default_role','subscriber','yes'),(52,'db_version','27916','yes'),(53,'uploads_use_yearmonth_folders','1','yes'),(54,'upload_path','','yes'),(55,'blog_public','0','yes'),(56,'default_link_category','2','yes'),(57,'show_on_front','page','yes'),(58,'tag_base','','yes'),(59,'show_avatars','','yes'),(60,'avatar_rating','G','yes'),(61,'upload_url_path','','yes'),(62,'thumbnail_size_w','150','yes'),(63,'thumbnail_size_h','150','yes'),(64,'thumbnail_crop','1','yes'),(65,'medium_size_w','300','yes'),(66,'medium_size_h','300','yes'),(67,'avatar_default','mystery','yes'),(68,'large_size_w','1024','yes'),(69,'large_size_h','1024','yes'),(70,'image_default_link_type','file','yes'),(71,'image_default_size','','yes'),(72,'image_default_align','','yes'),(73,'close_comments_for_old_posts','','yes'),(74,'close_comments_days_old','14','yes'),(75,'thread_comments','1','yes'),(76,'thread_comments_depth','5','yes'),(77,'page_comments','','yes'),(78,'comments_per_page','50','yes'),(79,'default_comments_page','newest','yes'),(80,'comment_order','asc','yes'),(81,'sticky_posts','a:0:{}','yes'),(82,'widget_categories','a:1:{s:12:\"_multiwidget\";i:1;}','yes'),(83,'widget_text','a:2:{i:4;a:3:{s:5:\"title\";s:0:\"\";s:4:\"text\";s:431:\"<a class=\"twitter-timeline\" href=\"https://twitter.com/search?q=ucdavis\" data-widget-id=\"476085765508591616\">Tweets about \"ucdavis\"</a>\r\n<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?\'http\':\'https\';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+\"://platform.twitter.com/widgets.js\";fjs.parentNode.insertBefore(js,fjs);}}(document,\"script\",\"twitter-wjs\");</script>\r\n\";s:6:\"filter\";b:0;}s:12:\"_multiwidget\";i:1;}','yes'),(84,'widget_rss','a:0:{}','yes'),(85,'uninstall_plugins','a:2:{s:23:\"len-slider/ls-index.php\";a:2:{i:0;s:9:\"LenSlider\";i:1;s:26:\"lenslider_plugin_uninstall\";}s:34:\"easing-slider/easingsliderlite.php\";a:2:{i:0;s:16:\"EasingSliderLite\";i:1;s:12:\"do_uninstall\";}}','no'),(86,'timezone_string','','yes'),(87,'page_for_posts','0','yes'),(88,'page_on_front','30','yes'),(89,'default_post_format','0','yes'),(90,'link_manager_enabled','0','yes'),(91,'initial_db_version','26691','yes'),(92,'wp_user_roles','a:5:{s:13:\"administrator\";a:2:{s:4:\"name\";s:13:\"Administrator\";s:12:\"capabilities\";a:65:{s:13:\"switch_themes\";b:1;s:11:\"edit_themes\";b:1;s:16:\"activate_plugins\";b:1;s:12:\"edit_plugins\";b:1;s:10:\"edit_users\";b:1;s:10:\"edit_files\";b:1;s:14:\"manage_options\";b:1;s:17:\"moderate_comments\";b:1;s:17:\"manage_categories\";b:1;s:12:\"manage_links\";b:1;s:12:\"upload_files\";b:1;s:6:\"import\";b:1;s:15:\"unfiltered_html\";b:1;s:10:\"edit_posts\";b:1;s:17:\"edit_others_posts\";b:1;s:20:\"edit_published_posts\";b:1;s:13:\"publish_posts\";b:1;s:10:\"edit_pages\";b:1;s:4:\"read\";b:1;s:8:\"level_10\";b:1;s:7:\"level_9\";b:1;s:7:\"level_8\";b:1;s:7:\"level_7\";b:1;s:7:\"level_6\";b:1;s:7:\"level_5\";b:1;s:7:\"level_4\";b:1;s:7:\"level_3\";b:1;s:7:\"level_2\";b:1;s:7:\"level_1\";b:1;s:7:\"level_0\";b:1;s:17:\"edit_others_pages\";b:1;s:20:\"edit_published_pages\";b:1;s:13:\"publish_pages\";b:1;s:12:\"delete_pages\";b:1;s:19:\"delete_others_pages\";b:1;s:22:\"delete_published_pages\";b:1;s:12:\"delete_posts\";b:1;s:19:\"delete_others_posts\";b:1;s:22:\"delete_published_posts\";b:1;s:20:\"delete_private_posts\";b:1;s:18:\"edit_private_posts\";b:1;s:18:\"read_private_posts\";b:1;s:20:\"delete_private_pages\";b:1;s:18:\"edit_private_pages\";b:1;s:18:\"read_private_pages\";b:1;s:12:\"delete_users\";b:1;s:12:\"create_users\";b:1;s:17:\"unfiltered_upload\";b:1;s:14:\"edit_dashboard\";b:1;s:14:\"update_plugins\";b:1;s:14:\"delete_plugins\";b:1;s:15:\"install_plugins\";b:1;s:13:\"update_themes\";b:1;s:14:\"install_themes\";b:1;s:11:\"update_core\";b:1;s:10:\"list_users\";b:1;s:12:\"remove_users\";b:1;s:9:\"add_users\";b:1;s:13:\"promote_users\";b:1;s:18:\"edit_theme_options\";b:1;s:13:\"delete_themes\";b:1;s:6:\"export\";b:1;s:16:\"slideshow_slides\";b:1;s:19:\"slideshow_galleries\";b:1;s:18:\"slideshow_settings\";b:1;}}s:6:\"editor\";a:2:{s:4:\"name\";s:6:\"Editor\";s:12:\"capabilities\";a:34:{s:17:\"moderate_comments\";b:1;s:17:\"manage_categories\";b:1;s:12:\"manage_links\";b:1;s:12:\"upload_files\";b:1;s:15:\"unfiltered_html\";b:1;s:10:\"edit_posts\";b:1;s:17:\"edit_others_posts\";b:1;s:20:\"edit_published_posts\";b:1;s:13:\"publish_posts\";b:1;s:10:\"edit_pages\";b:1;s:4:\"read\";b:1;s:7:\"level_7\";b:1;s:7:\"level_6\";b:1;s:7:\"level_5\";b:1;s:7:\"level_4\";b:1;s:7:\"level_3\";b:1;s:7:\"level_2\";b:1;s:7:\"level_1\";b:1;s:7:\"level_0\";b:1;s:17:\"edit_others_pages\";b:1;s:20:\"edit_published_pages\";b:1;s:13:\"publish_pages\";b:1;s:12:\"delete_pages\";b:1;s:19:\"delete_others_pages\";b:1;s:22:\"delete_published_pages\";b:1;s:12:\"delete_posts\";b:1;s:19:\"delete_others_posts\";b:1;s:22:\"delete_published_posts\";b:1;s:20:\"delete_private_posts\";b:1;s:18:\"edit_private_posts\";b:1;s:18:\"read_private_posts\";b:1;s:20:\"delete_private_pages\";b:1;s:18:\"edit_private_pages\";b:1;s:18:\"read_private_pages\";b:1;}}s:6:\"author\";a:2:{s:4:\"name\";s:6:\"Author\";s:12:\"capabilities\";a:10:{s:12:\"upload_files\";b:1;s:10:\"edit_posts\";b:1;s:20:\"edit_published_posts\";b:1;s:13:\"publish_posts\";b:1;s:4:\"read\";b:1;s:7:\"level_2\";b:1;s:7:\"level_1\";b:1;s:7:\"level_0\";b:1;s:12:\"delete_posts\";b:1;s:22:\"delete_published_posts\";b:1;}}s:11:\"contributor\";a:2:{s:4:\"name\";s:11:\"Contributor\";s:12:\"capabilities\";a:5:{s:10:\"edit_posts\";b:1;s:4:\"read\";b:1;s:7:\"level_1\";b:1;s:7:\"level_0\";b:1;s:12:\"delete_posts\";b:1;}}s:10:\"subscriber\";a:2:{s:4:\"name\";s:10:\"Subscriber\";s:12:\"capabilities\";a:2:{s:4:\"read\";b:1;s:7:\"level_0\";b:1;}}}','yes'),(93,'widget_search','a:1:{s:12:\"_multiwidget\";i:1;}','yes'),(94,'widget_recent-posts','a:1:{s:12:\"_multiwidget\";i:1;}','yes'),(95,'widget_recent-comments','a:1:{s:12:\"_multiwidget\";i:1;}','yes'),(96,'widget_archives','a:1:{s:12:\"_multiwidget\";i:1;}','yes'),(97,'widget_meta','a:1:{s:12:\"_multiwidget\";i:1;}','yes'),(98,'sidebars_widgets','a:5:{s:19:\"wp_inactive_widgets\";a:0:{}s:9:\"sidebar-1\";a:0:{}s:9:\"sidebar-2\";a:1:{i:0;s:6:\"text-4\";}s:9:\"sidebar-3\";a:0:{}s:13:\"array_version\";i:3;}','yes'),(99,'_transient_random_seed','92d20903064cb90368fc12891807438c','yes'),(100,'logged_in_key','B.*yS.@s,38af$G*>a9.C5>7V6C2xDFyM1X|a9<T+qYIHw83[~vw*dfrs&>)$9!.','yes'),(101,'logged_in_salt','2<[GUs6r@`G_tZjbNOw.@~~R,QVv~u2fA|a087(mV8c^N3Vx2^HoznW:V}p%}PF.','yes'),(102,'cron','a:5:{i:1402641662;a:3:{s:16:\"wp_version_check\";a:1:{s:32:\"40cd750bba9870f18aada2478b24840a\";a:3:{s:8:\"schedule\";s:10:\"twicedaily\";s:4:\"args\";a:0:{}s:8:\"interval\";i:43200;}}s:17:\"wp_update_plugins\";a:1:{s:32:\"40cd750bba9870f18aada2478b24840a\";a:3:{s:8:\"schedule\";s:10:\"twicedaily\";s:4:\"args\";a:0:{}s:8:\"interval\";i:43200;}}s:16:\"wp_update_themes\";a:1:{s:32:\"40cd750bba9870f18aada2478b24840a\";a:3:{s:8:\"schedule\";s:10:\"twicedaily\";s:4:\"args\";a:0:{}s:8:\"interval\";i:43200;}}}i:1402643640;a:1:{s:20:\"wp_maybe_auto_update\";a:1:{s:32:\"40cd750bba9870f18aada2478b24840a\";a:3:{s:8:\"schedule\";s:10:\"twicedaily\";s:4:\"args\";a:0:{}s:8:\"interval\";i:43200;}}}i:1402684871;a:1:{s:19:\"wp_scheduled_delete\";a:1:{s:32:\"40cd750bba9870f18aada2478b24840a\";a:3:{s:8:\"schedule\";s:5:\"daily\";s:4:\"args\";a:0:{}s:8:\"interval\";i:86400;}}}i:1402685481;a:1:{s:30:\"wp_scheduled_auto_draft_delete\";a:1:{s:32:\"40cd750bba9870f18aada2478b24840a\";a:3:{s:8:\"schedule\";s:5:\"daily\";s:4:\"args\";a:0:{}s:8:\"interval\";i:86400;}}}s:7:\"version\";i:2;}','yes'),(109,'auth_key','ACFL$:BICIEU:D{!;v_tn=5>:_G?+zvW%dgwLeRGL)tWOXB7r#+qQHb3tJ1;*;jC','yes'),(110,'auth_salt','P~;CBLP4J>n;ui+^%wD-6LbQ(Xhh?*%vr=IDx.|v/fg@x}<w`Hw*#(RaZxTiggA<','yes'),(111,'nonce_key','jjTq?[EKDv.Mn!jkhx=B<FRz>u!9 z1lkBJ$zVqF%`X:mWA&g(G};Yj&rDbXH%If','yes'),(112,'nonce_salt','f&TWX?~!,&d+gq(7e)rQf)IgRgstQ^V2CP|liu6C/:XGH%G}U-4-xO#f 9$vFD~Z','yes'),(133,'theme_mods_twentyfourteen','a:1:{s:16:\"sidebars_widgets\";a:2:{s:4:\"time\";i:1392057897;s:4:\"data\";a:4:{s:19:\"wp_inactive_widgets\";a:0:{}s:9:\"sidebar-1\";a:6:{i:0;s:8:\"search-2\";i:1;s:14:\"recent-posts-2\";i:2;s:17:\"recent-comments-2\";i:3;s:10:\"archives-2\";i:4;s:12:\"categories-2\";i:5;s:6:\"meta-2\";}s:9:\"sidebar-2\";a:0:{}s:9:\"sidebar-3\";a:0:{}}}}','yes'),(134,'current_theme','Child of Twenty Twelve','yes'),(135,'theme_mods_child_of_2012','a:10:{i:0;b:0;s:12:\"header_image\";s:103:\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/cropped-genetics_lab_home_header.png\";s:18:\"nav_menu_locations\";a:2:{s:7:\"primary\";i:2;s:9:\"secondary\";i:4;}s:16:\"header_textcolor\";s:6:\"7f7f7f\";s:16:\"background_color\";s:6:\"e6e6e6\";s:16:\"background_image\";s:0:\"\";s:17:\"background_repeat\";s:9:\"no-repeat\";s:21:\"background_position_x\";s:6:\"center\";s:21:\"background_attachment\";s:6:\"scroll\";s:17:\"header_image_data\";a:5:{s:13:\"attachment_id\";i:20;s:3:\"url\";s:103:\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/cropped-genetics_lab_home_header.png\";s:13:\"thumbnail_url\";s:103:\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/cropped-genetics_lab_home_header.png\";s:5:\"width\";i:700;s:6:\"height\";i:182;}}','yes'),(136,'theme_switched','','yes'),(141,'nav_menu_options','a:2:{i:0;b:0;s:8:\"auto_add\";a:0:{}}','yes'),(148,'theme_mods_twentytwelve','a:4:{i:0;b:0;s:12:\"header_image\";s:104:\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/cropped-genetics_lab_home_header1.png\";s:17:\"header_image_data\";O:8:\"stdClass\":5:{s:13:\"attachment_id\";i:32;s:3:\"url\";s:104:\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/cropped-genetics_lab_home_header1.png\";s:13:\"thumbnail_url\";s:104:\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/cropped-genetics_lab_home_header1.png\";s:6:\"height\";i:182;s:5:\"width\";i:700;}s:16:\"sidebars_widgets\";a:2:{s:4:\"time\";i:1393612345;s:4:\"data\";a:4:{s:19:\"wp_inactive_widgets\";a:0:{}s:9:\"sidebar-1\";a:0:{}s:9:\"sidebar-2\";a:0:{}s:9:\"sidebar-3\";a:0:{}}}}','yes'),(215,'recently_activated','a:16:{s:33:\"show-sliders/atw-show-sliders.php\";i:1402603791;s:31:\"smooth-slider/smooth-slider.php\";i:1402603682;s:35:\"smooth-dynamic-slider/sdsslider.php\";i:1402603654;s:23:\"wowslider/wowslider.php\";i:1402603579;s:35:\"cyclone-slider-2/cyclone-slider.php\";i:1402603179;s:34:\"easing-slider/easingsliderlite.php\";i:1402603103;s:35:\"infinite-slider/infinite-slider.php\";i:1402600457;s:41:\"easyrotator-for-wordpress/easyrotator.php\";i:1402007291;s:38:\"meteor-slides/meteor-slides-plugin.php\";i:1402006810;s:30:\"slidedeck2/slidedeck2-lite.php\";i:1402006446;s:23:\"ml-slider/ml-slider.php\";i:1402006406;s:45:\"wp-content-slideshow/wp-content-slideshow.php\";i:1402006058;s:29:\"addi-simple-slider/slider.php\";i:1402003379;s:23:\"len-slider/ls-index.php\";i:1402001435;s:29:\"arconix-flexslider/plugin.php\";i:1402000466;s:31:\"simple-slider-ssp/wp-slider.php\";i:1402000197;}','yes'),(219,'acf_version','4.3.8','yes'),(499,'category_children','a:0:{}','yes'),(854,'theme_mods_child_of_2011','a:2:{i:0;b:0;s:16:\"sidebars_widgets\";a:2:{s:4:\"time\";i:1394643070;s:4:\"data\";a:6:{s:19:\"wp_inactive_widgets\";a:0:{}s:9:\"sidebar-1\";a:0:{}s:9:\"sidebar-2\";a:0:{}s:9:\"sidebar-3\";a:0:{}s:9:\"sidebar-4\";N;s:9:\"sidebar-5\";N;}}}','yes'),(857,'theme_mods_responsive','a:2:{i:0;b:0;s:16:\"sidebars_widgets\";a:2:{s:4:\"time\";i:1394642020;s:4:\"data\";a:13:{s:19:\"wp_inactive_widgets\";a:0:{}s:12:\"main-sidebar\";a:0:{}s:13:\"right-sidebar\";a:0:{}s:12:\"left-sidebar\";a:0:{}s:17:\"left-sidebar-half\";N;s:18:\"right-sidebar-half\";N;s:13:\"home-widget-1\";N;s:13:\"home-widget-2\";N;s:13:\"home-widget-3\";N;s:14:\"gallery-widget\";N;s:15:\"colophon-widget\";N;s:10:\"top-widget\";N;s:13:\"footer-widget\";N;}}}','yes'),(858,'responsive_theme_options','a:34:{s:10:\"breadcrumb\";b:0;s:10:\"cta_button\";b:0;s:12:\"minified_css\";b:0;s:10:\"front_page\";i:1;s:13:\"home_headline\";N;s:16:\"home_subheadline\";N;s:17:\"home_content_area\";N;s:8:\"cta_text\";N;s:7:\"cta_url\";N;s:16:\"featured_content\";N;s:24:\"google_site_verification\";s:0:\"\";s:22:\"bing_site_verification\";s:0:\"\";s:23:\"yahoo_site_verification\";s:0:\"\";s:23:\"site_statistics_tracker\";s:0:\"\";s:11:\"twitter_uid\";s:0:\"\";s:12:\"facebook_uid\";s:0:\"\";s:12:\"linkedin_uid\";s:0:\"\";s:11:\"youtube_uid\";s:0:\"\";s:11:\"stumble_uid\";s:0:\"\";s:7:\"rss_uid\";s:0:\"\";s:15:\"google_plus_uid\";s:0:\"\";s:13:\"instagram_uid\";s:0:\"\";s:13:\"pinterest_uid\";s:0:\"\";s:8:\"yelp_uid\";s:0:\"\";s:9:\"vimeo_uid\";s:0:\"\";s:14:\"foursquare_uid\";s:0:\"\";s:21:\"responsive_inline_css\";s:0:\"\";s:25:\"responsive_inline_js_head\";s:0:\"\";s:31:\"responsive_inline_css_js_footer\";s:0:\"\";s:26:\"static_page_layout_default\";s:20:\"content-sidebar-page\";s:26:\"single_post_layout_default\";s:20:\"content-sidebar-page\";s:31:\"blog_posts_index_layout_default\";s:20:\"content-sidebar-page\";s:14:\"googleplus_uid\";s:0:\"\";s:15:\"stumbleupon_uid\";s:0:\"\";}','yes'),(859,'theme_mods_child_of_responsive','a:2:{i:0;b:0;s:16:\"sidebars_widgets\";a:2:{s:4:\"time\";i:1394642119;s:4:\"data\";a:13:{s:19:\"wp_inactive_widgets\";a:0:{}s:12:\"main-sidebar\";a:0:{}s:13:\"right-sidebar\";a:0:{}s:12:\"left-sidebar\";a:0:{}s:17:\"left-sidebar-half\";N;s:18:\"right-sidebar-half\";N;s:13:\"home-widget-1\";N;s:13:\"home-widget-2\";N;s:13:\"home-widget-3\";N;s:14:\"gallery-widget\";N;s:15:\"colophon-widget\";N;s:10:\"top-widget\";N;s:13:\"footer-widget\";N;}}}','yes'),(1098,'auto_core_update_notified','a:4:{s:4:\"type\";s:7:\"success\";s:5:\"email\";s:21:\"pasalessi@ucdavis.edu\";s:7:\"version\";s:5:\"3.9.1\";s:9:\"timestamp\";i:1401233119;}','yes'),(1101,'db_upgraded','','yes'),(1104,'rewrite_rules','a:108:{s:9:\"people/?$\";s:26:\"index.php?post_type=people\";s:39:\"people/feed/(feed|rdf|rss|rss2|atom)/?$\";s:43:\"index.php?post_type=people&feed=$matches[1]\";s:34:\"people/(feed|rdf|rss|rss2|atom)/?$\";s:43:\"index.php?post_type=people&feed=$matches[1]\";s:26:\"people/page/([0-9]{1,})/?$\";s:44:\"index.php?post_type=people&paged=$matches[1]\";s:11:\"projects/?$\";s:28:\"index.php?post_type=projects\";s:41:\"projects/feed/(feed|rdf|rss|rss2|atom)/?$\";s:45:\"index.php?post_type=projects&feed=$matches[1]\";s:36:\"projects/(feed|rdf|rss|rss2|atom)/?$\";s:45:\"index.php?post_type=projects&feed=$matches[1]\";s:28:\"projects/page/([0-9]{1,})/?$\";s:46:\"index.php?post_type=projects&paged=$matches[1]\";s:47:\"category/(.+?)/feed/(feed|rdf|rss|rss2|atom)/?$\";s:52:\"index.php?category_name=$matches[1]&feed=$matches[2]\";s:42:\"category/(.+?)/(feed|rdf|rss|rss2|atom)/?$\";s:52:\"index.php?category_name=$matches[1]&feed=$matches[2]\";s:35:\"category/(.+?)/page/?([0-9]{1,})/?$\";s:53:\"index.php?category_name=$matches[1]&paged=$matches[2]\";s:17:\"category/(.+?)/?$\";s:35:\"index.php?category_name=$matches[1]\";s:44:\"tag/([^/]+)/feed/(feed|rdf|rss|rss2|atom)/?$\";s:42:\"index.php?tag=$matches[1]&feed=$matches[2]\";s:39:\"tag/([^/]+)/(feed|rdf|rss|rss2|atom)/?$\";s:42:\"index.php?tag=$matches[1]&feed=$matches[2]\";s:32:\"tag/([^/]+)/page/?([0-9]{1,})/?$\";s:43:\"index.php?tag=$matches[1]&paged=$matches[2]\";s:14:\"tag/([^/]+)/?$\";s:25:\"index.php?tag=$matches[1]\";s:45:\"type/([^/]+)/feed/(feed|rdf|rss|rss2|atom)/?$\";s:50:\"index.php?post_format=$matches[1]&feed=$matches[2]\";s:40:\"type/([^/]+)/(feed|rdf|rss|rss2|atom)/?$\";s:50:\"index.php?post_format=$matches[1]&feed=$matches[2]\";s:33:\"type/([^/]+)/page/?([0-9]{1,})/?$\";s:51:\"index.php?post_format=$matches[1]&paged=$matches[2]\";s:15:\"type/([^/]+)/?$\";s:33:\"index.php?post_format=$matches[1]\";s:34:\"people/[^/]+/attachment/([^/]+)/?$\";s:32:\"index.php?attachment=$matches[1]\";s:44:\"people/[^/]+/attachment/([^/]+)/trackback/?$\";s:37:\"index.php?attachment=$matches[1]&tb=1\";s:64:\"people/[^/]+/attachment/([^/]+)/feed/(feed|rdf|rss|rss2|atom)/?$\";s:49:\"index.php?attachment=$matches[1]&feed=$matches[2]\";s:59:\"people/[^/]+/attachment/([^/]+)/(feed|rdf|rss|rss2|atom)/?$\";s:49:\"index.php?attachment=$matches[1]&feed=$matches[2]\";s:59:\"people/[^/]+/attachment/([^/]+)/comment-page-([0-9]{1,})/?$\";s:50:\"index.php?attachment=$matches[1]&cpage=$matches[2]\";s:27:\"people/([^/]+)/trackback/?$\";s:33:\"index.php?people=$matches[1]&tb=1\";s:47:\"people/([^/]+)/feed/(feed|rdf|rss|rss2|atom)/?$\";s:45:\"index.php?people=$matches[1]&feed=$matches[2]\";s:42:\"people/([^/]+)/(feed|rdf|rss|rss2|atom)/?$\";s:45:\"index.php?people=$matches[1]&feed=$matches[2]\";s:35:\"people/([^/]+)/page/?([0-9]{1,})/?$\";s:46:\"index.php?people=$matches[1]&paged=$matches[2]\";s:42:\"people/([^/]+)/comment-page-([0-9]{1,})/?$\";s:46:\"index.php?people=$matches[1]&cpage=$matches[2]\";s:27:\"people/([^/]+)(/[0-9]+)?/?$\";s:45:\"index.php?people=$matches[1]&page=$matches[2]\";s:23:\"people/[^/]+/([^/]+)/?$\";s:32:\"index.php?attachment=$matches[1]\";s:33:\"people/[^/]+/([^/]+)/trackback/?$\";s:37:\"index.php?attachment=$matches[1]&tb=1\";s:53:\"people/[^/]+/([^/]+)/feed/(feed|rdf|rss|rss2|atom)/?$\";s:49:\"index.php?attachment=$matches[1]&feed=$matches[2]\";s:48:\"people/[^/]+/([^/]+)/(feed|rdf|rss|rss2|atom)/?$\";s:49:\"index.php?attachment=$matches[1]&feed=$matches[2]\";s:48:\"people/[^/]+/([^/]+)/comment-page-([0-9]{1,})/?$\";s:50:\"index.php?attachment=$matches[1]&cpage=$matches[2]\";s:36:\"projects/[^/]+/attachment/([^/]+)/?$\";s:32:\"index.php?attachment=$matches[1]\";s:46:\"projects/[^/]+/attachment/([^/]+)/trackback/?$\";s:37:\"index.php?attachment=$matches[1]&tb=1\";s:66:\"projects/[^/]+/attachment/([^/]+)/feed/(feed|rdf|rss|rss2|atom)/?$\";s:49:\"index.php?attachment=$matches[1]&feed=$matches[2]\";s:61:\"projects/[^/]+/attachment/([^/]+)/(feed|rdf|rss|rss2|atom)/?$\";s:49:\"index.php?attachment=$matches[1]&feed=$matches[2]\";s:61:\"projects/[^/]+/attachment/([^/]+)/comment-page-([0-9]{1,})/?$\";s:50:\"index.php?attachment=$matches[1]&cpage=$matches[2]\";s:29:\"projects/([^/]+)/trackback/?$\";s:35:\"index.php?projects=$matches[1]&tb=1\";s:49:\"projects/([^/]+)/feed/(feed|rdf|rss|rss2|atom)/?$\";s:47:\"index.php?projects=$matches[1]&feed=$matches[2]\";s:44:\"projects/([^/]+)/(feed|rdf|rss|rss2|atom)/?$\";s:47:\"index.php?projects=$matches[1]&feed=$matches[2]\";s:37:\"projects/([^/]+)/page/?([0-9]{1,})/?$\";s:48:\"index.php?projects=$matches[1]&paged=$matches[2]\";s:44:\"projects/([^/]+)/comment-page-([0-9]{1,})/?$\";s:48:\"index.php?projects=$matches[1]&cpage=$matches[2]\";s:29:\"projects/([^/]+)(/[0-9]+)?/?$\";s:47:\"index.php?projects=$matches[1]&page=$matches[2]\";s:25:\"projects/[^/]+/([^/]+)/?$\";s:32:\"index.php?attachment=$matches[1]\";s:35:\"projects/[^/]+/([^/]+)/trackback/?$\";s:37:\"index.php?attachment=$matches[1]&tb=1\";s:55:\"projects/[^/]+/([^/]+)/feed/(feed|rdf|rss|rss2|atom)/?$\";s:49:\"index.php?attachment=$matches[1]&feed=$matches[2]\";s:50:\"projects/[^/]+/([^/]+)/(feed|rdf|rss|rss2|atom)/?$\";s:49:\"index.php?attachment=$matches[1]&feed=$matches[2]\";s:50:\"projects/[^/]+/([^/]+)/comment-page-([0-9]{1,})/?$\";s:50:\"index.php?attachment=$matches[1]&cpage=$matches[2]\";s:48:\".*wp-(atom|rdf|rss|rss2|feed|commentsrss2)\\.php$\";s:18:\"index.php?feed=old\";s:20:\".*wp-app\\.php(/.*)?$\";s:19:\"index.php?error=403\";s:18:\".*wp-register.php$\";s:23:\"index.php?register=true\";s:32:\"feed/(feed|rdf|rss|rss2|atom)/?$\";s:27:\"index.php?&feed=$matches[1]\";s:27:\"(feed|rdf|rss|rss2|atom)/?$\";s:27:\"index.php?&feed=$matches[1]\";s:20:\"page/?([0-9]{1,})/?$\";s:28:\"index.php?&paged=$matches[1]\";s:27:\"comment-page-([0-9]{1,})/?$\";s:39:\"index.php?&page_id=30&cpage=$matches[1]\";s:41:\"comments/feed/(feed|rdf|rss|rss2|atom)/?$\";s:42:\"index.php?&feed=$matches[1]&withcomments=1\";s:36:\"comments/(feed|rdf|rss|rss2|atom)/?$\";s:42:\"index.php?&feed=$matches[1]&withcomments=1\";s:44:\"search/(.+)/feed/(feed|rdf|rss|rss2|atom)/?$\";s:40:\"index.php?s=$matches[1]&feed=$matches[2]\";s:39:\"search/(.+)/(feed|rdf|rss|rss2|atom)/?$\";s:40:\"index.php?s=$matches[1]&feed=$matches[2]\";s:32:\"search/(.+)/page/?([0-9]{1,})/?$\";s:41:\"index.php?s=$matches[1]&paged=$matches[2]\";s:14:\"search/(.+)/?$\";s:23:\"index.php?s=$matches[1]\";s:47:\"author/([^/]+)/feed/(feed|rdf|rss|rss2|atom)/?$\";s:50:\"index.php?author_name=$matches[1]&feed=$matches[2]\";s:42:\"author/([^/]+)/(feed|rdf|rss|rss2|atom)/?$\";s:50:\"index.php?author_name=$matches[1]&feed=$matches[2]\";s:35:\"author/([^/]+)/page/?([0-9]{1,})/?$\";s:51:\"index.php?author_name=$matches[1]&paged=$matches[2]\";s:17:\"author/([^/]+)/?$\";s:33:\"index.php?author_name=$matches[1]\";s:69:\"([0-9]{4})/([0-9]{1,2})/([0-9]{1,2})/feed/(feed|rdf|rss|rss2|atom)/?$\";s:80:\"index.php?year=$matches[1]&monthnum=$matches[2]&day=$matches[3]&feed=$matches[4]\";s:64:\"([0-9]{4})/([0-9]{1,2})/([0-9]{1,2})/(feed|rdf|rss|rss2|atom)/?$\";s:80:\"index.php?year=$matches[1]&monthnum=$matches[2]&day=$matches[3]&feed=$matches[4]\";s:57:\"([0-9]{4})/([0-9]{1,2})/([0-9]{1,2})/page/?([0-9]{1,})/?$\";s:81:\"index.php?year=$matches[1]&monthnum=$matches[2]&day=$matches[3]&paged=$matches[4]\";s:39:\"([0-9]{4})/([0-9]{1,2})/([0-9]{1,2})/?$\";s:63:\"index.php?year=$matches[1]&monthnum=$matches[2]&day=$matches[3]\";s:56:\"([0-9]{4})/([0-9]{1,2})/feed/(feed|rdf|rss|rss2|atom)/?$\";s:64:\"index.php?year=$matches[1]&monthnum=$matches[2]&feed=$matches[3]\";s:51:\"([0-9]{4})/([0-9]{1,2})/(feed|rdf|rss|rss2|atom)/?$\";s:64:\"index.php?year=$matches[1]&monthnum=$matches[2]&feed=$matches[3]\";s:44:\"([0-9]{4})/([0-9]{1,2})/page/?([0-9]{1,})/?$\";s:65:\"index.php?year=$matches[1]&monthnum=$matches[2]&paged=$matches[3]\";s:26:\"([0-9]{4})/([0-9]{1,2})/?$\";s:47:\"index.php?year=$matches[1]&monthnum=$matches[2]\";s:43:\"([0-9]{4})/feed/(feed|rdf|rss|rss2|atom)/?$\";s:43:\"index.php?year=$matches[1]&feed=$matches[2]\";s:38:\"([0-9]{4})/(feed|rdf|rss|rss2|atom)/?$\";s:43:\"index.php?year=$matches[1]&feed=$matches[2]\";s:31:\"([0-9]{4})/page/?([0-9]{1,})/?$\";s:44:\"index.php?year=$matches[1]&paged=$matches[2]\";s:13:\"([0-9]{4})/?$\";s:26:\"index.php?year=$matches[1]\";s:27:\".?.+?/attachment/([^/]+)/?$\";s:32:\"index.php?attachment=$matches[1]\";s:37:\".?.+?/attachment/([^/]+)/trackback/?$\";s:37:\"index.php?attachment=$matches[1]&tb=1\";s:57:\".?.+?/attachment/([^/]+)/feed/(feed|rdf|rss|rss2|atom)/?$\";s:49:\"index.php?attachment=$matches[1]&feed=$matches[2]\";s:52:\".?.+?/attachment/([^/]+)/(feed|rdf|rss|rss2|atom)/?$\";s:49:\"index.php?attachment=$matches[1]&feed=$matches[2]\";s:52:\".?.+?/attachment/([^/]+)/comment-page-([0-9]{1,})/?$\";s:50:\"index.php?attachment=$matches[1]&cpage=$matches[2]\";s:20:\"(.?.+?)/trackback/?$\";s:35:\"index.php?pagename=$matches[1]&tb=1\";s:40:\"(.?.+?)/feed/(feed|rdf|rss|rss2|atom)/?$\";s:47:\"index.php?pagename=$matches[1]&feed=$matches[2]\";s:35:\"(.?.+?)/(feed|rdf|rss|rss2|atom)/?$\";s:47:\"index.php?pagename=$matches[1]&feed=$matches[2]\";s:28:\"(.?.+?)/page/?([0-9]{1,})/?$\";s:48:\"index.php?pagename=$matches[1]&paged=$matches[2]\";s:35:\"(.?.+?)/comment-page-([0-9]{1,})/?$\";s:48:\"index.php?pagename=$matches[1]&cpage=$matches[2]\";s:20:\"(.?.+?)(/[0-9]+)?/?$\";s:47:\"index.php?pagename=$matches[1]&page=$matches[2]\";s:27:\"[^/]+/attachment/([^/]+)/?$\";s:32:\"index.php?attachment=$matches[1]\";s:37:\"[^/]+/attachment/([^/]+)/trackback/?$\";s:37:\"index.php?attachment=$matches[1]&tb=1\";s:57:\"[^/]+/attachment/([^/]+)/feed/(feed|rdf|rss|rss2|atom)/?$\";s:49:\"index.php?attachment=$matches[1]&feed=$matches[2]\";s:52:\"[^/]+/attachment/([^/]+)/(feed|rdf|rss|rss2|atom)/?$\";s:49:\"index.php?attachment=$matches[1]&feed=$matches[2]\";s:52:\"[^/]+/attachment/([^/]+)/comment-page-([0-9]{1,})/?$\";s:50:\"index.php?attachment=$matches[1]&cpage=$matches[2]\";s:20:\"([^/]+)/trackback/?$\";s:31:\"index.php?name=$matches[1]&tb=1\";s:40:\"([^/]+)/feed/(feed|rdf|rss|rss2|atom)/?$\";s:43:\"index.php?name=$matches[1]&feed=$matches[2]\";s:35:\"([^/]+)/(feed|rdf|rss|rss2|atom)/?$\";s:43:\"index.php?name=$matches[1]&feed=$matches[2]\";s:28:\"([^/]+)/page/?([0-9]{1,})/?$\";s:44:\"index.php?name=$matches[1]&paged=$matches[2]\";s:35:\"([^/]+)/comment-page-([0-9]{1,})/?$\";s:44:\"index.php?name=$matches[1]&cpage=$matches[2]\";s:20:\"([^/]+)(/[0-9]+)?/?$\";s:43:\"index.php?name=$matches[1]&page=$matches[2]\";s:16:\"[^/]+/([^/]+)/?$\";s:32:\"index.php?attachment=$matches[1]\";s:26:\"[^/]+/([^/]+)/trackback/?$\";s:37:\"index.php?attachment=$matches[1]&tb=1\";s:46:\"[^/]+/([^/]+)/feed/(feed|rdf|rss|rss2|atom)/?$\";s:49:\"index.php?attachment=$matches[1]&feed=$matches[2]\";s:41:\"[^/]+/([^/]+)/(feed|rdf|rss|rss2|atom)/?$\";s:49:\"index.php?attachment=$matches[1]&feed=$matches[2]\";s:41:\"[^/]+/([^/]+)/comment-page-([0-9]{1,})/?$\";s:50:\"index.php?attachment=$matches[1]&cpage=$matches[2]\";}','yes'),(1105,'can_compress_scripts','1','yes'),(1106,'_site_transient_timeout_wordpress_credits_en_US','1399500058','yes'),(1107,'_site_transient_wordpress_credits_en_US','a:2:{s:6:\"groups\";a:6:{s:15:\"project-leaders\";a:4:{s:4:\"name\";s:15:\"Project Leaders\";s:4:\"type\";s:6:\"titles\";s:7:\"shuffle\";b:1;s:4:\"data\";a:6:{s:4:\"ryan\";a:4:{i:0;s:10:\"Ryan Boren\";i:1;s:32:\"c22398fb9602c967d1dac8174f4a1a4e\";i:2;s:4:\"ryan\";i:3;s:14:\"Lead Developer\";}s:11:\"markjaquith\";a:4:{i:0;s:12:\"Mark Jaquith\";i:1;s:32:\"097a87a525e317519b5ee124820012fb\";i:2;s:11:\"markjaquith\";i:3;s:14:\"Lead Developer\";}s:4:\"matt\";a:4:{i:0;s:14:\"Matt Mullenweg\";i:1;s:32:\"767fc9c115a1b989744c755db47feb60\";i:2;s:4:\"matt\";i:3;s:23:\"Cofounder, Project Lead\";}s:5:\"nacin\";a:4:{i:0;s:12:\"Andrew Nacin\";i:1;s:32:\"01cfe9feaafb068590891bbd1f6a7f5a\";i:2;s:5:\"nacin\";i:3;s:14:\"Lead Developer\";}s:6:\"azaozz\";a:4:{i:0;s:10:\"Andrew Ozz\";i:1;s:32:\"4e84843ebff0918d72ade21c6ee7b1e4\";i:2;s:6:\"azaozz\";i:3;s:14:\"Lead Developer\";}s:5:\"westi\";a:4:{i:0;s:14:\"Peter Westwood\";i:1;s:32:\"5b3b5ce04dd402124aba53142b3e47f6\";i:2;s:5:\"westi\";i:3;s:14:\"Lead Developer\";}}}s:15:\"core-developers\";a:4:{s:4:\"name\";s:23:\"Contributing Developers\";s:4:\"type\";s:6:\"titles\";s:7:\"shuffle\";b:0;s:4:\"data\";a:6:{s:4:\"dd32\";a:4:{i:0;s:10:\"Dion Hulse\";i:1;s:32:\"fa63bbe285f64ba10985551154474e25\";i:2;s:4:\"dd32\";i:3;s:14:\"Core Developer\";}s:4:\"koop\";a:4:{i:0;s:17:\"Daryl Koopersmith\";i:1;s:32:\"26671c19c0d5722d0b1c101fd9d35175\";i:2;s:4:\"koop\";i:3;s:14:\"Core Developer\";}s:5:\"duck_\";a:4:{i:0;s:8:\"Jon Cave\";i:1;s:32:\"c422d58d80cf37281d34221f21a4a557\";i:2;s:5:\"duck_\";i:3;s:14:\"Core Developer\";}s:5:\"helen\";a:4:{i:0;s:16:\"Helen Hou-Sand\";i:1;s:32:\"6506162ada6b700b151ad8a187f65842\";i:2;s:5:\"helen\";i:3;s:14:\"Core Developer\";}s:7:\"ocean90\";a:4:{i:0;s:17:\"Dominik Schilling\";i:1;s:32:\"3e8e161d97d793bd8fc2dcd62583bb76\";i:2;s:7:\"ocean90\";i:3;s:0:\"\";}s:14:\"SergeyBiryukov\";a:4:{i:0;s:15:\"Sergey Biryukov\";i:1;s:32:\"750b7b0fcd855389264c2b1294d61bd6\";i:2;s:14:\"SergeyBiryukov\";i:3;s:0:\"\";}}}s:23:\"contributing-developers\";a:4:{s:4:\"name\";b:0;s:4:\"type\";s:6:\"titles\";s:7:\"shuffle\";b:1;s:4:\"data\";a:6:{s:14:\"wonderboymusic\";a:4:{i:0;s:12:\"Scott Taylor\";i:1;s:32:\"0e9fc8ea22885ed718ee37d31dcffb8c\";i:2;s:14:\"wonderboymusic\";i:3;s:0:\"\";}s:11:\"johnbillion\";a:4:{i:0;s:15:\"John Blackbourn\";i:1;s:32:\"0000ba6dd1b089e1746abbfe6281ee3b\";i:2;s:11:\"johnbillion\";i:3;s:0:\"\";}s:12:\"DrewAPicture\";a:4:{i:0;s:11:\"Drew Jaynes\";i:1;s:32:\"95c934fa2c3362794bf62ff8c59ada08\";i:2;s:12:\"DrewAPicture\";i:3;s:0:\"\";}s:12:\"lancewillett\";a:4:{i:0;s:13:\"Lance Willett\";i:1;s:32:\"ddc339d9565077190a08a9e5da7a5f44\";i:2;s:12:\"lancewillett\";i:3;s:0:\"\";}s:6:\"gcorne\";a:4:{i:0;s:17:\"Gregory Cornelius\";i:1;s:32:\"16836597f6d030b7ea01750a33e058df\";i:2;s:6:\"gcorne\";i:3;s:0:\"\";}s:11:\"DH-Shredder\";a:4:{i:0;s:13:\"Mike Schroder\";i:1;s:32:\"76424a001dc6b3ebb4faca0c567800c4\";i:2;s:11:\"DH-Shredder\";i:3;s:0:\"\";}}}s:16:\"recent-rockstars\";a:4:{s:4:\"name\";b:0;s:4:\"type\";s:6:\"titles\";s:7:\"shuffle\";b:1;s:4:\"data\";a:18:{s:15:\"adamsilverstein\";a:4:{i:0;s:16:\"Adam Silverstein\";i:1;s:32:\"08001a0a424aed1382375a446ed690d6\";i:2;s:15:\"adamsilverstein\";i:3;s:0:\"\";}s:5:\"avryl\";a:4:{i:0;s:17:\"Janneke Van Dorpe\";i:1;s:32:\"7c9d3a0262734566951c66a12c9a7720\";i:2;s:5:\"avryl\";i:3;s:0:\"\";}s:3:\"ehg\";a:4:{i:0;s:12:\"Chris Blower\";i:1;s:32:\"dac3fa4f660db27bff792344b3d1df34\";i:2;s:3:\"ehg\";i:3;s:0:\"\";}s:13:\"EmpireOfLight\";a:4:{i:0;s:10:\"Ben Dunkle\";i:1;s:32:\"1772cd53f2747f71c8455ac820606afc\";i:2;s:13:\"EmpireOfLight\";i:3;s:0:\"\";}s:8:\"ethitter\";a:4:{i:0;s:12:\"Erick Hitter\";i:1;s:32:\"fcf17d516139b240f405706d6fc8fc10\";i:2;s:8:\"ethitter\";i:3;s:0:\"\";}s:10:\"jeremyfelt\";a:4:{i:0;s:11:\"Jeremy Felt\";i:1;s:32:\"d1759b1c669981b7c52ec9a97d19e6bd\";i:2;s:10:\"jeremyfelt\";i:3;s:0:\"\";}s:6:\"jorbin\";a:4:{i:0;s:12:\"Aaron Jorbin\";i:1;s:32:\"b3e2b94eb305bf95a1bee11bc7705fb7\";i:2;s:6:\"jorbin\";i:3;s:0:\"\";}s:9:\"kovshenin\";a:4:{i:0;s:20:\"Konstantin Kovshenin\";i:1;s:32:\"8585b20b69e8ff4b0ba737accfdbb31b\";i:2;s:9:\"kovshenin\";i:3;s:0:\"\";}s:8:\"kpdesign\";a:4:{i:0;s:11:\"Kim Parsell\";i:1;s:32:\"c2f7f079b6d2610c97715b55f12b5ba5\";i:2;s:8:\"kpdesign\";i:3;s:0:\"\";}s:6:\"matveb\";a:4:{i:0;s:15:\"Matas Ventura\";i:1;s:32:\"3f8389957d0c59bb0bd5433e86b53a19\";i:2;s:6:\"matveb\";i:3;s:0:\"\";}s:4:\"mcsf\";a:4:{i:0;s:14:\"Miguel Fonseca\";i:1;s:32:\"7e89b8105f7e05afa2872e411d3f76ad\";i:2;s:4:\"mcsf\";i:3;s:0:\"\";}s:9:\"melchoyce\";a:4:{i:0;s:10:\"Mel Choyce\";i:1;s:32:\"2dc245d4cf9e5579dfb0e13e7dcb75db\";i:2;s:9:\"melchoyce\";i:3;s:0:\"\";}s:15:\"michael-arestad\";a:4:{i:0;s:15:\"Michael Arestad\";i:1;s:32:\"46e0f1b10224bb4e832ea9633acee2b8\";i:2;s:15:\"michael-arestad\";i:3;s:0:\"\";}s:8:\"obenland\";a:4:{i:0;s:19:\"Konstantin Obenland\";i:1;s:32:\"2370ea5912750f4cb0f3c51ae1cbca55\";i:2;s:8:\"obenland\";i:3;s:0:\"\";}s:5:\"pento\";a:4:{i:0;s:15:\"Gary Pendergast\";i:1;s:32:\"1ad9e5c98d81c6815a65dab5b6e1f669\";i:2;s:5:\"pento\";i:3;s:0:\"\";}s:12:\"shaunandrews\";a:4:{i:0;s:13:\"Shaun Andrews\";i:1;s:32:\"b7fdd6477cc13ca16e8358a0725bc02c\";i:2;s:12:\"shaunandrews\";i:3;s:0:\"\";}s:8:\"smashcut\";a:4:{i:0;s:12:\"Michael Pick\";i:1;s:32:\"b6ebbee8aa2ee353cd189fa0071df169\";i:2;s:8:\"smashcut\";i:3;s:0:\"\";}s:11:\"westonruter\";a:4:{i:0;s:12:\"Weston Ruter\";i:1;s:32:\"22ed378fbf1d918ef43a45b2a1f34634\";i:2;s:11:\"westonruter\";i:3;s:0:\"\";}}}s:5:\"props\";a:4:{s:4:\"name\";s:33:\"Core Contributors to WordPress %s\";s:12:\"placeholders\";a:1:{i:0;s:3:\"3.9\";}s:4:\"type\";s:4:\"list\";s:4:\"data\";a:238:{s:13:\"aaroncampbell\";s:17:\"Aaron D. Campbell\";s:7:\"kawauso\";s:11:\"Adam Harley\";s:7:\"adelval\";s:7:\"adelval\";s:4:\"ajay\";s:4:\"Ajay\";s:5:\"akeda\";s:11:\"Akeda Bagus\";s:6:\"xknown\";s:11:\"Alex Concha\";s:5:\"aliso\";s:14:\"Alison Barrett\";s:15:\"collinsinternet\";s:13:\"Allan Collins\";s:8:\"sabreuse\";s:22:\"Amy Hendrix (sabreuse)\";s:7:\"afercia\";s:13:\"Andrea Fercia\";s:8:\"norcross\";s:15:\"Andrew Norcross\";s:5:\"rarst\";s:24:\"Andrey \"Rarst\" Savchenko\";s:9:\"andykeith\";s:10:\"Andy Keith\";s:4:\"andy\";s:12:\"Andy Skelton\";s:7:\"atimmer\";s:16:\"Anton Timmermans\";s:9:\"aubreypwd\";s:15:\"Aubrey Portwood\";s:5:\"barry\";s:5:\"Barry\";s:7:\"toszcze\";s:18:\"Bartosz Romanowski\";s:8:\"bassgang\";s:8:\"bassgang\";s:7:\"bcworkz\";s:7:\"bcworkz\";s:13:\"empireoflight\";s:10:\"Ben Dunkle\";s:5:\"neoxx\";s:14:\"Bernhard Riedl\";s:9:\"bigdawggi\";s:9:\"bigdawggi\";s:11:\"bobbingwide\";s:11:\"bobbingwide\";s:9:\"bobbravo2\";s:10:\"Bob Gregor\";s:9:\"bradparbs\";s:9:\"bradparbs\";s:5:\"bradt\";s:14:\"Brad Touesnard\";s:5:\"bramd\";s:5:\"bramd\";s:7:\"kraftbj\";s:13:\"Brandon Kraft\";s:9:\"brasofilo\";s:9:\"brasofilo\";s:9:\"bravokeyl\";s:9:\"bravokeyl\";s:6:\"bpetty\";s:11:\"Bryan Petty\";s:7:\"cgaffga\";s:7:\"cgaffga\";s:12:\"chiragswadia\";s:13:\"Chirag Swadia\";s:6:\"chouby\";s:6:\"Chouby\";s:12:\"chriseverson\";s:12:\"chriseverson\";s:14:\"chrisguitarguy\";s:14:\"chrisguitarguy\";s:12:\"cmmarslender\";s:16:\"Chris Marslender\";s:10:\"c3mdigital\";s:14:\"Chris Olbekson\";s:10:\"chrisscott\";s:11:\"Chris Scott\";s:6:\"cfinke\";s:17:\"Christopher Finke\";s:7:\"ciantic\";s:7:\"ciantic\";s:8:\"antorome\";s:21:\"Comparativa de Bancos\";s:8:\"cojennin\";s:15:\"Connor Jennings\";s:6:\"corphi\";s:6:\"Corphi\";s:14:\"corvannoorloos\";s:16:\"Cor van Noorloos\";s:10:\"cramdesign\";s:10:\"cramdesign\";s:15:\"danielbachhuber\";s:16:\"Daniel Bachhuber\";s:10:\"redsweater\";s:27:\"Daniel Jalkut (Red Sweater)\";s:11:\"dannydehaan\";s:13:\"Danny de Haan\";s:6:\"dpe415\";s:5:\"DaveE\";s:9:\"eightface\";s:23:\"Dave Kellam (eightface)\";s:13:\"davidakennedy\";s:16:\"David A. Kennedy\";s:13:\"davidanderson\";s:14:\"David Anderson\";s:13:\"davidmarichal\";s:14:\"David Marichal\";s:17:\"denis-de-bernardy\";s:17:\"Denis de Bernardy\";s:12:\"dougwollison\";s:13:\"Doug Wollison\";s:12:\"drewapicture\";s:26:\"Drew Jaynes (DrewAPicture)\";s:11:\"drprotocols\";s:11:\"DrProtocols\";s:6:\"dustyf\";s:16:\"Dustin Filippini\";s:11:\"eatingrules\";s:11:\"eatingrules\";s:6:\"plocha\";s:4:\"edik\";s:10:\"eliorivero\";s:11:\"Elio Rivero\";s:4:\"enej\";s:4:\"enej\";s:9:\"ericlewis\";s:17:\"Eric Andrew Lewis\";s:8:\"evarlese\";s:13:\"Erica Varlese\";s:8:\"ericmann\";s:9:\"Eric Mann\";s:11:\"ejdanderson\";s:13:\"Evan Anderson\";s:9:\"fahmiadib\";s:10:\"Fahmi Adib\";s:8:\"fboender\";s:8:\"fboender\";s:11:\"frank-klein\";s:11:\"Frank Klein\";s:7:\"garyc40\";s:8:\"Gary Cao\";s:5:\"garyj\";s:10:\"Gary Jones\";s:8:\"genkisan\";s:8:\"genkisan\";s:10:\"soulseekah\";s:17:\"Gennady Kovshenin\";s:15:\"georgestephanis\";s:16:\"George Stephanis\";s:14:\"grahamarmfield\";s:15:\"Graham Armfield\";s:8:\"vancoder\";s:13:\"Grant Mangham\";s:6:\"tivnet\";s:27:\"Gregory Karpinsky (@tivnet)\";s:5:\"hakre\";s:5:\"hakre\";s:5:\"hanni\";s:5:\"hanni\";s:8:\"ippetkov\";s:8:\"ippetkov\";s:7:\"ipstenu\";s:22:\"Ipstenu (Mika Epstein)\";s:8:\"jdgrimes\";s:11:\"J.D. Grimes\";s:12:\"jackreichert\";s:13:\"Jack Reichert\";s:9:\"_jameslee\";s:8:\"jameslee\";s:7:\"janrenn\";s:7:\"janrenn\";s:5:\"jaycc\";s:5:\"JayCC\";s:11:\"jeffsebring\";s:12:\"Jeff Sebring\";s:7:\"jenmylo\";s:8:\"Jen Mylo\";s:5:\"jesin\";s:7:\"Jesin A\";s:6:\"jayjdk\";s:24:\"Jesper Johansen (jayjdk)\";s:14:\"jnielsendotnet\";s:14:\"jnielsendotnet\";s:6:\"jartes\";s:10:\"Joan Artes\";s:9:\"joedolson\";s:10:\"Joe Dolson\";s:8:\"joehoyle\";s:9:\"Joe Hoyle\";s:15:\"johnjamesjacoby\";s:17:\"John James Jacoby\";s:10:\"johnpbloch\";s:13:\"John P. Bloch\";s:10:\"johnregan3\";s:10:\"John Regan\";s:6:\"jond3r\";s:23:\"Jonas Bolinder (jond3r)\";s:11:\"joostdevalk\";s:13:\"Joost de Valk\";s:7:\"shelob9\";s:12:\"Josh Pollock\";s:14:\"joshuaabenazer\";s:15:\"Joshua Abenazer\";s:10:\"jstraitiff\";s:10:\"jstraitiff\";s:8:\"juliobox\";s:12:\"Julio Potier\";s:9:\"kopepasah\";s:16:\"Justin Kopepasah\";s:13:\"justinsainton\";s:14:\"Justin Sainton\";s:10:\"kadamwhite\";s:12:\"K.Adam White\";s:7:\"trepmal\";s:16:\"Kailey (trepmal)\";s:8:\"kasparsd\";s:7:\"Kaspars\";s:6:\"ryelle\";s:10:\"Kelly Dwan\";s:7:\"kerikae\";s:7:\"kerikae\";s:12:\"kworthington\";s:17:\"Kevin Worthington\";s:6:\"kwight\";s:10:\"Kirk Wight\";s:7:\"kitchin\";s:7:\"kitchin\";s:7:\"klihelp\";s:7:\"klihelp\";s:6:\"knutsp\";s:13:\"Knut Sparhell\";s:6:\"drozdz\";s:15:\"Krzysiek Drozdz\";s:11:\"leewillis77\";s:10:\"Lee Willis\";s:7:\"lkwdwrd\";s:7:\"lkwdwrd\";s:8:\"lpointet\";s:8:\"lpointet\";s:8:\"spmlucas\";s:13:\"Lucas Karpiuk\";s:10:\"ldebrouwer\";s:14:\"Luc De Brouwer\";s:11:\"mark8barnes\";s:11:\"Mark Barnes\";s:12:\"markoheijnen\";s:13:\"Marko Heijnen\";s:9:\"marventus\";s:9:\"Marventus\";s:13:\"iammattthomas\";s:20:\"Matt (Thomas) Miklic\";s:7:\"mjbanks\";s:10:\"Matt Banks\";s:7:\"mboynes\";s:14:\"Matthew Boynes\";s:6:\"mdbitz\";s:14:\"Matthew Denton\";s:7:\"mattheu\";s:20:\"Matthew Haines-Young\";s:11:\"mattonomics\";s:11:\"mattonomics\";s:8:\"mattyrob\";s:8:\"mattyrob\";s:9:\"maxcutler\";s:10:\"Max Cutler\";s:8:\"mcadwell\";s:8:\"mcadwell\";s:7:\"meloniq\";s:7:\"meloniq\";s:10:\"michelwppi\";s:22:\"Michel - xiligroup dev\";s:8:\"gradyetc\";s:10:\"Mike Burns\";s:10:\"mikecorkum\";s:10:\"mikecorkum\";s:12:\"mikehansenme\";s:11:\"Mike Hansen\";s:10:\"mikemanger\";s:10:\"mikemanger\";s:12:\"mikeschinkel\";s:13:\"Mike Schinkel\";s:11:\"dh-shredder\";s:13:\"Mike Schroder\";s:15:\"mitchoyoshitaka\";s:35:\"mitcho (Michael Yoshitaka Erlewine)\";s:6:\"batmoo\";s:15:\"Mohammad Jangda\";s:11:\"morganestes\";s:12:\"Morgan Estes\";s:5:\"mor10\";s:22:\"Morten Rand-Hendriksen\";s:3:\"Nao\";s:12:\"Naoko Takano\";s:7:\"alex-ye\";s:14:\"Nashwan Doaqan\";s:8:\"nendeb55\";s:8:\"nendeb55\";s:16:\"celloexpressions\";s:11:\"Nick Halsey\";s:25:\"nicolealleyinteractivecom\";s:13:\"Nicole Arnold\";s:4:\"nikv\";s:19:\"Nikhil Vimal (NikV)\";s:7:\"nivijah\";s:8:\"Nivi Jah\";s:9:\"nofearinc\";s:9:\"nofearinc\";s:14:\"nunomorgadinho\";s:15:\"Nuno Morgadinho\";s:5:\"olivm\";s:5:\"olivM\";s:5:\"jbkkd\";s:11:\"Omer Korner\";s:11:\"originalexe\";s:11:\"OriginalEXE\";s:10:\"oso96_2000\";s:10:\"oso96_2000\";s:11:\"patricknami\";s:11:\"patricknami\";s:7:\"pbearne\";s:11:\"Paul Bearne\";s:6:\"djpaul\";s:10:\"Paul Gibbs\";s:9:\"paulwilde\";s:10:\"Paul Wilde\";s:9:\"pavelevap\";s:9:\"pavelevap\";s:17:\"philiparthurmoore\";s:19:\"Philip Arthur Moore\";s:7:\"mordauk\";s:17:\"Pippin Williamson\";s:11:\"nprasath002\";s:17:\"Prasath Nadarajah\";s:11:\"prettyboymp\";s:11:\"prettyboymp\";s:7:\"raamdev\";s:8:\"Raam Dev\";s:11:\"rachelbaker\";s:11:\"rachelbaker\";s:11:\"ramonchiara\";s:11:\"ramonchiara\";s:11:\"mauryaratan\";s:16:\"Ram Ratan Maurya\";s:6:\"ounziw\";s:18:\"Rescuework Support\";s:9:\"rhyswynne\";s:10:\"Rhys Wynne\";s:14:\"ricardocorreia\";s:15:\"Ricardo Correia\";s:11:\"richard2222\";s:11:\"richard2222\";s:10:\"theorboman\";s:15:\"Richard Sweeney\";s:11:\"iamfriendly\";s:12:\"Richard Tape\";s:8:\"rickalee\";s:20:\"Ricky Lee Whittemore\";s:11:\"miqrogroove\";s:13:\"Robert Chapin\";s:9:\"robmiller\";s:9:\"robmiller\";s:13:\"rodrigosprimo\";s:13:\"Rodrigo Primo\";s:13:\"romaimperator\";s:13:\"romaimperator\";s:10:\"roothorick\";s:10:\"roothorick\";s:8:\"ruudjoyo\";s:9:\"ruud@joyo\";s:6:\"rmccue\";s:10:\"Ryan McCue\";s:7:\"salcode\";s:14:\"Sal Ferrarello\";s:6:\"otto42\";s:18:\"Samuel Wood (Otto)\";s:6:\"sandyr\";s:7:\"Sandeep\";s:8:\"scottlee\";s:9:\"Scott Lee\";s:11:\"coffee2code\";s:12:\"Scott Reilly\";s:8:\"greglone\";s:12:\"ScreenfeedFr\";s:6:\"scribu\";s:6:\"scribu\";s:6:\"sdasse\";s:6:\"sdasse\";s:6:\"bootsz\";s:10:\"Sean Butze\";s:10:\"seanchayes\";s:10:\"Sean Hayes\";s:10:\"nessworthy\";s:15:\"Sean Nessworthy\";s:14:\"sergeybiryukov\";s:15:\"Sergey Biryukov\";s:10:\"shahpranaf\";s:10:\"shahpranaf\";s:9:\"shinichin\";s:9:\"ShinichiN\";s:5:\"pross\";s:13:\"Simon Prosser\";s:13:\"simonwheatley\";s:14:\"Simon Wheatley\";s:7:\"siobhan\";s:7:\"Siobhan\";s:7:\"siobhyb\";s:24:\"Siobhan Bamber (siobhyb)\";s:8:\"sirzooro\";s:8:\"sirzooro\";s:12:\"solarissmoke\";s:12:\"solarissmoke\";s:8:\"sonjanyc\";s:8:\"sonjanyc\";s:14:\"spencerfinnell\";s:15:\"Spencer Finnell\";s:11:\"piontkowski\";s:19:\"Spencer Piontkowski\";s:11:\"stephcook22\";s:11:\"stephcook22\";s:6:\"netweb\";s:13:\"Stephen Edgar\";s:13:\"stephenharris\";s:14:\"Stephen Harris\";s:7:\"sbruner\";s:12:\"Steve Bruner\";s:11:\"stevenkword\";s:11:\"Steven Word\";s:8:\"miyauchi\";s:17:\"Takayuki Miyauchi\";s:8:\"tanner-m\";s:14:\"Tanner Moushey\";s:8:\"tlovett1\";s:13:\"Taylor Lovett\";s:6:\"tbrams\";s:6:\"tbrams\";s:10:\"tellyworth\";s:10:\"tellyworth\";s:8:\"tobiasbg\";s:8:\"TobiasBg\";s:8:\"tomauger\";s:9:\"Tom Auger\";s:7:\"willmot\";s:11:\"Tom Willmot\";s:13:\"topher1kenobe\";s:6:\"Topher\";s:9:\"topquarky\";s:9:\"topquarky\";s:10:\"zodiac1978\";s:18:\"Torsten Landsiedel\";s:4:\"toru\";s:4:\"Toru\";s:7:\"wpsmith\";s:12:\"Travis Smith\";s:11:\"umeshsingla\";s:11:\"Umesh Kumar\";s:18:\"undergroundnetwork\";s:18:\"undergroundnetwork\";s:8:\"varunagw\";s:8:\"VarunAgw\";s:5:\"wawco\";s:5:\"wawco\";s:8:\"wokamoto\";s:8:\"wokamoto\";s:6:\"xsonic\";s:6:\"xsonic\";s:5:\"yoavf\";s:10:\"Yoav Farhi\";s:10:\"yurivictor\";s:11:\"Yuri Victor\";s:9:\"zbtirrell\";s:12:\"Zach Tirrell\";s:13:\"vanillalounge\";s:13:\"Ze Fontainhas\";}}s:9:\"libraries\";a:3:{s:4:\"name\";s:18:\"External Libraries\";s:4:\"type\";s:9:\"libraries\";s:4:\"data\";a:28:{i:0;a:2:{i:0;s:11:\"Backbone.js\";i:1;s:22:\"http://backbonejs.org/\";}i:1;a:2:{i:0;s:10:\"Class POP3\";i:1;s:24:\"http://squirrelmail.org/\";}i:2;a:2:{i:0;s:16:\"Color Animations\";i:1;s:39:\"http://plugins.jquery.com/project/color\";}i:3;a:2:{i:0;s:15:\"Horde Text Diff\";i:1;s:22:\"http://pear.horde.org/\";}i:4;a:2:{i:0;s:11:\"hoverIntent\";i:1;s:45:\"http://plugins.jquery.com/project/hoverIntent\";}i:5;a:2:{i:0;s:13:\"imgAreaSelect\";i:1;s:42:\"http://odyniec.net/projects/imgareaselect/\";}i:6;a:2:{i:0;s:4:\"Iris\";i:1;s:34:\"https://github.com/Automattic/Iris\";}i:7;a:2:{i:0;s:6:\"jQuery\";i:1;s:18:\"http://jquery.com/\";}i:8;a:2:{i:0;s:9:\"jQuery UI\";i:1;s:20:\"http://jqueryui.com/\";}i:9;a:2:{i:0;s:14:\"jQuery Hotkeys\";i:1;s:41:\"https://github.com/tzuryby/jquery.hotkeys\";}i:10;a:2:{i:0;s:22:\"jQuery serializeObject\";i:1;s:49:\"http://benalman.com/projects/jquery-misc-plugins/\";}i:11;a:2:{i:0;s:12:\"jQuery.query\";i:1;s:46:\"http://plugins.jquery.com/project/query-object\";}i:12;a:2:{i:0;s:14:\"jQuery.suggest\";i:1;s:41:\"http://plugins.jquery.com/project/suggest\";}i:13;a:2:{i:0;s:21:\"jQuery UI Touch Punch\";i:1;s:27:\"http://touchpunch.furf.com/\";}i:14;a:2:{i:0;s:5:\"json2\";i:1;s:43:\"https://github.com/douglascrockford/JSON-js\";}i:15;a:2:{i:0;s:7:\"Masonry\";i:1;s:28:\"http://masonry.desandro.com/\";}i:16;a:2:{i:0;s:15:\"MediaElement.js\";i:1;s:26:\"http://mediaelementjs.com/\";}i:17;a:2:{i:0;s:6:\"PclZip\";i:1;s:33:\"http://www.phpconcept.net/pclzip/\";}i:18;a:2:{i:0;s:6:\"PemFTP\";i:1;s:50:\"http://www.phpclasses.org/browse/package/1743.html\";}i:19;a:2:{i:0;s:6:\"phpass\";i:1;s:31:\"http://www.openwall.com/phpass/\";}i:20;a:2:{i:0;s:9:\"PHPMailer\";i:1;s:55:\"http://code.google.com/a/apache-extras.org/p/phpmailer/\";}i:21;a:2:{i:0;s:8:\"Plupload\";i:1;s:24:\"http://www.plupload.com/\";}i:22;a:2:{i:0;s:9:\"SimplePie\";i:1;s:21:\"http://simplepie.org/\";}i:23;a:2:{i:0;s:27:\"The Incutio XML-RPC Library\";i:1;s:34:\"http://scripts.incutio.com/xmlrpc/\";}i:24;a:2:{i:0;s:8:\"Thickbox\";i:1;s:32:\"http://jquery.com/demo/thickbox/\";}i:25;a:2:{i:0;s:7:\"TinyMCE\";i:1;s:23:\"http://www.tinymce.com/\";}i:26;a:2:{i:0;s:13:\"Underscore.js\";i:1;s:24:\"http://underscorejs.org/\";}i:27;a:2:{i:0;s:6:\"zxcvbn\";i:1;s:30:\"https://github.com/lowe/zxcvbn\";}}}}s:4:\"data\";a:2:{s:8:\"profiles\";s:32:\"http://profiles.wordpress.org/%s\";s:7:\"version\";s:3:\"3.9\";}}','yes'),(1112,'_site_transient_update_core','O:8:\"stdClass\":4:{s:7:\"updates\";a:1:{i:0;O:8:\"stdClass\":10:{s:8:\"response\";s:6:\"latest\";s:8:\"download\";s:59:\"https://downloads.wordpress.org/release/wordpress-3.9.1.zip\";s:6:\"locale\";s:5:\"en_US\";s:8:\"packages\";O:8:\"stdClass\":5:{s:4:\"full\";s:59:\"https://downloads.wordpress.org/release/wordpress-3.9.1.zip\";s:10:\"no_content\";s:70:\"https://downloads.wordpress.org/release/wordpress-3.9.1-no-content.zip\";s:11:\"new_bundled\";s:71:\"https://downloads.wordpress.org/release/wordpress-3.9.1-new-bundled.zip\";s:7:\"partial\";b:0;s:8:\"rollback\";b:0;}s:7:\"current\";s:5:\"3.9.1\";s:7:\"version\";s:5:\"3.9.1\";s:11:\"php_version\";s:5:\"5.2.4\";s:13:\"mysql_version\";s:3:\"5.0\";s:11:\"new_bundled\";s:3:\"3.8\";s:15:\"partial_version\";s:0:\"\";}}s:12:\"last_checked\";i:1402603776;s:15:\"version_checked\";s:5:\"3.9.1\";s:12:\"translations\";a:0:{}}','yes'),(1148,'_site_transient_update_themes','O:8:\"stdClass\":4:{s:12:\"last_checked\";i:1402603776;s:7:\"checked\";a:5:{s:13:\"child_of_2012\";s:5:\"1.0.0\";s:12:\"twentyeleven\";s:3:\"1.8\";s:14:\"twentyfourteen\";s:3:\"1.1\";s:14:\"twentythirteen\";s:3:\"1.2\";s:12:\"twentytwelve\";s:3:\"1.4\";}s:8:\"response\";a:0:{}s:12:\"translations\";a:0:{}}','yes'),(1167,'metaslider_systemcheck','a:2:{s:16:\"wordPressVersion\";b:0;s:12:\"imageLibrary\";b:0;}','yes'),(1168,'ml-slider_children','a:0:{}','yes'),(1182,'slidedeck_cohort','a:4:{s:4:\"name\";s:7:\"ecf8915\";s:9:\"variation\";s:0:\"\";s:4:\"year\";s:4:\"2014\";s:5:\"month\";s:2:\"06\";}','yes'),(1183,'slidedeck_partner','','yes'),(1184,'slidedeck_version','2.3.9','yes'),(1185,'slidedeck_license','LITE','yes'),(1186,'slidedeck2_installed','1401998826','yes'),(1187,'slidedeck2_lite_installed','1401998826','yes'),(1188,'_transient_timeout_e7b99956fa29ced8818fecc1a1e2b796','1407182836','no'),(1189,'_transient_e7b99956fa29ced8818fecc1a1e2b796','a:5:{s:7:\"headers\";a:7:{s:6:\"server\";s:5:\"nginx\";s:4:\"date\";s:29:\"Thu, 05 Jun 2014 20:07:16 GMT\";s:12:\"content-type\";s:24:\"application/octet-stream\";s:14:\"content-length\";s:3:\"789\";s:13:\"last-modified\";s:29:\"Thu, 01 Nov 2012 14:39:27 GMT\";s:10:\"connection\";s:5:\"close\";s:13:\"accept-ranges\";s:5:\"bytes\";}s:4:\"body\";s:789:\"{\n    \"tier_5\":{\n        \"manage\":{\n            \"message_text\":\"Get more lenses!\"\n        },\n        \"edit\":{\n            \"message_text\":\"Get more sources!\"\n        },\n        \"lenses\":{\n            \"message_text\":\"Copy and edit lenses!\"\n        }\n    },\n    \"tier_10\":{\n        \"manage\":{\n            \"message_text\":\"Get more custom slides\"\n        },\n        \"edit\":{\n            \"message_text\":\"Get the Classic lens!\"\n        },\n        \"lenses\":{\n            \"message_text\":\"Copy and edit lenses!\"\n        }\n    },\n    \"tier_20\":{\n        \"manage\":{\n            \"message_text\":\"Get more custom slides\"\n        },\n        \"edit\":{\n            \"message_text\":\"Use SlideDeck on more sites\"\n        },\n        \"lenses\":{\n            \"message_text\":\"Copy and edit lenses!\"\n        }\n    }\n}\";s:8:\"response\";a:2:{s:4:\"code\";i:200;s:7:\"message\";s:2:\"OK\";}s:7:\"cookies\";a:0:{}s:8:\"filename\";N;}','no'),(1190,'slidedeck2_global_options','a:2:{s:21:\"anonymous_stats_optin\";b:0;s:25:\"anonymous_stats_has_opted\";b:1;}','yes'),(1191,'slidedeck-cache-buster','2','yes'),(1202,'vslider_options','a:41:{s:7:\"slideNr\";i:3;s:5:\"width\";i:630;s:6:\"height\";i:280;s:3:\"spw\";i:7;s:3:\"sph\";i:5;s:5:\"delay\";i:3000;s:6:\"sDelay\";i:30;s:7:\"opacity\";s:3:\"0.7\";s:10:\"titleSpeed\";i:1500;s:6:\"effect\";s:0:\"\";s:10:\"navigation\";s:4:\"true\";s:9:\"stickynav\";s:5:\"false\";s:5:\"links\";s:4:\"true\";s:7:\"buttons\";s:4:\"true\";s:10:\"hoverPause\";s:5:\"false\";s:10:\"fontFamily\";s:28:\"Arial, Helvetica, sans-serif\";s:9:\"titleFont\";i:16;s:8:\"fontSize\";i:12;s:9:\"textColor\";s:6:\"FFFFFF\";s:7:\"bgColor\";s:6:\"222222\";s:9:\"customImg\";s:4:\"true\";s:10:\"catchimage\";s:4:\"true\";s:5:\"chars\";i:200;s:7:\"excerpt\";s:4:\"true\";s:6:\"slide1\";s:84:\"http://localhost/~psalessi/Genetics_Lab/wp-content/plugins/vslider/images/slide1.jpg\";s:6:\"slide2\";s:84:\"http://localhost/~psalessi/Genetics_Lab/wp-content/plugins/vslider/images/slide2.jpg\";s:6:\"slide3\";s:84:\"http://localhost/~psalessi/Genetics_Lab/wp-content/plugins/vslider/images/slide3.jpg\";s:6:\"target\";s:4:\"none\";s:8:\"navstyle\";s:4:\"none\";s:8:\"arrstyle\";s:4:\"none\";s:8:\"navplace\";s:17:\"10px 0 10px 100px\";s:6:\"layout\";s:13:\"stripe-bottom\";s:11:\"borderWidth\";s:1:\"5\";s:12:\"borderRadius\";s:1:\"0\";s:11:\"borderColor\";s:6:\"FFFFFF\";s:9:\"holdermar\";s:19:\"10px 10px 10px 10px\";s:11:\"holderfloat\";s:4:\"none\";s:8:\"timthumb\";i:1;s:7:\"quality\";s:2:\"80\";s:10:\"vnavenable\";i:0;s:7:\"randimg\";i:0;}','yes'),(1203,'vslider_support','0','yes'),(1206,'soliloquy_upgrade','1','yes'),(1213,'_transient_timeout_feed_a6071078a3039a78554a2d9018cfbe6a','1402043520','no'),(1214,'_transient_feed_a6071078a3039a78554a2d9018cfbe6a','a:4:{s:5:\"child\";a:1:{s:0:\"\";a:1:{s:3:\"rss\";a:1:{i:0;a:6:{s:4:\"data\";s:3:\"\n\n\n\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:7:\"version\";s:3:\"2.0\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:1:{s:0:\"\";a:1:{s:7:\"channel\";a:1:{i:0;a:6:{s:4:\"data\";s:36:\"\n	\n	\n	\n	\n	\n	\n		\n		\n	\n	\n		\n		\n		\n		\n	\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:3:{s:0:\"\";a:7:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:39:\"Arconix Computers  arconix-flexslider\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:20:\"http://arconixpc.com\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:52:\"Web Design and Technology Services for RI, MA and CT\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:13:\"lastBuildDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Wed, 04 Jun 2014 13:00:35 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"language\";a:1:{i:0;a:5:{s:4:\"data\";s:5:\"en-US\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:9:\"generator\";a:1:{i:0;a:5:{s:4:\"data\";s:29:\"http://wordpress.org/?v=3.9.1\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"item\";a:5:{i:0;a:6:{s:4:\"data\";s:45:\"\n		\n		\n		\n		\n		\n				\n		\n\n		\n		\n				\n			\n		\n		\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:5:{s:0:\"\";a:7:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:24:\"Flexslider v0.5 released\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:50:\"http://arconixpc.com/2012/flexslider-v0-5-released\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:59:\"http://arconixpc.com/2012/flexslider-v0-5-released#comments\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 15 Nov 2012 16:00:50 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"category\";a:2:{i:0;a:5:{s:4:\"data\";s:8:\"Releases\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}i:1;a:5:{s:4:\"data\";s:18:\"arconix-flexslider\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:27:\"http://arconixpc.com/?p=862\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:539:\"<p>Having been about six months since the last release and needing some improvements, I recently finished updates to the Flexslider plugin. With this release, some significant strides have been made towards the goal of adapting this plugin to fit more applications. While it still focusing on displaying content that already exists (vs creating a &#8216;slide&#8217; [&#8230;]</p>\n<p>This post first appeared on <a rel=\"nofollow\" href=\"http://arconixpc.com\">Arconix Computers - Web Design and Technology Services for RI, MA and CT</a></p>\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:4:\"John\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:40:\"http://purl.org/rss/1.0/modules/content/\";a:1:{s:7:\"encoded\";a:1:{i:0;a:5:{s:4:\"data\";s:1610:\"<p>Having been about six months since the last release and needing some improvements, I recently finished updates to the <a title=\"Arconix FlexSlider\" href=\"http://arconixpc.com/plugins/arconix-flexslider\">Flexslider plugin</a>. With this release, some significant strides have been made towards the goal of adapting this plugin to fit more applications. While it still focusing on displaying content that already exists (vs creating a &#8216;slide&#8217; post type to display on the front end), the flexslider can now be used as more than simply a rotating image slider. This release consists almost entirely of user-requested improvements, so please let your voice be heard. Head over to the<a href=\"http://arcnx.co/aptrello\">Trello board</a>or contact me to add your ideas. There were 2 notable updates and a few minor ones.</p>\n<ul>\n<li>The flexslider can now be called via shortcode &#8212; add [ ac-flexslider ] (minus the spaces) to your post or page or other content area &#8212; in addition to the existing widget.</li>\n<li>The post-type&#8217;s content can be displayed in its entirety or just the excerpt</li>\n</ul>\n<p>That should greatly broaden the appeal of the plugin and hopefully allow it to be used in a lot more areas.Head over to the <a href=\"http://arcnx.co/afs\">plugin page</a> on WordPress to read the changelog for all the changes, if you&#8217;re interested.</p>\n<p>Be sure to let me know if you run into any issues.</p>\n<p>This post first appeared on <a rel=\"nofollow\" href=\"http://arconixpc.com\">Arconix Computers - Web Design and Technology Services for RI, MA and CT</a></p>\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:36:\"http://wellformedweb.org/CommentAPI/\";a:1:{s:10:\"commentRss\";a:1:{i:0;a:5:{s:4:\"data\";s:55:\"http://arconixpc.com/2012/flexslider-v0-5-released/feed\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:38:\"http://purl.org/rss/1.0/modules/slash/\";a:1:{s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:2:\"15\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:1;a:6:{s:4:\"data\";s:45:\"\n		\n		\n		\n		\n		\n				\n		\n\n		\n		\n				\n			\n		\n		\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:5:{s:0:\"\";a:7:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:26:\"FlexSlider Update Released\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:52:\"http://arconixpc.com/2012/flexslider-update-released\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:61:\"http://arconixpc.com/2012/flexslider-update-released#comments\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Tue, 17 Apr 2012 18:00:24 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"category\";a:2:{i:0;a:5:{s:4:\"data\";s:8:\"Releases\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}i:1;a:5:{s:4:\"data\";s:18:\"arconix-flexslider\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:27:\"http://arconixpc.com/?p=786\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:530:\"<p>I released v0.2 of the Arconix FlexSlider plugin last night. While there weren&#8217;t any major issues to begin with, I did add a couple user-requested feature: When creating the slider, we only pull in posts (or whatever post type you&#8217;re using) which have featured images, ignoring ones that don&#8217;t. I added additional image caption options, [&#8230;]</p>\n<p>This post first appeared on <a rel=\"nofollow\" href=\"http://arconixpc.com\">Arconix Computers - Web Design and Technology Services for RI, MA and CT</a></p>\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:4:\"John\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:40:\"http://purl.org/rss/1.0/modules/content/\";a:1:{s:7:\"encoded\";a:1:{i:0;a:5:{s:4:\"data\";s:1550:\"<p>I released v0.2 of the Arconix FlexSlider plugin last night. While there weren&#8217;t any major issues to begin with, I did add a couple user-requested feature:</p>\n<ol>\n<li>When creating the slider, we only pull in posts (or whatever post type you&#8217;re using) which have featured images, ignoring ones that don&#8217;t.</li>\n<li>I added additional image caption options, so instead of just being able to pull in the Post Title, we can also pull in the image title or the image caption. As always, &#8216;none&#8217; is still an option.</li>\n</ol>\n<p>I also added a developer option for filtering the post types that show up in the select box. WordPress has the following builtin post types &#8211; post, page, attachment, nav_menu_item and revision. Since the last 3 cannot have featured images, it makes no sense to show them. Initially I was indivdiually unsetting them, but that&#8217;s inefficient and not very flexible, so I created an array with all the post types I want excluded, and I wrapped that in a filter called &#8216;acfs_exclude_post_types&#8217; that can be modified by another plugin or the theme&#8217;s functions.php file. You can <a href=\"http://codex.wordpress.org/Function_Reference/apply_filters\">read more about filters here</a>.</p>\n<p>I hope you enjoy my latest release, and please let me know if you find any bugs or think something should be added.</p>\n<p>This post first appeared on <a rel=\"nofollow\" href=\"http://arconixpc.com\">Arconix Computers - Web Design and Technology Services for RI, MA and CT</a></p>\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:36:\"http://wellformedweb.org/CommentAPI/\";a:1:{s:10:\"commentRss\";a:1:{i:0;a:5:{s:4:\"data\";s:57:\"http://arconixpc.com/2012/flexslider-update-released/feed\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:38:\"http://purl.org/rss/1.0/modules/slash/\";a:1:{s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:1:\"1\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:2;a:6:{s:4:\"data\";s:45:\"\n		\n		\n		\n		\n		\n				\n		\n\n		\n		\n				\n			\n		\n		\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:5:{s:0:\"\";a:7:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:27:\"Arconix FlexSlider Released\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:53:\"http://arconixpc.com/2012/arconix-flexslider-released\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:62:\"http://arconixpc.com/2012/arconix-flexslider-released#comments\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Tue, 27 Mar 2012 14:45:51 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"category\";a:2:{i:0;a:5:{s:4:\"data\";s:8:\"Releases\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}i:1;a:5:{s:4:\"data\";s:18:\"arconix-flexslider\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:27:\"http://arconixpc.com/?p=766\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:514:\"<p>Arconix FlexSlider has been released. V0.1 can be downloaded from theWordPress website or added to your website from the WordPress admin page. Documentation can be found here, and help can be found here. Being able to choose which post-type you&#8217;d like the widget to work with, along with the image size, make this slider extremely [&#8230;]</p>\n<p>This post first appeared on <a rel=\"nofollow\" href=\"http://arconixpc.com\">Arconix Computers - Web Design and Technology Services for RI, MA and CT</a></p>\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:4:\"John\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:40:\"http://purl.org/rss/1.0/modules/content/\";a:1:{s:7:\"encoded\";a:1:{i:0;a:5:{s:4:\"data\";s:1214:\"<p><a title=\"Arconix FlexSlider\" href=\"http://arconixpc.com/plugins/arconix-flexslider\">Arconix FlexSlider</a> has been released. V0.1 can be downloaded from the<a href=\"http://arcnx.co/afs\">WordPress website</a> or added to your website from the WordPress admin page. Documentation can be found <a href=\"http://arcnx.co/afswiki\">here</a>, and help can be found <a href=\"http://arcnx.co/afshelp\">here</a>. Being able to choose which post-type you&#8217;d like the widget to work with, along with the image size, make this slider extremely flexible, something that I think a lot of people will like.</p>\n<p>As noted, this is a v0.1 release, and while I don&#8217;t know of any bugs, it&#8217;s not a very mature plugin. That&#8217;s where I need your help. Use the plugin and tell me how I can improve it. You can reach me through my <a title=\"Contact\" href=\"http://arconixpc.com/contact\">contact page</a>, <a href=\"http://arcnx.co/twitter\">Twitter</a>, <a href=\"http://arcnx.co/facebook\">Facebook</a> or <a href=\"http://arcnx.co/gplus\">Google +</a>.</p>\n<p>This post first appeared on <a rel=\"nofollow\" href=\"http://arconixpc.com\">Arconix Computers - Web Design and Technology Services for RI, MA and CT</a></p>\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:36:\"http://wellformedweb.org/CommentAPI/\";a:1:{s:10:\"commentRss\";a:1:{i:0;a:5:{s:4:\"data\";s:58:\"http://arconixpc.com/2012/arconix-flexslider-released/feed\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:38:\"http://purl.org/rss/1.0/modules/slash/\";a:1:{s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:1:\"4\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:3;a:6:{s:4:\"data\";s:45:\"\n		\n		\n		\n		\n		\n				\n		\n\n		\n		\n				\n			\n		\n		\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:5:{s:0:\"\";a:7:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:15:\"Flex, Baby Flex\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:40:\"http://arconixpc.com/2012/flex-baby-flex\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:49:\"http://arconixpc.com/2012/flex-baby-flex#comments\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Wed, 21 Mar 2012 14:30:01 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"category\";a:2:{i:0;a:5:{s:4:\"data\";s:4:\"News\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}i:1;a:5:{s:4:\"data\";s:18:\"arconix-flexslider\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:27:\"http://arconixpc.com/?p=745\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:522:\"<p>The latest plugin I&#8217;m working on is a responsive slider plugin. Now, I know what you&#8217;re probably thinking &#8211;Another slider plugin? There are already too many &#8212; and to a degree, I can&#8217;t argue. The WordPress Extend site is flooded with jQuery sliders, many of which are quite redundant. This plugin is different for 2 [&#8230;]</p>\n<p>This post first appeared on <a rel=\"nofollow\" href=\"http://arconixpc.com\">Arconix Computers - Web Design and Technology Services for RI, MA and CT</a></p>\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:4:\"John\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:40:\"http://purl.org/rss/1.0/modules/content/\";a:1:{s:7:\"encoded\";a:1:{i:0;a:5:{s:4:\"data\";s:2573:\"<p>The latest plugin I&#8217;m working on is a responsive slider plugin. Now, I know what you&#8217;re probably thinking &#8211;<em>Another slider plugin? There are already too many</em> &#8212; and to a degree, I can&#8217;t argue. The <a href=\"http://wordpress.org/extend/plugins/\">WordPress Extend</a> site is flooded with jQuery sliders, many of which are quite redundant. This plugin is different for 2 distinct reasons.</p>\n<h3>Fully Responsive</h3>\n<p>Responsive web design is all the rage right now, and for good reason. Designing multiple, separate sites to cater to PC&#8217;s, tablets and smartphones is difficult and time consuming (even if we ignore the potential of maintaining content in multiple places). Responsive design uses style changes to alter the website&#8217;s layout depending on the device the visitor is using. This means just 1 site, 1 set of content to serve to visitors, which is far more manageable. There are not a lot of sliders that are responsive by design, but the <a href=\"http://www.woothemes.com/flexslider/\">Flexslider jQuery script</a> that I&#8217;m using to power this plugin is. That&#8217;s not to say you need a responsive theme to use this slider, you don&#8217;t, but that means styling this plugin to fit a responsive WordPress theme is already that much easier.</p>\n<h3>More Flexible</h3>\n<p>Most slider plugins add a new post type (like &#8220;slides&#8221;), or hard code the &#8220;posts&#8221; type into their plugin. But what if you were using a <a title=\"Arconix Portfolio\" href=\"http://arconixpc.com/plugins/arconix-portfolio\">portfolio plugin</a> and you wanted to show a rotating slider of recent items from your portfolio on the front page of your site? If the plugin developer hadn&#8217;t created a slider widget with the plugin, you&#8217;d probably have to write one on your own. Not an easy task for most. This plugin allows you to select any registered post type to generate the slider, and as long as your post type supports post-thumbnails and you&#8217;re using Featured Images, this slider will work. It also allows you to choose from any image size available to display these images, and they&#8217;ll automatically be resized as the browser window shrinks.</p>\n<p>I&#8217;m not aware of any slider plugin that is that powerful and that flexible, and that&#8217;s where this plugin shines. Look for Arconix FlexSlider to debut shortly.</p>\n<p>This post first appeared on <a rel=\"nofollow\" href=\"http://arconixpc.com\">Arconix Computers - Web Design and Technology Services for RI, MA and CT</a></p>\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:36:\"http://wellformedweb.org/CommentAPI/\";a:1:{s:10:\"commentRss\";a:1:{i:0;a:5:{s:4:\"data\";s:45:\"http://arconixpc.com/2012/flex-baby-flex/feed\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:38:\"http://purl.org/rss/1.0/modules/slash/\";a:1:{s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:1:\"1\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:4;a:6:{s:4:\"data\";s:57:\"\n		\n		\n		\n		\n		\n				\n		\n		\n		\n		\n		\n\n		\n		\n				\n			\n		\n		\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:5:{s:0:\"\";a:7:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:35:\"Rolling Your Own CSS and Javascript\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:61:\"http://arconixpc.com/2012/rolling-your-own-css-and-javascript\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:70:\"http://arconixpc.com/2012/rolling-your-own-css-and-javascript#comments\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Mon, 09 Jan 2012 19:45:47 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"category\";a:6:{i:0;a:5:{s:4:\"data\";s:9:\"Tutorials\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}i:1;a:5:{s:4:\"data\";s:11:\"arconix-faq\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}i:2;a:5:{s:4:\"data\";s:18:\"arconix-flexslider\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}i:3;a:5:{s:4:\"data\";s:17:\"arconix-portfolio\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}i:4;a:5:{s:4:\"data\";s:18:\"arconix-shortcodes\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}i:5;a:5:{s:4:\"data\";s:20:\"arconix-testimonials\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:27:\"http://arconixpc.com/?p=687\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:498:\"<p>Normally if you want to modify the css or javascript of a plugin to better integrate with your site or add your own specific functionality, you only had a couple options. Edit the plugin files directly Try to override the css with your own through specificity and !important declarations Add your own javascript files None [&#8230;]</p>\n<p>This post first appeared on <a rel=\"nofollow\" href=\"http://arconixpc.com\">Arconix Computers - Web Design and Technology Services for RI, MA and CT</a></p>\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:4:\"John\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:40:\"http://purl.org/rss/1.0/modules/content/\";a:1:{s:7:\"encoded\";a:1:{i:0;a:5:{s:4:\"data\";s:1701:\"<p>Normally if you want to modify the css or javascript of a plugin to better integrate with your site or add your own specific functionality, you only had a couple options.</p>\n<ol>\n<li>Edit the plugin files directly</li>\n<li>Try to override the css with your own through specificity and <code>!important</code> declarations</li>\n<li>Add your own javascript files</li>\n</ol>\n<p>None of those options are all that attractive. Editing the plugin files directly is not upgrade friendly, overriding css means duplicate work and added complexity to your site&#8217;s stylesheet, and more javascript can only increase, not modify, functionality.</p>\n<p>To combat this issue, I&#8217;ve written my plugins in such a way that allows users an easy way to add their own css or javascript. If you want to modify the built-in CSS and can&#8217;t do it through custom classes, copy the CSS file you want to modify, make your changes and save it in your theme&#8217;s root directory. Just make sure the file name matches matches the plugin name, i.e. <code>arconix-shortcodes.css</code>. The same can be done with javascript (i.e. <code>arconix-shortcodes.js</code>) for modification of default parameters. The plugin searches the stylesheet and template directories for those files. If none are found, then it loads the plugin&#8217;s version.</p>\n<p>Making modifications in this manner ensures you won&#8217;t lose your changes if you upgrade the plugin, and makes it easy to keep modifications outside your style.css in the event you change themes.</p>\n<p>This post first appeared on <a rel=\"nofollow\" href=\"http://arconixpc.com\">Arconix Computers - Web Design and Technology Services for RI, MA and CT</a></p>\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:36:\"http://wellformedweb.org/CommentAPI/\";a:1:{s:10:\"commentRss\";a:1:{i:0;a:5:{s:4:\"data\";s:66:\"http://arconixpc.com/2012/rolling-your-own-css-and-javascript/feed\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:38:\"http://purl.org/rss/1.0/modules/slash/\";a:1:{s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:1:\"5\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}}}s:27:\"http://www.w3.org/2005/Atom\";a:1:{s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:0:\"\";s:7:\"attribs\";a:1:{s:0:\"\";a:3:{s:4:\"href\";s:48:\"http://arconixpc.com/tag/arconix-flexslider/feed\";s:3:\"rel\";s:4:\"self\";s:4:\"type\";s:19:\"application/rss+xml\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:44:\"http://purl.org/rss/1.0/modules/syndication/\";a:2:{s:12:\"updatePeriod\";a:1:{i:0;a:5:{s:4:\"data\";s:6:\"hourly\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:15:\"updateFrequency\";a:1:{i:0;a:5:{s:4:\"data\";s:1:\"1\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}}}}}}}}s:4:\"type\";i:128;s:7:\"headers\";a:10:{s:4:\"date\";s:29:\"Thu, 05 Jun 2014 20:31:55 GMT\";s:6:\"server\";s:6:\"Apache\";s:4:\"etag\";s:34:\"\"d39dec76033ac02cf1516da3491b7eb8\"\";s:10:\"x-pingback\";s:31:\"http://arconixpc.com/xmlrpc.php\";s:12:\"x-powered-by\";s:10:\"PHP/5.2.17\";s:12:\"x-robots-tag\";s:14:\"noindex,follow\";s:10:\"set-cookie\";s:111:\"clef_wp_session=6c912c3b1d86ac8cf4e670e0463edca8%7C%7C2835522232%7C%7C1402001756; path=/; domain=.arconixpc.com\";s:13:\"last-modified\";s:29:\"Wed, 04 Jun 2014 13:00:35 GMT\";s:10:\"connection\";s:5:\"close\";s:12:\"content-type\";s:23:\"text/xml; charset=UTF-8\";}s:5:\"build\";s:14:\"20130911120210\";}','no'),(1215,'_transient_timeout_feed_mod_a6071078a3039a78554a2d9018cfbe6a','1402043521','no'),(1216,'_transient_feed_mod_a6071078a3039a78554a2d9018cfbe6a','1402000321','no'),(1217,'widget_arconix-flexslider-widget','a:1:{s:12:\"_multiwidget\";i:1;}','yes'),(1229,'content_nav_active_bg','CCC','yes'),(1230,'content_sort','title','yes'),(1231,'content_order','ASC','yes'),(1232,'content_width','auto','yes'),(1233,'content_height','250px','yes'),(1234,'content_bg','FFF','yes'),(1235,'content_img_width','720','yes'),(1236,'content_img_height','250','yes'),(1237,'content_nav_width','240','yes'),(1238,'content_nav_height','31','yes'),(1239,'content_nav_bg','EEE','yes'),(1240,'content_nav_color','333','yes'),(1241,'content_nav_active_color','FFF','yes'),(1290,'easyrotator_compatibility_shortcodesultimate_isenabled','0','yes'),(1293,'Gallerypermissions','a:1:{s:13:\"administrator\";a:3:{i:0;s:6:\"slides\";i:1;s:9:\"galleries\";i:2;s:8:\"settings\";}}','yes'),(1294,'Galleryresizeimagescrop','Y','yes'),(1295,'Galleryimagespath','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/slideshow-gallery/','yes'),(1296,'Gallerystyles','a:11:{s:6:\"layout\";s:10:\"responsive\";s:9:\"resheight\";s:2:\"30\";s:13:\"resheighttype\";s:1:\"%\";s:12:\"resizeimages\";s:1:\"N\";s:5:\"width\";s:3:\"450\";s:6:\"height\";s:3:\"250\";s:6:\"border\";s:17:\"1px solid #CCCCCC\";s:10:\"background\";s:7:\"#000000\";s:14:\"infobackground\";s:7:\"#000000\";s:9:\"infocolor\";s:7:\"#ffffff\";s:11:\"thumbactive\";s:0:\"\";}','yes'),(1297,'Galleryfadespeed','10','yes'),(1298,'Galleryshownav','Y','yes'),(1299,'Gallerynavopacity','25','yes'),(1300,'Gallerynavhover','70','yes'),(1301,'Galleryinformation','Y','yes'),(1302,'Galleryinfospeed','20','yes'),(1304,'Gallerythumbnails','Y','yes'),(1305,'Gallerythumbwidth','100','yes'),(1306,'Gallerythumbheight','50','yes'),(1307,'Gallerythumbposition','top','yes'),(1308,'Gallerythumbopacity','70','yes'),(1309,'Gallerythumbscrollspeed','5','yes'),(1310,'Gallerythumbspacing','5','yes'),(1311,'Gallerythumbactive','#FFFFFF','yes'),(1312,'Galleryautoslide','Y','yes'),(1313,'Galleryautospeed','10','yes'),(1314,'Galleryalwaysauto','true','yes'),(1315,'Galleryimagesthickbox','Y','yes'),(1317,'Gallerysave','Save Configuration','yes'),(1351,'widget_pages','a:1:{s:12:\"_multiwidget\";i:1;}','yes'),(1388,'_transient_timeout_feed_ac0b00fe65abe10e0c5b588f3ed8c7ca','1402640355','no'),(1389,'_transient_feed_ac0b00fe65abe10e0c5b588f3ed8c7ca','a:4:{s:5:\"child\";a:1:{s:0:\"\";a:1:{s:3:\"rss\";a:1:{i:0;a:6:{s:4:\"data\";s:3:\"\n\n\n\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:7:\"version\";s:3:\"2.0\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:1:{s:0:\"\";a:1:{s:7:\"channel\";a:1:{i:0;a:6:{s:4:\"data\";s:51:\"\n	\n	\n	\n	\n	\n	\n		\n		\n	\n	\n		\n		\n		\n		\n		\n		\n		\n		\n		\n	\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:3:{s:0:\"\";a:7:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:14:\"WordPress News\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:25:\"http://wordpress.org/news\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:14:\"WordPress News\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:13:\"lastBuildDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 08 May 2014 18:40:58 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"language\";a:1:{i:0;a:5:{s:4:\"data\";s:5:\"en-US\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:9:\"generator\";a:1:{i:0;a:5:{s:4:\"data\";s:42:\"http://wordpress.org/?v=4.0-alpha-20140612\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"item\";a:10:{i:0;a:6:{s:4:\"data\";s:42:\"\n		\n		\n		\n		\n		\n				\n\n		\n		\n				\n			\n		\n		\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:5:{s:0:\"\";a:7:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:35:\"WordPress 3.9.1 Maintenance Release\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:50:\"http://wordpress.org/news/2014/05/wordpress-3-9-1/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:59:\"http://wordpress.org/news/2014/05/wordpress-3-9-1/#comments\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 08 May 2014 18:40:58 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"category\";a:1:{i:0;a:5:{s:4:\"data\";s:8:\"Releases\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:33:\"http://wordpress.org/news/?p=3241\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:385:\"After three weeks and more than 9 million downloads of WordPress 3.9, we&#8217;re pleased to announce that WordPress 3.9.1 is now available. This maintenance release fixes 34 bugs in 3.9, including numerous fixes for multisite networks, customizing widgets while previewing themes, and the updated visual editor. We&#8217;ve also made some improvements to the new audio/video [&#8230;]\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"Andrew Nacin\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:40:\"http://purl.org/rss/1.0/modules/content/\";a:1:{s:7:\"encoded\";a:1:{i:0;a:5:{s:4:\"data\";s:3077:\"<p>After three weeks and more than 9 million downloads of <a title=\"WordPress 3.9 Smith\" href=\"http://wordpress.org/news/2014/04/smith/\">WordPress 3.9</a>, we&#8217;re pleased to announce that WordPress 3.9.1 is now available.</p>\n<p>This maintenance release fixes 34 bugs in 3.9, including numerous fixes for multisite networks, customizing widgets while previewing themes, and the updated visual editor. We&#8217;ve also made some improvements to the new audio/video playlists feature and made some adjustments to improve performance. For a full list of changes, consult the <a href=\"https://core.trac.wordpress.org/query?milestone=3.9.1\">list of tickets</a> and the <a href=\"https://core.trac.wordpress.org/log/branches/3.9?rev=28353&amp;stop_rev=28154\">changelog</a>.</p>\n<p>If you are one of the millions already running WordPress 3.9, we&#8217;ve started rolling out automatic background updates for 3.9.1. For sites<a href=\"http://wordpress.org/plugins/background-update-tester/\">that support them</a>, of course.</p>\n<p><a href=\"http://wordpress.org/download/\">Download WordPress 3.9.1</a>or venture over to<strong>Dashboard  Updates</strong>and simply click &#8220;Update Now.&#8221;</p>\n<p>Thanks to all of these fine individuals for contributing to 3.9.1:<a href=\"http://profiles.wordpress.org/jorbin\">Aaron Jorbin</a>, <a href=\"http://profiles.wordpress.org/nacin\">Andrew Nacin</a>, <a href=\"http://profiles.wordpress.org/azaozz\">Andrew Ozz</a>, <a href=\"http://profiles.wordpress.org/rzen\">Brian Richards</a>, <a href=\"http://profiles.wordpress.org/ehg\">Chris Blower</a>, <a href=\"http://profiles.wordpress.org/jupiterwise\">Corey McKrill</a>, <a href=\"http://profiles.wordpress.org/danielbachhuber\">Daniel Bachhuber</a>, <a href=\"http://profiles.wordpress.org/ocean90\">Dominik Schilling</a>, <a href=\"http://profiles.wordpress.org/feedmeastraycat\">feedmeastraycat</a>, <a href=\"http://profiles.wordpress.org/gcorne\">Gregory Cornelius</a>, <a href=\"http://profiles.wordpress.org/helen\">Helen Hou-Sandi</a>, <a href=\"http://profiles.wordpress.org/imath\">imath</a>, <a href=\"http://profiles.wordpress.org/avryl\">Janneke Van Dorpe</a>, <a href=\"http://profiles.wordpress.org/jeremyfelt\">Jeremy Felt</a>, <a href=\"http://profiles.wordpress.org/johnbillion\">John Blackbourn</a>, <a href=\"http://profiles.wordpress.org/obenland\">Konstantin Obenland</a>, <a href=\"http://profiles.wordpress.org/lancewillett\">Lance Willett</a>, <a href=\"http://profiles.wordpress.org/m_i_n\">m_i_n</a>, <a href=\"http://profiles.wordpress.org/clorith\">Marius Jensen</a>, <a href=\"http://profiles.wordpress.org/markjaquith\">Mark Jaquith</a>, <a href=\"http://profiles.wordpress.org/dimadin\">Milan Dini</a>, <a href=\"http://profiles.wordpress.org/celloexpressions\">Nick Halsey</a>, <a href=\"http://profiles.wordpress.org/pavelevap\">pavelevap</a>, <a href=\"http://profiles.wordpress.org/wonderboymusic\">Scott Taylor</a>, <a href=\"http://profiles.wordpress.org/SergeyBiryukov\">Sergey Biryukov</a>, and <a href=\"http://profiles.wordpress.org/westonruter\">Weston Ruter</a>.</p>\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:36:\"http://wellformedweb.org/CommentAPI/\";a:1:{s:10:\"commentRss\";a:1:{i:0;a:5:{s:4:\"data\";s:55:\"http://wordpress.org/news/2014/05/wordpress-3-9-1/feed/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:38:\"http://purl.org/rss/1.0/modules/slash/\";a:1:{s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:1:\"0\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:1;a:6:{s:4:\"data\";s:42:\"\n		\n		\n		\n		\n		\n				\n\n		\n		\n				\n			\n		\n		\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:5:{s:0:\"\";a:7:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:25:\"WordPress 3.9 Smith\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:40:\"http://wordpress.org/news/2014/04/smith/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:49:\"http://wordpress.org/news/2014/04/smith/#comments\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Wed, 16 Apr 2014 18:33:44 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"category\";a:1:{i:0;a:5:{s:4:\"data\";s:8:\"Releases\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:33:\"http://wordpress.org/news/?p=3154\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:411:\"Version 3.9 of WordPress, named &#8220;Smith&#8221; in honor of jazz organistJimmy Smith, is availablefor downloador update in your WordPress dashboard. This release features a number of refinements that we hope you&#8217;ll love. A smoother media editingexperience Improved visual editing The updated visual editor has improved speed, accessibility, and mobile support. You can paste into the [&#8230;]\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:14:\"Matt Mullenweg\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:40:\"http://purl.org/rss/1.0/modules/content/\";a:1:{s:7:\"encoded\";a:1:{i:0;a:5:{s:4:\"data\";s:23297:\"<p>Version 3.9 of WordPress, named &#8220;Smith&#8221; in honor of jazz organist<a href=\"http://en.wikipedia.org/wiki/Jimmy_Smith_(musician)\">Jimmy Smith</a>, is available<a href=\"http://wordpress.org/download/\">for download</a>or update in your WordPress dashboard. This release features a number of refinements that we hope you&#8217;ll love.</p>\n<embed src=\"//v0.wordpress.com/player.swf?v=1.03\" type=\"application/x-shockwave-flash\" width=\"640\" height=\"360\" wmode=\"direct\" seamlesstabbing=\"true\" allowfullscreen=\"true\" allowscriptaccess=\"always\" overstretch=\"true\" flashvars=\"guid=sAiXhCfV&amp;isDynamicSeeking=true\" title=\"\"></embed>\n<h2 class=\"about-headline-callout\" style=\"text-align: center\">A smoother media editingexperience</h2>\n<div>\n<p><img class=\"alignright wp-image-3168\" src=\"//wordpress.org/news/files/2014/04/editor1-300x233.jpg\" alt=\"editor\" width=\"228\" height=\"177\" /></p>\n<h3>Improved visual editing</h3>\n<p>The updated visual editor has improved speed, accessibility, and mobile support. You can paste into the visual editor from your word processor without wasting time to clean up messy styling. (Yeah, were talking about you, Microsoft Word.)</p>\n</div>\n<div style=\"clear: both\"></div>\n<div>\n<p><img class=\"alignright wp-image-3170\" src=\"//wordpress.org/news/files/2014/04/image1-300x233.jpg\" alt=\"image\" width=\"228\" height=\"178\" /></p>\n<h3>Edit images easily</h3>\n<p>With quicker access to crop and rotation tools, its now much easier to edit your images while editing posts. You can also scale images directly in the editor to find just the right fit.</p>\n</div>\n<div style=\"clear: both\"></div>\n<div>\n<p><img class=\"alignright wp-image-3187\" src=\"//wordpress.org/news/files/2014/04/dragdrop1-300x233.jpg\" alt=\"dragdrop\" width=\"228\" height=\"178\" /></p>\n<h3>Drag and drop your images</h3>\n<p>Uploading your images is easier than ever. Just grab them from your desktop and drop them in the editor.</p>\n</div>\n<div style=\"clear: both\"></div>\n<hr />\n<h2 style=\"text-align: center\">Gallery previews</h2>\n<p><img class=\"aligncenter size-full wp-image-3169\" src=\"//wordpress.org/news/files/2014/04/gallery1.jpg\" alt=\"gallery\" width=\"980\" height=\"550\" /></p>\n<p>Galleries display a beautiful grid of images right in the editor, just like they do in your published post.</p>\n<hr />\n<h2 style=\"text-align: center\">Do more with audio and video</h2>\n\n<a href=\'http://wordpress.org/news/files/2014/04/AintMisbehavin.mp3\'>Ain\'t Misbehavin\'</a>\n<a href=\'http://wordpress.org/news/files/2014/04/DavenportBlues.mp3\'>Davenport Blues</a>\n<a href=\'http://wordpress.org/news/files/2014/04/JellyRollMorton-BuddyBoldensBlues.mp3\'>Buddy Bolden\'s Blues</a>\n<a href=\'http://wordpress.org/news/files/2014/04/Johnny_Hodges_Orchestra-Squaty_Roo-1941.mp3\'>Squaty Roo</a>\n<a href=\'http://wordpress.org/news/files/2014/04/Louisiana_Five-Dixie_Blues-1919.mp3\'>Dixie Blues</a>\n<a href=\'http://wordpress.org/news/files/2014/04/WolverineBlues.mp3\'>Wolverine Blues</a>\n\n<p>Images have galleries; now weve added simple audio and video playlists, so you can showcase your music and clips.</p>\n<hr />\n<h2 style=\"text-align: center\">Live widget and header previews</h2>\n<div style=\"width: 692px; max-width: 100%;\" class=\"wp-video\"><!--[if lt IE 9]><script>document.createElement(\'video\');</script><![endif]-->\n<video class=\"wp-video-shortcode\" id=\"video-3154-1\" width=\"692\" height=\"448\" preload=\"metadata\" controls=\"controls\"><source type=\"video/mp4\" src=\"//wordpress.org/news/files/2014/04/widgets.mp4?_=1\" /><a href=\"//wordpress.org/news/files/2014/04/widgets.mp4\">//wordpress.org/news/files/2014/04/widgets.mp4</a></video></div>\n<p>Add, edit, and rearrange your sites widgets right in the theme customizer. No save and surprise  preview your changes live and only save them when youre ready.</p>\n<p>The improved header image tool also lets you upload, crop, and manage headers while customizing your theme.</p>\n<hr />\n<h2 style=\"text-align: center\">Stunning new theme browser</h2>\n<p><img class=\"aligncenter size-full wp-image-3172\" src=\"//wordpress.org/news/files/2014/04/theme1.jpg\" alt=\"theme\" width=\"1003\" height=\"558\" /><br />\nLooking for a new theme should be easy and fun. Lose yourself in the boundless supply of free WordPress.org themes with the beautiful new theme browser.</p>\n<hr />\n<h2 style=\"text-align: center\">The Crew</h2>\n<p>This release was led by <a href=\"http://nacin.com/\">Andrew Nacin</a> and <a href=\"http://www.getsource.net/\">Mike Schroder</a>, with the help of these fine individuals. There are 267 contributors with props in this release, a new high:</p>\n<p><a href=\"http://profiles.wordpress.org/aaroncampbell\">Aaron D. Campbell</a>, <a href=\"http://profiles.wordpress.org/jorbin\">Aaron Jorbin</a>, <a href=\"http://profiles.wordpress.org/kawauso\">Adam Harley</a>, <a href=\"http://profiles.wordpress.org/adamsilverstein\">Adam Silverstein</a>, <a href=\"http://profiles.wordpress.org/adelval\">adelval</a>, <a href=\"http://profiles.wordpress.org/ajay\">Ajay</a>, <a href=\"http://profiles.wordpress.org/akeda\">Akeda Bagus</a>, <a href=\"http://profiles.wordpress.org/xknown\">Alex Concha</a>, <a href=\"http://profiles.wordpress.org/tellyworth\">Alex Shiels</a>, <a href=\"http://profiles.wordpress.org/aliso\">Alison Barrett</a>, <a href=\"http://profiles.wordpress.org/collinsinternet\">Allan Collins</a>, <a href=\"http://profiles.wordpress.org/sabreuse\">Amy Hendrix (sabreuse)</a>, <a href=\"http://profiles.wordpress.org/afercia\">Andrea Fercia</a>, <a href=\"http://profiles.wordpress.org/nacin\">Andrew Nacin</a>, <a href=\"http://profiles.wordpress.org/norcross\">Andrew Norcross</a>, <a href=\"http://profiles.wordpress.org/azaozz\">Andrew Ozz</a>, <a href=\"http://profiles.wordpress.org/rarst\">Andrey \"Rarst\" Savchenko</a>, <a href=\"http://profiles.wordpress.org/andykeith\">Andy Keith</a>, <a href=\"http://profiles.wordpress.org/andy\">Andy Skelton</a>, <a href=\"http://profiles.wordpress.org/atimmer\">Anton Timmermans</a>, <a href=\"http://profiles.wordpress.org/aubreypwd\">Aubrey Portwood</a>, <a href=\"http://profiles.wordpress.org/barry\">Barry</a>, <a href=\"http://profiles.wordpress.org/toszcze\">Bartosz Romanowski</a>, <a href=\"http://profiles.wordpress.org/bassgang\">bassgang</a>, <a href=\"http://profiles.wordpress.org/bcworkz\">bcworkz</a>, <a href=\"http://profiles.wordpress.org/empireoflight\">Ben Dunkle</a>, <a href=\"http://profiles.wordpress.org/neoxx\">Bernhard Riedl</a>, <a href=\"http://profiles.wordpress.org/bigdawggi\">bigdawggi</a>, <a href=\"http://profiles.wordpress.org/bobbravo2\">Bob Gregor</a>, <a href=\"http://profiles.wordpress.org/bobbingwide\">bobbingwide</a>, <a href=\"http://profiles.wordpress.org/bradt\">Brad Touesnard</a>, <a href=\"http://profiles.wordpress.org/bradparbs\">bradparbs</a>, <a href=\"http://profiles.wordpress.org/bramd\">bramd</a>, <a href=\"http://profiles.wordpress.org/kraftbj\">Brandon Kraft</a>, <a href=\"http://profiles.wordpress.org/brasofilo\">brasofilo</a>, <a href=\"http://profiles.wordpress.org/bravokeyl\">bravokeyl</a>, <a href=\"http://profiles.wordpress.org/bpetty\">Bryan Petty</a>, <a href=\"http://profiles.wordpress.org/cgaffga\">cgaffga</a>, <a href=\"http://profiles.wordpress.org/chiragswadia\">Chirag Swadia</a>, <a href=\"http://profiles.wordpress.org/chouby\">Chouby</a>, <a href=\"http://profiles.wordpress.org/ehg\">Chris Blower</a>, <a href=\"http://profiles.wordpress.org/cmmarslender\">Chris Marslender</a>, <a href=\"http://profiles.wordpress.org/c3mdigital\">Chris Olbekson</a>, <a href=\"http://profiles.wordpress.org/chrisscott\">Chris Scott</a>, <a href=\"http://profiles.wordpress.org/chriseverson\">chriseverson</a>, <a href=\"http://profiles.wordpress.org/chrisguitarguy\">chrisguitarguy</a>, <a href=\"http://profiles.wordpress.org/cfinke\">Christopher Finke</a>, <a href=\"http://profiles.wordpress.org/ciantic\">ciantic</a>, <a href=\"http://profiles.wordpress.org/antorome\">Comparativa de Bancos</a>, <a href=\"http://profiles.wordpress.org/cojennin\">Connor Jennings</a>, <a href=\"http://profiles.wordpress.org/corvannoorloos\">Cor van Noorloos</a>, <a href=\"http://profiles.wordpress.org/corphi\">Corphi</a>, <a href=\"http://profiles.wordpress.org/cramdesign\">cramdesign</a>, <a href=\"http://profiles.wordpress.org/danielbachhuber\">Daniel Bachhuber</a>, <a href=\"http://profiles.wordpress.org/redsweater\">Daniel Jalkut (Red Sweater)</a>, <a href=\"http://profiles.wordpress.org/dannydehaan\">Danny de Haan</a>, <a href=\"http://profiles.wordpress.org/koop\">Daryl Koopersmith</a>, <a href=\"http://profiles.wordpress.org/eightface\">Dave Kellam (eightface)</a>, <a href=\"http://profiles.wordpress.org/dpe415\">DaveE</a>, <a href=\"http://profiles.wordpress.org/davidakennedy\">David A. Kennedy</a>, <a href=\"http://profiles.wordpress.org/davidanderson\">David Anderson</a>, <a href=\"http://profiles.wordpress.org/davidmarichal\">David Marichal</a>, <a href=\"http://profiles.wordpress.org/denis-de-bernardy\">Denis de Bernardy</a>, <a href=\"http://profiles.wordpress.org/dd32\">Dion Hulse</a>, <a href=\"http://profiles.wordpress.org/ocean90\">Dominik Schilling</a>, <a href=\"http://profiles.wordpress.org/dougwollison\">Doug Wollison</a>, <a href=\"http://profiles.wordpress.org/drewapicture\">Drew Jaynes</a>, <a href=\"http://profiles.wordpress.org/drprotocols\">DrProtocols</a>, <a href=\"http://profiles.wordpress.org/dustyf\">Dustin Filippini</a>, <a href=\"http://profiles.wordpress.org/eatingrules\">eatingrules</a>, <a href=\"http://profiles.wordpress.org/plocha\">edik</a>, <a href=\"http://profiles.wordpress.org/eliorivero\">Elio Rivero</a>, <a href=\"http://profiles.wordpress.org/enej\">enej</a>, <a href=\"http://profiles.wordpress.org/ericlewis\">Eric Andrew Lewis</a>, <a href=\"http://profiles.wordpress.org/ericmann\">Eric Mann</a>, <a href=\"http://profiles.wordpress.org/evarlese\">Erica Varlese</a>, <a href=\"http://profiles.wordpress.org/ethitter\">Erick Hitter</a>, <a href=\"http://profiles.wordpress.org/ejdanderson\">Evan Anderson</a>, <a href=\"http://profiles.wordpress.org/fahmiadib\">Fahmi Adib</a>, <a href=\"http://profiles.wordpress.org/fboender\">fboender</a>, <a href=\"http://profiles.wordpress.org/frank-klein\">Frank Klein</a>, <a href=\"http://profiles.wordpress.org/garyc40\">Gary Cao</a>, <a href=\"http://profiles.wordpress.org/garyj\">Gary Jones</a>, <a href=\"http://profiles.wordpress.org/pento\">Gary Pendergast</a>, <a href=\"http://profiles.wordpress.org/genkisan\">genkisan</a>, <a href=\"http://profiles.wordpress.org/soulseekah\">Gennady Kovshenin</a>, <a href=\"http://profiles.wordpress.org/georgestephanis\">George Stephanis</a>, <a href=\"http://profiles.wordpress.org/grahamarmfield\">Graham Armfield</a>, <a href=\"http://profiles.wordpress.org/vancoder\">Grant Mangham</a>, <a href=\"http://profiles.wordpress.org/gcorne\">Gregory Cornelius</a>, <a href=\"http://profiles.wordpress.org/tivnet\">Gregory Karpinsky (@tivnet)</a>, <a href=\"http://profiles.wordpress.org/hakre\">hakre</a>, <a href=\"http://profiles.wordpress.org/hanni\">hanni</a>, <a href=\"http://profiles.wordpress.org/helen\">Helen Hou-Sand</a>, <a href=\"http://profiles.wordpress.org/ippetkov\">ippetkov</a>, <a href=\"http://profiles.wordpress.org/ipstenu\">Ipstenu (Mika Epstein)</a>, <a href=\"http://profiles.wordpress.org/jdgrimes\">J.D. Grimes</a>, <a href=\"http://profiles.wordpress.org/jackreichert\">Jack Reichert</a>, <a href=\"http://profiles.wordpress.org/_jameslee\">jameslee</a>, <a href=\"http://profiles.wordpress.org/avryl\">Janneke Van Dorpe</a>, <a href=\"http://profiles.wordpress.org/janrenn\">janrenn</a>, <a href=\"http://profiles.wordpress.org/jaycc\">JayCC</a>, <a href=\"http://profiles.wordpress.org/jeffsebring\">Jeff Sebring</a>, <a href=\"http://profiles.wordpress.org/jenmylo\">Jen Mylo</a>, <a href=\"http://profiles.wordpress.org/jeremyfelt\">Jeremy Felt</a>, <a href=\"http://profiles.wordpress.org/jesin\">Jesin A</a>, <a href=\"http://profiles.wordpress.org/jayjdk\">Jesper Johansen (jayjdk)</a>, <a href=\"http://profiles.wordpress.org/jnielsendotnet\">jnielsendotnet</a>, <a href=\"http://profiles.wordpress.org/jartes\">Joan Artes</a>, <a href=\"http://profiles.wordpress.org/joedolson\">Joe Dolson</a>, <a href=\"http://profiles.wordpress.org/joehoyle\">Joe Hoyle</a>, <a href=\"http://profiles.wordpress.org/johnbillion\">John Blackbourn</a>, <a href=\"http://profiles.wordpress.org/johnjamesjacoby\">John James Jacoby</a>, <a href=\"http://profiles.wordpress.org/johnpbloch\">John P. Bloch</a>, <a href=\"http://profiles.wordpress.org/johnregan3\">John Regan</a>, <a href=\"http://profiles.wordpress.org/duck_\">Jon Cave</a>, <a href=\"http://profiles.wordpress.org/jond3r\">Jonas Bolinder (jond3r)</a>, <a href=\"http://profiles.wordpress.org/joostdevalk\">Joost de Valk</a>, <a href=\"http://profiles.wordpress.org/shelob9\">Josh Pollock</a>, <a href=\"http://profiles.wordpress.org/joshuaabenazer\">Joshua Abenazer</a>, <a href=\"http://profiles.wordpress.org/jstraitiff\">jstraitiff</a>, <a href=\"http://profiles.wordpress.org/juliobox\">Julio Potier</a>, <a href=\"http://profiles.wordpress.org/kopepasah\">Justin Kopepasah</a>, <a href=\"http://profiles.wordpress.org/justinsainton\">Justin Sainton</a>, <a href=\"http://profiles.wordpress.org/kadamwhite\">K.Adam White</a>, <a href=\"http://profiles.wordpress.org/trepmal\">Kailey (trepmal)</a>, <a href=\"http://profiles.wordpress.org/kasparsd\">Kaspars</a>, <a href=\"http://profiles.wordpress.org/ryelle\">Kelly Dwan</a>, <a href=\"http://profiles.wordpress.org/kerikae\">kerikae</a>, <a href=\"http://profiles.wordpress.org/kworthington\">Kevin Worthington</a>, <a href=\"http://profiles.wordpress.org/kpdesign\">Kim Parsell</a>, <a href=\"http://profiles.wordpress.org/kwight\">Kirk Wight</a>, <a href=\"http://profiles.wordpress.org/kitchin\">kitchin</a>, <a href=\"http://profiles.wordpress.org/klihelp\">klihelp</a>, <a href=\"http://profiles.wordpress.org/knutsp\">Knut Sparhell</a>, <a href=\"http://profiles.wordpress.org/kovshenin\">Konstantin Kovshenin</a>, <a href=\"http://profiles.wordpress.org/obenland\">Konstantin Obenland</a>, <a href=\"http://profiles.wordpress.org/drozdz\">Krzysiek Drozdz</a>, <a href=\"http://profiles.wordpress.org/lancewillett\">Lance Willett</a>, <a href=\"http://profiles.wordpress.org/leewillis77\">Lee Willis</a>, <a href=\"http://profiles.wordpress.org/lpointet\">lpointet</a>, <a href=\"http://profiles.wordpress.org/ldebrouwer\">Luc De Brouwer</a>, <a href=\"http://profiles.wordpress.org/spmlucas\">Lucas Karpiuk</a>, <a href=\"http://profiles.wordpress.org/lkwdwrd\">Luke Woodward</a>, <a href=\"http://profiles.wordpress.org/mark8barnes\">Mark Barnes</a>, <a href=\"http://profiles.wordpress.org/markjaquith\">Mark Jaquith</a>, <a href=\"http://profiles.wordpress.org/markoheijnen\">Marko Heijnen</a>, <a href=\"http://profiles.wordpress.org/marventus\">Marventus</a>, <a href=\"http://profiles.wordpress.org/iammattthomas\">Matt (Thomas) Miklic</a>, <a href=\"http://profiles.wordpress.org/mjbanks\">Matt Banks</a>, <a href=\"http://profiles.wordpress.org/matt\">Matt Mullenweg</a>, <a href=\"http://profiles.wordpress.org/mboynes\">Matthew Boynes</a>, <a href=\"http://profiles.wordpress.org/mdbitz\">Matthew Denton</a>, <a href=\"http://profiles.wordpress.org/mattheu\">Matthew Haines-Young</a>, <a href=\"http://profiles.wordpress.org/mattonomics\">mattonomics</a>, <a href=\"http://profiles.wordpress.org/mattyrob\">mattyrob</a>, <a href=\"http://profiles.wordpress.org/matveb\">Matas Ventura</a>, <a href=\"http://profiles.wordpress.org/maxcutler\">Max Cutler</a>, <a href=\"http://profiles.wordpress.org/mcadwell\">mcadwell</a>, <a href=\"http://profiles.wordpress.org/melchoyce\">Mel Choyce</a>, <a href=\"http://profiles.wordpress.org/meloniq\">meloniq</a>, <a href=\"http://profiles.wordpress.org/michael-arestad\">Michael Arestad</a>, <a href=\"http://profiles.wordpress.org/michelwppi\">Michel - xiligroup dev</a>, <a href=\"http://profiles.wordpress.org/mcsf\">Miguel Fonseca</a>, <a href=\"http://profiles.wordpress.org/gradyetc\">Mike Burns</a>, <a href=\"http://profiles.wordpress.org/mikehansenme\">Mike Hansen</a>, <a href=\"http://profiles.wordpress.org/mikemanger\">Mike Manger</a>, <a href=\"http://profiles.wordpress.org/mikeschinkel\">Mike Schinkel</a>, <a href=\"http://profiles.wordpress.org/dh-shredder\">Mike Schroder</a>, <a href=\"http://profiles.wordpress.org/mikecorkum\">mikecorkum</a>, <a href=\"http://profiles.wordpress.org/mitchoyoshitaka\">mitcho (Michael Yoshitaka Erlewine)</a>, <a href=\"http://profiles.wordpress.org/batmoo\">Mohammad Jangda</a>, <a href=\"http://profiles.wordpress.org/morganestes\">Morgan Estes</a>, <a href=\"http://profiles.wordpress.org/mor10\">Morten Rand-Hendriksen</a>, <a href=\"http://profiles.wordpress.org/Nao\">Naoko Takano</a>, <a href=\"http://profiles.wordpress.org/alex-ye\">Nashwan Doaqan</a>, <a href=\"http://profiles.wordpress.org/nendeb55\">nendeb55</a>, <a href=\"http://profiles.wordpress.org/celloexpressions\">Nick Halsey</a>, <a href=\"http://profiles.wordpress.org/nicolealleyinteractivecom\">Nicole Arnold</a>, <a href=\"http://profiles.wordpress.org/nikv\">Nikhil Vimal (NikV)</a>, <a href=\"http://profiles.wordpress.org/nivijah\">Nivi Jah</a>, <a href=\"http://profiles.wordpress.org/nofearinc\">nofearinc</a>, <a href=\"http://profiles.wordpress.org/nunomorgadinho\">Nuno Morgadinho</a>, <a href=\"http://profiles.wordpress.org/olivm\">olivM</a>, <a href=\"http://profiles.wordpress.org/jbkkd\">Omer Korner</a>, <a href=\"http://profiles.wordpress.org/originalexe\">OriginalEXE</a>, <a href=\"http://profiles.wordpress.org/oso96_2000\">oso96_2000</a>, <a href=\"http://profiles.wordpress.org/patricknami\">patricknami</a>, <a href=\"http://profiles.wordpress.org/pbearne\">Paul Bearne</a>, <a href=\"http://profiles.wordpress.org/djpaul\">Paul Gibbs</a>, <a href=\"http://profiles.wordpress.org/paulwilde\">Paul Wilde</a>, <a href=\"http://profiles.wordpress.org/pavelevap\">pavelevap</a>, <a href=\"http://profiles.wordpress.org/westi\">Peter Westwood</a>, <a href=\"http://profiles.wordpress.org/philiparthurmoore\">Philip Arthur Moore</a>, <a href=\"http://profiles.wordpress.org/mordauk\">Pippin Williamson</a>, <a href=\"http://profiles.wordpress.org/nprasath002\">Prasath Nadarajah</a>, <a href=\"http://profiles.wordpress.org/prettyboymp\">prettyboymp</a>, <a href=\"http://profiles.wordpress.org/raamdev\">Raam Dev</a>, <a href=\"http://profiles.wordpress.org/rachelbaker\">rachelbaker</a>, <a href=\"http://profiles.wordpress.org/mauryaratan\">Ram Ratan Maurya</a>, <a href=\"http://profiles.wordpress.org/ramonchiara\">ramonchiara</a>, <a href=\"http://profiles.wordpress.org/ounziw\">Rescuework Support</a>, <a href=\"http://profiles.wordpress.org/rhyswynne\">Rhys Wynne</a>, <a href=\"http://profiles.wordpress.org/ricardocorreia\">Ricardo Correia</a>, <a href=\"http://profiles.wordpress.org/theorboman\">Richard Sweeney</a>, <a href=\"http://profiles.wordpress.org/iamfriendly\">Richard Tape</a>, <a href=\"http://profiles.wordpress.org/richard2222\">richard2222</a>, <a href=\"http://profiles.wordpress.org/rickalee\">Ricky Lee Whittemore</a>, <a href=\"http://profiles.wordpress.org/miqrogroove\">Robert Chapin</a>, <a href=\"http://profiles.wordpress.org/robmiller\">robmiller</a>, <a href=\"http://profiles.wordpress.org/rodrigosprimo\">Rodrigo Primo</a>, <a href=\"http://profiles.wordpress.org/romaimperator\">romaimperator</a>, <a href=\"http://profiles.wordpress.org/roothorick\">roothorick</a>, <a href=\"http://profiles.wordpress.org/ruudjoyo\">ruud@joyo</a>, <a href=\"http://profiles.wordpress.org/ryan\">Ryan Boren</a>, <a href=\"http://profiles.wordpress.org/rmccue\">Ryan McCue</a>, <a href=\"http://profiles.wordpress.org/salcode\">Sal Ferrarello</a>, <a href=\"http://profiles.wordpress.org/otto42\">Samuel Wood (Otto)</a>, <a href=\"http://profiles.wordpress.org/sandyr\">Sandeep</a>, <a href=\"http://profiles.wordpress.org/scottlee\">Scott Lee</a>, <a href=\"http://profiles.wordpress.org/coffee2code\">Scott Reilly</a>, <a href=\"http://profiles.wordpress.org/wonderboymusic\">Scott Taylor</a>, <a href=\"http://profiles.wordpress.org/greglone\">ScreenfeedFr</a>, <a href=\"http://profiles.wordpress.org/scribu\">scribu</a>, <a href=\"http://profiles.wordpress.org/sdasse\">sdasse</a>, <a href=\"http://profiles.wordpress.org/bootsz\">Sean Butze</a>, <a href=\"http://profiles.wordpress.org/seanchayes\">Sean Hayes</a>, <a href=\"http://profiles.wordpress.org/nessworthy\">Sean Nessworthy</a>, <a href=\"http://profiles.wordpress.org/sergeybiryukov\">Sergey Biryukov</a>, <a href=\"http://profiles.wordpress.org/shahpranaf\">shahpranaf</a>, <a href=\"http://profiles.wordpress.org/shaunandrews\">Shaun Andrews</a>, <a href=\"http://profiles.wordpress.org/shinichin\">ShinichiN</a>, <a href=\"http://profiles.wordpress.org/pross\">Simon Prosser</a>, <a href=\"http://profiles.wordpress.org/simonwheatley\">Simon Wheatley</a>, <a href=\"http://profiles.wordpress.org/siobhan\">Siobhan</a>, <a href=\"http://profiles.wordpress.org/siobhyb\">Siobhan Bamber (siobhyb)</a>, <a href=\"http://profiles.wordpress.org/sirzooro\">sirzooro</a>, <a href=\"http://profiles.wordpress.org/solarissmoke\">solarissmoke</a>, <a href=\"http://profiles.wordpress.org/sonjanyc\">sonjanyc</a>, <a href=\"http://profiles.wordpress.org/spencerfinnell\">Spencer Finnell</a>, <a href=\"http://profiles.wordpress.org/piontkowski\">Spencer Piontkowski</a>, <a href=\"http://profiles.wordpress.org/stephcook22\">stephcook22</a>, <a href=\"http://profiles.wordpress.org/netweb\">Stephen Edgar</a>, <a href=\"http://profiles.wordpress.org/stephenharris\">Stephen Harris</a>, <a href=\"http://profiles.wordpress.org/sbruner\">Steve Bruner</a>, <a href=\"http://profiles.wordpress.org/stevenkword\">Steven Word</a>, <a href=\"http://profiles.wordpress.org/miyauchi\">Takayuki Miyauchi</a>, <a href=\"http://profiles.wordpress.org/tanner-m\">Tanner Moushey</a>, <a href=\"http://profiles.wordpress.org/tlovett1\">Taylor Lovett</a>, <a href=\"http://profiles.wordpress.org/tbrams\">tbrams</a>, <a href=\"http://profiles.wordpress.org/tobiasbg\">TobiasBg</a>, <a href=\"http://profiles.wordpress.org/tomauger\">Tom Auger</a>, <a href=\"http://profiles.wordpress.org/willmot\">Tom Willmot</a>, <a href=\"http://profiles.wordpress.org/topher1kenobe\">Topher</a>, <a href=\"http://profiles.wordpress.org/topquarky\">topquarky</a>, <a href=\"http://profiles.wordpress.org/zodiac1978\">Torsten Landsiedel</a>, <a href=\"http://profiles.wordpress.org/toru\">Toru</a>, <a href=\"http://profiles.wordpress.org/wpsmith\">Travis Smith</a>, <a href=\"http://profiles.wordpress.org/umeshsingla\">Umesh Kumar</a>, <a href=\"http://profiles.wordpress.org/undergroundnetwork\">undergroundnetwork</a>, <a href=\"http://profiles.wordpress.org/varunagw\">VarunAgw</a>, <a href=\"http://profiles.wordpress.org/wawco\">wawco</a>, <a href=\"http://profiles.wordpress.org/westonruter\">Weston Ruter</a>, <a href=\"http://profiles.wordpress.org/wokamoto\">wokamoto</a>, <a href=\"http://profiles.wordpress.org/xsonic\">xsonic</a>, <a href=\"http://profiles.wordpress.org/yoavf\">Yoav Farhi</a>, <a href=\"http://profiles.wordpress.org/yurivictor\">Yuri Victor</a>, <a href=\"http://profiles.wordpress.org/zbtirrell\">Zach Tirrell</a>, and <a href=\"http://profiles.wordpress.org/vanillalounge\">Ze Fontainhas</a>. Also thanks to <a href=\"http://michaelpick.wordpress.com/\">Michael Pick</a> for producing the release video.</p>\n<p>If you want to follow along or help out, check out <a href=\"http://make.wordpress.org/\">Make WordPress</a> and our <a href=\"http://make.wordpress.org/core/\">core development blog</a>. Thanks for choosing WordPress. See you soon for version 4.0!</p>\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:36:\"http://wellformedweb.org/CommentAPI/\";a:1:{s:10:\"commentRss\";a:1:{i:0;a:5:{s:4:\"data\";s:45:\"http://wordpress.org/news/2014/04/smith/feed/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:38:\"http://purl.org/rss/1.0/modules/slash/\";a:1:{s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:1:\"0\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:2;a:6:{s:4:\"data\";s:45:\"\n		\n		\n		\n		\n		\n				\n		\n\n		\n		\n				\n			\n		\n		\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:5:{s:0:\"\";a:7:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:33:\"WordPress 3.9 Release Candidate 2\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:68:\"http://wordpress.org/news/2014/04/wordpress-3-9-release-candidate-2/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:77:\"http://wordpress.org/news/2014/04/wordpress-3-9-release-candidate-2/#comments\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Tue, 15 Apr 2014 09:47:36 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"category\";a:2:{i:0;a:5:{s:4:\"data\";s:11:\"Development\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}i:1;a:5:{s:4:\"data\";s:8:\"Releases\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:33:\"http://wordpress.org/news/?p=3151\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:356:\"The second release candidate for WordPress 3.9 is now available for testing. If you haven&#8217;t tested 3.9 yet, you&#8217;re running out of time! We made about five dozen changes since the first release candidate, and those changes are all helpfully summarized in our weekly post on the development blog. Probably the biggest fixesare to live [&#8230;]\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"Andrew Nacin\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:40:\"http://purl.org/rss/1.0/modules/content/\";a:1:{s:7:\"encoded\";a:1:{i:0;a:5:{s:4:\"data\";s:2273:\"<p>The second release candidate for WordPress 3.9 is now available for testing.</p>\n<p>If you haven&#8217;t tested 3.9 yet, you&#8217;re running out of time! We made about five dozen changes since the <a title=\"WordPress 3.9 Release Candidate\" href=\"//wordpress.org/news/2014/04/wordpress-3-9-release-candidate/\">first release candidate</a>, and those changes are all helpfully summarized <a href=\"//make.wordpress.org/core/?p=10237\">in our weekly post</a> on the development blog. Probably the biggest fixesare to live widget previews and the new theme browser, along with some extra TinyMCE compatibility and some RTL fixes.</p>\n<p><strong>Plugin authors:</strong>Could you test your plugins against 3.9, and if they&#8217;re compatible, make sure they are marked as tested up to 3.9?It only takes a few minutes and this really helps make launch easier.Be sure to follow along the core development blog; we&#8217;ve been posting<a href=\"//make.wordpress.org/core/tag/3-9-dev-notes/\">notes fordevelopers for 3.9</a>. (For example: <a href=\"//make.wordpress.org/core/2014/04/15/html5-galleries-captions-in-wordpress-3-9/\">HTML5</a>, <a href=\"//make.wordpress.org/core/2014/04/14/symlinked-plugins-in-wordpress-3-9/\">symlinks</a>, <a href=\"//make.wordpress.org/core/2014/04/07/mysql-in-wordpress-3-9/\">MySQL</a>, <a href=\"//make.wordpress.org/core/2014/04/11/plupload-2-x-in-wordpress-3-9/\">Plupload</a>.)</p>\n<p>To test WordPress 3.9 RC2, try the<a href=\"//wordpress.org/extend/plugins/wordpress-beta-tester/\">WordPress Beta Tester</a>plugin (youll want bleeding edge nightlies). Or you can<a href=\"//wordpress.org/wordpress-3.9-RC2.zip\">download the release candidate here</a>(zip).If youd like to learn more about whats new in WordPress 3.9, visit the nearly completeAbout screen in your dashboard (<strong><img src=\"//i0.wp.com/core.svn.wordpress.org/branches/3.6/wp-content/themes/twentyten/images/wordpress.png?w=692\" alt=\"\" width=\"16\" height=\"16\" /> About</strong>in the toolbar) and also check out<a title=\"WordPress 3.9 Beta 1\" href=\"//wordpress.org/news/2014/03/wordpress-3-9-beta-1/\">the Beta 1 post</a>.</p>\n<p><em>This is for testing,</em><br />\n<em>so not recommended for<br />\nproductionsitesyet.</em></p>\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:36:\"http://wellformedweb.org/CommentAPI/\";a:1:{s:10:\"commentRss\";a:1:{i:0;a:5:{s:4:\"data\";s:73:\"http://wordpress.org/news/2014/04/wordpress-3-9-release-candidate-2/feed/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:38:\"http://purl.org/rss/1.0/modules/slash/\";a:1:{s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:1:\"0\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:3;a:6:{s:4:\"data\";s:42:\"\n		\n		\n		\n		\n		\n				\n\n		\n		\n				\n			\n		\n		\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:5:{s:0:\"\";a:7:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:35:\"WordPress 3.8.3 Maintenance Release\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:50:\"http://wordpress.org/news/2014/04/wordpress-3-8-3/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:59:\"http://wordpress.org/news/2014/04/wordpress-3-8-3/#comments\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Mon, 14 Apr 2014 19:29:13 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"category\";a:1:{i:0;a:5:{s:4:\"data\";s:8:\"Releases\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:33:\"http://wordpress.org/news/?p=3145\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:338:\"WordPress 3.8.3 is now available to fix a small but unfortunate bug in the WordPress 3.8.2 security release. The &#8220;Quick Draft&#8221; tool on the dashboard screen was broken in the 3.8.2 update. If you tried to use it, your draft would disappear and it wouldn&#8217;t save. While we doubt anyone was writing a novella using [&#8230;]\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"Andrew Nacin\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:40:\"http://purl.org/rss/1.0/modules/content/\";a:1:{s:7:\"encoded\";a:1:{i:0;a:5:{s:4:\"data\";s:2339:\"<p>WordPress 3.8.3 is now available to fix a small but unfortunate bug in the <a title=\"WordPress 3.8.2 Security Release\" href=\"http://wordpress.org/news/2014/04/wordpress-3-8-2/\">WordPress 3.8.2 security release</a>.</p>\n<p>The &#8220;Quick Draft&#8221; tool on the dashboard screen was broken in the 3.8.2 update. If you tried to use it, your draft would disappear and it wouldn&#8217;t save. While we doubt anyone was writing a novella using this tool, <em>any</em> loss of content is unacceptable to us.</p>\n<p>We recognize how much trust you place in us to safeguard your content, and we take this responsibility very seriously. We&#8217;re sorry we let you down.</p>\n<p>We&#8217;ve all lost words we&#8217;ve written before, like an email thanks to a cat on the keyboard or a term paper to a blue screen of death. Over the last few WordPress releases, we&#8217;ve made a number of improvements to features like autosaves and revisions. With revisions, an old edit can always be restored. We&#8217;re trying our hardest to save your content somewhere even if your power goes out or your browser crashes. We even monitor your internet connection and prevent you from hitting that &#8220;Publish&#8221; button at the exact moment the coffee shop Wi-Fi has a hiccup.</p>\n<p>It&#8217;s <em>possible</em> that the quick draft you lost last week is still in the database, and just hidden from view. As an added complication, these &#8220;discarded drafts&#8221; normally get deleted after seven days, and it&#8217;s already been sixdays since the release. If we were able to rescue your draft, you&#8217;ll see it on the &#8220;All Posts&#8221; screen after you update to 3.8.3. (We&#8217;ll also be pushing 3.8.3 out as a background update, so you may just see a draft appear.)</p>\n<p>So, if you tried to jot down a quick idea last week, I hope WordPress has recovered it for you. Maybe it&#8217;ll turn into that novella.</p>\n<p><a href=\"http://wordpress.org/download/\">Download WordPress 3.8.3</a> or click &#8220;Update Now&#8221; on Dashboard  Updates.</p>\n<p><em>This affected version 3.7.2 as well, so we&#8217;re pushing a 3.7.3 to these installs, but we&#8217;d encourage you to update to the latest and greatest.</em></p>\n<hr />\n<p><em>Now for some good news:<br />\nWordPress 3.9 is near.<br />\nExpect it thisweek</em></p>\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:36:\"http://wellformedweb.org/CommentAPI/\";a:1:{s:10:\"commentRss\";a:1:{i:0;a:5:{s:4:\"data\";s:55:\"http://wordpress.org/news/2014/04/wordpress-3-8-3/feed/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:38:\"http://purl.org/rss/1.0/modules/slash/\";a:1:{s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:1:\"0\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:4;a:6:{s:4:\"data\";s:45:\"\n		\n		\n		\n		\n		\n				\n		\n\n		\n		\n				\n			\n		\n		\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:5:{s:0:\"\";a:7:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"WordPress 3.9 Release Candidate\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:66:\"http://wordpress.org/news/2014/04/wordpress-3-9-release-candidate/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:75:\"http://wordpress.org/news/2014/04/wordpress-3-9-release-candidate/#comments\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Tue, 08 Apr 2014 21:05:21 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"category\";a:2:{i:0;a:5:{s:4:\"data\";s:11:\"Development\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}i:1;a:5:{s:4:\"data\";s:8:\"Releases\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:33:\"http://wordpress.org/news/?p=3129\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:338:\"As teased earlier, the first release candidate for WordPress 3.9 is now available for testing! We hope to ship WordPress 3.9 next week, but we need your help to get there.If you havent tested 3.9 yet, theres no time like the present. (Please, not on a production site, unless youre adventurous.) To test WordPress 3.9 [&#8230;]\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"Andrew Nacin\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:40:\"http://purl.org/rss/1.0/modules/content/\";a:1:{s:7:\"encoded\";a:1:{i:0;a:5:{s:4:\"data\";s:2967:\"<p><a href=\"//wordpress.org/news/2014/04/wordpress-3-8-2/\">As teased earlier</a>, the first release candidate for WordPress 3.9 is now available for testing!</p>\n<p>We hope to ship WordPress 3.9 <em>next week</em>, but we need your help to get there.If you havent tested 3.9 yet, theres no time like the present. (Please, not on a production site, unless youre adventurous.)</p>\n<p>To test WordPress 3.9 RC1, try the<a href=\"//wordpress.org/extend/plugins/wordpress-beta-tester/\">WordPress Beta Tester</a>plugin (youll want bleeding edge nightlies). Or you can<a href=\"//wordpress.org/wordpress-3.9-RC1.zip\">download the release candidate here</a>(zip).If youd like to learn more about whats new in WordPress 3.9, visit the work-in-progress About screen in your dashboard (<strong><img src=\"//i0.wp.com/core.svn.wordpress.org/branches/3.6/wp-content/themes/twentyten/images/wordpress.png?w=692\" alt=\"\" width=\"16\" height=\"16\" /> About</strong>in the toolbar) and check out <a title=\"WordPress 3.9 Beta 1\" href=\"//wordpress.org/news/2014/03/wordpress-3-9-beta-1/\">the Beta 1 post</a>.</p>\n<p><strong>Think youve found a bug?</strong>Please post to the<a href=\"//wordpress.org/support/forum/alphabeta/\">Alpha/Beta area in the support forums</a>. If any known issues come up, youll be able to<a href=\"//core.trac.wordpress.org/report/5\">find them here</a>.</p>\n<p><strong>If you&#8217;re a plugin author</strong>, there are two important changesin particular to be aware of:</p>\n<ul>\n<li>TinyMCE received a major update, toversion4.0. Any editor plugins written for TinyMCE 3.x might require some updates. (If things broke, we&#8217;d like to hear about them so we can make adjustments.)For more, see TinyMCE&#8217;s<a href=\"http://www.tinymce.com/wiki.php/Tutorial:Migration_guide_from_3.x\">migration guide</a>and<a href=\"http://www.tinymce.com/wiki.php/api4:index\">API documentation</a>, and the notes onthe <a href=\"//make.wordpress.org/core/2014/01/18/tinymce-4-0-is-in-core/\">core development blog</a>.</li>\n<li>WordPress 3.9 now usesthe MySQLi Improved extension for sites runningPHP 5.5. Any plugins that made direct calls to <code>mysql_*</code> functions will experience some problems on these sites. For moreinformation, see the notes on the <a href=\"//make.wordpress.org/core/2014/04/07/mysql-in-wordpress-3-9/\">coredevelopment blog</a>.</li>\n</ul>\n<p>Be sure to follow along the core development blog, where we will be continuing to post <a href=\"//make.wordpress.org/core/tag/3-9-dev-notes/\">notes fordevelopers for 3.9</a>. (For example, read <a href=\"//make.wordpress.org/core/2014/03/27/masonry-in-wordpress-3-9/\">this</a> if you areusing Masonry in your theme.) And please, please update your plugin&#8217;s <em>Tested up to</em>version in the readme to 3.9 before April 16.</p>\n<p><em>Release candidate<br />\nThis haiku&#8217;s the easy one<br />\n3.9 is near</em></p>\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:36:\"http://wellformedweb.org/CommentAPI/\";a:1:{s:10:\"commentRss\";a:1:{i:0;a:5:{s:4:\"data\";s:71:\"http://wordpress.org/news/2014/04/wordpress-3-9-release-candidate/feed/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:38:\"http://purl.org/rss/1.0/modules/slash/\";a:1:{s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:1:\"0\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:5;a:6:{s:4:\"data\";s:45:\"\n		\n		\n		\n		\n		\n				\n		\n\n		\n		\n				\n			\n		\n		\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:5:{s:0:\"\";a:7:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:32:\"WordPress 3.8.2 Security Release\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:50:\"http://wordpress.org/news/2014/04/wordpress-3-8-2/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:59:\"http://wordpress.org/news/2014/04/wordpress-3-8-2/#comments\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Tue, 08 Apr 2014 19:04:44 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"category\";a:2:{i:0;a:5:{s:4:\"data\";s:8:\"Releases\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}i:1;a:5:{s:4:\"data\";s:8:\"Security\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:33:\"http://wordpress.org/news/?p=3124\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:355:\"WordPress 3.8.2 is now available. This is an important security release for all previous versions and we strongly encourage you to update your sites immediately. This releases fixes a weakness that could let an attacker force their way into your site by forging authentication cookies.This was discovered and fixed by Jon Cave of the WordPress [&#8230;]\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"Andrew Nacin\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:40:\"http://purl.org/rss/1.0/modules/content/\";a:1:{s:7:\"encoded\";a:1:{i:0;a:5:{s:4:\"data\";s:2272:\"<p>WordPress 3.8.2 is now available. This is an important security release for all previous versions and we strongly encourage you to update your sites immediately.</p>\n<p>This releases fixes a weakness that could let an attacker force their way into your site by forging authentication cookies.This was discovered and fixed by <a href=\"http://joncave.co.uk/\">Jon Cave</a> of the WordPress security team.</p>\n<p>It also contains a fix to prevent a user with the Contributor role from improperly publishing posts. Reported by <a href=\"http://edik.ch/\">edik</a>.</p>\n<p>This release also fixes nine bugs and contains three other security hardening changes:</p>\n<ul>\n<li>Pass along additional information when processing pingbacks to help hosts identify potentially abusive requests.</li>\n<li>Fix a low-impact SQL injection by trusted users. Reported by <a href=\"http://www.dxw.com/\">Tom Adams</a> of dxw.</li>\n<li>Prevent possible cross-domain scripting through Plupload, the third-party library WordPress uses for uploading files. Reported by <a href=\"http://szgru.website.pl/\">Szymon Gruszecki</a>.</li>\n</ul>\n<p>We appreciated <a href=\"http://codex.wordpress.org/FAQ_Security\">responsible disclosure</a> of thesesecurity issues directly to our security team. For more information on all of thechanges, see the <a href=\"http://codex.wordpress.org/Version_3.8.2\">release notes</a> or consult <a href=\"https://core.trac.wordpress.org/log/branches/3.8?rev=28057&amp;stop_rev=27024\">the list of changes</a>.</p>\n<p><a href=\"http://wordpress.org/download/\">Download WordPress 3.8.2</a> or venture over to <strong>Dashboard  Updates</strong> and simply click &#8220;Update Now.&#8221;</p>\n<p>Sites that support automatic background updates will be updated to WordPress 3.8.2 within 12 hours. If you are still on WordPress 3.7.1, you will be updated to 3.7.2, which contains the same security fixes as 3.8.2. We don&#8217;t support older versions, so please update to 3.8.2 for the latest and greatest.</p>\n<p>Already testing WordPress 3.9? The first release candidate is <a href=\"https://wordpress.org/wordpress-3.9-RC1.zip\">now available</a> (zip) and it contains these security fixes. Look for a full announcement later today; we expect to release 3.9 next week.</p>\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:36:\"http://wellformedweb.org/CommentAPI/\";a:1:{s:10:\"commentRss\";a:1:{i:0;a:5:{s:4:\"data\";s:55:\"http://wordpress.org/news/2014/04/wordpress-3-8-2/feed/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:38:\"http://purl.org/rss/1.0/modules/slash/\";a:1:{s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:1:\"0\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:6;a:6:{s:4:\"data\";s:45:\"\n		\n		\n		\n		\n		\n				\n		\n\n		\n		\n				\n			\n		\n		\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:5:{s:0:\"\";a:7:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:20:\"WordPress 3.9 Beta 3\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:55:\"http://wordpress.org/news/2014/03/wordpress-3-9-beta-3/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:64:\"http://wordpress.org/news/2014/03/wordpress-3-9-beta-3/#comments\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Sat, 29 Mar 2014 13:15:10 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"category\";a:2:{i:0;a:5:{s:4:\"data\";s:11:\"Development\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}i:1;a:5:{s:4:\"data\";s:8:\"Releases\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:33:\"http://wordpress.org/news/?p=3106\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:373:\"The third(and maybelast) beta of WordPress 3.9 is now available for download. Beta 3 includes more than200changes, including: New features like live widget previews and the new theme installer are now more ready for prime time, so check &#8216;em out. UI refinementswhen editing images and when working with media in the editor. We&#8217;ve also brought [&#8230;]\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"Andrew Nacin\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:40:\"http://purl.org/rss/1.0/modules/content/\";a:1:{s:7:\"encoded\";a:1:{i:0;a:5:{s:4:\"data\";s:2668:\"<p>The third(and maybelast) beta of WordPress 3.9 is now available for download.</p>\n<p>Beta 3 includes more than200<a href=\"https://core.trac.wordpress.org/log?rev=27850&amp;stop_rev=27639&amp;limit=300\">changes</a>, including:</p>\n<ul>\n<li>New features like live widget previews and the new theme installer are now more ready for prime time, so check &#8216;em out.</li>\n<li>UI refinementswhen editing images and when working with media in the editor. We&#8217;ve also brought back some of the advanced display settings for images.</li>\n<li>If you want to test out audio and videoplaylists, the links will appear in the media manager once you&#8217;veuploaded an audio or video file.</li>\n<li>For theme developers, we&#8217;ve added HTML5 caption support (<a class=\"reopened ticket\" title=\"task (blessed): HTML5 captions (reopened)\" href=\"https://core.trac.wordpress.org/ticket/26642\">#26642</a>) to match the new gallery support (<a class=\"closed ticket\" title=\"enhancement: HTML5 Galleries (closed: fixed)\" href=\"https://core.trac.wordpress.org/ticket/26697\">#26697</a>).</li>\n<li>The formatting function that turns straight quotes into smart quotes (among other things) underwentsome changes to drastically speed itup, so let us know if you see anything weird.</li>\n</ul>\n<p><strong>We need your help</strong>.We&#8217;re still aimingfor an April release, which means the next week will be critical foridentifying and squashing bugs. If you&#8217;re just joining us, please see<a href=\"https://wordpress.org/news/2014/03/wordpress-3-9-beta-1/\">the Beta 1 announcement post</a>for what to look out for.</p>\n<p><strong>If you think youve found a bug</strong>, you can post to the<a href=\"http://wordpress.org/support/forum/alphabeta\">Alpha/Beta area</a>in the support forums, where friendly moderators are standing by.<b>Plugin developers</b><strong>,</strong> if you haven&#8217;t tested WordPress 3.9 yet, now is the time  and be sure to update the &#8220;tested up to&#8221; version for your plugins so they&#8217;re listed as compatible with 3.9.</p>\n<p><strong>This software is still in development,</strong>so we dont recommend you run it on a production site. Consider setting up a test site just to play with the new version. To test WordPress 3.9, try the<a href=\"http://wordpress.org/extend/plugins/wordpress-beta-tester/\">WordPress Beta Tester</a>plugin (youll want bleeding edge nightlies). Or you can<a href=\"http://wordpress.org/wordpress-3.9-beta3.zip\">download the beta here</a>(zip).</p>\n<p><em>WordPress 3.9<br />\nLet&#8217;s make thedate official<br />\nIt&#8217;s April 16</em></p>\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:36:\"http://wellformedweb.org/CommentAPI/\";a:1:{s:10:\"commentRss\";a:1:{i:0;a:5:{s:4:\"data\";s:60:\"http://wordpress.org/news/2014/03/wordpress-3-9-beta-3/feed/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:38:\"http://purl.org/rss/1.0/modules/slash/\";a:1:{s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:1:\"0\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:7;a:6:{s:4:\"data\";s:45:\"\n		\n		\n		\n		\n		\n				\n		\n\n		\n		\n				\n			\n		\n		\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:5:{s:0:\"\";a:7:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:20:\"WordPress 3.9 Beta 2\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:55:\"http://wordpress.org/news/2014/03/wordpress-3-9-beta-2/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:64:\"http://wordpress.org/news/2014/03/wordpress-3-9-beta-2/#comments\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 20 Mar 2014 05:01:25 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"category\";a:2:{i:0;a:5:{s:4:\"data\";s:11:\"Development\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}i:1;a:5:{s:4:\"data\";s:8:\"Releases\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:33:\"http://wordpress.org/news/?p=3101\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:309:\"WordPress 3.9 Beta 2 is now available fortesting! We&#8217;ve made more than a hundred changes since Beta 1, but we still need your help if we&#8217;re going to hit our goal of an April release. For what to look out for, please head on over tothe Beta 1 announcement post. Some of the changes in [&#8230;]\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"Andrew Nacin\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:40:\"http://purl.org/rss/1.0/modules/content/\";a:1:{s:7:\"encoded\";a:1:{i:0;a:5:{s:4:\"data\";s:1901:\"<p>WordPress 3.9 Beta 2 is now available fortesting!</p>\n<p>We&#8217;ve made more than a hundred <a href=\"https://core.trac.wordpress.org/log?rev=27639&amp;stop_rev=27500&amp;limit=200\">changes</a> since Beta 1, but we still need your help if we&#8217;re going to hit our goal of an April release. For what to look out for, please head on over to<a href=\"https://wordpress.org/news/2014/03/wordpress-3-9-beta-1/\">the Beta 1 announcement post</a>. Some of the changes in Beta 2 include:</p>\n<ul>\n<li>Rendering of embedded audio and video players directly in the visual editor.</li>\n<li>Visual and functional improvements to the editor, the media manager, and theme installer.</li>\n<li>Various bug fixes to TinyMCE, the software behind the visual editor.</li>\n<li>Lots of fixesto widget management in the theme customizer.</li>\n</ul>\n<p>As always,<strong>if you think youve found a bug</strong>, you can post to the<a href=\"http://wordpress.org/support/forum/alphabeta\">Alpha/Beta area</a>in the support forums. Or, if youre comfortable writing a reproducible bug report,<a href=\"https://make.wordpress.org/core/reports/\">file one on the WordPress Trac</a>. There, you can also find<a href=\"http://core.trac.wordpress.org/tickets/major\">a list of known bugs</a>and<a href=\"http://core.trac.wordpress.org/query?status=closed&amp;group=component&amp;milestone=3.9\">everything weve fixed</a>so far.</p>\n<p><strong>This software is still in development,</strong>so we dont recommend you run it on a production site. Consider setting up a test site just to play with the new version. To test WordPress 3.9, try the<a href=\"http://wordpress.org/extend/plugins/wordpress-beta-tester/\">WordPress Beta Tester</a>plugin (youll want bleeding edge nightlies). Or you can<a href=\"http://wordpress.org/wordpress-3.9-beta2.zip\">download the beta here</a>(zip).</p>\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:36:\"http://wellformedweb.org/CommentAPI/\";a:1:{s:10:\"commentRss\";a:1:{i:0;a:5:{s:4:\"data\";s:60:\"http://wordpress.org/news/2014/03/wordpress-3-9-beta-2/feed/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:38:\"http://purl.org/rss/1.0/modules/slash/\";a:1:{s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:1:\"0\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:8;a:6:{s:4:\"data\";s:45:\"\n		\n		\n		\n		\n		\n				\n		\n\n		\n		\n				\n			\n		\n		\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:5:{s:0:\"\";a:7:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:20:\"WordPress 3.9 Beta 1\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:55:\"http://wordpress.org/news/2014/03/wordpress-3-9-beta-1/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:64:\"http://wordpress.org/news/2014/03/wordpress-3-9-beta-1/#comments\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Tue, 11 Mar 2014 13:42:19 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"category\";a:2:{i:0;a:5:{s:4:\"data\";s:11:\"Development\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}i:1;a:5:{s:4:\"data\";s:8:\"Releases\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:33:\"http://wordpress.org/news/?p=3083\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:329:\"I&#8217;m excited to announce that the first beta of WordPress 3.9 is now available for testing. WordPress 3.9 is due out next month &#8212; but in order to hit that goal, we need your help testing all of the goodies we&#8217;ve added: We updated TinyMCE, the software powering the visual editor, to the latest version. [&#8230;]\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"Andrew Nacin\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:40:\"http://purl.org/rss/1.0/modules/content/\";a:1:{s:7:\"encoded\";a:1:{i:0;a:5:{s:4:\"data\";s:6065:\"<p>I&#8217;m excited to announce that the <strong>first beta of WordPress 3.9</strong> is now available for testing.</p>\n<p>WordPress 3.9 is due out next month &#8212; but in order to hit that goal, <strong>we need your help</strong> testing all of the goodies we&#8217;ve added:</p>\n<ul>\n<li>We updated TinyMCE, the software powering the visual editor, to the latest version. Be on the lookout for cleaner markup. Also try the new paste handling &#8212; if you paste in a block of text from Microsoft Word, for example, it will no longer come out terrible. (The &#8220;Paste from Word&#8221; button you probably never noticed has been removed.) It&#8217;s possible some plugins that added stuff to the visual editor (like a new toolbar button) no longer work, so we&#8217;d like to hear about them (<a href=\"https://core.trac.wordpress.org/ticket/24067\">#24067</a>). (And be sure to <a href=\"http://wordpress.org/support/\">open a support thread</a> for the plugin author.)</li>\n<li>We&#8217;ve added <strong>widget management to live previews</strong> (the customizer). Please test editing, adding, and rearranging widgets! (<a href=\"https://core.trac.wordpress.org/ticket/27112\">#27112</a>) We&#8217;ve also added the ability to upload, crop, and manage header images, without needing to leave the preview. (<a href=\"https://core.trac.wordpress.org/ticket/21785\">#21785</a>)</li>\n<li>We brought 3.8&#8217;s beautiful new theme browsing experience to the <strong>theme installer</strong>. Checkit out! (<a title=\"View ticket\" href=\"https://core.trac.wordpress.org/ticket/27055\">#27055</a>)</li>\n<li><strong>Galleries</strong> now receive alivepreview in the editor. Upload some photos and insert a gallery to see this in action. (<a href=\"https://core.trac.wordpress.org/ticket/26959\">#26959</a>)</li>\n<li>You can now <strong>drag-and-drop</strong> images directly onto the editor to upload them. It can be a bit finicky, so try it and help us work out the kinks. (<a href=\"https://core.trac.wordpress.org/ticket/19845\">#19845</a>)</li>\n<li>Some things got improved around <strong>editing images</strong>. It&#8217;s a lot easier to make changes to an image after you insert it into a post (<a class=\"closed\" title=\"View ticket\" href=\"https://core.trac.wordpress.org/ticket/24409\">#24409</a>) and you no longer get kicked to a new window when you need to crop or rotate an image (<a href=\"https://core.trac.wordpress.org/ticket/21811\">#21811</a>).</li>\n<li>New <strong>audio/video playlists</strong>. Upload a few audio or video files to test these. (<a href=\"https://core.trac.wordpress.org/ticket/26631\">#26631</a>)</li>\n</ul>\n<p><strong>If you think youve found a bug</strong>, you can post to the <a href=\"http://wordpress.org/support/forum/alphabeta\">Alpha/Beta area</a> in the support forums. We&#8217;d love to hear from you! If youre comfortable writing a reproducible bug report, <a href=\"https://make.wordpress.org/core/reports/\">file one on the WordPress Trac</a>. There, you can also find <a href=\"http://core.trac.wordpress.org/tickets/major\">a list of known bugs</a> and <a href=\"http://core.trac.wordpress.org/query?status=closed&amp;group=component&amp;milestone=3.9\">everything weve fixed</a> so far.</p>\n<p><strong>This software is still in development,</strong> so we dont recommend you run it on a production site. Consider setting up a test site just to play with the new version. To test WordPress 3.9, try the <a href=\"http://wordpress.org/extend/plugins/wordpress-beta-tester/\">WordPress Beta Tester</a> plugin (youll want bleeding edge nightlies). Or you can <a href=\"http://wordpress.org/wordpress-3.9-beta1.zip\">download the beta here</a> (zip).</p>\n<hr />\n<p><strong>DEVELOPERS!</strong> Hello! There&#8217;s lots for you, too.</p>\n<p><strong>Please test your plugins and themes!</strong> There&#8217;s a lot of great stuff under the hood in 3.9 and we hope to blog a bit about them in the coming days. If you haven&#8217;t been reading the awesome <a href=\"http://make.wordpress.org/core/tag/week-in-core/\">weekly summaries</a>on the <a href=\"http://make.wordpress.org/core/\">main core development blog</a>, that&#8217;s a great place to start. (You should definitely follow that blog.) For now, here are some things to watch out for when testing:</p>\n<ul>\n<li>The <strong>load process in multisite</strong> got rewritten. If you notice any issues with your network, see<a href=\"https://core.trac.wordpress.org/ticket/27003\">#27003</a>.</li>\n<li>We now use the <strong>MySQL Improved (mysqli) database extension</strong> if you&#8217;re running a recent version of PHP (<a href=\"https://core.trac.wordpress.org/ticket/21663\">#21663</a>). Please test your plugins and see that everything works well, and please make sure you&#8217;re not calling <code>mysql_*</code> functions directly.</li>\n<li><strong>Autosave</strong> was refactored, so if you see any issues related to autosaving, heartbeat, etc., let us know (<a href=\"https://core.trac.wordpress.org/ticket/25272\">#25272</a>).</li>\n<li>Library updates, in particular Backbone 1.1 and Underscore 1.6 (<a href=\"https://core.trac.wordpress.org/ticket/26799\">#26799</a>). Also Masonry 3 (<a href=\"https://core.trac.wordpress.org/ticket/25351\">#25351</a>), PHPMailer (<a href=\"https://core.trac.wordpress.org/ticket/25560\">#25560</a>), Plupload (<a href=\"https://core.trac.wordpress.org/ticket/25663\">#25663</a>), and TinyMCE (<a href=\"https://core.trac.wordpress.org/ticket/24067\">#24067</a>).</li>\n<li>TinyMCE 4.0 is a <em>major</em> update. Please see TinyMCE&#8217;s <a href=\"http://www.tinymce.com/wiki.php/Tutorial:Migration_guide_from_3.x\">upgrade guide</a> and our <a href=\"https://core.trac.wordpress.org/ticket/24067\">implementation ticket</a> for more. If you have any questions or problems, please <a href=\"http://wordpress.org/support/forum/alphabeta\">open a thread in the support forums</a>.</li>\n</ul>\n<p>Happy testing!</p>\n<p><em><em>Lots of improvements<br />\nLittle things go a long way</em><br />\nPlease test beta one<br />\n</em></p>\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:36:\"http://wellformedweb.org/CommentAPI/\";a:1:{s:10:\"commentRss\";a:1:{i:0;a:5:{s:4:\"data\";s:60:\"http://wordpress.org/news/2014/03/wordpress-3-9-beta-1/feed/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:38:\"http://purl.org/rss/1.0/modules/slash/\";a:1:{s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:1:\"0\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:9;a:6:{s:4:\"data\";s:42:\"\n		\n		\n		\n		\n		\n				\n\n		\n		\n				\n			\n		\n		\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:5:{s:0:\"\";a:7:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:35:\"WordPress 3.8.1 Maintenance Release\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:50:\"http://wordpress.org/news/2014/01/wordpress-3-8-1/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:59:\"http://wordpress.org/news/2014/01/wordpress-3-8-1/#comments\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 23 Jan 2014 20:37:49 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"category\";a:1:{i:0;a:5:{s:4:\"data\";s:8:\"Releases\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:33:\"http://wordpress.org/news/?p=3063\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:358:\"After six weeks and more than 9.3 million downloads of WordPress 3.8, we&#8217;re pleased to announce WordPress 3.8.1 is now available. Version 3.8.1 is a maintenance releases that addresses 31 bugs in 3.8, including various fixes and improvements for the new dashboard design and new themes admin screen. An issue with taxonomy queries in WP_Query [&#8230;]\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"Andrew Nacin\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:40:\"http://purl.org/rss/1.0/modules/content/\";a:1:{s:7:\"encoded\";a:1:{i:0;a:5:{s:4:\"data\";s:3809:\"<p>After six weeks and more than <a href=\"http://wordpress.org/download/counter/\">9.3 million downloads</a> of WordPress 3.8, we&#8217;re pleased to announce WordPress 3.8.1 is now available.</p>\n<p>Version 3.8.1 is a maintenance releases that addresses 31 bugs in 3.8, including various fixes and improvements for the new dashboard design and new themes admin screen. An issue with taxonomy queries in WP_Query was resolved. And if you&#8217;ve been frustrated by submit buttons that won&#8217;t do anything when you click on them (or thought you were going crazy, like some of us), we&#8217;ve found and fixed this &#8220;dead zone&#8221; on submit buttons.</p>\n<p>It also contains a fix for <strong>embedding tweets</strong> (by placing the URL to the tweet on its own line), which was broken due to a recent Twitter API change. (For more on Embeds, see <a href=\"http://codex.wordpress.org/Embeds\">the Codex</a>.)</p>\n<p>For a full list of changes, consult the<a href=\"http://core.trac.wordpress.org/query?milestone=3.8.1\">list of tickets</a>and<a href=\"https://core.trac.wordpress.org/log/branches/3.8?rev=27018&amp;stop_rev=26862\">the changelog</a>. There&#8217;s also a <a href=\"http://make.wordpress.org/core/2014/01/22/wordpress-3-8-1-release-candidate/\">detailed summary</a> for developers on the development blog.</p>\n<p>If you are one of the millions already running WordPress 3.8, we will start rolling out automatic background updates for WordPress 3.8.1 in the next few hours. For sites <a href=\"http://wordpress.org/plugins/background-update-tester/\">that support them</a>, of course.</p>\n<p><a href=\"http://wordpress.org/download/\">Download WordPress 3.8.1</a>or venture over to<strong>Dashboard  Updates</strong>and simply click &#8220;Update Now.&#8221;</p>\n<p>Thanks to all of these fine individuals for contributing to 3.8.1:</p>\n<p><a href=\"http://profiles.wordpress.org/jorbin\">Aaron Jorbin</a>, <a href=\"http://profiles.wordpress.org/collinsinternet\">Allan Collins</a>, <a href=\"http://profiles.wordpress.org/nacin\">Andrew Nacin</a>, <a href=\"http://profiles.wordpress.org/azaozz\">Andrew Ozz</a>, <a href=\"http://profiles.wordpress.org/aubreypwd\">Aubrey Portwood</a>, <a href=\"http://profiles.wordpress.org/empireoflight\">Ben Dunkle</a>, <a href=\"http://profiles.wordpress.org/cojennin\">Connor Jennings</a>, <a href=\"http://profiles.wordpress.org/dd32\">Dion Hulse</a>, <a href=\"http://profiles.wordpress.org/ocean90\">Dominik Schilling</a>, <a href=\"http://profiles.wordpress.org/fboender\">fboender</a>, <a href=\"http://profiles.wordpress.org/avryl\">Janneke Van Dorpe</a>, <a href=\"http://profiles.wordpress.org/janrenn\">janrenn</a>, <a href=\"http://profiles.wordpress.org/joedolson\">Joe Dolson</a>, <a href=\"http://profiles.wordpress.org/johnbillion\">John Blackbourn</a>, <a href=\"#\">Jos Pino</a>, <a href=\"http://profiles.wordpress.org/kovshenin\">Konstantin Kovshenin</a>, <a href=\"http://profiles.wordpress.org/matveb\">Matias Ventura</a>, <a href=\"http://profiles.wordpress.org/mattheu\">Matthew Haines-Young</a>, <a href=\"http://profiles.wordpress.org/iammattthomas\">Matt Thomas</a>, <a href=\"http://profiles.wordpress.org/melchoyce\">Mel Choyce</a>, <a href=\"http://profiles.wordpress.org/batmoo\">Mohammad Jangda</a>, <a href=\"http://profiles.wordpress.org/morganestes\">Morgan Estes</a>, <a href=\"http://profiles.wordpress.org/nivijah\">nivijah</a>, <a href=\"http://profiles.wordpress.org/wonderboymusic\">Scott Taylor</a>, <a href=\"http://profiles.wordpress.org/sergeybiryukov\">Sergey Biryukov</a>, <a href=\"http://profiles.wordpress.org/undergroundnetwork\">undergroundnetwork</a>, and <a href=\"http://profiles.wordpress.org/yurivictor\">Yuri Victor</a>.</p>\n<p><em>WordPress three eight one<br />\nWe heard you didn&#8217;t like bugs<br />\nSo we took them out</em></p>\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:36:\"http://wellformedweb.org/CommentAPI/\";a:1:{s:10:\"commentRss\";a:1:{i:0;a:5:{s:4:\"data\";s:55:\"http://wordpress.org/news/2014/01/wordpress-3-8-1/feed/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:38:\"http://purl.org/rss/1.0/modules/slash/\";a:1:{s:8:\"comments\";a:1:{i:0;a:5:{s:4:\"data\";s:1:\"0\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}}}s:27:\"http://www.w3.org/2005/Atom\";a:1:{s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:0:\"\";s:7:\"attribs\";a:1:{s:0:\"\";a:3:{s:4:\"href\";s:31:\"http://wordpress.org/news/feed/\";s:3:\"rel\";s:4:\"self\";s:4:\"type\";s:19:\"application/rss+xml\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:44:\"http://purl.org/rss/1.0/modules/syndication/\";a:2:{s:12:\"updatePeriod\";a:1:{i:0;a:5:{s:4:\"data\";s:6:\"hourly\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:15:\"updateFrequency\";a:1:{i:0;a:5:{s:4:\"data\";s:1:\"1\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}}}}}}}}s:4:\"type\";i:128;s:7:\"headers\";a:8:{s:6:\"server\";s:5:\"nginx\";s:4:\"date\";s:29:\"Thu, 12 Jun 2014 18:19:15 GMT\";s:12:\"content-type\";s:23:\"text/xml; charset=UTF-8\";s:10:\"connection\";s:5:\"close\";s:4:\"vary\";s:15:\"Accept-Encoding\";s:10:\"x-pingback\";s:36:\"http://wordpress.org/news/xmlrpc.php\";s:13:\"last-modified\";s:29:\"Thu, 08 May 2014 18:40:58 GMT\";s:4:\"x-nc\";s:11:\"HIT lax 249\";}s:5:\"build\";s:14:\"20130911120210\";}','no'),(1390,'_transient_timeout_feed_mod_ac0b00fe65abe10e0c5b588f3ed8c7ca','1402640355','no'),(1391,'_transient_feed_mod_ac0b00fe65abe10e0c5b588f3ed8c7ca','1402597155','no'),(1392,'_transient_timeout_feed_867bd5c64f85878d03a060509cd2f92c','1402640355','no'),(1393,'_transient_feed_867bd5c64f85878d03a060509cd2f92c','a:4:{s:5:\"child\";a:1:{s:0:\"\";a:1:{s:3:\"rss\";a:1:{i:0;a:6:{s:4:\"data\";s:3:\"\n\n\n\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:7:\"version\";s:3:\"2.0\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:1:{s:0:\"\";a:1:{s:7:\"channel\";a:1:{i:0;a:6:{s:4:\"data\";s:61:\"\n	\n	\n	\n	\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:1:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:16:\"WordPress Planet\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://planet.wordpress.org/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"language\";a:1:{i:0;a:5:{s:4:\"data\";s:2:\"en\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:47:\"WordPress Planet - http://planet.wordpress.org/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"item\";a:50:{i:0;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:48:\"Ping-O-Matic: Tips on Growth From a Book Blogger\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:33:\"http://blog.pingomatic.com/?p=107\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:56:\"http://blog.pingomatic.com/2014/06/12/book-blogger-tips/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:4569:\"<p><a href=\"http://101books.net/\"><img class=\"aligncenter size-large wp-image-108\" alt=\"\" src=\"http://pingomatic.files.wordpress.com/2014/03/screen-shot-2014-03-06-at-12-13-28-pm.png?w=600&h=107\" /></a><br />\nOver on<a href=\"https://dailypost.wordpress.com/\" target=\"_blank\"><em>The Daily Post</em></a>, weve published interviews with writers on growth and readership. Robert Bruce, the blogger behind the popular book site,<a href=\"http://101books.net/\" target=\"_blank\"><em>101 Books</em></a>, shared bits of advice on building a loyal readership and successful niche blog. Here are snippets from his Q&amp;A:</p>\n<h3>You&#8217;ve got over 26,000 followers and counting. What&#8217;s your secret?</h3>\n<p>Theres really no secret. Its just steady, consistent posting over a long period of time.<em>101 Books</em>is more than three years old now, and Ive had more than 700 posts. When you post that often, people are bound to find you. Then, the key is to just write content that relates to them. Most people dont care about what you had for breakfast, but if you can help them learn something new, then theyll keep coming back.</p>\n<h3>What types of posts perform better?</h3>\n<p>The funny thing about my blog is that, even though its centered on <a href=\"http://101books.net/faq/\" target=\"_blank\">the 101 Books project</a>, these book reviews dont perform as well as the quirkier stuff.One of the most popular posts Ive had was a post about<a href=\"http://101books.net/2013/04/26/my-2-year-old-judges-books-by-their-covers/\" target=\"_blank\">my two-year-old son judging books by their covers</a>. I put a couple of classic book covers in front of him and asked him what he thought they were about. His answers were hilarious. That post took about 15 minutes to put together, but because it was unique and fresh, it became a hit.</p>\n<p>Obviously, list-style posts do well, and I probably tend to overuse them because of that. (Im not BuzzFeed.) Also, for whatever reason, people gravitate to more negative-sounding titles, like<a href=\"http://101books.net/?s=horrible+death\" target=\"_blank\">7 Words That Should Die A Horrible Death.</a></p>\n<h3>There are many blogs about books on the web. Why do you think yours has been so successful?</h3>\n<blockquote><p>I think people can easily get behind the idea of someone pursuing a crazy goal and the ups and downs that come with that.</p></blockquote>\n<p>Theres a lot of great book blogs out there, and a lot of bloggers who write incredibly detailed book reviews. My blog is a little different because I review books in small chunks; I take a small passage from a book and write about it. Or I write about some cool, unusual fact from the authors background. So I think it stands out a bit in the book blogging world.</p>\n<p>Plus, I think people can easily get behind the idea of someone pursuing a crazy goal and the ups and downs that come with that. Its like a literary version of the<em>Julie and Julia</em>book (and movie). Not that Im near as creative and successful as she was, but you get the point.</p>\n<h3>Do you have tips for someone who wants to focus their blog on books?</h3>\n<blockquote><p>Sometimes youve just got to push the publish button because an almost-perfect blog post is better than no post at all.</p></blockquote>\n<p>Be honest. Dont feel like you have to like a book or dislike a book because of what the critics say. On my blog, Im very vocal of my dislike for<em>Mrs. Dalloway</em>, but its my honest opinion.</p>\n<p>If you want to write a book blog with an academic voice, thats great. But youll probably realize that not many people will read it. I try to write about literature in an approachable way, and that style involves<em>forgetting</em>what my English literature professor taught me.</p>\n<p>I think its also important to<a href=\"http://101books.net/2013/04/19/3-tips-for-a-better-blog/\" target=\"_blank\">forget about being perfect</a>. Sometimes youve just got to push the publish button because an almost-perfect blog post is better than no post at all. Dont pass over the great in search of the perfect.</p><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/pingomatic.wordpress.com/107/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/pingomatic.wordpress.com/107/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=blog.pingomatic.com&blog=68432&post=107&subd=pingomatic&ref=&feed=1\" width=\"1\" height=\"1\" />\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 12 Jun 2014 16:00:25 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:20:\"Cheri Lucas Rowlands\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:1;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:106:\"WPTavern: Clef Partners With Softaculous To Provide 2-Step Authentication For Automated WordPress Installs\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wptavern.com/?p=24533\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:256:\"http://wptavern.com/clef-partners-with-softaculous-to-provide-2-step-authentication-for-automated-wordpress-installs?utm_source=rss&utm_medium=rss&utm_campaign=clef-partners-with-softaculous-to-provide-2-step-authentication-for-automated-wordpress-installs\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:3697:\"<p><a title=\"https://getclef.com/\" href=\"https://getclef.com/\">Clef</a>, the two-step authentication service that replaces usernames and passwords <a title=\"http://blog.getclef.com/two-steps-foward/\" href=\"http://blog.getclef.com/two-steps-foward/\">has announced</a> a strategic partnership with <a title=\"http://www.softaculous.com/\" href=\"http://www.softaculous.com/\">Softaculous</a>. Softaculous is similar to Fantastico in that it contains several scripts that can be auto installed on a webhosting account. The partnership enables automated WordPress installs from Softaculous to have Clef support out of the box.</p>\n<div id=\"attachment_24588\" class=\"wp-caption aligncenter\"><a href=\"http://i2.wp.com/wptavern.com/wp-content/uploads/2014/06/Clef+Softaculous.png?resize=1025%2C361\" rel=\"prettyphoto[24533]\"><img class=\"size-full wp-image-24588\" src=\"http://i2.wp.com/wptavern.com/wp-content/uploads/2014/06/Clef+Softaculous.png?resize=1025%2C361\" alt=\"Clef + Softaculous = Safer Auto Installs Of WordPress\" /></a><p class=\"wp-caption-text\">Clef + Softaculous = Safer Auto Installs Of WordPress</p></div>\n<p>In one simple step, webhosts will be able to configure their auto install of WordPress to support Clef by navigating to<strong>Softaculous Admin panel -&gt; Software -&gt; Advanced Settings</strong> and checking the box to enable Clef. Administrators can find more information on <a title=\"http://www.softaculous.com/docs/Advanced_Software_Settings\" href=\"http://www.softaculous.com/docs/Advanced_Software_Settings\">configuring the script here</a>. There&#8217;s also a <a title=\"http://demo.softaculous.com/index.php?act=adv_software\" href=\"http://demo.softaculous.com/index.php?act=adv_software\">live demo</a> to see how it works.</p>\n<p>Clef has also announced the beginning of a new program called <strong>SHIP</strong> or Safer Host Incentive Program. Clef will be made available for free for a host to use on its own domain if they make two-factor authentication an option for their customers. The service will remain free for sites with less than 100 people logging in.</p>\n<p>I asked Brennen Byrne,CEO and Co-founder of Clef, why is this partnership a big deal? He replied,</p>\n<blockquote><p>The future of WordPress security is about protecting every site, not just the big ones  including Clef in Softaculous WordPress installs will help establish a stronger baseline of security for everyone. Good security is something you should opt out of and hosts should be giving users better access to the security tools they need</p></blockquote>\n<p>Clef has experienced rapid growth since their launch.</p>\n<ul>\n<li>The Clef plugin is being downloaded more times each day than any other two-factor plugin is in a week</li>\n<li>Clef is being used by hedge funds to protect almost $2B in managed wealth</li>\n<li>Clef has doubled in size every six weeks since the beginning of the year</li>\n</ul>\n<p>When asked whether they tried to work with Fantastico as well as Softaculous, Byrne replied, &#8220;We&#8217;ve been working with Softaculous for a while to make this happen &#8212; they really love the product and it&#8217;s just been a matter of timing and development resources. We haven&#8217;t talked to any of the other installers, but we&#8217;ll definitely be looking to work with more of them soon.&#8221;</p>\n<p>With this partnership, thousands if not millions of WordPress auto installs will be safer immediately after installation. It also gives Clef an opportunity to vastly improve their reach into the WordPress ecosystem. Do you think this move will help make the web a safer place by improving the baseline of security for automated WordPress installations?</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 12 Jun 2014 15:10:29 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Jeff Chandler\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:2;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:36:\"Matt: Extended Interview with Forbes\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:21:\"http://ma.tt/?p=43838\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:52:\"http://ma.tt/2014/06/extended-interview-with-forbes/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:915:\"<p>J. J. Colao, <a href=\"http://ma.tt/2012/09/future-of-work/\">who covered Automattic for Forbes Magazine in 2012</a> and has a long history and experience with WordPress and Automattic, sat down with me for close to three hours in March and  <a href=\"http://www.forbes.com/sites/jjcolao/2014/06/11/an-extended-interview-with-wordpress-creator-matt-mullenweg/\">somehow managed to distill it down to just a few thousand words of interview</a>. (&#8220;13,500 words down to 2,800.&#8221;) I&#8217;m sheepish to link it because there are a lot of &#8220;I&#8221; statements and some nuance lost in the distillation, but JJ asks great questions and we cover a lot of ground that anyone who follows Automattic or the WordPress ecosystem I think will find interesting. <a href=\"http://www.forbes.com/sites/jjcolao/2014/06/11/an-extended-interview-with-wordpress-creator-matt-mullenweg/\">You can check it out here</a>.</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 12 Jun 2014 08:30:07 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:14:\"Matt Mullenweg\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:3;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:51:\"WPTavern: PressWork to Relaunch Under New Ownership\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wptavern.com/?p=24538\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:146:\"http://wptavern.com/presswork-to-relaunch-under-new-ownership?utm_source=rss&utm_medium=rss&utm_campaign=presswork-to-relaunch-under-new-ownership\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:3943:\"<p><a href=\"http://i2.wp.com/wptavern.com/wp-content/uploads/2014/06/presswork.png\" rel=\"prettyphoto[24538]\"><img src=\"http://i2.wp.com/wptavern.com/wp-content/uploads/2014/06/presswork.png?resize=976%2C352\" alt=\"presswork\" class=\"aligncenter size-full wp-image-24566\" /></a></p>\n<p>PressWork made a big splash in the WordPress theme world when it launched its free HTML5/CSS3 framework in the summer of 2011. The theme boasted a unique drag and drop front-end editor which allowed users to customize their sites without having to touch any code.</p>\n<p><a href=\"https://twitter.com/digibomb\" target=\"_blank\">Brendan Sera-Shriar</a> and <a href=\"https://twitter.com/bavotasan\" target=\"_blank\">Chris Bavota</a> put a great deal of effort into the framework only to <a href=\"http://wpcandy.com/reports/presswork-folds/#.U5kcAZSwKwI\" target=\"_blank\">close the doors</a> one year later after the two admitted that they could not successfully monetize it and no longer had time to devote to the project. The popularity of the framework gradually fizzled out over the years, as it languished on GitHub without updates.</p>\n<p>Today the new PressWork website <a href=\"http://presswork.io/presswork-is-back/\" target=\"_blank\">announced</a> that the project&#8217;s founders have given the keys over to WordPress developer Emil Uzelac. As an active member of the <a href=\"http://wptavern.com/a-day-in-the-life-of-a-wordpress-theme-reviewer\" target=\"_blank\">WordPress.org Theme Review Team</a>, Uzelac has a solid understanding of best practices and the passion to revive PressWork.</p>\n<p>I spoke with him to find out the details of the PressWork sale, only to discover that the founders actually gave him the project, branding and all, without no strings attached.</p>\n<p>&#8220;A couple of weeks ago I got in touch with c.bavota and asked if they have any future plans with the PressWork and if I could &#8216;take over,&#8217;&#8221; Uzelac told the Tavern. &#8220;The next day I received an e-mail along with c.bavotas &#8216;Yes, you can.&#8217;&#8221; The PressWork GitHub repo has now been transferred to Uzelac&#8217;s account. The original PressWork team seemed eager to offload the project to someone who would give it a good home. &#8220;I&#8217;m not really sure if the guys had any interest in it anymore,&#8221; Uzelac said. <strong>&#8220;I simply asked for it.&#8221;</strong></p>\n<p>If you&#8217;ve ever been given a non-working car, then you have a good foundation for understanding this transaction. PressWork received next to no contributions after the project was abandoned. Essentially, the framework needs a few updates to make it compatible with the latest version of WordPress, but there&#8217;s huge potential there if Uzelac can bring the project back up to speed.</p>\n<p>The fatal error of the original PressWork team was that they had no commercial backbone to sustain the framework&#8217;s continual development. Uzelac confirmed that he plans to relaunch with commercial products. <strong>&#8220;If I learned anything from developing the Responsive Theme, it&#8217;s that you have to add commercial products in the order to survive,&#8221;</strong> he said.</p>\n<p>Uzelac is in the process of updating the framework. &#8220;My main plan is to get the PressWork back in WordPress directory,&#8221; he said. &#8220;After that, a new modern look will be in order.&#8221;</p>\n<p>Before the original PressWork team closed the doors, they <a href=\"https://web.archive.org/web/20130112204202/http://presswork.me/\" target=\"_blank\">reported</a> that tens of thousands of WordPress users had downloaded the theme. The framework now has a chance at a strong future. Instead of disappearing into the black hole of abandoned WordPress projects, <a href=\"http://presswork.io/\" target=\"_blank\">PressWork</a> has been adopted by a developer who is passionate about restoring its usefulness. What do you think Uzelac can do with it?</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 12 Jun 2014 05:48:54 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Sarah Gooding\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:4;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:91:\"WPTavern: New Data Shows WordPress Support Forum Registrations Surged Between 2009 and 2012\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wptavern.com/?p=24511\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:226:\"http://wptavern.com/new-data-shows-wordpress-support-forum-registrations-surged-between-2009-and-2012?utm_source=rss&utm_medium=rss&utm_campaign=new-data-shows-wordpress-support-forum-registrations-surged-between-2009-and-2012\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:3430:\"<p>A few days ago, Siobhan McKeown published a graphic <a title=\"https://twitter.com/SiobhanPMcKeown/status/475424535534379010\" href=\"https://twitter.com/SiobhanPMcKeown/status/475424535534379010\">on Twitter</a> that shows sign ups to the WordPress.org support forums between 2005 and 2014. Notice the huge increase in registrations between 2009 and 2010.</p>\n<div id=\"attachment_24524\" class=\"wp-caption aligncenter\"><a href=\"http://i0.wp.com/wptavern.com/wp-content/uploads/2014/06/WPorgSupportForumSignups.png\" rel=\"prettyphoto[24511]\"><img class=\"size-full wp-image-24524\" src=\"http://i0.wp.com/wptavern.com/wp-content/uploads/2014/06/WPorgSupportForumSignups.png?resize=1025%2C702\" alt=\"Signups To The WordPress.org Support Forums\" /></a><p class=\"wp-caption-text\">Signups To The WordPress.org Support Forums</p></div>\n<p>I asked McKeown what could account for the surge and <a title=\"https://twitter.com/SiobhanPMcKeown/status/476137367590690816\" href=\"https://twitter.com/SiobhanPMcKeown/status/476137367590690816\">she believes</a> it&#8217;s the release of WordPress 3.0. WordPress 3.0 <a title=\"http://wordpress.org/news/2010/06/thelonious/\" href=\"http://wordpress.org/news/2010/06/thelonious/\">was released</a> on June 17, 2010 and was a blockbuster release. For the first time, WordPress Multisite was part of the same package as WordPress.</p>\n<p>As McKeown explains, a spike in signups could reflect a number of things:</p>\n<blockquote><p>a) that a release has generated a spike in users , and/or b) that something in a new WordPress release is complex enough to require additional help in the support forums.</p>\n<p>There could be other environmental factors that cause new signups &#8211; for example, the month that Movable Type changed its license (and WordPress 1.2 came out with the new plugin architecture) signups more than doubled and stayed higher in the subsequent months.</p></blockquote>\n<p>When asked what other plans she has in store for the data, McKeown plans on looking at monthly signups in relation to WordPress releases and will plot graphs that visually display the relation between WordPress releases to forum signups.</p>\n<h3>No Meaningful Results</h3>\n<p>The data is not publicly available due to privacy concerns and was obtained by McKeown through Samuel Wood by request for the <a title=\"https://github.com/wordpress/book\" href=\"https://github.com/wordpress/book\">WordPress history book</a> she is writing.</p>\n<p>According to Wood, when compiling the data, he did not attempt to account for spam accounts. In 2012, a CAPTCHA was added to the user registration area of the support forums but it was only partially successful at combating the problem. &#8220;Because of this, I don&#8217;t think any meaningful conclusions can be drawn from those numbers. They are interesting and show interesting trends, but not any meaningful results to base any actions off of.&#8221; Wood said.</p>\n<h3>What Data Points Would You Like To See?</h3>\n<p>Regardless of whether or not the data is scientifically accurate, I still find the numbers interesting. However, without being able to review the data in months instead of years, it will be hard to come up with any specific correlations. I think it would be neat to see an average of how long a new user is active on the forums and whether that average has increased or decreased over time.</p>\n<p>What ideas or trends can you gleam from the data?</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 12 Jun 2014 03:01:45 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Jeff Chandler\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:5;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:80:\"WPTavern: Michael Oeser of DER PRiNZ Acquires WordPress Theme Company ThemeShift\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wptavern.com/?p=24471\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:204:\"http://wptavern.com/michael-oeser-of-der-prinz-acquires-wordpress-theme-company-themeshift?utm_source=rss&utm_medium=rss&utm_campaign=michael-oeser-of-der-prinz-acquires-wordpress-theme-company-themeshift\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:3219:\"<p>In what has become a trend of late, another WordPress commercial theme shop has exchanged hands. ThemeShift, originally owned and operated by Simon Rimkus is now <a title=\"http://themeshift.com/news/themeshift-has-a-new-owner/\" href=\"http://themeshift.com/news/themeshift-has-a-new-owner/\">owned by Michael Oeser</a> of <a title=\"http://www.der-prinz.com/\" href=\"http://www.der-prinz.com/\">DER PRiNZ</a>, a German based WordPress theme and digital media company.</p>\n<p><a href=\"http://i0.wp.com/wptavern.com/wp-content/uploads/2014/06/ThemeShiftFeaturedImage.png\" rel=\"prettyphoto[24471]\"><img class=\"aligncenter size-full wp-image-24500\" src=\"http://i0.wp.com/wptavern.com/wp-content/uploads/2014/06/ThemeShiftFeaturedImage.png?resize=623%2C230\" alt=\"ThemeShift Featured Image\" /></a></p>\n<p>ThemeShift has been in the WordPress commercial theme space since 2009. In an announcement post by Rimkus, he states the main reason for selling the business is a lack of time.</p>\n<blockquote><p>However, I have to make this step because with a new WordPress project that I started Im not able to dedicate the time to ThemeShift the project actually deserves. As you might have noticed, things at ThemeShift werent really evolving lately.</p></blockquote>\n<p>Rimkus is focusing his efforts towards the WordPress framework <a href=\"http://wpcasa.com/\" target=\"_blank\">wpCasa</a> which will be part of <a title=\"http://wpsight.com/\" href=\"http://wpsight.com/\">WPSight</a>. As for ThemeShift, Oeser says there will be no price increase and the number of themes available will rise as he migrates themes fromDER PRiNZ to ThemeShift using the <a title=\"http://themeshift.com/frameshift/\" href=\"http://themeshift.com/frameshift/\">FrameShift</a> framework.</p>\n<p>The acquisition of ThemeShift is just one of many to happen in recent months. Here is a list of some other companies that have been acquired recently.</p>\n<ul>\n<li>Press75 <a title=\"http://wptavern.com/press75-acquired-by-westwerk-for-undisclosed-amount\" href=\"http://wptavern.com/press75-acquired-by-westwerk-for-undisclosed-amount\">acquired by Westwerk</a></li>\n<li>DevPress <a title=\"http://www.poststat.us/devin-price-new-owner-devpress/\" href=\"http://www.poststat.us/devin-price-new-owner-devpress/\">acquired by Deven Price</a></li>\n<li>Themelab <a title=\"http://wptavern.com/themelab-acquired-by-syed-balkhi\" href=\"http://wptavern.com/themelab-acquired-by-syed-balkhi\">acquired by Syed Balkhi</a></li>\n<li>CreativeMarket <a title=\"http://wptavern.com/creativemarket-acquired-by-the-makers-of-autocad\" href=\"http://wptavern.com/creativemarket-acquired-by-the-makers-of-autocad\">acquired by Autodesk</a></li>\n<li>Just The Lucid Themes <a title=\"http://www.poststat.us/upthemes-adds-lucid-theme-collection-reportoire/\" href=\"http://www.poststat.us/upthemes-adds-lucid-theme-collection-reportoire/\">acquired by UpThemes</a></li>\n<li>Standard Theme by 8bit <a title=\"http://www.woothemes.com/2013/09/standard-theme-lives-on/\" href=\"http://www.woothemes.com/2013/09/standard-theme-lives-on/\">acquired by WooThemes</a></li>\n</ul>\n<p>Let us know in the comments which WordPress commercial theme company you think will be the next to be acquired and why.</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 12 Jun 2014 02:36:19 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Jeff Chandler\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:6;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:72:\"WPTavern: WP Engine to Sponsor Two Full-Time WordPress Core Contributors\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wptavern.com/?p=24495\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:188:\"http://wptavern.com/wp-engine-to-sponsor-two-full-time-wordpress-core-contributors?utm_source=rss&utm_medium=rss&utm_campaign=wp-engine-to-sponsor-two-full-time-wordpress-core-contributors\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:4107:\"<div id=\"attachment_24519\" class=\"wp-caption aligncenter\"><a href=\"http://i2.wp.com/wptavern.com/wp-content/uploads/2014/06/open-source-job.png\" rel=\"prettyphoto[24495]\"><img src=\"http://i2.wp.com/wptavern.com/wp-content/uploads/2014/06/open-source-job.png?resize=700%2C330\" alt=\"photo credit: cc license - opensource.com\" class=\"size-full wp-image-24519\" /></a><p class=\"wp-caption-text\">photo credit: cc license &#8211; opensource.com</p></div>\n<p>WP Engine is currently on the hunt to hire two developers to work on the open source WordPress project full-time. This is a relatively new trend that has caught on among companies that sell WordPress products and/or services. Last November, 10up <a href=\"http://wptavern.com/10up-sponsors-helen-hou-sandi-to-work-full-time-on-wordpress-core\" target=\"_blank\">announced</a> that it would be sponsoring Helen Hou-Sandi to work full-time on WordPress core. Many other WordPress-related companies and agencies regularly sponsor a portion of their employees time to contribute to WordPress, WordCamps and other open source projects.</p>\n<p>The <a href=\"https://hire.jobvite.com/Jobvite/Job.aspx?m=nu65snwZ&o=34&j=oraPYfw8\" target=\"_blank\">WP Engine job advertisement</a> says that the developers selected will be responsible for submitting new functionality to the WordPress code base, making it more accessible and easy to use, keeping up with modern trends, and evangelizing the benefits of WordPress to the Austin, TX community and abroad. Tomas Puig, Head of marketing at WP Engine, said that they haven&#8217;t been getting enough of the right applicants so far, even though the jobs have been open since mid-April.</p>\n<p>Puig confirmed that the contributors WP Engine aims to hire will be 100% devoted to core and will occasionally interface with other open source projects such as NGINX. Applicants are not rigidly required to have contributed to core previously, but it certainly helps.</p>\n<p><strong>&#8220;Our goal is to have people who have contributed to core in the past and this enables them to realize their dream about it becoming a full time job,&#8221;</strong> Puig said. &#8220;Though if an amazing developer that hasn&#8217;t contributed applies we&#8217;d like them to talk with Nacin and see if there is a fit.&#8221;</p>\n<p>Puig said most of the current applicants have submitted patches to core, or plugins to the official directory. WP Engine has been speaking with WordPress lead developer Andrew Nacin to find recommendations for qualified applicants. The final hiring selection, however, will be done by WP Engine.</p>\n<p>What is WP Engine looking for, specifically? <strong>&#8220;Our first priorities is making sure they get along with the community and are fantastic engineers,&#8221;</strong> Puig said. &#8220;I&#8217;m really hopeful a known developer with commit access comes forward to apply.&#8221;</p>\n<p>WP Engine is hoping to relocate the selected developers to Austin, TX. &#8220;They do not have to relocate but our preference is that they are Austin or SF,&#8221; Puig said, as the employees would need to be on hand to fly in for company wide meetings.</p>\n<p>It&#8217;s not easy to find solid core contributors, much less ones willing to relocate. WordPress talent is in high demand and most of the developers of that caliber could work virtually anywhere. However, Puig is hopeful that they will find the right fit for giving back to the project.</p>\n<blockquote><p>If we&#8217;re going to fund full time positions we want them to be huge assets to the project. But in all honesty finding great developers for any company or project is hard. We have a few applications in process but I wanted to get the word out much farther.</p></blockquote>\n<p>Working on the WordPress open source project full time would be a dream job for many developers who love this software. The challenge for WP Engine is getting the word out and attracting high quality engineers. The prospect of two more high level contributors giving back to WordPress full time is good news and can only serve to propel the project further ahead.</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Wed, 11 Jun 2014 20:06:04 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Sarah Gooding\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:7;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:54:\"WordPress.tv: Dave Winter: Doin It with Admin Style\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=35617\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:68:\"http://wordpress.tv/2014/06/11/dave-winter-doin-it-with-admin-style/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:660:\"<div id=\"v-o7lhFh0N-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/35617/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/35617/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=35617&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/11/dave-winter-doin-it-with-admin-style/\"><img alt=\"Dave Winter: Doin It with Admin Style\" src=\"http://videos.videopress.com/o7lhFh0N/video-16e99b1948_scruberthumbnail_0.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Wed, 11 Jun 2014 18:55:55 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:8;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:62:\"WPTavern: Editor: A Beautiful Free WordPress Theme for Writers\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wptavern.com/?p=24437\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:166:\"http://wptavern.com/editor-a-beautiful-free-wordpress-theme-for-writers?utm_source=rss&utm_medium=rss&utm_campaign=editor-a-beautiful-free-wordpress-theme-for-writers\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:3697:\"<p><a href=\"https://array.is/\" target=\"_blank\">Array</a>, the WordPress theme shop formerly known as Okay Themes, has branched out into WordPress.org with the release of <a href=\"http://wordpress.org/themes/editor\" target=\"_blank\">Editor</a>, a beautiful new blogging theme. Editor is stamped with Array&#8217;s trademark simplicity and bold style. It was designed to have images, videos, and text scale down nicely for reading on any mobile device.</p>\n<p><a href=\"http://i2.wp.com/wptavern.com/wp-content/uploads/2014/06/editor.jpg\" rel=\"prettyphoto[24437]\"><img src=\"http://i2.wp.com/wptavern.com/wp-content/uploads/2014/06/editor.jpg?resize=880%2C660\" alt=\"editor\" class=\"aligncenter size-full wp-image-24477\" /></a></p>\n<p>Editor includes options in the customizer for adding your own logo and setting the featured posts category that will appear in the sidebar. You can also assign a background color or image for the site and a color for the sidebar.</p>\n<p>The use of big, bold typography and large format featured images makes Editor a stunning theme. As it was designed for writers, it includes unique styling for blockquotes and pull quotes to highlight information within articles.</p>\n<p><a href=\"http://i1.wp.com/wptavern.com/wp-content/uploads/2014/06/gallery.jpg\" rel=\"prettyphoto[24437]\"><img src=\"http://i1.wp.com/wptavern.com/wp-content/uploads/2014/06/gallery.jpg?resize=300%2C268\" alt=\"gallery\" class=\"alignright size-medium wp-image-24491\" /></a>Editor was created to work nicely with several Jetpack modules, including infinite scroll, contact forms, carousels and mosaic galleries. This enables the theme to incorporate styles for more functionality while not having to bear the burden of supporting the associated plugins.</p>\n<p>Check out a <a href=\"http://themes.array.is/editor/\" target=\"_blank\">live demo</a> of the theme to see it in action, and make sure to view the <a href=\"http://themes.array.is/editor/style-guide/\" target=\"_blank\">Style Guide</a> to see pull quotes, tables, highlights, and gallery examples.</p>\n<p>Mike McAlister decided to rebrand Array after <a href=\"http://wptavern.com/okay-themes-pulls-out-of-themeforest-rebrands-as-array\" target=\"_blank\">pulling his products out of ThemeForest</a>. His team was excited to give something back to the WordPress world, hence the release of Editor.</p>\n<p>&#8220;With the recent launch of Array, we thought it was a good opportunity to give back to the community on a larger scale and create a theme that would empower people to publish beautiful content with minimal effort,&#8221; McAlister said. His approach to releasing a free theme is rather unique in that it&#8217;s not a &#8220;lite version&#8221; of a better, commercial copy:</p>\n<blockquote><p>Although Editor is free, it is by no means limited or any less capable than a commercial theme. We put the same attention to detail into Editor as we do all of the themes in the Array collection. It was important to us that we create a theme that didnt feel free or have any catch. No up-selling, no upgrade nags, no funny business. Just a unique and well-crafted theme thats super easy to use from the second you activate it.</p></blockquote>\n<p>This is Array&#8217;s first theme on WordPress.org and if the experiment goes well, the team hopes to release another. McAlister said that they&#8217;ll be submitting Editor to WordPress.com in the coming days, so if you&#8217;re hosted there you may be able to use it soon.</p>\n<p>Right now, it&#8217;s only available for self-hosted WordPress blogs. Download <a href=\"http://wordpress.org/themes/editor\" target=\"_blank\">Editor</a> for free on WordPress.org or via your admin themes browser.</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Wed, 11 Jun 2014 18:45:46 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Sarah Gooding\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:9;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:85:\"WordPress.tv: Tabitha Marie Chapman: Functions.php vs Plugins  The Ultimate Battle\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=35664\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:98:\"http://wordpress.tv/2014/06/11/tabitha-marie-chapman-functions-php-vs-plugins-the-ultimate-battle/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:725:\"<div id=\"v-wgf3NEMR-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/35664/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/35664/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=35664&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/11/tabitha-marie-chapman-functions-php-vs-plugins-the-ultimate-battle/\"><img alt=\"Tabitha Marie Chapman: Functions.php vs Plugins &#8211; The Ultimate Battle\" src=\"http://videos.videopress.com/wgf3NEMR/video-c108f56300_scruberthumbnail_0.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Wed, 11 Jun 2014 18:18:32 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:10;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:84:\"WordPress.tv: Steve Zehngut: Build a WordPress Theme with Foundation and Underscores\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=35662\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:101:\"http://wordpress.tv/2014/06/11/steve-zehngut-build-a-wordpress-theme-with-foundation-and-underscores/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:723:\"<div id=\"v-ck2dL1QI-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/35662/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/35662/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=35662&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/11/steve-zehngut-build-a-wordpress-theme-with-foundation-and-underscores/\"><img alt=\"Steve Zehngut: Build a WordPress Theme with Foundation and Underscores\" src=\"http://videos.videopress.com/ck2dL1QI/video-4b084bef36_scruberthumbnail_0.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Wed, 11 Jun 2014 18:08:38 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:11;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:35:\"WPTavern: Why Comments Still Matter\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wptavern.com/?p=24451\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:114:\"http://wptavern.com/why-comments-still-matter?utm_source=rss&utm_medium=rss&utm_campaign=why-comments-still-matter\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:11374:\"<div id=\"attachment_23428\" class=\"wp-caption aligncenter\"><a href=\"http://i0.wp.com/wptavern.com/wp-content/uploads/2014/05/business-coffee.jpg\" rel=\"prettyphoto[24451]\"><img src=\"http://i0.wp.com/wptavern.com/wp-content/uploads/2014/05/business-coffee.jpg?resize=1022%2C488\" alt=\"photo credit: DaveFayram - cc\" class=\"size-full wp-image-23428\" /></a><p class=\"wp-caption-text\">photo credit: <a href=\"http://www.flickr.com/photos/davefayram/4326574134/\">DaveFayram</a> &#8211; <a href=\"http://creativecommons.org/licenses/by/2.0/\">cc</a></p></div>\n<p>Earlier this year Copyblogger set the blogosphere ablaze with its controversial <a href=\"http://www.copyblogger.com/removing-blog-comments/\" target=\"_blank\">decision to turn off comments</a>. The announcement cited the burden of moderation, spam, and the availability of other conversation &#8220;outposts&#8221; as the main reasons for closing comments. It also gave the impression that Copyblogger had in many ways outgrown comments:</p>\n<blockquote><p>Blog comments are an amazing resource for any blog when its getting started. Its a super convenient way to take the pulse of your audience&#8230;. And for eight years, comments have been a fantastic thing here on Copyblogger.</p>\n<p>Were fortunate enough (mainly because Copybloggers been around for so long now) that we have a lot of thriving outposts where conversation happens.</p></blockquote>\n<p>Copyblogger isn&#8217;t the only site to turn the lights off on commenters. Popular Science, powered by Drupal, <a href=\"http://www.popsci.com/science/article/2013-09/why-were-shutting-our-comments\">shut off comments</a> in September 2013, citing trolls, spambots, and a &#8220;fractious minority&#8221; detracting from its ability to foster intelligent conversation and debate. The editors felt that commenters often cast aspersion on the the validity of the science presented, prefacing the announcment with: <strong>&#8220;Comments can be bad for science. That&#8217;s why, here at PopularScience.com, we&#8217;re shutting them off.&#8221;</strong></p>\n<h3>The Rising Trend of Turning Comments Off</h3>\n<p>Copyblogger is likely the most influential WordPress site to adopt a comments-off approach to blogging, and <a href=\"http://mattgemmell.com/comments-off/\" target=\"_blank\">many</a> <a href=\"http://govcon.net/why-we-turned-off-comments-on-the-usa-gov-blog/\" target=\"_blank\">other</a> <a href=\"http://www.twistimage.com/blog/archives/shutting-down-blog-comments/\" target=\"_blank\">sites</a> have followed suit.</p>\n<p>The other day I noticed an influx of comment-killing plugins coming into the WordPress Plugin Directory. <a href=\"http://wordpress.org/plugins/simple-no-comments/\" target=\"_blank\">Simple No Comments</a> is a plugin that turns them off upon activation. <a href=\"http://wordpress.org/plugins/wp-disable-comments/\" target=\"_blank\">WP Disable Comments</a> entered the directory right around the same time with more fine-grained control for disabling comments globally or for specific post types, categories, tags, referrers, etc. <a href=\"http://wordpress.org/plugins/hear-no-evil/\" target=\"_blank\">Hear No Evil</a> is a relatively new one that blocks comments for a WordPress site or on a site-by-site basis in a multisite network.</p>\n<p>WordPress users are constantly on the hunt for better tools to help manage commenting and mitigate the unrelenting onslaught of spam. Publications turn comments off for different reasons, but it&#8217;s rarely due to the fact that they do not appreciate the comments left by genuine community members. Oftentimes the burden of spam moderation becomes greater than the benefit of conversation on posts.</p>\n<p>Comment moderation was introduced in <a href=\"http://wordpress.org/news/2004/01/wordpress-10/\">WordPress 1.0</a> in 2004 but at that time spammers were operating at a fraction of their current strength. Akismet <a href=\"http://blog.akismet.com/2014/06/01/may-stats-roundup/\" target=\"_blank\">reports</a> that the amount of spam it caught in May 2014 is up 4% from April, and up 269% from May 2012. This burden increases every year and even with powerful spam-blocking plugins in place, many will slip through the cracks.</p>\n<p>The solution for sites where interaction isn&#8217;t critical? Turn the comments off. Unfortunately, many sites do not have this luxury.</p>\n<h3>Removing Comments Suffocates Conversation</h3>\n<p>Traditionally, a blog post has been an invitation to a conversation. The weblog medium soared to popularity because of its interactive nature, which allowed readers to leave comments. Some might <a href=\"http://en.wikipedia.org/wiki/Blog#cite_note-2\" target=\"_blank\">contend</a> that this singular feature distinguishes the medium from other informational, static websites.</p>\n<p>Conversation and community are the natural byproducts of allowing open comments. They aren&#8217;t a utility to propel your blog to a state of maturity where you finally shed comments in a burst of enlightenment. They have intrinsic value. <span class=\"pullquote alignleft\">Open comments are the affirmation that your blog is a conversation and not a soapbox.</span> Ultimately, allowing comments on your posts is about valuing conversation. Encouraging interaction on other social channels, while turning it off at home on your blog, sends a mixed message.</p>\n<p>When Copyblogger turned off comments, it encouraged readers to take the conversation to social networks and to write responses on their own blogs as an alternative to commenting on Copyblogger.com. How can you ask your readers to promote you on social networks and link to you from their blogs, when they&#8217;re not welcome at your home on the web? This is a conversation killer for those who have no motivation to write their own response posts or participate on social networks.</p>\n<h3>Removing Comments Fragments the Conversation</h3>\n<div id=\"attachment_24462\" class=\"wp-caption aligncenter\"><a href=\"http://i2.wp.com/wptavern.com/wp-content/uploads/2014/06/shattered.jpg\" rel=\"prettyphoto[24451]\"><img src=\"http://i2.wp.com/wptavern.com/wp-content/uploads/2014/06/shattered.jpg?resize=1024%2C564\" alt=\"photo credit: Nils Geylen - cc\" class=\"size-full wp-image-24462\" /></a><p class=\"wp-caption-text\">photo credit: <a href=\"http://www.flickr.com/photos/napfisk/389135782/\">Nils Geylen</a> &#8211; <a href=\"http://creativecommons.org/licenses/by-nc-sa/2.0/\">cc</a></p></div>\n<p>Allowing social media to be the primary outposts for conversation on your content may bring some decent interaction for a short time, but posts sent via these channels soon disappear under the heavy stream of cat pictures, location checkins, Candy Crush invitations and every form of distraction.</p>\n<p>Furthermore, a conversation happening in many different places becomes severely fragmented, diluted, and difficult to track. The quality of the conversation starts to plummet. However, with comments open on your website, you have the opportunity for the brightest minds to respond to each other in one public location, not limited to x number of characters or the commenter&#8217;s social connections.</p>\n<p>If your blog is your home on the web, then everything important that you have to say should be said in your posts and in their comments. <span class=\"pullquote alignleft\">Social networks come and go but your blog is forever.</span> If all the resulting conversations of your content are happening on social networks, what will you do when those data silos move on and all that content is lost?</p>\n<p>Sonia Simone, co-founder of Copyblogger Media, <a href=\"http://www.top5seo.co.uk/copyblogger-comments/#comment-5726\" target=\"_blank\">commented</a> on a blog responding to their announcement about removing comments, indicating that she isn&#8217;t concerned if the exchange of ideas is ultimately lost on social networks:</p>\n<blockquote><p>Basically, philosophically, I think it comes down to the fact that I dont feel a need to own or preserve my conversations. What endures about a conversation is how I grow and learn from it  and that cant be taken away.</p>\n<p>If twitter or Facebook or G+ gets taken out by a meteor tomorrow and I lose those conversations, thats all right with me  because the important part is how the conversations have changed me.</p></blockquote>\n<p>What about the millions of others who could learn from or be changed by reading those conversations? Those who read a post months or years after its publish date will probably miss the corresponding social network post and will not have the opportunity to interact with others who have expertise and related knowledge to share on the topic.</p>\n<p>At times it can be disheartening for writers to see an empty comments section on an influential post with a wildly active conversation that moves straight to Twitter. Writers are then forced to chase the conversation in multiple places on the web and must be willing to follow wherever it goes. Many valuable responses are lost in social network posts that quickly evaporate and aren&#8217;t easy to search.</p>\n<p>Blog comments, however, are an enduring home for conversations, many of which result in the exchange of knowledge and ideas. Providing a home for conversations is unfortunately accompanied by the great burden of combating spam and moderating comments. Don&#8217;t give up, though. Our anti-spam tools continue to get better and someday you&#8217;ll be glad that you fought through all the bots to keep real communication alive in your comments. These conversations are the true gold of the internet.</p>\n<h3>Open Comments Are for Those Who Know They Haven&#8217;t Arrived</h3>\n<p>In years of working with WordPress, I&#8217;ve probably learned just as much from blog comments as I have from reading an author&#8217;s original post. At the Tavern, we&#8217;re strong advocates of commenting, because it&#8217;s often there that you&#8217;ll find the rest of the story unfolds. Multiple perspectives combine to form the big picture and the full story emerges when you have the community engaged.</p>\n<p>In online journalism, comments present a huge advantage over the traditional paper newspaper. They enable any citizen to correct misinformation and offer expertise that the author may be missing. You can&#8217;t know everything and you can&#8217;t see every side of a story. The comments section gives the story an opportunity to have a life of its own.</p>\n<p>Many users have expressed <a href=\"http://www.poststat.us/wordpress-comments/\" target=\"_blank\">dissatisfaction with WordPress&#8217; native comment system</a>, and third party systems like <a href=\"http://wptavern.com/the-idea-of-sponsored-comments-disqusts-me\" target=\"_blank\">Disqus</a> don&#8217;t enjoy a much better reputation. These are all part of the growing pains of comments evolving on the web, but don&#8217;t let their drawbacks drive you to close comments completely.</p>\n<p>Leaving comments open on blogs is an adventure. Sure, you may have to deal with spam, vitriol, and people who wrongly assume your blog&#8217;s comments are a support forum. But for the breakthrough gems of real communication and exchange, we leave comments open. It&#8217;s the readers who make a blog worthwhile. You can never outgrow conversation with them, no matter how big your brand or blog becomes.</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Wed, 11 Jun 2014 03:53:19 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Sarah Gooding\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:12;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:71:\"WPTavern: DMCA Takedown Notice Issued Against Fork Of WP Migrate DB Pro\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wptavern.com/?p=24426\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:186:\"http://wptavern.com/dmca-takedown-notice-issued-against-fork-of-wp-migrate-db-pro?utm_source=rss&utm_medium=rss&utm_campaign=dmca-takedown-notice-issued-against-fork-of-wp-migrate-db-pro\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:3470:\"<p><a title=\"http://wordpress.org/plugins/wp-migrate-db/\" href=\"http://wordpress.org/plugins/wp-migrate-db/\">WP DB Migrate</a> by Brad Touesnard is a great plugin that makes it easy to migrate databases from one WordPress site to another. When we <a title=\"http://wptavern.com/wordpress-plugin-review-wp-db-migrate\" href=\"http://wptavern.com/wordpress-plugin-review-wp-db-migrate\">highlighted the plugin</a> in late 2013, a commenter by the name of Sean Lang <a title=\"http://wptavern.com/wordpress-plugin-review-wp-db-migrate#comment-53663\" href=\"http://wptavern.com/wordpress-plugin-review-wp-db-migrate#comment-53663\">published a link</a> to a <a title=\"https://github.com/slang800/wp-migrate-db\" href=\"https://github.com/slang800/wp-migrate-db\">Github repository</a> containing his fork of the pro version. Forking and making available a pro version of a GPL licensed plugin for free is acceptable under the <a title=\"http://wordpress.org/about/gpl/\" href=\"http://wordpress.org/about/gpl/\">terms of the GPL license</a>.</p>\n<div id=\"attachment_24447\" class=\"wp-caption aligncenter\"><a href=\"http://i1.wp.com/wptavern.com/wp-content/uploads/2014/06/WPMigrateDBDMCATakeDownNotice.png\" rel=\"prettyphoto[24426]\"><img class=\"size-full wp-image-24447\" src=\"http://i1.wp.com/wptavern.com/wp-content/uploads/2014/06/WPMigrateDBDMCATakeDownNotice.png?resize=1025%2C288\" alt=\"DMCA Takedown Notice Against A Fork Of WP Migrate DB Pro\" /></a><p class=\"wp-caption-text\">DMCA Takedown Notice Against A Fork Of WP Migrate DB Pro</p></div>\n<p>I recently discovered the Github repository page hosting the forked version of WP Migrate DB Pro is offline due to a <a title=\"https://github.com/github/dmca/blob/master/2014-05-28-Delicious-Brains.md\" href=\"https://github.com/github/dmca/blob/master/2014-05-28-Delicious-Brains.md\">DMCA Takedown request</a>. According to the request, the offending plugin contained a README.md file that used copyrighted content from Delicious Brains Inc. without their permission.</p>\n<blockquote><p>The Github user slang800 has forked our repository found at <a href=\"https://github.com/bradt/wp-migrate-db\">https://github.com/bradt/wp-migrate-db</a>. This action alone would have been fine, but he has also added a new file named README.md containing content copied from our website that we did not give him permission to use.</p></blockquote>\n<p>I confirmed with Touesnard that the DMCA Takedown notice had nothing to do with the code hosted on the repository. Although several commercial themes and plugins are licensed under the GPL, it doesn&#8217;t mean you can violate a company&#8217;s trademark or infringe on their copyrights. If you&#8217;re going to fork a commercial WordPress product and redistribute it, here are a few tips to avoid trouble.</p>\n<ul>\n<li>Rename it. Keeping the same product name confuses users</li>\n<li>Maintain the copyright of the original author</li>\n<li>Do not use trademarked images, logos, etc.</li>\n<li>Create your own set of documentation and support system</li>\n</ul>\n<p>Before forking, try working with the developer to have your features or fixes integrated into the product. Most of the time, they&#8217;ll gladly work with you to have a specific feature added, especially if there&#8217;s already a patch. If your WordPress products are GPL licensed, I&#8217;d like to know how you&#8217;ve handled trademark and copyright infringement in the comments. What advice do you have for other business owners?</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Tue, 10 Jun 2014 22:58:41 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Jeff Chandler\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:13;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:39:\"WPTavern: WP Remote Now Supports Zapier\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wptavern.com/?p=23805\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:122:\"http://wptavern.com/wp-remote-now-supports-zapier?utm_source=rss&utm_medium=rss&utm_campaign=wp-remote-now-supports-zapier\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:3344:\"<p><a title=\"https://wpremote.com/\" href=\"https://wpremote.com/\">WP Remote</a>, the free WordPress site monitoring service, has announced they now <a title=\"https://wpremote.com/2014/05/29/wp-remote-added-zapier/\" href=\"https://wpremote.com/2014/05/29/wp-remote-added-zapier/\">support Zapier</a>. Zapier is a collection of over 250 web apps that allow you to connect applications to automate tasks. Between WP Remote&#8217;s API and Zapier, the company hopes to give users the power to enhance their work day with more automation. The plan includes:</p>\n<ul>\n<li>Free (does not require a paid plan)</li>\n<li>20 Triggers to choose from</li>\n<li>Your sites will refresh while youre not logged in</li>\n<li><a href=\"http://zpr.io/ggKy\">Zapier</a> will automatically push changes to your connected apps as they come in (15-30 minute delay)</li>\n</ul>\n<p>When I asked Noel Tock, Partner and Head of Products for WP Remote, why they allowed Zapier to be free and what distinct advantages does hooking up with the service give the company, he replied:</p>\n<blockquote><p>Just as WP Remote was born out of something we had initially built for ourselves, we wanted to extend some of our newer efficiencies (as a WordPress agency and product company) to our WP Remote end users.</p>\n<p>Furthermore, connecting WP Remote and Zapier isnt a ground breaking feature, but simply a smarter way for your tools to talk to one another. Unless were shipping unique or innovative enhancements (or features that have internal expenses associated such as backups), were quite happy to spread the love. As developers, freelancers or web designers, we simply use so many tools today that giving web applications the ability to communicate should be the norm, and not a paid upgrade.</p></blockquote>\n<p>Connecting sites to Zapier provides an assortment of opportunities to monitor and log events. For example, you can connect a WP Remote site to Pushover in Zapier and trigger it to send you push notifications on specific items such as failed backups or when new updates are available. Or, you could create an audit log by connecting a site to a Google Docs spreadsheet.</p>\n<div id=\"attachment_23819\" class=\"wp-caption aligncenter\"><a href=\"http://i0.wp.com/wptavern.com/wp-content/uploads/2014/05/WPRemoteZapierAuditing.jpg\" rel=\"prettyphoto[23805]\"><img class=\"size-full wp-image-23819\" src=\"http://i0.wp.com/wptavern.com/wp-content/uploads/2014/05/WPRemoteZapierAuditing.jpg?resize=1025%2C720\" alt=\"Creating Audit Logs With WPRemote and Zapier\" /></a><p class=\"wp-caption-text\">Creating Audit Logs With WPRemote and Zapier</p></div>\n<p>Tock said the company wants to see what types of creative Zaps are created by users, &#8220;weve shipped this integration with 20 different triggers specifically so we could see what creative ways people come up with to use it.&#8221;</p>\n<p>WP Remote is not the first WordPress company to take advantage of Zapier. RocketGenius <a title=\"http://www.gravityforms.com/add-ons/zapier/\" href=\"http://www.gravityforms.com/add-ons/zapier/\">created a commercial add-on</a> to extend the functionality of GravityForms instead of creating 100 separate add-ons. Whether it&#8217;s a product or service, tapping into Zapier and its catalog of web apps is a quick and easy way to enhance what a product or service can do.</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Tue, 10 Jun 2014 20:48:50 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Jeff Chandler\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:14;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:60:\"WordPress.tv: Sara Cannon: KidsCamp Atlanta  Smart Design\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=35277\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:73:\"http://wordpress.tv/2014/06/10/sara-cannon-kidscamp-atlanta-smart-design/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:675:\"<div id=\"v-DlbdjkFk-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/35277/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/35277/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=35277&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/10/sara-cannon-kidscamp-atlanta-smart-design/\"><img alt=\"Sara Cannon: KidsCamp Atlanta &#8211; Smart Design\" src=\"http://videos.videopress.com/DlbdjkFk/video-f424a8a7f3_scruberthumbnail_0.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Tue, 10 Jun 2014 18:40:16 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:15;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:69:\"WordPress.tv: Jon Brown: Finding Work/Life Balance as a Remote Worker\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=35609\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:85:\"http://wordpress.tv/2014/06/10/jon-brown-finding-worklife-balance-as-a-remote-worker/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:686:\"<div id=\"v-EQYDzy5J-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/35609/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/35609/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=35609&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/10/jon-brown-finding-worklife-balance-as-a-remote-worker/\"><img alt=\"Jon Brown: Finding Work/Life Balance as a Remote Worker\" src=\"http://videos.videopress.com/EQYDzy5J/video-2de904c5d8_std.original.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Tue, 10 Jun 2014 18:25:45 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:16;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:78:\"WordPress.tv: Andrea Middleton: WordPress Community: Choose Your Own Adventure\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=35612\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:94:\"http://wordpress.tv/2014/06/10/andrea-middleton-wordpress-community-choose-your-own-adventure/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:710:\"<div id=\"v-pwasL9GA-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/35612/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/35612/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=35612&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/10/andrea-middleton-wordpress-community-choose-your-own-adventure/\"><img alt=\"Andrea Middleton: WordPress Community: Choose Your Own Adventure\" src=\"http://videos.videopress.com/pwasL9GA/video-1db6043a03_scruberthumbnail_1.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Tue, 10 Jun 2014 18:13:33 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:17;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:67:\"WPTavern: So Simple: A Free Minimalist WordPress Theme from Press75\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wptavern.com/?p=24363\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:176:\"http://wptavern.com/so-simple-a-free-minimalist-wordpress-theme-from-press75?utm_source=rss&utm_medium=rss&utm_campaign=so-simple-a-free-minimalist-wordpress-theme-from-press75\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:2755:\"<p>Last week the big news in the world of WordPress themes was <a href=\"http://wptavern.com/press75-acquired-by-westwerk-for-undisclosed-amount\" target=\"_blank\">Westwerk&#8217;s acquisition of Press75</a>, one of the oldest theme shops originally founded by <a href=\"http://jason.sc/\" target=\"_blank\">Jason Schuller</a> in 2008. The folks at Westwerk began implementing changes immediately and announced new <a href=\"https://press75.com/join/\" target=\"_blank\">pricing options</a> the same day the shop&#8217;s new owner was introduced. Single theme purchases are now back in the mix.</p>\n<p><a href=\"https://press75.com/view/theme/so-simple/\" target=\"_blank\">So Simple</a> was one of the themes in Press75&#8242;s smaller subscription package, but after new management revised the pricing structure, they decided to offer it for free.</p>\n<p><a href=\"http://i1.wp.com/wptavern.com/wp-content/uploads/2014/06/so-simple.png\" rel=\"prettyphoto[24363]\"><img src=\"http://i1.wp.com/wptavern.com/wp-content/uploads/2014/06/so-simple.png?resize=852%2C486\" alt=\"so-simple\" class=\"aligncenter size-full wp-image-24402\" /></a></p>\n<p>As you can see, this super minimalist theme is almost entirely free of post meta, displaying only the published date. So Simple allows you to select a page to assign for the intro area within the WordPress customizer. This unique intro area disappears when the visitor clicks on the circular indicator attached to the right side of the viewport.  Check out the <a href=\"http://www.press75.com/themes/so-simple/\" target=\"_blank\">live demo</a> to see how it works.</p>\n<p>The theme adds options to the post edit screen which allow you to set a light or dark text color, or a custom permalink. The featured image you assign serves as a responsive fullscreen background image for the post in both the archive and single post views.</p>\n<p>So Simple is probably most notable for what it removes, rather than what it adds. It has no menu, no header and no widget areas, so these areas are not visible in the admin when the theme is active.</p>\n<p>The post writing experience is simplified as well, with tags and categories removed from the editing screen. (Please note that the admin tweaks can be disabled via the customizer.) The idea in using So Simple is to provide a distraction-free writing and reading experience.</p>\n<p>This theme is a natural choice for Press75 to make into a free product, as it doesn&#8217;t promise much but caters to a specific type of website owner. If bare bones minimalism is your style and you don&#8217;t mind a severely pruned post editing screen, check out So Simple. You can <a href=\"https://press75.com/view/theme/so-simple/\" target=\"_blank\">download it</a> for free from the Press75.com website.</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Mon, 09 Jun 2014 21:35:00 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Sarah Gooding\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:18;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:112:\"WPTavern: WordPress Multi-Environment Config: Work Seamlessly Between Development, Staging, and Production Sites\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wptavern.com/?p=24365\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:262:\"http://wptavern.com/wordpress-multi-environment-config-work-seamlessly-between-development-staging-and-production-sites?utm_source=rss&utm_medium=rss&utm_campaign=wordpress-multi-environment-config-work-seamlessly-between-development-staging-and-production-sites\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:5783:\"<div id=\"attachment_24392\" class=\"wp-caption aligncenter\"><a href=\"http://i0.wp.com/wptavern.com/wp-content/uploads/2014/06/control-panel.png\" rel=\"prettyphoto[24365]\"><img src=\"http://i0.wp.com/wptavern.com/wp-content/uploads/2014/06/control-panel.png?resize=1024%2C487\" alt=\"photo credit: Matteo Bagnoli - cc\" class=\"size-full wp-image-24392\" /></a><p class=\"wp-caption-text\">photo credit: <a href=\"http://www.flickr.com/photos/matteo_bagnoli/5030486811/\">Matteo Bagnoli</a> &#8211; <a href=\"http://creativecommons.org/licenses/by-sa/2.0/\">cc</a></p></div>\n<p>WordPress doesn&#8217;t have built-in support for developers working in multiple environments, i.e. local, staging, and production. It&#8217;s no surprise, given that this workflow isn&#8217;t part of the vast majority of users&#8217; lives when publishing with WordPress.</p>\n<p>Nevertheless, running a separate development site can be a <a href=\"http://codex.wordpress.org/Running_a_Development_Copy_of_WordPress\" target=\"_blank\">complex and tedious endeavor</a> and WordPress doesn&#8217;t make it very easy. The process usually involves updating the database settings in wp-config.php and running a SQL change on the database to update the &#8216;wp_siteurl&#8217; and &#8216;wp_home&#8217; settings. Developers have different tools to help streamline the task of moving between different environments, but it&#8217;s still a pain.</p>\n<h3>WordPress Multi-Environment Config</h3>\n<p><a href=\"http://twitter.com/simonrjones\" target=\"_blank\">Simon Jones</a> is the founder and technical director of <a href=\"http://www.studio24.net/\" target=\"_blank\">Studio 24</a>, a web development agency that works extensively with WordPress. He believes that the platform should have better support for those working in multiple environments and has created an extension to the WP config system to elegantly support this.</p>\n<p>&#8220;In order to efficiently work on a project I prefer to have all the setup for different environments in the website code,&#8221; Jones said. <strong>&#8220;So when we push changes to staging things just work.&#8221;</strong> To that end, he created the <a href=\"https://github.com/studio24/wordpress-multi-env-config\" target=\"_blank\">WordPress Multi-Environment Config</a> project. It supports using WordPress in local development, staging, and production environments without having to edit your config files or update the site URLS in your database.</p>\n<p>Jones used an approach based on FocusLabs work on the <a href=\"https://github.com/focuslabllc/ee-master-config\" target=\"_blank\">ExpressionEngine master config</a>. It splits the config files out into:</p>\n<ul>\n<li><strong>wp-config.default.php</strong> &#8211; any shared settings used across all environments</li>\n<li><strong>wp-config.{environment}.php</strong> &#8211; the settings for a specific environment</li>\n<li><strong>wp-config.env.php</strong> &#8211; sets up which URLs work for which environments</li>\n<li><strong>wp-config.php</strong> &#8211; the original WP config file which now contains loading code</li>\n</ul>\n<p>The way it works is that you copy any shared settings, i.e. auth keys and salts, into wp-config.default.php. You then copy your unique settings for each environment, i.e. database access details, to the wp-config{environment}.php file. In the wp-config.env.php file you then specify which URLs are defined for which environment. You can also set the environment via a webserver environment variable if you like.</p>\n<p>Once all of this is in place, WordPress will load the default config settings first, followed by the correct environment config based on either the settings in wp-config.env.php or the webserver environment variable. Your configuration then works seamlessly across your local, staging, and production URLs.</p>\n<p>Jones&#8217; experience with agency work inspired him to create the WordPress multi-environment config project. &#8220;Like many web developers and digital agencies, at Studio 24 we work on multiple environments for client websites,&#8221; he said. &#8220;<span class=\"pullquote alignleft\">In my experience most CMSs dont have great support for multiple environments</span>, its usually a case of local config files or changes to the database when files are pushed live. Even Drupal is only just starting to look at <a href=\"http://drupalwatchdog.com/2/2/drupal-8-preview\" target=\"_blank\">adding support for multiple environments in Drupal 8</a>.&#8221;</p>\n<p>Jones believes that WordPress needs multiple environment support in core. <strong>&#8220;Very few CMSs really consider this and its an essential feature for people working on the modern web,&#8221;</strong> he said. &#8220;With all the great improvements in WordPress over the past few years, features to help developers deploy and work on multiple environments would be great to see.&#8221;</p>\n<p>A better configuration management system would allow WordPress developers to be more efficient when working with enterprise environments, which often have more complex deployment workflows. Smaller agencies, individuals and anyone building WordPress sites may also benefit. Even if most WordPress users don&#8217;t work on their sites in multiple environments, there might be a way to make configuration management easier for those who do.</p>\n<p>In the meantime, Jones&#8217; WordPress Multiple Environment Config provides a nice solution. Although it doesn&#8217;t handle migration of database data between environments, this setup offers a clean and convenient way to manage configuration when moving between sites. For more details on how to install and use it, check out the <a href=\"https://github.com/studio24/wordpress-multi-env-config\" target=\"_blank\">project&#8217;s page on GitHub</a>.</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Mon, 09 Jun 2014 19:43:00 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Sarah Gooding\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:19;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:68:\"WordPress.tv: Erick Arb: WordPress Navigation in Responsive Design\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=35307\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:86:\"http://wordpress.tv/2014/06/09/erick-arbe-wordpress-navigation-in-responsive-design-2/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:692:\"<div id=\"v-z0I9xwxp-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/35307/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/35307/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=35307&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/09/erick-arbe-wordpress-navigation-in-responsive-design-2/\"><img alt=\"Erick Arb: WordPress Navigation in Responsive Design\" src=\"http://videos.videopress.com/z0I9xwxp/video-0da0e4ba29_scruberthumbnail_0.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Mon, 09 Jun 2014 18:46:09 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:20;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:65:\"WordPress.tv: Manuel Schmalstieg: Date Management for Minimalists\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=35514\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:82:\"http://wordpress.tv/2014/06/09/manuel-schmalstieg-date-management-for-minimalists/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:688:\"<div id=\"v-UH4N1iBS-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/35514/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/35514/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=35514&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/09/manuel-schmalstieg-date-management-for-minimalists/\"><img alt=\"Manuel Schmalstieg: Date Management for Minimalists\" src=\"http://videos.videopress.com/UH4N1iBS/manuel-schmalstieg-wcch14_std.original.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Mon, 09 Jun 2014 18:32:45 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:21;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:52:\"WordPress.tv: Andrea Middleton: A World of WordCamps\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=35279\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:69:\"http://wordpress.tv/2014/06/09/andrea-middleton-a-world-of-wordcamps/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:659:\"<div id=\"v-3ejMeKuQ-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/35279/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/35279/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=35279&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/09/andrea-middleton-a-world-of-wordcamps/\"><img alt=\"Andrea Middleton: A World of WordCamps\" src=\"http://videos.videopress.com/3ejMeKuQ/video-0e21ffc629_scruberthumbnail_0.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Mon, 09 Jun 2014 18:22:56 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:22;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:88:\"WordPress.tv: Michael Cain &amp; Konstantin Obenland: Cain &amp; Obenland In The Morning\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=35588\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:65:\"http://wordpress.tv/2014/06/09/cain-obenland-in-the-morning-wcoc/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:693:\"<div id=\"v-hgCxahT6-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/35588/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/35588/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=35588&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/09/cain-obenland-in-the-morning-wcoc/\"><img alt=\"Michael Cain and Konstantin Obenland &#8211; Cain & Obenland In The Morning[2].mp4\" src=\"http://videos.videopress.com/hgCxahT6/video-66aa1c1700_std.original.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Mon, 09 Jun 2014 16:38:09 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:23;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:68:\"WordPress.tv: Eric Mann: Ten Lessons on Customer-centric Development\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=35261\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:85:\"http://wordpress.tv/2014/06/08/eric-mann-ten-lessons-on-customer-centric-development/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:691:\"<div id=\"v-CRbbhJVm-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/35261/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/35261/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=35261&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/08/eric-mann-ten-lessons-on-customer-centric-development/\"><img alt=\"Eric Mann: Ten Lessons on Customer-centric Development\" src=\"http://videos.videopress.com/CRbbhJVm/video-14af328cda_scruberthumbnail_0.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Sun, 08 Jun 2014 18:56:15 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:24;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:63:\"WordPress.tv: Clinton Warren: An Accidental Career In WordPress\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=35319\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:80:\"http://wordpress.tv/2014/06/08/clinton-warren-an-accidental-career-in-wordpress/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:681:\"<div id=\"v-bNd7Oif6-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/35319/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/35319/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=35319&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/08/clinton-warren-an-accidental-career-in-wordpress/\"><img alt=\"Clinton Warren: An Accidental Career In WordPress\" src=\"http://videos.videopress.com/bNd7Oif6/video-baca84f43e_scruberthumbnail_0.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Sun, 08 Jun 2014 18:49:26 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:25;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:92:\"WordPress.tv: Helen Hou-Sandi: How I Convinced My Boss to Let Me Work on WordPress Full-time\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=35404\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:109:\"http://wordpress.tv/2014/06/08/helen-hou-sandi-how-i-convinced-my-boss-to-let-me-work-on-wordpress-full-time/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:739:\"<div id=\"v-fWQ4N99p-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/35404/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/35404/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=35404&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/08/helen-hou-sandi-how-i-convinced-my-boss-to-let-me-work-on-wordpress-full-time/\"><img alt=\"Helen Hou-Sandi: How I Convinced My Boss to Let Me Work on WordPress Full-time\" src=\"http://videos.videopress.com/fWQ4N99p/video-5802861622_scruberthumbnail_0.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Sun, 08 Jun 2014 18:04:48 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:26;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:57:\"WordPress.tv: Chris Lema: Getting The Most From Your Blog\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=35585\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:74:\"http://wordpress.tv/2014/06/07/chris-lema-getting-the-most-from-your-blog/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:663:\"<div id=\"v-E24Ef0Qi-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/35585/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/35585/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=35585&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/07/chris-lema-getting-the-most-from-your-blog/\"><img alt=\"Chris Lema: Getting The Most From Your Blog\" src=\"http://videos.videopress.com/E24Ef0Qi/video-f6e527e6e8_std.original.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Sat, 07 Jun 2014 23:25:09 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:27;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:66:\"Alex King: Heres Crowd Favorites own Brandon Dove at the \";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://alexking.org/?p=20175\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:41:\"http://alexking.org/blog/2014/06/07/20175\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:257:\"<div>\n<p>Here&#8217;s <a href=\"http://crowdfavorite.com\">Crowd Favorite</a>&#8216;s own <a href=\"http://brandondove.com/\">Brandon Dove</a> at the <a href=\"http://2014.oc.wordcamp.org/\">WordCamp Orange County</a> golf tournament yesterday.</p>\n<p></p>\n</div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Sat, 07 Jun 2014 17:45:55 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:4:\"Alex\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:28;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:80:\"WordPress.tv: James Laws: Ninja Forms: A Case Study Of The Freemium Plugin Model\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=35104\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:96:\"http://wordpress.tv/2014/06/07/james-laws-ninja-forms-a-case-study-of-the-freemium-plugin-model/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:714:\"<div id=\"v-G6Qclaje-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/35104/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/35104/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=35104&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/07/james-laws-ninja-forms-a-case-study-of-the-freemium-plugin-model/\"><img alt=\"James Laws: Ninja Forms: A Case Study Of The Freemium Plugin Model\" src=\"http://videos.videopress.com/G6Qclaje/video-3088daedcd_scruberthumbnail_0.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Sat, 07 Jun 2014 09:00:37 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:7:\"Michael\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:29;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:56:\"WordPress.tv: Andrew Nacin: WordCamp Connecticut Keynote\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=35467\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:73:\"http://wordpress.tv/2014/06/07/andrew-nacin-wordcamp-connecticut-keynote/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:661:\"<div id=\"v-yt8XUtjj-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/35467/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/35467/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=35467&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/07/andrew-nacin-wordcamp-connecticut-keynote/\"><img alt=\"STANFORD WP KEYNOTE Andrew Nacin.mp4\" src=\"http://videos.videopress.com/yt8XUtjj/video-a857e0bf7e_scruberthumbnail_0.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Sat, 07 Jun 2014 08:00:50 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:30;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:72:\"WordPress.tv: Jos Esteban Mucientes Manso: Un tema para cada necesidad\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=32504\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:88:\"http://wordpress.tv/2014/06/07/jose-esteban-mucientes-manso-un-tema-para-cada-necesidad/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:698:\"<div id=\"v-UCTGa87i-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/32504/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/32504/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=32504&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/07/jose-esteban-mucientes-manso-un-tema-para-cada-necesidad/\"><img alt=\"Jos Esteban Mucientes Manso: Un tema para cada necesidad\" src=\"http://videos.videopress.com/UCTGa87i/video-3cbd171905_scruberthumbnail_0.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Sat, 07 Jun 2014 07:00:05 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:31;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:80:\"WPTavern: bbPress 2.5.4 Security Release Available: Immediate Update Recommended\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wptavern.com/?p=24326\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:202:\"http://wptavern.com/bbpress-2-5-4-security-release-available-immediate-update-recommended?utm_source=rss&utm_medium=rss&utm_campaign=bbpress-2-5-4-security-release-available-immediate-update-recommended\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:1965:\"<p><a href=\"http://i1.wp.com/wptavern.com/wp-content/uploads/2014/06/bbpress.png\" rel=\"prettyphoto[24326]\"><img src=\"http://i1.wp.com/wptavern.com/wp-content/uploads/2014/06/bbpress.png?resize=1025%2C378\" alt=\"bbpress\" class=\"aligncenter size-full wp-image-24338\" /></a></p>\n<p>If you have a WordPress site with a bbPress-powered forum, you may want to set aside a few minutes this weekend to perform updates. bbPress 2.5.4 was <a href=\"http://bbpress.org/blog/2014/06/bbpress-2-5-4-security-bugfix-release/\" target=\"_blank\">released</a> today with five bug fixes, including an important fix for a security vulnerability.</p>\n<p>bbPress project lead John James Jacoby announced the release, crediting IT Security Reaseacher<a href=\"https://hackerone.com/mazengamal\" target=\"_blank\"> Mazen Gamal Mesbah</a> for identifying and responsibly disclosing the displayed user field security issue. Mesbah has been instrumental in helping to identify multiple security bugs for Yahoo, Automattic, Mailchimp, Sony, Eventbrite, and was most recently honored in Microsoft&#8217;s security <a href=\"http://technet.microsoft.com/en-us/security/cc308589.aspx\" target=\"_blank\">hall of fame</a>.</p>\n<p>The security <a href=\"https://bbpress.trac.wordpress.org/ticket/2610\" target=\"_blank\">fix</a> in bbPress 2.5.4 adds proper escaping for displayed user fields and data when editing a user. Should you spend time updating over the weekend? Yes, if at all possible. The bbPress team recommends that you update your sites as soon as you can.  While the update wasn&#8217;t announced as critical, bbPress contributors found the important enough to fix within 24 hours of being notified, a feat which Jacoby notes is quite impressive for an all-volunteer team.</p>\n<p>The bbPress 2.6 release date has been pushed back to the end of June to allow contributors some more time to enjoy the summer. In the meantime, take a few minutes this weekend to update your bbPress sites to 2.5.4.</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Sat, 07 Jun 2014 04:22:33 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Sarah Gooding\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:32;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:87:\"WPTavern: New Plugin Adds Syntax Highlighting to the WordPress Plugin and Theme Editors\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wptavern.com/?p=24286\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:218:\"http://wptavern.com/new-plugin-adds-syntax-highlighting-to-the-wordpress-plugin-and-theme-editors?utm_source=rss&utm_medium=rss&utm_campaign=new-plugin-adds-syntax-highlighting-to-the-wordpress-plugin-and-theme-editors\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:3862:\"<p>The WordPress plugin and theme editors have a basic, bare bones implementation of a text editor. They function well but are rather plain and limited in features. This may be due in part to the fact that many WordPress contributors cannot agree on whether or not the admin plugin and theme editors should be included in core.</p>\n<p>This hotly debated topic was reignited on the Tavern last year when we featured the controversial <a href=\"http://wptavern.com/should-code-revisions-be-added-to-the-wordpress-core\" target=\"_blank\">Code Revisions project</a>, which added native revisions to the admin theme and plugin editors. The project was part of Google Summer of Code in 2013 and there was some discussion about the possibility of adding it to core.</p>\n<p>Whether or not you agree on having the admin editors in core, there are many plugin developers who are happy to extend the feature for those who use it. <a href=\"http://wordpress.org/plugins/syntax-highlight/\" target=\"_blank\">Syntax Highlight</a> is a new plugin that adds syntax highlighting and a few other handy bits to the editors by incorporating the open source <a href=\"http://ace.c9.io/\" target=\"_blank\">Ace Editor</a>.</p>\n<p><a href=\"http://i0.wp.com/wptavern.com/wp-content/uploads/2014/06/syntax-highlighter.jpg\" rel=\"prettyphoto[24286]\"><img src=\"http://i0.wp.com/wptavern.com/wp-content/uploads/2014/06/syntax-highlighter.jpg?resize=1025%2C548\" alt=\"syntax-highlighter\" class=\"aligncenter size-full wp-image-24288\" /></a></p>\n<p>The plugin adds AJAX saving through CTRL+S and has support for keyboard shortcuts (keybindings: Vim, Emacs and <a href=\"https://github.com/ajaxorg/ace/wiki/Default-Keyboard-Shortcuts\" target=\"_blank\">Default</a>). If you make edits and attempt to leave the page, the plugin will ask you whether or not you want to leave when there are unsaved changes.</p>\n<p>Syntax Highlight enables editors to launch in fullscreen mode (CTRL+Enter) for an experience similar to the distraction free writing mode:</p>\n<p><a href=\"http://i0.wp.com/wptavern.com/wp-content/uploads/2014/06/fullscreen.jpg\" rel=\"prettyphoto[24286]\"><img src=\"http://i0.wp.com/wptavern.com/wp-content/uploads/2014/06/fullscreen.jpg?resize=1025%2C548\" alt=\"fullscreen\" class=\"aligncenter size-full wp-image-24300\" /></a></p>\n<p>It also includes a lighter theme which can be turned on via the plugin&#8217;s settings page:</p>\n<p><a href=\"http://i2.wp.com/wptavern.com/wp-content/uploads/2014/06/lighter-theme.jpg\" rel=\"prettyphoto[24286]\"><img src=\"http://i2.wp.com/wptavern.com/wp-content/uploads/2014/06/lighter-theme.jpg?resize=1025%2C546\" alt=\"lighter-theme\" class=\"aligncenter size-full wp-image-24299\" /></a></p>\n<p>The settings panel gives you the option to set the default tab size, use soft tabs, turn on word wrap, use line numbers, set key bindings and enable full line selection.</p>\n<p><a href=\"http://i2.wp.com/wptavern.com/wp-content/uploads/2014/06/settings.jpg\" rel=\"prettyphoto[24286]\"><img src=\"http://i2.wp.com/wptavern.com/wp-content/uploads/2014/06/settings.jpg?resize=1025%2C546\" alt=\"settings\" class=\"aligncenter size-full wp-image-24303\" /></a></p>\n<p>The plugin enables search and replace with regular expressions (CTRL+F, CTRL+H). It also supports all the other features included in the <a href=\"http://ace.c9.io/\" target=\"_blank\">Ace Editor</a>, including drag and drop text using the mouse, automatic indenting, live syntax checking (currently JavaScript/CoffeeScript/CSS/XQuery), and more.</p>\n<p>If you frequently use the admin plugin and theme editors and want to beef them up with syntax highlighting, this plugin is a decent option. At the very least, it makes the code more readable if you only use the editors for reference. You can download <a href=\"http://wordpress.org/plugins/syntax-highlight/\" target=\"_blank\">Syntax Highlight</a> from WordPress.org.</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Fri, 06 Jun 2014 23:20:57 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Sarah Gooding\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:33;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:65:\"WPTavern: WordPress for Android 2.9 Debuts Blog Discovery Feature\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wptavern.com/?p=24259\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:174:\"http://wptavern.com/wordpress-for-android-2-9-debuts-blog-discovery-feature?utm_source=rss&utm_medium=rss&utm_campaign=wordpress-for-android-2-9-debuts-blog-discovery-feature\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:2990:\"<p>Version 2.9 of the WordPress for Android app was <a href=\"http://android.wordpress.org/2014/06/06/version-2-9-of-wordpress-for-android/\" target=\"_blank\">released</a> today, debuting its Blog Discovery feature as a new part of the Reader. Blog Discovery can be found by clicking the tag icon in the top right corner of the Reader. This feature helps users find new blogs to read, based on recommendations, and allows you to preview a blog&#8217;s posts before electing to follow it.</p>\n<p><a href=\"http://i1.wp.com/wptavern.com/wp-content/uploads/2014/06/blog-discovery-feature.gif\" rel=\"prettyphoto[24259]\"><img src=\"http://i1.wp.com/wptavern.com/wp-content/uploads/2014/06/blog-discovery-feature.gif?resize=540%2C847\" alt=\"blog-discovery-feature\" class=\"aligncenter size-full wp-image-24264\" /></a></p>\n<p>The Reader has been updated to allow users to easily manage tags and blog subscriptions within the mobile app. You&#8217;ll notice that the reblogging interface is also sporting a new design.</p>\n<h3>Improved Publish Icon Button</h3>\n<p><a href=\"http://i1.wp.com/wptavern.com/wp-content/uploads/2014/06/draft-save.png\" rel=\"prettyphoto[24259]\"><img src=\"http://i1.wp.com/wptavern.com/wp-content/uploads/2014/06/draft-save.png?resize=300%2C500\" alt=\"draft-save\" class=\"alignright size-large wp-image-24273\" /></a>The 2.9 release solves a long-standing issue with the ambiguous UI of the save/publish button. In the past, it was unclear whether or not you were saving a draft or publishing a post, which discouraged some users from using the app to save drafts on the go.</p>\n<p>Version 2.9 clarifies the UI by replacing the publish icon button with a contextual text button. The new button displays the action it initiates, based on the current task, ie. saving a draft, publishing, scheduling, or updating a post. This is now more in line with how a user might expect WordPress to work based on the desktop experience.</p>\n<h3>UI Enhancements: Notifications Powered by Simperium, App Now Supports Pull-to-Refresh Gesture</h3>\n<p>Notifications now use <a href=\"https://simperium.com/\" target=\"_blank\">Simperium</a> technology for faster syncing. Stats also refresh faster and version 2.9 reintroduces the refresh button for all refreshable views. It makes use of Android&#8217;s native pull-to-refresh gesture. Posts will be auto-saved when you close the &#8220;edit post&#8221; view, so the save dialog has been removed entirely for simplification.</p>\n<p>Version 2.9 adds Hebrew and Basque to its available translations. It also introduces <a href=\"http://en.wikipedia.org/wiki/Server_Name_Indication\" target=\"_blank\">SNI (Server Name Indication) </a>support.</p>\n<p>Support for Gingerbread (Android 2.3) was dropped, as <a href=\"http://wptavern.com/wordpress-for-android-will-no-longer-support-gingerbread\" target=\"_blank\">announced</a> last month. If you&#8217;re upgrading to WordPress for Android 2.9, you&#8217;ll need to be running Android 4.0 or later on your device.</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Fri, 06 Jun 2014 19:34:02 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Sarah Gooding\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:34;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:82:\"WordPress.tv: Francisco Javier Carazo Gil: WordPress como plataforma de desarrollo\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=25609\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:99:\"http://wordpress.tv/2014/06/06/francisco-javier-carazo-gil-wordpress-como-plataforma-de-desarrollo/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:719:\"<div id=\"v-u20E7Fpe-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/25609/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/25609/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=25609&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/06/francisco-javier-carazo-gil-wordpress-como-plataforma-de-desarrollo/\"><img alt=\"Francisco Javier Carazo Gil: WordPress como plataforma de desarrollo\" src=\"http://videos.videopress.com/u20E7Fpe/video-d6104776da_scruberthumbnail_0.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Fri, 06 Jun 2014 09:15:12 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:35;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:48:\"WP Android: Version 2.9 of WordPress for Android\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:36:\"http://android.wordpress.org/?p=1065\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:77:\"http://android.wordpress.org/2014/06/06/version-2-9-of-wordpress-for-android/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:4519:\"<p>The WordPress for Android 2.9 release is now available in the Google Play Store. This release includes some exciting new features, enhancements, and bug fixes.</p>\n<h2>Blog Discovery</h2>\n<p>Blog discovery is a new feature in the Reader that lets you:</p>\n<ul>\n<li>Find new blogs (based on recommendations).</li>\n<li>Preview a blog and read posts before following it.</li>\n<li>Manage your tags and blog subscriptions.</li>\n</ul>\n<p><a href=\"http://wpandroid.files.wordpress.com/2014/06/blog-discovery-feature.gif\"><img class=\"aligncenter size-full wp-image-1059\" src=\"http://wpandroid.files.wordpress.com/2014/06/blog-discovery-feature.gif?w=840\" alt=\"blog-discovery-feature\" /></a></p>\n<h2>Publish Icon Button</h2>\n<p>We replaced the publish icon button witha contextual text button. Whether you&#8217;re saving a draft, publishing or scheduling a post, or updating one, this new button will displayyour action, depending on yourcurrent task.</p>\n\n<a href=\"http://android.wordpress.org/?attachment_id=1063\"><img width=\"90\" height=\"150\" src=\"http://wpandroid.files.wordpress.com/2014/06/screencap-2014-06-04t17-47-230800.png?w=90&h=150\" class=\"attachment-thumbnail\" alt=\"\" /></a>\n<a href=\"http://android.wordpress.org/?attachment_id=1060\"><img width=\"90\" height=\"150\" src=\"http://wpandroid.files.wordpress.com/2014/06/screencap-2014-06-04t17-46-510800.png?w=90&h=150\" class=\"attachment-thumbnail\" alt=\"\" /></a>\n<a href=\"http://android.wordpress.org/?attachment_id=1061\"><img width=\"90\" height=\"150\" src=\"http://wpandroid.files.wordpress.com/2014/06/screencap-2014-06-04t17-46-570800.png?w=90&h=150\" class=\"attachment-thumbnail\" alt=\"\" /></a>\n<a href=\"http://android.wordpress.org/?attachment_id=1062\"><img width=\"90\" height=\"150\" src=\"http://wpandroid.files.wordpress.com/2014/06/screencap-2014-06-04t17-47-060800.png?w=90&h=150\" class=\"attachment-thumbnail\" alt=\"\" /></a>\n<a href=\"http://android.wordpress.org/?attachment_id=1064\"><img width=\"90\" height=\"150\" src=\"http://wpandroid.files.wordpress.com/2014/06/screencap-2014-06-04t18-42-390800.png?w=90&h=150\" class=\"attachment-thumbnail\" alt=\"\" /></a>\n\n<h2>Faster Notifications and Stats Refresh</h2>\n<p>We updated the Notifications feature to use <a href=\"https://simperium.com/\">Simperium</a> technology, which will sync your notifications quickly and efficiently.</p>\n<p>We also know you love viewing your Stats, so we improved them to refresh faster than ever before.</p>\n<h2>Interface Improvements</h2>\n<ul>\n<li>Reintroduction of the refresh button in all refreshable views, along with the pull-to-refresh gesture.</li>\n<li>Pull-to-refresh tip bar has been replaced by a less aggressive, self-hiding message.</li>\n<li>Save dialog has been removed, andall posts are now auto-saved when you close the edit post view.</li>\n<li>Reblogging interface redesign in the Reader.</li>\n<li>Sharing image, video, text, or link via WordPress for Android now remembers the previous choice.</li>\n</ul>\n<h2>General Changes</h2>\n<ul>\n<li>Posts and pages auto-save feature has been improved.</li>\n<li>Fixed bugs related to statistics (only affecting Jetpack users) and image handling.</li>\n<li>Reader improvements to fill gaps in time between two syncs.</li>\n<li>As <a href=\"http://android.wordpress.org/2014/05/16/wordpress-for-android-no-longer-paying-the-gingerbread-tax/\">we announced earlier</a>, we dropped Android 2.3 support. Current (2.9) and later versions need Android 4.0 or later.</li>\n<li>New translations: Hebrew and Basque.</li>\n<li>SNI (Server Name Indication) support.</li>\n<li>Minor bug fixes.</li>\n</ul>\n<h2>Whats Next?</h2>\n<p>A big thanks to all of the contributors who worked on this release: <a href=\"https://github.com/beaucollins\">@beaucollins</a>, <a href=\"https://github.com/daniloercoli\">@daniloercoli</a>, <a href=\"https://github.com/maxme\">@maxme</a>, <a href=\"https://github.com/nbradbury\">@nbradbury</a>, <a href=\"https://github.com/roundhill\">@roundhill</a>, and<a href=\"https://github.com/sendhil\">@sendhil</a>! You can keep up with the development progress at <a href=\"http://make.wordpress.org/mobile\">http://make.wordpress.org/mobile</a>andcan also follow the app on Twitter <a href=\"http://twitter.com/wpandroid\">@WPAndroid</a>. If you need support or want to send us suggestions, please visit <a href=\"http://android.forums.wordpress.org/\">our forums</a>.</p><img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=android.wordpress.org&blog=9426921&post=1065&subd=wpandroid&ref=&feed=1\" width=\"1\" height=\"1\" />\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Fri, 06 Jun 2014 08:34:22 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:6:\"Maxime\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:36;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:124:\"WordPress.tv: Fran Moreno: Themes premium desde la perspectiva del creador de contenidos, del diseador y del desarrollador\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=25604\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:139:\"http://wordpress.tv/2014/06/06/fran-moreno-themes-premium-desde-la-perspectiva-del-creador-de-contenidos-del-disenador-y-del-desarrollador/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:701:\"<div id=\"v-iRjlQwSY-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/25604/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/25604/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=25604&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/06/fran-moreno-themes-premium-desde-la-perspectiva-del-creador-de-contenidos-del-disenador-y-del-desarrollador/\"><img alt=\"themes.mp4\" src=\"http://videos.videopress.com/iRjlQwSY/video-3bd011e887_scruberthumbnail_0.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Fri, 06 Jun 2014 08:00:07 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:37;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:79:\"WordPress.tv: Rafael Poveda: Trabajando en Comunidad  WordPress desde dentro\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=25241\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:92:\"http://wordpress.tv/2014/06/06/rafael-poveda-trabajando-en-comunidad-wordpress-desde-dentro/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:713:\"<div id=\"v-U8CvZNdU-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/25241/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/25241/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=25241&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/06/rafael-poveda-trabajando-en-comunidad-wordpress-desde-dentro/\"><img alt=\"Rafael Poveda: Trabajando en Comunidad &#8211; WordPress desde dentro\" src=\"http://videos.videopress.com/U8CvZNdU/video-735f74913e_scruberthumbnail_0.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Fri, 06 Jun 2014 07:00:09 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:38;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:95:\"WPTavern: Preview the Press This Bookmarklet Redesign, Alpha Plugin Now Ready for Testing\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wptavern.com/?p=24209\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:220:\"http://wptavern.com/preview-the-press-this-bookmarklet-redesign-alpha-plugin-now-ready-for-testing?utm_source=rss&utm_medium=rss&utm_campaign=preview-the-press-this-bookmarklet-redesign-alpha-plugin-now-ready-for-testing\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:3821:\"<p>The &#8220;<a href=\"http://codex.wordpress.org/Press_This\" target=\"_blank\">Press This</a>&#8221; browser bookmarklet in WordPress is on its way to a <a href=\"http://make.wordpress.org/ui/2014/06/04/press-this/\" target=\"_blank\">complete redesign</a> via the <a href=\"http://make.wordpress.org/core/features-as-plugins/\" target=\"_blank\">features-as-plugins</a> development path. If you&#8217;ve never used it before, the feature allows you to make quick posts to your WordPress site from anywhere on the web. It&#8217;s actually a pretty smart tool for posting and can automatically detect media and grab the proper embed codes.</p>\n<p>The redesign project, led by <a href=\"http://profiles.wordpress.org/michael-arestad\" target=\"_blank\">Michael Arestad</a>, is currently in the design stage, but an ultra alpha public release of the <a href=\"http://wordpress.org/plugins/press-this/\" target=\"_blank\">Press This</a> plugin is now available for testing. Here&#8217;s a sneak peak of the redesign progress thus far:</p>\n<div id=\"attachment_24222\" class=\"wp-caption aligncenter\"><a href=\"http://i1.wp.com/wptavern.com/wp-content/uploads/2014/06/press-this.png\" rel=\"prettyphoto[24209]\"><img src=\"http://i1.wp.com/wptavern.com/wp-content/uploads/2014/06/press-this.png?resize=1025%2C653\" alt=\"Press This redesign progress\" class=\"size-full wp-image-24222\" /></a><p class=\"wp-caption-text\">Press This redesign progress</p></div>\n<p>Parts of the design are still in flux (placement and behavior of WYSIWYG, modal pop-out, embeds, etc), as the bookmarklet is being completely rebuilt with a focus on automation and speed. Arestad posted a 30-second video demo that walks you through posting with the redesigned bookmarklet:<br />\n<div class=\"wp-video\"><!--[if lt IE 9]><script>document.createElement(\'video\');</script><![endif]-->\n<a href=\"http://wptavern.com/wp-content/uploads/2014/06/press-this-progress.mp4\">http://wptavern.com/wp-content/uploads/2014/06/press-this-progress.mp4</a></div><div class=\"media-shortcode-extend\"><div class=\"media-info video-info\"><ul class=\"media-meta\"><li><span class=\"prep\">Run Time</span> <span class=\"data\">0:29</span></li><li><span class=\"prep\">Dimensions</span> <span class=\"data\">1,280 &#215; 800</span></li><li><span class=\"prep\">File Name</span> <span class=\"data\"><a href=\"http://wptavern.com/wp-content/uploads/2014/06/press-this-progress.mp4\">press-this-progress.mp4</a></span></li><li><span class=\"prep\">File Size</span> <span class=\"data\">4.63 MB</span></li><li><span class=\"prep\">File Type</span> <span class=\"data\">MP4</span></li><li><span class=\"prep\">Mime Type</span> <span class=\"data\">video/quicktime</span></li></ul></div><a class=\"media-info-toggle\">Video Info</a></div></p>\n<p>This iteration of Press This is currently powered by AJAX, but Arestad reports that it&#8217;s ready to be switched to use the <a href=\"http://wptavern.com/json-rest-api-slated-for-wordpress-4-1-release\" target=\"_blank\">WP-API</a> endpoints as they become available. It&#8217;s also backward compatible with the old bookmarklet but is built to override /wp-admin/press-this.php and the bookmarklet JS code in /wp-admin/tools.php with the new behavior.</p>\n<p>The complete overhaul of the bookmarklet, with its speedy 2-step publishing capabilities, is bound to win the feature some new users. The contributors working on it are aiming for &#8220;lightning fast posting&#8221; and the progress so far looks promising.</p>\n<p>When testing the <a href=\"http://wordpress.org/plugins/press-this/\" target=\"_blank\">alpha plugin</a>, be advised that it&#8217;s currently undergoing heavy development and will undoubtedly have some bugs. If you want to join in the discussion, feel free to jump in on the <a href=\"http://corepressthis.wordpress.com\" target=\"_blank\">Press This Working Group blog</a>.</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Fri, 06 Jun 2014 02:35:25 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Sarah Gooding\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:39;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:78:\"WPTavern: Semicolon is Possibly the Cleanest Free Magazine Theme for WordPress\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wptavern.com/?p=23953\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:200:\"http://wptavern.com/semicolon-is-possibly-the-cleanest-free-magazine-theme-for-wordpress?utm_source=rss&utm_medium=rss&utm_campaign=semicolon-is-possibly-the-cleanest-free-magazine-theme-for-wordpress\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:3105:\"<p>WordPress magazine themes are notorious for being packed to the gills with content, widgets, post meta, and advertising. Many have a chaotic feel as a byproduct of trying to pack as much content as possible into the homepage.</p>\n<p><a href=\"http://wordpress.org/themes/semicolon\" target=\"_blank\">Semicolon</a> is a shining exception among free WordPress magazine themes in that it features a super minimalist design with an emphasis on large, readable typography.</p>\n<p><a href=\"http://i2.wp.com/wptavern.com/wp-content/uploads/2014/06/semicolon.png\" rel=\"prettyphoto[23953]\"><img src=\"http://i2.wp.com/wptavern.com/wp-content/uploads/2014/06/semicolon.png?resize=880%2C660\" alt=\"semicolon\" class=\"aligncenter size-full wp-image-24147\" /></a></p>\n<p>Semicolon was created by <a href=\"http://kovshenin.com/\" target=\"_blank\">Konstantin Kovshenin</a>, a WordPress plugin and theme developer who is currently a code wrangler at Automattic.  It is based on the <a href=\"http://underscores.me/\" target=\"_blank\">Underscores</a> starter theme. The unique grid layout is 100% responsive and works beautifully on mobile devices.</p>\n<p><a href=\"http://i1.wp.com/wptavern.com/wp-content/uploads/2014/06/semicolon-mobile.png\" rel=\"prettyphoto[23953]\"><img src=\"http://i1.wp.com/wptavern.com/wp-content/uploads/2014/06/semicolon-mobile.png?resize=281%2C500\" alt=\"semicolon-mobile\" class=\"alignright size-large wp-image-24193\" /></a>You can further personalize the theme by adding your own background image or setting the background color with the WordPress customizer. It also supports two menus: a primary navigation menu and a social menu than utilizes the <a href=\"http://genericons.com/\" target=\"_blank\">Genericons</a> vector icon webfont.</p>\n<p>Check out a live demo of Semicolon in action at <a href=\"http://semicolon.kovshenin.com/\" target=\"_blank\">semicolon.kovshenin.com</a>.</p>\n<p>Semicolon manages featured posts using Jetpack&#8217;s Featured Content module, so you&#8217;ll need to have that plugin installed in order to take advantage of this feature. You can then assign a unique tag for your featured posts via the customizer.</p>\n<p>Single posts are followed by an author bio section that can be updated via Users  Your Profile in the admin. A list of related posts is also shown below the content. It&#8217;s generated by a simple algorithm that fetches recent posts in the same category.</p>\n<p>The theme includes four widget areas. Unlike many other WordPress magazine themes, the widget areas are not used to determine the layout for the main content area. The primary, secondary and tertiary widgets are all located in the sidebar. The primary one is highlighted with a blue background and works best for a notice or information you want to feature. The fourth widget area is for the footer, where any widgets added will be stacked horizontally up to a maximum of three.</p>\n<p>Want to test Semicolon on your site? <a href=\"http://wordpress.org/themes/semicolon\" target=\"_blank\">Download</a> it for free from WordPress.org or install it directly via the themes browser in the admin.</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 05 Jun 2014 20:29:21 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Sarah Gooding\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:40;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:52:\"WordPress.tv: Brad Parbs: Writing Extensible Plugins\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=35263\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:69:\"http://wordpress.tv/2014/06/05/brad-parbs-writing-extensible-plugins/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:659:\"<div id=\"v-hrL5ccBA-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/35263/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/35263/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=35263&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/05/brad-parbs-writing-extensible-plugins/\"><img alt=\"Brad Parbs: Writing Extensible Plugins\" src=\"http://videos.videopress.com/hrL5ccBA/video-e797f9a98d_scruberthumbnail_0.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 05 Jun 2014 18:26:33 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:41;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:102:\"WPTavern: WordPress.com to Implement SSL for All Subdomains, Joins the Fight Against Mass Surveillance\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wptavern.com/?p=24161\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:246:\"http://wptavern.com/wordpress-com-to-implement-ssl-for-all-subdomains-joins-the-fight-against-mass-surveillance?utm_source=rss&utm_medium=rss&utm_campaign=wordpress-com-to-implement-ssl-for-all-subdomains-joins-the-fight-against-mass-surveillance\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:2381:\"<p><a href=\"http://i0.wp.com/wptavern.com/wp-content/uploads/2014/06/reset-the-net.png\" rel=\"prettyphoto[24161]\"><img src=\"http://i0.wp.com/wptavern.com/wp-content/uploads/2014/06/reset-the-net.png?resize=1025%2C453\" alt=\"reset-the-net\" class=\"aligncenter size-full wp-image-24163\" /></a></p>\n<p>Today Automattic joins Google, Mozilla, Twitter, Reddit, and other companies across the web in announcing its support for <a href=\"https://www.resetthenet.org/\" target=\"_blank\">Reset the Net</a> campaign against mass surveillance, which takes place on the internet today.</p>\n<p>Because the NSA continues to exploit weak links in internet security in order to spy on the world, the only way for companies to fight back is to build better security into their products. The campaign encourages citizens to commit to using apps that protect their devices from prying government eyes.</p>\n<p>In an effort that goes far beyond simply displaying a banner, Automattic <a href=\"http://en.blog.wordpress.com/2014/06/05/reset-the-net/\" target=\"_blank\">announced</a> that it will be implementing SSL for all *.wordpress.com subdomains by the end of the year. Paul Sieminski, General Counsel for Automattic, reaffirmed the company&#8217;s commitment to secure the connection between users and WordPress.com websites:</p>\n<blockquote><p>If weve learned anything over the past year, its that encryption, when done correctly, works. If we properly encrypt our sites and devices, we can make mass surveillance much more difficult.</p></blockquote>\n<p>As of May this year, 409 million people view more than 14.5 billion pages each month on the WordPress.com network, which includes Jetpack-enabled self-hosted sites. By the end of the year, SSL will be in place for anyone viewing a site hosted on WordPress.com. It&#8217;s important to note that SSL is already forced for WordPress.com admin areas.</p>\n<p>If you want to join in this historic push to secure the web, visit<a href=\"https://www.resetthenet.org/\" target=\"_blank\"> Reset the Net</a> to learn about specific ways that you can protect yourself and others to make mass government surveillance more difficult. Self-hosted WordPress sites can join in the campaign with the <a href=\"http://wordpress.org/plugins/reset-the-net-splash-screen/\" target=\"_blank\">Reset the Net splash screen</a> plugin, available for free on WordPress.org.</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 05 Jun 2014 17:26:01 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Sarah Gooding\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:42;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:86:\"WordPress.tv: Scott Offord: Creating and Promoting Compelling Content Through Blogging\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=34789\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:103:\"http://wordpress.tv/2014/06/05/scott-offord-creating-and-promoting-compelling-content-through-blogging/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:727:\"<div id=\"v-mWiu88t0-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/34789/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/34789/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=34789&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/05/scott-offord-creating-and-promoting-compelling-content-through-blogging/\"><img alt=\"Scott Offord: Creating And Promoting Compelling Content Through Blogging\" src=\"http://videos.videopress.com/mWiu88t0/video-066f69c912_scruberthumbnail_0.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 05 Jun 2014 08:00:05 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:43;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:72:\"WordPress.tv: Adrian Zumbrunnen: On distraction-free reading experiences\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=35490\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:89:\"http://wordpress.tv/2014/06/05/adrian-zumbrunnen-on-distraction-free-reading-experiences/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:701:\"<div id=\"v-kthC8PH3-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/35490/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/35490/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=35490&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/05/adrian-zumbrunnen-on-distraction-free-reading-experiences/\"><img alt=\"Adrian Zumbrunnen: On distraction free reading experiences\" src=\"http://videos.videopress.com/kthC8PH3/adrian-zumbrunnen-wcch14_std.original.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 05 Jun 2014 07:00:07 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:44;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:27:\"Matt: iOS 8 Embedded WebKit\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:21:\"http://ma.tt/?p=43835\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:43:\"http://ma.tt/2014/06/ios-8-embedded-webkit/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:606:\"<p><a href=\"http://9to5mac.com/2014/06/03/ios-8-webkit-changes-finally-allow-all-apps-to-have-the-same-performance-as-safari/\">iOS 8 WebKit changes finally allow all apps to have the same performance as Safari</a>. I was just asked about the future of the mobile web at last night&#8217;s WP talk in Singapore. (Which had about 300 people there, great turnout!) There are still a lot of issues for the open web in a closed mobile world, but things like this are a great step in the right direction. Another reason I can&#8217;t wait for iOS 8. Hat tip: <a href=\"http://bummyti.me/\">Matt Bumgardner</a>.</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 05 Jun 2014 06:01:49 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:14:\"Matt Mullenweg\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:45;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:54:\"WPTavern: WordPress in Vietnamese: Now 100% Translated\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wptavern.com/?p=24119\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:148:\"http://wptavern.com/wordpress-in-vietnamese-now-100-translated?utm_source=rss&utm_medium=rss&utm_campaign=wordpress-in-vietnamese-now-100-translated\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:3909:\"<div id=\"attachment_24140\" class=\"wp-caption aligncenter\"><a href=\"http://i2.wp.com/wptavern.com/wp-content/uploads/2014/06/vietnamese-flag.jpg\" rel=\"prettyphoto[24119]\"><img src=\"http://i2.wp.com/wptavern.com/wp-content/uploads/2014/06/vietnamese-flag.jpg?resize=1024%2C492\" alt=\"photo credit: patrikmloeff - cc\" class=\"size-full wp-image-24140\" /></a><p class=\"wp-caption-text\">photo credit: <a href=\"http://www.flickr.com/photos/bupia/807902145/\">patrikmloeff</a> &#8211; <a href=\"http://creativecommons.org/licenses/by-nc-nd/2.0/\">cc</a></p></div>\n<p>This month marks a big milestone for WordPress in Vietnamese. It is now 100% translated, thanks in large part to the folks at <a href=\"http://foobla.com/\" target=\"_blank\">foobla</a> who volunteered a great deal of time and effort on the final push to finish translating the software into Vietnamese.</p>\n<p>New WordPress translations don&#8217;t magically happen &#8211; they actually take quite a bit of manual work. It requires having a nuanced understanding of both languages in addition to a decent level of WordPress expertise for context. You forget how many words are in WordPress until you see pages and pages of strings in need of translation:</p>\n<p><a href=\"http://i1.wp.com/wptavern.com/wp-content/uploads/2014/06/translate-strings.jpg\" rel=\"prettyphoto[24119]\"><img src=\"http://i1.wp.com/wptavern.com/wp-content/uploads/2014/06/translate-strings.jpg?resize=984%2C525\" alt=\"translate-strings\" class=\"aligncenter size-full wp-image-24128\" /></a></p>\n<p>Back in December 2013, the <a href=\"http://wpvi.wordpress.com/2013/12/13/current-state-of-vietnamese-translations-for-wordpress-org/\" target=\"_blank\">progress of the Vietnamese translation</a> was slow and it was nowhere near finished. Coverage was sparse with /trunk/ strings at 69% complete, WordPress admin strings at 46% and network admin for multisite at just 6%. The three most recent default themes were all 0% translated.</p>\n<p>Today&#8217;s <a href=\"http://wpvi.wordpress.com/2014/06/04/wordpress-now-100-translated-into-vietnamese/\" target=\"_blank\">announcement</a> of the fully complete translation is a huge victory for anyone wanting to use WordPress in Vietnamese. <a href=\"https://twitter.com/philip_arthur\" target=\"_blank\">Philip Arthur Moore</a>, who has been heavily involved in this effort, said that contributors will now change their focus to the Vietnamese WordPress.org site:</p>\n<blockquote><p>Our next steps are to update <a href=\"http://vi.wordpress.org/\" target=\"_blank\">vi.wordpress.org</a> with the updated language packages, and make sure that when WordPress users in Vietnam to go vi.wordpress.org, they know how to begin using WordPress.</p></blockquote>\n<p>Translations are an often overlooked area of WordPress contribution, but are a necessary part of expanding the software&#8217;s use to non-English speaking parts of the world. Internationalization stats on <a href=\"http://wpcentral.io/internationalization/\" target=\"_blank\">WP Central</a> show that only 41 locals are up to date on WordPress package translations and 22 locales are behind by two or more major versions. 60 locales do not yet have a package available.</p>\n<p>More completed translations and better support for installing WordPress in any language may be the catalyst to pushing global WordPress usage stats over the 25% mark. Andrew Nacin recently set out an ambitious <a href=\"http://wptavern.com/major-internationalization-improvements-planned-for-wordpress-4-0\" target=\"_blank\">plan to improve internationalization</a> across the board for WordPress core and is aiming at getting these updates into the 4.0 release.</p>\n<p>With the planned changes in place, WordPress core will have more elegant ways to get people on track to using the software in their own languages. 2014 is shaping up to be an important year for making WordPress friendlier for its users across the globe.</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Wed, 04 Jun 2014 21:05:15 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Sarah Gooding\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:46;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:104:\"WPTavern: WordPress.com VIP Releases Presentation Documents and Resources Under Creative Commons License\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wptavern.com/?p=24081\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:252:\"http://wptavern.com/wordpress-com-vip-releases-presentation-documents-and-resources-under-creative-commons-license?utm_source=rss&utm_medium=rss&utm_campaign=wordpress-com-vip-releases-presentation-documents-and-resources-under-creative-commons-license\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:4319:\"<p><a href=\"http://i0.wp.com/wptavern.com/wp-content/uploads/2014/06/documattic.jpg\" rel=\"prettyphoto[24081]\"><img src=\"http://i0.wp.com/wptavern.com/wp-content/uploads/2014/06/documattic.jpg?resize=992%2C453\" alt=\"documattic\" class=\"aligncenter size-full wp-image-24107\" /></a></p>\n<p><a href=\"http://vip.wordpress.com/\" target=\"_blank\">WordPress.com VIP</a> announced <a href=\"https://github.com/Automattic/Documattic\" target=\"_blank\">Documattic</a> today, a new GitHub repository to house presentations and resources shared with the community. The VIP team is regularly engaged in the task of educating new clients on the strengths and features of the WordPress platform. Sara Rosso explained in the <a href=\"http://vip.wordpress.com/2014/06/04/documattic-wordpress-com-vip-presentations-resources-on-github/\" target=\"_blank\">announcement</a> why they decided to share the relevant documents:</p>\n<blockquote><p>We spend a lot of time explaining the great features of WordPress to client IT, security, and editorial teams, and work with partners and agencies to provide up-to-date information about WordPress software and its vast community of developers, designers, content creators, and site owners.</p></blockquote>\n<p>The VIP team plans to share documents that will focus mainly on technical specifications, capabilities, features, and other information with a marketing, project or educational purpose.</p>\n<p>Documattic currently contains three high quality resources. One is a recent presentation on <a href=\"https://github.com/Automattic/Documattic/tree/master/Presentations/Trends_in_Enterprise_WP_Content\" target=\"_blank\">Trends in Enterprise WordPress Content</a> by Sara Rosso. It&#8217;s available in both PDF and RevealJS slide formats.</p>\n<p><a href=\"https://github.com/Automattic/Documattic/tree/master/Presentations/Trends_in_Enterprise_WP_Content\"><img src=\"http://i0.wp.com/wptavern.com/wp-content/uploads/2014/06/trends-in-enterprise-content.jpg?resize=1024%2C768\" alt=\"trends-in-enterprise-content\" class=\"aligncenter size-full wp-image-24090\" /></a></p>\n<p>The repository also includes a document highlighting FAQs regarding <a href=\"https://github.com/Automattic/Documattic/tree/master/Security\" target=\"_blank\">WordPress security</a>. It&#8217;s basically a Q&amp;A style document filled with questions that are commonly fielded by most WordPress development agencies when speaking with clients who are new to the platform.</p>\n<p><a href=\"http://i1.wp.com/wptavern.com/wp-content/uploads/2014/06/wp-security.jpg\" rel=\"prettyphoto[24081]\"><img src=\"http://i1.wp.com/wptavern.com/wp-content/uploads/2014/06/wp-security.jpg?resize=632%2C374\" alt=\"wp-security\" class=\"aligncenter size-full wp-image-24095\" /></a></p>\n<p>The third resource is a <a href=\"https://github.com/Automattic/Documattic/tree/master/WordPress_for_Government\" target=\"_blank\">WordPress in Government FAQ</a>, which answers questions such as:</p>\n<ul>\n<li>How is WordPress being used in government?</li>\n<li>Who uses WordPress in government?</li>\n<li>Why would a government agency use WordPress over other CMS / custom-built solutions?</li>\n<li>How big is the WordPress developer ecosystem?</li>\n</ul>\n<p>These documents can be useful to any WordPress agency looking to attract more enterprise clients, as they provide an excellent starting point for articulating the strengths and capabilities of the platform.</p>\n<p>The WordPress.com VIP team invites other agencies to contribute to the repository and they plan to continue curating it and building it up to be a collaborative resource open to the community.</p>\n<p>Earlier this year Automattic added a <a href=\"https://github.com/Automattic/legalmattic\" target=\"_blank\">Legalmattic</a> repository to share its legal documents under a similar Creative Commons license. The repository contains legal documents used on WordPress.com including its Terms of Service. There&#8217;s no doubt some will find these documents useful, although the <a href=\"https://github.com/Automattic/Documattic\" target=\"_blank\">Documattic</a> repository is likely to have a broader appeal.</p>\n<p>Anyone is welcome to reuse and edit these documents as necessary, provided you link back to the original source, as specified in the Creative Commons license applicable to the shared document.</p>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Wed, 04 Jun 2014 17:37:30 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Sarah Gooding\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:47;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:72:\"WordPress.tv: Jose Antonio Maroto Fernandez: Tu WordPress 100% legal\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=32517\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:84:\"http://wordpress.tv/2014/06/04/jose-antonio-maroto-fernandez-tu-wordpress-100-legal/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:694:\"<div id=\"v-lSryyHXj-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/32517/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/32517/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=32517&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/04/jose-antonio-maroto-fernandez-tu-wordpress-100-legal/\"><img alt=\"Jose Antonio Maroto Fernandez: Tu WordPress 100% legal\" src=\"http://videos.videopress.com/lSryyHXj/video-9d0fc4cac3_scruberthumbnail_0.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Wed, 04 Jun 2014 09:00:10 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:48;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:66:\"WordPress.tv: Enrique Rodriguez Vallejo: Ya ramos responsables\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=32512\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:80:\"http://wordpress.tv/2014/06/04/enrique-rodriguez-vallejo-ya-eramos-responsables/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:684:\"<div id=\"v-2IohVD3o-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/32512/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/32512/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=32512&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/04/enrique-rodriguez-vallejo-ya-eramos-responsables/\"><img alt=\"Enrique Rodriguez Vallejo: Ya ramos responsables\" src=\"http://videos.videopress.com/2IohVD3o/video-eae210ae03_scruberthumbnail_0.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Wed, 04 Jun 2014 08:00:05 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:49;a:6:{s:4:\"data\";s:13:\"\n	\n	\n	\n	\n	\n	\n\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:149:\"WordPress.tv: Ricardo Prieto Antunez, David Borrallo Rodriguez, Veronica Valenzuela Jimenez: WordPress y la personalizacion en el ecommerce\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:28:\"http://wordpress.tv/?p=32507\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:107:\"http://wordpress.tv/2014/06/04/jose-esteban-mucientes-manso-wordpress-y-la-personalizacion-en-el-ecommerce/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:794:\"<div id=\"v-vhmWfYwU-1\" class=\"video-player\">\n</div><br />  <a rel=\"nofollow\" href=\"http://feeds.wordpress.com/1.0/gocomments/wptv.wordpress.com/32507/\"><img alt=\"\" border=\"0\" src=\"http://feeds.wordpress.com/1.0/comments/wptv.wordpress.com/32507/\" /></a> <img alt=\"\" border=\"0\" src=\"http://stats.wordpress.com/b.gif?host=wordpress.tv&blog=5089392&post=32507&subd=wptv&ref=&feed=1\" width=\"1\" height=\"1\" /><div><a href=\"http://wordpress.tv/2014/06/04/jose-esteban-mucientes-manso-wordpress-y-la-personalizacion-en-el-ecommerce/\"><img alt=\"Ricardo Prieto Antunez, David Borrallo Rodriguez, Veronica Valenzuela Jimenez: WordPress y la personalizacion en el ecommerce\" src=\"http://videos.videopress.com/vhmWfYwU/video-eff73e9174_scruberthumbnail_0.jpg\" width=\"160\" height=\"120\" /></a></div>\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Wed, 04 Jun 2014 07:00:55 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:12:\"WordPress.tv\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}}}}}}}}}}}}s:4:\"type\";i:128;s:7:\"headers\";a:9:{s:6:\"server\";s:5:\"nginx\";s:4:\"date\";s:29:\"Thu, 12 Jun 2014 18:19:15 GMT\";s:12:\"content-type\";s:8:\"text/xml\";s:14:\"content-length\";s:6:\"144804\";s:10:\"connection\";s:5:\"close\";s:4:\"vary\";s:15:\"Accept-Encoding\";s:13:\"last-modified\";s:29:\"Thu, 12 Jun 2014 18:00:31 GMT\";s:4:\"x-nc\";s:11:\"HIT lax 249\";s:13:\"accept-ranges\";s:5:\"bytes\";}s:5:\"build\";s:14:\"20130911120210\";}','no'),(1394,'_transient_timeout_feed_mod_867bd5c64f85878d03a060509cd2f92c','1402640355','no'),(1395,'_transient_feed_mod_867bd5c64f85878d03a060509cd2f92c','1402597155','no'),(1396,'_transient_timeout_feed_b9388c83948825c1edaef0d856b7b109','1402640355','no'),(1397,'_transient_feed_b9388c83948825c1edaef0d856b7b109','a:4:{s:5:\"child\";a:1:{s:0:\"\";a:1:{s:3:\"rss\";a:1:{i:0;a:6:{s:4:\"data\";s:3:\"\n	\n\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:7:\"version\";s:3:\"2.0\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:1:{s:0:\"\";a:1:{s:7:\"channel\";a:1:{i:0;a:6:{s:4:\"data\";s:72:\"\n		\n		\n		\n		\n		\n		\n				\n\n		\n		\n		\n		\n		\n		\n		\n		\n		\n		\n		\n		\n		\n		\n		\n\n	\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:7:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:39:\"WordPress Plugins  View: Most Popular\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:44:\"http://wordpress.org/plugins/browse/popular/\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:39:\"WordPress Plugins  View: Most Popular\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:8:\"language\";a:1:{i:0;a:5:{s:4:\"data\";s:5:\"en-US\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 12 Jun 2014 18:08:32 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:9:\"generator\";a:1:{i:0;a:5:{s:4:\"data\";s:25:\"http://bbpress.org/?v=1.1\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"item\";a:15:{i:0;a:6:{s:4:\"data\";s:30:\"\n			\n			\n			\n			\n			\n			\n					\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:22:\"WordPress SEO by Yoast\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:53:\"http://wordpress.org/plugins/wordpress-seo/#post-8321\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 01 Jan 2009 20:34:44 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:34:\"8321@http://wordpress.org/plugins/\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:131:\"Improve your WordPress SEO: Write better content and have a fully optimized WordPress site using the WordPress SEO plugin by Yoast.\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Joost de Valk\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:1;a:6:{s:4:\"data\";s:30:\"\n			\n			\n			\n			\n			\n			\n					\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:19:\"Google XML Sitemaps\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:63:\"http://wordpress.org/plugins/google-sitemap-generator/#post-132\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Fri, 09 Mar 2007 22:31:32 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:33:\"132@http://wordpress.org/plugins/\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:105:\"This plugin will generate a special XML sitemap which will help search engines to better index your blog.\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:5:\"arnee\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:2;a:6:{s:4:\"data\";s:30:\"\n			\n			\n			\n			\n			\n			\n					\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:14:\"Contact Form 7\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:54:\"http://wordpress.org/plugins/contact-form-7/#post-2141\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 02 Aug 2007 12:45:03 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:34:\"2141@http://wordpress.org/plugins/\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:54:\"Just another contact form plugin. Simple but flexible.\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:16:\"Takayuki Miyoshi\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:3;a:6:{s:4:\"data\";s:30:\"\n			\n			\n			\n			\n			\n			\n					\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:19:\"All in One SEO Pack\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:58:\"http://wordpress.org/plugins/all-in-one-seo-pack/#post-753\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Fri, 30 Mar 2007 20:08:18 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:33:\"753@http://wordpress.org/plugins/\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:126:\"All in One SEO Pack is a WordPress SEO plugin to automatically optimize your WordPress blog for Search Engines such as Google.\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:8:\"uberdose\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:4;a:6:{s:4:\"data\";s:30:\"\n			\n			\n			\n			\n			\n			\n					\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:7:\"Akismet\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:45:\"http://wordpress.org/plugins/akismet/#post-15\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Fri, 09 Mar 2007 22:11:30 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:32:\"15@http://wordpress.org/plugins/\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:98:\"Akismet checks your comments against the Akismet web service to see if they look like spam or not.\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:14:\"Matt Mullenweg\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:5;a:6:{s:4:\"data\";s:30:\"\n			\n			\n			\n			\n			\n			\n					\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:24:\"Jetpack by WordPress.com\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:48:\"http://wordpress.org/plugins/jetpack/#post-23862\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 20 Jan 2011 02:21:38 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:35:\"23862@http://wordpress.org/plugins/\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:104:\"Supercharge your WordPress site with powerful features previously only available to WordPress.com users.\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:9:\"Tim Moore\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:6;a:6:{s:4:\"data\";s:30:\"\n			\n			\n			\n			\n			\n			\n					\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:33:\"WooCommerce - excelling eCommerce\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:52:\"http://wordpress.org/plugins/woocommerce/#post-29860\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Mon, 05 Sep 2011 08:13:36 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:35:\"29860@http://wordpress.org/plugins/\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:97:\"WooCommerce is a powerful, extendable eCommerce plugin that helps you sell anything. Beautifully.\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:9:\"WooThemes\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:7;a:6:{s:4:\"data\";s:30:\"\n			\n			\n			\n			\n			\n			\n					\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:20:\"MailPoet Newsletters\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:59:\"http://wordpress.org/plugins/wysija-newsletters/#post-32629\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Fri, 02 Dec 2011 17:09:16 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:35:\"32629@http://wordpress.org/plugins/\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:94:\"Send newsletters, post notifications or autoresponders from WordPress easily, and beautifully.\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:14:\"MailPoet Staff\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:8;a:6:{s:4:\"data\";s:30:\"\n			\n			\n			\n			\n			\n			\n					\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:18:\"Wordfence Security\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:50:\"http://wordpress.org/plugins/wordfence/#post-29832\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Sun, 04 Sep 2011 03:13:51 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:35:\"29832@http://wordpress.org/plugins/\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:137:\"Wordfence Security is a free enterprise class security and performance plugin that makes your site up to 50 times faster and more secure.\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:9:\"Wordfence\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:9;a:6:{s:4:\"data\";s:30:\"\n			\n			\n			\n			\n			\n			\n					\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:18:\"WordPress Importer\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:59:\"http://wordpress.org/plugins/wordpress-importer/#post-18101\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 20 May 2010 17:42:45 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:35:\"18101@http://wordpress.org/plugins/\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:101:\"Import posts, pages, comments, custom fields, categories, tags and more from a WordPress export file.\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:14:\"Brian Colinger\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:10;a:6:{s:4:\"data\";s:30:\"\n			\n			\n			\n			\n			\n			\n					\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:21:\"WPtouch Mobile Plugin\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:47:\"http://wordpress.org/plugins/wptouch/#post-5468\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Thu, 01 May 2008 04:58:09 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:34:\"5468@http://wordpress.org/plugins/\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:63:\"Create a slick mobile WordPress website with just a few clicks.\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:17:\"BraveNewCode Inc.\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:11;a:6:{s:4:\"data\";s:30:\"\n			\n			\n			\n			\n			\n			\n					\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:15:\"NextGEN Gallery\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:55:\"http://wordpress.org/plugins/nextgen-gallery/#post-1169\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Mon, 23 Apr 2007 20:08:06 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:34:\"1169@http://wordpress.org/plugins/\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:121:\"The most popular WordPress gallery plugin and one of the most popular plugins of all time with over 10 million downloads.\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:9:\"Alex Rabe\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:12;a:6:{s:4:\"data\";s:30:\"\n			\n			\n			\n			\n			\n			\n					\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:46:\"iThemes Security (formerly Better WP Security)\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:59:\"http://wordpress.org/plugins/better-wp-security/#post-21738\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Fri, 22 Oct 2010 22:06:05 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:35:\"21738@http://wordpress.org/plugins/\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:63:\"The easiest, most effective way to secure WordPress in seconds.\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:13:\"Chris Wiegman\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:13;a:6:{s:4:\"data\";s:30:\"\n			\n			\n			\n			\n			\n			\n					\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:33:\"Google Analytics Dashboard for WP\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:74:\"http://wordpress.org/plugins/google-analytics-dashboard-for-wp/#post-50539\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Sun, 10 Mar 2013 17:07:11 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:35:\"50539@http://wordpress.org/plugins/\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:148:\"Displays Google Analytics Reports and Real-Time Statistics in your Dashboard. Automatically inserts the tracking code in every page of your website.\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:10:\"Alin Marcu\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}i:14;a:6:{s:4:\"data\";s:30:\"\n			\n			\n			\n			\n			\n			\n					\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";s:5:\"child\";a:2:{s:0:\"\";a:5:{s:5:\"title\";a:1:{i:0;a:5:{s:4:\"data\";s:16:\"TinyMCE Advanced\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:56:\"http://wordpress.org/plugins/tinymce-advanced/#post-2082\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:7:\"pubDate\";a:1:{i:0;a:5:{s:4:\"data\";s:31:\"Wed, 27 Jun 2007 15:00:26 +0000\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:4:\"guid\";a:1:{i:0;a:5:{s:4:\"data\";s:34:\"2082@http://wordpress.org/plugins/\";s:7:\"attribs\";a:1:{s:0:\"\";a:1:{s:11:\"isPermaLink\";s:5:\"false\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}s:11:\"description\";a:1:{i:0;a:5:{s:4:\"data\";s:71:\"Enables the advanced features of TinyMCE, the WordPress WYSIWYG editor.\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}s:32:\"http://purl.org/dc/elements/1.1/\";a:1:{s:7:\"creator\";a:1:{i:0;a:5:{s:4:\"data\";s:10:\"Andrew Ozz\";s:7:\"attribs\";a:0:{}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}}}s:27:\"http://www.w3.org/2005/Atom\";a:1:{s:4:\"link\";a:1:{i:0;a:5:{s:4:\"data\";s:0:\"\";s:7:\"attribs\";a:1:{s:0:\"\";a:3:{s:4:\"href\";s:45:\"http://wordpress.org/plugins/rss/view/popular\";s:3:\"rel\";s:4:\"self\";s:4:\"type\";s:19:\"application/rss+xml\";}}s:8:\"xml_base\";s:0:\"\";s:17:\"xml_base_explicit\";b:0;s:8:\"xml_lang\";s:0:\"\";}}}}}}}}}}}}s:4:\"type\";i:128;s:7:\"headers\";a:7:{s:6:\"server\";s:5:\"nginx\";s:4:\"date\";s:29:\"Thu, 12 Jun 2014 18:19:15 GMT\";s:12:\"content-type\";s:23:\"text/xml; charset=UTF-8\";s:10:\"connection\";s:5:\"close\";s:4:\"vary\";s:15:\"Accept-Encoding\";s:13:\"last-modified\";s:29:\"Thu, 01 Jan 2009 20:34:44 GMT\";s:4:\"x-nc\";s:11:\"HIT lax 249\";}s:5:\"build\";s:14:\"20130911120210\";}','no'),(1398,'_transient_timeout_feed_mod_b9388c83948825c1edaef0d856b7b109','1402640355','no'),(1399,'_transient_feed_mod_b9388c83948825c1edaef0d856b7b109','1402597155','no'),(1400,'_transient_timeout_plugin_slugs','1402691068','no'),(1401,'_transient_plugin_slugs','a:2:{i:0;s:30:\"advanced-custom-fields/acf.php\";i:1;s:39:\"slideshow-gallery/slideshow-gallery.php\";}','no'),(1402,'_transient_timeout_dash_4077549d03da2e451c8b5f002294ff51','1402640355','no'),(1403,'_transient_dash_4077549d03da2e451c8b5f002294ff51','<div class=\"rss-widget\"><ul><li><a class=\'rsswidget\' href=\'http://wordpress.org/news/2014/05/wordpress-3-9-1/\'>WordPress 3.9.1 Maintenance Release</a> <span class=\"rss-date\">May 8, 2014</span><div class=\"rssSummary\">After three weeks and more than 9 million downloads of WordPress 3.9, were pleased to announce that WordPress 3.9.1 is now available. This maintenance release fixes 34 bugs in 3.9, including numerous fixes for multisite networks, customizing widgets while previewing themes, and the updated visual editor. Weve also made some improvements to the new audio/vi</div></li></ul></div><div class=\"rss-widget\"><ul><li><a class=\'rsswidget\' href=\'http://blog.pingomatic.com/2014/06/12/book-blogger-tips/\' title=\'Over onThe Daily Post, weve published interviews with writers on growth and readership. Robert Bruce, the blogger behind the popular book site,101 Books, shared bits of advice on building a loyal readership and successful niche blog. Here are snippets from his Q&amp;A: Youve got over 26,000 followers and counting. Whats your secret? Theres really no se\'>Ping-O-Matic: Tips on Growth From a Book Blogger</a></li><li><a class=\'rsswidget\' href=\'http://wptavern.com/clef-partners-with-softaculous-to-provide-2-step-authentication-for-automated-wordpress-installs?utm_source=rss&#038;utm_medium=rss&#038;utm_campaign=clef-partners-with-softaculous-to-provide-2-step-authentication-for-automated-wordpress-installs\' title=\'Clef, the two-step authentication service that replaces usernames and passwords has announced a strategic partnership with Softaculous. Softaculous is similar to Fantastico in that it contains several scripts that can be auto installed on a webhosting account. The partnership enables automated WordPress installs from Softaculous to have Clef support out of t\'>WPTavern: Clef Partners With Softaculous To Provide 2-Step Authentication For Automated WordPress Installs</a></li><li><a class=\'rsswidget\' href=\'http://ma.tt/2014/06/extended-interview-with-forbes/\' title=\'J. J. Colao, who covered Automattic for Forbes Magazine in 2012 and has a long history and experience with WordPress and Automattic, sat down with me for close to three hours in March and somehow managed to distill it down to just a few thousand words of interview. (13,500 words down to 2,800.) Im sheepish to link it because there are a lot of I stateme\'>Matt: Extended Interview with Forbes</a></li></ul></div><div class=\"rss-widget\"><ul><li class=\'dashboard-news-plugin\'><span>Popular Plugin:</span> <a href=\'http://wordpress.org/plugins/wordfence/\' class=\'dashboard-news-plugin-link\'>Wordfence Security</a>&nbsp;<span>(<a href=\'plugin-install.php?tab=plugin-information&amp;plugin=wordfence&amp;_wpnonce=bde56c4076&amp;TB_iframe=true&amp;width=600&amp;height=800\' class=\'thickbox\' title=\'Wordfence Security\'>Install</a>)</span></li></ul></div>','no'),(1406,'Galleryinfohideonmobile','1','yes'),(1408,'_site_transient_timeout_browser_e3b8dcbd7101a0203e0e42781b74b09b','1403205068','yes'),(1409,'_site_transient_browser_e3b8dcbd7101a0203e0e42781b74b09b','a:9:{s:8:\"platform\";s:9:\"Macintosh\";s:4:\"name\";s:6:\"Chrome\";s:7:\"version\";s:13:\"35.0.1916.114\";s:10:\"update_url\";s:28:\"http://www.google.com/chrome\";s:7:\"img_src\";s:49:\"http://s.wordpress.org/images/browsers/chrome.png\";s:11:\"img_src_ssl\";s:48:\"https://wordpress.org/images/browsers/chrome.png\";s:15:\"current_version\";s:2:\"18\";s:7:\"upgrade\";b:0;s:8:\"insecure\";b:0;}','yes'),(1423,'_site_transient_timeout_poptags_40cd750bba9870f18aada2478b24840a','1402614253','yes'),(1424,'_site_transient_poptags_40cd750bba9870f18aada2478b24840a','a:40:{s:6:\"widget\";a:3:{s:4:\"name\";s:6:\"widget\";s:4:\"slug\";s:6:\"widget\";s:5:\"count\";s:4:\"4463\";}s:4:\"post\";a:3:{s:4:\"name\";s:4:\"Post\";s:4:\"slug\";s:4:\"post\";s:5:\"count\";s:4:\"2778\";}s:6:\"plugin\";a:3:{s:4:\"name\";s:6:\"plugin\";s:4:\"slug\";s:6:\"plugin\";s:5:\"count\";s:4:\"2683\";}s:5:\"admin\";a:3:{s:4:\"name\";s:5:\"admin\";s:4:\"slug\";s:5:\"admin\";s:5:\"count\";s:4:\"2196\";}s:5:\"posts\";a:3:{s:4:\"name\";s:5:\"posts\";s:4:\"slug\";s:5:\"posts\";s:5:\"count\";s:4:\"2110\";}s:7:\"sidebar\";a:3:{s:4:\"name\";s:7:\"sidebar\";s:4:\"slug\";s:7:\"sidebar\";s:5:\"count\";s:4:\"1756\";}s:6:\"google\";a:3:{s:4:\"name\";s:6:\"google\";s:4:\"slug\";s:6:\"google\";s:5:\"count\";s:4:\"1553\";}s:7:\"twitter\";a:3:{s:4:\"name\";s:7:\"twitter\";s:4:\"slug\";s:7:\"twitter\";s:5:\"count\";s:4:\"1513\";}s:8:\"comments\";a:3:{s:4:\"name\";s:8:\"comments\";s:4:\"slug\";s:8:\"comments\";s:5:\"count\";s:4:\"1465\";}s:6:\"images\";a:3:{s:4:\"name\";s:6:\"images\";s:4:\"slug\";s:6:\"images\";s:5:\"count\";s:4:\"1449\";}s:4:\"page\";a:3:{s:4:\"name\";s:4:\"page\";s:4:\"slug\";s:4:\"page\";s:5:\"count\";s:4:\"1401\";}s:9:\"shortcode\";a:3:{s:4:\"name\";s:9:\"shortcode\";s:4:\"slug\";s:9:\"shortcode\";s:5:\"count\";s:4:\"1340\";}s:5:\"image\";a:3:{s:4:\"name\";s:5:\"image\";s:4:\"slug\";s:5:\"image\";s:5:\"count\";s:4:\"1308\";}s:8:\"facebook\";a:3:{s:4:\"name\";s:8:\"Facebook\";s:4:\"slug\";s:8:\"facebook\";s:5:\"count\";s:4:\"1157\";}s:3:\"seo\";a:3:{s:4:\"name\";s:3:\"seo\";s:4:\"slug\";s:3:\"seo\";s:5:\"count\";s:4:\"1116\";}s:5:\"links\";a:3:{s:4:\"name\";s:5:\"links\";s:4:\"slug\";s:5:\"links\";s:5:\"count\";s:4:\"1096\";}s:9:\"wordpress\";a:3:{s:4:\"name\";s:9:\"wordpress\";s:4:\"slug\";s:9:\"wordpress\";s:5:\"count\";s:3:\"999\";}s:7:\"gallery\";a:3:{s:4:\"name\";s:7:\"gallery\";s:4:\"slug\";s:7:\"gallery\";s:5:\"count\";s:3:\"955\";}s:6:\"social\";a:3:{s:4:\"name\";s:6:\"social\";s:4:\"slug\";s:6:\"social\";s:5:\"count\";s:3:\"955\";}s:7:\"widgets\";a:3:{s:4:\"name\";s:7:\"widgets\";s:4:\"slug\";s:7:\"widgets\";s:5:\"count\";s:3:\"789\";}s:3:\"rss\";a:3:{s:4:\"name\";s:3:\"rss\";s:4:\"slug\";s:3:\"rss\";s:5:\"count\";s:3:\"782\";}s:5:\"pages\";a:3:{s:4:\"name\";s:5:\"pages\";s:4:\"slug\";s:5:\"pages\";s:5:\"count\";s:3:\"781\";}s:6:\"jquery\";a:3:{s:4:\"name\";s:6:\"jquery\";s:4:\"slug\";s:6:\"jquery\";s:5:\"count\";s:3:\"769\";}s:5:\"email\";a:3:{s:4:\"name\";s:5:\"email\";s:4:\"slug\";s:5:\"email\";s:5:\"count\";s:3:\"766\";}s:5:\"media\";a:3:{s:4:\"name\";s:5:\"media\";s:4:\"slug\";s:5:\"media\";s:5:\"count\";s:3:\"703\";}s:4:\"ajax\";a:3:{s:4:\"name\";s:4:\"AJAX\";s:4:\"slug\";s:4:\"ajax\";s:5:\"count\";s:3:\"678\";}s:5:\"video\";a:3:{s:4:\"name\";s:5:\"video\";s:4:\"slug\";s:5:\"video\";s:5:\"count\";s:3:\"662\";}s:10:\"javascript\";a:3:{s:4:\"name\";s:10:\"javascript\";s:4:\"slug\";s:10:\"javascript\";s:5:\"count\";s:3:\"651\";}s:7:\"content\";a:3:{s:4:\"name\";s:7:\"content\";s:4:\"slug\";s:7:\"content\";s:5:\"count\";s:3:\"619\";}s:10:\"buddypress\";a:3:{s:4:\"name\";s:10:\"buddypress\";s:4:\"slug\";s:10:\"buddypress\";s:5:\"count\";s:3:\"610\";}s:5:\"photo\";a:3:{s:4:\"name\";s:5:\"photo\";s:4:\"slug\";s:5:\"photo\";s:5:\"count\";s:3:\"592\";}s:4:\"link\";a:3:{s:4:\"name\";s:4:\"link\";s:4:\"slug\";s:4:\"link\";s:5:\"count\";s:3:\"583\";}s:4:\"feed\";a:3:{s:4:\"name\";s:4:\"feed\";s:4:\"slug\";s:4:\"feed\";s:5:\"count\";s:3:\"579\";}s:5:\"login\";a:3:{s:4:\"name\";s:5:\"login\";s:4:\"slug\";s:5:\"login\";s:5:\"count\";s:3:\"579\";}s:6:\"photos\";a:3:{s:4:\"name\";s:6:\"photos\";s:4:\"slug\";s:6:\"photos\";s:5:\"count\";s:3:\"568\";}s:8:\"category\";a:3:{s:4:\"name\";s:8:\"category\";s:4:\"slug\";s:8:\"category\";s:5:\"count\";s:3:\"532\";}s:7:\"youtube\";a:3:{s:4:\"name\";s:7:\"youtube\";s:4:\"slug\";s:7:\"youtube\";s:5:\"count\";s:3:\"525\";}s:4:\"spam\";a:3:{s:4:\"name\";s:4:\"spam\";s:4:\"slug\";s:4:\"spam\";s:5:\"count\";s:3:\"524\";}s:8:\"security\";a:3:{s:4:\"name\";s:8:\"security\";s:4:\"slug\";s:8:\"security\";s:5:\"count\";s:3:\"510\";}s:5:\"share\";a:3:{s:4:\"name\";s:5:\"Share\";s:4:\"slug\";s:5:\"share\";s:5:\"count\";s:3:\"507\";}}','yes'),(1425,'_site_transient_timeout_theme_roots','1402605259','yes'),(1426,'_site_transient_theme_roots','a:5:{s:13:\"child_of_2012\";s:7:\"/themes\";s:12:\"twentyeleven\";s:7:\"/themes\";s:14:\"twentyfourteen\";s:7:\"/themes\";s:14:\"twentythirteen\";s:7:\"/themes\";s:12:\"twentytwelve\";s:7:\"/themes\";}','yes'),(1430,'sds_slider_whi','s:77:\"a:3:{s:5:\"width\";s:3:\"400\";s:6:\"height\";s:3:\"320\";s:8:\"interval\";s:4:\"1800\";}\";','yes'),(1438,'_site_transient_update_plugins','O:8:\"stdClass\":4:{s:12:\"last_checked\";i:1402603777;s:7:\"checked\";a:3:{s:30:\"advanced-custom-fields/acf.php\";s:5:\"4.3.8\";s:33:\"show-sliders/atw-show-sliders.php\";s:5:\"1.0.8\";s:39:\"slideshow-gallery/slideshow-gallery.php\";s:7:\"1.4.4.3\";}s:8:\"response\";a:0:{}s:12:\"translations\";a:0:{}}','yes'),(1440,'_transient_is_multi_author','0','yes');
/*!40000 ALTER TABLE `wp_options` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_postmeta`
--

DROP TABLE IF EXISTS `wp_postmeta`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_postmeta` (
  `meta_id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
  `post_id` bigint(20) unsigned NOT NULL DEFAULT '0',
  `meta_key` varchar(255) DEFAULT NULL,
  `meta_value` longtext,
  PRIMARY KEY (`meta_id`),
  KEY `post_id` (`post_id`),
  KEY `meta_key` (`meta_key`)
) ENGINE=InnoDB AUTO_INCREMENT=2928 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_postmeta`
--

LOCK TABLES `wp_postmeta` WRITE;
/*!40000 ALTER TABLE `wp_postmeta` DISABLE KEYS */;
INSERT INTO `wp_postmeta` VALUES (26,10,'_edit_last','1'),(27,10,'_wp_page_template','default'),(28,10,'_edit_lock','1402341864:1'),(29,12,'_edit_last','1'),(30,12,'_wp_page_template','page-templates/publication_page.php'),(31,12,'_edit_lock','1394641110:1'),(32,14,'_edit_last','1'),(33,14,'_wp_page_template','default'),(34,14,'_edit_lock','1402598605:1'),(35,16,'_edit_last','1'),(36,16,'_wp_page_template','default'),(37,16,'_edit_lock','1402345802:1'),(45,20,'_wp_attached_file','2014/02/cropped-genetics_lab_home_header.png'),(46,20,'_wp_attachment_context','custom-header'),(47,20,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:700;s:6:\"height\";i:182;s:4:\"file\";s:44:\"2014/02/cropped-genetics_lab_home_header.png\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:44:\"cropped-genetics_lab_home_header-150x150.png\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:9:\"image/png\";}s:6:\"medium\";a:4:{s:4:\"file\";s:43:\"cropped-genetics_lab_home_header-300x78.png\";s:5:\"width\";i:300;s:6:\"height\";i:78;s:9:\"mime-type\";s:9:\"image/png\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:44:\"cropped-genetics_lab_home_header-624x162.png\";s:5:\"width\";i:624;s:6:\"height\";i:162;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(48,20,'_wp_attachment_is_custom_header','child_of_2012'),(49,21,'_menu_item_type','custom'),(50,21,'_menu_item_menu_item_parent','0'),(51,21,'_menu_item_object_id','21'),(52,21,'_menu_item_object','custom'),(53,21,'_menu_item_target',''),(54,21,'_menu_item_classes','a:1:{i:0;s:0:\"\";}'),(55,21,'_menu_item_xfn',''),(56,21,'_menu_item_url','http://localhost/~psalessi/Genetics_Lab/'),(58,22,'_menu_item_type','post_type'),(59,22,'_menu_item_menu_item_parent','0'),(60,22,'_menu_item_object_id','10'),(61,22,'_menu_item_object','page'),(62,22,'_menu_item_target',''),(63,22,'_menu_item_classes','a:1:{i:0;s:0:\"\";}'),(64,22,'_menu_item_xfn',''),(65,22,'_menu_item_url',''),(85,25,'_menu_item_type','post_type'),(86,25,'_menu_item_menu_item_parent','0'),(87,25,'_menu_item_object_id','12'),(88,25,'_menu_item_object','page'),(89,25,'_menu_item_target',''),(90,25,'_menu_item_classes','a:1:{i:0;s:0:\"\";}'),(91,25,'_menu_item_xfn',''),(92,25,'_menu_item_url',''),(94,26,'_menu_item_type','post_type'),(95,26,'_menu_item_menu_item_parent','0'),(96,26,'_menu_item_object_id','16'),(97,26,'_menu_item_object','page'),(98,26,'_menu_item_target',''),(99,26,'_menu_item_classes','a:1:{i:0;s:0:\"\";}'),(100,26,'_menu_item_xfn',''),(101,26,'_menu_item_url',''),(112,28,'_menu_item_type','post_type'),(113,28,'_menu_item_menu_item_parent','0'),(114,28,'_menu_item_object_id','14'),(115,28,'_menu_item_object','page'),(116,28,'_menu_item_target',''),(117,28,'_menu_item_classes','a:1:{i:0;s:0:\"\";}'),(118,28,'_menu_item_xfn',''),(119,28,'_menu_item_url',''),(125,30,'_edit_last','1'),(126,30,'_edit_lock','1402604705:1'),(127,30,'_wp_page_template','page-templates/front-page.php'),(160,69,'_edit_last','1'),(161,69,'field_52fbc541cfb03','a:11:{s:3:\"key\";s:19:\"field_52fbc541cfb03\";s:5:\"label\";s:15:\"Profile Picture\";s:4:\"name\";s:16:\"_profile_picture\";s:4:\"type\";s:5:\"image\";s:12:\"instructions\";s:22:\"Upload profile picture\";s:8:\"required\";s:1:\"0\";s:11:\"save_format\";s:3:\"url\";s:12:\"preview_size\";s:9:\"thumbnail\";s:7:\"library\";s:3:\"all\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:0:\"\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:0;}'),(163,69,'position','side'),(164,69,'layout','no_box'),(165,69,'hide_on_screen',''),(166,69,'name',''),(167,69,'work_position',''),(168,69,'email',''),(169,69,'work_address',''),(170,69,'work_phone',''),(171,69,'curriculum_vitae',''),(172,69,'_edit_lock','1393006235:1'),(208,72,'_edit_last','1'),(209,72,'field_52fbce8765544','a:10:{s:3:\"key\";s:19:\"field_52fbce8765544\";s:5:\"label\";s:16:\"Curriculum Vitae\";s:4:\"name\";s:17:\"_curriculum_vitae\";s:4:\"type\";s:4:\"file\";s:12:\"instructions\";s:27:\"Upload a curriculum vitae! \";s:8:\"required\";s:1:\"0\";s:11:\"save_format\";s:3:\"url\";s:7:\"library\";s:3:\"all\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:0:\"\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:0;}'),(211,72,'position','side'),(212,72,'layout','no_box'),(213,72,'hide_on_screen',''),(214,72,'name',''),(215,72,'work_position',''),(216,72,'email',''),(217,72,'work_address',''),(218,72,'work_phone',''),(219,72,'curriculum_vitae',''),(220,72,'_edit_lock','1393006226:1'),(248,80,'_edit_last','1'),(249,80,'_edit_lock','1395336702:1'),(251,81,'_wp_attached_file','2014/02/IMG_1612.jpg'),(252,81,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:300;s:6:\"height\";i:225;s:4:\"file\";s:20:\"2014/02/IMG_1612.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:20:\"IMG_1612-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";d:7.0999999999999996;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:24:\"Canon PowerShot SD800 IS\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:1283605073;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";s:3:\"4.6\";s:3:\"iso\";i:0;s:13:\"shutter_speed\";s:6:\"0.0025\";s:5:\"title\";s:0:\"\";}}'),(253,82,'_wp_attached_file','2014/02/Meek-Meek-CV-2013.pdf'),(254,83,'_wp_attached_file','2014/02/shapeimage_2.png'),(255,83,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:700;s:6:\"height\";i:120;s:4:\"file\";s:24:\"2014/02/shapeimage_2.png\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:24:\"shapeimage_2-150x120.png\";s:5:\"width\";i:150;s:6:\"height\";i:120;s:9:\"mime-type\";s:9:\"image/png\";}s:6:\"medium\";a:4:{s:4:\"file\";s:23:\"shapeimage_2-300x51.png\";s:5:\"width\";i:300;s:6:\"height\";i:51;s:9:\"mime-type\";s:9:\"image/png\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:24:\"shapeimage_2-624x106.png\";s:5:\"width\";i:624;s:6:\"height\";i:106;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(256,80,'_thumbnail_id','83'),(257,80,'profile_picture','81'),(258,80,'_profile_picture','81'),(259,80,'curriculum_vitae','82'),(260,80,'_curriculum_vitae','82'),(261,80,'work_position','Junior Programmer'),(262,80,'email','pasalessi@ucdavis.edu'),(263,80,'work_address','2323 Shasta Drive Apt. #40\r\nDavis, CA 95616'),(264,80,'work_phone','(626) 340-5993'),(265,80,'education','UC Davis 2012-2016 CS Major'),(266,80,'research_interests','I like pie and cake and food and all the jazz.'),(267,80,'research_projects','I\'m currently making this site! :D'),(268,80,'publications','Hey there. How are you?'),(305,80,'_work_position','Assistant Project Scientist'),(307,80,'_work_address','University of California, Davis\r\nDepartment of Animal Science\r\nOne Shields Avenue, Davis, CA 95616'),(325,80,'_work_phone','(530) 752-0802'),(330,80,'__profile_picture','field_52fbc541cfb03'),(331,80,'__curriculum_vitae','field_52fbce8765544'),(336,80,'_email','mhmeek@ucdavis.edu'),(345,82,'_edit_lock','1394046736:1'),(346,81,'_edit_lock','1394046761:1'),(413,93,'_edit_last','1'),(414,93,'field_530503a0f8e71','a:11:{s:3:\"key\";s:19:\"field_530503a0f8e71\";s:5:\"label\";s:9:\"Education\";s:4:\"name\";s:10:\"_education\";s:4:\"type\";s:7:\"wysiwyg\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:13:\"default_value\";s:0:\"\";s:7:\"toolbar\";s:4:\"full\";s:12:\"media_upload\";s:3:\"yes\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:0:\"\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:0;}'),(416,93,'position','normal'),(417,93,'layout','no_box'),(418,93,'hide_on_screen',''),(419,93,'_edit_lock','1393276569:1'),(423,80,'__education','field_530503a0f8e71'),(428,80,'_education','PhD, Ecology, University of California-Davis, 2010\r\n\r\nDissertation: Invasion biology of three hydrozoan species in the San Francisco Estuary, CA\r\n\r\nB.S., Zoology and Biology, Fisheries Minor, University of Washington, 2000'),(442,80,'_research_interests','My primary research interests are in the ecological and evolutionary processes that generate and maintain genetic diversity within and among populations, with an emphasis on applying an understanding of these patterns and processes to problems in conservation and management.'),(443,80,'_research_projects','ECOLOGY AND POPULATION GENOMICS OF CALIFORNIA CENTRAL VALLEY SALMONIDS\r\n\r\nI am investigating the population genomics of the Endangered Species Act listed California Central Valley Chinook salmon (<em>Oncorhynchus tshawytscha</em>). I am using next-generation sequencing techniques (RAD-seq) to evaluate population structure within the Central Valley and to look for signatures of natural selection across the genome. This work will elucidate the genomic mechanisms leading to adaptive variation in life history traits in Chinook salmon populations. Additionally, I am identifying a panel of single nucleotide polymorphism (SNP) markers that will improve our ability to distinguish between the various Chinook salmon run types. This work will improve our understanding of the genomic structure involved in adaptive variation among populations. It will be informative in the management of this imperiled species, including evaluating potential source populations for reintroduction efforts, genetic monitoring of populations in the face of climate change, and identifying juveniles to run type.\r\n\r\nHABITAT USE OF JUVENILE CHINOOK SALMON IN THE SACRAMENTO-SAN JOAQUIN DELTA\r\n\r\nSpring and winter run Chinook salmon are listed on the Endangered Species Act and very little is known about the how juveniles from the runs differentially use habitat. Through collaborations with the Dept. of Water Resources, Bureau of Reclamation, and other researchers at UC Davis, I am using molecular techniques to quantify the proportional use of flood plain habitat (the Yolo Bypass) and the mainstream Sacramento R. by each of the spring, winter, fall, and late-fall Chinook runs. I am also investigating the natal origins of the fish sampled to determine if fish from different streams have different migration pathways. This information will aid our understanding of how different populations within the same species partition habitat in space and time, as well as allow managers to better direct restoration and conservation efforts to protect threatened and endangered populations.\r\n\r\nCONSERVATION OF EXTREMELY SMALL POPULATIONS\r\n\r\nThere are over 300 species in the United States that are listed as Critically Endangered under the IUCN Red List. These are the most imperiled populations we know of, yet there are many impediments to their conservation and protection, including large gaps in our understanding of how to protect them. In order to identify what the major data gaps are for the conservation of species on the verge of extinction, I convened and organized a special interdisciplinary symposium and workshop on the topic: <a title=\"save_small_pops\" href=\"http://animalscience.ucdavis.edu/SaveSmallPops\" target=\"_blank\">animalscience.ucdavis.edu/SaveSmallPops</a>. I am now collaborating with the workshop participants to collate the understanding gained from this workshop. We are working on a manuscript to identify the greatest impediments to the conservation of extremely small populations.\r\n\r\n<a title=\"Brackish Water Hydrozoans\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/brackish-water-hydrozoans/\">INVASION BIOLOGY AND MOLECULAR ECOLOGY OF BRACKISH WATER HYDROZOANS IN THE SAN FRANCISCO ESTUARY</a>\r\n\r\nA suite of four predatory hydrozoan species (<em>Maeotias marginata</em>, <em>Moerisia sp.</em>, <em>Blackfordia virginica</em>, and <em>Cordylophora caspia</em>) have invaded and become abundant in the San Francisco Estuary (SFE). Our current level of knowledge regarding the basic biology and ecology of these organisms is alarmingly poor in light of both their possibly negative effect on the SFE ecosystem and the increasing trends in jellyfish blooms around the globe. My research seeks to investigate the potential effects of these species on the SFE ecosystem, to determine the key factors allowing successful establishment and spread of these species, and to predict future effects and spread of the invasions. Please <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">see the project page</a> for more information.'),(444,80,'_publications','Wintzer, A., <strong>M. Meek</strong>, and P. Moyle. 2013. Abundance trends and diel feeding ecology of Blackfordia virginica (Mayer, 1910), a non-native hydrozoan in the lower Napa and Petaluma Rivers, California (USA). Aquatic Invasions. 8(2): 147-156. <a title=\"Wintzer_2013.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Mariah_Meek_files/Wintzer%20et%20al%202013.pdf\">pdf</a>\r\n\r\n<strong>Meek, M.</strong>, A. Wintzer, N. Sheperd, and B. May. 2012a. Genetic diversity and reproductive mode in two non-native hydromedusae, Maeotias marginata and Moerisia sp., in the Upper San Francisco Estuary, California. Biological Invasions. 15(1): 199-212. <a title=\"Meel_2012a.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Mariah_Meek_files/2012%20Meek%20et%20al.pdf\">pdf</a>\r\n\r\n<strong>Meek, M.</strong> A. Wintzer, W. Wetzel, and B. May. 2012b. Climate change likely to facilitate the invasion of the non- native hydroid, Cordylophora caspia, in the San Francisco Estuary, CA. PLoS ONE. 7(10): e46373. doi:10.1371/journal.pone.0046373. <a title=\"Meek_2012b.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Mariah_Meek_files/Meek%20et%20al%202012%20PLoS%20ONE.pdf\">pdf</a>\r\n\r\nBlickley, J., K. Deiner, K. Garbach, I. Lacher, <strong>M. Meek</strong>*, L. Porensky, M. Wilkerson, E. Winford, and M. Schwartz. 2012. A graduate student\'s guide to necessary skill sets for conservation careers outside academia. Conservation Biology. 27(1): 24-34. *Order for first 8 authors determined alphabetically as all contributed equally. <a title=\"Blickley_2012.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Mariah_Meek_files/Blickley%20et%20al.pdf\">pdf</a>\r\n\r\nWintzer, A., <strong>M. Meek</strong>, P. Moyle, and B. May. 2011a. Ecological insights into the polyp stage of non-native hydrozoans in the San Francisco Estuary. Aquatic Ecology. 5(2): 151-161. <a title=\"Wintzer_2011a.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Mariah_Meek_files/wintzer%20et%20al%202010-aquatic%20ecology-polyp%20paper.pdf\">pdf</a>\r\n\r\nWintzer, A., <strong>M. Meek</strong>, and P. Moyle. 2011b. Trophic ecology of two non-native hydrozoans in the upper San Francisco Estuary: implications for the Pelagic Organism Decline. Marine and Freshwater Res. 62(8): 952-961. Cover story. <a title=\"Wintzer_2011b.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Mariah_Meek_files/Wintzer%20et%20al.%202011.pdf\">pdf</a>\r\n\r\nWintzer, A., <strong>M. Meek</strong>, and P. Moyle. 2011c. Life history and population dynamics of Moerisia sp., a non-native hydrozoan in the upper San Francisco Estuary (U.S.A.). Estuarine Coastal and Shelf Sci. DOI:10.1016/j.ecss.2011.05.017. <a title=\"Wintzer_2011c.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Mariah_Meek_files/Wintzer%20et%20al%202011b.pdf\">pdf</a>\r\n\r\n<strong>Meek, M.</strong>, M. Baerwald, A. Wintzer, and B. May. 2009. Isolation and characterization of microsatellite loci in two non-native hydromedusae in the San Francisco Estuary: Maeotias marginata and Moerisia sp. Conservation Genetics Resources. 1(1): 205-208. <a title=\"Meek_2009.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Mariah_Meek_files/Meek%20et%20al%202009%20Con%20Genet%20Res.pdf\">pdf</a>'),(445,80,'_educationn','hey there\r\n\r\nhow are you'),(446,80,'__educationn','field_530503a0f8e71'),(450,94,'_edit_last','1'),(451,94,'field_530507cebdd0e','a:11:{s:3:\"key\";s:19:\"field_530507cebdd0e\";s:5:\"label\";s:18:\"Research Interests\";s:4:\"name\";s:19:\"_research_interests\";s:4:\"type\";s:7:\"wysiwyg\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:13:\"default_value\";s:0:\"\";s:7:\"toolbar\";s:4:\"full\";s:12:\"media_upload\";s:3:\"yes\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:0:\"\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:0;}'),(453,94,'position','normal'),(454,94,'layout','no_box'),(455,94,'hide_on_screen',''),(456,94,'_edit_lock','1393006217:1'),(457,95,'_edit_last','1'),(458,95,'field_530507efae298','a:11:{s:3:\"key\";s:19:\"field_530507efae298\";s:5:\"label\";s:17:\"Research Projects\";s:4:\"name\";s:18:\"_research_projects\";s:4:\"type\";s:7:\"wysiwyg\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:13:\"default_value\";s:0:\"\";s:7:\"toolbar\";s:4:\"full\";s:12:\"media_upload\";s:3:\"yes\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:0:\"\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:0;}'),(460,95,'position','normal'),(461,95,'layout','no_box'),(462,95,'hide_on_screen',''),(463,95,'_edit_lock','1393006208:1'),(464,96,'_edit_last','1'),(465,96,'field_5305081a9bade','a:11:{s:3:\"key\";s:19:\"field_5305081a9bade\";s:5:\"label\";s:12:\"Publications\";s:4:\"name\";s:13:\"_publications\";s:4:\"type\";s:7:\"wysiwyg\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:13:\"default_value\";s:0:\"\";s:7:\"toolbar\";s:4:\"full\";s:12:\"media_upload\";s:3:\"yes\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:0:\"\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:0;}'),(467,96,'position','normal'),(468,96,'layout','no_box'),(469,96,'hide_on_screen',''),(470,96,'_edit_lock','1393006198:1'),(482,80,'__research_interests','field_530507cebdd0e'),(483,80,'__research_projects','field_530507efae298'),(484,80,'__publications','field_5305081a9bade'),(494,97,'_edit_last','1'),(495,97,'field_530795285c18b','a:11:{s:3:\"key\";s:19:\"field_530795285c18b\";s:5:\"label\";s:13:\"Project Image\";s:4:\"name\";s:14:\"_project_image\";s:4:\"type\";s:5:\"image\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:11:\"save_format\";s:3:\"url\";s:12:\"preview_size\";s:9:\"thumbnail\";s:7:\"library\";s:3:\"all\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:0:\"\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:0;}'),(497,97,'position','side'),(498,97,'layout','no_box'),(499,97,'hide_on_screen',''),(500,97,'_edit_lock','1393011950:1'),(503,96,'rule','a:5:{s:5:\"param\";s:9:\"post_type\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:6:\"people\";s:8:\"order_no\";i:0;s:8:\"group_no\";i:0;}'),(504,95,'rule','a:5:{s:5:\"param\";s:9:\"post_type\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:6:\"people\";s:8:\"order_no\";i:0;s:8:\"group_no\";i:0;}'),(505,94,'rule','a:5:{s:5:\"param\";s:9:\"post_type\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:6:\"people\";s:8:\"order_no\";i:0;s:8:\"group_no\";i:0;}'),(506,72,'rule','a:5:{s:5:\"param\";s:9:\"post_type\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:6:\"people\";s:8:\"order_no\";i:0;s:8:\"group_no\";i:0;}'),(507,69,'rule','a:5:{s:5:\"param\";s:9:\"post_type\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:6:\"people\";s:8:\"order_no\";i:0;s:8:\"group_no\";i:0;}'),(508,93,'rule','a:5:{s:5:\"param\";s:9:\"post_type\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:6:\"people\";s:8:\"order_no\";i:0;s:8:\"group_no\";i:0;}'),(509,97,'rule','a:5:{s:5:\"param\";s:9:\"post_type\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:8:\"projects\";s:8:\"order_no\";i:0;s:8:\"group_no\";i:0;}'),(519,100,'_edit_last','1'),(520,100,'field_53079733f3610','a:11:{s:3:\"key\";s:19:\"field_53079733f3610\";s:5:\"label\";s:36:\"Background and Significance of Study\";s:4:\"name\";s:11:\"_background\";s:4:\"type\";s:7:\"wysiwyg\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:13:\"default_value\";s:0:\"\";s:7:\"toolbar\";s:4:\"full\";s:12:\"media_upload\";s:3:\"yes\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:2:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:0;}'),(521,100,'rule','a:5:{s:5:\"param\";s:9:\"post_type\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:8:\"projects\";s:8:\"order_no\";i:0;s:8:\"group_no\";i:0;}'),(522,100,'position','normal'),(523,100,'layout','no_box'),(524,100,'hide_on_screen',''),(525,100,'_edit_lock','1393010594:1'),(526,101,'_edit_last','1'),(527,101,'field_5307975cdf5ef','a:11:{s:3:\"key\";s:19:\"field_5307975cdf5ef\";s:5:\"label\";s:21:\"Materials and Methods\";s:4:\"name\";s:22:\"_materials_and_methods\";s:4:\"type\";s:7:\"wysiwyg\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:13:\"default_value\";s:0:\"\";s:7:\"toolbar\";s:4:\"full\";s:12:\"media_upload\";s:3:\"yes\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:0:\"\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:0;}'),(529,101,'position','normal'),(530,101,'layout','no_box'),(531,101,'hide_on_screen',''),(532,101,'_edit_lock','1393007342:1'),(533,102,'_edit_last','1'),(534,102,'field_53079783f8e53','a:11:{s:3:\"key\";s:19:\"field_53079783f8e53\";s:5:\"label\";s:7:\"Results\";s:4:\"name\";s:8:\"_results\";s:4:\"type\";s:7:\"wysiwyg\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:13:\"default_value\";s:0:\"\";s:7:\"toolbar\";s:4:\"full\";s:12:\"media_upload\";s:3:\"yes\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:0:\"\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:0;}'),(536,102,'position','normal'),(537,102,'layout','no_box'),(538,102,'hide_on_screen',''),(539,102,'_edit_lock','1393007320:1'),(540,103,'_edit_last','1'),(541,103,'field_530797af2f229','a:11:{s:3:\"key\";s:19:\"field_530797af2f229\";s:5:\"label\";s:11:\"Future Work\";s:4:\"name\";s:12:\"_future_work\";s:4:\"type\";s:7:\"wysiwyg\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:13:\"default_value\";s:0:\"\";s:7:\"toolbar\";s:4:\"full\";s:12:\"media_upload\";s:3:\"yes\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:0:\"\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:0;}'),(543,103,'position','normal'),(544,103,'layout','no_box'),(545,103,'hide_on_screen',''),(546,103,'_edit_lock','1393007378:1'),(547,104,'_edit_last','1'),(548,104,'field_530797d1e08ae','a:11:{s:3:\"key\";s:19:\"field_530797d1e08ae\";s:5:\"label\";s:13:\"Collaborators\";s:4:\"name\";s:14:\"_collaborators\";s:4:\"type\";s:7:\"wysiwyg\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:13:\"default_value\";s:0:\"\";s:7:\"toolbar\";s:4:\"full\";s:12:\"media_upload\";s:3:\"yes\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:2:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:0;}'),(549,104,'rule','a:5:{s:5:\"param\";s:9:\"post_type\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:8:\"projects\";s:8:\"order_no\";i:0;s:8:\"group_no\";i:0;}'),(550,104,'position','normal'),(551,104,'layout','no_box'),(552,104,'hide_on_screen',''),(553,104,'_edit_lock','1393007035:1'),(554,105,'_edit_last','1'),(555,105,'field_530797f0836ce','a:11:{s:3:\"key\";s:19:\"field_530797f0836ce\";s:5:\"label\";s:16:\"Literature Cited\";s:4:\"name\";s:17:\"_literature_cited\";s:4:\"type\";s:7:\"wysiwyg\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:13:\"default_value\";s:0:\"\";s:7:\"toolbar\";s:4:\"full\";s:12:\"media_upload\";s:3:\"yes\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:0:\"\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:0;}'),(557,105,'position','normal'),(558,105,'layout','no_box'),(559,105,'hide_on_screen',''),(560,105,'_edit_lock','1393010776:1'),(561,107,'_wp_attached_file','2014/02/DSC01370.jpg'),(562,107,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:216;s:6:\"height\";i:162;s:4:\"file\";s:20:\"2014/02/DSC01370.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:20:\"DSC01370-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";d:8;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:9:\"CYBERSHOT\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:1063815808;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";s:1:\"7\";s:3:\"iso\";s:3:\"100\";s:13:\"shutter_speed\";s:9:\"0.0015625\";s:5:\"title\";s:0:\"\";}}'),(563,107,'_wp_attachment_image_alt','California golden trout project image of a hand holding a california golden trout'),(564,106,'_edit_last','1'),(565,106,'_edit_lock','1395335055:1'),(568,109,'_wp_attached_file','2014/02/gold_trout_map.jpg'),(569,109,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:319;s:6:\"height\";i:558;s:4:\"file\";s:26:\"2014/02/gold_trout_map.jpg\";s:5:\"sizes\";a:2:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:26:\"gold_trout_map-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}s:6:\"medium\";a:4:{s:4:\"file\";s:26:\"gold_trout_map-171x300.jpg\";s:5:\"width\";i:171;s:6:\"height\";i:300;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(570,110,'_wp_attached_file','2014/02/key_to_gold_trout_map.gif'),(571,110,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:226;s:6:\"height\";i:31;s:4:\"file\";s:33:\"2014/02/key_to_gold_trout_map.gif\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:32:\"key_to_gold_trout_map-150x31.gif\";s:5:\"width\";i:150;s:6:\"height\";i:31;s:9:\"mime-type\";s:9:\"image/gif\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(572,106,'_project_image','107'),(573,106,'__project_image','field_530795285c18b'),(574,106,'_background','California golden trout <em>Oncorhynchus mykiss aguabonita</em> were used extensively in the first half of the 20th century to stock lakes and streams throughout the western United States (Pister 1991). The source of these stocks have been attributed to an introduction of fish from either Golden Trout Creek (a tributary of the Kern River) or the South Fork Kern River (Stanley Stephens, California Department of Fish and Game, pers. comm..) to fishless Mulkey Creek (a tributary of the South Fork Kern River) prior to 1876. Twelve fish from Mulkey Creek were subsequently transplanted into Cottonwood Creek (1876) and from there into the Cottonwood Lakes (1891), which served as the source populations for stocks planted throughout the Sierra Nevada, including headwater Lakes draining into the Golden Trout Creek watershed (i.e. Johnson, Chicken Springs, and the Rocky Basin Lakes).\r\n\r\nRecent evidence based on protein allozyme analyses have indicated that the Cottonwood Creek and Cottonwood Lakes populations of California golden trout may have become introgressed with alleles from introduced rainbow trout O. m. mykiss, and that introgressed fish may have been subsequently transplanted into Johnson, Chicken Springs, and the Rocky Basin Lakes (Leary 1997). This was confirmed by a recent study (Cordes et al. 2006) of the Golden Trout Creek drainage, which confirmed low and localized areas of introgression at the site of these introductions. A more recent study evaluated South Fork Kern River populations of California golden trout and found a strikingly different pattern of introgresssion (Figure 1), with high levels of rainbow trout influence present in the lower reaches and tributaries of the South Fork Kern, corresponding to rainbow trout stocking in the lower reaches of this river (Cordes et al. 2003).\r\n\r\n[caption id=\"attachment_109\" align=\"aligncenter\" width=\"319\"]<img class=\"size-full wp-image-109\" alt=\"California golden trout map image\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gold_trout_map.jpg\" width=\"319\" height=\"558\" /> Figure 1. Comparison of admixture levels detected in Golden Trout Creek and South Fork Kern River California golden trout populations.[/caption]\r\n<p style=\"text-align: center;\"><img class=\"size-medium wp-image-110 aligncenter\" style=\"font-size: 1rem; line-height: 1;\" alt=\"key_to_gold_trout_map\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/key_to_gold_trout_map.gif\" width=\"226\" height=\"31\" /></p>\r\n<p style=\"text-align: center;\"></p>'),(575,106,'__background','field_53079733f3610'),(576,106,'_literature_cited','Cordes, J.F., M.R. Stephens, M.A. Blumberg, and B.P. May. 2006. Identifying Introgressive Hybridization in Native Populations of California Golden Trout Based on Molecular Markers. Transactions of the American Fisheries Society. 135:110128. <a title=\"Cordes_etal_2005TAFS135.pdf\" href=\"http://genome-lab.ucdavis.edu/People/Stephens/Cordes_etal_2005TAFS135.pdf\">[PDF]</a>\r\n\r\nCordes, J.F., M.A. Blumberg, G.A.E. Gall, and B. May. 2001. Genetic status of California golden trout populations in the headwaters of Golden Trout Creek. Report to the California Department of Fish and Game Threatened Trout Committee. September 2001. 34 pp.\r\n\r\nCordes, J. F., M. R. Stephens, and B. P. May. 2003. Genetic Status of California Golden Trout in the South Fork Kern River and Transplanted Populations: Report to the Threatened Trout Committee, California Department of Fish and Game (in prep). [PDF]\r\n\r\nBagley, M., G.A.E. Gall, and B. May. 1998. Genetic analysis of 1997 trout collections. Report to California Department of Fish and Game, Threatened Trout Committee. August 1998. 18 pp.\r\n\r\nGall, G.E., and B. May. 1997. Trout of the Kern River Basin: A genetic analysis of Little Kern River and Golden Trout Creek populations. Report to California Department of Fish and Game, Threatened Trout Committee. September 1997. 55 pp.\r\n\r\nLeary, R.F. 1997. Letter to Eric Gerstung, California Department of Fish and Game.\r\n\r\nPister, P. 1991. Golden trout (<em>Oncorhynchus aguabonita</em>). In Trout, J. Stoltz and J. Schnell (eds.). Stackpole Books, Harrisburg. pp.280-285.\r\n\r\nStephens, M.R., N.W. Clipperton, and B.P. May. Single Nucleotide Polymorphism markers improve hybridization assessment for native California golden trout.'),(577,106,'__literature_cited','field_530797f0836ce'),(578,106,'_collaborators','This work is progressing in collaboration with the California Department of Fish and Game (CDFG).\r\n\r\nFunding is provided by the CDFG and the US Fish and Wildlife Service.'),(579,106,'__collaborators','field_530797d1e08ae'),(580,106,'_future_work','We are currently applying SNP markers to assessment of introgression in the Golden Trout Creek watershed. A manuscript of comparison between SNP and microsatellite admixture estimates in California golden trout is in preparation (Stephens et al., in prep)\r\n\r\n<img class=\"aligncenter size-full wp-image-116\" alt=\"future_work_river\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/future_work_river.jpg\" width=\"300\" height=\"225\" />'),(581,106,'__future_work','field_530797af2f229'),(582,106,'_materials_and_methods','<strong><em>Sample Collections</em>:</strong>Fin clips samples of California golden trout were collected by California Department of Fish and Game personnel throughout the Golden Trout Creek and South Fork Kern River drainages. Rainbow trout samples were taken from hatchery and wild populations to use as references.\r\n\r\n<strong><em>Data Collection and Analysis</em>:</strong>\r\n<ul>\r\n	<li>Six microsatellite loci</li>\r\n	<li>One single copy nuclear minisatellite</li>\r\n	<li>18 SNP loci</li>\r\n	<li>Admixture analysis using ADMIX and STRUCTURE</li>\r\n</ul>'),(583,106,'__materials_and_methods','field_5307975cdf5ef'),(584,106,'_results','Microsatellite loci were used to assess population structure and introgression in California golden trout in Golden Trout Creek (Cordes et al. 2006) and the South Fork Kern River (Cordes et al. 2003; manuscript in review). Recently developed SNP markers were used in a comparative study of microsatellite versus SNP marker estimates of introgression (Stephens et al., <em>in prep</em>). This study has confirmed the ability of SNP markers to assess introgression in California golden trout populations in the South Fork Kern (Figure 2).\r\n\r\n[caption id=\"attachment_118\" align=\"aligncenter\" width=\"685\"]<img class=\"size-full wp-image-118\" alt=\"California golden trout data analysis graph image\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/red_green_graph.jpg\" width=\"685\" height=\"126\" /> Figure 2. STRUCTURE representation of SNP data analysis for South Fork Kern California golden trout.<br />Each vertical bar represents an individual fish, with populations grouped by population number (as in Table 1) on the x-axis. The green bar color indicates \"rainbow trout\" group membership, while red denotes \"golen\" group membership. The presence of green, therefore, indicates rainbow trout admixture in a given individual at levels measured by proportion on the y-axis.[/caption]\r\n<p style=\"text-align: center;\"></p>'),(585,106,'__results','field_53079783f8e53'),(599,102,'rule','a:5:{s:5:\"param\";s:9:\"post_type\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:8:\"projects\";s:8:\"order_no\";i:0;s:8:\"group_no\";i:0;}'),(600,101,'rule','a:5:{s:5:\"param\";s:9:\"post_type\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:8:\"projects\";s:8:\"order_no\";i:0;s:8:\"group_no\";i:0;}'),(602,103,'rule','a:5:{s:5:\"param\";s:9:\"post_type\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:8:\"projects\";s:8:\"order_no\";i:0;s:8:\"group_no\";i:0;}'),(613,114,'_menu_item_type','custom'),(614,114,'_menu_item_menu_item_parent','0'),(615,114,'_menu_item_object_id','114'),(616,114,'_menu_item_object','custom'),(617,114,'_menu_item_target',''),(618,114,'_menu_item_classes','a:1:{i:0;s:0:\"\";}'),(619,114,'_menu_item_xfn',''),(620,114,'_menu_item_url','http://localhost/~psalessi/Genetics_Lab/people/'),(622,115,'_menu_item_type','custom'),(623,115,'_menu_item_menu_item_parent','0'),(624,115,'_menu_item_object_id','115'),(625,115,'_menu_item_object','custom'),(626,115,'_menu_item_target',''),(627,115,'_menu_item_classes','a:1:{i:0;s:0:\"\";}'),(628,115,'_menu_item_xfn',''),(629,115,'_menu_item_url','http://localhost/~psalessi/Genetics_Lab/projects/'),(631,106,'_project_excerpt','Assessment of rainbow trout introgression with the species utilizing both microsatellite markers and SNPs'),(632,106,'_contact_email','mrstephens@ucdavis.edu '),(633,116,'_wp_attached_file','2014/02/future_work_river.jpg'),(634,116,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:300;s:6:\"height\";i:225;s:4:\"file\";s:29:\"2014/02/future_work_river.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:29:\"future_work_river-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(637,118,'_wp_attached_file','2014/02/red_green_graph.jpg'),(638,118,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:685;s:6:\"height\";i:126;s:4:\"file\";s:27:\"2014/02/red_green_graph.jpg\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:27:\"red_green_graph-150x126.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:126;s:9:\"mime-type\";s:10:\"image/jpeg\";}s:6:\"medium\";a:4:{s:4:\"file\";s:26:\"red_green_graph-300x55.jpg\";s:5:\"width\";i:300;s:6:\"height\";i:55;s:9:\"mime-type\";s:10:\"image/jpeg\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:27:\"red_green_graph-624x114.jpg\";s:5:\"width\";i:624;s:6:\"height\";i:114;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(639,106,'_contact_name','Molly Stephens'),(640,105,'rule','a:5:{s:5:\"param\";s:9:\"post_type\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:8:\"projects\";s:8:\"order_no\";i:0;s:8:\"group_no\";i:0;}'),(641,119,'_wp_attached_file','2014/02/gold_trout_header.png'),(642,119,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:700;s:6:\"height\";i:120;s:4:\"file\";s:29:\"2014/02/gold_trout_header.png\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:29:\"gold_trout_header-150x120.png\";s:5:\"width\";i:150;s:6:\"height\";i:120;s:9:\"mime-type\";s:9:\"image/png\";}s:6:\"medium\";a:4:{s:4:\"file\";s:28:\"gold_trout_header-300x51.png\";s:5:\"width\";i:300;s:6:\"height\";i:51;s:9:\"mime-type\";s:9:\"image/png\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:29:\"gold_trout_header-624x106.png\";s:5:\"width\";i:624;s:6:\"height\";i:106;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(643,106,'_thumbnail_id','119'),(651,121,'_edit_last','1'),(652,121,'field_5307b5fae265a','a:11:{s:3:\"key\";s:19:\"field_5307b5fae265a\";s:5:\"label\";s:13:\"Past Projects\";s:4:\"name\";s:14:\"_past_projects\";s:4:\"type\";s:7:\"wysiwyg\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:13:\"default_value\";s:0:\"\";s:7:\"toolbar\";s:4:\"full\";s:12:\"media_upload\";s:3:\"yes\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"1\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:19:\"field_53177905a02ac\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:1:\"1\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:1;}'),(654,121,'position','normal'),(655,121,'layout','no_box'),(656,121,'hide_on_screen',''),(657,121,'_edit_lock','1394047403:1'),(659,122,'_edit_last','1'),(660,122,'_edit_lock','1395335067:1'),(661,123,'_wp_attached_file','2014/02/past_projects_header.png'),(662,123,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:706;s:6:\"height\";i:121;s:4:\"file\";s:32:\"2014/02/past_projects_header.png\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:32:\"past_projects_header-150x121.png\";s:5:\"width\";i:150;s:6:\"height\";i:121;s:9:\"mime-type\";s:9:\"image/png\";}s:6:\"medium\";a:4:{s:4:\"file\";s:31:\"past_projects_header-300x51.png\";s:5:\"width\";i:300;s:6:\"height\";i:51;s:9:\"mime-type\";s:9:\"image/png\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:32:\"past_projects_header-624x106.png\";s:5:\"width\";i:624;s:6:\"height\";i:106;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(663,124,'_wp_attached_file','2014/02/past_projects_profile.jpg'),(664,124,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:232;s:6:\"height\";i:141;s:4:\"file\";s:33:\"2014/02/past_projects_profile.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:33:\"past_projects_profile-150x141.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:141;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";d:2.7999999999999998;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:7:\"EX-Z750\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:1126793902;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";s:3:\"7.9\";s:3:\"iso\";i:0;s:13:\"shutter_speed\";s:5:\"0.008\";s:5:\"title\";s:0:\"\";}}'),(665,122,'_thumbnail_id','123'),(669,122,'_project_image','124'),(670,122,'__project_image','field_530795285c18b'),(671,122,'_past_projects','<strong>Elk subspecies</strong>\r\n\r\nMeredith, E. P., J.A. Rodzen, J. D. Banks, R. Schaefer, H. B. Ernest, T. R. Famula and B. P. May. Microsatellite analysis of three subspecies of Elk (Cervus elaphus) in California. Journal of Mammalogy 88(3):801-808 (<a title=\"200_Meredith_elk.pdf\" href=\"http://genome-lab.ucdavis.edu/Publications/2007%20Meredith-elk.pdf\">pdf</a>).\r\n\r\n<strong>Kearney\'s bluestar</strong>\r\n\r\nTopinka, J.R., A.J. Donovan, and B. May. Characterization of microsatellite loci in Kearneys bluestar (<em>Amsonia kearneyana</em>) and cross-amplification in other <em>Amsonia</em> species. Mol. Ecol. Notes. 4:710-712 (<a title=\"Topinka_Amsonia_2004.pdf\" href=\"http://genome-lab.ucdavis.edu/Publications/Topinka%20Amsonia%202004.pdf\">pdf</a>).\r\n\r\n<strong>Klamath basin suckers</strong>\r\n\r\nTranah, G.J., J.J. Agresti, and B. May. New microsatellite loci for suckers (Catostomidae): primer homology in Catostomus, Chasmistes, and Deltistes. Mol. Ecol. Notes 1:55-60 (<a title=\"Tranah_2001_primer_note.pdf\" href=\"http://genome-lab.ucdavis.edu/Publications/Tranah%202001%20primer%20note.pdf\">pdf</a>).\r\n\r\n<strong>Cowhead Lake Tui Chub</strong>\r\n\r\n<strong>Blue Whale population structure</strong>\r\n\r\n<strong>Green crab invasions</strong>\r\n\r\n<strong>Ground Squirrels</strong>\r\n\r\nMay, B., T.A. Gavin, P.W. Sherman, and T.M. Korves. Characterization of microsatellite loci in the Northern Idaho ground squirrel, Spermophilus brunneus Mol. Ecol. 6:399-400 (<a title=\"1997_Spermophilus_brunneus.pd\" href=\"http://genome-lab.ucdavis.edu/Publications/1997%20Spermophilus%20brunneus.pdf\">pdf</a>).\r\n\r\n<strong>Klamath River steelhead</strong>\r\n\r\nPapa R., J.A. Israel, F. N. Marzan, and B. May. Assessment of genetic variation among reproductive ecotypes of Klamath River steelhead reveals differentiation associated with different run-timings. J. Appl. Icthy (<a title=\"2007_Papa_steelhead.pdf\" href=\"http://genome-lab.ucdavis.edu/Publications/2007%20Papa_steelhead.pdf\">pdf</a>).\r\n\r\n<strong>Mekong Delta Giant Catfish</strong>\r\n\r\nHogan et al. 2004. Imperiled Giants of the Mekong: Ecologists struggle to understandand protectSoutheast Asia\'s large migratory catfish. Am. Sci. 92:228-237.\r\n\r\nHogan, Z. S. and B. May. Twenty-seven new microsatellites for the migratory Asian catfish family Pangasiidae. Mol. Ecol. Notes. 2:38-41 (<a title=\"Hogan_2002_primer_note.pdf\" href=\"http://genome-lab.ucdavis.edu/Publications/Hogan%202002%20primer%20note.pdf\">pdf</a>).\r\n\r\n<strong>Modoc sucker</strong>\r\n\r\n<strong>Mohave Tui Chub</strong>\r\n\r\nLeunda, P.M., R. Miranda, J. Madoz, S. Parmenter, Y. Chen, and B. May. Threatened fishes of the world: Siphateles bicolor snyderi (Miller, 1973) (Cyprinidae). Env. Biol. Fish. 73:109-110 (<a title=\"2005_Leunda_etal.pdf\" href=\"http://genome-lab.ucdavis.edu/Publications/2005%20Leunda%20etal.pdf\">pdf</a>).\r\n\r\n<strong>Navarro River Steelhead</strong>\r\n\r\n<strong>Owens Tui Chub</strong>\r\n\r\nChen, Y., S. Parmenter, and B. May. Introgression between Lahontan and endangered Owens tui chubs, and apparent discovery of a new tui chub in the Owens Valley, California. Conserv. Genet. 8:221-238 (<a title=\"2007_Chen_Owens_tui.pdf\" href=\"http://genome-lab.ucdavis.edu/Publications/2007_Chen_Owens_tui.pdf\">pdf</a>).\r\n\r\n<strong>Pallid and Shovelnose Sturgeon</strong>\r\n\r\nTranah, G.J., D.E. Campton and B. May. Genetic evidence for hybridization of pallid and shovelnose sturgeon. J. Hered. 95:474-480 (<a title=\"Tranah_J_Heredity.pdf\" href=\"http://genome-lab.ucdavis.edu/Publications/Tranah%20J%20Heredity.pdf\">pdf</a>).\r\n\r\nTranah, G.J., H.L. Kincaid, C.C. Krueger, D.E. Campton, B. May. Reproductive isolation in sympatric populations of pallid and shovelnose sturgeon. N. Am. J. Fish. Man. 21:367-373 (<a title=\"2001TranahEtAl_NAJFM.pdf\" href=\"http://genome-lab.ucdavis.edu/Publications/2001TranahEtAl_NAJFM.pdf\">pdf</a>).\r\n\r\n<strong>Rainbow trout IHN resistance</strong>\r\n\r\nRodriguez, F., S. LaPatra, S. Williams, T. Famula and B. May. Genetic markers associated with resistance to infectious hematopoietic necrosis in rainbow and steelhead trout (Oncorhynchus mykiss) backcrosses. Aquaculture 241:93-115 (<a title=\"Rodriguez_et_al_B1_D._2004.pdf\" href=\"http://genome-lab.ucdavis.edu/Publications/Rodriguez_et_al%5B1%5D._2004.pdf\">pdf</a>).\r\n\r\n<strong>Red Swamp Crayfish</strong>\r\n\r\nBelfiore, N.M. and B. May. Variable microsatellite loci in red swamp crayfish, Procambarus clarkii, and their characterization in other crayfish taxa. Mol. Ecol. 12:2231-2234 (<a title=\"2000_red_swamp_crayfish.pdf\" href=\"http://genome-lab.ucdavis.edu/Publications/2000%20red%20swamp%20crayfish.pdf\">pdf</a>).\r\n\r\n<strong>Sacramento sucker</strong>\r\n\r\nWhitehead et al. 2003. Genetic variation among interconnected populations of Catostomus occidentalis: implications for distinguishing impacts of contaminants from biogeographic structuring. Mol. Ecol. 12: 2817-2833.\r\n\r\n<strong>Tilapia mapping and QTL detection</strong>\r\n\r\nMoen, T., J.J. Agresti, A. Cnaani, H. Moses, T.R. Famula, G. Hulata, G.A.E. Gall, B. May. A genome scan of a four-way tilapia cross supports the existence of a quantitative trait locus for cold tolerance on linkage group 23. Aqua. Res. 35:893-904 (<a title=\"Moen_e_al_2004.pdf\" href=\"http://genome-lab.ucdavis.edu/Publications/Moen%20et%20al%202004.pdf\">pdf</a>).\r\n\r\nAgresti, J.J., S. Seki, A. Cnaani, S. Poompuang, E. M. Hallerman, N. Umiel, G. Hulata, G. A.E. Gall, and B. May. Breeding new strains of tilapia: development of an artificial center of origin and linkage map based on AFLP and microsatellite loci. Aquaculture. 185:43-56 (<a title=\"2000_Tilapia.pdf\" href=\"http://genome-lab.ucdavis.edu/Publications/2000%20Tilapia.pdf\">pdf</a>).\r\n\r\n<strong>White sturgeon aquaculture</strong>\r\n\r\nRodzen, J.A., T. Famula, and B. May. Estimation of parentage and relatedness in the polyploid white sturgeon Acipenser transmontanus using a dominant marker approach for duplicated microsatellite loci. Aquaculture 232:165-182 (<a title=\"Rodzen_Aquaculture_paper.pdf\" href=\"http://genome-lab.ucdavis.edu/Publications/Rodzen%20Aquaculture%20paper.pdf\">pdf</a>).\r\n\r\nRodzen, J.A. and B. May. Inheritance of microsatellite loci in the white sturgeon (Acipenser transmontanus). Genome 45:1064-1076 (<a title=\"Rodzen_and_May_Genome_2002.pdf\" href=\"http://genome-lab.ucdavis.edu/Publications/Rodzen%20and%20May%20Genome%202002.pdf\">pdf</a>).\r\n\r\nSmith, C.T., R.J. Nelson, S. Pollard, E. Rubridge, S.J. McKay, J. Rodzen, B. May, and B. Koop. Population genetic analysis of white sturgeon (Acipenser transmontanus) in the Fraser River. J. Appl. Icthy. 18:307-312 (<a title=\"2002_Acipenser_transmontanus.pdf\" href=\"http://genome-lab.ucdavis.edu/Publications/2002%20Acipenser%20transmontanus.pdf\">pdf</a>).\r\n\r\n<strong>Yellow-bellied marmots</strong>\r\n\r\nFloyd, C.H., D. Van Vuren, and B. May. Marmots on Great Basin mountaintops: using genetics to test an ecological paradigm. Ecology. 86: 2145-2153 (<a title=\"Floyd_marmot_2005.pdf\" href=\"http://genome-lab.ucdavis.edu/Publications/Floyd%20marmot%202005.pdf\">pdf</a>).'),(672,122,'__past_projects','field_5307b5fae265a'),(673,122,'_background',''),(674,122,'__background','field_53079733f3610'),(675,122,'_materials_and_methods',''),(676,122,'__materials_and_methods','field_5307975cdf5ef'),(677,122,'_results',''),(678,122,'__results','field_53079783f8e53'),(679,122,'_future_work',''),(680,122,'__future_work','field_530797af2f229'),(681,122,'_collaborators',''),(682,122,'__collaborators','field_530797d1e08ae'),(683,122,'_literature_cited',''),(684,122,'__literature_cited','field_530797f0836ce'),(685,125,'_wp_attached_file','2014/02/publications_page_header.png'),(686,125,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:706;s:6:\"height\";i:121;s:4:\"file\";s:36:\"2014/02/publications_page_header.png\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:36:\"publications_page_header-150x121.png\";s:5:\"width\";i:150;s:6:\"height\";i:121;s:9:\"mime-type\";s:9:\"image/png\";}s:6:\"medium\";a:4:{s:4:\"file\";s:35:\"publications_page_header-300x51.png\";s:5:\"width\";i:300;s:6:\"height\";i:51;s:9:\"mime-type\";s:9:\"image/png\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:36:\"publications_page_header-624x106.png\";s:5:\"width\";i:624;s:6:\"height\";i:106;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(687,12,'_thumbnail_id','125'),(690,129,'_edit_last','1'),(691,129,'_edit_lock','1394222021:1'),(692,129,'_thumbnail_id','125'),(693,129,'_wp_page_template','page-templates/publication_page.php'),(703,135,'_edit_last','1'),(704,135,'_edit_lock','1394223035:1'),(705,135,'_wp_page_template','page-templates/publication_page.php'),(715,140,'_edit_last','1'),(716,140,'_edit_lock','1394223412:1'),(717,140,'_wp_page_template','page-templates/publication_page.php'),(718,142,'_thumbnail_id','125'),(719,142,'_edit_last','1'),(720,142,'_wp_page_template','page-templates/publication_page.php'),(721,142,'_edit_lock','1394223547:1'),(722,140,'_thumbnail_id','125'),(723,135,'_thumbnail_id','125'),(745,106,'_past_projects',''),(746,106,'__past_projects','field_5307b5fae265a'),(750,160,'_wp_attached_file','2014/02/partners_header.png'),(751,160,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:712;s:6:\"height\";i:122;s:4:\"file\";s:27:\"2014/02/partners_header.png\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:27:\"partners_header-150x122.png\";s:5:\"width\";i:150;s:6:\"height\";i:122;s:9:\"mime-type\";s:9:\"image/png\";}s:6:\"medium\";a:4:{s:4:\"file\";s:26:\"partners_header-300x51.png\";s:5:\"width\";i:300;s:6:\"height\";i:51;s:9:\"mime-type\";s:9:\"image/png\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:27:\"partners_header-624x106.png\";s:5:\"width\";i:624;s:6:\"height\";i:106;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(752,10,'_thumbnail_id','160'),(766,171,'_wp_attached_file','2014/02/BLMLogo.jpg'),(767,171,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:123;s:6:\"height\";i:123;s:4:\"file\";s:19:\"2014/02/BLMLogo.jpg\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(768,172,'_wp_attached_file','2014/02/DFGshield.jpg'),(769,172,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:102;s:6:\"height\";i:135;s:4:\"file\";s:21:\"2014/02/DFGshield.jpg\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(770,173,'_wp_attached_file','2014/02/droppedImage_1.jpg'),(771,173,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:121;s:6:\"height\";i:121;s:4:\"file\";s:26:\"2014/02/droppedImage_1.jpg\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(772,174,'_wp_attached_file','2014/02/droppedImage_2.jpg'),(773,174,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:188;s:6:\"height\";i:54;s:4:\"file\";s:26:\"2014/02/droppedImage_2.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:25:\"droppedImage_2-150x54.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:54;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(774,175,'_wp_attached_file','2014/02/droppedImage_3.jpg'),(775,175,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:93;s:6:\"height\";i:93;s:4:\"file\";s:26:\"2014/02/droppedImage_3.jpg\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(776,176,'_wp_attached_file','2014/02/droppedImage_4.jpg'),(777,176,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:124;s:6:\"height\";i:72;s:4:\"file\";s:26:\"2014/02/droppedImage_4.jpg\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(778,177,'_wp_attached_file','2014/02/droppedImage_5.jpg'),(779,177,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:213;s:6:\"height\";i:68;s:4:\"file\";s:26:\"2014/02/droppedImage_5.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:25:\"droppedImage_5-150x68.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:68;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(780,178,'_wp_attached_file','2014/02/droppedImage_6.jpg'),(781,178,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:86;s:6:\"height\";i:113;s:4:\"file\";s:26:\"2014/02/droppedImage_6.jpg\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(782,179,'_wp_attached_file','2014/02/droppedImage_7.jpg'),(783,179,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:149;s:6:\"height\";i:102;s:4:\"file\";s:26:\"2014/02/droppedImage_7.jpg\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(784,180,'_wp_attached_file','2014/02/droppedImage_8.jpg'),(785,180,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:244;s:6:\"height\";i:70;s:4:\"file\";s:26:\"2014/02/droppedImage_8.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:25:\"droppedImage_8-150x70.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:70;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(786,181,'_wp_attached_file','2014/02/droppedImage_9.jpg'),(787,181,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:119;s:6:\"height\";i:119;s:4:\"file\";s:26:\"2014/02/droppedImage_9.jpg\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(788,182,'_wp_attached_file','2014/02/droppedImage_10.jpg'),(789,182,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:157;s:6:\"height\";i:123;s:4:\"file\";s:27:\"2014/02/droppedImage_10.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:27:\"droppedImage_10-150x123.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:123;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(790,183,'_wp_attached_file','2014/02/droppedImage_11.jpg'),(791,183,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:192;s:6:\"height\";i:71;s:4:\"file\";s:27:\"2014/02/droppedImage_11.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:26:\"droppedImage_11-150x71.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:71;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(792,184,'_wp_attached_file','2014/02/droppedImage.jpg'),(793,184,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:233;s:6:\"height\";i:46;s:4:\"file\";s:24:\"2014/02/droppedImage.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:23:\"droppedImage-150x46.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:46;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(794,185,'_wp_attached_file','2014/02/droppedImage.png'),(795,185,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:97;s:6:\"height\";i:140;s:4:\"file\";s:24:\"2014/02/droppedImage.png\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(796,186,'_wp_attached_file','2014/02/dwr-logobig_home.jpg'),(797,186,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:125;s:6:\"height\";i:125;s:4:\"file\";s:28:\"2014/02/dwr-logobig_home.jpg\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(798,187,'_wp_attached_file','2014/02/shapeimage_4.png'),(799,187,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:167;s:6:\"height\";i:112;s:4:\"file\";s:24:\"2014/02/shapeimage_4.png\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:24:\"shapeimage_4-150x112.png\";s:5:\"width\";i:150;s:6:\"height\";i:112;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(800,193,'_wp_attached_file','2014/02/U_S__Fish__and__Wildlife_Service.jpg'),(801,193,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:103;s:6:\"height\";i:124;s:4:\"file\";s:44:\"2014/02/U_S__Fish__and__Wildlife_Service.jpg\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(802,194,'_wp_attached_file','2014/02/USForestServiceLogo.jpg'),(803,194,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:108;s:6:\"height\";i:121;s:4:\"file\";s:31:\"2014/02/USForestServiceLogo.jpg\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(804,195,'_wp_attached_file','2014/02/logo.jpg'),(805,195,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:165;s:6:\"height\";i:57;s:4:\"file\";s:16:\"2014/02/logo.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:15:\"logo-150x57.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:57;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(806,199,'_edit_last','1'),(807,199,'field_530b901b0ca70','a:13:{s:3:\"key\";s:19:\"field_530b901b0ca70\";s:5:\"label\";s:16:\"People Catergory\";s:4:\"name\";s:16:\"_people_category\";s:4:\"type\";s:5:\"radio\";s:12:\"instructions\";s:43:\"Others are not displayed on the People page\";s:8:\"required\";s:1:\"1\";s:7:\"choices\";a:10:{s:8:\"Director\";s:8:\"Director\";s:18:\"Project Scientists\";s:18:\"Project Scientists\";s:15:\"Adjunct Faculty\";s:15:\"Adjunct Faculty\";s:17:\"Graduate Students\";s:17:\"Graduate Students\";s:11:\"Technicians\";s:11:\"Technicians\";s:45:\"Associates in Agricultural Experiment Station\";s:45:\"Associates in Agricultural Experiment Station\";s:22:\"Undergraduate Students\";s:22:\"Undergraduate Students\";s:19:\"High School Interns\";s:19:\"High School Interns\";s:10:\"Volunteers\";s:10:\"Volunteers\";s:6:\"Others\";s:6:\"Others\";}s:12:\"other_choice\";s:1:\"0\";s:17:\"save_other_choice\";s:1:\"0\";s:13:\"default_value\";s:14:\"Uncatergorized\";s:6:\"layout\";s:8:\"vertical\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:2:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:0;}'),(809,199,'position','normal'),(810,199,'layout','no_box'),(811,199,'hide_on_screen',''),(812,199,'_edit_lock','1395078107:1'),(816,80,'_people_catergory','5'),(817,80,'__people_catergory','field_530b901b0ca70'),(818,80,'_wp_old_slug','paul-salessi'),(820,80,'_people_category','Project Scientists'),(821,80,'__people_category','field_530b901b0ca70'),(836,202,'_edit_last','1'),(837,202,'_edit_lock','1394645933:1'),(840,204,'_wp_attached_file','2014/02/director_header.png'),(841,204,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:700;s:6:\"height\";i:120;s:4:\"file\";s:27:\"2014/02/director_header.png\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:27:\"director_header-150x120.png\";s:5:\"width\";i:150;s:6:\"height\";i:120;s:9:\"mime-type\";s:9:\"image/png\";}s:6:\"medium\";a:4:{s:4:\"file\";s:26:\"director_header-300x51.png\";s:5:\"width\";i:300;s:6:\"height\";i:51;s:9:\"mime-type\";s:9:\"image/png\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:27:\"director_header-624x106.png\";s:5:\"width\";i:624;s:6:\"height\";i:106;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(842,202,'_thumbnail_id','204'),(844,202,'_email','bpmay@ucdavis.edu'),(845,202,'_work_address','University of California, Davis\r\nDepartment of Animal Science\r\nOne Shields Avenue, Davis, CA 95616'),(847,202,'_profile_picture','205'),(848,202,'__profile_picture','field_52fbc541cfb03'),(849,202,'_curriculum_vitae',''),(850,202,'__curriculum_vitae','field_52fbce8765544'),(851,202,'_people_category','Director'),(852,202,'__people_category','field_530b901b0ca70'),(853,202,'_education',''),(854,202,'__education','field_530503a0f8e71'),(855,202,'_research_interests',''),(856,202,'__research_interests','field_530507cebdd0e'),(857,202,'_research_projects',''),(858,202,'__research_projects','field_530507efae298'),(859,202,'_publications',''),(860,202,'__publications','field_5305081a9bade'),(861,205,'_wp_attached_file','2014/02/director_profile.jpg'),(862,205,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:300;s:6:\"height\";i:290;s:4:\"file\";s:28:\"2014/02/director_profile.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:28:\"director_profile-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(867,208,'_edit_last','1'),(868,208,'field_5310d6b20e086','a:10:{s:3:\"key\";s:19:\"field_5310d6b20e086\";s:5:\"label\";s:6:\"Alumni\";s:4:\"name\";s:7:\"_alumni\";s:4:\"type\";s:10:\"true_false\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:7:\"message\";s:33:\"Is this person an alumnus/alumna?\";s:13:\"default_value\";s:1:\"0\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:2:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:0;}'),(870,208,'position','side'),(871,208,'layout','no_box'),(872,208,'hide_on_screen',''),(873,208,'_edit_lock','1402602764:1'),(878,199,'field_5310daa13a8eb','a:11:{s:3:\"key\";s:19:\"field_5310daa13a8eb\";s:5:\"label\";s:12:\"Director Bio\";s:4:\"name\";s:13:\"_director_bio\";s:4:\"type\";s:7:\"wysiwyg\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:13:\"default_value\";s:0:\"\";s:7:\"toolbar\";s:4:\"full\";s:12:\"media_upload\";s:3:\"yes\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"1\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:19:\"field_530b901b0ca70\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:8:\"Director\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:1;}'),(886,202,'_alumni','0'),(887,202,'__alumni','field_5310d6b20e086'),(888,202,'_director_bio','Dr. May has been an Adjunct Professor and Director of the Genomic Variation Laboratory at UC Davis since 1995. He was Director of the Cornell Laboratory for Ecological and Evolutionary Genetics for the prior fourteen years. He has authored over 185 scientific publications using molecular genetic data to answer a broad suite of biological questions, within the framework of more fundamental questions such as:\r\n<ul>\r\n	<li>How do we identify populations for preservation?</li>\r\n	<li>How do we measure loss of genetic variability?</li>\r\n	<li>What remnants of native populations remain after extensive stocking with non-indigenous populations?</li>\r\n	<li>How different must two populations be for them to be maintained and managed separately?</li>\r\n</ul>\r\nCurrently he and the graduate students, post-docs, and project scientists in his program work closely with biologists from a variety of state, federal, provincial, and tribal agencies on threatened and endangered fish species, seeking to maintain population health and integrity.'),(889,202,'__director_bio','field_5310daa13a8eb'),(894,202,'_work_position','Director'),(915,217,'_edit_last','1'),(916,217,'_edit_lock','1394476437:1'),(917,217,'_wp_page_template','default'),(918,219,'_wp_attached_file','2014/02/protocols_header.png'),(919,219,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:723;s:6:\"height\";i:124;s:4:\"file\";s:28:\"2014/02/protocols_header.png\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:28:\"protocols_header-150x124.png\";s:5:\"width\";i:150;s:6:\"height\";i:124;s:9:\"mime-type\";s:9:\"image/png\";}s:6:\"medium\";a:4:{s:4:\"file\";s:27:\"protocols_header-300x51.png\";s:5:\"width\";i:300;s:6:\"height\";i:51;s:9:\"mime-type\";s:9:\"image/png\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:28:\"protocols_header-624x107.png\";s:5:\"width\";i:624;s:6:\"height\";i:107;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(920,217,'_thumbnail_id','219'),(921,220,'_edit_last','1'),(922,220,'_edit_lock','1393624661:1'),(923,220,'_wp_page_template','default'),(926,223,'_edit_last','1'),(927,223,'_edit_lock','1402345489:1'),(928,223,'_wp_page_template','default'),(929,226,'_wp_attached_file','2014/02/GVL_logo_green.jpg'),(930,226,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:177;s:6:\"height\";i:186;s:4:\"file\";s:26:\"2014/02/GVL_logo_green.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:26:\"GVL_logo_green-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(931,230,'_edit_last','1'),(932,230,'_edit_lock','1402346291:1'),(933,231,'_wp_attached_file','2014/02/gvl_map_freeway.jpg'),(934,231,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:604;s:6:\"height\";i:455;s:4:\"file\";s:27:\"2014/02/gvl_map_freeway.jpg\";s:5:\"sizes\";a:2:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:27:\"gvl_map_freeway-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}s:6:\"medium\";a:4:{s:4:\"file\";s:27:\"gvl_map_freeway-300x225.jpg\";s:5:\"width\";i:300;s:6:\"height\";i:225;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(935,232,'_wp_attached_file','2014/02/gvl_map_meyer_hall.jpg'),(936,232,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:558;s:6:\"height\";i:497;s:4:\"file\";s:30:\"2014/02/gvl_map_meyer_hall.jpg\";s:5:\"sizes\";a:2:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:30:\"gvl_map_meyer_hall-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}s:6:\"medium\";a:4:{s:4:\"file\";s:30:\"gvl_map_meyer_hall-300x267.jpg\";s:5:\"width\";i:300;s:6:\"height\";i:267;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(937,230,'_wp_page_template','default'),(938,235,'_wp_attached_file','2014/02/GVL_logo_blue.png'),(939,235,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:207;s:6:\"height\";i:217;s:4:\"file\";s:25:\"2014/02/GVL_logo_blue.png\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:25:\"GVL_logo_blue-150x150.png\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(942,239,'_wp_attached_file','2014/02/kern_community_foundation_grant.png'),(943,239,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:240;s:6:\"height\";i:181;s:4:\"file\";s:43:\"2014/02/kern_community_foundation_grant.png\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:43:\"kern_community_foundation_grant-150x150.png\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(944,246,'_wp_attached_file','2014/02/gvl_home_catfish.png'),(945,246,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:110;s:6:\"height\";i:83;s:4:\"file\";s:28:\"2014/02/gvl_home_catfish.png\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(946,247,'_wp_attached_file','2014/02/gvl_home_owl.png'),(947,247,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:110;s:6:\"height\";i:83;s:4:\"file\";s:24:\"2014/02/gvl_home_owl.png\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(948,248,'_wp_attached_file','2014/02/gvl_home_salmon.png'),(949,248,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:110;s:6:\"height\";i:83;s:4:\"file\";s:27:\"2014/02/gvl_home_salmon.png\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(950,246,'_wp_attachment_image_alt','Home page support us image of catfish'),(951,247,'_wp_attachment_image_alt','Home page projects image of owl'),(952,248,'_wp_attachment_image_alt','Home page salmon genetics meeting 2012 image of person holding a salmon'),(953,271,'_wp_attached_file','2014/02/support_header-e1402598171853.png'),(954,271,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:700;s:6:\"height\";i:400;s:4:\"file\";s:41:\"2014/02/support_header-e1402598171853.png\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:41:\"support_header-e1402598171853-150x150.png\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:9:\"image/png\";}s:6:\"medium\";a:4:{s:4:\"file\";s:41:\"support_header-e1402598171853-300x171.png\";s:5:\"width\";i:300;s:6:\"height\";i:171;s:9:\"mime-type\";s:9:\"image/png\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:41:\"support_header-e1402598171853-624x356.png\";s:5:\"width\";i:624;s:6:\"height\";i:356;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(955,14,'_thumbnail_id','271'),(956,272,'_wp_attached_file','2014/02/home_header.png'),(957,272,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:700;s:6:\"height\";i:406;s:4:\"file\";s:23:\"2014/02/home_header.png\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:23:\"home_header-150x150.png\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:9:\"image/png\";}s:6:\"medium\";a:4:{s:4:\"file\";s:23:\"home_header-300x174.png\";s:5:\"width\";i:300;s:6:\"height\";i:174;s:9:\"mime-type\";s:9:\"image/png\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:23:\"home_header-624x361.png\";s:5:\"width\";i:624;s:6:\"height\";i:361;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(959,273,'_wp_attached_file','2014/02/resources_header.png'),(960,273,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:700;s:6:\"height\";i:120;s:4:\"file\";s:28:\"2014/02/resources_header.png\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:28:\"resources_header-150x120.png\";s:5:\"width\";i:150;s:6:\"height\";i:120;s:9:\"mime-type\";s:9:\"image/png\";}s:6:\"medium\";a:4:{s:4:\"file\";s:27:\"resources_header-300x51.png\";s:5:\"width\";i:300;s:6:\"height\";i:51;s:9:\"mime-type\";s:9:\"image/png\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:28:\"resources_header-624x106.png\";s:5:\"width\";i:624;s:6:\"height\";i:106;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(961,16,'_thumbnail_id','273'),(962,274,'_wp_attached_file','2014/02/links_header.png'),(963,274,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:700;s:6:\"height\";i:120;s:4:\"file\";s:24:\"2014/02/links_header.png\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:24:\"links_header-150x120.png\";s:5:\"width\";i:150;s:6:\"height\";i:120;s:9:\"mime-type\";s:9:\"image/png\";}s:6:\"medium\";a:4:{s:4:\"file\";s:23:\"links_header-300x51.png\";s:5:\"width\";i:300;s:6:\"height\";i:51;s:9:\"mime-type\";s:9:\"image/png\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:24:\"links_header-624x106.png\";s:5:\"width\";i:624;s:6:\"height\";i:106;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(964,220,'_thumbnail_id','274'),(965,275,'_wp_attached_file','2014/02/contact_header.png'),(966,275,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:700;s:6:\"height\";i:120;s:4:\"file\";s:26:\"2014/02/contact_header.png\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:26:\"contact_header-150x120.png\";s:5:\"width\";i:150;s:6:\"height\";i:120;s:9:\"mime-type\";s:9:\"image/png\";}s:6:\"medium\";a:4:{s:4:\"file\";s:25:\"contact_header-300x51.png\";s:5:\"width\";i:300;s:6:\"height\";i:51;s:9:\"mime-type\";s:9:\"image/png\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:26:\"contact_header-624x106.png\";s:5:\"width\";i:624;s:6:\"height\";i:106;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(967,223,'_thumbnail_id','275'),(968,276,'_wp_attached_file','2014/02/directions_headr.png'),(969,276,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:723;s:6:\"height\";i:124;s:4:\"file\";s:28:\"2014/02/directions_headr.png\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:28:\"directions_headr-150x124.png\";s:5:\"width\";i:150;s:6:\"height\";i:124;s:9:\"mime-type\";s:9:\"image/png\";}s:6:\"medium\";a:4:{s:4:\"file\";s:27:\"directions_headr-300x51.png\";s:5:\"width\";i:300;s:6:\"height\";i:51;s:9:\"mime-type\";s:9:\"image/png\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:28:\"directions_headr-624x107.png\";s:5:\"width\";i:624;s:6:\"height\";i:107;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(970,230,'_thumbnail_id','276'),(971,277,'_edit_last','1'),(972,277,'_wp_page_template','page-templates/alumni_page.php'),(973,277,'_edit_lock','1402006573:1'),(974,276,'_edit_lock','1394043572:1'),(975,276,'_edit_last','1'),(976,80,'_alumni','1'),(977,80,'__alumni','field_5310d6b20e086'),(978,280,'_wp_attached_file','2014/03/alumni_header.png'),(979,280,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:700;s:6:\"height\";i:120;s:4:\"file\";s:25:\"2014/03/alumni_header.png\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:25:\"alumni_header-150x120.png\";s:5:\"width\";i:150;s:6:\"height\";i:120;s:9:\"mime-type\";s:9:\"image/png\";}s:6:\"medium\";a:4:{s:4:\"file\";s:24:\"alumni_header-300x51.png\";s:5:\"width\";i:300;s:6:\"height\";i:51;s:9:\"mime-type\";s:9:\"image/png\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:25:\"alumni_header-624x106.png\";s:5:\"width\";i:624;s:6:\"height\";i:106;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(980,277,'_thumbnail_id','280'),(981,280,'_edit_lock','1402598815:1'),(982,280,'_edit_last','1'),(983,280,'_wp_attachment_image_alt','Alumni page featured image of a fish\'s length being measured'),(984,276,'_wp_attachment_image_alt','Directions page featured image of a fish'),(1076,20,'_edit_lock','1394046803:1'),(1077,275,'_edit_lock','1394043606:1'),(1078,275,'_edit_last','1'),(1079,275,'_wp_attachment_image_alt','Contact us page featured image of two people holding a net in the water'),(1080,274,'_edit_lock','1394043644:1'),(1081,274,'_wp_attachment_image_alt','Links page featured image of tall grass'),(1082,274,'_edit_last','1'),(1083,273,'_edit_lock','1394043709:1'),(1084,273,'_wp_attachment_image_alt','Resources page featured image of a large lake during sunset'),(1085,273,'_edit_last','1'),(1086,272,'_edit_lock','1394043774:1'),(1087,272,'_wp_attachment_image_alt','Home page featured image of green marsh with rocky hills'),(1088,272,'_edit_last','1'),(1089,271,'_edit_lock','1402605136:1'),(1090,271,'_wp_attachment_image_alt','Support us page featured image of a small fish being held in a hand'),(1091,271,'_edit_last','1'),(1092,248,'_edit_lock','1394043896:1'),(1093,248,'_edit_last','1'),(1094,246,'_edit_lock','1402008604:1'),(1095,246,'_edit_last','1'),(1096,247,'_edit_lock','1402008501:1'),(1097,247,'_edit_last','1'),(1098,239,'_edit_lock','1394044076:1'),(1099,239,'_wp_attachment_image_alt','Kern community foundation grant image of two men holding a check'),(1100,239,'_edit_last','1'),(1101,235,'_edit_lock','1394044136:1'),(1102,235,'_wp_attachment_image_alt','Genomic variation laboratory blue logo image'),(1103,235,'_edit_last','1'),(1104,231,'_edit_lock','1394044239:1'),(1105,231,'_wp_attachment_image_alt','Directions page map of airport to lab image'),(1106,231,'_edit_last','1'),(1107,232,'_edit_lock','1394044281:1'),(1108,232,'_wp_attachment_image_alt','Directions page map of freeway to lab image'),(1109,232,'_edit_last','1'),(1110,226,'_edit_lock','1394044319:1'),(1111,226,'_wp_attachment_image_alt','Genomic variation laboratory green logo image'),(1112,226,'_edit_last','1'),(1113,219,'_edit_lock','1394044382:1'),(1114,219,'_wp_attachment_image_alt','Protocols page featured image of a brown fish on a board'),(1115,219,'_edit_last','1'),(1116,205,'_edit_lock','1394044424:1'),(1117,205,'_wp_attachment_image_alt','Bernie may profile image'),(1118,205,'_edit_last','1'),(1119,204,'_edit_lock','1394044475:1'),(1120,204,'_wp_attachment_image_alt','Bernie may featured image of a brown fish'),(1121,204,'_edit_last','1'),(1122,195,'_edit_lock','1394044518:1'),(1123,195,'_wp_attachment_image_alt','Partners page kern community foundation logo image'),(1124,195,'_edit_last','1'),(1125,194,'_edit_lock','1394044572:1'),(1126,194,'_wp_attachment_image_alt','Partners page us forest service logo image'),(1127,194,'_edit_last','1'),(1128,187,'_edit_lock','1394044633:1'),(1129,187,'_wp_attachment_image_alt','Partners page kootenai tribe of idaho logo image'),(1130,187,'_edit_last','1'),(1131,186,'_edit_lock','1394044756:1'),(1132,186,'_wp_attachment_image_alt','Partners page california department of water resources logo image'),(1133,186,'_edit_last','1'),(1134,185,'_edit_lock','1394044847:1'),(1135,185,'_wp_attachment_image_alt','Partners page nevada division of wildlife logo image'),(1136,185,'_edit_last','1'),(1137,184,'_edit_lock','1394044893:1'),(1138,184,'_wp_attachment_image_alt','Partners page cramer fish sciences logo image'),(1139,184,'_edit_last','1'),(1140,171,'_edit_lock','1394044983:1'),(1141,171,'_wp_attachment_image_alt','Partners page us bureau of land management logo image'),(1142,171,'_edit_last','1'),(1143,182,'_edit_lock','1394045079:1'),(1144,183,'_edit_lock','1394045112:1'),(1145,180,'_edit_lock','1394045197:1'),(1146,181,'_edit_lock','1394045263:1'),(1147,178,'_edit_lock','1394045314:1'),(1148,182,'_wp_attachment_image_alt','Partners page great lakes fishery trust logo image'),(1149,182,'_edit_last','1'),(1150,183,'_wp_attachment_image_alt','Partners page idaho power logo image'),(1151,183,'_edit_last','1'),(1152,180,'_wp_attachment_image_alt','Partners page centro de investigaciones biologicas del noroeste logo image'),(1153,180,'_edit_last','1'),(1154,181,'_wp_attachment_image_alt','Partners page us national oceanic and atmospheric administration logo image'),(1155,181,'_edit_last','1'),(1156,178,'_wp_attachment_image_alt','Partners page oregon fish and wildlife logo image'),(1157,178,'_edit_last','1'),(1158,176,'_edit_lock','1394045370:1'),(1159,175,'_edit_lock','1394045405:1'),(1160,174,'_edit_lock','1394045443:1'),(1161,173,'_edit_lock','1394045666:1'),(1162,172,'_edit_lock','1394045549:1'),(1163,160,'_edit_lock','1394045583:1'),(1164,125,'_edit_lock','1394045641:1'),(1165,176,'_wp_attachment_image_alt','Partners page sea grant logo image'),(1166,176,'_edit_last','1'),(1167,175,'_wp_attachment_image_alt','Partners page delta stewardship council logo image'),(1168,175,'_edit_last','1'),(1169,174,'_wp_attachment_image_alt','Partners page uc mexus logo image'),(1170,174,'_edit_last','1'),(1171,173,'_wp_attachment_image_alt','Partners page us department of the navy logo image'),(1172,173,'_edit_last','1'),(1173,172,'_wp_attachment_image_alt','Partners page california department of fish and game'),(1174,172,'_edit_last','1'),(1175,160,'_wp_attachment_image_alt','Partners page featured image of tall grass'),(1176,160,'_edit_last','1'),(1177,125,'_wp_attachment_image_alt','Publications page featured image of people fishing next to tall grass'),(1178,125,'_edit_last','1'),(1179,179,'_edit_lock','1394045692:1'),(1180,177,'_edit_lock','1394045734:1'),(1181,179,'_wp_attachment_image_alt','Partners page usda logo image'),(1182,179,'_edit_last','1'),(1183,177,'_wp_attachment_image_alt','Partners page pacific rim research program logo image'),(1184,177,'_edit_last','1'),(1185,124,'_edit_lock','1394047687:1'),(1186,123,'_edit_lock','1394045840:1'),(1187,119,'_edit_lock','1394045899:1'),(1188,118,'_edit_lock','1394047019:1'),(1190,116,'_edit_lock','1394046163:1'),(1191,124,'_wp_attachment_image_alt','Past projects project image of a hand holding a tadpole'),(1192,124,'_edit_last','1'),(1193,123,'_wp_attachment_image_alt','Past projects featured image of a brown fish sitting on a board'),(1194,123,'_edit_last','1'),(1195,119,'_wp_attachment_image_alt','California golden trout featured image of a fish swimming in water'),(1196,119,'_edit_last','1'),(1197,118,'_wp_attachment_image_alt','California golden trout data analysis graph image'),(1198,118,'_edit_last','1'),(1201,116,'_wp_attachment_image_alt','California golden trout gathering data image of people fishing in a river'),(1202,116,'_edit_last','1'),(1203,109,'_edit_lock','1394046467:1'),(1204,110,'_edit_lock','1394046593:1'),(1206,107,'_edit_lock','1394046642:1'),(1207,83,'_edit_lock','1394046700:1'),(1208,109,'_wp_attachment_image_alt','California golden trout map image'),(1209,109,'_edit_last','1'),(1212,110,'_wp_attachment_image_alt','California golden trout map key image'),(1213,110,'_edit_last','1'),(1214,107,'_edit_last','1'),(1215,83,'_wp_attachment_image_alt','People default featured image of moving water'),(1216,83,'_edit_last','1'),(1217,82,'_edit_last','1'),(1218,81,'_wp_attachment_image_alt','Mariah meek profile image'),(1219,81,'_edit_last','1'),(1220,20,'_wp_attachment_image_alt','Cropped home page featured image'),(1221,20,'_edit_last','1'),(1222,121,'field_53177905a02ac','a:10:{s:3:\"key\";s:19:\"field_53177905a02ac\";s:5:\"label\";s:33:\"Is this a post for past projects?\";s:4:\"name\";s:32:\"is_this_a_post_for_past_projects\";s:4:\"type\";s:10:\"true_false\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:7:\"message\";s:0:\"\";s:13:\"default_value\";s:1:\"0\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:2:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:0;}'),(1225,121,'rule','a:5:{s:5:\"param\";s:9:\"post_type\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:8:\"projects\";s:8:\"order_no\";i:0;s:8:\"group_no\";i:0;}'),(1226,106,'is_this_a_post_for_past_projects','0'),(1227,106,'_is_this_a_post_for_past_projects','field_53177905a02ac'),(1231,122,'is_this_a_post_for_past_projects','1'),(1232,122,'_is_this_a_post_for_past_projects','field_53177905a02ac'),(1233,307,'_edit_last','1'),(1234,307,'_edit_lock','1394226745:1'),(1235,307,'_thumbnail_id','219'),(1236,307,'_wp_page_template','default'),(1237,309,'_edit_last','1'),(1238,309,'_edit_lock','1394228183:1'),(1239,309,'_wp_page_template','default'),(1240,309,'_thumbnail_id','219'),(1241,312,'_edit_last','1'),(1242,312,'_edit_lock','1394227035:1'),(1243,312,'_wp_page_template','default'),(1244,312,'_thumbnail_id','219'),(1245,316,'_edit_last','1'),(1246,316,'_edit_lock','1394227069:1'),(1247,316,'_wp_page_template','default'),(1248,316,'_thumbnail_id','219'),(1249,319,'_edit_last','1'),(1250,319,'_edit_lock','1394227095:1'),(1251,319,'_thumbnail_id','219'),(1252,319,'_wp_page_template','default'),(1253,321,'_edit_last','1'),(1254,321,'_edit_lock','1394228020:1'),(1255,321,'_thumbnail_id','219'),(1256,321,'_wp_page_template','default'),(1257,326,'_edit_last','1'),(1258,326,'_edit_lock','1394226925:1'),(1259,326,'_wp_page_template','default'),(1260,326,'_thumbnail_id','219'),(1261,331,'_edit_last','1'),(1262,331,'_edit_lock','1394227319:1'),(1263,331,'_wp_page_template','default'),(1264,331,'_thumbnail_id','219'),(1265,333,'_edit_last','1'),(1266,333,'_edit_lock','1394228065:1'),(1267,333,'_thumbnail_id','219'),(1268,333,'_wp_page_template','default'),(1269,335,'_edit_last','1'),(1270,335,'_edit_lock','1394228090:1'),(1271,335,'_thumbnail_id','219'),(1272,335,'_wp_page_template','default'),(1273,337,'_edit_last','1'),(1274,337,'_edit_lock','1394228179:1'),(1275,337,'_thumbnail_id','219'),(1276,337,'_wp_page_template','default'),(1277,340,'_edit_last','1'),(1278,340,'_edit_lock','1394228174:1'),(1279,340,'_thumbnail_id','219'),(1280,340,'_wp_page_template','default'),(1281,351,'_edit_last','1'),(1282,351,'_edit_lock','1394229199:1'),(1283,351,'_wp_page_template','default'),(1284,351,'_thumbnail_id','219'),(1285,357,'_edit_last','1'),(1286,357,'_edit_lock','1394229460:1'),(1287,357,'_wp_page_template','default'),(1288,357,'_thumbnail_id','219'),(1289,361,'_edit_last','1'),(1290,361,'_edit_lock','1394473982:1'),(1291,361,'_wp_page_template','default'),(1292,361,'_thumbnail_id','219'),(1293,363,'_edit_last','1'),(1294,363,'_edit_lock','1394474009:1'),(1295,363,'_thumbnail_id','219'),(1296,363,'_wp_page_template','default'),(1297,365,'_edit_last','1'),(1298,365,'_edit_lock','1394473565:1'),(1299,365,'_thumbnail_id','219'),(1300,365,'_wp_page_template','default'),(1301,367,'_edit_last','1'),(1302,367,'_edit_lock','1394473807:1'),(1303,367,'_thumbnail_id','219'),(1304,367,'_wp_page_template','default'),(1305,374,'_edit_last','1'),(1306,374,'_edit_lock','1394476694:1'),(1307,374,'_thumbnail_id','219'),(1308,374,'_wp_page_template','default'),(1311,383,'_edit_last','1'),(1312,383,'_edit_lock','1394476319:1'),(1313,383,'_thumbnail_id','219'),(1314,383,'_wp_page_template','default'),(1315,390,'_edit_last','1'),(1316,390,'_edit_lock','1394477218:1'),(1317,391,'_wp_attached_file','2014/03/past_alumni_header.png'),(1318,391,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:700;s:6:\"height\";i:120;s:4:\"file\";s:30:\"2014/03/past_alumni_header.png\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:30:\"past_alumni_header-150x120.png\";s:5:\"width\";i:150;s:6:\"height\";i:120;s:9:\"mime-type\";s:9:\"image/png\";}s:6:\"medium\";a:4:{s:4:\"file\";s:29:\"past_alumni_header-300x51.png\";s:5:\"width\";i:300;s:6:\"height\";i:51;s:9:\"mime-type\";s:9:\"image/png\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:30:\"past_alumni_header-624x106.png\";s:5:\"width\";i:624;s:6:\"height\";i:106;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(1319,391,'_wp_attachment_image_alt','Past alumni page featured image of a wide fish being measured'),(1320,390,'_thumbnail_id','391'),(1321,390,'_wp_page_template','default'),(1322,395,'_edit_last','1'),(1323,395,'_edit_lock','1395075659:1'),(1324,396,'_wp_attached_file','2014/03/little_kern_golden_trout_graph_red_green.jpg'),(1325,396,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:680;s:6:\"height\";i:93;s:4:\"file\";s:52:\"2014/03/little_kern_golden_trout_graph_red_green.jpg\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:51:\"little_kern_golden_trout_graph_red_green-150x93.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:93;s:9:\"mime-type\";s:10:\"image/jpeg\";}s:6:\"medium\";a:4:{s:4:\"file\";s:51:\"little_kern_golden_trout_graph_red_green-300x41.jpg\";s:5:\"width\";i:300;s:6:\"height\";i:41;s:9:\"mime-type\";s:10:\"image/jpeg\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:51:\"little_kern_golden_trout_graph_red_green-624x85.jpg\";s:5:\"width\";i:624;s:6:\"height\";i:85;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(1326,397,'_wp_attached_file','2014/03/little_kern_golden_trout_map.jpg'),(1327,397,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:329;s:6:\"height\";i:397;s:4:\"file\";s:40:\"2014/03/little_kern_golden_trout_map.jpg\";s:5:\"sizes\";a:2:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:40:\"little_kern_golden_trout_map-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}s:6:\"medium\";a:4:{s:4:\"file\";s:40:\"little_kern_golden_trout_map-248x300.jpg\";s:5:\"width\";i:248;s:6:\"height\";i:300;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(1328,397,'_wp_attachment_image_alt','Little kern golden trout map image'),(1329,396,'_wp_attachment_image_alt','Little kern golden trout graph of rainbow trout introgression'),(1330,398,'_wp_attached_file','2014/03/little_kern_golden_trout_project_image.jpg'),(1331,398,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:213;s:6:\"height\";i:153;s:4:\"file\";s:50:\"2014/03/little_kern_golden_trout_project_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:50:\"little_kern_golden_trout_project_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";d:3.3999999999999999;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:7:\"DMC-TS2\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:1317727846;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";s:3:\"5.2\";s:3:\"iso\";s:2:\"80\";s:13:\"shutter_speed\";s:5:\"0.025\";s:5:\"title\";s:0:\"\";}}'),(1332,398,'_wp_attachment_image_alt','Little kern golden trout project image of woman holding a trout'),(1333,395,'_project_excerpt','Quantification of introgression level with rainbow trout through the use of SNPs and microsatellite markers'),(1334,395,'_contact_name','Molly Stephens'),(1335,395,'_contact_email','mrstephens@ucdavis.edu '),(1336,395,'_project_image','398'),(1337,395,'__project_image','field_530795285c18b'),(1338,395,'is_this_a_post_for_past_projects','0'),(1339,395,'_is_this_a_post_for_past_projects','field_53177905a02ac'),(1340,395,'_background','The Little Kern golden trout (LKGT) is one of two golden trout subspecies designated as the California State fish. Both golden trout subspecies were endemic to the Kern River watershed prior to European settlement, with the Little Kern form restricted to the Little Kern River above a natural barrier (Evans et al. 1973). Extensive stocking of non-native rainbow trout in the basin during the first half of this century led to the near extinction of the LKGT due to hybridization. Planting of non-native trout ceased in the 1950s, and the California Department of Fish and Game (CDFG) began surveys in 1965 to initiate restoration efforts. Allozyme electrophoretic analyses begun in 1976 at UC Davis ultimately identified six pure populations of GT-LK occupying only ten miles of stream out of the 100 miles of historic range (Gall and May 1997 and references therein). The Little Kern golden trout was federally listed as a Threatened species under the Endangered Species Act in 1978.\r\n\r\nRestoration efforts began in 1975 with the first rotenone treatment by the California Department of Fish and Game. In addition to chemical treatments, restoration measures have included restocking of treated waters with pure LKGT, construction of barriers to the upstream movement of non-native trout, habitat improvement of streams damaged by cattle grazing, public education, and continued monitoring of fish populations, their genetic integrity, and habitat conditions. Although restoration was believed complete in 1996, subsequent genetic analyses identified a number of streams and lakes within the system that had been stocked with one strain of LKGT that contained hybridized trout.\r\n\r\nThe purpose our research is to identify hybridized populations of GT-LK and quantify levels of introgression with rainbow trout using microsatellite and newly developed SNP markers. This information will be used in the management and conservation of the species.\r\n\r\n[caption id=\"attachment_397\" align=\"aligncenter\" width=\"329\"]<img class=\"size-full wp-image-397\" alt=\"Little kern golden trout map image\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/little_kern_golden_trout_map.jpg\" width=\"329\" height=\"397\" /> Figure 1. Distribution of Little Kern golden trout broodstock during restoration efforts. (From Gall 1997).[/caption]'),(1341,395,'__background','field_53079733f3610'),(1342,395,'_materials_and_methods','<ul>\r\n	<li>28 populations (5 reference Little Kern broodstock, 5 hatchery and wild rainbow trout)</li>\r\n	<li>18 O. <em>mykiss</em> microsatellite loci</li>\r\n	<li>10 SNP loci</li>\r\n</ul>'),(1343,395,'__materials_and_methods','field_5307975cdf5ef'),(1344,395,'_results','Comparative analysis of SNP and microsatellite data reveal similar estimates of location and levels of rainbow trout introgression (Figure 2). A manuscript detailing results of comparative analysis is in preparation (Stephens et al., <em>in prep</em>).\r\n\r\n[caption id=\"attachment_396\" align=\"aligncenter\" width=\"680\"]<img class=\"size-full wp-image-396\" alt=\"Little kern golden trout graph of rainbow trout introgression\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/little_kern_golden_trout_graph_red_green.jpg\" width=\"680\" height=\"93\" /> Figure 2. STRUCTURE Bayesian analysis of Little Kern golden trout SPN data. Populations 1-5 are Little Kern broodstock populations, populations 6-23 and 25 are LKGT populations, and 26-30 are hatchery and wild rainbow trout. Each vertical bar represents an individual fish, with individuals grouped by population number (as in Table 4) on the x-axis. Bar colors represent golden trout (red) and rainbow trout (green), with the proportion of green indicating of the level of rainbow trout introgression in a given individual.[/caption]'),(1345,395,'__results','field_53079783f8e53'),(1346,395,'_future_work',''),(1347,395,'__future_work','field_530797af2f229'),(1348,395,'_collaborators','California Department of Fish and Game (CDFG)\r\n\r\nU.S. Forest Service (USFS)\r\n\r\nFunding is provided by the CDFG and the US Fish and Wildlife Service'),(1349,395,'__collaborators','field_530797d1e08ae'),(1350,395,'_literature_cited','Evans, W.A., R.C. Smith, and M. Bell. 1973. A reconnaissance survey of the fish resources of the Little Kern River drainage, California. U.S. Dept. Agriculture Forest Ser., Region 5, and Ca. Dept. Fish Game Draft (Prelininary Report). 41 p.\r\n\r\nGall, G.E., and B. May. 1997. Trout of the Kern River Basin: A genetic analysis of Little Kern River and Golden Trout Creek populations. Report to California Department of Fish and Game, Threatened Trout Committee. September 1997. 55 pp.\r\n\r\nStephens, M.R., N.W. Clipperton, and B.P. May. Conservation genetic status of native Little Kern golden trout using SNP and microsatellite markers. (<em>in prep</em>)'),(1351,395,'__literature_cited','field_530797f0836ce'),(1352,400,'_wp_attached_file','2014/03/little_kern_header.png'),(1353,400,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:700;s:6:\"height\";i:120;s:4:\"file\";s:30:\"2014/03/little_kern_header.png\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:30:\"little_kern_header-150x120.png\";s:5:\"width\";i:150;s:6:\"height\";i:120;s:9:\"mime-type\";s:9:\"image/png\";}s:6:\"medium\";a:4:{s:4:\"file\";s:29:\"little_kern_header-300x51.png\";s:5:\"width\";i:300;s:6:\"height\";i:51;s:9:\"mime-type\";s:9:\"image/png\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:30:\"little_kern_header-624x106.png\";s:5:\"width\";i:624;s:6:\"height\";i:106;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(1354,400,'_wp_attachment_image_alt','Little kern project page featured image of a fish in water'),(1355,395,'_thumbnail_id','400'),(1364,402,'_edit_last','1'),(1365,402,'_edit_lock','1394645927:1'),(1366,402,'_thumbnail_id','83'),(1367,403,'_wp_attached_file','2014/03/Mandi_Finger_profile_image.jpg'),(1368,403,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:240;s:6:\"height\";i:181;s:4:\"file\";s:38:\"2014/03/Mandi_Finger_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:38:\"Mandi_Finger_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(1369,403,'_wp_attachment_image_alt','Mandi finger profile image'),(1370,402,'_work_position','Postdoctoral Researcher'),(1371,402,'_email','ajfinger@ucdavis.edu'),(1372,402,'_work_address','Department of Animal Science, Meyer Hall\r\nOne Shields Avenue, Davis, CA 95616'),(1373,402,'_work_phone','(530) 752-6351'),(1374,402,'_profile_picture','403'),(1375,402,'__profile_picture','field_52fbc541cfb03'),(1376,402,'_curriculum_vitae',''),(1377,402,'__curriculum_vitae','field_52fbce8765544'),(1378,402,'_alumni','1'),(1379,402,'__alumni','field_5310d6b20e086'),(1380,402,'_people_category','Project Scientists'),(1381,402,'__people_category','field_530b901b0ca70'),(1382,402,'_education','B.S., Ecology and Evolutionary Biology, UC Santa Cruz, 2003\r\n\r\nPh.D., Graduate Group in Ecology, UC Davis, 2012'),(1383,402,'__education','field_530503a0f8e71'),(1384,402,'_research_interests','I have a broad range of interests and enjoy collaborating with agency biologists and other researchers to use genetic tools to manage threatened and endangered species. Some of these species include the Owens pupfish, Paiute cutthroat trout, and the delta smelt. I am also interested in using newer technologies, such as next-generation sequencing, to find candidate loci that are under selection, such as those for disease resistance or spawn timing. Additionally, I am working with Postdoctoral Researcher Alan Hitch, Ph.D. to collect molecular data to identify and describe new species of bats collected in Indonesia, and Adjunct Faculty Josh Hull Ph.D. on a population genetics study of White-tailed Hawks and Red-shouldered Hawks.'),(1385,402,'__research_interests','field_530507cebdd0e'),(1386,402,'_research_projects','Genetic management of a hatchery population of the Delta smelt (<em>Hypomesus transpacificus</em>)\r\n\r\nDetermining parentage and mating strategies in a natural spawning experiment with Delta smelt (<em>Hypomesus transpacificus</em>)\r\n\r\nUsing next-generation sequencing techniques to inform the reintroduction of the threatened Paiute cutthroat trout (<em>Oncorhynchus clarkii seleniris</em>)\r\n\r\nConservation and landscape genetics and sex-determination of White-tailed (<em>Buteo albicaudatus</em>) and Red-shouldered Hawks (<em>Buteo lineatus</em>) in North America\r\n\r\nPhylogenetics and systematics of Indonesian bats using cytochrome-b sequence data\r\n\r\nCurrent and historical phylogeography of tui chubs (Gila bicolor) in the Walker Basin using microsatellites and mitochondrial sequence data'),(1387,402,'__research_projects','field_530507efae298'),(1388,402,'_publications','<span style=\"text-decoration: underline;\">Published</span>\r\n\r\n<strong>Finger, A. J.</strong>, Stephens, M. R., Clipperton, N. C. and May B. P. (2009) Characterization of Six Single Nucleotide Polymorphisms Distinguishing Lahontan and Paiute cutthroat trouts. <em>Molecular Ecology Resources</em> 9:759-763.\r\n\r\n<strong>Finger, A. J.</strong>, Fishes of the Tuolumne River, book chapter <em>in</em> Confluence v1.0 eds. Jeffrey Mount, Sabra Purdy. University of California press, creative commons licensing. (V. 1.0). <a title=\"ConfluenceTuolumneV1.pdf\" href=\"http://watershed.ucdavis.edu/tuolumne/resources/ConfluenceTuolumneV1.pdf\">http://watershed.ucdavis.edu/tuolumne/resources/ConfluenceTuolumneV1.pdf</a>.\r\n\r\n<strong>Finger, A. J.</strong>, Anderson, E. A., Stephens, M. R., and May, B. P. (2010) Application of a method for estimating effective population size and admixture using diagnostic SNPs: Implications for conservation of threatened Paiute cutthroat trout in Silver King Creek, California. <em>Canadian Journal of Fisheries and Aquatic Sciences</em>, 68:1369-1386.\r\n\r\n<span style=\"text-decoration: underline;\">Manuscripts in Progress</span>\r\n\r\n<strong>Finger, A. J.</strong>, Parmenter, S. P. and May, B. P. Microsatellites reveal genetic drift and population bottlenecks in refuge populations of the Owens pupfish (Cyprinodon radiosus). (submitted with minor revisions to Transactions of the American Fisheries Society)\r\n\r\nStephens, M. R., <strong>Finger, A. J.</strong>, Clipperton, N. C., and May, B.P. Conservation genetic status of native Little Kern golden trout using SNP and microsatellite markers.'),(1389,402,'__publications','field_5305081a9bade'),(1390,405,'_edit_last','1'),(1391,405,'_edit_lock','1394645913:1'),(1392,406,'_wp_attached_file','2014/03/andrea_schreier_cv_2012.pdf'),(1393,407,'_wp_attached_file','2014/03/andrea_schreier_profile_image.jpg'),(1394,407,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:300;s:6:\"height\";i:225;s:4:\"file\";s:41:\"2014/03/andrea_schreier_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:41:\"andrea_schreier_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";d:2.7999999999999998;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:8:\"DMC-FX12\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:1252917782;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";s:3:\"5.8\";s:3:\"iso\";s:3:\"100\";s:13:\"shutter_speed\";s:9:\"0.0015625\";s:5:\"title\";s:0:\"\";}}'),(1395,407,'_wp_attachment_image_alt','Andrea Schreier Profile Image'),(1396,405,'_work_position','Postdoctoral Researcher\r\nAlumna of the Graduate Group of Ecology'),(1397,405,'_email','amdrauch@ucdavis.edu'),(1398,405,'_work_address','One Shields Ave, Davis, CA  95616'),(1399,405,'_work_phone','(530) 752-0664'),(1400,405,'_profile_picture','407'),(1401,405,'__profile_picture','field_52fbc541cfb03'),(1402,405,'_curriculum_vitae','406'),(1403,405,'__curriculum_vitae','field_52fbce8765544'),(1404,405,'_alumni','1'),(1405,405,'__alumni','field_5310d6b20e086'),(1406,405,'_people_category','Project Scientists'),(1407,405,'__people_category','field_530b901b0ca70'),(1408,405,'_education','PhD, UC Davis, 2012\r\n\r\nMS, Purdue University, 2006\r\n\r\nBS, Hillsdale College, 2003'),(1409,405,'__education','field_530503a0f8e71'),(1410,405,'_research_interests','I am generally interested in using molecular techniques to understand ecological and evolutionary characteristics of fish and other organisms. I am particularly interested in the conservation, ecology, and evolution of ancient polyploid fishes such as sturgeon and paddlefishes.'),(1411,405,'__research_interests','field_530507cebdd0e'),(1412,405,'_research_projects','<strong>Inference of ploidy level in white sturgeon:</strong> Sturgeon belong to the family Acipenseridae, which is thought to be derived from an ancestor with 60 chromosomes. Extant sturgeons have 120, 250, or 360 chromosomes and there is some controversy over the evolutionary ploidy level for these groups. I followed the inheritance of eight microsatellite loci in 15 known families of white sturgeon to infer ploidy level from the mode of inheritance and gene copy number at each locus.\r\n\r\n<strong>Examination of white sturgeon population structure across the species range:</strong> Due to the highly duplicated nature of the white sturgeon genome, little work had been done to evaluate population structure until recently. With the help of agency biologists across the West Coast, I assembled a collection of tissue samples from white sturgeon inhabiting nearly all reaches of the Sacramento-San Joaquin, Columbia-Snake, and Fraser River drainages. I used microsatellite markers to examine population structure within and among drainages and used these to address critical unknowns in white sturgeon conservation and management.\r\n\r\n<strong>Genetic monitoring of Kootenai River white sturgeon conservation aquaculture program:</strong> I use neutral markers to monitor genetic diversity loss in a conservation aquaculture program for an endangered white sturgeon population suffering from decades of recruitment failure. Genetic data are also used for identification of individual broodstock and parentage analysis.\r\n\r\n<strong>Examining effectiveness of two conservation aquaculture techniques for preserving genetic diversity in Upper Columbia River white sturgeon:</strong> Traditional sturgeon conservation aquaculture involves the capture of wild, sexually mature adults, spawning these adults in a hatchery, and rearing of their offspring until they are eventually released into the natural habitat. An alternative to this model is larval capture, where fisheries biologists capture larvae that have been naturally spawned and bring them to a hatchery for rearing. I am comparing the amount of genetic diversity preserved by each technique in a conservation aquaculture program for the Upper Columbia River white sturgeon.\r\n\r\n<strong>Evaluating incidence of spontaneous autopolyploidy in white sturgeon aquaculture programs:</strong> In my dissertation research I discovered individuals of abnormal ploidy at a commercial white sturgeon caviar farm in California. It is uncertain what aspect of artificial culture is resulting in spontaneous genome duplication. Because culture techniques are similar in commercial and conservation aquaculture, it is possible that this phenomenon is occurring in conservation hatcheries for white sturgeon. In collaboration with Daphne Gille, I am working to determine the incidence of unintentional induction of abnormal ploidy on the commercial caviar farm and in the Kootenai River white sturgeon conservation aquaculture program.\r\n\r\n<strong>Estimating heritability of ovarian adiposity and caviar yield in cultured white sturgeon:</strong> Increasing the efficiency and cost-effectiveness of commercial caviar production furthers sturgeon conservation goals by providing a sustainable alternative to wild-harvested caviar. Current white sturgeon caviar production is hindered by the incidence of ovarian adiposity in farm populations. Individuals with ovarian adiposity, or fatty ovaries, produce low yields of low quality caviar. Daphne Gille and I are collaborating with an interdisciplinary team of scientists to determine the influence of genetic and environmental factors on the ovarian adiposity trait. If found to be heritable, the identification of genetic markers associated with genes controlling this trait will aid in selective breeding to reduce its incidence in farm populations.\r\n\r\n<strong>Genetic management plan for Snake River white sturgeon:</strong> In collaboration with scientists at Idaho Power Corporation, I am using information on the population structure of white sturgeon within the Columbia/Snake River system generated by my dissertation research to write a genetic management plan for the Snake River. The management plan will delineate white sturgeon management units based on genetic differentiation among regions in the River and provide recommendations for broodstock selection for conservation aquaculture.'),(1413,405,'__research_projects','field_530507efae298'),(1414,405,'_publications','<strong>Drauch Schreier, A.</strong>, B. Mahardja, and B. May. Variable patterns of population structure revealed across the range of the ancient octoploid white sturgeon, <em>Acipenser transmontanus. Submitted to Transactions of the American Fisheries Society.</em>\r\n\r\nNelson, T. C., P. Doukakis, S. T. Lindley, <strong>A. Drauch Schreier</strong>, J. E. Hightower, L. R. Hildebrand, R. E. Whitlock, and M. A. H. Webb. Modern technologies for an ancient fish: tools to inform management of migratory sturgeon stocks. <em>Submitted to PLoS ONE.</em>\r\n\r\n<strong>Drauch Schreier, A.</strong>, B. Mahardja, and B. May. 2012. Hierarchical patterns of population structure in the endangered Fraser River white sturgeon, Acipenser transmontanus, and implications for conservation. <em>In press,</em> Canadian Journal of Fisheries and Aquatic Sciences.\r\n\r\n<strong>Drauch Schreier, A.</strong>, J. Rodzen, S. Ireland, and B. May. 2011. Genetic techniques inform conservation aquaculture of the endangered Kootenai River white sturgeon, <em>Acipenser transmontanus.</em> Endangered Species Research 16:65-75.\r\n\r\n<strong>Drauch Schreier, A.</strong>, D. A. Gille, B. Mahardja, and B. May. 2011. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon, <em>Acipenser transmontanus</em>. Journal of Applied Ichthyology 27(Suppl 2): 24-33.\r\n\r\nAnders, P., <strong>A. Drauch Schreier</strong>, J. Rodzen, M. Powell, and S. Narum. 2011. A review of genetic evaluation tools for conservation and management of sturgeons: Roles, benefits, and limitations. Journal of Applied Ichthyology 27(Suppl 2): 3-11.\r\n\r\n<strong>Drauch, A. M.</strong>, B. E. Fisher, E. K. Latch, J. A. Fike, and O. E. Rhodes, Jr. 2008. Evaluation of a remnant lake sturgeon populations utility as a source for reintroductions in the Ohio River system. Conservation Genetics 9:1195-1209.\r\n\r\nBrk, K. B., <strong>A. M. Drauch</strong>, J. A. Israel, J. K. Pedroia, J. Rodzen, and B. P. May. 2008. Development of new microsatellite DNA primers for green and white sturgeon. Conservation Genetics 9:973-979.\r\n\r\nFike, J. A., <strong>A. M. Drauch</strong>, J. C. Beasley, G. Dharmarajan, and O. E. Rhodes, Jr. 2007. Development of fourteen multiplexed microsatellite loci for raccoons (Procyon lotor). Molecular Ecology Notes 7: 525-527.\r\n\r\n<strong>Drauch, A. M.</strong> and O. E. Rhodes, Jr. 2007. Genetic evaluation of the lake sturgeon reintroduction program in the Mississippi and Missouri Rivers. North American Journal of Fisheries Management 27(2): 434-442.'),(1415,405,'__publications','field_5305081a9bade'),(1416,405,'_thumbnail_id','83'),(1417,408,'_edit_last','1'),(1418,408,'_edit_lock','1395080957:1'),(1419,409,'_wp_attached_file','2014/03/melinda_baerwald_2012_cv.pdf'),(1420,410,'_wp_attached_file','2014/03/Melinda_Baerwald_profile_image.jpg'),(1421,410,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:219;s:6:\"height\";i:242;s:4:\"file\";s:42:\"2014/03/Melinda_Baerwald_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:42:\"Melinda_Baerwald_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";d:2.7000000000000002;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:21:\"Canon PowerShot S2 IS\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";s:1:\"6\";s:3:\"iso\";s:3:\"100\";s:13:\"shutter_speed\";s:17:\"0.016666666666667\";s:5:\"title\";s:0:\"\";}}'),(1422,410,'_wp_attachment_image_alt','Melinda baerwald profile image'),(1423,408,'_work_position','Assistant Project Scientist'),(1424,408,'_email','mrbaerwald@ucdavis.edu '),(1425,408,'_work_address','Department of Animal Science\r\nUniversity of California, Davis\r\nOne Shields Avenue, Davis, CA 95616'),(1426,408,'_work_phone','(530) 754-4155 '),(1427,408,'_profile_picture','410'),(1428,408,'__profile_picture','field_52fbc541cfb03'),(1429,408,'_curriculum_vitae','409'),(1430,408,'__curriculum_vitae','field_52fbce8765544'),(1431,408,'_alumni','0'),(1432,408,'__alumni','field_5310d6b20e086'),(1433,408,'_people_category','Project Scientists'),(1434,408,'__people_category','field_530b901b0ca70'),(1435,408,'_education','Ph.D., Genetics, University of California Davis\r\n\r\nBS, Biology, Indiana University'),(1436,408,'__education','field_530503a0f8e71'),(1437,408,'_research_interests','Im interested in using population genetic/genomic approaches to aid in the conservation of threatened species. Specific current areas of research interest include understanding the genetic basis of traits influenced by selection (e.g., disease resistance, run timing in Chinook salmon, smoltification in O. mykiss), linking predation patterns with environmental factors, and evaluating spatial distribution and population trends for ecosystems using environmental DNA. While my current research primarily focuses on fish species, I am not taxon-focused and enjoy working on any species with interesting biological questions that can be addressed using the tools of conservation/population genetics.'),(1438,408,'__research_interests','field_530507cebdd0e'),(1439,408,'_research_projects','<ol>\r\n	<li><strong>San Joaquin River Restoration Project (SJRRP):</strong>Provide genetic guidance and research for the SJRRP, a state and federal multi-agency led long-term project, which is restoring water flows to the upper portion of the San Joaquin River and will re-establish salmonids in this previously dewatered region.</li>\r\n	<li><strong>Genetic basis of whirling disease resistance in rainbow trout:</strong>Collaborating with the Colorado Division of Wildlife to genetically determine if stocking efforts to establish a resistant rainbow trout lineage in the Colorado River basin are proving successful.</li>\r\n	<li><strong>Population genetics of the Sacramento splittail:</strong>Project includes marker development, estimation of effective population size, taxonomic classification, and individual-based genetic assignment. My role on the project is to oversee contract administration with funding agency and mentor graduate student conducting genetic analyses.</li>\r\n	<li><strong>Fish predation patterns in the San Francisco Estuarys Delta:</strong>Collaborating with CA Dept. of Water Resources and Cramer Fish Sciences to detect prey DNA in the gut contents of predatory species. In the long-term, we hope to link predation patterns with specific environmental factors that may be influencing predation rates. To date, we have focused on the invasive Mississippi silversides predation on the threatened delta smelt but are currently creating new genetic assays to assess predation for many other prey species (e.g., longfin smelt, splittail, white and green sturgeon, Chinook salmon, rainbow trout).</li>\r\n</ol>'),(1440,408,'__research_projects','field_530507efae298'),(1441,408,'_publications','<strong>Baerwald MR</strong>, Schumer G, Schreier B, May B. 2011. TaqMan assays for the genetic identification of delta smelt (<em>Hypomesus transpacificus</em>) and wakasagi smelt (<em>Hypomesus nipponensis</em>). <em>Mol Ecol Res</em> 11: 784-785.\r\n\r\n<strong>Baerwald MR</strong>, Petersen JP, Hedrick RP, Schisler GJ, May B. 2011. A major effect quantitative trait locus for whirling disease resistance identified in rainbow trout (<em>Oncorhynchus mykiss</em>). <em>Heredity</em> 106: 920-926.\r\n\r\n<strong>Baerwald MR</strong>, Stephens MR, Bork KS, Meek MH, Tomalty K, May B. 2011. Spring-run\r\n\r\nChinook salmon genetic management plan  San Joaquin River Restoration Program. 118 pgs. Scientific genetic management plan for the CA Dept. of Fish and Game.\r\n\r\nSommer T, Reece K, Feyrer F, Brown R, <strong>Baerwald M</strong>. 2010. Splittail persistence in the Petaluma River. <em>IEP Newsletter</em> 23: 7-8.\r\n\r\nMeek MH, <strong>Baerwald MR</strong>, Wintzer AP, May B. 2009. Isolation and characterization of microsatellite loci in two non-native hydromedusae in the San Francisco Estuary: <em>Maeotias marginata</em> and <em>Moerisia</em> sp. <em>Conserv Genet Res</em> 1: 205-208.\r\n\r\nPincheira R, <strong>Baerwald M</strong>, Dunbar JD, Donner DB. 2009. Sall2 is a novel p75NTR-interacting protein that links NGF signaling to cell cycle progression and neurite outgrowth. <em>EMBO</em> J 28: 261-273.\r\n\r\nFisch K, Petersen J, <strong>Baerwald MR</strong>, Pedroia J, May B. 2009. Characterization of 24 microsatellite loci in delta smelt, <em>Hypomesus transpacificus</em>, and their cross-species amplification in two other smelt species of the Osmeridae family. <em>Mol Ecol Res</em> 9: 405-408.\r\n\r\n<strong>Baerwald MR</strong>, Feyrer FV, May B. 2008. Distribution of genetically differentiated splittail populations during the non-spawning season. <em>Trans Am Fish Soc</em> 137: 1335-1345.\r\n\r\n<strong>Baerwald MR</strong>, Welsh AB, Hedrick RP, May B. 2008. Discovery of genes implicated in whirling disease infection and resistance in rainbow trout using genome-wide expression profiling. <em>BMC Genomics</em> 9: 37.\r\n\r\nFeyrer F, Hobbs J, <strong>Baerwald M</strong>, Sommer T, Yin QZ, Clark K, May B, Bennett W. 2007. Otolith microchemistry provides information complimentary to microsatellite DNA for a migratory fish. <em>Trans Am Fish Soc</em> 136: 469-476.\r\n\r\n<strong>Baerwald M</strong>, Bien V, Feyrer F, May B. 2007. Genetic analysis reveals two distinct splittail (<em>Pogonichthys macrolepidotus</em>) populations. Conserv Genet 8: 159-167.\r\n\r\n<strong>Baerwald MR</strong>, May B. 2004. Characterization of microsatellite loci for five members of the minnow family Cyprinidae found in the Sacramento  San Joaquin Delta and its tributaries. <em>Mol Ecol Notes</em> 4: 385-390.\r\n\r\nGustin JA, Pincheira R, Mayo LD, Ozes ON, Kessler KM, <strong>Baerwald MR</strong>, Korgaonkar CK, Donner DB. 2004. Tumor necrosis factor activates CRE-binding protein through a p38 MAPK/MSK1 signaling pathway in endothelial cells. <em>Am J Physiol  Cell Physiol</em> 286: C547-C555.\r\n\r\nGuo DQ, Wu LW, Dunbar JD, Ozes ON, Mayo LD, Kessler KM, Gustin JA, <strong>Baerwald MR</strong>, Jaffe EA, Warren RS, Donner DB. 2000. Tumor necrosis factor employs a protein-tyrosine phosphatase to inhibit activation of KDR and vascular endothelial cell growth factor-induced endothelial cell proliferation. <em>J Biol Chem</em> 275: 11216-11221.\r\n\r\nWu LW, Mayo LD, Dunbar JD, Kessler KM, <strong>Baerwald MR</strong>, Jaffe EA, Wang D, Warren RS, Donner DB. 2000. Utilization of distinct signaling pathways by receptors for vascular endothelial cell growth factor and other mitogens in the induction of endothelial cell proliferation. <em>J Biol Chem</em> 275: 5096-5103.'),(1442,408,'__publications','field_5305081a9bade'),(1443,408,'_thumbnail_id','83'),(1444,412,'_wp_attached_file','2014/03/molly_stephens_2012_cv.pdf'),(1445,413,'_wp_attached_file','2014/03/molly_stephens_grassy_field_image.jpg'),(1446,413,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:470;s:6:\"height\";i:351;s:4:\"file\";s:45:\"2014/03/molly_stephens_grassy_field_image.jpg\";s:5:\"sizes\";a:2:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:45:\"molly_stephens_grassy_field_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}s:6:\"medium\";a:4:{s:4:\"file\";s:45:\"molly_stephens_grassy_field_image-300x224.jpg\";s:5:\"width\";i:300;s:6:\"height\";i:224;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(1447,414,'_wp_attached_file','2014/03/molly_stephens_profile_image.jpg'),(1448,414,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:280;s:6:\"height\";i:216;s:4:\"file\";s:40:\"2014/03/molly_stephens_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:40:\"molly_stephens_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(1449,414,'_wp_attachment_image_alt','Molly stephens profile image'),(1450,411,'_edit_last','1'),(1451,411,'_edit_lock','1395763965:1'),(1452,411,'_thumbnail_id','83'),(1453,413,'_wp_attachment_image_alt','Molly stephens grassy field image of a woman working in a field'),(1454,411,'_work_position','Assistant Project Scientist'),(1455,411,'_email','mrstephens@ucdavis.edu'),(1456,411,'_work_address','UC Davis, Dept. of Animal Science\r\nOne Shields Avenue, Davis, CA 95616'),(1457,411,'_work_phone','530.754.4155 (office)\r\n530.752.6351 (lab)'),(1458,411,'_profile_picture','414'),(1459,411,'__profile_picture','field_52fbc541cfb03'),(1460,411,'_curriculum_vitae','412'),(1461,411,'__curriculum_vitae','field_52fbce8765544'),(1462,411,'_alumni','0'),(1463,411,'__alumni','field_5310d6b20e086'),(1464,411,'_people_category','Project Scientists'),(1465,411,'__people_category','field_530b901b0ca70'),(1466,411,'_education','PhD, Ecology 2007 UC Davis\r\n\r\nMA, Biology 2001 Sonoma State University\r\n\r\nBS, Biology, 1997 Westmont College'),(1467,411,'__education','field_530503a0f8e71'),(1468,411,'_research_interests','Introgressive hybridization between native trout species and introduced hatchery rainbow trout; hybridization dynamics of secondary contact zones; species restoration and reintroduction genetics, threatened and endangered species management.'),(1469,411,'__research_interests','field_530507cebdd0e'),(1470,411,'_research_projects','<strong>Hybridization and Conservation genetics of native trout:</strong>\r\n\r\nSeveral of my research projects examine native trout species of California using SNP and microsatellite markers to detect and quantify patterns of hatchery rainbow trout introgression. Our studies on <a title=\"California Golden Trout (Oncorhynchus mykiss aguabonita)\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/california-golden-trout-oncorhynchus-mykiss-aguabonita/\">California golden trout</a> and <a title=\"Little Kern Golden Trout (Oncorhynchus mykiss whitei)\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/little-kern-golden-trout-oncorhynchus-mykiss-whitei/\">Little Kern golden trout</a> evaluate the genetic purity of within-basin and out-of-basin transplanted populations, describe introgression patterns on geographic and temporal scales, and examine the genetic consequences of previous intensive watershed-level species restoration efforts for these species. Genetic management plans are also being created to aid the translation of scientific research to adaptive management of these native species. I collaborate with Bjorn Erickson on Kern River rainbow trout projects, examining extant KRRT populations both for conservation and for potential native broodstock development. I also work on Paiute cutthroat trout with Mandi Finger and am continuing some of Rachel Simmons work on native redband trout.\r\n\r\n<strong>San Joaquin River Restoration Project:</strong>\r\n\r\nThis project, funded by California Department of Fish and Game San Joaquin River Restoration Program, supports the creation of management documents related to the planned reintroduction of spring run Chinook salmon to the San Joaquin River; this includes a Hatchery and Genetic Management Plan (an effort led by former graduate student, Karrigan Bork) covering the planned production of fish at the proposed Conservation Hatchery facility, and a Genetic Management Plan (Baerwald et al. 2010), which outlines broader genetic considerations for stock selection and monitoring of reintroduced populations. This project also supports genetic research on San Joaquin salmonid species, including genetic stock identification and genetic monitoring of Chinook salmon using SNP markers, additional SNP marker discovery efforts for run-timing identification in Chinook salmon, RNAseq gene expression analysis of thermal tolerance in Chinook salmon led by graduate student Kat Tomalty, and other developing projects.\r\n\r\n<img class=\"aligncenter size-full wp-image-413\" alt=\"Molly stephens grassy field image of a woman working in a field\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/molly_stephens_grassy_field_image.jpg\" width=\"470\" height=\"351\" />'),(1471,411,'__research_projects','field_530507efae298'),(1472,411,'_publications','Teglas, M.B., B.P. May, P.R. Crosbie, <strong>M.R. Stephens</strong>, and W.M. Boyce. 2005. Genetic structure of the tick <em>Ornithodoros coriaceus</em> (Acari: Argasidae) in California, Nevada and Oregon. <em>Journal of Medical Entomology</em> 42(3):247-253.\r\n\r\nAmanda J. Finger, Eric C. Anderson, <strong>Molly R. Stephens</strong>, Bernard P. May. 2011. Application of a method for estimating effective population size and admixture using diagnostic single nucleotide polymorphisms (SNPs): implications for conservation of threatened Paiute cutthroat trout (<em>Oncorhynchus clarkii seleniris</em>) in Silver King Creek, California. <em>Canadian Journal of Fisheries and Aquatic Sciences</em>. 68(8): 1369-1386.\r\n\r\nFinger, A.J., <strong>M.R. Stephens</strong>, N.W. Clipperton and B. May. 2009. Six diagnostic SNP markers for detecting introgression between cutthroat and rainbow trout. <em>Molecular Ecology Resources</em>. 9:759-763.\r\n\r\n<strong>Stephens, M.R.</strong>, N.W. Clipperton, and B.P. May. 2009. Subspecies-informative TaqMan probes for evaluating introgression between native golden trout and introduced rainbow trout. <em>Molecular Ecology Resources</em>. 9:339-343.\r\n\r\nSprowles, A.E., <strong>M.R. Stephens</strong>, N.W. Clipperton, and B.P. May. 2006. Fishing for SNPs: a targeted locus approach for Single Nucleotide Polymorphism discovery in rainbow trout. <em>Transactions of the American Fisheries Society.</em>135:1698-1721.\r\n\r\nCordes, J.F., <strong>M.R. Stephens</strong>, M.A. Blumberg, and B.P. May. 2006. Identifying introgressive hybridization in native populations of California golden trout based on molecular markers. <em>Transactions of the American Fisheries Society. </em>134:110-128.'),(1473,411,'__publications','field_5305081a9bade'),(1474,417,'_edit_last','1'),(1475,417,'_edit_lock','1395080904:1'),(1476,418,'_wp_attached_file','2014/03/josh_hull_header.png'),(1477,418,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:712;s:6:\"height\";i:122;s:4:\"file\";s:28:\"2014/03/josh_hull_header.png\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:28:\"josh_hull_header-150x122.png\";s:5:\"width\";i:150;s:6:\"height\";i:122;s:9:\"mime-type\";s:9:\"image/png\";}s:6:\"medium\";a:4:{s:4:\"file\";s:27:\"josh_hull_header-300x51.png\";s:5:\"width\";i:300;s:6:\"height\";i:51;s:9:\"mime-type\";s:9:\"image/png\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:28:\"josh_hull_header-624x106.png\";s:5:\"width\";i:624;s:6:\"height\";i:106;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(1478,419,'_wp_attached_file','2014/03/josh_hull_profile_image.jpg'),(1479,419,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:300;s:6:\"height\";i:200;s:4:\"file\";s:35:\"2014/03/josh_hull_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:35:\"josh_hull_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";d:4.5;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:9:\"NIKON D70\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:1108186195;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";s:2:\"40\";s:3:\"iso\";s:3:\"200\";s:13:\"shutter_speed\";s:17:\"0.033333333333333\";s:5:\"title\";s:0:\"\";}}'),(1480,419,'_wp_attachment_image_alt','Josh hull profile image'),(1481,418,'_wp_attachment_image_alt','Josh Hull Featured Image of an eagle flying'),(1482,417,'_thumbnail_id','418'),(1483,417,'_work_position','Assistant Adjunct Professor,\r\nDepartment of Animal Science, UC Davis\r\n\r\nDivision Chief, Recovery Division, \r\nSacramento Fish and Wildlife Office, US Fish and Wildlife Service'),(1484,417,'_email','jmhull@ucdavis.edu'),(1485,417,'_work_address','Ecology and Avian Science Graduate Groups'),(1487,417,'_profile_picture','419'),(1488,417,'__profile_picture','field_52fbc541cfb03'),(1489,417,'_curriculum_vitae',''),(1490,417,'__curriculum_vitae','field_52fbce8765544'),(1491,417,'_alumni','0'),(1492,417,'__alumni','field_5310d6b20e086'),(1493,417,'_people_category','Adjunct Faculty'),(1494,417,'__people_category','field_530b901b0ca70'),(1495,417,'_education','PhD, Ecology, UC Davis, 2007\r\n\r\nMA, Biology, Sonoma State University, 2002\r\n\r\nBS, Biology, Dominican University of California, 1998'),(1496,417,'__education','field_530503a0f8e71'),(1497,417,'_research_interests','Conservation genetics, avian ecology and migration, and the conservation of rare and endangered plant and animal species.'),(1498,417,'__research_interests','field_530507cebdd0e'),(1499,417,'_research_projects','Population genetics and systematics of North American merlin (<em>Falco columbarius</em>) subspecies\r\n\r\nDemographic modeling of great gray owls (<em>Strix nebulosa</em>) in California using passive genetic techniques\r\n\r\nMonitoring of great gray owl (<em>Strix nebulosa</em>) habitat using autonomous recording units\r\n\r\nMonitoring of California clapper rail (<em>Rallus longirostrus obsoletus</em>) habitat using autonomous recording units\r\n\r\nDevelopment of morphometric key to accurately sex white-tailed hawks\r\n\r\nPopulation genetics of red-shouldered (<em>Buteo lineatus</em>) hawks in eastern North America'),(1500,417,'__research_projects','field_530507efae298'),(1501,417,'_publications','<strong>J Hull</strong>, S Pitzer, A Fish, H Ernest, A Hull. <strong>2012</strong>. Differential migration patterns for five raptor species in central Coastal California. Journal of Raptor Research.\r\n\r\nBollmer J, <strong>J Hull</strong>, H Ernest, J Sarasola, P Parker. <strong>2011</strong>. Reduced adaptive and neutral variation in the Galpagos Hawk, an island endemic. BMC Evolutionary Biology, 11, 143. doi:10.1186/1471-2148-11-143.\r\n\r\n<strong>Hull J</strong>, D Mindell, S Talbot, E Kay, H Hoekstra, H Ernest. <strong>2010</strong>. Population structure and plumage polymorphism: The intraspecific evolutionary relationships of a polymorphic raptor, <em>Buteo jamaicensis harlani</em>. BMC Evolutionary Biology, 10, 224. doi:10.1186/1471-2148-10-224\r\n\r\n<strong>Hull J</strong>, A Fish, J Keane, S Mori, B Sacks, A Hull. <strong>2010</strong>. Estimation of species identification error: implications for raptor migration counts and trend estimation. Journal of Wildlife Management, 76, 1326-1334. doi: 10.2193/2009-255.\r\n\r\n<strong>Hull J</strong>, J Keane, W Savage, E Jepsen, S Godwin, J Shafer, R Gerhardt, C Stermer, H Ernest. <strong>2010</strong>. Range-wide genetic differentiation among North American great gray owls (<em>Strix nebulosa</em>) reveals a distinct lineage restricted to the Sierra Nevada, California. Molecular Phylogenetics and Evolution, 56, 212-221.\r\n\r\n<span style=\"line-height: 1.714285714; font-size: 1rem;\"><strong>Hull J</strong>, J Keane, L Tell, H Ernest. <strong>2010</strong>. West Nile virus antibody surveillance in three Sierra Nevada raptors: Spotted Owls, Great Gray Owls, and Northern Goshawks. The Condor, 112, 168-172.</span>\r\n\r\n<span style=\"line-height: 1.714285714; font-size: 1rem;\">Whiteman N, V Dosanjh, R Palma, <strong>J Hull</strong>, R Kimball, PC Snchez, P Parker. <strong>2009</strong>. Molecular and morphological divergence in a pair of bird species and their ectoparasites. Journal of Parasitology, 95, 1372-1382.</span>\r\n\r\nBrown S, <strong>J Hull</strong>, D Updike, S Fain, H Ernest. <strong>2009</strong>. Black bear population genetics in California: Signatures of population structure, competitive release and historical translocation. Journal of Mammalogy, 90, 1066-1074.\r\n\r\n<strong>Hull J</strong>, H Ernest, J Harley, A Fish, A Hull. <strong>2009</strong>. Differential migration between discrete populations of juvenile Red-tailed Hawks (<em>Buteo jamaicensis</em>). Auk, 126, 389-396.\r\n\r\n<strong>Hull J</strong>, B Stroble, C Boal, C Dykstra, A Hull, A Irish, A Fish, H Ernest. <strong>2008</strong>. Comparative phylogeography and population genetics within <em>Buteo lineatus</em> reveals evidence of distinct evolutionary lineages. Molecular Phylogenetics and Evolution, 49, 988-996.\r\n\r\n<strong>Hull J</strong>, W Savage, J Bollmer, R Kimball, N Whiteman, P Parker, and H Ernest. <strong>2008</strong>. On the origin of the Galpagos hawk: an examination of phenotypic differentiation and mitochondrial paraphyly. Biological Journal of the Linnean Society, 95, 779-789.\r\n\r\n<strong>Hull J</strong>, J Keane, L Tell, and H Ernest. <strong>2008</strong>. Development of 37 microsatellite loci for Great Gray Owls (<em>Strix nebulosa</em>) with cross amplification in Spotted Owls (<em>Strix occidentalis</em>) and Barred Owls (<em>Strix varia</em>). Conservation Genetics, 9, 1357-1361.\r\n\r\n<strong>Hull J</strong>, R Anderson, M Bradbury, J Estep, and H Ernest. <strong>2008</strong>. Population structure and genetic diversity in Swainsons Hawks (<em>Buteo swainsoni</em>): implications for conservation. Conservation Genetics, 9, 305-316.\r\n\r\n<strong>Hull J</strong>, A Hull, B Sacks, J Smith, and H Ernest. <strong>2008</strong>. Landscape characteristics influence morphological and genetic differentiation in a widespread raptor (<em>Buteo jamaicensis</em>). Molecular Ecology, 17, 810-824.\r\n\r\nPitzer S, <strong>J Hull</strong>, H Ernest, and A Hull. <strong>2008</strong>. Sex determination of three raptor species using morphology and molecular techniques. Journal of Field Ornithology, 79, 71-79.\r\n\r\n<strong>Hull J</strong>, W Savage, J Smith, N Murphy, L Cullen, A Hutchins and H Ernest. <strong>2007</strong>.Further evidence of hybridization in Buteo: Swainsons Hawks (<em>Buteo swainsoni</em>)  Red-tailed Hawks (<em>B. jamaicensis</em>) from Alberta and Utah. The Wilson Journal of Ornithology, 119, 579-584.\r\n\r\n<strong>Hull J</strong>, D Tufts, JR Topinka, B May, and HB Ernest. <strong>2007</strong>. Development of 19 microsatellite loci for Swainsons Hawks (<em>Buteo swainsoni</em>) and other <em>Buteos</em>. Molecular Ecology Notes, 7, 346-349.\r\n\r\n<strong>Hull J</strong>, A Hull, W Reisen, Y Fang, and HB Ernest. <strong>2006</strong>. Variation of West Nile Virus antibody prevalence in migrating and wintering hawks in central California. The Condor, 108, 435-439.\r\n\r\n<strong>Hull J</strong> and DJ Girman. <strong>2005</strong>. Effects of Holocene climate change on the historical demography of migrating Sharp-shinned Hawks (<em>Accipiter striatus velox</em>) in North America. Molecular Ecology, 14, 159-170.'),(1502,417,'__publications','field_5305081a9bade'),(1504,420,'_edit_last','1'),(1505,420,'_edit_lock','1394645814:1'),(1506,421,'_wp_attached_file','2014/03/CV-Scott-Brandl.doc'),(1507,422,'_wp_attached_file','2014/03/scott_brandl_profile_image.jpg'),(1508,422,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:241;s:6:\"height\";i:252;s:4:\"file\";s:38:\"2014/03/scott_brandl_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:38:\"scott_brandl_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(1509,422,'_wp_attachment_image_alt','Scott Brandl Profile Image'),(1510,420,'_thumbnail_id','83'),(1511,420,'_work_position','PhD Student\r\nGenetics Graduate Group'),(1512,420,'_email','scbrandl@ucdavis.edu'),(1513,420,'_work_address','630 Anderson Road, Davis, CA 95616'),(1514,420,'_work_phone','(907) 382-0613'),(1515,420,'_profile_picture','422'),(1516,420,'__profile_picture','field_52fbc541cfb03'),(1517,420,'_curriculum_vitae','421'),(1518,420,'__curriculum_vitae','field_52fbce8765544'),(1519,420,'_alumni','0'),(1520,420,'__alumni','field_5310d6b20e086'),(1521,420,'_people_category','Graduate Students'),(1522,420,'__people_category','field_530b901b0ca70'),(1523,420,'_education','B.S. Cell Biology, University of Minnesota Duluth, 2006'),(1524,420,'__education','field_530503a0f8e71'),(1525,420,'_research_interests','I am interested in using DNA barcodes to detect species in different environments. In practice species detection can mean a number of things, including detecting prey in a predators stomach or residual DNA in water samples, among others.'),(1526,420,'__research_interests','field_530507cebdd0e'),(1527,420,'_research_projects','Predation plays an important role in the decline of several fishes native to the Sacramento-San Joaquin river delta. We hope to shed some light on these predator-prey interactions by identifying the stomach contents of non-native predators. Traditional stomach content studies are limited by how digested the prey are inside the gut. We can bypass this issue by identifying species by their DNA. I am developing and applying genetic (qPCR) assays to identify at-risk species in predators guts. This information will be important in creating a plan for management and restoration of these species.'),(1528,420,'__research_projects','field_530507efae298'),(1529,420,'_publications',''),(1530,420,'__publications','field_5305081a9bade'),(1531,423,'_edit_last','1'),(1532,423,'_edit_lock','1394645802:1'),(1533,424,'_wp_attached_file','2014/03/Erickson_CV.doc'),(1534,425,'_wp_attached_file','2014/03/Bjorn_erickson_profile_image.jpg'),(1535,425,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:300;s:6:\"height\";i:232;s:4:\"file\";s:40:\"2014/03/Bjorn_erickson_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:40:\"Bjorn_erickson_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(1536,425,'_wp_attachment_image_alt','Bjorn erickson profile image'),(1537,423,'_thumbnail_id','83'),(1538,423,'_work_position','PhD candidate\r\nGraduate Group in Ecology'),(1539,423,'_email','pberickson@ucdavis.edu'),(1540,423,'_work_address','2403 Meyer Hall, UC Davis'),(1541,423,'_work_phone','(530) 752-6351'),(1542,423,'_profile_picture','425'),(1543,423,'__profile_picture','field_52fbc541cfb03'),(1544,423,'_curriculum_vitae','424'),(1545,423,'__curriculum_vitae','field_52fbce8765544'),(1546,423,'_alumni','0'),(1547,423,'__alumni','field_5310d6b20e086'),(1548,423,'_people_category','Graduate Students'),(1549,423,'__people_category','field_530b901b0ca70'),(1550,423,'_education',''),(1551,423,'__education','field_530503a0f8e71'),(1552,423,'_research_interests','I am interested in using genetic techniques to inform the conservation and management of imperiled species.'),(1553,423,'__research_interests','field_530507cebdd0e'),(1554,423,'_research_projects','I am working on genetic methods to detect introgression in the Kern River rainbow trout, an endemic subspecies of rainbow trout that has been negatively affected by the stocking of non-native fish. I am using both SNPs and microsatellites to measure introgression, and characterize the remaining populations of native fish. This information is essential for both management of extant wild stocks, and to guide the development and utilization of native broodstock for use in hatchery production.'),(1555,423,'__research_projects','field_530507efae298'),(1556,423,'_publications',''),(1557,423,'__publications','field_5305081a9bade'),(1558,426,'_edit_last','1'),(1559,426,'_edit_lock','1395075317:1'),(1560,427,'_wp_attached_file','2014/03/kristy_deiner_2012_cv.pdf'),(1561,428,'_wp_attached_file','2014/03/kristy_deiner_hiking_mountain_image.jpg'),(1562,428,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:334;s:6:\"height\";i:250;s:4:\"file\";s:47:\"2014/03/kristy_deiner_hiking_mountain_image.jpg\";s:5:\"sizes\";a:2:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:47:\"kristy_deiner_hiking_mountain_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}s:6:\"medium\";a:4:{s:4:\"file\";s:47:\"kristy_deiner_hiking_mountain_image-300x224.jpg\";s:5:\"width\";i:300;s:6:\"height\";i:224;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(1563,429,'_wp_attached_file','2014/03/kristy_deiner_profile_image.jpg'),(1564,429,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:194;s:6:\"height\";i:235;s:4:\"file\";s:39:\"2014/03/kristy_deiner_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:39:\"kristy_deiner_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(1565,429,'_wp_attachment_image_alt','Kristy deiner profile image'),(1566,428,'_wp_attachment_image_alt','Kristy deiner hiking a mountain image'),(1567,426,'_thumbnail_id','83'),(1568,426,'_work_position','PhD Candidate\r\nGraduate Group in Ecology'),(1569,426,'_email','kldeiner@ucdavis.edu'),(1570,426,'_work_address','University of California, Davis\r\nDepartment of Animal Science\r\nOne Shields Avenue, Davis, CA 95616'),(1572,426,'_profile_picture','429'),(1573,426,'__profile_picture','field_52fbc541cfb03'),(1574,426,'_curriculum_vitae','427'),(1575,426,'__curriculum_vitae','field_52fbce8765544'),(1576,426,'_alumni','1'),(1577,426,'__alumni','field_5310d6b20e086'),(1578,426,'_people_category','Graduate Students'),(1579,426,'__people_category','field_530b901b0ca70'),(1580,426,'_education','BS Biology, Sonoma State University, 1997-2001\r\n\r\nMS Biology, Sonoma State University 2002-2004\r\n\r\nPhD UC Davis 2007-present'),(1581,426,'__education','field_530503a0f8e71'),(1582,426,'_research_interests','I am interested in investigating ecological and evolutionary mechanisms that play a role in generating and maintaining biodiversity. Ultimately, I would like to work as a museum curator and eventually be involved with a non-profit organization to develop a network of stakeholders and foster scientific communication in an international setting focused on alpine ecosystem conservation.\r\n\r\n<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/kristy_deiner_hiking_mountain_image.jpg\"><img class=\"aligncenter size-full wp-image-428\" alt=\"Kristy deiner hiking a mountain image\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/kristy_deiner_hiking_mountain_image.jpg\" width=\"334\" height=\"250\" /></a>'),(1583,426,'__research_interests','field_530507cebdd0e'),(1584,426,'_research_projects','<ol>\r\n	<li><strong>Increasing resolution of species list using barcodes-</strong> investigating how taxonomic resolution of a community can be increased by barcode identification. Collaborators: Roland Knapp, Dave Herbst, Daniel Bioano and Bernie May</li>\r\n	<li><strong>Trophic filters in community phylogenetics-</strong> investigating the effect of trout presence on community phylogenetic diversity of aquatic midges. Collaborators: Roland Knapp, Bruce Hammock, Monika Winder, Arjun Sivasundar, Oliver Heiri and Bernie May</li>\r\n	<li><strong>Cryptic species and phylogeography of alpine fairy shrimp in the Sierra Nevada Mountain Range-</strong> investigating species distributions and population genetic structure of alpine fairy shrimp (Branchinecta spp.) Collaborators: Roland Knapp, Christopher Rodgers, and Bernie May</li>\r\n	<li><strong>Conservation genetics of endangered vernal pool fairy shrimp-</strong> developing molecular markers and surveying population genetic structure for species recovery planning. Collaborators: Josh Hull and Bernie May</li>\r\n</ol>'),(1585,426,'__research_projects','field_530507efae298'),(1586,426,'_publications','<a title=\"Kristy Deiner Google Scholar Profile\" href=\"http://scholar.google.com/citations?user=KdbiEfsAAAAJ\" target=\"_blank\"><strong>Google Scholar Profile</strong></a>\r\n\r\n<strong>Deiner, K.</strong>, Lemmon, A.R., Mack A., Fleischer R.C., Dumbacher, J.P. (2011) A passerine birds evolution corroborates the geologic history of the island of New Guinea. PLoS One 6(5): e19479.\r\n\r\nE. P. Dahlhoff, S.L. Fearnley, D. A. Bruce, A. G. Gibbs, R. Stoneking, D.M. McMillan, <strong>K. Deiner</strong>, J. T. Smiley, N. E. Rank (2008) Effects of temperature on physiology and reproductive success of a montane leaf beetle: implications for persistence of native populations enduring climate change. Physiological and Biochemical Zoology 81(6):718732. Cited 15 times.\r\n\r\nDumbacher, J.P., <strong>K. Deiner</strong>, L. Thompson, R.C. Fleischer (2008) Phylogeny of the avian genus Pitohui and the evolution of toxicity in birds. Molecular phylogenetics and Evolution 49:774781. Cited 13 times.\r\n\r\n<strong>Deiner, K.</strong>, Garza, J. C., Coey, R., Girman, D. J (2007) The effect of landscape features on the genetic structure of rainbow trout (Oncorhynchus mykiss) in the Russian River Watershed. Conservation Genetics 8(2): 437-454. Cited 26 times.'),(1587,426,'__publications','field_5305081a9bade'),(1588,430,'_edit_last','1'),(1589,430,'_edit_lock','1394646689:1'),(1590,431,'_wp_attached_file','2014/03/Alex-Van-Dam-CV-i-24-12.doc'),(1591,432,'_wp_attached_file','2014/03/alex_van_dam_profile_image.jpg'),(1592,432,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:300;s:6:\"height\";i:225;s:4:\"file\";s:38:\"2014/03/alex_van_dam_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:38:\"alex_van_dam_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(1594,434,'_wp_attached_file','2014/03/alexander_gaos_profile_image.jpg'),(1595,434,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:257;s:6:\"height\";i:241;s:4:\"file\";s:40:\"2014/03/alexander_gaos_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:40:\"alexander_gaos_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(1596,435,'_wp_attached_file','2014/03/joe_medley_profile_image.jpg'),(1597,435,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:190;s:6:\"height\";i:284;s:4:\"file\";s:36:\"2014/03/joe_medley_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:36:\"joe_medley_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";d:4.5;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:10:\"NIKON D200\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:1187123157;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";s:2:\"20\";s:3:\"iso\";s:3:\"640\";s:13:\"shutter_speed\";s:17:\"0.016666666666667\";s:5:\"title\";s:0:\"\";}}'),(1598,436,'_wp_attached_file','2014/03/katharine_tomalty_profile_image.jpg'),(1599,436,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:300;s:6:\"height\";i:234;s:4:\"file\";s:43:\"2014/03/katharine_tomalty_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:43:\"katharine_tomalty_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(1600,436,'_wp_attachment_image_alt','Katharine tomalty profile image'),(1601,430,'_thumbnail_id','83'),(1603,430,'_email','kmtomalty@ucdavis.edu'),(1604,430,'_work_address','2403 Meyer Hall One Shields Ave, Davis, CA  95616'),(1605,430,'_work_phone','(530) 752- 6351'),(1606,430,'_profile_picture','436'),(1607,430,'__profile_picture','field_52fbc541cfb03'),(1608,430,'_curriculum_vitae',''),(1609,430,'__curriculum_vitae','field_52fbce8765544'),(1610,430,'_alumni','0'),(1611,430,'__alumni','field_5310d6b20e086'),(1612,430,'_people_category','Graduate Students'),(1613,430,'__people_category','field_530b901b0ca70'),(1614,430,'_education','Masters, Biological Sciences, Stanford University, 2006\r\n\r\nBS, Biological Sciences, Stanford University, 2006'),(1615,430,'__education','field_530503a0f8e71'),(1616,430,'_research_interests','I am interested in using genetic tools to address questions in conservation biology and ecology. I am particularly interested in how disease and genetics interact to impact free-ranging wildlife populations.'),(1617,430,'__research_interests','field_530507cebdd0e'),(1618,430,'_research_projects','Transcriptional response to acute thermal stress in juvenile Chinook salmon\r\n\r\nThis project uses RNAseq to investigate how short periods of thermal stress may affect outmigrating juvenile Chinook salmon on a cellular level.'),(1619,430,'__research_projects','field_530507efae298'),(1620,430,'_publications','Prakash S, McLendon HM, Dubreuil CI, Ghose A, Hwa J, Dennehy KA, <strong>Tomalty KM</strong>, Clark KL, Van Vactor D, Clandinin TR. 2009. Complex interactions amongst N-cadherin, DLAR, and Liprin-alpha regulate Drosophila photoreceptor axon targeting. Developmental Biology. 336(1):10-9.\r\n\r\nMast JD, <strong>Tomalty KM</strong>, Vogel H, Clandinin TR. 2008. Reactive oxygen species act remotely to cause synapse loss in a Drosophila model of developmental mitochondrial encephalopathy. Development. 135(15):2669-79.\r\n\r\nLogan CA, Elizabeth SA, Haupt AJ, <strong>Tomalty KM</strong>, and Palumbi SR. An impediment to consumer choice: Overfished species are sold as Pacific red snapper. 2008. Biological Conservation. 141(6):1591-99.'),(1621,430,'__publications','field_5305081a9bade'),(1622,437,'_edit_last','1'),(1623,437,'field_5320970428360','a:12:{s:3:\"key\";s:19:\"field_5320970428360\";s:5:\"label\";s:8:\"Position\";s:4:\"name\";s:14:\"_work_position\";s:4:\"type\";s:8:\"textarea\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:13:\"default_value\";s:0:\"\";s:11:\"placeholder\";s:0:\"\";s:9:\"maxlength\";s:0:\"\";s:10:\"formatting\";s:2:\"br\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:0:\"\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:0;}'),(1625,437,'position','normal'),(1626,437,'layout','default'),(1627,437,'hide_on_screen',''),(1628,437,'_edit_lock','1394645117:1'),(1631,430,'_work_position','Graduate Student and Veterinary Student\r\nGenetics Graduate Group\r\nUC Davis School of Veterinary Medicine2'),(1632,430,'__work_position','field_5320970428360'),(1633,437,'field_532097d86af9d','a:12:{s:3:\"key\";s:19:\"field_532097d86af9d\";s:5:\"label\";s:6:\"E-Mail\";s:4:\"name\";s:6:\"_email\";s:4:\"type\";s:8:\"textarea\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:13:\"default_value\";s:0:\"\";s:11:\"placeholder\";s:0:\"\";s:9:\"maxlength\";s:0:\"\";s:10:\"formatting\";s:2:\"br\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:2:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:1;}'),(1635,437,'field_53209806c81da','a:12:{s:3:\"key\";s:19:\"field_53209806c81da\";s:5:\"label\";s:12:\"Work Address\";s:4:\"name\";s:13:\"_work_address\";s:4:\"type\";s:8:\"textarea\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:13:\"default_value\";s:0:\"\";s:11:\"placeholder\";s:0:\"\";s:9:\"maxlength\";s:0:\"\";s:10:\"formatting\";s:2:\"br\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:0:\"\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:2;}'),(1636,437,'field_5320982dc81db','a:12:{s:3:\"key\";s:19:\"field_5320982dc81db\";s:5:\"label\";s:12:\"Phone Number\";s:4:\"name\";s:11:\"_work_phone\";s:4:\"type\";s:8:\"textarea\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:13:\"default_value\";s:0:\"\";s:11:\"placeholder\";s:0:\"\";s:9:\"maxlength\";s:0:\"\";s:10:\"formatting\";s:2:\"br\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:0:\"\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:3;}'),(1638,437,'rule','a:5:{s:5:\"param\";s:9:\"post_type\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:6:\"people\";s:8:\"order_no\";i:0;s:8:\"group_no\";i:0;}'),(1639,430,'__email','field_532097d86af9d'),(1640,430,'__work_address','field_53209806c81da'),(1641,430,'__work_phone','field_5320982dc81db'),(1642,202,'__work_position','field_5320970428360'),(1643,202,'__email','field_532097d86af9d'),(1644,202,'__work_address','field_53209806c81da'),(1645,202,'_work_phone',''),(1646,202,'__work_phone','field_5320982dc81db'),(1647,411,'__work_position','field_5320970428360'),(1648,411,'__email','field_532097d86af9d'),(1649,411,'__work_address','field_53209806c81da'),(1650,411,'__work_phone','field_5320982dc81db'),(1651,438,'_edit_last','1'),(1652,438,'field_532099cee4ec6','a:12:{s:3:\"key\";s:19:\"field_532099cee4ec6\";s:5:\"label\";s:32:\"Brief description of the project\";s:4:\"name\";s:16:\"_project_excerpt\";s:4:\"type\";s:8:\"textarea\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:13:\"default_value\";s:0:\"\";s:11:\"placeholder\";s:0:\"\";s:9:\"maxlength\";s:0:\"\";s:10:\"formatting\";s:2:\"br\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:0:\"\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:0;}'),(1654,438,'position','normal'),(1655,438,'layout','default'),(1656,438,'hide_on_screen',''),(1657,438,'_edit_lock','1394645505:1'),(1658,438,'rule','a:5:{s:5:\"param\";s:9:\"post_type\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:8:\"projects\";s:8:\"order_no\";i:0;s:8:\"group_no\";i:0;}'),(1659,439,'_edit_last','1'),(1660,439,'field_53209a3637a8e','a:14:{s:3:\"key\";s:19:\"field_53209a3637a8e\";s:5:\"label\";s:12:\"Contact Name\";s:4:\"name\";s:13:\"_contact_name\";s:4:\"type\";s:4:\"text\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:13:\"default_value\";s:0:\"\";s:11:\"placeholder\";s:0:\"\";s:7:\"prepend\";s:0:\"\";s:6:\"append\";s:0:\"\";s:10:\"formatting\";s:4:\"html\";s:9:\"maxlength\";s:0:\"\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:0:\"\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:0;}'),(1661,439,'field_53209a5537a8f','a:14:{s:3:\"key\";s:19:\"field_53209a5537a8f\";s:5:\"label\";s:14:\"Contact E-Mail\";s:4:\"name\";s:14:\"_contact_email\";s:4:\"type\";s:4:\"text\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"0\";s:13:\"default_value\";s:0:\"\";s:11:\"placeholder\";s:0:\"\";s:7:\"prepend\";s:0:\"\";s:6:\"append\";s:0:\"\";s:10:\"formatting\";s:4:\"html\";s:9:\"maxlength\";s:0:\"\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:0:\"\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:1;}'),(1663,439,'position','normal'),(1664,439,'layout','default'),(1665,439,'hide_on_screen',''),(1666,439,'_edit_lock','1394645635:1'),(1667,439,'rule','a:5:{s:5:\"param\";s:9:\"post_type\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:8:\"projects\";s:8:\"order_no\";i:0;s:8:\"group_no\";i:0;}'),(1668,426,'__work_position','field_5320970428360'),(1669,426,'__email','field_532097d86af9d'),(1670,426,'__work_address','field_53209806c81da'),(1671,426,'_work_phone',''),(1672,426,'__work_phone','field_5320982dc81db'),(1673,423,'__work_position','field_5320970428360'),(1674,423,'__email','field_532097d86af9d'),(1675,423,'__work_address','field_53209806c81da'),(1676,423,'__work_phone','field_5320982dc81db'),(1677,420,'__work_position','field_5320970428360'),(1678,420,'__email','field_532097d86af9d'),(1679,420,'__work_address','field_53209806c81da'),(1680,420,'__work_phone','field_5320982dc81db'),(1681,417,'__work_position','field_5320970428360'),(1682,417,'__email','field_532097d86af9d'),(1683,417,'__work_address','field_53209806c81da'),(1684,417,'_work_phone',''),(1685,417,'__work_phone','field_5320982dc81db'),(1686,408,'__work_position','field_5320970428360'),(1687,408,'__email','field_532097d86af9d'),(1688,408,'__work_address','field_53209806c81da'),(1689,408,'__work_phone','field_5320982dc81db'),(1690,405,'__work_position','field_5320970428360'),(1691,405,'__email','field_532097d86af9d'),(1692,405,'__work_address','field_53209806c81da'),(1693,405,'__work_phone','field_5320982dc81db'),(1694,402,'__work_position','field_5320970428360'),(1695,402,'__email','field_532097d86af9d'),(1696,402,'__work_address','field_53209806c81da'),(1697,402,'__work_phone','field_5320982dc81db'),(1698,80,'__work_position','field_5320970428360'),(1699,80,'__email','field_532097d86af9d'),(1700,80,'__work_address','field_53209806c81da'),(1701,80,'__work_phone','field_5320982dc81db'),(1702,440,'_edit_last','1'),(1703,440,'_edit_lock','1394646671:1'),(1704,432,'_wp_attachment_image_alt','Alex van dam profile image'),(1705,440,'_thumbnail_id','83'),(1706,440,'_profile_picture','432'),(1707,440,'__profile_picture','field_52fbc541cfb03'),(1708,440,'_curriculum_vitae','431'),(1709,440,'__curriculum_vitae','field_52fbce8765544'),(1710,440,'_alumni','0'),(1711,440,'__alumni','field_5310d6b20e086'),(1712,440,'_work_position','PhD Candidate\r\nEntomology Graduate Group\r\n'),(1713,440,'__work_position','field_5320970428360'),(1714,440,'_email','vandama01@gmail.com'),(1715,440,'__email','field_532097d86af9d'),(1716,440,'_work_address','Department of Animal Science\r\nUniversity of California Davis\r\nOne Shields Avenue, Davis, CA 95616'),(1717,440,'__work_address','field_53209806c81da'),(1718,440,'_work_phone','(530) 752-6351'),(1719,440,'__work_phone','field_5320982dc81db'),(1720,440,'_people_category','Graduate Students'),(1721,440,'__people_category','field_530b901b0ca70'),(1722,440,'_education','M.S. Entomology UC Riverside 2007\r\n\r\nB.S. Entomology UC Riverside 2004'),(1723,440,'__education','field_530503a0f8e71'),(1724,440,'_research_interests','Domestication process in phytophagous insects and host-plant switching in their wild relatives.\r\n\r\nUsing molecular markers to delineate the geographic origins of the cochineal insect (<em>Dactylopius coccus</em>) that has been used as a brilliant red dye since pre-Columbian times in Peru and Mexico.\r\n\r\nInvestigating genetic evidence of domestication in <em>D.coccus.</em>\r\n\r\nMolecular Phylogenetics of <em>Dactylopius spp</em>. identifying host-plant switching and imbedded lineages within recognized morphological species.'),(1725,440,'__research_interests','field_530507cebdd0e'),(1726,440,'_research_projects',''),(1727,440,'__research_projects','field_530507efae298'),(1728,440,'_publications','<strong>Van Dam, A.</strong>, May B.P. 2012. A new species of <em>Dactylopius</em> (Hemiptera: Dactylopiidae) from the Chihuahuan Desert. <em>ZOOTAXA</em>. Currently in Review by Editor.\r\n\r\n<strong>Van Dam, A.</strong>, Householder J.E., Lubinsky P. 2010. <em>Vanilla bicolor</em> (Orchidaceae) from the Peruvian Amazon: auto-fertilization in <em>Vanilla</em> and notes on floral phenology. <em>Genetic Resources and Crop Evolution.</em> 57:473-480.\r\n\r\nWalton, W.E., <strong>Van Dam A.R.</strong>, Popko D.A. 2009. Ovipositional respones of two <em>Culex</em> (Diptera: Culicidae) species to larvivorous fish. <em>Journal of Medical Entomology</em>. 46(6):1338-1343.\r\n\r\n<strong>Van Dam, A.</strong>, Walton W. E. 2008. The effect of predatory fish exudates on the ovipositioning behavior of three mosquito species: <em>Culex quinquefasciatus</em>, <em>Aedes aegypti</em>, and <em>Culex tarsalis</em>. <em>Medical and Veterinary Entomology</em>. 22: 399-404.\r\n\r\n<strong>Van Dam, A.R.</strong>, Van Dam M.H. 2008. Impact of off-road vehicle use on dune endemic Coleoptera. <em>Annals of the Entomological Society of America</em>.101(2):411-417 <em>Annals of the ESA</em> March issue<strong> [COVER ARTICLE]</strong>.\r\n\r\n<strong>Van Dam, A.</strong>, Walton W. E. 2007. Comparison of mosquito control provided by the arroyo chub (<em>Gila orcutti</em>) and the mosquitofish (<em>Gambusia affinis</em>). <em>American Journal of Mosquito Control</em>. 23(4): 430-441.\r\n\r\n<strong>Van Dam, A.</strong>, Van Dam M. 2007. Cacti and succulents of the Socorro sand dunes. <em>The Cactus and Succulent Journal</em>. 79(3): 131-134.\r\n\r\n<strong>Van Dam, A.R.</strong>, Van Dam M.H., Heraty J.M. 2007. A comparison of pyrethrum fogging and screen-sweep netting of Hymenoptera in Southern California chaparral. <em>Journal of Hymenoptera Research</em>. 16(1): 192-205.\r\n\r\nLubinsky, P., Van Dam M.H., <strong>Van Dam A.R.</strong> 2006. Pollination of <em>Vanilla</em> and evolution in Orchidaceae. <em>Lindleyana</em> 75(12): 926-929.\r\n\r\nVan Dam, M.H., <strong>Van Dam A.R.</strong> 2006. Description of the larvae of <em>Pseudocotalpa sonorica</em> Hardy (Scarabaeidae: Rutelinae: Rutelini) with notes on life history. <em>Coleopterists Bulletin</em>, 60(1): 31-35.\r\n\r\nVan Dam, M.H., <strong>Van Dam A.R.</strong>, Wilcox M.D. 2006. Description of the third-instar larva and adult male of <em>Megasoma sleeperi</em> Hardy (Scarabaeidae: Dynastinae). <em>Coleopterists Bulletin</em> 60(1): 59-67.'),(1729,440,'__publications','field_5305081a9bade'),(1730,441,'_edit_last','1'),(1731,441,'_edit_lock','1394647001:1'),(1732,434,'_wp_attachment_image_alt','Alexander gaos profile image'),(1733,442,'_wp_attached_file','2014/03/alexander_gaos_2012_cv.pdf'),(1734,441,'_thumbnail_id','83'),(1735,441,'_profile_picture','434'),(1736,441,'__profile_picture','field_52fbc541cfb03'),(1737,441,'_curriculum_vitae','442'),(1738,441,'__curriculum_vitae','field_52fbce8765544'),(1739,441,'_alumni','0'),(1740,441,'__alumni','field_5310d6b20e086'),(1741,441,'_work_position','PhD Student\r\nExecutive Director of Eastern Pacific Hawksbill Initiative\r\nGraduate Group in Ecology - Joint Doctoral Program with San Diego State University'),(1742,441,'__work_position','field_5320970428360'),(1743,441,'_email','gaos@hawksbill.org'),(1744,441,'__email','field_532097d86af9d'),(1745,441,'_work_address',''),(1746,441,'__work_address','field_53209806c81da'),(1747,441,'_work_phone','(619) 818-0041'),(1748,441,'__work_phone','field_5320982dc81db'),(1749,441,'_people_category','Graduate Students'),(1750,441,'__people_category','field_530b901b0ca70'),(1751,441,'_education','BA Environmental Studies - California State University, East Bay - 2001\r\n\r\nMSc in Ecology - San Diego State University - 2011'),(1752,441,'__education','field_530503a0f8e71'),(1753,441,'_research_interests','Spatial ecology of hawksbill turtles\r\n\r\nGenetic stock structure and phylogeography of hawksbill turtles'),(1754,441,'__research_interests','field_530507cebdd0e'),(1755,441,'_research_projects',''),(1756,441,'__research_projects','field_530507efae298'),(1757,441,'_publications','<strong>Gaos, A.R.</strong>, Lewison, R.L., Liles, M., Nichols, W.J., Baquero, A., Hasbn, C.R., Urteaga, J. &amp; J.A. Seminoff. In Press. Spatial Ecology of Critically Endangered Hawksbill Turtles: Implications for Conservation and Management. Marine Ecology Progress Series.\r\n\r\n<strong>Gaos, A.R.</strong> and I. Yaez. 2012. Saving the Eastern Pacific Hawksbill from Extinction: Last Chance or Chance Lost. In: Seminoff, J.A. &amp; Wallace, B.P. (Eds.). Sea Turtles of the Eastern Pacific Ocean. University of Arizona Press.\r\n\r\n<strong>Gaos, A.R.</strong>, Lewison, R.L., Liles, M., Nichols, W.J., Baquero, A., Hasbn, C.R., Vasquez, M.J., Urteaga, J. &amp; J.A. Seminoff. 2011. Shifting the life-history paradigm: discovery of novel habitat use by hawksbill turtles. Biology Letters, 14 DOI:10.1098/rsbl.2011.0603\r\n\r\nLemons, G.E., Lewison, R., Komoroske, L., <strong>Gaos, A.R.</strong>, Lai, C., and J.A. Seminoff. 2011. 2011. Trophic Ecology of green sea turtles in a highly urbanized bay: insights from stable isotopes and mixing models. Journal of experimental marine biology and ecology.\r\n\r\n<strong>Gaos, A.R.</strong>, Abreu, A., Alfaro, J.A., Amorocho, D., Arauz, R., Baquero, A., Briseo, R., Chacn, D., Dueas, C., Hasbn, C., Liles, M., Mariona, G., Muccio, C., Muoz, J.P., Nichols, W.J., Seminoff, J.A., Vsquez, M., Urteaga, J., Wallace, B., Yaez, I.L., and P. Zrate. 2010. Signs of Hope in the Eastern Pacific: International Collaboration Reveals Encouraging Status for Severely Depleted Population of Hawksbill Turtles. Oryx, 1-7 DOI:10.1017/S0030605310000773\r\n\r\nAlfaro-Shigueto, J.A., Mangel, J.C., Caceres, C., Seminoff, J.A.,<strong> Gaos. A.</strong>, and I. Yaez. 2010. Hawksbill turtles in Peruvian coastal fisheries. Marine Turtle Newsletter. 129:19\r\n\r\nPeckham, S.H., Maldonado-Diaz, D., Koch, V., Mancini, A., <strong>Gaos, A.</strong>, Tinker, M.T. and W.J. Nichols. 2008. High mortality of loggerhead turtles due to bycatch, human consumption and strandings at Baja California Sur, Mexico, 2003 to 2007. Endangered Species Research. 5: 171-183\r\n\r\n<strong>Gaos, A.R.</strong>, Arauz, R. and I. Yaez. 2006. Hawksbill Turtles on the Pacific Coast of Costa Rica. Marine Turtle Newsletter. 112:14'),(1758,441,'__publications','field_5305081a9bade'),(1759,443,'_edit_last','1'),(1760,443,'_edit_lock','1394647179:1'),(1761,435,'_wp_attachment_image_alt','Joe medley profile image'),(1762,443,'_thumbnail_id','83'),(1763,443,'_profile_picture','435'),(1764,443,'__profile_picture','field_52fbc541cfb03'),(1765,443,'_curriculum_vitae',''),(1766,443,'__curriculum_vitae','field_52fbce8765544'),(1767,443,'_alumni','0'),(1768,443,'__alumni','field_5310d6b20e086'),(1769,443,'_work_position','PhD Candidate\r\nEcology Graduate Group\r\nMajor Adviser: Dr. Josh Hull\r\n'),(1770,443,'__work_position','field_5320970428360'),(1771,443,'_email','jrmedley@ucdavis.edu'),(1772,443,'__email','field_532097d86af9d'),(1773,443,'_work_address',''),(1774,443,'__work_address','field_53209806c81da'),(1775,443,'_work_phone',''),(1776,443,'__work_phone','field_5320982dc81db'),(1777,443,'_people_category','Graduate Students'),(1778,443,'__people_category','field_530b901b0ca70'),(1779,443,'_education','B.S. Cell and Developmental Biology, 2005, University of California at Santa Barbara\r\n\r\nM.S. Avian Science, Expected Spring 2012, University of California at Davis'),(1780,443,'__education','field_530503a0f8e71'),(1781,443,'_research_interests','Conservation and management of birds\r\n\r\nBioacoustics and soundscape ecology\r\n\r\nDevelopment and assessment of non-invasive survey methodologies'),(1782,443,'__research_interests','field_530507cebdd0e'),(1783,443,'_research_projects','Occupancy and Demographic Monitoring of California Great Gray Owls (<em>Strix nebulosa</em>) in the Central Sierra Nevada.\r\n\r\nDevelopment of scientifically based survey protocol for Great Gray Owls\r\n\r\nMolecular genetic relationships among North American Merlin (<em>Falco columbarius</em>) subspecies\r\n\r\nBioacoustic monitoring of California Clapper Rails (<em>Rallus longirostris obsoletus</em>)'),(1784,443,'__research_projects','field_530507efae298'),(1785,443,'_publications',''),(1786,443,'__publications','field_5305081a9bade'),(1787,444,'_edit_last','1'),(1788,444,'_edit_lock','1394822616:1'),(1789,445,'_wp_attached_file','2014/03/alisha_goodbla_2012_cv.pdf'),(1790,446,'_wp_attached_file','2014/03/alisha_goodbla_profile_image.jpg'),(1791,446,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:225;s:6:\"height\";i:300;s:4:\"file\";s:40:\"2014/03/alisha_goodbla_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:40:\"alisha_goodbla_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";d:2.7999999999999998;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:22:\"Canon PowerShot SD1000\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:1329899636;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";s:3:\"5.8\";s:3:\"iso\";s:3:\"200\";s:13:\"shutter_speed\";s:17:\"0.016666666666667\";s:5:\"title\";s:0:\"\";}}'),(1792,447,'_wp_attached_file','2014/03/antonia_wong_profile_image.jpg'),(1793,447,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:260;s:6:\"height\";i:250;s:4:\"file\";s:38:\"2014/03/antonia_wong_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:38:\"antonia_wong_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(1794,448,'_wp_attached_file','2014/03/emily_ringelman_profile_image.jpg'),(1795,448,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:191;s:6:\"height\";i:285;s:4:\"file\";s:41:\"2014/03/emily_ringelman_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:41:\"emily_ringelman_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(1796,446,'_wp_attachment_image_alt','Alisha goodbla profile image'),(1797,448,'_wp_attachment_image_alt','Emily ringelman profile image'),(1798,447,'_wp_attachment_image_alt','Antonia wong profile image'),(1799,444,'_thumbnail_id','83'),(1800,444,'_profile_picture','446'),(1801,444,'__profile_picture','field_52fbc541cfb03'),(1802,444,'_curriculum_vitae','445'),(1803,444,'__curriculum_vitae','field_52fbce8765544'),(1804,444,'_alumni','0'),(1805,444,'__alumni','field_5310d6b20e086'),(1806,444,'_work_position',' Lab Technician'),(1807,444,'__work_position','field_5320970428360'),(1808,444,'_email','amgoodbla@ucdavis.edu'),(1809,444,'__email','field_532097d86af9d'),(1810,444,'_work_address','Department of Animal Science, Meyer Hall\r\nOne Shields Avenue, Davis, CA 95616'),(1811,444,'__work_address','field_53209806c81da'),(1812,444,'_work_phone',''),(1813,444,'__work_phone','field_5320982dc81db'),(1814,444,'_people_category','Technicians'),(1815,444,'__people_category','field_530b901b0ca70'),(1816,444,'_education','MSc. University College Dublin, Ireland 2012\r\n\r\nB.S. in Biology, Emphasis in Chemistry, University of Washington, Seattle, WA 2006'),(1817,444,'__education','field_530503a0f8e71'),(1818,444,'_research_interests',''),(1819,444,'__research_interests','field_530507cebdd0e'),(1820,444,'_research_projects','San Joaquin restoration project'),(1821,444,'__research_projects','field_530507efae298'),(1822,444,'_publications','Meredith RW, Janecka JE, Gatesy J, Ryder OA, Fisher CA, Teeling EC, <strong>Goodbla A</strong>, Eizirik E, Simo TLL, Stadler T, Rabosky DL, Honeycutt RL, Flynn JJ, Ingram CM, Steiner C, Williams TL, Robinson TJ, Burk-Herrick A, Westerman M, Ayoub NA, Springer MS, Murphy WJ. 2011. Impacts of the Cretaceous Terrestrial Revolution and KPg Extinction on Extant Mammal Diversification. Science, 334, 521-524.\r\n\r\n&nbsp;\r\n\r\nWinans GA , McHenry M , Baker J , Elz A, <strong>Goodbla A</strong> , Iwamoto E , Kuligowski D , Miller KM , Small MP , Spruell P , Van-Doornik D . 2008. Genetic inventory of anadromous Pacific salmonids of the Elwha River prior to dam removal. Northwest Science, 82, 128141.'),(1823,444,'__publications','field_5305081a9bade'),(1827,449,'_edit_last','1'),(1828,449,'_edit_lock','1394822764:1'),(1829,449,'_thumbnail_id','83'),(1830,449,'_profile_picture','448'),(1831,449,'__profile_picture','field_52fbc541cfb03'),(1832,449,'_curriculum_vitae',''),(1833,449,'__curriculum_vitae','field_52fbce8765544'),(1834,449,'_alumni','0'),(1835,449,'__alumni','field_5310d6b20e086'),(1836,449,'_work_position','Junior Specialist'),(1837,449,'__work_position','field_5320970428360'),(1838,449,'_email','earingelman@ucdavis.edu'),(1839,449,'__email','field_532097d86af9d'),(1840,449,'_work_address',''),(1841,449,'__work_address','field_53209806c81da'),(1842,449,'_work_phone','530.752.6351'),(1843,449,'__work_phone','field_5320982dc81db'),(1844,449,'_people_category','Technicians'),(1845,449,'__people_category','field_530b901b0ca70'),(1846,449,'_education','University of Minnesota- Twin Cities, BS in Biology, 2007'),(1847,449,'__education','field_530503a0f8e71'),(1848,449,'_research_interests',''),(1849,449,'__research_interests','field_530507cebdd0e'),(1850,449,'_research_projects',''),(1851,449,'__research_projects','field_530507efae298'),(1852,449,'_publications',''),(1853,449,'__publications','field_5305081a9bade'),(1854,450,'_edit_last','1'),(1855,450,'_edit_lock','1394827972:1'),(1857,452,'_wp_attached_file','2014/03/antonia_wong_2012_cv.doc'),(1858,450,'_thumbnail_id','83'),(1859,450,'_profile_picture','447'),(1860,450,'__profile_picture','field_52fbc541cfb03'),(1861,450,'_curriculum_vitae','452'),(1862,450,'__curriculum_vitae','field_52fbce8765544'),(1863,450,'_alumni','0'),(1864,450,'__alumni','field_5310d6b20e086'),(1865,450,'_work_position','Junior Specialist'),(1866,450,'__work_position','field_5320970428360'),(1867,450,'_email','kapwong@ucdavis.edu'),(1868,450,'__email','field_532097d86af9d'),(1869,450,'_work_address','One Shields Avenue, Davis, CA 95616'),(1870,450,'__work_address','field_53209806c81da'),(1871,450,'_work_phone',''),(1872,450,'__work_phone','field_5320982dc81db'),(1873,450,'_people_category','Technicians'),(1874,450,'__people_category','field_530b901b0ca70'),(1875,450,'_education','B.S. Genetics, University of California, Davis, 2010\r\n\r\nMinor in Sociology'),(1876,450,'__education','field_530503a0f8e71'),(1877,450,'_research_interests',''),(1878,450,'__research_interests','field_530507cebdd0e'),(1879,450,'_research_projects','I work on SNPs analysis of Chinook salmons as part of the San Joaquin River Restoration Project.'),(1880,450,'__research_projects','field_530507efae298'),(1881,450,'_publications',''),(1882,450,'__publications','field_5305081a9bade'),(1883,453,'_edit_last','1'),(1884,453,'field_5323512c57543','a:10:{s:3:\"key\";s:19:\"field_5323512c57543\";s:5:\"label\";s:33:\"Link this person to another site?\";s:4:\"name\";s:19:\"_link_to_profile_tf\";s:4:\"type\";s:10:\"true_false\";s:12:\"instructions\";s:56:\"Does this person have a profile located on another site?\";s:8:\"required\";s:1:\"0\";s:7:\"message\";s:0:\"\";s:13:\"default_value\";s:1:\"0\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"0\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:4:\"null\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:0:\"\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:0;}'),(1885,453,'field_5323516957544','a:14:{s:3:\"key\";s:19:\"field_5323516957544\";s:5:\"label\";s:15:\"Link to Profile\";s:4:\"name\";s:16:\"_link_to_profile\";s:4:\"type\";s:4:\"text\";s:12:\"instructions\";s:0:\"\";s:8:\"required\";s:1:\"1\";s:13:\"default_value\";s:0:\"\";s:11:\"placeholder\";s:0:\"\";s:7:\"prepend\";s:0:\"\";s:6:\"append\";s:0:\"\";s:10:\"formatting\";s:4:\"html\";s:9:\"maxlength\";s:0:\"\";s:17:\"conditional_logic\";a:3:{s:6:\"status\";s:1:\"1\";s:5:\"rules\";a:1:{i:0;a:3:{s:5:\"field\";s:19:\"field_5323512c57543\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:1:\"1\";}}s:8:\"allorany\";s:3:\"all\";}s:8:\"order_no\";i:1;}'),(1887,453,'position','side'),(1888,453,'layout','default'),(1889,453,'hide_on_screen',''),(1890,453,'_edit_lock','1394824313:1'),(1891,453,'rule','a:5:{s:5:\"param\";s:9:\"post_type\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:6:\"people\";s:8:\"order_no\";i:0;s:8:\"group_no\";i:0;}'),(1892,450,'_link_to_profile_tf','0'),(1893,450,'__link_to_profile_tf','field_5323512c57543'),(1894,450,'_link_to_profile','http://www.cvm.umn.edu/vbs/faculty/Mickelson/lab/home.html'),(1895,450,'__link_to_profile','field_5323516957544'),(1899,456,'_wp_attached_file','2014/03/louise_conrad_profile_image.jpg'),(1900,456,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:194;s:6:\"height\";i:231;s:4:\"file\";s:39:\"2014/03/louise_conrad_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:39:\"louise_conrad_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";d:10.800000000000001;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:11:\"COOLPIX P50\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:1246462447;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";s:4:\"16.1\";s:3:\"iso\";s:2:\"64\";s:13:\"shutter_speed\";s:10:\"0.00459136\";s:5:\"title\";s:0:\"\";}}'),(1901,456,'_wp_attachment_image_alt','Louise conrad profile image'),(1902,454,'_edit_last','1'),(1903,454,'_link_to_profile_tf','1'),(1904,454,'__link_to_profile_tf','field_5323512c57543'),(1905,454,'_link_to_profile','http://www.water.ca.gov/aes/staff/conrad.cfm'),(1906,454,'__link_to_profile','field_5323516957544'),(1907,454,'_profile_picture','456'),(1908,454,'__profile_picture','field_52fbc541cfb03'),(1909,454,'_curriculum_vitae',''),(1910,454,'__curriculum_vitae','field_52fbce8765544'),(1911,454,'_alumni','0'),(1912,454,'__alumni','field_5310d6b20e086'),(1913,454,'_work_position',''),(1914,454,'__work_position','field_5320970428360'),(1915,454,'_email',''),(1916,454,'__email','field_532097d86af9d'),(1917,454,'_work_address',''),(1918,454,'__work_address','field_53209806c81da'),(1919,454,'_work_phone',''),(1920,454,'__work_phone','field_5320982dc81db'),(1921,454,'_people_category','Associates in Agricultural Experiment Station'),(1922,454,'__people_category','field_530b901b0ca70'),(1923,454,'_education',''),(1924,454,'__education','field_530503a0f8e71'),(1925,454,'_research_interests',''),(1926,454,'__research_interests','field_530507cebdd0e'),(1927,454,'_research_projects',''),(1928,454,'__research_projects','field_530507efae298'),(1929,454,'_publications',''),(1930,454,'__publications','field_5305081a9bade'),(1931,454,'_edit_lock','1394826595:1'),(1964,458,'_edit_last','1'),(1965,458,'_edit_lock','1394827995:1'),(1966,458,'_link_to_profile_tf','0'),(1967,458,'__link_to_profile_tf','field_5323512c57543'),(1968,458,'_profile_picture',''),(1969,458,'__profile_picture','field_52fbc541cfb03'),(1970,458,'_curriculum_vitae',''),(1971,458,'__curriculum_vitae','field_52fbce8765544'),(1972,458,'_alumni','0'),(1973,458,'__alumni','field_5310d6b20e086'),(1974,458,'_work_position',''),(1975,458,'__work_position','field_5320970428360'),(1976,458,'_email',''),(1977,458,'__email','field_532097d86af9d'),(1978,458,'_work_address',''),(1979,458,'__work_address','field_53209806c81da'),(1980,458,'_work_phone',''),(1981,458,'__work_phone','field_5320982dc81db'),(1982,458,'_people_category','Undergraduate Students'),(1983,458,'__people_category','field_530b901b0ca70'),(1984,458,'_education',''),(1985,458,'__education','field_530503a0f8e71'),(1986,458,'_research_interests',''),(1987,458,'__research_interests','field_530507cebdd0e'),(1988,458,'_research_projects',''),(1989,458,'__research_projects','field_530507efae298'),(1990,458,'_publications',''),(1991,458,'__publications','field_5305081a9bade'),(1992,459,'_edit_last','1'),(1993,459,'_edit_lock','1394828058:1'),(1994,459,'_link_to_profile_tf','0'),(1995,459,'__link_to_profile_tf','field_5323512c57543'),(1996,459,'_profile_picture',''),(1997,459,'__profile_picture','field_52fbc541cfb03'),(1998,459,'_curriculum_vitae',''),(1999,459,'__curriculum_vitae','field_52fbce8765544'),(2000,459,'_alumni','0'),(2001,459,'__alumni','field_5310d6b20e086'),(2002,459,'_work_position',''),(2003,459,'__work_position','field_5320970428360'),(2004,459,'_email',''),(2005,459,'__email','field_532097d86af9d'),(2006,459,'_work_address',''),(2007,459,'__work_address','field_53209806c81da'),(2008,459,'_work_phone',''),(2009,459,'__work_phone','field_5320982dc81db'),(2010,459,'_people_category','Undergraduate Students'),(2011,459,'__people_category','field_530b901b0ca70'),(2012,459,'_education',''),(2013,459,'__education','field_530503a0f8e71'),(2014,459,'_research_interests',''),(2015,459,'__research_interests','field_530507cebdd0e'),(2016,459,'_research_projects',''),(2017,459,'__research_projects','field_530507efae298'),(2018,459,'_publications',''),(2019,459,'__publications','field_5305081a9bade'),(2020,460,'_edit_last','1'),(2021,460,'_edit_lock','1394828075:1'),(2022,460,'_link_to_profile_tf','0'),(2023,460,'__link_to_profile_tf','field_5323512c57543'),(2024,460,'_profile_picture',''),(2025,460,'__profile_picture','field_52fbc541cfb03'),(2026,460,'_curriculum_vitae',''),(2027,460,'__curriculum_vitae','field_52fbce8765544'),(2028,460,'_alumni','0'),(2029,460,'__alumni','field_5310d6b20e086'),(2030,460,'_work_position',''),(2031,460,'__work_position','field_5320970428360'),(2032,460,'_email',''),(2033,460,'__email','field_532097d86af9d'),(2034,460,'_work_address',''),(2035,460,'__work_address','field_53209806c81da'),(2036,460,'_work_phone',''),(2037,460,'__work_phone','field_5320982dc81db'),(2038,460,'_people_category','Undergraduate Students'),(2039,460,'__people_category','field_530b901b0ca70'),(2040,460,'_education',''),(2041,460,'__education','field_530503a0f8e71'),(2042,460,'_research_interests',''),(2043,460,'__research_interests','field_530507cebdd0e'),(2044,460,'_research_projects',''),(2045,460,'__research_projects','field_530507efae298'),(2046,460,'_publications',''),(2047,460,'__publications','field_5305081a9bade'),(2048,461,'_edit_last','1'),(2049,461,'_edit_lock','1394828085:1'),(2050,461,'_link_to_profile_tf','0'),(2051,461,'__link_to_profile_tf','field_5323512c57543'),(2052,461,'_profile_picture',''),(2053,461,'__profile_picture','field_52fbc541cfb03'),(2054,461,'_curriculum_vitae',''),(2055,461,'__curriculum_vitae','field_52fbce8765544'),(2056,461,'_alumni','0'),(2057,461,'__alumni','field_5310d6b20e086'),(2058,461,'_work_position',''),(2059,461,'__work_position','field_5320970428360'),(2060,461,'_email',''),(2061,461,'__email','field_532097d86af9d'),(2062,461,'_work_address',''),(2063,461,'__work_address','field_53209806c81da'),(2064,461,'_work_phone',''),(2065,461,'__work_phone','field_5320982dc81db'),(2066,461,'_people_category','Undergraduate Students'),(2067,461,'__people_category','field_530b901b0ca70'),(2068,461,'_education',''),(2069,461,'__education','field_530503a0f8e71'),(2070,461,'_research_interests',''),(2071,461,'__research_interests','field_530507cebdd0e'),(2072,461,'_research_projects',''),(2073,461,'__research_projects','field_530507efae298'),(2074,461,'_publications',''),(2075,461,'__publications','field_5305081a9bade'),(2076,462,'_edit_last','1'),(2077,462,'_edit_lock','1394828302:1'),(2078,462,'_link_to_profile_tf','0'),(2079,462,'__link_to_profile_tf','field_5323512c57543'),(2080,462,'_profile_picture',''),(2081,462,'__profile_picture','field_52fbc541cfb03'),(2082,462,'_curriculum_vitae',''),(2083,462,'__curriculum_vitae','field_52fbce8765544'),(2084,462,'_alumni','0'),(2085,462,'__alumni','field_5310d6b20e086'),(2086,462,'_work_position',''),(2087,462,'__work_position','field_5320970428360'),(2088,462,'_email',''),(2089,462,'__email','field_532097d86af9d'),(2090,462,'_work_address',''),(2091,462,'__work_address','field_53209806c81da'),(2092,462,'_work_phone',''),(2093,462,'__work_phone','field_5320982dc81db'),(2094,462,'_people_category','High School Interns'),(2095,462,'__people_category','field_530b901b0ca70'),(2096,462,'_education',''),(2097,462,'__education','field_530503a0f8e71'),(2098,462,'_research_interests',''),(2099,462,'__research_interests','field_530507cebdd0e'),(2100,462,'_research_projects',''),(2101,462,'__research_projects','field_530507efae298'),(2102,462,'_publications',''),(2103,462,'__publications','field_5305081a9bade'),(2104,463,'_edit_last','1'),(2105,463,'_edit_lock','1394828314:1'),(2106,463,'_link_to_profile_tf','0'),(2107,463,'__link_to_profile_tf','field_5323512c57543'),(2108,463,'_profile_picture',''),(2109,463,'__profile_picture','field_52fbc541cfb03'),(2110,463,'_curriculum_vitae',''),(2111,463,'__curriculum_vitae','field_52fbce8765544'),(2112,463,'_alumni','0'),(2113,463,'__alumni','field_5310d6b20e086'),(2114,463,'_work_position',''),(2115,463,'__work_position','field_5320970428360'),(2116,463,'_email',''),(2117,463,'__email','field_532097d86af9d'),(2118,463,'_work_address',''),(2119,463,'__work_address','field_53209806c81da'),(2120,463,'_work_phone',''),(2121,463,'__work_phone','field_5320982dc81db'),(2122,463,'_people_category','Volunteers'),(2123,463,'__people_category','field_530b901b0ca70'),(2124,463,'_education',''),(2125,463,'__education','field_530503a0f8e71'),(2126,463,'_research_interests',''),(2127,463,'__research_interests','field_530507cebdd0e'),(2128,463,'_research_projects',''),(2129,463,'__research_projects','field_530507efae298'),(2130,463,'_publications',''),(2131,463,'__publications','field_5305081a9bade'),(2132,464,'_edit_last','1'),(2133,464,'_edit_lock','1394828451:1'),(2134,465,'_wp_attached_file','2014/03/gregg_schumer_profile_image.jpg'),(2135,465,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:274;s:6:\"height\";i:379;s:4:\"file\";s:39:\"2014/03/gregg_schumer_profile_image.jpg\";s:5:\"sizes\";a:2:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:39:\"gregg_schumer_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}s:6:\"medium\";a:4:{s:4:\"file\";s:39:\"gregg_schumer_profile_image-216x300.jpg\";s:5:\"width\";i:216;s:6:\"height\";i:300;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";d:5;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:11:\"FinePix F20\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";b:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";s:1:\"8\";s:3:\"iso\";s:3:\"100\";s:13:\"shutter_speed\";s:18:\"0.0018181818181818\";s:5:\"title\";s:0:\"\";}}'),(2136,465,'_wp_attachment_image_alt','Gregg Schumer Profile Image'),(2137,464,'_link_to_profile_tf','1'),(2138,464,'__link_to_profile_tf','field_5323512c57543'),(2139,464,'_link_to_profile','http://www.fishsciences.net/staff/schumer_gregg.php'),(2140,464,'__link_to_profile','field_5323516957544'),(2141,464,'_profile_picture','465'),(2142,464,'__profile_picture','field_52fbc541cfb03'),(2143,464,'_curriculum_vitae',''),(2144,464,'__curriculum_vitae','field_52fbce8765544'),(2145,464,'_alumni','0'),(2146,464,'__alumni','field_5310d6b20e086'),(2147,464,'_work_position',''),(2148,464,'__work_position','field_5320970428360'),(2149,464,'_email',''),(2150,464,'__email','field_532097d86af9d'),(2151,464,'_work_address',''),(2152,464,'__work_address','field_53209806c81da'),(2153,464,'_work_phone',''),(2154,464,'__work_phone','field_5320982dc81db'),(2155,464,'_people_category','Associates in Agricultural Experiment Station'),(2156,464,'__people_category','field_530b901b0ca70'),(2157,464,'_education',''),(2158,464,'__education','field_530503a0f8e71'),(2159,464,'_research_interests',''),(2160,464,'__research_interests','field_530507cebdd0e'),(2161,464,'_research_projects',''),(2162,464,'__research_projects','field_530507efae298'),(2163,464,'_publications',''),(2164,464,'__publications','field_5305081a9bade'),(2165,466,'_edit_last','1'),(2166,466,'_edit_lock','1394828938:1'),(2167,467,'_wp_attached_file','2014/03/kathleen_fisch_2012_cv.pdf'),(2168,468,'_wp_attached_file','2014/03/kathleen_fisch_profile_image.jpg'),(2169,468,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:300;s:6:\"height\";i:298;s:4:\"file\";s:40:\"2014/03/kathleen_fisch_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:40:\"kathleen_fisch_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(2170,468,'_wp_attachment_image_alt','Kathleen fisch profile image'),(2172,466,'_thumbnail_id','83'),(2173,466,'_link_to_profile_tf','0'),(2174,466,'__link_to_profile_tf','field_5323512c57543'),(2175,466,'_profile_picture','468'),(2176,466,'__profile_picture','field_52fbc541cfb03'),(2177,466,'_curriculum_vitae','467'),(2178,466,'__curriculum_vitae','field_52fbce8765544'),(2179,466,'_alumni','1'),(2180,466,'__alumni','field_5310d6b20e086'),(2181,466,'_work_position','California Sea Grant Delta Science Program\r\nPostdoctoral Fellow, San Diego Zoo Institute for Conservation Research\r\nAssociate, UC Davis Agricultural Experiment Station'),(2182,466,'__work_position','field_5320970428360'),(2183,466,'_email','KMFisch@ucdavis.edu\r\nKFisch@sandiegozoo.org\r\n'),(2184,466,'__email','field_532097d86af9d'),(2185,466,'_work_address','San Diego Zoo Institute for Conservation Research\r\n15600 San Pasqual Valley Road\r\nEscondido, CA 92027-7000'),(2186,466,'__work_address','field_53209806c81da'),(2187,466,'_work_phone','(760) 747-8702 ext. 5781'),(2188,466,'__work_phone','field_5320982dc81db'),(2189,466,'_people_category','Associates in Agricultural Experiment Station'),(2190,466,'__people_category','field_530b901b0ca70'),(2191,466,'_education','Ph.D. Ecology, UC Davis 2007-2011\r\n\r\nB.A. Integrative Biology, UC Berkeley 2002-2006'),(2192,466,'__education','field_530503a0f8e71'),(2193,466,'_research_interests','I am interested in broad questions of evolutionary population genetics, especially those with a direct conservation application. Through my dissertation work on the wild and captive population genetics of delta smelt, I have become increasingly interested in questions of captive population genetic management and the impacts of reintroduction of captive animals on wild populations.'),(2194,466,'__research_interests','field_530507cebdd0e'),(2195,466,'_research_projects','<strong>Saving San Francisco Bay-Delta native fishes: Hatchery management and reintroduction strategies</strong>\r\n\r\nNative fishes in the San Francisco Bay-Delta and Central Valley have recently seen a drastic decline in abundance, resulting in an increasing need to bring these fish into conservation hatcheries to prevent their extinction. A number of mandated hatchery-related projects exist in the Central Valley as part of NOAA Fisheries and US Fish and Wildlife Biological Opinions. Current and proposed hatchery operations make it imperative that we study appropriate genetic management, as planning for hatchery genetic management is necessary to determine the best management strategies that account for possible reintroductions and dynamics between hatchery and wild fish populations. Ideally, fish hatcheries would be managed to promote the long-term genetic and demographic viability of their populations, but genetic management is often lacking due to logistical constraints. Many captive fish populations accumulate detrimental genetic changes that decrease the genetic integrity of wild populations when their stocks are used for reintroduction. To improve fish hatchery management and reintroduction success, this study evaluates three methods of incorporating molecular data into hatchery management plans and their utility in preserving the genetic integrity of wild supplemented populations in the context of hatchery activities in the San Francisco Bay-Delta and Central Valley. As hatchery genetic management is a relatively contemporary idea, in addition to improving the logistics of hatchery genetic management, this study aims to establish a new culture of hatchery management, which should incorporate considerations of the genetic health of both captive and wild populations.\r\n\r\n<strong>Delta smelt refugial population genetic management &amp; monitoring</strong>\r\n\r\nA refugial population for the threatened delta smelt is currently held at the Fish Conservation &amp; Culture Laboratory (FCCL), a satellite facility of University of California, Davis (UC Davis), in collaboration with the Genomic Variation Lab (GVL) at UC Davis, and the US Fish and Wildlife Service. The goal of the delta smelt refugial population is to provide a genetic bank of delta smelt that will provide a safeguard for the species in the event of extinction in the wild. The intensive genetic management and monitoring of the refugial population of delta smelt is necessary to minimize genetic divergence of the captive population from the wild population. The delta smelt refugial population will be managed to be as similar to the wild population as possible, so in the event of species extinction in the wild, delta smelt may be reintroduced into the system contingent upon a thorough reintroduction and habitat remediation plan. Such a reintroduction, however, is not a current goal of the program. The genetic integrity of the captive population depends on intensive genetic management and monitoring, which are vital to the success of the delta smelt refugial population. In addition, the genetic management plan developed for the delta smelt refugial population may be adapted and applied to the captive propagation of other endangered species to safeguard them from species extinction.'),(2196,466,'__research_projects','field_530507efae298'),(2197,466,'_publications','<strong>Fisch, K.</strong>, Henderson, J., Burton, R. &amp; B. May. 2011. Population genetics and conservation implications for the endangered delta smelt in the San Francisco Bay-Delta. <em>Conservation Genetics</em> 12(6): 1421-1434.\r\n\r\nIsrael, J., <strong>Fisch, K.</strong>, Turner, T., &amp; R. Waples. 2011. Conservation of native Bay-Delta fishes: past experience and future considerations for artificial propagation of Chinook salmon, delta smelt, and green sturgeon. <em>San Francisco Estuary &amp; Watershed Science</em> 9(1): 1-20.\r\n\r\n<strong>Fisch, K.</strong>, Mahardja, B., Rettinghouse, T., Ellison, L., Tigan, G., Lindberg, J. &amp; B. May. 2010. Captive Breeding Plan for the Endangered Delta Smelt Refugial Population: Genetic Management and Fish Rearing Modifications for 2010. <em>Interagency Ecological Program Newsletter</em> 23(3): 13-20.\r\n\r\n<strong>Fisch, K.</strong>, Rettinghouse, T., Ellison, L., Tigan, G., Lindberg, J. &amp; B. May. 2009. Delta smelt refugial population development and genetic management2009 season summary. <em>Interagency Ecological Program Newsletter</em> 22(3): 3-9.\r\n\r\n<strong>Fisch, K.</strong>, Petersen, J., Pedroia, J., Baerwald, M. &amp; B. May. 2009. Characterization of 24 microsatellite loci in delta smelt, Hypomesus transpacificus, and their cross-species amplification in two other smelt species of the Osmeridae family. <em>Molecular Ecology Resources</em> 9(1): 405-408.'),(2198,466,'__publications','field_5305081a9bade'),(2199,470,'_edit_last','1'),(2200,470,'_edit_lock','1395079766:1'),(2201,471,'_wp_attached_file','2014/03/josh_israel_profile_image.jpg'),(2202,471,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:209;s:6:\"height\";i:300;s:4:\"file\";s:37:\"2014/03/josh_israel_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:37:\"josh_israel_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(2203,471,'_wp_attachment_image_alt','Josh israel profile image'),(2204,470,'_thumbnail_id','83'),(2205,470,'_link_to_profile_tf','0'),(2206,470,'__link_to_profile_tf','field_5323512c57543'),(2207,470,'_profile_picture','471'),(2208,470,'__profile_picture','field_52fbc541cfb03'),(2209,470,'_curriculum_vitae',''),(2210,470,'__curriculum_vitae','field_52fbce8765544'),(2211,470,'_alumni','1'),(2212,470,'__alumni','field_5310d6b20e086'),(2213,470,'_work_position','Fish Biologist\r\nAssociate in the College of Agriculture and Environmental Sciences\r\nEcology Graduate Group\r\n'),(2214,470,'__work_position','field_5320970428360'),(2215,470,'_email','jaisrael@usbr.gov\r\n'),(2216,470,'__email','field_532097d86af9d'),(2217,470,'_work_address','Bay Delta Office\r\nUS Bureau of Reclamation\r\n840 I Street, Suite 140\r\nSacramento, CA'),(2218,470,'__work_address','field_53209806c81da'),(2219,470,'_work_phone','(916) 414-2417'),(2220,470,'__work_phone','field_5320982dc81db'),(2221,470,'_people_category','Associates in Agricultural Experiment Station'),(2222,470,'__people_category','field_530b901b0ca70'),(2223,470,'_education','Ph.D. UC Davis, 2007\r\n\r\nB.S. UC Santa Cruz, 1995'),(2224,470,'__education','field_530503a0f8e71'),(2225,470,'_research_interests','Migratory fish management and ecology\r\n\r\nPopulation demographics\r\n\r\nTelemetry\r\n\r\nGenetics'),(2226,470,'__research_interests','field_530507cebdd0e'),(2227,470,'_research_projects',''),(2228,470,'__research_projects','field_530507efae298'),(2229,470,'_publications','<strong>Israel, J.A.</strong>, M. Neuman, M. L. Moser, S. T. Lindley, B. W. McCovey Jr., D. L. Erickson, P. Klimley. Understanding the biology and ecology of green sturgeon (<em>Acipenser medirostris</em>) for evaluating threats to this imperiled fish. <em>Journal of Applied Icthyology</em>. In revision.\r\n\r\nKatz, J., P.B. Moyle, R.M. Quiones, <strong>J.A. Israel</strong>, and S. Purdy. Impending extinction of salmon, steelhead, and trout (Salmonidae) in California. <em>Environmental Biology of Fishes.</em> Online First. DOI: 10.1007/s10641-012-9974-8\r\n\r\n<strong>Israel, J.A</strong>., K. Fisch, T. Turner, and R. Waples. 2011. The use of artificial propagation as a tool for native fish conservation in the western US: what we need to know for the preservation of Central Valley salmonids, delta smelt, and green sturgeon. <em>San Francisco Estuary and Watershed Sciences</em> 9. Retrieved from: <a title=\"San Francisco Estuary &amp; Watershed\" href=\"http://escholarship.ucop.edu/uc/item/9r80d47p\" target=\"_blank\">http://escholarship.ucop.edu/uc/item/9r80d47p</a>.\r\n\r\n<strong>Israel, J.A.</strong> and B. May. 2010. Characterization and evaluation of polymorphic microsatellite markers in the anadromous fish <em>Spirinchus thaleichthys</em>. <em>Conservation Genetics Resources</em> 2: 227-230.\r\n\r\n<strong>Israel, J.A.</strong> and B. May. 2010. Indirect genetic estimates of breeding population size in the polyploid green sturgeon (<em>Acipenser medirostris</em>). <em>Molecular Ecology</em> 19:1058-1070.\r\n\r\n<strong>Israel, J.A.</strong>, K.J. Bando, E.C. Anderson, and B. May. 2009. Stock complexity in North American green sturgeon: the utility of polysomic markers in mixed stock fishery analysis <em>Canadian Journal of Fisheries and Aquatic Science</em> 66:1491-1504.\r\n\r\nBrk, K., A. Drauch, <strong>J.A Israel</strong>, J. Pedroia, J. Rodzen, and B. May. 2008. Development of new microsatellite primers for green and white sturgeon. 9:973-979.\r\n\r\nKlimley, A.P., P.A. Allen, <strong>J.A. Israel</strong> and J.T. Kelly. 2007. Green sturgeon and its environment: Introduction. <em>Environmental Biology of Fishes</em>, 79:187-190.\r\n\r\nKlimley, A.P., P.A. Allen, <strong>J.A. Israel</strong> and J.T. Kelly. 2007. Green Sturgeon and its Environment: Past, Present, and Future <em>Environmental Biology of Fishes</em>, 79:415-421.\r\n\r\nShmigirilov A.P., A.A. Mednikova, <strong>J.A. Israel.</strong> 2007. Comparison of biology of the Sakhalin sturgeon, Amur sturgeon, and kaluga from the Amur River, Sea of Okhotsk, and Sea of Japan Biogeographic Province. <em>Environmental Biology of Fishes</em>, 79:383-395.\r\n\r\nPapa, R., <strong>J.A. Israel</strong>, F. N. Marzano, and B. May. 2007. Assessment of Genetic Variation among reproductive ecotypes of Klamath River Steelhead reveals differentiation associated with different run-timings. <em>Journal of Applied Ichthyology</em> 23:142-146.\r\n\r\nMoyle, P. and <strong>J.A. Israel</strong>. 2005. Untested assumptions: effectiveness of screening diversions for conservation of fish populations. <em>Fisheries</em>: 30(5): 20-28.\r\n\r\nI<strong>srael, J.A.</strong>, M. Blumberg, J. Cordes and B. May. 2004. Geographic patterns of genetic differentiation among western U.S. collections of North American green sturgeon (<em>Acipenser medirostris</em>). <em>North American Journal of Fisheries Management</em> 24: 922-931.\r\n\r\nJ.F. Cordes, <strong>Israel J.A.</strong>, and B. May. 2004. Conservation of Paiute cutthroat trout: measuring the genetic legacy of population transplants and hybridization in an endemic California salmonid. <em>California Fish and Game Bulletin</em> 90(3): 101-118.'),(2230,470,'__publications','field_5305081a9bade'),(2231,473,'_wp_attached_file','2014/03/brian_schreier_profile_image.jpg'),(2232,473,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:316;s:6:\"height\";i:302;s:4:\"file\";s:40:\"2014/03/brian_schreier_profile_image.jpg\";s:5:\"sizes\";a:2:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:40:\"brian_schreier_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}s:6:\"medium\";a:4:{s:4:\"file\";s:40:\"brian_schreier_profile_image-300x286.jpg\";s:5:\"width\";i:300;s:6:\"height\";i:286;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";d:11;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:13:\"Canon EOS D60\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:1272281112;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";s:2:\"80\";s:3:\"iso\";s:3:\"200\";s:13:\"shutter_speed\";s:5:\"0.004\";s:5:\"title\";s:0:\"\";}}'),(2233,473,'_wp_attachment_image_alt','Brian schreier profile image'),(2234,472,'_edit_last','1'),(2235,472,'_edit_lock','1395080922:1'),(2236,472,'_link_to_profile_tf','1'),(2237,472,'__link_to_profile_tf','field_5323512c57543'),(2238,472,'_link_to_profile','http://www.water.ca.gov/aes/staff/schreier.cfm'),(2239,472,'__link_to_profile','field_5323516957544'),(2240,472,'_profile_picture','473'),(2241,472,'__profile_picture','field_52fbc541cfb03'),(2242,472,'_curriculum_vitae',''),(2243,472,'__curriculum_vitae','field_52fbce8765544'),(2244,472,'_alumni','0'),(2245,472,'__alumni','field_5310d6b20e086'),(2246,472,'_work_position',''),(2247,472,'__work_position','field_5320970428360'),(2248,472,'_email',''),(2249,472,'__email','field_532097d86af9d'),(2250,472,'_work_address',''),(2251,472,'__work_address','field_53209806c81da'),(2252,472,'_work_phone',''),(2253,472,'__work_phone','field_5320982dc81db'),(2254,472,'_people_category','Associates in Agricultural Experiment Station'),(2255,472,'__people_category','field_530b901b0ca70'),(2256,472,'_education',''),(2257,472,'__education','field_530503a0f8e71'),(2258,472,'_research_interests',''),(2259,472,'__research_interests','field_530507cebdd0e'),(2260,472,'_research_projects',''),(2261,472,'__research_projects','field_530507efae298'),(2262,472,'_publications',''),(2263,472,'__publications','field_5305081a9bade'),(2264,474,'_edit_last','1'),(2265,474,'_edit_lock','1394830377:1'),(2266,475,'_wp_attached_file','2014/03/scott_blankenship_profile_image.jpg'),(2267,475,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:334;s:6:\"height\";i:224;s:4:\"file\";s:43:\"2014/03/scott_blankenship_profile_image.jpg\";s:5:\"sizes\";a:2:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:43:\"scott_blankenship_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}s:6:\"medium\";a:4:{s:4:\"file\";s:43:\"scott_blankenship_profile_image-300x201.jpg\";s:5:\"width\";i:300;s:6:\"height\";i:201;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";d:5;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:27:\"Canon EOS DIGITAL REBEL XTi\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:1330148515;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";s:2:\"60\";s:3:\"iso\";s:3:\"400\";s:13:\"shutter_speed\";s:17:\"0.016666666666667\";s:5:\"title\";s:0:\"\";}}'),(2268,475,'_wp_attachment_image_alt','Scott blankenship profile image'),(2269,474,'_link_to_profile_tf','1'),(2270,474,'__link_to_profile_tf','field_5323512c57543'),(2271,474,'_link_to_profile','http://www.fishsciences.net/staff/blankenship_scott.php'),(2272,474,'__link_to_profile','field_5323516957544'),(2273,474,'_profile_picture','475'),(2274,474,'__profile_picture','field_52fbc541cfb03'),(2275,474,'_curriculum_vitae',''),(2276,474,'__curriculum_vitae','field_52fbce8765544'),(2277,474,'_alumni','0'),(2278,474,'__alumni','field_5310d6b20e086'),(2279,474,'_work_position',''),(2280,474,'__work_position','field_5320970428360'),(2281,474,'_email',''),(2282,474,'__email','field_532097d86af9d'),(2283,474,'_work_address',''),(2284,474,'__work_address','field_53209806c81da'),(2285,474,'_work_phone',''),(2286,474,'__work_phone','field_5320982dc81db'),(2287,474,'_people_category','Associates in Agricultural Experiment Station'),(2288,474,'__people_category','field_530b901b0ca70'),(2289,474,'_education',''),(2290,474,'__education','field_530503a0f8e71'),(2291,474,'_research_interests',''),(2292,474,'__research_interests','field_530507cebdd0e'),(2293,474,'_research_projects',''),(2294,474,'__research_projects','field_530507efae298'),(2295,474,'_publications',''),(2296,474,'__publications','field_5305081a9bade'),(2297,476,'_edit_last','1'),(2298,476,'_edit_lock','1394830490:1'),(2299,477,'_wp_attached_file','2014/03/jessica_petersen_profile_image.png'),(2300,477,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:110;s:6:\"height\";i:83;s:4:\"file\";s:42:\"2014/03/jessica_petersen_profile_image.png\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(2301,477,'_wp_attachment_image_alt','Jessica petersen profile image'),(2302,476,'_link_to_profile_tf','1'),(2303,476,'__link_to_profile_tf','field_5323512c57543'),(2304,476,'_link_to_profile','http://www.cvm.umn.edu/vbs/faculty/Mickelson/lab/home.html'),(2305,476,'__link_to_profile','field_5323516957544'),(2306,476,'_profile_picture','477'),(2307,476,'__profile_picture','field_52fbc541cfb03'),(2308,476,'_curriculum_vitae',''),(2309,476,'__curriculum_vitae','field_52fbce8765544'),(2310,476,'_alumni','1'),(2311,476,'__alumni','field_5310d6b20e086'),(2312,476,'_work_position',''),(2313,476,'__work_position','field_5320970428360'),(2314,476,'_email',''),(2315,476,'__email','field_532097d86af9d'),(2316,476,'_work_address',''),(2317,476,'__work_address','field_53209806c81da'),(2318,476,'_work_phone',''),(2319,476,'__work_phone','field_5320982dc81db'),(2320,476,'_people_category','Associates in Agricultural Experiment Station'),(2321,476,'__people_category','field_530b901b0ca70'),(2322,476,'_education',''),(2323,476,'__education','field_530503a0f8e71'),(2324,476,'_research_interests',''),(2325,476,'__research_interests','field_530507cebdd0e'),(2326,476,'_research_projects',''),(2327,476,'__research_projects','field_530507efae298'),(2328,476,'_publications',''),(2329,476,'__publications','field_5305081a9bade'),(2330,478,'_edit_last','1'),(2331,478,'_edit_lock','1394830564:1'),(2332,479,'_wp_attached_file','2014/03/ryan_lew_profile_image.png'),(2333,479,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:110;s:6:\"height\";i:82;s:4:\"file\";s:34:\"2014/03/ryan_lew_profile_image.png\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(2334,479,'_wp_attachment_image_alt','Ryan lew profile image'),(2335,478,'_thumbnail_id','83'),(2336,478,'_link_to_profile_tf','0'),(2337,478,'__link_to_profile_tf','field_5323512c57543'),(2338,478,'_profile_picture','479'),(2339,478,'__profile_picture','field_52fbc541cfb03'),(2340,478,'_curriculum_vitae',''),(2341,478,'__curriculum_vitae','field_52fbce8765544'),(2342,478,'_alumni','0'),(2343,478,'__alumni','field_5310d6b20e086'),(2344,478,'_work_position',''),(2345,478,'__work_position','field_5320970428360'),(2346,478,'_email',''),(2347,478,'__email','field_532097d86af9d'),(2348,478,'_work_address',''),(2349,478,'__work_address','field_53209806c81da'),(2350,478,'_work_phone',''),(2351,478,'__work_phone','field_5320982dc81db'),(2352,478,'_people_category','Graduate Students'),(2353,478,'__people_category','field_530b901b0ca70'),(2354,478,'_education',''),(2355,478,'__education','field_530503a0f8e71'),(2356,478,'_research_interests',''),(2357,478,'__research_interests','field_530507cebdd0e'),(2358,478,'_research_projects',''),(2359,478,'__research_projects','field_530507efae298'),(2360,478,'_publications',''),(2361,478,'__publications','field_5305081a9bade'),(2362,480,'_edit_last','1'),(2363,480,'_edit_lock','1395075747:1'),(2364,481,'_wp_attached_file','2014/03/brian_mahardja_2013_cv.pdf'),(2365,482,'_wp_attached_file','2014/03/brian_mahardja_profile_image.jpg'),(2366,482,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:300;s:6:\"height\";i:225;s:4:\"file\";s:40:\"2014/03/brian_mahardja_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:40:\"brian_mahardja_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";d:8;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:7:\"EX-Z750\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:1291039889;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";s:3:\"7.9\";s:3:\"iso\";i:0;s:13:\"shutter_speed\";s:5:\"0.008\";s:5:\"title\";s:0:\"\";}}'),(2367,482,'_wp_attachment_image_alt','Brian mahardja profile image'),(2368,480,'_thumbnail_id','83'),(2369,480,'_link_to_profile_tf','0'),(2370,480,'__link_to_profile_tf','field_5323512c57543'),(2371,480,'_profile_picture','482'),(2372,480,'__profile_picture','field_52fbc541cfb03'),(2373,480,'_curriculum_vitae','481'),(2374,480,'__curriculum_vitae','field_52fbce8765544'),(2375,480,'_alumni','0'),(2376,480,'__alumni','field_5310d6b20e086'),(2377,480,'_work_position','M.S. Student\r\nGraduate Group of Ecology'),(2378,480,'__work_position','field_5320970428360'),(2379,480,'_email','bmahardja@ucdavis.edu'),(2380,480,'__email','field_532097d86af9d'),(2381,480,'_work_address',''),(2382,480,'__work_address','field_53209806c81da'),(2383,480,'_work_phone','(530) 752-6351'),(2384,480,'__work_phone','field_5320982dc81db'),(2385,480,'_people_category','Graduate Students'),(2386,480,'__people_category','field_530b901b0ca70'),(2387,480,'_education','2009  University of California, Davis - B. S. in Genetics &amp; Anthropology\r\n\r\nExpected in 2013 - University of California, Davis - M. S. in Ecology'),(2388,480,'__education','field_530503a0f8e71'),(2389,480,'_research_interests','My main research interest lies in the use of molecular tools to aid conservation management of threatened species. Although the primary focus of my research has been on the population structure and evolutionary history of native freshwater Californian fish species, I believe that the application of molecular techniques transcends taxon boundaries and may answer crucial biological questions for imperiled species in any location.'),(2390,480,'__research_interests','field_530507cebdd0e'),(2391,480,'_research_projects','<strong>Demographic connectivity and viability of two distinct splittail (</strong><em>Pogonichthys macrolepidotus</em><strong>) populations in the San Francisco estuary</strong>\r\n\r\nDue to habitat alteration, overexploitation, and introduction of exotic species, most of the native fish fauna in the San Francisco Estuary are experiencing a population decline. The splittail (<em>Pogonichthys macrolepidotus</em>) is Californian cyprinid fish species which resides in the bay during the summer and migrates upstream to spawn in floodplains and river margins in the winter and spring. This species was once listed as a threatened species under the Endangered Species Act and is still considered as a Species of Special Concern by the California Department of Fish and Game. Lower abundance indices in recent years and an ongoing status review present a clear need for a better understanding of the population dynamics and life history of the species.\r\n\r\nMy research is a part of a multifaceted study on the connectivity of different splittail populations funded by the Delta Science Program. In 2007, research conducted by Dr. Melinda Baerwald (currently a project scientist here in the Department of Animal Science) revealed that splittails are comprised of two distinct populations: one from the Sacramento and San Joaquin Rivers, and another from the Petaluma and Napa Rivers. Past studies on the species have focused on the Central Valley population and little is known about the less abundant Petaluma-Napa population. The focus of my study will be to estimate the effective and census population size of each population as well as to assign captured adult splittail to their respective population of origin. We will develop new microsatellite markers for this study in addition to the thirteen original microsatellite loci utilized in the previous research. We hypothesize that the Petaluma-Napa River population will be much lower in abundance due to a more sparse spawning habitat. Population and genetic diversity estimates acquired from this study will further our understanding of the ever-changing San Francisco Estuary ecosystem and provide much needed information for a better management of the species.'),(2392,480,'__research_projects','field_530507efae298'),(2393,480,'_publications','Schreier, A.D., B. Mahardja, and B. May. 2013. Hierarchical patterns of population structure in the endangered Fraser River white sturgeon, <em>Acipenser transmontanus</em>, and implications for conservation. Can. J. Fish. &amp; Aquat. Sci. 69:1968-1980 (<a title=\"2013_Schreier_Fraser_river_white_sturgeon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Brian_Mahardja_files/2013%20-%20Schreier%20Fraser%20river%20white%20sturgeon.pdf\">pdf</a>).\r\n\r\nMahardja, B., B. May, and M. Baerwald. 2012. Characterization of 36 additional microsatellite loci in splittail (<em>Pogonichthys macrolepidotus</em>) and cross-amplification in five other native Californian cyprinid species. Cons. Gen. Res. 4(4):917-921 (<a title=\"2012_Mahardja_et_al._Cyprinid_Microsat_note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Brian_Mahardja_files/2012%20Mahardja%20et%20al.%20-%20Cyprinid%20Microsat%20note.pdf\">pdf</a>).\r\n\r\nSchreier, A.D., D. Gille, B. Mahardja, and B. May. 2011. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon, <em>Acipenser transmontanus</em>. J. Appl. Ichthiol. 27:24-33 (<a title=\"2011_Schreier_et_al._Octoploid_origin_sturgeon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Brian_Mahardja_files/2011%20Schreier%20et%20al.%20-%20Octoploid%20origin%20sturgeon.pdf\">pdf</a>).'),(2394,480,'__publications','field_5305081a9bade'),(2395,483,'_edit_last','1'),(2396,483,'_edit_lock','1395083068:1'),(2397,484,'_wp_attached_file','2014/03/Daphne_Gille_cv.pdf'),(2398,485,'_wp_attached_file','2014/03/daphne_gille_profile_image.jpg'),(2399,485,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:241;s:6:\"height\";i:300;s:4:\"file\";s:38:\"2014/03/daphne_gille_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:38:\"daphne_gille_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";d:2.7999999999999998;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:12:\"FinePix E550\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:1172935048;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";s:3:\"7.2\";s:3:\"iso\";s:3:\"100\";s:13:\"shutter_speed\";s:8:\"0.003125\";s:5:\"title\";s:0:\"\";}}'),(2400,485,'_wp_attachment_image_alt','Daphne_gille_profile_image'),(2401,483,'_thumbnail_id','83'),(2402,486,'_wp_attached_file','2014/03/daphne_gille_seal_image.jpg'),(2403,486,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:216;s:6:\"height\";i:161;s:4:\"file\";s:35:\"2014/03/daphne_gille_seal_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:35:\"daphne_gille_seal_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(2404,487,'_wp_attached_file','2014/03/daphne_gille_seal_on_beach_image.jpg'),(2405,487,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:232;s:6:\"height\";i:154;s:4:\"file\";s:44:\"2014/03/daphne_gille_seal_on_beach_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:44:\"daphne_gille_seal_on_beach_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(2406,488,'_wp_attached_file','2014/03/daphne_gille_turtle_on_beach_image.jpg'),(2407,488,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:200;s:6:\"height\";i:150;s:4:\"file\";s:46:\"2014/03/daphne_gille_turtle_on_beach_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:46:\"daphne_gille_turtle_on_beach_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(2408,486,'_wp_attachment_image_alt','Daphne gille seal image of her posing with a seal'),(2409,487,'_wp_attachment_image_alt','Daphne gille seal on beach image'),(2410,488,'_wp_attachment_image_alt',''),(2411,483,'_link_to_profile_tf','0'),(2412,483,'__link_to_profile_tf','field_5323512c57543'),(2413,483,'_profile_picture','485'),(2414,483,'__profile_picture','field_52fbc541cfb03'),(2415,483,'_curriculum_vitae','484'),(2416,483,'__curriculum_vitae','field_52fbce8765544'),(2417,483,'_alumni','0'),(2418,483,'__alumni','field_5310d6b20e086'),(2419,483,'_work_position','PhD Candidate\r\nGenetics Graduate Group'),(2420,483,'__work_position','field_5320970428360'),(2421,483,'_email','dagille@ucdavis.edu'),(2422,483,'__email','field_532097d86af9d'),(2423,483,'_work_address','University of California, Davis\r\nDepartment of Animal Science\r\nOne Shields Avenue, Davis, CA 95616'),(2424,483,'__work_address','field_53209806c81da'),(2425,483,'_work_phone','(530) 752-6351 (lab)'),(2426,483,'__work_phone','field_5320982dc81db'),(2427,483,'_people_category','Graduate Students'),(2428,483,'__people_category','field_530b901b0ca70'),(2429,483,'_education','BS - Biochemistry &amp; Molecular Biology, University of California Santa Cruz\r\n\r\nMS - Organismal Biology, Conservation &amp; Ecology (Emphasis: Conservation Genetics), San Jose State University'),(2430,483,'__education','field_530503a0f8e71'),(2431,483,'_research_interests','I am broadly interested in using genetic tools to answer ecological questions and to inform management decisions regarding the conservation of endangered species and populations. My dissertation research will focus on improving white sturgeon (<em>Acipenser transmontanus</em>) caviar farming and aquaculture practices in the United States in an effort to reduce fishing pressure on wild stocks. Specifically, I will evaluate the heritability and identify genetic markers that are associated with high caviar yield and minimal ovarian adiposity. Other projects in wildlife conservation include the evaluation of population structure and genetic integrity of desert bighorn sheep in Arizona and determination of prevalence and characterization of the genetic strains of leptospirosis in California wild pigs.'),(2432,483,'__research_interests','field_530507cebdd0e'),(2433,483,'_research_projects','<strong>Improving Parentage Assignment Accuracy Within Mixed Family Cohorts of Captive-Reared White Sturgeon</strong>\r\n\r\nOne obstacle that must be overcome in order to conduct experiments that evaluate the contribution of genetics to caviar yield and ovarian adiposity is the resolution of familial relationships and the construction of a reliable pedigree. Current methods to resolve familial relationships among captive reared white sturgeon use too few markers and are not accurate enough to proceed with heritability studies. I will test and incorporate six additional microsatellite markers (for a panel of 15 total microsatellite markers) into white sturgeon parentage analysis and identify the minimum number required to increase the accuracy of assignment to a target of 95% within families reared at a caviar farm in California.\r\n\r\n<strong>Heritability of Caviar Yield and Ovarian Adiposity in Mature Captive-Reared Female White Sturgeon</strong>\r\n\r\nOne common problem that threatens the efficiency and success of caviar farming in the United States is highly variable roe yield in mature female white sturgeon associated with accumulation of fat in the ovaries. White sturgeon with lean ovaries tend to produce a greater yield of superior quality roe compared to those with fatty ovaries. While environmental, developmental, and genetic factors may all influence caviar yield and ovarian adiposity, the role of each factor is poorly understood. I will assess the role of genetics in determining caviar yield and ovarian adiposity by calculating narrow-sense heritability in mature female white sturgeon under different environmental conditions at a caviar farm in California.\r\n\r\n<strong>Identification of Genetic Markers Associated With High Caviar Yield and Lean Ovaries</strong>\r\n\r\nVery little is known about the white sturgeon genome, much less about the genes that are associated with caviar yield and ovarian adiposity. I will use Restriction-Site Associated DNA Sequencing (RADseq) using the Illumina platform to identify thousands of single nucleotide polymorphisms (SNPs) throughout the white sturgeon genome. As sturgeon are complex polyploids, I will estimate the dosage of each SNP and use single-dose markers to construct a linkage map from which QTLs associated with high caviar yield and lean ovaries will be identified.\r\n\r\n<img class=\"size-full wp-image-487 alignleft\" alt=\"Daphne gille seal on beach image\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/daphne_gille_seal_on_beach_image.jpg\" width=\"232\" height=\"154\" /><img class=\"size-full wp-image-488 alignleft\" alt=\"Daphne gille turtle on beach image\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/daphne_gille_turtle_on_beach_image.jpg\" width=\"200\" height=\"150\" /><img class=\"size-full wp-image-486 alignleft\" alt=\"Daphne gille seal image of her posing with a seal\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/daphne_gille_seal_image.jpg\" width=\"216\" height=\"161\" />'),(2434,483,'__research_projects','field_530507efae298'),(2435,483,'_publications','Ernest H, Harmeling B, Goldberg L, Well J, <strong>Gille D</strong>. Yellow-billed Magpie genetic diversity and population structure prior to entry of West Nile Virus. <em>In prep</em>.\r\n\r\nSingh VK, Christensen J, Fatanmi OO, <strong>Gille D</strong>, Ducey EJ, Wise SY, Singh PK, Karsunky H, Sedello AK. Myeloid progenitors: a radiation countermeasure that is effective when initiated days after irradiation. <em>Submitted to Radiation Research</em>.\r\n\r\nChristensen JL, <strong>Gille DA</strong>, Smith-Berdan SK, Sagy M, Domen J, Sedello AK. Ex Vivo derived, pooled allogenic myeloid progenitors give rise to cells of the myelo-erythroid lineages in vivo and protect mice from lethal fungal challenge. <em>Submitted to Stem Cells and Development</em>.\r\n\r\nDomen J, Christensen JL, Wahedi M, Danenberg E, <strong>Gille D</strong>, Smith-Berdan S, Fong T, and Brown JMY. Ex Vivo Expanded Cryopreserved Allogeneic Myeloid Progenitors Protect Against Lethal Invasive Aspergillosis In Neutropenic Mice. <em>Submitted to Experimental Hematology</em>.\r\n\r\nDrauch Schreier A, <strong>Gille DA</strong>, Mahardja B, May B. 2011. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon, Acipenser transmontanus. J Appl Ichthyol. 27(Suppl. 2):24-33.\r\n\r\nSmith-Berdan S,<strong> Gille D</strong>, Weissman IL, Christensen, JL. 2007. Reversal of autoimmune disease in lupus-prone NZB/NZW mice by nonmyeloablative transplantation of purified allogeneic hematopoietic stem cells. Blood. 110(4): 1370-1378.'),(2436,483,'__publications','field_5305081a9bade'),(2437,489,'_edit_last','1'),(2438,489,'_edit_lock','1395078175:1'),(2439,490,'_wp_attached_file','2014/03/k_jun_bando_2012_cv.pdf'),(2440,491,'_wp_attached_file','2014/03/k_jun_bando_profile_image.jpg'),(2441,491,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:300;s:6:\"height\";i:225;s:4:\"file\";s:37:\"2014/03/k_jun_bando_profile_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:37:\"k_jun_bando_profile_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(2442,491,'_wp_attachment_image_alt','K Jun Bando Profile Image'),(2443,489,'_thumbnail_id','418'),(2444,489,'_link_to_profile_tf','0'),(2445,489,'__link_to_profile_tf','field_5323512c57543'),(2446,489,'_profile_picture','491'),(2447,489,'__profile_picture','field_52fbc541cfb03'),(2448,489,'_curriculum_vitae','490'),(2449,489,'__curriculum_vitae','field_52fbce8765544'),(2450,489,'_alumni','1'),(2451,489,'__alumni','field_5310d6b20e086'),(2452,489,'_work_position','Special Advisor to the Commander,\r\nUnited States Africa Command'),(2453,489,'__work_position','field_5320970428360'),(2454,489,'_email',''),(2455,489,'__email','field_532097d86af9d'),(2456,489,'_work_address',''),(2457,489,'__work_address','field_53209806c81da'),(2458,489,'_work_phone',''),(2459,489,'__work_phone','field_5320982dc81db'),(2460,489,'_people_category','Others'),(2461,489,'__people_category','field_530b901b0ca70'),(2462,489,'_education','Ph.D., Ecology, Marine Ecology Area of Emphasis, UC Davis, 2005\r\n\r\nDissertation: Ecological and evolutionary dynamics of Zostera japonica and Spartina alterniflora invasions in the eastern Pacific\r\n\r\nCoadvisors: Marcel Holyoak and Don Strong'),(2463,489,'__education','field_530503a0f8e71'),(2464,489,'_research_interests','My scientific research focuses on investigating the evolutionary ecology of marine invasions and the molecular population genetics of polyploid species. Additional areas of research have included nutrient and grazer influences on seagrass population dynamics, and the consequences of ecosystem engineering and positive feedbacks for the population growth and spatial spread of invasive species. I have also worked as an international affairs practitioner in support of U.S. diplomatic and defense efforts, and am currently focused on facilitating complex decision-making on national security issues.'),(2465,489,'__research_interests','field_530507cebdd0e'),(2466,489,'_research_projects',''),(2467,489,'__research_projects','field_530507efae298'),(2468,489,'_publications',''),(2469,489,'__publications','field_5305081a9bade'),(2470,426,'_link_to_profile_tf','0'),(2471,426,'__link_to_profile_tf','field_5323512c57543'),(2472,208,'rule','a:5:{s:5:\"param\";s:9:\"post_type\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:6:\"people\";s:8:\"order_no\";i:0;s:8:\"group_no\";i:0;}'),(2473,395,'__project_excerpt','field_532099cee4ec6'),(2474,395,'__contact_name','field_53209a3637a8e'),(2475,395,'__contact_email','field_53209a5537a8f'),(2478,492,'_edit_last','1'),(2479,492,'_edit_lock','1395078144:1'),(2480,493,'_wp_attached_file','2014/03/karrigan_bork_profile_image.png'),(2481,493,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:110;s:6:\"height\";i:83;s:4:\"file\";s:39:\"2014/03/karrigan_bork_profile_image.png\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(2482,494,'_wp_attached_file','2014/03/rachel_schwartz_profile_image.png'),(2483,494,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:110;s:6:\"height\";i:83;s:4:\"file\";s:41:\"2014/03/rachel_schwartz_profile_image.png\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(2484,495,'_wp_attached_file','2014/03/rachel_simmons_profile_image.png'),(2485,495,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:108;s:6:\"height\";i:83;s:4:\"file\";s:40:\"2014/03/rachel_simmons_profile_image.png\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(2486,496,'_wp_attached_file','2014/03/rick_topinka_profile_image.png'),(2487,496,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:110;s:6:\"height\";i:83;s:4:\"file\";s:38:\"2014/03/rick_topinka_profile_image.png\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(2488,497,'_wp_attached_file','2014/03/yonjiu_chen_profile_image.png'),(2489,497,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:110;s:6:\"height\";i:83;s:4:\"file\";s:37:\"2014/03/yonjiu_chen_profile_image.png\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(2490,493,'_wp_attachment_image_alt','Karrigan bork profile image'),(2491,494,'_wp_attachment_image_alt','Rachel schwartz profile image'),(2492,495,'_wp_attachment_image_alt','Rachel simmons profile image'),(2493,496,'_wp_attachment_image_alt','Rick topinka profile image'),(2494,497,'_wp_attachment_image_alt','Yonjiu chen profile image'),(2495,492,'_thumbnail_id','83'),(2496,492,'_link_to_profile_tf','0'),(2497,492,'__link_to_profile_tf','field_5323512c57543'),(2498,492,'_profile_picture','493'),(2499,492,'__profile_picture','field_52fbc541cfb03'),(2500,492,'_curriculum_vitae',''),(2501,492,'__curriculum_vitae','field_52fbce8765544'),(2502,492,'_alumni','1'),(2503,492,'__alumni','field_5310d6b20e086'),(2504,492,'_work_position',''),(2505,492,'__work_position','field_5320970428360'),(2506,492,'_email',''),(2507,492,'__email','field_532097d86af9d'),(2508,492,'_work_address',''),(2509,492,'__work_address','field_53209806c81da'),(2510,492,'_work_phone',''),(2511,492,'__work_phone','field_5320982dc81db'),(2512,492,'_people_category','Others'),(2513,492,'__people_category','field_530b901b0ca70'),(2514,492,'_education','Ph. D. Ecology UC Davis 2011\r\n\r\nJD Stanford 2009'),(2515,492,'__education','field_530503a0f8e71'),(2516,492,'_research_interests',''),(2517,492,'__research_interests','field_530507cebdd0e'),(2518,492,'_research_projects',''),(2519,492,'__research_projects','field_530507efae298'),(2520,492,'_publications',''),(2521,492,'__publications','field_5305081a9bade'),(2523,199,'rule','a:5:{s:5:\"param\";s:9:\"post_type\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:6:\"people\";s:8:\"order_no\";i:0;s:8:\"group_no\";i:0;}'),(2524,499,'_edit_last','1'),(2525,499,'_edit_lock','1395080858:1'),(2526,499,'_thumbnail_id','83'),(2527,499,'_link_to_profile_tf','0'),(2528,499,'__link_to_profile_tf','field_5323512c57543'),(2529,499,'_profile_picture','494'),(2530,499,'__profile_picture','field_52fbc541cfb03'),(2531,499,'_curriculum_vitae',''),(2532,499,'__curriculum_vitae','field_52fbce8765544'),(2533,499,'_alumni','1'),(2534,499,'__alumni','field_5310d6b20e086'),(2535,499,'_work_position',''),(2536,499,'__work_position','field_5320970428360'),(2537,499,'_email',''),(2538,499,'__email','field_532097d86af9d'),(2539,499,'_work_address',''),(2540,499,'__work_address','field_53209806c81da'),(2541,499,'_work_phone',''),(2542,499,'__work_phone','field_5320982dc81db'),(2543,499,'_people_category','Others'),(2544,499,'__people_category','field_530b901b0ca70'),(2545,499,'_education','Ph.D. Ecology UC Davis 2008'),(2546,499,'__education','field_530503a0f8e71'),(2547,499,'_research_interests',''),(2548,499,'__research_interests','field_530507cebdd0e'),(2549,499,'_research_projects',''),(2550,499,'__research_projects','field_530507efae298'),(2551,499,'_publications',''),(2552,499,'__publications','field_5305081a9bade'),(2553,500,'_edit_last','1'),(2554,500,'_edit_lock','1395080855:1'),(2555,500,'_thumbnail_id','83'),(2556,500,'_link_to_profile_tf','0'),(2557,500,'__link_to_profile_tf','field_5323512c57543'),(2558,500,'_profile_picture','495'),(2559,500,'__profile_picture','field_52fbc541cfb03'),(2560,500,'_curriculum_vitae',''),(2561,500,'__curriculum_vitae','field_52fbce8765544'),(2562,500,'_alumni','1'),(2563,500,'__alumni','field_5310d6b20e086'),(2564,500,'_work_position',''),(2565,500,'__work_position','field_5320970428360'),(2566,500,'_email',''),(2567,500,'__email','field_532097d86af9d'),(2568,500,'_work_address',''),(2569,500,'__work_address','field_53209806c81da'),(2570,500,'_work_phone',''),(2571,500,'__work_phone','field_5320982dc81db'),(2572,500,'_people_category','Others'),(2573,500,'__people_category','field_530b901b0ca70'),(2574,500,'_education','Ph.D. Genetics UC Davis 2010'),(2575,500,'__education','field_530503a0f8e71'),(2576,500,'_research_interests',''),(2577,500,'__research_interests','field_530507cebdd0e'),(2578,500,'_research_projects',''),(2579,500,'__research_projects','field_530507efae298'),(2580,500,'_publications',''),(2581,500,'__publications','field_5305081a9bade'),(2582,501,'_edit_last','1'),(2583,501,'_edit_lock','1395080850:1'),(2584,501,'_thumbnail_id','83'),(2585,501,'_link_to_profile_tf','0'),(2586,501,'__link_to_profile_tf','field_5323512c57543'),(2587,501,'_profile_picture','496'),(2588,501,'__profile_picture','field_52fbc541cfb03'),(2589,501,'_curriculum_vitae',''),(2590,501,'__curriculum_vitae','field_52fbce8765544'),(2591,501,'_alumni','1'),(2592,501,'__alumni','field_5310d6b20e086'),(2593,501,'_work_position',''),(2594,501,'__work_position','field_5320970428360'),(2595,501,'_email',''),(2596,501,'__email','field_532097d86af9d'),(2597,501,'_work_address',''),(2598,501,'__work_address','field_53209806c81da'),(2599,501,'_work_phone',''),(2600,501,'__work_phone','field_5320982dc81db'),(2601,501,'_people_category','Others'),(2602,501,'__people_category','field_530b901b0ca70'),(2603,501,'_education','Ph.D. Ecology UC Davis 2006'),(2604,501,'__education','field_530503a0f8e71'),(2605,501,'_research_interests',''),(2606,501,'__research_interests','field_530507cebdd0e'),(2607,501,'_research_projects',''),(2608,501,'__research_projects','field_530507efae298'),(2609,501,'_publications',''),(2610,501,'__publications','field_5305081a9bade'),(2611,502,'_edit_last','1'),(2612,502,'_edit_lock','1395080841:1'),(2613,503,'_wp_attached_file','2014/03/amy_welsh_profile_image.png'),(2614,503,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:110;s:6:\"height\";i:83;s:4:\"file\";s:35:\"2014/03/amy_welsh_profile_image.png\";s:5:\"sizes\";a:0:{}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(2615,503,'_wp_attachment_image_alt','Amy Welsh Profile Image'),(2616,502,'_thumbnail_id','83'),(2617,502,'_link_to_profile_tf','0'),(2618,502,'__link_to_profile_tf','field_5323512c57543'),(2619,502,'_profile_picture','503'),(2620,502,'__profile_picture','field_52fbc541cfb03'),(2621,502,'_curriculum_vitae',''),(2622,502,'__curriculum_vitae','field_52fbce8765544'),(2623,502,'_alumni','1'),(2624,502,'__alumni','field_5310d6b20e086'),(2625,502,'_work_position',''),(2626,502,'__work_position','field_5320970428360'),(2627,502,'_email',''),(2628,502,'__email','field_532097d86af9d'),(2629,502,'_work_address',''),(2630,502,'__work_address','field_53209806c81da'),(2631,502,'_work_phone',''),(2632,502,'__work_phone','field_5320982dc81db'),(2633,502,'_people_category','Others'),(2634,502,'__people_category','field_530b901b0ca70'),(2635,502,'_education','Ph.D. Ecology UC Davis 2006'),(2636,502,'__education','field_530503a0f8e71'),(2637,502,'_research_interests',''),(2638,502,'__research_interests','field_530507cebdd0e'),(2639,502,'_research_projects',''),(2640,502,'__research_projects','field_530507efae298'),(2641,502,'_publications',''),(2642,502,'__publications','field_5305081a9bade'),(2643,504,'_edit_last','1'),(2644,504,'_edit_lock','1395080833:1'),(2645,504,'_thumbnail_id','83'),(2646,504,'_link_to_profile_tf','0'),(2647,504,'__link_to_profile_tf','field_5323512c57543'),(2648,504,'_profile_picture','497'),(2649,504,'__profile_picture','field_52fbc541cfb03'),(2650,504,'_curriculum_vitae',''),(2651,504,'__curriculum_vitae','field_52fbce8765544'),(2652,504,'_alumni','1'),(2653,504,'__alumni','field_5310d6b20e086'),(2654,504,'_work_position',''),(2655,504,'__work_position','field_5320970428360'),(2656,504,'_email',''),(2657,504,'__email','field_532097d86af9d'),(2658,504,'_work_address',''),(2659,504,'__work_address','field_53209806c81da'),(2660,504,'_work_phone',''),(2661,504,'__work_phone','field_5320982dc81db'),(2662,504,'_people_category','Others'),(2663,504,'__people_category','field_530b901b0ca70'),(2664,504,'_education','Ph.D. Ecology UC Davis 2005'),(2665,504,'__education','field_530503a0f8e71'),(2666,504,'_research_interests',''),(2667,504,'__research_interests','field_530507cebdd0e'),(2668,504,'_research_projects',''),(2669,504,'__research_projects','field_530507efae298'),(2670,504,'_publications',''),(2671,504,'__publications','field_5305081a9bade'),(2672,80,'_link_to_profile_tf','0'),(2673,80,'__link_to_profile_tf','field_5323512c57543'),(2674,408,'_link_to_profile_tf','0'),(2675,408,'__link_to_profile_tf','field_5323512c57543'),(2676,417,'_link_to_profile_tf','0'),(2677,417,'__link_to_profile_tf','field_5323512c57543'),(2678,507,'_edit_last','1'),(2679,507,'_edit_lock','1395335885:1'),(2680,508,'_wp_attached_file','2014/03/brackish_water_hydro_jellyfish_single_image.jpg'),(2681,508,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:214;s:6:\"height\";i:160;s:4:\"file\";s:55:\"2014/03/brackish_water_hydro_jellyfish_single_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:55:\"brackish_water_hydro_jellyfish_single_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(2682,509,'_wp_attached_file','2014/03/brackish_water_hydro_jellyfish_two_image.jpg'),(2683,509,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:215;s:6:\"height\";i:162;s:4:\"file\";s:52:\"2014/03/brackish_water_hydro_jellyfish_two_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:52:\"brackish_water_hydro_jellyfish_two_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(2684,510,'_wp_attached_file','2014/03/brackish_water_hydro_project_image.jpg'),(2685,510,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:214;s:6:\"height\";i:160;s:4:\"file\";s:46:\"2014/03/brackish_water_hydro_project_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:46:\"brackish_water_hydro_project_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(2686,510,'_wp_attachment_image_alt','Brackish water hydros project image of two water hydrozoans'),(2687,509,'_wp_attachment_image_alt','Brackish water hydros jellyfish two image'),(2688,508,'_wp_attachment_image_alt','Brackish water hydro jellyfish single image'),(2689,511,'_wp_attached_file','2014/03/brackish_water_hydros_header_image.png'),(2690,511,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:700;s:6:\"height\";i:120;s:4:\"file\";s:46:\"2014/03/brackish_water_hydros_header_image.png\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:46:\"brackish_water_hydros_header_image-150x120.png\";s:5:\"width\";i:150;s:6:\"height\";i:120;s:9:\"mime-type\";s:9:\"image/png\";}s:6:\"medium\";a:4:{s:4:\"file\";s:45:\"brackish_water_hydros_header_image-300x51.png\";s:5:\"width\";i:300;s:6:\"height\";i:51;s:9:\"mime-type\";s:9:\"image/png\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:46:\"brackish_water_hydros_header_image-624x106.png\";s:5:\"width\";i:624;s:6:\"height\";i:106;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(2691,511,'_wp_attachment_image_alt','Brackish water hydros featured image of two hydrozoans'),(2692,507,'_thumbnail_id','511'),(2693,507,'_project_image','510'),(2694,507,'__project_image','field_530795285c18b'),(2695,507,'_project_excerpt','Invasion biology and molecular ecology of brackish water hydrozoans in the San Francisco Estuary'),(2696,507,'__project_excerpt','field_532099cee4ec6'),(2697,507,'_contact_name','Mariah Meek'),(2698,507,'__contact_name','field_53209a3637a8e'),(2699,507,'_contact_email','mhmeek@ucdavis.edu'),(2700,507,'__contact_email','field_53209a5537a8f'),(2701,507,'is_this_a_post_for_past_projects','0'),(2702,507,'_is_this_a_post_for_past_projects','field_53177905a02ac'),(2703,507,'_background','A suite of four predatory hydrozoan species (Maeotias marginata, Moerisia sp., Blackfordia virginica, and Cordylophora caspia) have invaded and become abundant in the San Francisco Estuary (SFE). Our current level of knowledge regarding the basic biology and ecology of these organisms is alarmingly poor in light of both their possibly negative effect on the SFE ecosystem and the increasing trends in jellyfish blooms around the globe. My research seeks to investigate the potential effects of these species on the SFE ecosystem, to determine the key factors allowing successful establishment and spread of these species, and to predict future effects and spread of the invasions.\r\n\r\n<img class=\"size-full wp-image-508 alignnone\" alt=\"Brackish water hydro jellyfish single image\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/brackish_water_hydro_jellyfish_single_image.jpg\" width=\"214\" height=\"160\" /> <img class=\"size-full wp-image-510 alignnone\" alt=\"Brackish water hydros project image of two water hydrozoans\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/brackish_water_hydro_project_image.jpg\" width=\"214\" height=\"160\" /> <img class=\"size-full wp-image-509 alignnone\" alt=\"Brackish water hydros jellyfish two image\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/brackish_water_hydro_jellyfish_two_image.jpg\" width=\"215\" height=\"162\" />\r\n<p style=\"margin-left: auto; margin-right: auto;\">This research project involves two interrelated components:</p>\r\n\r\n<h2 style=\"margin-left: auto; margin-right: auto;\">1.<span style=\"text-decoration: underline;\">Genetic</span><span style=\"text-decoration: underline;\"> studies:</span></h2>\r\n<p style=\"text-align: left;\"><strong>Marker Development</strong></p>\r\n<p style=\"text-align: left;\">I developed a suite of microsatellite genetic markers for <em>M. marginata</em> and <em>Moerisia sp.</em> to investigate population genetic diversity and reproductive mode in these invasions. See <a title=\"Springer_link_Meek_et_al_2010\" href=\"http://link.springer.com/article/10.1007%2Fs12686-009-9050-7?LI=true#page-1\">Meek et al. 2010</a></p>\r\n<p style=\"text-align: left;\"><strong>Evaluation of clonal diversity and mode of reproduction</strong></p>\r\n<p style=\"text-align: left;\">Life history characteristics and reproductive strategy may play a significant role in invasion success and spread. Asexual reproduction can confer an advantage during the establishment and spread of an invasive species, as asexually reproducing species can avoid the demographic constraints of small population size during the beginning of the invasion. Additionally, the potential for preservation of co-adapted gene complexes with high phenotypic plasticity and broad tolerance ranges is greater under asexual reproduction. However, asexually reproducing invasive species may sometimes be less successful due to low amounts of genetic variation, limited recombination, and reduced ability to purge deleterious mutations.</p>\r\n<p style=\"text-align: left;\">In my PhD work, I investigated:</p>\r\n\r\n<ul style=\"text-align: left;\">\r\n	<li>The level of genetic diversity in the invasive populations of <em>Moerisia sp.</em> and <em>M. marginata</em> in the San Francisco Estuary</li>\r\n	<li>The relative contributions of asexual and sexual reproduction to these invasions.</li>\r\n</ul>\r\n<p style=\"text-align: left;\">I employed the microsatellite markers I developed (see above) and medusae and polyp samples collected from throughout Suisun Marsh. With this research, I am working to understand how the ability to reproduce both asexually and sexually aids in the establishment and spread of these invasive species. See <a title=\"Meek_et_al_2013.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Mariahs_Jellyfish_files/Meek%20et%20al%202013.pdf\">Meek et al. 2013</a></p>\r\n\r\n<h2 style=\"text-align: left;\">2. <span style=\"text-decoration: underline;\">Laboratory</span><span style=\"text-decoration: underline;\"> studies to determine feeding rates and ecophysiological characteristics:</span></h2>\r\n<p style=\"text-align: left;\">Jellyfish and other hydrozoans have proven to be important, yet understudied, predators in marine and estuarine systems. According to the Bad Suisun Bay Hypothesis, invasive species have changed the food web in the brackish water habitats of the SFE. We do not currently understand how the invasive hydrozoans in the SFE fit into the trophic ecology of the estuary. In order to assess their potential impact, it is important to understand the basic feeding biology of these species.</p>\r\n<p style=\"text-align: left;\">In order to begin to understand the trophic ecology of the invasive hydrozoans in the SFE, in collaboration with Alpa Wintzer, I have conducted a series of experiments to determine potential feeding rates and the factors that impact feeding rates. Our question is</p>\r\n\r\n<ul>\r\n	<li>How does time to feed and prey density impact feeding rates in <em>M. marginata</em> and <em>Moerisia sp.</em> medusae?</li>\r\n	<li>Additionally, we have conducted a series of experiments using <em>C. caspia</em> polyps we cultured in the lab to look at the effect of temperature and salinity on growth rates and their combined effect on feeding rates. With this work, we are addressing the following questions:</li>\r\n	<li>What are the reproductive rates of <em>C. caspia</em> under different temperature and salinity conditions?</li>\r\n	<li>How do different temperature and salinity conditions affect feeding rates in <em>C. caspia</em>?</li>\r\n</ul>\r\n<p style=\"text-align: left;\">We are synthesizing the information gained from this work with field collected gut content data to get a better understanding of what these invasives eat, how much and how fast they can prey upon zooplankton, and what the effects of differing temperature and salinity conditions may have on growth and consumption rates. See <a title=\"Plos_one_Meek_et_al_2012\" href=\"http://www.plosone.org/article/info:doi/10.1371/journal.pone.0046373\">Meek et al. 2012</a>.</p>\r\n<p style=\"text-align: left;\">The expected outcome of this work will be a clearer understanding of the effects of several abundant and novel hydrozoan predators in the system and the role of reproductive mode and genetic diversity in the invasions. Through this work, I will gain a better understanding of what makes these invasives successful and predict how the invasion may spread and adapt in the years to come.</p>'),(2704,507,'__background','field_53079733f3610'),(2705,507,'_materials_and_methods',''),(2706,507,'__materials_and_methods','field_5307975cdf5ef'),(2707,507,'_results',''),(2708,507,'__results','field_53079783f8e53'),(2709,507,'_future_work',''),(2710,507,'__future_work','field_530797af2f229'),(2711,507,'_collaborators',''),(2712,507,'__collaborators','field_530797d1e08ae'),(2713,507,'_literature_cited',''),(2714,507,'__literature_cited','field_530797f0836ce'),(2715,122,'_project_excerpt',''),(2716,122,'__project_excerpt','field_532099cee4ec6'),(2717,122,'_contact_name',''),(2718,122,'__contact_name','field_53209a3637a8e'),(2719,122,'_contact_email',''),(2720,122,'__contact_email','field_53209a5537a8f'),(2721,512,'_edit_last','1'),(2722,512,'_edit_lock','1402346565:1'),(2723,513,'_wp_attached_file','2014/03/genetic_monitoring_sturgeon_project_image.jpg'),(2724,513,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:343;s:6:\"height\";i:258;s:4:\"file\";s:53:\"2014/03/genetic_monitoring_sturgeon_project_image.jpg\";s:5:\"sizes\";a:2:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:53:\"genetic_monitoring_sturgeon_project_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}s:6:\"medium\";a:4:{s:4:\"file\";s:53:\"genetic_monitoring_sturgeon_project_image-300x225.jpg\";s:5:\"width\";i:300;s:6:\"height\";i:225;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(2725,513,'_wp_attachment_image_alt','Genetic monitoring sturgeon project image'),(2726,514,'_wp_attached_file','2014/03/white_sturgeo_header_image.png'),(2727,514,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:700;s:6:\"height\";i:120;s:4:\"file\";s:38:\"2014/03/white_sturgeo_header_image.png\";s:5:\"sizes\";a:3:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:38:\"white_sturgeo_header_image-150x120.png\";s:5:\"width\";i:150;s:6:\"height\";i:120;s:9:\"mime-type\";s:9:\"image/png\";}s:6:\"medium\";a:4:{s:4:\"file\";s:37:\"white_sturgeo_header_image-300x51.png\";s:5:\"width\";i:300;s:6:\"height\";i:51;s:9:\"mime-type\";s:9:\"image/png\";}s:14:\"post-thumbnail\";a:4:{s:4:\"file\";s:38:\"white_sturgeo_header_image-624x106.png\";s:5:\"width\";i:624;s:6:\"height\";i:106;s:9:\"mime-type\";s:9:\"image/png\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";i:0;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:0:\"\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:0;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";i:0;s:3:\"iso\";i:0;s:13:\"shutter_speed\";i:0;s:5:\"title\";s:0:\"\";}}'),(2728,514,'_wp_attachment_image_alt','White sturgeon projects featured image of a white sturgeon'),(2729,512,'_thumbnail_id','514'),(2730,512,'_project_image','513'),(2731,512,'__project_image','field_530795285c18b'),(2732,512,'_project_excerpt',''),(2733,512,'__project_excerpt','field_532099cee4ec6'),(2734,512,'_contact_name',''),(2735,512,'__contact_name','field_53209a3637a8e'),(2736,512,'_contact_email',''),(2737,512,'__contact_email','field_53209a5537a8f'),(2738,512,'is_this_a_post_for_past_projects','0'),(2739,512,'_is_this_a_post_for_past_projects','field_53177905a02ac'),(2740,512,'_background','The ESA-listed Kootenai River white sturgeon population has suffered from negligible recruitment for the past 20-30 years. The Kootenai Tribe of Idaho initiated a conservation aquaculture program for this population in 1993 to maintain it in the absence of natural recruitment. Currently, nearly all recruitment is thought to be derived from the aquaculture program. Because tissue samples have been archived for most adult sturgeon brought into the hatchery, the Tribe has a tissue archive of all or nearly all parents contributing offspring since the 1990s.\r\n\r\n<img class=\"aligncenter size-full wp-image-513\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/genetic_monitoring_sturgeon_project_image.jpg\" alt=\"Genetic monitoring sturgeon project image\" width=\"343\" height=\"258\" />\r\n\r\n&nbsp;\r\n\r\nThe Genome Variation Lab (GVL) is continuing work with the Kootenai Tribe, using genetic techniques to inform the management of the Kootenai River white sturgeon population. We are collecting microsatellite genotype data on all adult white sturgeon sampled by the Tribe in order to monitor changes in genetic diversity over time. The Tribe is striving to maximize genetic diversity in their conservation aquaculture program in order to preserve the populations ability to adapt to changing environmental conditions. Additionally, we are using microsatellite data to conduct parentage analysis to determine family relationships among all adult sturgeon brought into the conservation aquaculture program as broodstock. By understanding how potential broodstock are related to each other, the Tribe can avoid inbreeding, or the crossing of close relatives that may reduce the fitness of the endangered population.\r\n\r\nPrevious work in the GVL used eight microsatellite loci to assign parentage full-sibling families whose parents were known. Low levels of genetic variation resulted in suboptimal assignment accuracy. We have optimized nine additional microsatellite loci and used these in combination with the previous eight loci to increase the accuracy of parentage assignment within known full-sib families produced by the Kootenai hatchery.\r\n\r\n&nbsp;\r\n\r\nPrevious work in the GVL used eight microsatellite loci to assign parentage full-sibling families whose parents were known. Low levels of genetic variation resulted in suboptimal assignment accuracy. We have optimized nine additional microsatellite loci and used these in combination with the previous eight loci to increase the accuracy of parentage assignment within known full-sib families produced by the Kootenai hatchery\r\n\r\nA third aspect of genetic monitoring includes monitoring of genome size. The recent discovery of individuals of abnormal ploidy in a commercially farmed population of white sturgeon indicates that some aspect of sturgeon culture is unintentionally inducing spontaneous autopolyploidy. As sturgeon culture techniques are similar for commercial and conservation aquaculture, this finding raised concerns that genome duplication could be induced in the Kootenai River conservation aquaculture program. We are currently using flow cytometry to monitor genome size of broodstock and offspring of the 2011 and 2012 year classes in the Kootenai hatchery.'),(2741,512,'__background','field_53079733f3610'),(2742,512,'_materials_and_methods',''),(2743,512,'__materials_and_methods','field_5307975cdf5ef'),(2744,512,'_results','The results of the genetic monitoring work are described in an open access peer reviewed publication in the journal Endangered Species Research.<a title=\"Endangered_species_research_Kootenai_River_white_sturgeon\" href=\"http://www.int-res.com/abstracts/esr/v16/n1/p65-75/\">http://www.int-res.com/abstracts/esr/v16/n1/p65-75/</a>\r\n\r\nThe genome size monitoring aspect of this project is still ongoing.'),(2745,512,'__results','field_53079783f8e53'),(2746,512,'_future_work',''),(2747,512,'__future_work','field_530797af2f229'),(2748,512,'_collaborators',''),(2749,512,'__collaborators','field_530797d1e08ae'),(2750,512,'_literature_cited',''),(2751,512,'__literature_cited','field_530797f0836ce'),(2752,515,'_edit_last','1'),(2753,515,'_edit_lock','1402341628:1'),(2754,515,'_thumbnail_id','514'),(2755,515,'_project_image',''),(2756,515,'__project_image','field_530795285c18b'),(2757,515,'_project_excerpt',''),(2758,515,'__project_excerpt','field_532099cee4ec6'),(2759,515,'_contact_name',''),(2760,515,'__contact_name','field_53209a3637a8e'),(2761,515,'_contact_email',''),(2762,515,'__contact_email','field_53209a5537a8f'),(2763,515,'is_this_a_post_for_past_projects','0'),(2764,515,'_is_this_a_post_for_past_projects','field_53177905a02ac'),(2765,515,'_background','White sturgeon are large, diadromous fishes with a high dispersal capability that were presumed to exhibit some degree of spawning site fidelity. Spawning is known to occur in the Sacramento-San Joaquin, Columbia-Snake, and Fraser River drainages, although white sturgeon may be encountered in marine habitat or coastal streams outside of their natal drainage. Because of the complex nature of the duplicated white sturgeon genome, many questions regarding the population structure and dispersal behavior of white sturgeon remained unanswered. In the project, thirteen microsatellite loci were used to examine white sturgeon population structure both within and among drainages known to support spawning populations.'),(2766,515,'__background','field_53079733f3610'),(2767,515,'_materials_and_methods',''),(2768,515,'__materials_and_methods','field_5307975cdf5ef'),(2769,515,'_results','I used the program Structure (Pritchard et al. 2000) to infer the number of K populations both within each drainage (Sacramento-San Joaquin, Columbia-Snake, Fraser) and among drainages across the species range. Different patterns of population structure were revealed in each drainage. A single population was identified in the Sacramento-San Joaquin system despite that fact that spawning has been documented on both the Sacramento and San Joaquin Rivers. A complex isolation by distance pattern was observed in the Columbia-Snake drainage, with local gene flow among geographically proximate spawning sites but higher levels of genetic divergence among samples collected at opposite ends of the drainage. The genetic distinctness of the Kootenai River population within the Columbia-Snake was confirmed. Interestingly, the Fraser River system exhibited a hierarchical pattern of population structure, with strong levels of genetic divergence on either side of a putative phylogeographic barrier (Hells Gate) and additional substructure above that barrier. Distinct spawning populations in the Fraser River (above Hells Gate) exist in the absence of any physical barrier suggesting that white sturgeon exhibit stronger spawning site fidelity in that system. Additionally, the federally listed Upper Fraser River population was revealed to consist of a mixture of individuals from the Middle Fraser and Nechako River populations.\r\n\r\nPopulation genetic analysis of white sturgeon samples collected in marine habitat or non-natal estuaries sought to elucidate marine migration behavior of white sturgeon. Although long-distance marine migrations of white sturgeon are known from mark-recapture data, it is unknown whether all populations with access to the ocean (Sacramento-San Joaquin, Lower Columbia, Lower Fraser) equally contribute marine migrants. More extensive analysis of marine migrants and potential source populations should better elucidate marine habitat use by this species.\r\n\r\nA more detailed description of Fraser River population structure will soon be available in a manuscript in press in the Canadian Journal of Fisheries and Aquatic Sciences. A second manuscript describing population structure within the Sacramento-San Joaquin and Columbia-Snake drainages as well as among-drainage patterns of structure has been submitted to the Transactions of the American Fisheries Society.'),(2770,515,'__results','field_53079783f8e53'),(2771,515,'_future_work',''),(2772,515,'__future_work','field_530797af2f229'),(2773,515,'_collaborators',''),(2774,515,'__collaborators','field_530797d1e08ae'),(2775,515,'_literature_cited',''),(2776,515,'__literature_cited','field_530797f0836ce'),(2777,516,'_edit_last','1'),(2778,516,'_edit_lock','1402601805:1'),(2779,517,'_wp_attached_file','2014/03/inference-_of_ploidy_many_sturgeons_image.jpg'),(2780,517,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:242;s:6:\"height\";i:181;s:4:\"file\";s:53:\"2014/03/inference-_of_ploidy_many_sturgeons_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:53:\"inference-_of_ploidy_many_sturgeons_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";d:2.7999999999999998;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:8:\"DMC-FX12\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:1282119300;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";s:3:\"5.8\";s:3:\"iso\";s:3:\"250\";s:13:\"shutter_speed\";s:17:\"0.033333333333333\";s:5:\"title\";s:0:\"\";}}'),(2781,518,'_wp_attached_file','2014/03/inference_of_ploidy_many_sturgeons_closeup_image.jpg'),(2782,518,'_wp_attachment_metadata','a:5:{s:5:\"width\";i:244;s:6:\"height\";i:183;s:4:\"file\";s:60:\"2014/03/inference_of_ploidy_many_sturgeons_closeup_image.jpg\";s:5:\"sizes\";a:1:{s:9:\"thumbnail\";a:4:{s:4:\"file\";s:60:\"inference_of_ploidy_many_sturgeons_closeup_image-150x150.jpg\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:10:\"image/jpeg\";}}s:10:\"image_meta\";a:10:{s:8:\"aperture\";d:2.7999999999999998;s:6:\"credit\";s:0:\"\";s:6:\"camera\";s:8:\"DMC-FX12\";s:7:\"caption\";s:0:\"\";s:17:\"created_timestamp\";i:1282119891;s:9:\"copyright\";s:0:\"\";s:12:\"focal_length\";s:3:\"5.8\";s:3:\"iso\";s:3:\"125\";s:13:\"shutter_speed\";s:17:\"0.033333333333333\";s:5:\"title\";s:0:\"\";}}'),(2783,517,'_wp_attachment_image_alt','Inference of ploidy project image of many sturgeons in swimming'),(2784,518,'_wp_attachment_image_alt','Inference of ploidy many sturgeons closeup image'),(2785,516,'_thumbnail_id','514'),(2786,516,'_project_image','517'),(2787,516,'__project_image','field_530795285c18b'),(2788,516,'_project_excerpt',''),(2789,516,'__project_excerpt','field_532099cee4ec6'),(2790,516,'_contact_name',''),(2791,516,'__contact_name','field_53209a3637a8e'),(2792,516,'_contact_email',''),(2793,516,'__contact_email','field_53209a5537a8f'),(2794,516,'is_this_a_post_for_past_projects','0'),(2795,516,'_is_this_a_post_for_past_projects','field_53177905a02ac'),(2796,516,'_background','Sturgeon belong to the family Acipenseridae, which is thought to be derived from an ancestor with 60 chromosomes. Within extant sturgeon species, various levels of ploidy have been inferred. Some authors propose that 120 chromosome species should be considered diploid, while others suggest they are ancient tetraploids. The same goes for the 250 chromosome group, with some scientists maintaining these species are tetraploid while others have inferred octoploidy. Inheritance data can be used to infer chromosome behavior at meiosis, from which ploidy level can be inferred. I sampled half- and full-sibling white sturgeon families from a caviar farm in central California. I genotyped eight parents and 48 offspring (N = 720) from each of 15 families of known parentage to determine mode of inheritance and gene copy number for eight microsatellite loci.\r\n\r\n<img class=\"size-full wp-image-518 alignleft\" alt=\"Inference of ploidy many sturgeons closeup image\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/inference_of_ploidy_many_sturgeons_closeup_image.jpg\" width=\"244\" height=\"183\" /> <img class=\"size-full wp-image-517 alignleft\" alt=\"Inference of ploidy project image of many sturgeons in swimming\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/inference-_of_ploidy_many_sturgeons_image.jpg\" width=\"242\" height=\"181\" />'),(2797,516,'__background','field_53079733f3610'),(2798,516,'_materials_and_methods',''),(2799,516,'__materials_and_methods','field_5307975cdf5ef'),(2800,516,'_results','My data suggest that the white sturgeon is an ancient octoploid. This is corroborated by previous GVL research on white sturgeon inheritance, which found that several microsatellites in this organism were present in more than four copies. We also discovered alternative ploidy states within the farmed population and future work will examine the prevalence, cause, and implications of spontaneous polyploidy in farmed white sturgeon. This work was published in 2011 in the Journal of Applied Ichthyology (<a title=\"2011_Journal_of_applied_ichthyology_white_sturgeons\" href=\"http://onlinelibrary.wiley.com/doi/10.1111/j.1439-0426.2011.01873.x/abstract\">http://onlinelibrary.wiley.com/doi/10.1111/j.1439-0426.2011.01873.x/abstract</a>)\r\n\r\nAnother impetus for doing this study was to ascertain whether these eight microsatellite loci were appropriate to use in molecular ecological studies of white sturgeon. Because of the high gene copy number in white sturgeon, it is impossible to apply standard population genetic tests for Hardy Weinberg expectations or linkage equilibrium. Microsatellite alleles must be scored as dominant present/absence loci, as dosage cannot be scored from electrophoretic data. By following the transmission of alleles from parents to offspring in 15 half-sibling families of known parentage, we were able to determine that with the exception of one locus tested, these newly developed microsatellite loci conform to Mendelian expectations and were appropriate for use in population genetic analysis.'),(2801,516,'__results','field_53079783f8e53'),(2802,516,'_future_work',''),(2803,516,'__future_work','field_530797af2f229'),(2804,516,'_collaborators',''),(2805,516,'__collaborators','field_530797d1e08ae'),(2806,516,'_literature_cited',''),(2807,516,'__literature_cited','field_530797f0836ce'),(2818,526,'ml-slider_settings','a:35:{s:4:\"type\";s:4:\"coin\";s:6:\"random\";s:5:\"false\";s:8:\"cssClass\";s:0:\"\";s:8:\"printCss\";s:4:\"true\";s:7:\"printJs\";s:4:\"true\";s:5:\"width\";s:0:\"\";s:6:\"height\";s:3:\"298\";s:3:\"spw\";s:1:\"7\";s:3:\"sph\";s:1:\"5\";s:5:\"delay\";s:4:\"3000\";s:6:\"sDelay\";s:2:\"30\";s:7:\"opacity\";s:3:\"0.7\";s:10:\"titleSpeed\";s:3:\"500\";s:6:\"effect\";s:6:\"random\";s:10:\"navigation\";b:1;s:5:\"links\";s:4:\"true\";s:10:\"hoverPause\";s:4:\"true\";s:5:\"theme\";s:7:\"default\";s:9:\"direction\";s:10:\"horizontal\";s:7:\"reverse\";s:5:\"false\";s:14:\"animationSpeed\";s:3:\"600\";s:8:\"prevText\";s:1:\"<\";s:8:\"nextText\";s:1:\">\";s:6:\"slices\";s:2:\"15\";s:6:\"center\";s:5:\"false\";s:9:\"smartCrop\";s:4:\"true\";s:12:\"carouselMode\";s:5:\"false\";s:14:\"carouselMargin\";s:1:\"5\";s:6:\"easing\";s:6:\"linear\";s:8:\"autoPlay\";s:5:\"false\";s:11:\"thumb_width\";i:150;s:12:\"thumb_height\";i:100;s:9:\"fullWidth\";s:5:\"false\";s:10:\"noConflict\";s:5:\"false\";s:12:\"smoothHeight\";s:5:\"false\";}'),(2819,204,'ml-slider_type','image'),(2820,204,'_wp_attachment_backup_sizes','a:4:{s:15:\"resized-150x150\";a:5:{s:4:\"path\";s:89:\"/Users/psalessi/Sites/Genetics_Lab/wp-content/uploads/2014/02/director_header-150x150.png\";s:4:\"file\";s:27:\"director_header-150x150.png\";s:5:\"width\";i:150;s:6:\"height\";i:120;s:9:\"mime-type\";s:9:\"image/png\";}s:15:\"resized-700x300\";a:5:{s:4:\"path\";s:89:\"/Users/psalessi/Sites/Genetics_Lab/wp-content/uploads/2014/02/director_header-700x300.png\";s:4:\"file\";s:27:\"director_header-700x300.png\";s:5:\"width\";i:700;s:6:\"height\";i:120;s:9:\"mime-type\";s:9:\"image/png\";}s:15:\"resized-280x120\";a:5:{s:4:\"path\";s:89:\"/Users/psalessi/Sites/Genetics_Lab/wp-content/uploads/2014/02/director_header-280x120.png\";s:4:\"file\";s:27:\"director_header-280x120.png\";s:5:\"width\";i:280;s:6:\"height\";i:120;s:9:\"mime-type\";s:9:\"image/png\";}s:13:\"resized-0x120\";a:5:{s:4:\"path\";s:87:\"/Users/psalessi/Sites/Genetics_Lab/wp-content/uploads/2014/02/director_header-0x120.png\";s:4:\"file\";s:25:\"director_header-0x120.png\";s:5:\"width\";i:700;s:6:\"height\";i:120;s:9:\"mime-type\";s:9:\"image/png\";}}'),(2821,20,'ml-slider_type','image'),(2822,20,'_wp_attachment_backup_sizes','a:2:{s:15:\"resized-424x182\";a:5:{s:4:\"path\";s:106:\"/Users/psalessi/Sites/Genetics_Lab/wp-content/uploads/2014/02/cropped-genetics_lab_home_header-424x182.png\";s:4:\"file\";s:44:\"cropped-genetics_lab_home_header-424x182.png\";s:5:\"width\";i:424;s:6:\"height\";i:182;s:9:\"mime-type\";s:9:\"image/png\";}s:13:\"resized-0x182\";a:5:{s:4:\"path\";s:104:\"/Users/psalessi/Sites/Genetics_Lab/wp-content/uploads/2014/02/cropped-genetics_lab_home_header-0x182.png\";s:4:\"file\";s:42:\"cropped-genetics_lab_home_header-0x182.png\";s:5:\"width\";i:700;s:6:\"height\";i:182;s:9:\"mime-type\";s:9:\"image/png\";}}'),(2823,527,'slidedeck_source','posts'),(2824,527,'slidedeck_lens','tool-kit'),(2825,527,'slidedeck_options','a:62:{s:12:\"total_slides\";i:5;s:4:\"size\";s:6:\"medium\";s:5:\"width\";i:500;s:6:\"height\";i:500;s:8:\"overlays\";s:5:\"hover\";s:13:\"overlays_open\";b:0;s:16:\"show-front-cover\";b:0;s:15:\"show-back-cover\";b:0;s:11:\"accentColor\";s:7:\"#3ea0c1\";s:9:\"titleFont\";s:10:\"sans-serif\";s:8:\"bodyFont\";s:10:\"sans-serif\";s:9:\"hyphenate\";b:0;s:12:\"activeCorner\";b:0;s:10:\"hideSpines\";b:1;s:13:\"image_scaling\";s:5:\"cover\";s:14:\"cache_duration\";i:30;s:11:\"show-author\";b:1;s:18:\"show-author-avatar\";b:1;s:14:\"linkAuthorName\";b:0;s:15:\"show-link-slide\";b:0;s:11:\"date-format\";s:7:\"timeago\";s:21:\"titleLengthWithImages\";i:50;s:24:\"titleLengthWithoutImages\";i:35;s:10:\"show-title\";b:1;s:9:\"linkTitle\";b:1;s:23:\"excerptLengthWithImages\";i:100;s:26:\"excerptLengthWithoutImages\";i:200;s:12:\"show-excerpt\";b:1;s:13:\"show-readmore\";b:1;s:10:\"linkTarget\";s:6:\"_blank\";s:18:\"display-nav-arrows\";s:5:\"hover\";s:4:\"keys\";b:1;s:6:\"scroll\";b:0;s:5:\"touch\";b:0;s:17:\"continueScrolling\";b:0;s:14:\"touchThreshold\";i:100;s:9:\"indexType\";s:7:\"numbers\";s:5:\"start\";i:1;s:9:\"randomize\";b:0;s:8:\"autoPlay\";b:0;s:16:\"autoPlayInterval\";i:5;s:5:\"cycle\";b:1;s:15:\"slideTransition\";s:5:\"slide\";s:5:\"speed\";i:750;s:10:\"transition\";s:5:\"swing\";s:18:\"preferredImageSize\";s:4:\"auto\";s:16:\"postsImageSource\";s:7:\"content\";s:9:\"post_type\";s:4:\"post\";s:14:\"post_type_sort\";s:6:\"recent\";s:12:\"filter_terms\";a:0:{}s:13:\"filter_by_tax\";b:0;s:13:\"query_any_all\";s:3:\"any\";s:23:\"use-custom-post-excerpt\";b:0;s:5:\"frame\";s:5:\"frame\";s:13:\"text-position\";s:13:\"title-pos-top\";s:10:\"text-color\";s:10:\"title-dark\";s:14:\"lensVariations\";s:4:\"dark\";s:19:\"navigation-position\";s:14:\"nav-pos-bottom\";s:15:\"navigation-type\";s:8:\"nav-dots\";s:16:\"navigation-style\";s:11:\"nav-default\";s:11:\"arrow-style\";s:12:\"arrowstyle-7\";s:15:\"nav-arrow-style\";s:17:\"nav-arrow-style-1\";}'),(2826,528,'_edit_lock','1401999267:1'),(2827,529,'_edit_lock','1401999821:1'),(2828,518,'_wp_attachment_url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/inference_of_ploidy_many_sturgeons_closeup_image.jpg'),(2829,517,'_wp_attachment_url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/inference-_of_ploidy_many_sturgeons_image.jpg'),(2830,514,'_wp_attachment_url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/white_sturgeo_header_image.png'),(2831,513,'_wp_attachment_url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/genetic_monitoring_sturgeon_project_image.jpg'),(2832,511,'_wp_attachment_url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/brackish_water_hydros_header_image.png'),(2833,510,'_wp_attachment_url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/brackish_water_hydro_project_image.jpg'),(2834,509,'_wp_attachment_url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/brackish_water_hydro_jellyfish_two_image.jpg'),(2835,508,'_wp_attachment_url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/brackish_water_hydro_jellyfish_single_image.jpg'),(2836,503,'_wp_attachment_url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/amy_welsh_profile_image.png'),(2837,497,'_wp_attachment_url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/yonjiu_chen_profile_image.png'),(2838,496,'_wp_attachment_url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/rick_topinka_profile_image.png'),(2839,495,'_wp_attachment_url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/rachel_simmons_profile_image.png'),(2840,494,'_wp_attachment_url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/rachel_schwartz_profile_image.png'),(2841,493,'_wp_attachment_url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/karrigan_bork_profile_image.png'),(2842,491,'_wp_attachment_url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/k_jun_bando_profile_image.jpg'),(2843,488,'_wp_attachment_url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/daphne_gille_turtle_on_beach_image.jpg'),(2844,486,'_wp_attachment_url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/daphne_gille_seal_image.jpg'),(2845,487,'_wp_attachment_url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/daphne_gille_seal_on_beach_image.jpg'),(2846,485,'_wp_attachment_url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/daphne_gille_profile_image.jpg'),(2847,482,'_wp_attachment_url','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/brian_mahardja_profile_image.jpg'),(2848,529,'_edit_last','1'),(2849,529,'slides','a:2:{i:0;a:4:{s:10:\"attachment\";s:3:\"518\";s:5:\"label\";s:10:\"Change ME!\";s:4:\"type\";s:5:\"image\";s:5:\"image\";a:7:{s:2:\"id\";s:3:\"518\";s:3:\"url\";s:119:\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/inference_of_ploidy_many_sturgeons_closeup_image.jpg\";s:3:\"alt\";s:48:\"Inference of ploidy many sturgeons closeup image\";s:4:\"link\";s:119:\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/inference_of_ploidy_many_sturgeons_closeup_image.jpg\";s:7:\"caption\";s:0:\"\";s:5:\"sizes\";a:5:{s:9:\"thumbnail\";s:127:\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/inference_of_ploidy_many_sturgeons_closeup_image-150x150.jpg\";s:6:\"medium\";s:119:\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/inference_of_ploidy_many_sturgeons_closeup_image.jpg\";s:5:\"large\";s:119:\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/inference_of_ploidy_many_sturgeons_closeup_image.jpg\";s:4:\"full\";s:119:\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/inference_of_ploidy_many_sturgeons_closeup_image.jpg\";s:14:\"post-thumbnail\";s:119:\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/inference_of_ploidy_many_sturgeons_closeup_image.jpg\";}s:5:\"title\";s:48:\"Inference of Ploidy Many Sturgeons Closeup Image\";}}i:1;a:4:{s:10:\"attachment\";s:3:\"518\";s:5:\"label\";s:11:\"Change ME!2\";s:4:\"type\";s:5:\"image\";s:5:\"image\";a:7:{s:2:\"id\";s:3:\"518\";s:3:\"url\";s:119:\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/inference_of_ploidy_many_sturgeons_closeup_image.jpg\";s:3:\"alt\";s:48:\"Inference of ploidy many sturgeons closeup image\";s:4:\"link\";s:119:\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/inference_of_ploidy_many_sturgeons_closeup_image.jpg\";s:7:\"caption\";s:0:\"\";s:5:\"sizes\";a:5:{s:9:\"thumbnail\";s:127:\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/inference_of_ploidy_many_sturgeons_closeup_image-150x150.jpg\";s:6:\"medium\";s:119:\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/inference_of_ploidy_many_sturgeons_closeup_image.jpg\";s:5:\"large\";s:119:\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/inference_of_ploidy_many_sturgeons_closeup_image.jpg\";s:4:\"full\";s:119:\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/inference_of_ploidy_many_sturgeons_closeup_image.jpg\";s:14:\"post-thumbnail\";s:119:\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/inference_of_ploidy_many_sturgeons_closeup_image.jpg\";}s:5:\"title\";s:48:\"Inference of Ploidy Many Sturgeons Closeup Image\";}}}'),(2850,529,'skin','default'),(2851,529,'options','a:17:{s:9:\"animation\";s:5:\"slide\";s:9:\"slideshow\";b:1;s:12:\"h_responsive\";b:1;s:6:\"height\";s:0:\"\";s:12:\"w_responsive\";b:1;s:5:\"width\";s:0:\"\";s:9:\"direction\";s:10:\"horizontal\";s:11:\"cycle_speed\";s:1:\"5\";s:15:\"animation_speed\";s:3:\"0.6\";s:11:\"control_nav\";b:1;s:13:\"direction_nav\";b:1;s:12:\"keyboard_nav\";b:1;s:9:\"touch_nav\";b:1;s:11:\"caption_box\";b:1;s:8:\"linkable\";b:1;s:14:\"pause_on_hover\";b:1;s:20:\"thumbnail_navigation\";b:1;}'),(2870,10,'content_slider','1'),(2871,277,'content_slider','1'),(2872,30,'content_slider',''),(2873,543,'_edit_lock','1402003180:1'),(2874,543,'_edit_last','1'),(2875,543,'_ssd_gallery_images','[\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/contact_header.png\",\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/white_sturgeo_header_image.png\",\"\",\"\",\"\"]'),(2876,543,'_ssd_caption_text','[\"This is awesome\",\"Woohoooo\",\"\",\"\",\"\"]'),(2877,543,'_ssd_autoplay',''),(2878,543,'_ssd_timeout','4000'),(2879,543,'_ssd_speed','500'),(2880,543,'_ssd_pager','enabled'),(2881,543,'_ssd_nav','enabled'),(2882,543,'_ssd_rand',''),(2883,543,'_ssd_pause',''),(2884,543,'_ssd_pauseControls',''),(2885,543,'_ssd_prev','Previous'),(2886,543,'_ssd_next','Next'),(2887,543,'_ssd_id','rslides'),(2888,543,'_ssd_namespace','rslides'),(2889,543,'_ssd_select','eclipse'),(2890,543,'_ssd_pagination','enabled'),(2891,547,'_edit_lock','1402603000:1'),(2892,547,'_edit_last','1'),(2901,547,'_cycloneslider_metas','a:3:{i:0;a:26:{s:2:\"id\";i:272;s:4:\"type\";s:5:\"image\";s:4:\"link\";s:0:\"\";s:5:\"title\";s:3:\"Sup\";s:11:\"description\";s:12:\"Yeah rock on\";s:11:\"link_target\";s:5:\"_self\";s:7:\"img_alt\";s:0:\"\";s:9:\"img_title\";s:0:\"\";s:20:\"enable_slide_effects\";i:0;s:2:\"fx\";s:7:\"default\";s:5:\"speed\";s:0:\"\";s:7:\"timeout\";s:0:\"\";s:10:\"tile_count\";s:1:\"7\";s:10:\"tile_delay\";s:3:\"100\";s:13:\"tile_vertical\";s:4:\"true\";s:11:\"video_thumb\";s:0:\"\";s:9:\"video_url\";s:0:\"\";s:5:\"video\";s:0:\"\";s:6:\"custom\";s:0:\"\";s:11:\"youtube_url\";s:0:\"\";s:15:\"youtube_related\";s:5:\"false\";s:9:\"vimeo_url\";s:0:\"\";s:11:\"testimonial\";s:0:\"\";s:18:\"testimonial_author\";s:0:\"\";s:16:\"testimonial_link\";s:0:\"\";s:23:\"testimonial_link_target\";s:5:\"_self\";}i:1;a:26:{s:2:\"id\";i:160;s:4:\"type\";s:5:\"image\";s:4:\"link\";s:0:\"\";s:5:\"title\";s:9:\"Hey there\";s:11:\"description\";s:13:\"fffffffriends\";s:11:\"link_target\";s:5:\"_self\";s:7:\"img_alt\";s:0:\"\";s:9:\"img_title\";s:0:\"\";s:20:\"enable_slide_effects\";i:0;s:2:\"fx\";s:7:\"default\";s:5:\"speed\";s:0:\"\";s:7:\"timeout\";s:0:\"\";s:10:\"tile_count\";s:1:\"7\";s:10:\"tile_delay\";s:3:\"100\";s:13:\"tile_vertical\";s:4:\"true\";s:11:\"video_thumb\";s:0:\"\";s:9:\"video_url\";s:0:\"\";s:5:\"video\";s:0:\"\";s:6:\"custom\";s:0:\"\";s:11:\"youtube_url\";s:0:\"\";s:15:\"youtube_related\";s:5:\"false\";s:9:\"vimeo_url\";s:0:\"\";s:11:\"testimonial\";s:0:\"\";s:18:\"testimonial_author\";s:0:\"\";s:16:\"testimonial_link\";s:0:\"\";s:23:\"testimonial_link_target\";s:5:\"_self\";}i:2;a:26:{s:2:\"id\";i:125;s:4:\"type\";s:5:\"image\";s:4:\"link\";s:0:\"\";s:5:\"title\";s:10:\"///ummmmmm\";s:11:\"description\";s:12:\"so hi people\";s:11:\"link_target\";s:5:\"_self\";s:7:\"img_alt\";s:0:\"\";s:9:\"img_title\";s:0:\"\";s:20:\"enable_slide_effects\";i:0;s:2:\"fx\";s:7:\"default\";s:5:\"speed\";s:0:\"\";s:7:\"timeout\";s:0:\"\";s:10:\"tile_count\";s:1:\"7\";s:10:\"tile_delay\";s:3:\"100\";s:13:\"tile_vertical\";s:4:\"true\";s:11:\"video_thumb\";s:0:\"\";s:9:\"video_url\";s:0:\"\";s:5:\"video\";s:0:\"\";s:6:\"custom\";s:0:\"\";s:11:\"youtube_url\";s:0:\"\";s:15:\"youtube_related\";s:5:\"false\";s:9:\"vimeo_url\";s:0:\"\";s:11:\"testimonial\";s:0:\"\";s:18:\"testimonial_author\";s:0:\"\";s:16:\"testimonial_link\";s:0:\"\";s:23:\"testimonial_link_target\";s:5:\"_self\";}}'),(2902,547,'_cycloneslider_settings','a:15:{s:8:\"template\";s:4:\"dark\";s:2:\"fx\";s:4:\"fade\";s:7:\"timeout\";i:4000;s:5:\"speed\";i:1000;s:5:\"width\";i:960;s:6:\"height\";i:600;s:11:\"hover_pause\";s:4:\"true\";s:14:\"show_prev_next\";i:1;s:8:\"show_nav\";i:1;s:10:\"tile_count\";i:7;s:10:\"tile_delay\";i:100;s:13:\"tile_vertical\";s:4:\"true\";s:6:\"random\";i:0;s:6:\"resize\";i:1;s:16:\"width_management\";s:10:\"responsive\";}'),(2903,555,'_menu_item_type','custom'),(2904,555,'_menu_item_menu_item_parent','0'),(2905,555,'_menu_item_object_id','555'),(2906,555,'_menu_item_object','custom'),(2907,555,'_menu_item_target',''),(2908,555,'_menu_item_classes','a:1:{i:0;s:0:\"\";}'),(2909,555,'_menu_item_xfn',''),(2910,555,'_menu_item_url','http://ucdavis.edu/'),(2912,556,'_menu_item_type','custom'),(2913,556,'_menu_item_menu_item_parent','0'),(2914,556,'_menu_item_object_id','556'),(2915,556,'_menu_item_object','custom'),(2916,556,'_menu_item_target',''),(2917,556,'_menu_item_classes','a:1:{i:0;s:0:\"\";}'),(2918,556,'_menu_item_xfn',''),(2919,556,'_menu_item_url','http://animalscience.ucdavis.edu/'),(2921,230,'_wp_trash_meta_status','publish'),(2922,230,'_wp_trash_meta_time','1402346440'),(2923,223,'_wp_trash_meta_status','publish'),(2924,223,'_wp_trash_meta_time','1402346444'),(2925,511,'_edit_lock','1402597204:1'),(2926,510,'_edit_lock','1402597216:1'),(2927,271,'_wp_attachment_backup_sizes','a:4:{s:9:\"full-orig\";a:3:{s:5:\"width\";i:700;s:6:\"height\";i:400;s:4:\"file\";s:18:\"support_header.png\";}s:14:\"thumbnail-orig\";a:4:{s:4:\"file\";s:26:\"support_header-150x150.png\";s:5:\"width\";i:150;s:6:\"height\";i:150;s:9:\"mime-type\";s:9:\"image/png\";}s:11:\"medium-orig\";a:4:{s:4:\"file\";s:26:\"support_header-300x171.png\";s:5:\"width\";i:300;s:6:\"height\";i:171;s:9:\"mime-type\";s:9:\"image/png\";}s:19:\"post-thumbnail-orig\";a:4:{s:4:\"file\";s:26:\"support_header-624x356.png\";s:5:\"width\";i:624;s:6:\"height\";i:356;s:9:\"mime-type\";s:9:\"image/png\";}}');
/*!40000 ALTER TABLE `wp_postmeta` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_posts`
--

DROP TABLE IF EXISTS `wp_posts`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_posts` (
  `ID` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
  `post_author` bigint(20) unsigned NOT NULL DEFAULT '0',
  `post_date` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `post_date_gmt` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `post_content` longtext NOT NULL,
  `post_title` text NOT NULL,
  `post_excerpt` text NOT NULL,
  `post_status` varchar(20) NOT NULL DEFAULT 'publish',
  `comment_status` varchar(20) NOT NULL DEFAULT 'open',
  `ping_status` varchar(20) NOT NULL DEFAULT 'open',
  `post_password` varchar(20) NOT NULL DEFAULT '',
  `post_name` varchar(200) NOT NULL DEFAULT '',
  `to_ping` text NOT NULL,
  `pinged` text NOT NULL,
  `post_modified` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `post_modified_gmt` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `post_content_filtered` longtext NOT NULL,
  `post_parent` bigint(20) unsigned NOT NULL DEFAULT '0',
  `guid` varchar(255) NOT NULL DEFAULT '',
  `menu_order` int(11) NOT NULL DEFAULT '0',
  `post_type` varchar(20) NOT NULL DEFAULT 'post',
  `post_mime_type` varchar(100) NOT NULL DEFAULT '',
  `comment_count` bigint(20) NOT NULL DEFAULT '0',
  PRIMARY KEY (`ID`),
  KEY `post_name` (`post_name`),
  KEY `type_status_date` (`post_type`,`post_status`,`post_date`,`ID`),
  KEY `post_parent` (`post_parent`),
  KEY `post_author` (`post_author`)
) ENGINE=InnoDB AUTO_INCREMENT=577 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_posts`
--

LOCK TABLES `wp_posts` WRITE;
/*!40000 ALTER TABLE `wp_posts` DISABLE KEYS */;
INSERT INTO `wp_posts` VALUES (10,1,'2014-02-10 18:53:38','2014-02-10 18:53:38','<div class=\"partner_images\">\r\n<h2 style=\"text-align: center;\">Funders and Associates of the Genomic Variation Laboratory</h2>\r\n<p style=\"text-align: center;\"><a title=\"US Fish &amp; Wildlife Service\" href=\"http://www.fws.gov/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-193\" style=\"font-size: 1rem; line-height: 1;\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/U_S__Fish__and__Wildlife_Service.jpg\" alt=\"U_S__Fish__and__Wildlife_Service\" width=\"103\" height=\"124\" /></a><a title=\"California Department of Water Resources\" href=\"http://www.water.ca.gov/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-186\" style=\"font-size: 1rem; line-height: 1;\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/dwr-logobig_home.jpg\" alt=\"dwr-logobig_home\" width=\"125\" height=\"125\" /></a><a title=\"US Forest Service\" href=\"http://www.fs.fed.us/\" target=\"_blank\"><img class=\"alignnone size-medium wp-image-194\" style=\"font-size: 1rem; line-height: 1;\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/USForestServiceLogo.jpg\" alt=\"USForestServiceLogo\" width=\"108\" height=\"121\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" title=\"US Department of the Interior, Bureau of Land Management\" href=\"http://www.blm.gov/wo/st/en.html\" target=\"_blank\"><img class=\"alignnone size-full wp-image-171\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/BLMLogo.jpg\" alt=\"BLMLogo\" width=\"123\" height=\"123\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" title=\"California Department of Fish &amp; Game\" href=\"http://www.dfg.ca.gov/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-172\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/DFGshield.jpg\" alt=\"DFGshield\" width=\"102\" height=\"135\" /></a><span style=\"line-height: 1.714285714; font-size: 1rem;\"></span><a title=\"Cramer Fish Sciences\" href=\"http://www.fishsciences.net/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-184\" style=\"font-size: 1rem; line-height: 1;\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.jpg\" alt=\"droppedImage\" width=\"233\" height=\"46\" /></a><a title=\"Kern Community Foundation\" href=\"http://www.kernfoundation.org/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-195\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/logo.jpg\" alt=\"logo\" width=\"165\" height=\"57\" /></a><a title=\"Kootenai Tribe of Idaho\" href=\"http://www.kootenai.org/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-187\" style=\"font-size: 1rem; line-height: 1;\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/shapeimage_4.png\" alt=\"shapeimage_4\" width=\"167\" height=\"112\" /></a><span style=\"line-height: 1.714285714; font-size: 1rem;\"><img class=\"alignnone size-full wp-image-173\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_1.jpg\" alt=\"droppedImage_1\" width=\"121\" height=\"121\" /><img class=\"alignnone size-full wp-image-174\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_2.jpg\" alt=\"droppedImage_2\" width=\"188\" height=\"54\" /><img class=\"alignnone size-full wp-image-175\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_3.jpg\" alt=\"droppedImage_3\" width=\"93\" height=\"93\" /><img class=\"alignnone size-full wp-image-176\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_4.jpg\" alt=\"droppedImage_4\" width=\"124\" height=\"72\" /><img class=\"alignnone size-full wp-image-177\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_5.jpg\" alt=\"droppedImage_5\" width=\"213\" height=\"68\" /><img class=\"alignnone size-full wp-image-178\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_6.jpg\" alt=\"droppedImage_6\" width=\"86\" height=\"113\" /><img class=\"alignnone size-full wp-image-179\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_7.jpg\" alt=\"droppedImage_7\" width=\"149\" height=\"102\" /></span><img class=\"alignnone size-full wp-image-181\" style=\"font-size: 1rem; line-height: 1;\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_9.jpg\" alt=\"droppedImage_9\" width=\"119\" height=\"119\" /><img class=\"alignnone size-full wp-image-185\" style=\"font-size: 1rem; line-height: 1;\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.png\" alt=\"droppedImage\" width=\"97\" height=\"140\" /><img class=\"alignnone size-full wp-image-180\" style=\"font-size: 1rem; line-height: 1;\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_8.jpg\" alt=\"droppedImage_8\" width=\"244\" height=\"70\" /><img class=\"alignnone size-full wp-image-182\" style=\"font-size: 1rem; line-height: 1;\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_10.jpg\" alt=\"droppedImage_10\" width=\"157\" height=\"123\" /><span style=\"line-height: 1.714285714; font-size: 1rem;\"><img class=\"alignnone size-full wp-image-183\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_11.jpg\" alt=\"droppedImage_11\" width=\"192\" height=\"71\" /></span></p>\r\n\r\n</div>','Partners','','publish','open','open','','partners','','','2014-06-05 20:54:43','2014-06-05 20:54:43','',0,'http://localhost/~psalessi/Genetics_Lab/?page_id=10',0,'page','',0),(11,1,'2014-02-10 18:53:38','2014-02-10 18:53:38','','Partners','','inherit','open','open','','10-revision-v1','','','2014-02-10 18:53:38','2014-02-10 18:53:38','',10,'http://localhost/~psalessi/Genetics_Lab/?p=11',0,'revision','',0),(12,1,'2014-02-10 18:53:48','2014-02-10 18:53:48','<strong>2013</strong>\r\n\r\nChen, Y., S. Parmenter, and B. May. Genetic characterization of variation and management implications for the endangeredMohave tui chub. Cons. Gen. 14:11-20. (<a title=\"Publications_files/2013 - Chen Mohave tui chub.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2013%20-%20Chen%20Mohave%20tui%20chub.pdf\">pdf</a>)\r\n\r\nMeek, M. H., A. P. Wintzer, N. Shepherd, and B. May. Genetic diversity and reproductive mode in two non-native hydromedusae, <em>Maeotias marginata</em> and <em>Moerisia</em> sp., in the upper San Francisco Estuary, California. Biol. Invasions. 15(1): 199-212. (<a title=\"Publications_files/2012 Meek et al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Meek%20et%20al.pdf\">pdf</a>)\r\n\r\nSchreier, A.D., B. Mahardja, and B. May. Hierarchical patterns of population structure in the endangered Fraser River white sturgeon, Acipenser transmontanus, and implications for conservation. Can. J. Fish. &amp; Aquat. Sci. 69:1968-1980. (<a title=\"Publications_files/2013 - Schreier Fraser river white sturgeon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2013%20-%20Schreier%20Fraser%20river%20white%20sturgeon.pdf\">pdf</a>)\r\n\r\n<strong>2012</strong>\r\n\r\nBaerwald, M.R., B.M. Schreier, G. Schumer, and B. May. Detection of Threatened Delta Smelt in the Gut Contents of the Invasive Mississippi Silverside in the San Francisco Estuary Using TaqMan Assays. Trans. Am. Fish. Soc. 141:1600-1607. (<a title=\"Publications_files/2012 - Baerwald gut contents.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20-%20Baerwald%20gut%20contents.pdf\">pdf</a>)\r\n\r\nMahardja, B., B. May, and M. Baerwald. Characterization of 36 additional microsatellite loci in splittail (<em>Pogonichthys macrolepidotus</em>) and cross-amplification in five other native Californian cyprinid species. Cons. Gen. Res. 4(4):917-921. (<a title=\"Publications_files/2012 Meek et al - Climate change.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Mahardja%20et%20al.%20-%20Cyprinid%20Microsat%20note.pdf\">pdf</a>)\r\n\r\nMeek, M., A. Wintzer, W. Wetzel, and B. May. Climate change likely to facilitate the invasion of the non-native hydroid, Cordylophora caspia, in the San Francisco Estuary, CA. PLos ONE. 7(10): e46373. doi:10.1371/journal.pone.0046373 (<a title=\"Publications_files/2012 - Petersen lion paw scallop.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Meek%20et%20al%20-%20Climate%20change.pdf\">pdf</a>)\r\n\r\nPetersen, J.L., M.R. Baerwald, J.M. Ibarra, and B. May. A first-generation linkage map of the Pacific lion-paw scallop: initial evidence of QTL for size traits and markers linked to orange shell color. Aquaculture. 350-353:200-209. (<a title=\"Publications_files/2012 - Petersen lion paw scallop.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20-%20Petersen%20lion%20paw%20scallop.pdf\">pdf</a>)\r\n\r\nSchreier, A. D., J. Rodzen, S. Ireland, and B. May. Genetic techniques inform conservation aquaculture of the endangered Kootenai River white sturgeon, <em>Acipenser transmontanus</em>. Endang. Species Res. 16:65-75. (<a title=\"Publications_files/2012 Schreier et al. - Kootenai.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Schreier%20et%20al.%20-%20Kootenai.pdf \">pdf</a>)\r\n\r\nSukmanomon, S., W. Kamonrat, S. Poompuang, T. Nguyen, D. M. Bartley, B. May, U. Na-Nakorn. Genetic changes, intra- and interspecific introgression in farmed Nile tilapia in Thailand. Aquaculture 324-325:44-54. (<a title=\"Publications_files/2012 - Sukmanomon Tilapia.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20-%20Sukmanomon%20%20Tilapia.pdf\">pdf</a>)\r\n\r\nVan Dam, A.R. and B. May. A new species of Dactylopius Costa (<em>Dactylopius gracilipilus</em> sp. nov.) (Hemiptera: Coccoidea: Dactylopiidae) from the Chihuahuan Desert, Texas, U.S.A. . Zootaxa 3573:33-39. (<a title=\"Publications_files/2012 - Van Dam new species.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20-%20Van%20Dam%20new%20species.pdf\">pdf</a>)\r\n\r\n<strong>2011</strong>\r\n\r\nBaerwald, M., J. Petersen, R. Hedrick, G. Schisler, and B. May. A major effect quantitative trait locus for whirling disease resistance identified in rainbow trout (<em>Oncorhynchus mykiss</em>). Heredity. 106:920-926. (<a title=\"Publications_files/2011 - Baerwald et al. - QTL whirling.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20-%20Baerwald%20et%20al.%20-%20QTL%20whirling.pdf\">pdf</a>)\r\n\r\nBaerwald, M., G. Schumer, B. Schreier, and B. May. TaqMan assays for the genetic identification of delta smelt (Hypomesus transpacificus) and wakasagi smelt (<em>Hypomesus nipponensis</em>). Mol. Ecol. Res. 11:784-785. (<a title=\"Publications_files/2011 - Baerwald TaqMan assays.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20-%20Baerwald%20TaqMan%20assays.pdf\">pdf</a>)\r\n\r\nBaerwald, M.R., M. R. Stephens, K. S. Bork, M. H. Meek, K. Tomalty, and B. May. 2011. Spring-run Chinook salmon genetic management plan  San Joaquin River Restoration Program. 118 pgs. Scientific genetic management plan for the CA Dept. of Fish and Game. (<a title=\"Publications_files/GMP_SpringRunChinook_SJR_UCD_GVL_FINAL.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/GMP_SpringRunChinook_SJR_UCD_GVL_FINAL.pdf\">pdf</a>)\r\n\r\nFinger, A. J., E. C. Anderson, M. R. Stephens, and B. P. May. Application of a method for estimating effective population size and admixture using diagnostic SNPs: Implications for conservation of threatened Paiute cutthroat trout (<em>Oncorhynchus clarkii seleniris</em>) in Silver King Creek, California. Can. J. Fish. Aquat. Sci. 68:1369-1386 (<a title=\"Publications_files/2011 - Finger et al. - Effective pop size of Paiute cutthroat.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20-%20Finger%20et%20al.%20-%20Effective%20pop%20size%20of%20Paiute%20cutthroat.pdf\">pdf</a>).\r\n\r\nFisch, K.M., J.M. Henderson, R.S. Burton, and B. May. Population genetics and conservation implications for the endangered delta smelt in the San Francisco Bay-Delta. Conservation Genetics. 12:1421-1434 (<a title=\"Publications_files/2011 Fisch et al. Delta smelt pop gen.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20Fisch%20et%20al.%20Delta%20smelt%20pop%20gen.pdf\">pdf</a>).\r\n\r\nFloyd, C. H., D.H. Van Vuren, K.R. Crooks, K.L. Jones, D.K. Garcelon, N.M. Belfiore, J.W. Dragoo, and B. May. Genetic differentiation of island spotted skunks, <em>Spilogale gracilis amphiala.</em> J. Mam. 92(1):148-158 (<a title=\"Publications_files/2011 Floyd et al. skunks.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20Floyd%20et%20al.%20skunks.pdf\">pdf</a>).\r\n\r\nMcEachern, M.B., D.H. Van Vuren, C.H. Floyd, B. May, and J.M. Eadie.Bottlenecks and rescue effects in a fluctuating population of golden-mantled ground squirrels (<em>Spermophilus lateralis</em>). Con. Gen. 12:285-296 (<a title=\"Publications_files/2011 McEachern ground squirrels.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20McEachern%20ground%20squirrels.pdf\">pdf</a>).\r\n\r\nMejia , O., L. G. Herrera, B. May, R. A. Medellin,and J. J. Flores-Martinez. Effective population size dynamics of Myotis vivesi during the Pleistocene and Holocene climatic changes. Acta Chiropterologica. 13:33-40 (<a title=\"Publications_files/2011 Mejia et al .pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20Mejia%20et%20al%20.pdf\">pdf</a>).\r\n\r\nSchreier, A.D., D. Gille, B. Mahardja, and B. May. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon,<em> Acipenser transmontanus</em>. J. Appl. Ichthiol. 27:24-33 (<a title=\"Publications_files/2011 Schreier et al. - Octoploid origin sturgeon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20Schreier%20et%20al.%20-%20Octoploid%20origin%20sturgeon.pdf\">pdf</a>).\r\n\r\nWintzer, A.P., M.H. Meek, P.B. Moyle, and B. May. Ecological insights into the polyp stage of non-native hydrozoans in the San Francisco Estuary. Aquat. Ecol. 45:151161 (<a title=\"Publications_files/2011 Wintzer - Aquatic Ecology.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20Wintzer%20-%20Aquatic%20Ecology.pdf\">pdf</a>).\r\n\r\n<strong>2010</strong>\r\n\r\nFloyd, C. H., J.J. Flores Martnez, , L.G. Herrera, O. Mejia, and B. May. Conserving the endangered Mexican fishing bat (<em>Myotis vivesi</em>): genetic variation indicates extensive gene flow among islands in the Gulf of California. Cons. Gen. 11:813-82 (<a title=\"Publications_files/2010 Floyd et al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2010%20Floyd%20et%20al.pdf\">pdf</a>).\r\n\r\nIsrael, J.A. and B. May. Indirect estimates of breeding population size in the polyploid green sturgeon (<em>Acipenser medirostris</em>). Mol. Ecol. 19:1058-107 (<a title=\"Publications_files/2010 Israel Mol. Ecol..pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2010%20Israel%20Mol.%20Ecol..pdf\">pdf</a>).\r\n\r\nIsrael, J.A. and B. May. Characterization and evaluation of polymorphic microsatellite markers in the anadromous fish <em>Spirinchus thaleichthys</em>. Cons. Gen. Res. online (<a title=\"Publications_files/2010 Israel &amp; May - Spirinchus microsat.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2010%20Israel%20%26%20May%20-%20Spirinchus%20microsat.pdf\">pdf</a>).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Nuclear and mtDNA lineage diversity in wild and cultured Pacific lion-paw scallop (Baja California Peninsula, Mexico). Mar. Biol. 157:27512767 (<a title=\"Publications_files/2010 Petersen et al. wild and cultured.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2010%20Petersen%20et%20al.%20wild%20and%20cultured.pdf\">pdf</a>).\r\n\r\n<strong>2009</strong>\r\n\r\nChen, Y., S. Reid, and B. May. Genetic relationships among tui chub populations in the Cow Head and surrounding basins of the Northwestern Great Basin. Cons. Gen. 10:101-114 (<a title=\"Publications_files/2009_Chen_great_basin_tui.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009_Chen_great_basin_tui.pdf\">pdf</a>).\r\n\r\nFinger, A.J., M.R. Stephens, N.W. Clipperton and B. May. Six diagnostic SNP markers for detecting introgression between cutthroat and rainbow trout. Mol. Ecol. Res. 9:759-763 (<a title=\"Publications_files/2009 - Finger_six_SNPs.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009%20-%20Finger_six_SNPs.pdf\">pdf</a>).\r\n\r\nFisch, K., J.L. Petersen, M.R. Baerwald, and B.P. May. Characterization of 24 microsatellite loci in delta smelt, <em>Hypomesus transpacificus</em>, and their cross-species amplification in two other smelt species of the Osmeridae family. Mol. Ecol. Res. 9:405-408 (<a title=\"Publications_files/2009_Fisch_primer_smelt.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009_Fisch_primer_smelt.pdf\">pdf</a>).\r\n\r\nIsrael, J.A., K.J. Bando, E.C. Anderson, and B. May. Polyploid microsatellite data reveal stock complexity among estuarine North American green sturgeon (<em>Acipenser medirostris</em>). Can. J. Fish. Aqu. Sci. 66:1491-1504 (<a title=\"Publications_files/2009 Israel green sturgeon stock.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009%20Israel%20green%20sturgeon%20stock.pdf\">pdf</a>).\r\n\r\nMeek, M., M. Baerwald, A. Wintzer, and B. May. Isolation and characterization of microsatellite loci in two non-native hydromedusae in the San Francisco Estuary: <em>Maeotias marginata</em> and <em>Moerisia</em> sp. Cons. Gen. Res. 1:205-208 (<a title=\"Publications_files/2009 - Meek et al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009%20-%20Meek%20et%20al.pdf\">pdf</a>).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Thirty-seven additional microsatellite loci in the Pacific lion-paw scallop (<em>Nodipecten subnodosus</em>) and cross-amplification in other Pectinids. Cons. Gen. Res. 1:101-105 (<a title=\"Publications_files/Petersen et al. 2009 - microsatellite.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/Petersen%20et%20al.%202009%20-%20microsatellite.pdf\">pdf</a>).\r\n\r\nSimmons, R. E., P. Lavretsky, and B. May. Introgressive Hybridization of Redband Trout in the Upper McCloud River Watershed. Trans. Am. Fish. Soc. 139:201-213 (<a title=\"Publications_files/2009 - Simmons_redband_introgression.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009%20-%20Simmons_redband_introgression.pdf\">pdf</a>).\r\n\r\nStephens, M.R., N.W. Clipperton, and B.P. May. Subspecies-informative TaqMan probes for evaluating introgression between native golden trout and introduced rainbow trout. Mol. Ecol. Res. 9:339-343 (<a title=\"Publications_files/2009_Stephens_SNP_primers.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009_Stephens_SNP_primers.pdf\">pdf</a>).\r\n\r\n<strong>Earlier publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications','','publish','open','open','','publications','','','2014-03-12 16:20:19','2014-03-12 16:20:19','',0,'http://localhost/~psalessi/Genetics_Lab/?page_id=12',0,'page','',0),(13,1,'2014-02-10 18:53:48','2014-02-10 18:53:48','','Publications','','inherit','open','open','','12-revision-v1','','','2014-02-10 18:53:48','2014-02-10 18:53:48','',12,'http://localhost/~psalessi/Genetics_Lab/?p=13',0,'revision','',0),(14,1,'2014-02-10 18:54:00','2014-02-10 18:54:00','The Conservation Genetics Fund supports the Genomic Variation Laboratory (GVL) at UC Davis. Your donation supports both quality science that informs species conservation and the students that generate that science.\r\n\r\nThe Genomic Variation Laboratory works with wildlife managers to help answer questions essential for species conservation. For example, <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">our work</a> helps define species and their phylogenetic relationships and informs the genetics of reintroductions. We study topics such as evolution, predation, aquaculture, population genetics structure, hybridization, and genetic management for reintroduction of salmonids.\r\n\r\nOur lab trains undergraduate and graduate <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">students</a> in many different molecular biological techniques, preparing them for a wide variety of careers. Bernie May is very supportive of career development and encourages his students to attend conferences, participate in internships, and network with agency biologists. We also host visiting researchers from outside the university to work with us in our lab.\r\n\r\n[caption id=\"attachment_239\" align=\"aligncenter\" width=\"240\"]<img class=\"size-full wp-image-239\" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /> $89,045 Kern Community Foundation Grant[/caption]\r\n<h2 style=\"text-align: center;\">Ways to Make a Donation:</h2>\r\n<ol>\r\n	<li>Online:Donate through the secure UC Davis donation website <a title=\"Giving to UC Davis\" href=\"https://awc.ucdavis.edu/MakeAGift.aspx?alloccat=130202120\" target=\"_blank\">HERE</a>.</li>\r\n	<li>By Mail:Mail your check made out to University of California Regents toBernie May, Meyer Hall, 1 Shields Avenue, Davis, CA 95616. Please <a title=\"E-Mail Bernie May\" href=\"mailto:bpmay@ucdavis.edu?subject=Mail%20Donation\">email Bernie May</a> to let him know the check is coming.</li>\r\n	<li>Learn more about making a donation:Contact Dr. Bernie May through <a title=\"E-Mail Bernie May\" href=\"mailto:bpmay@ucdavis.edu?subject=Mail%20Donation\">email</a> or phone (530) 754-8123</li>\r\n</ol>','Support Us','','publish','open','open','','support-us','','','2014-03-05 19:40:15','2014-03-05 19:40:15','',0,'http://localhost/~psalessi/Genetics_Lab/?page_id=14',0,'page','',0),(15,1,'2014-02-10 18:54:00','2014-02-10 18:54:00','','Support Us','','inherit','open','open','','14-revision-v1','','','2014-02-10 18:54:00','2014-02-10 18:54:00','',14,'http://localhost/~psalessi/Genetics_Lab/?p=15',0,'revision','',0),(16,1,'2014-02-10 18:54:13','2014-02-10 18:54:13','<strong><a title=\"Protocols and Procedures\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/\">Protocol and Procedures</a></strong>\r\n\r\n<strong><a title=\"Links\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/links/\">Links</a></strong>\r\n<h3>Contact Us<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_blue.png\"><img class=\"alignright size-full wp-image-235\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_blue.png\" alt=\"Genomic variation laboratory blue logo image\" width=\"207\" height=\"217\" /></a></h3>\r\n<p style=\"text-align: left;\">2403 Meyer Hall, Department of Animal Science\r\nUniversity of California, DavisOne Shields Avenue Davis, CA 95616</p>\r\nPhone: (530) 752-6351\r\nFax: (530) 752-0175\r\n<h3>Directions</h3>\r\n<h4><span style=\"text-decoration: underline;\">From Sac. Metro Airport:</span></h4>\r\nGet on I-5 south from airport Blvd.\r\n\r\nTake I-80 West towards San Francisco.\r\n<h4><span style=\"text-decoration: underline;\"><strong>From I-80 east or west:</strong></span></h4>\r\nExit at the \"U.C. Davis/Old Davis Road\" exit and travel North.\r\n\r\nFollow Old Davis Road and go straight through first stop sign.\r\n\r\nCross bridge and turn left at first intersection, La Rue\r\n\r\nTurn right on Bioletti Way, and left in the parking lot. There is a permit dispenser just as you enter from this side. Parking is $7 per day.\r\n\r\nMeyer Hall is on the corner of Bioletti and La Rue.\r\n\r\nThe GVL is on the second floor in Rm. 2403 in the north building of Meyer Hall.\r\n\r\n<iframe class=\"aligncenter\" style=\"border: 0;\" src=\"https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d6241.639725534423!2d-121.75234279909584!3d38.53792040586795!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x8085291ab1c780f9%3A0xc365a885cdb2e287!2sMeyer+Hall!5e0!3m2!1sen!2sus!4v1402333214210\" width=\"600\" height=\"450\" frameborder=\"0\"></iframe>','Resources','','publish','open','open','','resources','','','2014-06-09 20:32:24','2014-06-09 20:32:24','',0,'http://localhost/~psalessi/Genetics_Lab/?page_id=16',0,'page','',0),(17,1,'2014-02-10 18:54:13','2014-02-10 18:54:13','','Resources','','inherit','open','open','','16-revision-v1','','','2014-02-10 18:54:13','2014-02-10 18:54:13','',16,'http://localhost/~psalessi/Genetics_Lab/?p=17',0,'revision','',0),(20,1,'2014-02-10 18:58:21','2014-02-10 18:58:21','http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/cropped-genetics_lab_home_header.png','Cropped Home Page Featured Image','Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla id pulvinar ipsum, in mattis sem. Vestibulum id purus commodo, blandit diam eu, consectetur lacus. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Nullam velit turpis, dapibus eu nulla pharetra, tincidunt tincidunt erat. Sed tortor urna, porta vel ante sed, fringilla fermentum tortor. Vestibulum a pretium leo, in consectetur ante. Mauris pellentesque, risus ut sodales facilisis, erat sapien laoreet sem, vitae laoreet nulla nulla vitae quam. Nunc tristique, erat at vestibulum ali','inherit','open','open','','cropped-genetics_lab_home_header-png','','','2014-02-10 18:58:21','2014-02-10 18:58:21','',0,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/cropped-genetics_lab_home_header.png',1,'attachment','image/png',0),(21,1,'2014-02-10 18:59:53','2014-02-10 18:59:53','','Home','','publish','open','open','','home','','','2014-03-17 19:55:46','2014-03-17 19:55:46','',0,'http://localhost/~psalessi/Genetics_Lab/?p=21',1,'nav_menu_item','',0),(22,1,'2014-02-10 18:59:53','2014-02-10 18:59:53',' ','','','publish','open','open','','22','','','2014-03-17 19:55:46','2014-03-17 19:55:46','',0,'http://localhost/~psalessi/Genetics_Lab/?p=22',4,'nav_menu_item','',0),(25,1,'2014-02-10 18:59:53','2014-02-10 18:59:53',' ','','','publish','open','open','','25','','','2014-03-17 19:55:46','2014-03-17 19:55:46','',0,'http://localhost/~psalessi/Genetics_Lab/?p=25',5,'nav_menu_item','',0),(26,1,'2014-02-10 18:59:53','2014-02-10 18:59:53',' ','','','publish','open','open','','26','','','2014-03-17 19:55:46','2014-03-17 19:55:46','',0,'http://localhost/~psalessi/Genetics_Lab/?p=26',7,'nav_menu_item','',0),(28,1,'2014-02-10 18:59:53','2014-02-10 18:59:53',' ','','','publish','open','open','','28','','','2014-03-17 19:55:46','2014-03-17 19:55:46','',0,'http://localhost/~psalessi/Genetics_Lab/?p=28',6,'nav_menu_item','',0),(30,1,'2014-02-10 19:06:41','2014-02-10 19:06:41','[caption id=\"attachment_239\" align=\"alignleft\" width=\"270\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\" wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"270\" height=\"195\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','publish','open','open','','home','','','2014-06-12 20:27:23','2014-06-12 20:27:23','',0,'http://localhost/~psalessi/Genetics_Lab/?page_id=30',0,'page','',0),(31,1,'2014-02-10 19:06:41','2014-02-10 19:06:41','','Home','','inherit','open','open','','30-revision-v1','','','2014-02-10 19:06:41','2014-02-10 19:06:41','',30,'http://localhost/~psalessi/Genetics_Lab/?p=31',0,'revision','',0),(69,1,'2014-02-12 19:03:38','2014-02-12 19:03:38','','Ppl_Profile Picture Upload','','publish','closed','closed','','acf_ppl_profile-picture-upload','','','2014-02-21 18:10:35','2014-02-21 18:10:35','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=acf&#038;p=69',0,'acf','',0),(72,1,'2014-02-12 19:42:57','2014-02-12 19:42:57','','Ppl_Curriculum Vitae Upload','','publish','closed','closed','','acf_ppl_curriculum-vitae-upload','','','2014-02-21 18:10:26','2014-02-21 18:10:26','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=acf&#038;p=72',1,'acf','',0),(80,1,'2014-02-14 19:33:37','2014-02-14 19:33:37','','Mariah Meek','','publish','closed','closed','','mariah-meek','','','2014-03-20 17:33:33','2014-03-20 17:33:33','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=80',0,'people','',0),(81,1,'2014-02-14 19:33:00','2014-02-14 19:33:00','','Mariah Meek Profile Image','','inherit','closed','open','','img_1612','','','2014-02-14 19:33:00','2014-02-14 19:33:00','',80,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/IMG_1612.jpg',0,'attachment','image/jpeg',0),(82,1,'2014-02-14 19:33:09','2014-02-14 19:33:09','','Mariah Meek Curriculum Vitae 2013 PDF','','inherit','closed','open','','meek-meek-cv-2013','','','2014-02-14 19:33:09','2014-02-14 19:33:09','',80,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/Meek-Meek-CV-2013.pdf',0,'attachment','application/pdf',0),(83,1,'2014-02-14 19:33:26','2014-02-14 19:33:26','','People Default Featured Image','','inherit','closed','open','','shapeimage_2','','','2014-02-14 19:33:26','2014-02-14 19:33:26','',80,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/shapeimage_2.png',0,'attachment','image/png',0),(93,1,'2014-02-19 19:19:21','2014-02-19 19:19:21','','Ppl_Education Field','','publish','closed','closed','','acf_ppl_education-field','','','2014-02-21 18:10:48','2014-02-21 18:10:48','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=acf&#038;p=93',0,'acf','',0),(94,1,'2014-02-19 19:37:04','2014-02-19 19:37:04','','Ppl_Research Interests Field','','publish','closed','closed','','acf_ppl_research-interests-field','','','2014-02-21 18:10:17','2014-02-21 18:10:17','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=acf&#038;p=94',1,'acf','',0),(95,1,'2014-02-19 19:37:49','2014-02-19 19:37:49','','Ppl_Research Projects Field','','publish','closed','closed','','acf_ppl_research-projects-field','','','2014-02-21 18:10:08','2014-02-21 18:10:08','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=acf&#038;p=95',2,'acf','',0),(96,1,'2014-02-19 19:38:25','2014-02-19 19:38:25','','Ppl_Publications Field','','publish','closed','closed','','acf_ppl_publications-field','','','2014-02-21 18:09:58','2014-02-21 18:09:58','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=acf&#038;p=96',3,'acf','',0),(97,1,'2014-02-21 18:04:57','2014-02-21 18:04:57','','Pro_Project Image','','publish','closed','closed','','acf_pro_project-image','','','2014-02-21 18:10:59','2014-02-21 18:10:59','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=acf&#038;p=97',0,'acf','',0),(100,1,'2014-02-21 18:13:30','2014-02-21 18:13:30','','Pro_Background','','publish','closed','closed','','acf_pro_background','','','2014-02-21 18:13:30','2014-02-21 18:13:30','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=acf&#038;p=100',0,'acf','',0),(101,1,'2014-02-21 18:14:07','2014-02-21 18:14:07','','Pro_Materials and Methods','','publish','closed','closed','','acf_pro_materials-and-methods','','','2014-02-21 18:29:02','2014-02-21 18:29:02','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=acf&#038;p=101',2,'acf','',0),(102,1,'2014-02-21 18:14:44','2014-02-21 18:14:44','','Pro_Results','','publish','closed','closed','','acf_pro_results','','','2014-02-21 18:28:39','2014-02-21 18:28:39','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=acf&#038;p=102',0,'acf','',0),(103,1,'2014-02-21 18:15:25','2014-02-21 18:15:25','','Pro_Future Work','','publish','closed','closed','','acf_pro_future-work','','','2014-02-21 18:29:38','2014-02-21 18:29:38','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=acf&#038;p=103',0,'acf','',0),(104,1,'2014-02-21 18:16:03','2014-02-21 18:16:03','','Pro_Collaborators','','publish','closed','closed','','acf_pro_collaborators','','','2014-02-21 18:16:03','2014-02-21 18:16:03','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=acf&#038;p=104',0,'acf','',0),(105,1,'2014-02-21 18:16:49','2014-02-21 18:16:49','','Pro_Literature Cited','','publish','closed','closed','','acf_pro_literature-cited','','','2014-02-21 19:26:16','2014-02-21 19:26:16','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=acf&#038;p=105',0,'acf','',0),(106,1,'2014-02-21 18:22:58','2014-02-21 18:22:58','','California Golden Trout (<em>Oncorhynchus mykiss aguabonita</em>)','','publish','closed','closed','','california-golden-trout-oncorhynchus-mykiss-aguabonita','','','2014-03-10 19:06:19','2014-03-10 19:06:19','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=projects&#038;p=106',0,'projects','',0),(107,1,'2014-02-21 18:17:54','2014-02-21 18:17:54','','California Golden Trout Project Image','','inherit','closed','open','','dsc01370','','','2014-02-21 18:17:54','2014-02-21 18:17:54','',106,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/DSC01370.jpg',0,'attachment','image/jpeg',0),(109,1,'2014-02-21 18:21:44','2014-02-21 18:21:44','','California Golden Trout Map Image','Figure 1. Comparison of admixture levels detected in Golden Trout Creek and South Fork Kern River California golden trout populations.','inherit','closed','open','','gold_trout_map','','','2014-02-21 18:21:44','2014-02-21 18:21:44','',106,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gold_trout_map.jpg',0,'attachment','image/jpeg',0),(110,1,'2014-02-21 18:21:44','2014-02-21 18:21:44','','California Golden Trout Map Key Image','','inherit','closed','open','','key_to_gold_trout_map','','','2014-02-21 18:21:44','2014-02-21 18:21:44','',106,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/key_to_gold_trout_map.gif',0,'attachment','image/gif',0),(114,1,'2014-02-21 18:53:50','2014-02-21 18:53:50','','People','','publish','closed','open','','people','','','2014-03-17 19:55:46','2014-03-17 19:55:46','',0,'http://localhost/~psalessi/Genetics_Lab/?p=114',2,'nav_menu_item','',0),(115,1,'2014-02-21 18:53:50','2014-02-21 18:53:50','','Projects','','publish','closed','open','','projects','','','2014-03-17 19:55:46','2014-03-17 19:55:46','',0,'http://localhost/~psalessi/Genetics_Lab/?p=115',3,'nav_menu_item','',0),(116,1,'2014-02-21 18:59:20','2014-02-21 18:59:20','','California Golden Trout Gathering Data Image','','inherit','closed','open','','future_work_river','','','2014-02-21 18:59:20','2014-02-21 18:59:20','',106,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/future_work_river.jpg',0,'attachment','image/jpeg',0),(118,1,'2014-02-21 18:59:21','2014-02-21 18:59:21','','California Golden Trout Data Analysis Graph Image','Figure 2. STRUCTURE representation of SNP data analysis for South Fork Kern California golden trout.\nEach vertical bar represents an individual fish, with populations grouped by population number (as in Table 1) on the x-axis. The green bar color indicates \"rainbow trout\" group membership, while red denotes \"golden\" group membership. The presence of green, therefore, indicates rainbow trout admixture in a given individual at levels measured by proportion on the y-axis.','inherit','closed','open','','red_green_graph','','','2014-02-21 18:59:21','2014-02-21 18:59:21','',106,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/red_green_graph.jpg',0,'attachment','image/jpeg',0),(119,1,'2014-02-21 20:12:19','2014-02-21 20:12:19','','California Golden Trout Featured Image','','inherit','closed','open','','gold_trout_header','','','2014-02-21 20:12:19','2014-02-21 20:12:19','',106,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gold_trout_header.png',0,'attachment','image/png',0),(121,1,'2014-02-21 20:25:03','2014-02-21 20:25:03','','Pro_Past Projects','','publish','closed','closed','','acf_pro_past-projects','','','2014-03-05 19:23:23','2014-03-05 19:23:23','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=acf&#038;p=121',0,'acf','',0),(122,1,'2014-02-20 20:31:23','2014-02-20 20:31:23','','Past Projects','','publish','closed','closed','','past-projects','','','2014-03-20 17:06:19','2014-03-20 17:06:19','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=projects&#038;p=122',0,'projects','',0),(123,1,'2014-02-21 20:31:08','2014-02-21 20:31:08','','Past Projects Featured Image','','inherit','closed','open','','past_projects_header','','','2014-02-21 20:31:08','2014-02-21 20:31:08','',122,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/past_projects_header.png',0,'attachment','image/png',0),(124,1,'2014-02-21 20:31:08','2014-02-21 20:31:08','','Past Projects Project Image','','inherit','closed','open','','past_projects_profile','','','2014-02-21 20:31:08','2014-02-21 20:31:08','',122,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/past_projects_profile.jpg',0,'attachment','image/jpeg',0),(125,1,'2014-02-21 20:34:32','2014-02-21 20:34:32','','Publications Page Featured Image','','inherit','closed','open','','publications_page_header','','','2014-02-21 20:34:32','2014-02-21 20:34:32','',12,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/publications_page_header.png',0,'attachment','image/png',0),(127,1,'2014-02-21 20:38:22','2014-02-21 20:38:22','<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/publications_2009-2013.png\"><img class=\"aligncenter size-full wp-image-126\" alt=\"publications_2009-2013\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/publications_2009-2013.png\" width=\"707\" height=\"1860\" /></a>','Publications','','inherit','closed','open','','12-revision-v1','','','2014-02-21 20:38:22','2014-02-21 20:38:22','',12,'http://localhost/~psalessi/Genetics_Lab/12-revision-v1/',0,'revision','',0),(128,1,'2014-02-21 20:44:17','2014-02-21 20:44:17','','Publications','','inherit','closed','open','','12-revision-v1','','','2014-02-21 20:44:17','2014-02-21 20:44:17','',12,'http://localhost/~psalessi/Genetics_Lab/12-revision-v1/',0,'revision','',0),(129,1,'2014-02-21 20:47:00','2014-02-21 20:47:00','<strong>2008</strong>\r\n\r\nBaerwald, M., V. Bien, F. Feyrer, and B. May. Distribution of genetically differentiated splittail populations during nonspawning season. Trans. Am. Fish. Soc. 137:1335-1345 (<a title=\"Publications_2_files/2008 - Baerwald_Distrib_splittail.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008%20-%20Baerwald_Distrib_splittail.pdf\">pdf</a>).\r\n\r\nBaerwald, M., A. Welsh, R. Hedrick, and B. May. Discovery of genes implicated in whirling disease infection and resistance in rainbow trout using genome-wide expression profiling. BMC Genomics 9:37 (<a title=\"Publications_2_files/2008_Baerwald_microarray.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008_Baerwald_microarray.pdf\">pdf</a>).\r\n\r\nBrk, K.S, A. Drauch, J. Israel, J. Pedroia, J. Rodzen, and B. May. Development of new microsatellite primers for green and white sturgeon. Cons. Gen. 9:973-979 (<a title=\"Publications_2_files/2008_KBork_Primer_Note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008_KBork_Primer_Note.pdf\">pdf</a>)\r\n\r\nPetersen, J. L., A. M. Ibarra, J. L. Ramirez, and B. May. An induced mass spawn of the hermaphroditic lion-paw scallop: Genetic assignment of maternal and paternal parentage. J. Hered. 99:337-348 (<a title=\"Publications_2_files/2008 - Petersen et al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008%20-%20Petersen%20et%20al.pdf\">pdf</a>).\r\n\r\nSchwartz, R.S. and B. May. Genetic Evaluation of Isolated Populations for Use in Reintroductions Reveals Significant Genetic Bottlenecks in Potential Stocks of Sacramento Perch. Trans. Am. Fish. Soc. 137:1764-1777 (<a title=\"Publications_2_files/2008_Schwartz_-_sac_perch.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008_Schwartz_-_sac_perch.pdf\">pdf</a>).\r\n\r\nWelsh, A., T. Hill, H. Quinlan, C. Robinson, and B. May. Genetic assessment of lake sturgeon population structure in the Great Lakes. N. Am. J. Fish. Man. 28:572591 (<a title=\"Publications_2_files/2008_Welsh_et_al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008_Welsh_et_al.pdf\">pdf</a>).\r\n\r\nWilliamson, K.S., R. Phillips, and B. May. Characterization of a chromosomal rearrangement responsible for producing \"apparent\" XY-female fall Chinook salmon in California. J. Hered. 99:483-490 (<a title=\"Publications_2_files/2008_Williamson_Heredity.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008_Williamson_Heredity.pdf\">pdf</a>).\r\n\r\n<strong>2007</strong>\r\n\r\nBaerwald, M.R., V. Bien, F. Feyrer, and B. May. Genetic analysis reveals two distinct Sacramento splittail (<em>Pogonichthyes macrolepidotus</em>) populations. Cons. Gen. 8:159-167. (<a title=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007_Baerwald_Genetic%20analysis%20splittail.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007_Baerwald_Genetic%20analysis%20splittail.pdf\">pdf</a>).\r\n\r\nChen, Y., S. Parmenter, and B. May. Introgression between Lahontan and endangered Owens tui chubs, and apparent discovery of a new tui chub in the Owens Valley, California. Conserv. Genet. 8:221-238. (<a title=\"Publications_2_files/2007_Chen_Owens_tui.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007_Chen_Owens_tui.pdf\">pdf</a>).\r\n\r\nFeyrer, F., J. Hobbs, M. Baerwald, T. Sommer, Q. Yin, K. Clark, B. May and W. Bennett. Otolith microchemistry provides information complementary to microsatellite DNA for migratory fish. Trans. Amer. Fish. Soc. 136:469-476. (<a title=\"Publications_2_files/2007 Feyrer_otoliths.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007%20Feyrer_otoliths.pdf\">pdf</a>)\r\n\r\nHull, J.M., D. Tufts, J. Topinka, B. May, and H. Ernest. Development of 19 microsatellite loci for Swainson\'s hawks (<em>Buteo swainsoni</em>) and other buteos. Molecular Ecology Notes 7:346-349. (<a title=\"Publications_2_files/2008_Hull_Swainsons.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008_Hull_Swainsons.pdf\">pdf</a>).\r\n\r\nLindley, S.T., R. Schick, E. Mora, P. Adams, J. Anderson, S. Greene, C. Hanson, B. May, D. McEwan, R. MacFarlane, C. Swanson, and J. Williams. Framework for Assessing Viability of Threatened and Endangered Chinook Salmon and Steelhead in the Sacramento-San Joaquin Basin . San Francisco Estuary and Watershed Science. Vol. 5, Issue 1 (1-26), Article 4. (<a title=\"Publications_2_files/2007_Lindley_viability.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007_Lindley_viability.pdf\">pdf</a>).\r\n\r\nMeredith, E. P., J.A. Rodzen, J. D. Banks, R. Schaefer, H. B. Ernest, T. R. Famula and B. P. May. Microsatellite analysis of three subspecies of Elk (<em>Cervus elaphus</em>) in California. Journal of Mammalogy 88(3):801-808. (<a title=\"Publications_2_files/2007 Meredith-elk.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007%20Meredith-elk.pdf\">pdf</a>)\r\n\r\nPapa R., J.A. Israel, F. N. Marzan, and B. May. Assessment of genetic variation among reproductive ecotypes of Klamath River steelhead reveals differentiation associated with different run-timings. J. Appl. Icthy. (<a title=\"Publications_2_files/2007 Papa_steelhead.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007%20Papa_steelhead.pdf\">pdf</a>)\r\n\r\nWilliamson, K. and B. May. Mitochondrial DNA Haplotype Diversity in Apparent XY Female Fall-Run and Spring-Run Chinook Salmon in California\'s Central Valley. Trans. Amer. Fish. Soc. 136: 1480-1486 (<a title=\"Publications_2_files/2007_Williamson_-_mtDNA_chinook.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007_Williamson_-_mtDNA_chinook.pdf\">pdf</a>).\r\n\r\n<strong>2006</strong>\r\n\r\nCordes, J.F., M.R. Stephens, M.A. Blumberg, and B. May. Identifying introgressive hybridization in native populations of California Golden Trout based on molecular markers. Trans. Am. Fish. Soc. 135:110-128. (<a title=\"Publications_2_files/2006_Cordes_etal.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2006_Cordes_etal.pdf\">pdf</a>).\r\n\r\nIbarra, A.M., J.L. Petersen, T.R. Famula, and B. May. Characterization of 35 microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and their cross-species amplification in four other scallops of the Pectinidae family. Mol. Ecol. Notes. 6:153-156 (<a title=\"Publications_2_files/2006 Ibarra_scallop_primers.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2006%20Ibarra_scallop_primers.pdf\">pdf</a>).\r\n\r\nLindley, S.T., R.S. Schick, A. Agrawal, M. Goslin, T. Pearson, E. Mora, J. J. Anderson, B.P. May, S. Greene, C. Hanson, A. Low, D. McEwan, R.B. MacFarlane, C. Swanson and J.G. Williams. Historical population structure of Central Valley steelhead and its alteration by dams. SF Est. Wtshd. Sci. 4(1): Article 3 (<a title=\"Publications_2_files/2006 Lindley___Steelhead.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2006%20Lindley___Steelhead.pdf\">pdf</a>).\r\n\r\nSprowles, A.E., M.R. Stephens, N.W. Clipperton, and B.P. May. Fishing for SNPs: a targeted locus approach for Single Nucleotide Polymorphism discovery in rainbow trout. Trans. Am. Fish. Soc. 135:1698-1721. (<a title=\"Publications_2_files/2006_Sprowles_SNPs.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2006_Sprowles_SNPs.pdf\">pdf</a>).\r\n\r\nTranah, G.J., and B. P. May. Patterns of Intra- and Interspecies Genetic Diversity in Klamath River Basin Suckers. Trans. Am Fish. Soc. 135:306316 (<a title=\"Publications_2_files/2006 Tranah_May.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2006%20Tranah_May.pdf\">pdf</a>).\r\n\r\nWelsh, A. and B. May. Development and standardization of disomic microsatellite markers for lake sturgeon genetic studies. J. Appl. Icthy. (<a title=\"Publications_2_files/2006 welsh standardization.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2006%20welsh%20standardization.pdf\">pdf</a>).\r\n\r\n<strong>2005</strong>\r\n\r\nBayard de Volo, S., RT Reynolds, JR Topinka, B. May and MF Antolin. Population genetic structure and the utility of muliti-locus genotyping for mark-recapture studies of northern goshawks (Accipiter gentilis). J. Raptor Res. 39:286-295. (<a title=\"Publications_2_files/2005 - Bayard_de_volo_etal.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20-%20Bayard_de_volo_etal.pdf\">pdf</a>)\r\n\r\nCordes, J.F., D.L. Perkins, H.L.Kincaid, and B. May. Genetic analysis of fish genomes and populations: allozyme variation within and among Atlantic salmon from downeast rivers of Maine. J. Fish Biology. 67:104-117. (<a title=\"Publications_2_files/2005 - Cordes Atlantic salmon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20-%20Cordes%20Atlantic%20salmon.pdf\">pdf</a>)\r\n\r\nFloyd, C.H., D. Van Vuren, and B. May. Marmots on Great Basin mountaintops: using genetics to test an ecological paradigm. Ecology. 86: 2145-2153. (<a title=\"Publications_2_files/2005 - Floyd marmot.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20-%20Floyd%20marmot.pdf\">pdf</a>)\r\n\r\nFlores Martnez, J.J., C. H. Floyd, L.G. Herrera, and B. May. Genetic variation and population size of the endangered fishing bat, Myotis vivesi, in Isla Partida. Pp. 187-192 in Contribuciones Mastozoolgicas en Homenaje a Bernardo Villa (V. Snchez-Cordero y R. A. Medelln, eds.). Instituto de Biologa, UNAM; Instituto de Ecologa, UNAM; CONABIO. Mxico. 706 pp.\r\n\r\nLeunda, P.M., R. Miranda, J. Madoz, S. Parmenter, Y. Chen, and B. May. Threatened fishes of the world: Siphateles bicolor snyderi (Miller, 1973) (Cyprinidae). Env. Biol. Fish. 73:109-110 (<a title=\"Publications_2_files/2005 Leunda etal.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20Leunda%20etal.pdf\">pdf</a>)\r\n\r\nRodrigue, K.L., B.P. May, T.R. Famula, and M.E. Delany. Meiotic instability of chicken ultra-long telomeres and mapping of a 2.8 megabase array to the W-sex chromosome. Chrom. Res.13:581-591. (<a title=\"Publications_2_files/2005 - Rodrigue___chicken.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20-%20Rodrigue___chicken.pdf\">pdf</a>)\r\n\r\nTeglas, M.B., B. May, P.R. Crosbie, M.R. Stephens, and W.M. Boyce. Genetic structure of the tick Ornithodoros coriaceus (Acarina: Argasidae) in California, Nevada and Oregon. J. Med. Entomol. 42:247-253.\r\n\r\nWilliamson, K. and B. May. Homogenization of fall-run chinook salmon gene pools in the Central Valley of California, USA. N. Am. J. Fish. Man. 25:9931009. (<a title=\"Publications_2_files/2005 - Williamson chinook homogenize.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20-%20Williamson%20chinook%20homogenize.pdf\">pdf</a>)\r\n\r\nWilliamson, K. and B. May. Inheritance studies implicate a genetic mechanism for apparent sex-reversal in Chinook salmon. Trans. Am. Fish. Soc. 134:12531261. (<a title=\"Publications_2_files/2005 - Williamson sex.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20-%20Williamson%20sex.pdf\">pdf</a>)\r\n\r\n<strong>Later Publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/\">2009-2013</a></li>\r\n</ul>\r\n<strong>Earlier Publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications 2005-2008','','publish','closed','open','','publications-2005-2008','','','2014-03-07 19:55:32','2014-03-07 19:55:32','',12,'http://localhost/~psalessi/Genetics_Lab/?page_id=129',0,'page','',0),(130,1,'2014-02-21 20:47:00','2014-02-21 20:47:00','','Publications 2005-2008','','inherit','closed','open','','129-revision-v1','','','2014-02-21 20:47:00','2014-02-21 20:47:00','',129,'http://localhost/~psalessi/Genetics_Lab/129-revision-v1/',0,'revision','',0),(132,1,'2014-03-07 19:08:53','2014-03-07 19:08:53','<strong>2013</strong>\n\nChen, Y., S. Parmenter, and B. May. Genetic characterization of variation and management implications for the endangeredMohave tui chub. Cons. Gen. 14:11-20. (<a title=\"Publications_files/2013 - Chen Mohave tui chub.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2013%20-%20Chen%20Mohave%20tui%20chub.pdf\">pdf</a>)\n\nMeek, M. H., A. P. Wintzer, N. Shepherd, and B. May. Genetic diversity and reproductive mode in two non-native hydromedusae, <em>Maeotias marginata</em> and <em>Moerisia</em> sp., in the upper San Francisco Estuary, California. Biol. Invasions. 15(1): 199-212. (<a title=\"Publications_files/2012 Meek et al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Meek%20et%20al.pdf\">pdf</a>)\n\nSchreier, A.D., B. Mahardja, and B. May. Hierarchical patterns of population structure in the endangered Fraser River white sturgeon, Acipenser transmontanus, and implications for conservation. Can. J. Fish. &amp; Aquat. Sci. 69:1968-1980. (<a title=\"Publications_files/2013 - Schreier Fraser river white sturgeon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2013%20-%20Schreier%20Fraser%20river%20white%20sturgeon.pdf\">pdf</a>)\n\n<strong>2012</strong>\n\nBaerwald, M.R., B.M. Schreier, G. Schumer, and B. May. Detection of Threatened Delta Smelt in the Gut Contents of the Invasive Mississippi Silverside in the San Francisco Estuary Using TaqMan Assays. Trans. Am. Fish. Soc. 141:1600-1607. (<a title=\"Publications_files/2012 - Baerwald gut contents.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20-%20Baerwald%20gut%20contents.pdf\">pdf</a>)\n\nMahardja, B., B. May, and M. Baerwald. Characterization of 36 additional microsatellite loci in splittail (<em>Pogonichthys macrolepidotus</em>) and cross-amplification in five other native Californian cyprinid species. Cons. Gen. Res. 4(4):917-921. (<a title=\"Publications_files/2012 Meek et al - Climate change.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Mahardja%20et%20al.%20-%20Cyprinid%20Microsat%20note.pdf\">pdf</a>)\n\nMeek, M., A. Wintzer, W. Wetzel, and B. May. Climate change likely to facilitate the invasion of the non-native hydroid, Cordylophora caspia, in the San Francisco Estuary, CA. PLos ONE. 7(10): e46373. doi:10.1371/journal.pone.0046373 (<a title=\"Publications_files/2012 - Petersen lion paw scallop.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Meek%20et%20al%20-%20Climate%20change.pdf\">pdf</a>)\n\nPetersen, J.L., M.R. Baerwald, J.M. Ibarra, and B. May. A first-generation linkage map of the Pacific lion-paw scallop: initial evidence of QTL for size traits and markers linked to orange shell color. Aquaculture. 350-353:200-209. (<a title=\"Publications_files/2012 - Petersen lion paw scallop.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20-%20Petersen%20lion%20paw%20scallop.pdf\">pdf</a>)\n\nSchreier, A. D., J. Rodzen, S. Ireland, and B. May. Genetic techniques inform conservation aquaculture of the endangered Kootenai River white sturgeon, <em>Acipenser transmontanus</em>. Endang. Species Res. 16:65-75. (<a title=\"Publications_files/2012 Schreier et al. - Kootenai.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Schreier%20et%20al.%20-%20Kootenai.pdf \">pdf</a>)\n\nSukmanomon, S., W. Kamonrat, S. Poompuang, T. Nguyen, D. M. Bartley, B. May, U. Na-Nakorn. Genetic changes, intra- and interspecific introgression in farmed Nile tilapia in Thailand. Aquaculture 324-325:44-54. (<a title=\"Publications_files/2012 - Sukmanomon Tilapia.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20-%20Sukmanomon%20%20Tilapia.pdf\">pdf</a>)\n\nVan Dam, A.R. and B. May. A new species of Dactylopius Costa (<em>Dactylopius gracilipilus</em> sp. nov.) (Hemiptera: Coccoidea: Dactylopiidae) from the Chihuahuan Desert, Texas, U.S.A. . Zootaxa 3573:33-39. (<a title=\"Publications_files/2012 - Van Dam new species.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20-%20Van%20Dam%20new%20species.pdf\">pdf</a>)\n\n<strong>2011</strong>\n\nBaerwald, M., J. Petersen, R. Hedrick, G. Schisler, and B. May. A major effect quantitative trait locus for whirling disease resistance identified in rainbow trout (<em>Oncorhynchus mykiss</em>). Heredity. 106:920-926. (<a title=\"Publications_files/2011 - Baerwald et al. - QTL whirling.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20-%20Baerwald%20et%20al.%20-%20QTL%20whirling.pdf\">pdf</a>)\n\nBaerwald, M., G. Schumer, B. Schreier, and B. May. TaqMan assays for the genetic identification of delta smelt (Hypomesus transpacificus) and wakasagi smelt (<em>Hypomesus nipponensis</em>). Mol. Ecol. Res. 11:784-785. (<a title=\"Publications_files/2011 - Baerwald TaqMan assays.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20-%20Baerwald%20TaqMan%20assays.pdf\">pdf</a>)\n\nBaerwald, M.R., M. R. Stephens, K. S. Bork, M. H. Meek, K. Tomalty, and B. May. 2011. Spring-run Chinook salmon genetic management plan  San Joaquin River Restoration Program. 118 pgs. Scientific genetic management plan for the CA Dept. of Fish and Game. (<a title=\"Publications_files/GMP_SpringRunChinook_SJR_UCD_GVL_FINAL.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/GMP_SpringRunChinook_SJR_UCD_GVL_FINAL.pdf\">pdf</a>)\n\nFinger, A. J., E. C. Anderson, M. R. Stephens, and B. P. May. Application of a method for estimating effective population size and admixture using diagnostic SNPs: Implications for conservation of threatened Paiute cutthroat trout (<em>Oncorhynchus clarkii seleniris</em>) in Silver King Creek, California. Can. J. Fish. Aquat. Sci. 68:1369-1386 (<a title=\"Publications_files/2011 - Finger et al. - Effective pop size of Paiute cutthroat.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20-%20Finger%20et%20al.%20-%20Effective%20pop%20size%20of%20Paiute%20cutthroat.pdf\">pdf</a>).\n\nFisch, K.M., J.M. Henderson, R.S. Burton, and B. May. Population genetics and conservation implications for the endangered delta smelt in the San Francisco Bay-Delta. Conservation Genetics. 12:1421-1434 (<a title=\"Publications_files/2011 Fisch et al. Delta smelt pop gen.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20Fisch%20et%20al.%20Delta%20smelt%20pop%20gen.pdf\">pdf</a>).\n\nFloyd, C. H., D.H. Van Vuren, K.R. Crooks, K.L. Jones, D.K. Garcelon, N.M. Belfiore, J.W. Dragoo, and B. May. Genetic differentiation of island spotted skunks, <em>Spilogale gracilis amphiala.</em> J. Mam. 92(1):148-158 (<a title=\"Publications_files/2011 Floyd et al. skunks.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20Floyd%20et%20al.%20skunks.pdf\">pdf</a>).\n\nMcEachern, M.B., D.H. Van Vuren, C.H. Floyd, B. May, and J.M. Eadie.Bottlenecks and rescue effects in a fluctuating population of golden-mantled ground squirrels (<em>Spermophilus lateralis</em>). Con. Gen. 12:285-296 (<a title=\"Publications_files/2011 McEachern ground squirrels.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20McEachern%20ground%20squirrels.pdf\">pdf</a>).\n\nMejia , O., L. G. Herrera, B. May, R. A. Medellin,and J. J. Flores-Martinez. Effective population size dynamics of Myotis vivesi during the Pleistocene and Holocene climatic changes. Acta Chiropterologica. 13:33-40 (<a title=\"Publications_files/2011 Mejia et al .pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20Mejia%20et%20al%20.pdf\">pdf</a>).\n\nSchreier, A.D., D. Gille, B. Mahardja, and B. May. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon,<em> Acipenser transmontanus</em>. J. Appl. Ichthiol. 27:24-33 (<a title=\"Publications_files/2011 Schreier et al. - Octoploid origin sturgeon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20Schreier%20et%20al.%20-%20Octoploid%20origin%20sturgeon.pdf\">pdf</a>).\n\nWintzer, A.P., M.H. Meek, P.B. Moyle, and B. May. Ecological insights into the polyp stage of non-native hydrozoans in the San Francisco Estuary. Aquat. Ecol. 45:151161 (<a title=\"Publications_files/2011 Wintzer - Aquatic Ecology.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20Wintzer%20-%20Aquatic%20Ecology.pdf\">pdf</a>).\n\n<strong>2010</strong>\n\nFloyd, C. H., J.J. Flores Martnez, , L.G. Herrera, O. Mejia, and B. May. Conserving the endangered Mexican fishing bat (<em>Myotis vivesi</em>): genetic variation indicates extensive gene flow among islands in the Gulf of California. Cons. Gen. 11:813-82 (<a title=\"Publications_files/2010 Floyd et al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2010%20Floyd%20et%20al.pdf\">pdf</a>).\n\nIsrael, J.A. and B. May. Indirect estimates of breeding population size in the polyploid green sturgeon (<em>Acipenser medirostris</em>). Mol. Ecol. 19:1058-107 (<a title=\"Publications_files/2010 Israel Mol. Ecol..pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2010%20Israel%20Mol.%20Ecol..pdf\">pdf</a>).\n\nIsrael, J.A. and B. May. Characterization and evaluation of polymorphic microsatellite markers in the anadromous fish <em>Spirinchus thaleichthys</em>. Cons. Gen. Res. online (<a title=\"Publications_files/2010 Israel &amp; May - Spirinchus microsat.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2010%20Israel%20%26%20May%20-%20Spirinchus%20microsat.pdf\">pdf</a>).\n\nPetersen, J.L., J.M. Ibarra, and B. May. Nuclear and mtDNA lineage diversity in wild and cultured Pacific lion-paw scallop (Baja California Peninsula, Mexico). Mar. Biol. 157:27512767 (<a title=\"Publications_files/2010 Petersen et al. wild and cultured.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2010%20Petersen%20et%20al.%20wild%20and%20cultured.pdf\">pdf</a>).\n\n<strong>2009</strong>\n\nChen, Y., S. Reid, and B. May. Genetic relationships among tui chub populations in the Cow Head and surrounding basins of the Northwestern Great Basin. Cons. Gen. 10:101-114 (<a title=\"Publications_files/2009_Chen_great_basin_tui.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009_Chen_great_basin_tui.pdf\">pdf</a>).\n\nFinger, A.J., M.R. Stephens, N.W. Clipperton and B. May. Six diagnostic SNP markers for detecting introgression between cutthroat and rainbow trout. Mol. Ecol. Res. 9:759-763 (<a title=\"Publications_files/2009 - Finger_six_SNPs.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009%20-%20Finger_six_SNPs.pdf\">pdf</a>).\n\nFisch, K., J.L. Petersen, M.R. Baerwald, and B.P. May. Characterization of 24 microsatellite loci in delta smelt, <em>Hypomesus transpacificus</em>, and their cross-species amplification in two other smelt species of the Osmeridae family. Mol. Ecol. Res. 9:405-408 (<a title=\"Publications_files/2009_Fisch_primer_smelt.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009_Fisch_primer_smelt.pdf\">pdf</a>).\n\nIsrael, J.A., K.J. Bando, E.C. Anderson, and B. May. Polyploid microsatellite data reveal stock complexity among estuarine North American green sturgeon (<em>Acipenser medirostris</em>). Can. J. Fish. Aqu. Sci. 66:1491-1504 (<a title=\"Publications_files/2009 Israel green sturgeon stock.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009%20Israel%20green%20sturgeon%20stock.pdf\">pdf</a>).\n\nMeek, M., M. Baerwald, A. Wintzer, and B. May. Isolation and characterization of microsatellite loci in two non-native hydromedusae in the San Francisco Estuary: Maeotias marginata and Moerisia sp. Cons. Gen. Res. 1:205-208 (<a title=\"Publications_files/2009 - Meek et al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009%20-%20Meek%20et%20al.pdf\">pdf</a>).\n\nPetersen, J.L., J.M. Ibarra, and B. May. Thirty-seven additional microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and cross-amplification in other Pectinids. Cons. Gen. Res. 1:101-105 (<a title=\"Publications_files/Petersen et al. 2009 - microsatellite.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/Petersen%20et%20al.%202009%20-%20microsatellite.pdf\">pdf</a>).\n\nSimmons, R. E., P. Lavretsky, and B. May. Introgressive Hybridization of Redband Trout in the Upper McCloud River Watershed. Trans. Am. Fish. Soc. 139:201-213 (<a title=\"Publications_files/2009 - Simmons_redband_introgression.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009%20-%20Simmons_redband_introgression.pdf\">pdf</a>).\n\nStephens, M.R., N.W. Clipperton, and B.P. May. Subspecies-informative TaqMan probes for evaluating introgression between native golden trout and introduced rainbow trout. Mol. Ecol. Res. 9:339-343 (<a title=\"Publications_files/2009_Stephens_SNP_primers.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009_Stephens_SNP_primers.pdf\">pdf</a>).\n\n<strong>Earlier publications:</strong>\n<ul>\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\n</ul>','Publications','','inherit','closed','open','','12-autosave-v1','','','2014-03-07 19:08:53','2014-03-07 19:08:53','',12,'http://localhost/~psalessi/Genetics_Lab/12-autosave-v1/',0,'revision','',0),(133,1,'2014-02-21 21:02:15','2014-02-21 21:02:15','<strong>2013</strong>\r\n\r\nChen, Y., S. Parmenter, and B. May. Genetic characterization of variation and management implications for the endangered Mohave tui chub. Cons. Gen. 14:11-20. (pdf)\r\n\r\nMeek, M. H., A. P. Wintzer, N. Shepherd, and B. May. Genetic diversity and reproductive mode in two non-native hydromedusae, Maeotias marginata and Moerisia sp., in the upper San Francisco Estuary, California. Biol. Invasions. 15(1): 199-212. (pdf)\r\n\r\nSchreier, A.D., B. Mahardja, and B. May. Hierarchical patterns of population structure in the endangered Fraser River white sturgeon, Acipenser transmontanus, and implications for conservation. Can. J. Fish. &amp; Aquat. Sci. 69:1968-1980. (pdf)\r\n\r\n<strong>2012</strong>\r\n\r\nBaerwald, M.R., B.M. Schreier, G. Schumer, and B. May. Detection of Threatened Delta Smelt in the Gut Contents of the Invasive Mississippi Silverside in the San Francisco Estuary Using TaqMan Assays. Trans. Am. Fish. Soc. 141:1600-1607. (pdf)\r\n\r\nMahardja, B., B. May, and M. Baerwald. Characterization of 36 additional microsatellite loci in splittail (Pogonichthys macrolepidotus) and cross-amplification in five other native Californian cyprinid species. Cons. Gen. Res. 4(4):917-921. (pdf)\r\n\r\nMeek, M., A. Wintzer, W. Wetzel, and B. May. Climate change likely to facilitate the invasion of the non-native hydroid, Cordylophora caspia, in the San Francisco Estuary, CA. PLos ONE. 7(10): e46373. doi:10.1371/journal.pone.0046373 (pdf)\r\n\r\nPetersen, J.L., M.R. Baerwald, J.M. Ibarra, and B. May. A first-generation linkage map of the Pacific lion-paw scallop: initial evidence of QTL for size traits and markers linked to orange shell color. Aquaculture. 350-353:200-209. (pdf)\r\n\r\nSchreier, A. D., J. Rodzen, S. Ireland, and B. May. Genetic techniques inform conservation aquaculture of the endangered Kootenai River white sturgeon, Acipenser transmontanus. Endang. Species Res. 16:65-75. (pdf)\r\n\r\nSukmanomon, S., W. Kamonrat, S. Poompuang, T. Nguyen, D. M. Bartley, B. May, U. Na-Nakorn. Genetic changes, intra- and interspecific introgression in farmed Nile tilapia in Thailand. Aquaculture 324-325:44-54. (pdf)\r\n\r\nVan Dam, A.R. and B. May. A new species of Dactylopius Costa (Dactylopius gracilipilus sp. nov.) (Hemiptera: Coccoidea: Dactylopiidae) from the Chihuahuan Desert, Texas, U.S.A. . Zootaxa 3573:33-39. (pdf)\r\n\r\n<strong>2011</strong>\r\n\r\nBaerwald, M., J. Petersen, R. Hedrick, G. Schisler, and B. May. A major effect quantitative trait locus for whirling disease resistance identified in rainbow trout (Oncorhynchus mykiss). Heredity. 106:920-926. (pdf)\r\n\r\nBaerwald, M., G. Schumer, B. Schreier, and B. May. TaqMan assays for the genetic identification of delta smelt (Hypomesus transpacificus) and wakasagi smelt (Hypomesus nipponensis). Mol. Ecol. Res. 11:784-785. (pdf)\r\n\r\nBaerwald, M.R., M. R. Stephens, K. S. Bork, M. H. Meek, K. Tomalty, and B. May. 2011. Spring-run Chinook salmon genetic management plan  San Joaquin River Restoration Program. 118 pgs. Scientific genetic management plan for the CA Dept. of Fish and Game. (pdf)\r\n\r\nFinger, A. J., E. C. Anderson, M. R. Stephens, and B. P. May. Application of a method for estimating effective population size and admixture using diagnostic SNPs: Implications for conservation of threatened Paiute cutthroat trout (Oncorhynchus clarkii seleniris) in Silver King Creek, California. Can. J. Fish. Aquat. Sci. 68:1369-1386 (pdf).\r\n\r\nFisch, K.M., J.M. Henderson, R.S. Burton, and B. May. Population genetics and conservation implications for the endangered delta smelt in the San Francisco Bay-Delta. Conservation Genetics. 12:1421-1434 (pdf).\r\n\r\nFloyd, C. H., D.H. Van Vuren, K.R. Crooks, K.L. Jones, D.K. Garcelon, N.M. Belfiore, J.W. Dragoo, and B. May. Genetic differentiation of island spotted skunks, Spilogale gracilis amphiala. J. Mam. 92(1):148-158 (pdf).\r\n\r\nMcEachern, M.B., D.H. Van Vuren, C.H. Floyd, B. May, and J.M. Eadie.Bottlenecks and rescue effects in a fluctuating population of golden-mantled ground squirrels (Spermophilus lateralis). Con. Gen. 12:285-296 (pdf).\r\n\r\nMejia , O., L. G. Herrera, B. May, R. A. Medellin,and J. J. Flores-Martinez. Effective population size dynamics of Myotis vivesi during the Pleistocene and Holocene climatic changes. Acta Chiropterologica. 13:33-40 (pdf).\r\n\r\nSchreier, A.D., D. Gille, B. Mahardja, and B. May. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon, Acipenser transmontanus. J. Appl. Ichthiol. 27:24-33 (pdf).\r\n\r\nWintzer, A.P., M.H. Meek, P.B. Moyle, and B. May. Ecological insights into the polyp stage of non-native hydrozoans in the San Francisco Estuary. Aquat. Ecol. 45:151161 (pdf).\r\n\r\n<strong>2010</strong>\r\n\r\nFloyd, C. H., J.J. Flores Martnez, , L.G. Herrera, O. Mejia, and B. May. Conserving the endangered Mexican fishing bat (Myotis vivesi): genetic variation indicates extensive gene flow among islands in the Gulf of California. Cons. Gen. 11:813-82 (pdf).\r\n\r\nIsrael, J.A. and B. May. Indirect estimates of breeding population size in the polyploid green sturgeon (Acipenser medirostris). Mol. Ecol. 19:1058-107 (pdf).\r\n\r\nIsrael, J.A. and B. May. Characterization and evaluation of polymorphic microsatellite markers in the anadromous fish Spirinchus thaleichthys. Cons. Gen. Res. online (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Nuclear and mtDNA lineage diversity in wild and cultured Pacific lion-paw scallop (Baja California Peninsula, Mexico). Mar. Biol. 157:27512767 (pdf).\r\n\r\n<strong>2009</strong>\r\n\r\nChen, Y., S. Reid, and B. May. Genetic relationships among tui chub populations in the Cow Head and surrounding basins of the Northwestern Great Basin. Cons. Gen. 10:101-114 (pdf).\r\n\r\nFinger, A.J., M.R. Stephens, N.W. Clipperton and B. May. Six diagnostic SNP markers for detecting introgression between cutthroat and rainbow trout. Mol. Ecol. Res. 9:759-763 (pdf).\r\n\r\nFisch, K., J.L. Petersen, M.R. Baerwald, and B.P. May. Characterization of 24 microsatellite loci in delta smelt, Hypomesus transpacificus, and their cross-species amplification in two other smelt species of the Osmeridae family. Mol. Ecol. Res. 9:405-408 (pdf).\r\n\r\nIsrael, J.A., K.J. Bando, E.C. Anderson, and B. May. Polyploid microsatellite data reveal stock complexity among estuarine North American green sturgeon (Acipenser medirostris). Can. J. Fish. Aqu. Sci. 66:1491-1504 (pdf).\r\n\r\nMeek, M., M. Baerwald, A. Wintzer, and B. May. Isolation and characterization of microsatellite loci in two non-native hydromedusae in the San Francisco Estuary: Maeotias marginata and Moerisia sp. Cons. Gen. Res. 1:205-208 (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Thirty-seven additional microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and cross-amplification in other Pectinids. Cons. Gen. Res. 1:101-105 (pdf).\r\n\r\nSimmons, R. E., P. Lavretsky, and B. May. Introgressive Hybridization of Redband Trout in the Upper McCloud River Watershed. Trans. Am. Fish. Soc. 139:201-213 (pdf).\r\n\r\nStephens, M.R., N.W. Clipperton, and B.P. May. Subspecies-informative TaqMan probes for evaluating introgression between native golden trout and introduced rainbow trout. Mol. Ecol. Res. 9:339-343 (pdf).\r\n\r\n<strong>Earlier publications:</strong>\r\n<ul>\r\n	<li>2005-2008</li>\r\n	<li>2000-2004</li>\r\n	<li>1993-1999</li>\r\n	<li>1990-1992</li>\r\n</ul>\r\n&nbsp;','Publications','','inherit','closed','open','','12-revision-v1','','','2014-02-21 21:02:15','2014-02-21 21:02:15','',12,'http://localhost/~psalessi/Genetics_Lab/12-revision-v1/',0,'revision','',0),(134,1,'2014-02-21 21:04:03','2014-02-21 21:04:03','<strong>2013</strong>\r\n\r\nChen, Y., S. Parmenter, and B. May. Genetic characterization of variation and management implications for the endangeredMohave tui chub. Cons. Gen. 14:11-20. (pdf)\r\n\r\nMeek, M. H., A. P. Wintzer, N. Shepherd, and B. May. Genetic diversity and reproductive mode in two non-native hydromedusae, Maeotias marginata and Moerisia sp., in the upper San Francisco Estuary, California. Biol. Invasions. 15(1): 199-212. (pdf)\r\n\r\nSchreier, A.D., B. Mahardja, and B. May. Hierarchical patterns of population structure in the endangered Fraser River white sturgeon, Acipenser transmontanus, and implications for conservation. Can. J. Fish. &amp; Aquat. Sci. 69:1968-1980. (pdf)\r\n\r\n<strong>2012</strong>\r\n\r\nBaerwald, M.R., B.M. Schreier, G. Schumer, and B. May. Detection of Threatened Delta Smelt in the Gut Contents of the Invasive Mississippi Silverside in the San Francisco Estuary Using TaqMan Assays. Trans. Am. Fish. Soc. 141:1600-1607. (pdf)\r\n\r\nMahardja, B., B. May, and M. Baerwald. Characterization of 36 additional microsatellite loci in splittail (Pogonichthys macrolepidotus) and cross-amplification in five other native Californian cyprinid species. Cons. Gen. Res. 4(4):917-921. (pdf)\r\n\r\nMeek, M., A. Wintzer, W. Wetzel, and B. May. Climate change likely to facilitate the invasion of the non-native hydroid, Cordylophora caspia, in the San Francisco Estuary, CA. PLos ONE. 7(10): e46373. doi:10.1371/journal.pone.0046373 (pdf)\r\n\r\nPetersen, J.L., M.R. Baerwald, J.M. Ibarra, and B. May. A first-generation linkage map of the Pacific lion-paw scallop: initial evidence of QTL for size traits and markers linked to orange shell color. Aquaculture. 350-353:200-209. (pdf)\r\n\r\nSchreier, A. D., J. Rodzen, S. Ireland, and B. May. Genetic techniques inform conservation aquaculture of the endangered Kootenai River white sturgeon, Acipenser transmontanus. Endang. Species Res. 16:65-75. (pdf)\r\n\r\nSukmanomon, S., W. Kamonrat, S. Poompuang, T. Nguyen, D. M. Bartley, B. May, U. Na-Nakorn. Genetic changes, intra- and interspecific introgression in farmed Nile tilapia in Thailand. Aquaculture 324-325:44-54. (pdf)\r\n\r\nVan Dam, A.R. and B. May. A new species of Dactylopius Costa (Dactylopius gracilipilus sp. nov.) (Hemiptera: Coccoidea: Dactylopiidae) from the Chihuahuan Desert, Texas, U.S.A. . Zootaxa 3573:33-39. (pdf)\r\n\r\n<strong>2011</strong>\r\n\r\nBaerwald, M., J. Petersen, R. Hedrick, G. Schisler, and B. May. A major effect quantitative trait locus for whirling disease resistance identified in rainbow trout (Oncorhynchus mykiss). Heredity. 106:920-926. (pdf)\r\n\r\nBaerwald, M., G. Schumer, B. Schreier, and B. May. TaqMan assays for the genetic identification of delta smelt (Hypomesus transpacificus) and wakasagi smelt (Hypomesus nipponensis). Mol. Ecol. Res. 11:784-785. (pdf)\r\n\r\nBaerwald, M.R., M. R. Stephens, K. S. Bork, M. H. Meek, K. Tomalty, and B. May. 2011. Spring-run Chinook salmon genetic management plan  San Joaquin River Restoration Program. 118 pgs. Scientific genetic management plan for the CA Dept. of Fish and Game. (pdf)\r\n\r\nFinger, A. J., E. C. Anderson, M. R. Stephens, and B. P. May. Application of a method for estimating effective population size and admixture using diagnostic SNPs: Implications for conservation of threatened Paiute cutthroat trout (Oncorhynchus clarkii seleniris) in Silver King Creek, California. Can. J. Fish. Aquat. Sci. 68:1369-1386 (pdf).\r\n\r\nFisch, K.M., J.M. Henderson, R.S. Burton, and B. May. Population genetics and conservation implications for the endangered delta smelt in the San Francisco Bay-Delta. Conservation Genetics. 12:1421-1434 (pdf).\r\n\r\nFloyd, C. H., D.H. Van Vuren, K.R. Crooks, K.L. Jones, D.K. Garcelon, N.M. Belfiore, J.W. Dragoo, and B. May. Genetic differentiation of island spotted skunks, Spilogale gracilis amphiala. J. Mam. 92(1):148-158 (pdf).\r\n\r\nMcEachern, M.B., D.H. Van Vuren, C.H. Floyd, B. May, and J.M. Eadie.Bottlenecks and rescue effects in a fluctuating population of golden-mantled ground squirrels (Spermophilus lateralis). Con. Gen. 12:285-296 (pdf).\r\n\r\nMejia , O., L. G. Herrera, B. May, R. A. Medellin,and J. J. Flores-Martinez. Effective population size dynamics of Myotis vivesi during the Pleistocene and Holocene climatic changes. Acta Chiropterologica. 13:33-40 (pdf).\r\n\r\nSchreier, A.D., D. Gille, B. Mahardja, and B. May. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon, Acipenser transmontanus. J. Appl. Ichthiol. 27:24-33 (pdf).\r\n\r\nWintzer, A.P., M.H. Meek, P.B. Moyle, and B. May. Ecological insights into the polyp stage of non-native hydrozoans in the San Francisco Estuary. Aquat. Ecol. 45:151161 (pdf).\r\n\r\n<strong>2010</strong>\r\n\r\nFloyd, C. H., J.J. Flores Martnez, , L.G. Herrera, O. Mejia, and B. May. Conserving the endangered Mexican fishing bat (Myotis vivesi): genetic variation indicates extensive gene flow among islands in the Gulf of California. Cons. Gen. 11:813-82 (pdf).\r\n\r\nIsrael, J.A. and B. May. Indirect estimates of breeding population size in the polyploid green sturgeon (Acipenser medirostris). Mol. Ecol. 19:1058-107 (pdf).\r\n\r\nIsrael, J.A. and B. May. Characterization and evaluation of polymorphic microsatellite markers in the anadromous fish Spirinchus thaleichthys. Cons. Gen. Res. online (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Nuclear and mtDNA lineage diversity in wild and cultured Pacific lion-paw scallop (Baja California Peninsula, Mexico). Mar. Biol. 157:27512767 (pdf).\r\n\r\n<strong>2009</strong>\r\n\r\nChen, Y., S. Reid, and B. May. Genetic relationships among tui chub populations in the Cow Head and surrounding basins of the Northwestern Great Basin. Cons. Gen. 10:101-114 (pdf).\r\n\r\nFinger, A.J., M.R. Stephens, N.W. Clipperton and B. May. Six diagnostic SNP markers for detecting introgression between cutthroat and rainbow trout. Mol. Ecol. Res. 9:759-763 (pdf).\r\n\r\nFisch, K., J.L. Petersen, M.R. Baerwald, and B.P. May. Characterization of 24 microsatellite loci in delta smelt, Hypomesus transpacificus, and their cross-species amplification in two other smelt species of the Osmeridae family. Mol. Ecol. Res. 9:405-408 (pdf).\r\n\r\nIsrael, J.A., K.J. Bando, E.C. Anderson, and B. May. Polyploid microsatellite data reveal stock complexity among estuarine North American green sturgeon (Acipenser medirostris). Can. J. Fish. Aqu. Sci. 66:1491-1504 (pdf).\r\n\r\nMeek, M., M. Baerwald, A. Wintzer, and B. May. Isolation and characterization of microsatellite loci in two non-native hydromedusae in the San Francisco Estuary: Maeotias marginata and Moerisia sp. Cons. Gen. Res. 1:205-208 (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Thirty-seven additional microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and cross-amplification in other Pectinids. Cons. Gen. Res. 1:101-105 (pdf).\r\n\r\nSimmons, R. E., P. Lavretsky, and B. May. Introgressive Hybridization of Redband Trout in the Upper McCloud River Watershed. Trans. Am. Fish. Soc. 139:201-213 (pdf).\r\n\r\nStephens, M.R., N.W. Clipperton, and B.P. May. Subspecies-informative TaqMan probes for evaluating introgression between native golden trout and introduced rainbow trout. Mol. Ecol. Res. 9:339-343 (pdf).\r\n\r\n<strong>Earlier publications:</strong>\r\n<ul>\r\n	<li>2005-2008</li>\r\n	<li>2000-2004</li>\r\n	<li>1993-1999</li>\r\n	<li>1990-1992</li>\r\n</ul>','Publications','','inherit','closed','open','','12-revision-v1','','','2014-02-21 21:04:03','2014-02-21 21:04:03','',12,'http://localhost/~psalessi/Genetics_Lab/12-revision-v1/',0,'revision','',0),(135,1,'2014-02-21 21:05:56','2014-02-21 21:05:56','<strong>2004</strong>\r\n\r\nBaerwald, M.R. and B. May. Characterization of microsatellite loci for five members of the minnow family Cyprinidae found in the Sacramento - San Joaquin Delta. Mol. Ecol. Notes. 4:385-390 (<a title=\"Publications_3_files/2004 - baerwald primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20baerwald%20primer%20note.pdf\">pdf</a>).\r\n\r\nCordes, J.F., J.A. Israel and B. May. Conservation of Paiute cutthroat: the genetic legacy of population transplants in an endemic California salmonid. CA Fish and Game. 90-101-118 (<a title=\"Publications_3_files/2004 - Cordes cutthroat.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Cordes%20cutthroat.pdf\">pdf</a>).\r\n\r\nFlores-Martnez, J.J., C. Floyd, L.G. Herrera, and B. May. Genetic variation and population size in the fishing bat, Myotis vivesi, in Isla Partida. In: Homenaje a Bernardo Villa, Eds., R. Medelln and V. Snchez-Cordero. Publ. Universidad Nacional Autnoma de Mxico. Pp. 185-190 (<a title=\"Publications_3_files/2004_fishing bats.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004_fishing%20bats.pdf\">pdf</a>).\r\n\r\nHogan, Z.S., P.B. Moyle, B. May, M.J. Vander Zanden and I.G. Baird. Imperiled Giants of the Mekong: Ecologists struggle to understandand protectSoutheast Asia\'s large migratory catfish. Am. Sci. 92:228-237b (<a title=\"Publications_3_files/2004 - Hogan Am Sci.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Hogan%20Am%20Sci.pdf\">pdf</a>).\r\n\r\nIsrael, J.A., J.F. Cordes, M.A. Blumberg, and B. May. Geographic patterns of genetic differentiation among collections of green sturgeon. N. Am. J. Fish. Man. 24:922-931 (<a title=\"Publications_3_files/2004 - Israel green sturgeon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Israel%20green%20sturgeon.pdf\">pdf</a>).\r\n\r\nMoen, T., J.J. Agresti, A. Cnaani, H. Moses, T.R. Famula, G. Hulata, G.A.E. Gall, B. May. A genome scan of a four-way tilapia cross supports the existence of a quantitative trait locus for cold tolerance on linkage group 23. Aqua. Res. 35:893-904 (<a title=\"Publications_3_files/2004 - Moen etal.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Moen%20etal.pdf\">pdf</a>).\r\n\r\nRodriguez, F., S. LaPatra, S. Williams, T. Famula and B. May. Genetic markers associated with resistance to infectious hematopoietic necrosis in rainbow and steelhead trout (Oncorhynchus mykiss) backcrosses. Aquaculture 241:93-115 (<a title=\"Publications_3_files/2004 - Rodriguez_et_al[1].pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Rodriguez_et_al%5B1%5D.pdf\">pdf</a>).\r\n\r\nRodzen, J.A., T. Famula, and B. May. Estimation of parentage and relatedness in the polyploid white sturgeon Acipenser transmontanus using a dominant marker approach for duplicated microsatellite loci. Aquaculture 232:165-182 (<a title=\"Publications_3_files/2004 - Rodzen Aquaculture paper.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Rodzen%20Aquaculture%20paper.pdf\">pdf</a>).\r\n\r\nSchwartz, R.S. and B. May. Characterization of microsatellite loci in Sacramento perch (Archoplites interruptus). Mol. Ecol. Notes. 4:694-697 (<a title=\"Publications_3_files/2004 - Schwartz primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Schwartz%20primer%20note.pdf\">pdf</a>).\r\n\r\nStadelmann, B., L.G. Herrera, J. Arroyo-Cabrales, J.J. Flores-Martnez, B.P. May, and M. Ruedi. Molecular systematics of the fishing bat Myotis (Pizonyz) vivesi. J. Mammo. 85:133139 (<a title=\"Publications_3_files/2004 - Stadelmann.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Stadelmann.pdf\">pdf</a>).\r\n\r\nTopinka, J.R., A.J. Donovan, and B. May. Characterization of microsatellite loci in Kearneys bluestar (Amsonia kearneyana) and cross-amplification in other Amsonia species. Mol. Ecol. Notes. 4:710-712 (<a title=\"Publications_3_files/2004 - Topinka Amsonia.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Topinka%20Amsonia.pdf\">pdf</a>).\r\n\r\nTopinka, J.R. and B. May. Development of polymorphic microsatellite loci in the northern goshawk (Accipiter gentilis atricapillus) and cross-amplification in other raptor species. Cons. Gen. 5:861-864 (<a title=\"Publications_3_files/2004 - Topinka goshawk.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Topinka%20goshawk.pdf\">pdf</a>).\r\n\r\nTranah, G.J., D.E. Campton and B. May. Genetic evidence for hybridization of pallid and shovelnose sturgeon. J. Hered. 95:474-480 (<a title=\"Publications_3_files/2004 - Tranah J Heredity.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Tranah%20J%20Heredity.pdf\">pdf</a>).\r\n\r\n<strong>2003</strong>\r\n\r\nBeyer, J. and B. May. A graph-theoretic approach to the partition of individuals into full-sib families. Mol. Ecol. 12:2243-2250 (<a title=\"Publications_3_files/2003 - Beyer-Jen MS.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2003%20-%20Beyer-Jen%20MS.pdf\">pdf</a>).\r\n\r\nErnest, H., W.M. Boyce, V. Bleich, B. May, S. Stiver, and S. Torres. Genetic structure of mountain lion (Puma concolor) populations in California. Cons. Gen. 3:353-366 (<a title=\"Publications_3_files/2003 - Ernest.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2003%20-%20Ernest.pdf\">pdf</a>).\r\n\r\nMay, B. Allozyme variation. In: Population Genetics: Principles and Applications for Fisheries Scientists, Ed. E. Hallerman. Amer. Fish. Soc. Pp. 23-36.\r\n\r\nMcQuown, E., C.C. Krueger, H.L. Kincaid, G.A.E. Gall, and B. May. Genetic comparison of lake sturgeon (Acipenser fulvescens) populations: differentiation based on allelic frequencies at seven microsatellite loci. J. Great Lakes Res. 29:3-13 (<a title=\"Publications_3_files/2003 McQuown_JGLR.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2003%20McQuown_JGLR.pdf\">pdf</a>).\r\n\r\nTranah, G.J., M.J. Bagley, J.J. Agresti, and B. May. Development of codominant markers for identifying species hybrids. Cons. Gen. 4:537-541 (<a title=\"Publications_3_files/2003 - Tranah codominant markers1.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2003%20-%20Tranah%20codominant%20markers1.pdf\">pdf</a>).\r\n\r\nWelsh, A.B., M. Blumberg, and B. May. Identification of microsatellite loci in lake sturgeon, Acipenser fulvescens, and their variability in green sturgeon, A. medirostris. Mol. Ecol. Notes 3:47-55 (<a title=\"Publications_3_files/2003 - Welsh primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2003%20-%20Welsh%20%20primer%20note.pdf\">pdf</a>).\r\n\r\nWhitehead, A., S.L. Anderson, K.M Kuivila, J.L. Roach, and B. May. Genetic variation among interconnected populations of Catostomus occidentalis: implications for distinguishing impacts of contaminants from biogeographic structuring. Mol. Ecol. 12: 2817-2833 (<a title=\"Publications_3_files/2003 - Whitehead etal 1.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2003%20-%20Whitehead%20etal%201.pdf\">pdf</a>).\r\n\r\n<strong>2002</strong>\r\n\r\nBlankenship, S.M., B. May, and D. Hedgecock. Evolution of a perfect simple-sequence-repeat locus in the context of its flanking sequence. Mol. Biol. Evol. 19:1943-1951 (<a title=\"Publications_3_files/2002 Blankenship, May.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20Blankenship,%20May.pdf\">pdf</a>).\r\n\r\nHogan, Z. S. and B. May. Twenty-seven new microsatellites for the migratory Asian catfish family Pangasiidae. Mol. Ecol. Notes. 2:38-41 (<a title=\"Publications_3_files/2002 - Hogan primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Hogan%20primer%20note.pdf\">pdf</a>).\r\n\r\nMcQuown, E., G.A.E. Gall, and B. May. Characterization and inheritance of six microsatellite loci in lake sturgeon (Acipenser fulvescens). Trans. Am. Fish. Soc. 131:299-307.\r\n\r\nMeredith, E.P. and B. May. Microsatellite loci in the Lahontan tui chub, Gila biocolor obesa, and their utililization in other chub species. Mol. Ecol. Notes. 2:156-158 (<a title=\"Publications_3_files/2002 - Meredith primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Meredith%20primer%20note.pdf\">pdf</a>).\r\n\r\nRodzen, J.A. and B. May. Inheritance of microsatellite loci in the white sturgeon (Acipenser transmontanus). Genome 45:1064-1076 (<a title=\"Publications_3_files/2002 - Rodzen and May Genome.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Rodzen%20and%20May%20Genome.pdf\">pdf</a>).\r\n\r\nSmith, C.T., R.J. Nelson, S. Pollard, E. Rubridge, S.J. McKay, J. Rodzen, B. May, and B. Koop. Population genetic analysis of white sturgeon (Acipenser transmontanus) in the Fraser River. J. Appl. Icthy. 18:307-312 (<a title=\"Publications_3_files/2002 - Acipenser transmontanus.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Acipenser%20transmontanus.pdf\">pdf</a>).\r\n\r\nWilliamson, K.S., J.F. Cordes, and B. May. Characterization of microsatellite loci in Chinook salmon (Oncorhynchus tshawytscha) and cross-species amplification in other salmonids. Mol. Ecol. Notes. 2:17-19 (<a title=\"Publications_3_files/2002 - Williamson primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Williamson%20primer%20note.pdf\">pdf</a>).\r\n\r\nWilliamson, K. and B. May. Incidence of phenotypic female chinook salmon (Oncorhynchus tshawytscha) positive for the Y-chromosome specific marker OtY1 in the Central Valley. J. Aquat. An. Health. 176-183 (<a title=\"Publications_3_files/2002 - Williamson_and_May_JAAH14.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Williamson_and_May_JAAH14.pdf\">pdf</a>).\r\n\r\nZhu, B., F. Zhou, H. Cao, Z. Shao, N. Zhao, B. May, and J. Chang. Analysis of genetic variation in the Chinese sturgeon, Acipenser sinensis: estimating the contribution of artificially produced larvae in a wild population J. Appl. Icthy. 18:301-306 (<a title=\"Publications_3_files/2002 - Acipenser sinensis.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Acipenser%20sinensis.pdf\">pdf</a>).\r\n\r\n<strong>2001</strong>\r\n\r\nBagley, M.J., S. Anderson, and B. May. Choice of methodology for assessing genetic impacts of environmental stressors: polymorphism and reproducibility of RAPD and AFLP fingerprints. Ecotoxicology 10:239-244 (<a title=\"Publications_3_files/2001 - RAPD and AFLP Fingerprints.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2001%20-%20RAPD%20and%20AFLP%20Fingerprints.pdf\">pdf</a>).\r\n\r\nJenneckens I., J.-N. Meyer, G. Hrstgen-Schwark, B. May, L. Debus, A. Ludwig. A fixed allele at microsatellite LS-39 exhibiting species-specificity for black caviar producer Acipenser stellatus. J. Appl. Icthyol. 17:39-42 (<a title=\"Publications_3_files/2001 - Acipenser stellatus.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2001%20-%20Acipenser%20stellatus.pdf\">pdf</a>).\r\n\r\nPyatskowit, J., C.C. Krueger, H.L. Kincaid, and B. May. Inheritance of microsatellite loci in the polyploid derivative lake sturgeon (Acipenser fulvescens). Genome 44:185-191 (<a title=\"Publications_3_files/2001 - pyatskowit.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2001%20-%20pyatskowit.pdf\">pdf</a>).\r\n\r\nTranah, G.J., J.J. Agresti, and B. May. New microsatellite loci for suckers (Catostomidae): primer homology in Catostomus, Chasmistes, and Deltistes. Mol. Ecol. Notes 1:55-60 (<a title=\"Publications_3_files/2001 - Tranah primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2001%20-%20Tranah%20primer%20note.pdf\">pdf</a>).\r\n\r\nTranah, G.J., H.L. Kincaid, C.C. Krueger, D.E. Campton, B. May. Reproductive isolation in sympatric populations of pallid and shovelnose sturgeon. N. Am. J. Fish. Man. 21:367-373 (<a title=\"Publications_3_files/2001 - Tranah_sturgeonEtAl_NAJFM.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2001%20-%20Tranah_sturgeonEtAl_NAJFM.pdf\">pdf</a>).\r\n\r\n<strong>2000</strong>\r\n\r\n<span style=\"line-height: 1.714285714; font-size: 1rem;\">Agresti, J.J., S. Seki, A. Cnaani, S. Poompuang, E. M. Hallerman, N. Umiel, G. Hulata, G. A.E. Gall, and B. May. Breeding new strains of tilapia: development of an artificial center of origin and linkage map based on AFLP and microsatellite loci. Aquaculture. 185:43-56 (<a title=\"Publications_3_files/2000 - Tilapia.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2000%20-%20Tilapia.pdf\">pdf</a>).</span>\r\n\r\nBelfiore, N.M. and B. May. Variable microsatellite loci in red swamp crayfish, Procambarus clarkii, and their characterization in other crayfish taxa. Mol. Ecol. 12:2231-2234 (<a title=\"Publications_3_files/2000 - Belfiore_red swamp crayfish.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2000%20-%20Belfiore_red%20swamp%20crayfish.pdf\">pdf</a>).\r\n\r\nErnest, H.B., M.C.T. Penedo, B. May, M. Syvanen, and W.M. Boyce. Molecular tracking of mountain lions in the Yosemite Valley region in California: genetic analysis using microsatellites and fecal DNA. Mol. Ecol. 9: 433-441 (<a title=\"Publications_3_files/2000 - Ernest_Mountain Lions.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2000%20-%20Ernest_Mountain%20Lions.pdf\">pdf</a>).\r\n\r\nLudwig, A., B. May, L. Debus, and I, Jenneckens. Heteroplasmy in the mtDNA Control Region of Sturgeon (Acipenser, Huso and Scaphirhynchus) Genetics 156:1933-1947 (<a title=\"Publications_3_files/2000 - Ludwig_sturgeon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2000%20-%20Ludwig_sturgeon.pdf\">pdf</a>).\r\n\r\nMcQuown, E.C., B.L. Sloss, R.J. Sheehan, J. Rodzen, G. Tranah, and B. May. Microsatellite analysis of genetic variation in sturgeon: new primer sequences for Scaphirynchus and Acipenser. Trans. Am. Fish. Soc. 129:1380-1388 (<a title=\"Publications_3_files/2000 - McQuowntafs1380.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2000%20-%20McQuowntafs1380.pdf\">pdf</a>).\r\n\r\n<strong>Later Publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/\">2009-2013</a></li>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n</ul>\r\n<strong>Earlier Publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications 2000-2004','','publish','closed','open','','publications-2000-2004','','','2014-03-07 20:12:27','2014-03-07 20:12:27','',12,'http://localhost/~psalessi/Genetics_Lab/?page_id=135',0,'page','',0),(136,1,'2014-02-21 21:05:56','2014-02-21 21:05:56','<strong>2004</strong>\r\n\r\nBaerwald, M.R. and B. May. Characterization of microsatellite loci for five members of the minnow family Cyprinidae found in the Sacramento - San Joaquin Delta. Mol. Ecol. Notes. 4:385-390 (pdf).\r\n\r\nCordes, J.F., J.A. Israel and B. May. Conservation of Paiute cutthroat: the genetic legacy of population transplants in an endemic California salmonid. CA Fish and Game. 90-101-118 (pdf).\r\n\r\nFlores-Martnez, J.J., C. Floyd, L.G. Herrera, and B. May. Genetic variation and population size in the fishing bat, Myotis vivesi, in Isla Partida. In: Homenaje a Bernardo Villa, Eds., R. Medelln and V. Snchez-Cordero. Publ. Universidad Nacional Autnoma de Mxico. Pp. 185-190 (pdf).\r\n\r\nHogan, Z.S., P.B. Moyle, B. May, M.J. Vander Zanden and I.G. Baird. Imperiled Giants of the Mekong: Ecologists struggle to understandand protectSoutheast Asia\'s large migratory catfish. Am. Sci. 92:228-237b (pdf).\r\n\r\nIsrael, J.A., J.F. Cordes, M.A. Blumberg, and B. May. Geographic patterns of genetic differentiation among collections of green sturgeon. N. Am. J. Fish. Man. 24:922-931 (pdf).\r\n\r\nMoen, T., J.J. Agresti, A. Cnaani, H. Moses, T.R. Famula, G. Hulata, G.A.E. Gall, B. May. A genome scan of a four-way tilapia cross supports the existence of a quantitative trait locus for cold tolerance on linkage group 23. Aqua. Res. 35:893-904 (pdf).\r\n\r\nRodriguez, F., S. LaPatra, S. Williams, T. Famula and B. May. Genetic markers associated with resistance to infectious hematopoietic necrosis in rainbow and steelhead trout (Oncorhynchus mykiss) backcrosses. Aquaculture 241:93-115 (pdf).\r\n\r\nRodzen, J.A., T. Famula, and B. May. Estimation of parentage and relatedness in the polyploid white sturgeon Acipenser transmontanus using a dominant marker approach for duplicated microsatellite loci. Aquaculture 232:165-182 (pdf).\r\n\r\nSchwartz, R.S. and B. May. Characterization of microsatellite loci in Sacramento perch (Archoplites interruptus). Mol. Ecol. Notes. 4:694-697 (pdf).\r\n\r\nStadelmann, B., L.G. Herrera, J. Arroyo-Cabrales, J.J. Flores-Martnez, B.P. May, and M. Ruedi. Molecular systematics of the fishing bat Myotis (Pizonyz) vivesi. J. Mammo. 85:133139 (pdf).\r\n\r\nTopinka, J.R., A.J. Donovan, and B. May. Characterization of microsatellite loci in Kearneys bluestar (Amsonia kearneyana) and cross-amplification in other Amsonia species. Mol. Ecol. Notes. 4:710-712 (pdf).\r\n\r\nTopinka, J.R. and B. May. Development of polymorphic microsatellite loci in the northern goshawk (Accipiter gentilis atricapillus) and cross-amplification in other raptor species. Cons. Gen. 5:861-864 (pdf).\r\n\r\nTranah, G.J., D.E. Campton and B. May. Genetic evidence for hybridization of pallid and shovelnose sturgeon. J. Hered. 95:474-480 (pdf).\r\n\r\n<strong>2003</strong>\r\n\r\nBeyer, J. and B. May. A graph-theoretic approach to the partition of individuals into full-sib families. Mol. Ecol. 12:2243-2250 (pdf).\r\n\r\nErnest, H., W.M. Boyce, V. Bleich, B. May, S. Stiver, and S. Torres. Genetic structure of mountain lion (Puma concolor) populations in California. Cons. Gen. 3:353-366 (pdf).\r\n\r\nMay, B. Allozyme variation. In: Population Genetics: Principles and Applications for Fisheries Scientists, Ed. E. Hallerman. Amer. Fish. Soc. Pp. 23-36.\r\n\r\nMcQuown, E., C.C. Krueger, H.L. Kincaid, G.A.E. Gall, and B. May. Genetic comparison of lake sturgeon (Acipenser fulvescens) populations: differentiation based on allelic frequencies at seven microsatellite loci. J. Great Lakes Res. 29:3-13 (pdf).\r\n\r\nTranah, G.J., M.J. Bagley, J.J. Agresti, and B. May. Development of codominant markers for identifying species hybrids. Cons. Gen. 4:537-541 (pdf).\r\n\r\nWelsh, A.B., M. Blumberg, and B. May. Identification of microsatellite loci in lake sturgeon, Acipenser fulvescens, and their variability in green sturgeon, A. medirostris. Mol. Ecol. Notes 3:47-55 (pdf).\r\n\r\nWhitehead, A., S.L. Anderson, K.M Kuivila, J.L. Roach, and B. May. Genetic variation among interconnected populations of Catostomus occidentalis: implications for distinguishing impacts of contaminants from biogeographic structuring. Mol. Ecol. 12: 2817-2833 (pdf).\r\n\r\n<strong>2002</strong>\r\n\r\nBlankenship, S.M., B. May, and D. Hedgecock. Evolution of a perfect simple-sequence-repeat locus in the context of its flanking sequence. Mol. Biol. Evol. 19:1943-1951 (pdf).\r\n\r\nHogan, Z. S. and B. May. Twenty-seven new microsatellites for the migratory Asian catfish family Pangasiidae. Mol. Ecol. Notes. 2:38-41 (pdf).\r\n\r\nMcQuown, E., G.A.E. Gall, and B. May. Characterization and inheritance of six microsatellite loci in lake sturgeon (Acipenser fulvescens). Trans. Am. Fish. Soc. 131:299-307.\r\n\r\nMeredith, E.P. and B. May. Microsatellite loci in the Lahontan tui chub, Gila biocolor obesa, and their utililization in other chub species. Mol. Ecol. Notes. 2:156-158 (pdf).\r\n\r\nRodzen, J.A. and B. May. Inheritance of microsatellite loci in the white sturgeon (Acipenser transmontanus). Genome 45:1064-1076 (pdf).\r\n\r\nSmith, C.T., R.J. Nelson, S. Pollard, E. Rubridge, S.J. McKay, J. Rodzen, B. May, and B. Koop. Population genetic analysis of white sturgeon (Acipenser transmontanus) in the Fraser River. J. Appl. Icthy. 18:307-312 (pdf).\r\n\r\nWilliamson, K.S., J.F. Cordes, and B. May. Characterization of microsatellite loci in Chinook salmon (Oncorhynchus tshawytscha) and cross-species amplification in other salmonids. Mol. Ecol. Notes. 2:17-19 (pdf).\r\n\r\nWilliamson, K. and B. May. Incidence of phenotypic female chinook salmon (Oncorhynchus tshawytscha) positive for the Y-chromosome specific marker OtY1 in the Central Valley. J. Aquat. An. Health. 176-183 (pdf).\r\n\r\nZhu, B., F. Zhou, H. Cao, Z. Shao, N. Zhao, B. May, and J. Chang. Analysis of genetic variation in the Chinese sturgeon, Acipenser sinensis: estimating the contribution of artificially produced larvae in a wild population J. Appl. Icthy. 18:301-306 (pdf).\r\n\r\n<strong>2001</strong>\r\n\r\nBagley, M.J., S. Anderson, and B. May. Choice of methodology for assessing genetic impacts of environmental stressors: polymorphism and reproducibility of RAPD and AFLP fingerprints. Ecotoxicology 10:239-244 (pdf).\r\n\r\nJenneckens I., J.-N. Meyer, G. Hrstgen-Schwark, B. May, L. Debus, A. Ludwig. A fixed allele at microsatellite LS-39 exhibiting species-specificity for black caviar producer Acipenser stellatus. J. Appl. Icthyol. 17:39-42 (pdf).\r\n\r\nPyatskowit, J., C.C. Krueger, H.L. Kincaid, and B. May. Inheritance of microsatellite loci in the polyploid derivative lake sturgeon (Acipenser fulvescens). Genome 44:185-191 (pdf).\r\n\r\nTranah, G.J., J.J. Agresti, and B. May. New microsatellite loci for suckers (Catostomidae): primer homology in Catostomus, Chasmistes, and Deltistes. Mol. Ecol. Notes 1:55-60 (pdf).\r\n\r\nTranah, G.J., H.L. Kincaid, C.C. Krueger, D.E. Campton, B. May. Reproductive isolation in sympatric populations of pallid and shovelnose sturgeon. N. Am. J. Fish. Man. 21:367-373 (pdf).\r\n\r\n<strong>2000</strong>\r\n\r\n<span style=\"line-height: 1.714285714; font-size: 1rem;\">Agresti, J.J., S. Seki, A. Cnaani, S. Poompuang, E. M. Hallerman, N. Umiel, G. Hulata, G. A.E. Gall, and B. May. Breeding new strains of tilapia: development of an artificial center of origin and linkage map based on AFLP and microsatellite loci. Aquaculture. 185:43-56 (pdf).</span>\r\n\r\nBelfiore, N.M. and B. May. Variable microsatellite loci in red swamp crayfish, Procambarus clarkii, and their characterization in other crayfish taxa. Mol. Ecol. 12:2231-2234 (pdf).\r\n\r\nErnest, H.B., M.C.T. Penedo, B. May, M. Syvanen, and W.M. Boyce. Molecular tracking of mountain lions in the Yosemite Valley region in California: genetic analysis using microsatellites and fecal DNA. Mol. Ecol. 9: 433-441 (pdf).\r\n\r\nLudwig, A., B. May, L. Debus, and I, Jenneckens. Heteroplasmy in the mtDNA Control Region of Sturgeon (Acipenser, Huso and Scaphirhynchus) Genetics 156:1933-1947 (pdf).\r\n\r\nMcQuown, E.C., B.L. Sloss, R.J. Sheehan, J. Rodzen, G. Tranah, and B. May. Microsatellite analysis of genetic variation in sturgeon: new primer sequences for Scaphirynchus and Acipenser. Trans. Am. Fish. Soc. 129:1380-1388 (pdf).\r\n\r\n&nbsp;\r\n\r\n&nbsp;','Publications 2000-2004','','inherit','closed','open','','135-revision-v1','','','2014-02-21 21:05:56','2014-02-21 21:05:56','',135,'http://localhost/~psalessi/Genetics_Lab/135-revision-v1/',0,'revision','',0),(137,1,'2014-02-21 21:07:04','2014-02-21 21:07:04','<strong>2008</strong>\r\n\r\nBaerwald, M., V. Bien, F. Feyrer, and B. May. Distribution of genetically differentiated splittail populations during nonspawning season. Trans. Am. Fish. Soc. 137:1335-1345 (pdf).\r\n\r\nBaerwald, M., A. Welsh, R. Hedrick, and B. May. Discovery of genes implicated in whirling disease infection and resistance in rainbow trout using genome-wide expression profiling. BMC Genomics 9:37 (pdf).\r\n\r\nBrk, K.S, A. Drauch, J. Israel, J. Pedroia, J. Rodzen, and B. May. Development of new microsatellite primers for green and white sturgeon. Cons. Gen. 9:973-979 (pdf)\r\n\r\nPetersen, J. L., A. M. Ibarra, J. L. Ramirez, and B. May. An induced mass spawn of the hermaphroditic lion-paw scallop: Genetic assignment of maternal and paternal parentage. J. Hered. 99:337-348 (pdf).\r\n\r\nSchwartz, R.S. and B. May. Genetic Evaluation of Isolated Populations for Use in Reintroductions Reveals Significant Genetic Bottlenecks in Potential Stocks of Sacramento Perch. Trans. Am. Fish. Soc. 137:1764-1777 (pdf).\r\n\r\nWelsh, A., T. Hill, H. Quinlan, C. Robinson, and B. May. Genetic assessment of lake sturgeon population structure in the Great Lakes. N. Am. J. Fish. Man. 28:572591 (pdf).\r\n\r\nWilliamson, K.S., R. Phillips, and B. May. Characterization of a chromosomal rearrangement responsible for producing \"apparent\" XY-female fall Chinook salmon in California. J. Hered. 99:483-490 (pdf).\r\n\r\n<strong>2007</strong>\r\n\r\nBaerwald, M.R., V. Bien, F. Feyrer, and B. May. Genetic analysis reveals two distinct Sacramento splittail (Pogonichthyes macrolepidotus) populations. Cons. Gen. 8:159-167. (pdf).\r\n\r\nChen, Y., S. Parmenter, and B. May. Introgression between Lahontan and endangered Owens tui chubs, and apparent discovery of a new tui chub in the Owens Valley, California. Conserv. Genet. 8:221-238. (pdf).\r\n\r\nFeyrer, F., J. Hobbs, M. Baerwald, T. Sommer, Q. Yin, K. Clark, B. May and W. Bennett. Otolith microchemistry provides information complementary to microsatellite DNA for migratory fish. Trans. Amer. Fish. Soc. 136:469-476. (pdf)\r\n\r\nHull, J.M., D. Tufts, J. Topinka, B. May, and H. Ernest. Development of 19 microsatellite loci for Swainson\'s hawks (Buteo swainsoni) and other buteos. Molecular Ecology Notes 7:346-349. (pdf).\r\n\r\nLindley, S.T., R. Schick, E. Mora, P. Adams, J. Anderson, S. Greene, C. Hanson, B. May, D. McEwan, R. MacFarlane, C. Swanson, and J. Williams. Framework for Assessing Viability of Threatened and Endangered Chinook Salmon and Steelhead in the Sacramento-San Joaquin Basin . San Francisco Estuary and Watershed Science. Vol. 5, Issue 1 (1-26), Article 4. (pdf).\r\n\r\nMeredith, E. P., J.A. Rodzen, J. D. Banks, R. Schaefer, H. B. Ernest, T. R. Famula and B. P. May. Microsatellite analysis of three subspecies of Elk (Cervus elaphus) in California. Journal of Mammalogy 88(3):801-808. (pdf)\r\n\r\nPapa R., J.A. Israel, F. N. Marzan, and B. May. Assessment of genetic variation among reproductive ecotypes of Klamath River steelhead reveals differentiation associated with different run-timings. J. Appl. Icthy. (pdf)\r\n\r\nWilliamson, K. and B. May. Mitochondrial DNA Haplotype Diversity in Apparent XY Female Fall-Run and Spring-Run Chinook Salmon in California\'s Central Valley. Trans. Amer. Fish. Soc. 136: 1480-1486 (pdf).\r\n\r\n<strong>2006</strong>\r\n\r\nCordes, J.F., M.R. Stephens, M.A. Blumberg, and B. May. Identifying introgressive hybridization in native populations of California Golden Trout based on molecular markers. Trans. Am. Fish. Soc. 135:110-128. (pdf).\r\n\r\nIbarra, A.M., J.L. Petersen, T.R. Famula, and B. May. Characterization of 35 microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and their cross-species amplification in four other scallops of the Pectinidae family. Mol. Ecol. Notes. 6:153-156 (pdf).\r\n\r\nLindley, S.T., R.S. Schick, A. Agrawal, M. Goslin, T. Pearson, E. Mora, J. J. Anderson, B.P. May, S. Greene, C. Hanson, A. Low, D. McEwan, R.B. MacFarlane, C. Swanson and J.G. Williams. Historical population structure of Central Valley steelhead and its alteration by dams. SF Est. Wtshd. Sci. 4(1): Article 3 (pdf).\r\n\r\nSprowles, A.E., M.R. Stephens, N.W. Clipperton, and B.P. May. Fishing for SNPs: a targeted locus approach for Single Nucleotide Polymorphism discovery in rainbow trout. Trans. Am. Fish. Soc. 135:1698-1721. (pdf).\r\n\r\nTranah, G.J., and B. P. May. Patterns of Intra- and Interspecies Genetic Diversity in Klamath River Basin Suckers. Trans. Am Fish. Soc. 135:306316 (pdf).\r\n\r\nWelsh, A. and B. May. Development and standardization of disomic microsatellite markers for lake sturgeon genetic studies. J. Appl. Icthy. (pdf).\r\n\r\n&nbsp;\r\n\r\n<strong>2005</strong>\r\n\r\nBayard de Volo, S., RT Reynolds, JR Topinka, B. May and MF Antolin. Population genetic structure and the utility of muliti-locus genotyping for mark-recapture studies of northern goshawks (Accipiter gentilis). J. Raptor Res. 39:286-295. (pdf)\r\n\r\nCordes, J.F., D.L. Perkins, H.L.Kincaid, and B. May. Genetic analysis of fish genomes and populations: allozyme variation within and among Atlantic salmon from downeast rivers of Maine. J. Fish Biology. 67:104-117. (pdf)\r\n\r\nFloyd, C.H., D. Van Vuren, and B. May. Marmots on Great Basin mountaintops: using genetics to test an ecological paradigm. Ecology. 86: 2145-2153. (pdf)\r\n\r\nFlores Martnez, J.J., C. H. Floyd, L.G. Herrera, and B. May. Genetic variation and population size of the endangered fishing bat, Myotis vivesi, in Isla Partida. Pp. 187-192 in Contribuciones Mastozoolgicas en Homenaje a Bernardo Villa (V. Snchez-Cordero y R. A. Medelln, eds.). Instituto de Biologa, UNAM; Instituto de Ecologa, UNAM; CONABIO. Mxico. 706 pp.\r\n\r\nLeunda, P.M., R. Miranda, J. Madoz, S. Parmenter, Y. Chen, and B. May. Threatened fishes of the world: Siphateles bicolor snyderi (Miller, 1973) (Cyprinidae). Env. Biol. Fish. 73:109-110 (pdf)\r\n\r\nRodrigue, K.L., B.P. May, T.R. Famula, and M.E. Delany. Meiotic instability of chicken ultra-long telomeres and mapping of a 2.8 megabase array to the W-sex chromosome. Chrom. Res.13:581-591. (pdf)\r\n\r\nTeglas, M.B., B. May, P.R. Crosbie, M.R. Stephens, and W.M. Boyce. Genetic structure of the tick Ornithodoros coriaceus (Acarina: Argasidae) in California, Nevada and Oregon. J. Med. Entomol. 42:247-253.\r\n\r\nWilliamson, K. and B. May. Homogenization of fall-run chinook salmon gene pools in the Central Valley of California, USA. N. Am. J. Fish. Man. 25:9931009. (pdf)\r\n\r\nWilliamson, K. and B. May. Inheritance studies implicate a genetic mechanism for apparent sex-reversal in Chinook salmon. Trans. Am. Fish. Soc. 134:12531261. (pdf)','Publications 2005-2008','','inherit','closed','open','','129-revision-v1','','','2014-02-21 21:07:04','2014-02-21 21:07:04','',129,'http://localhost/~psalessi/Genetics_Lab/129-revision-v1/',0,'revision','',0),(138,1,'2014-02-21 21:07:18','2014-02-21 21:07:18','<strong>2008</strong>\r\n\r\nBaerwald, M., V. Bien, F. Feyrer, and B. May. Distribution of genetically differentiated splittail populations during nonspawning season. Trans. Am. Fish. Soc. 137:1335-1345 (pdf).\r\n\r\nBaerwald, M., A. Welsh, R. Hedrick, and B. May. Discovery of genes implicated in whirling disease infection and resistance in rainbow trout using genome-wide expression profiling. BMC Genomics 9:37 (pdf).\r\n\r\nBrk, K.S, A. Drauch, J. Israel, J. Pedroia, J. Rodzen, and B. May. Development of new microsatellite primers for green and white sturgeon. Cons. Gen. 9:973-979 (pdf)\r\n\r\nPetersen, J. L., A. M. Ibarra, J. L. Ramirez, and B. May. An induced mass spawn of the hermaphroditic lion-paw scallop: Genetic assignment of maternal and paternal parentage. J. Hered. 99:337-348 (pdf).\r\n\r\nSchwartz, R.S. and B. May. Genetic Evaluation of Isolated Populations for Use in Reintroductions Reveals Significant Genetic Bottlenecks in Potential Stocks of Sacramento Perch. Trans. Am. Fish. Soc. 137:1764-1777 (pdf).\r\n\r\nWelsh, A., T. Hill, H. Quinlan, C. Robinson, and B. May. Genetic assessment of lake sturgeon population structure in the Great Lakes. N. Am. J. Fish. Man. 28:572591 (pdf).\r\n\r\nWilliamson, K.S., R. Phillips, and B. May. Characterization of a chromosomal rearrangement responsible for producing \"apparent\" XY-female fall Chinook salmon in California. J. Hered. 99:483-490 (pdf).\r\n\r\n<strong>2007</strong>\r\n\r\nBaerwald, M.R., V. Bien, F. Feyrer, and B. May. Genetic analysis reveals two distinct Sacramento splittail (Pogonichthyes macrolepidotus) populations. Cons. Gen. 8:159-167. (pdf).\r\n\r\nChen, Y., S. Parmenter, and B. May. Introgression between Lahontan and endangered Owens tui chubs, and apparent discovery of a new tui chub in the Owens Valley, California. Conserv. Genet. 8:221-238. (pdf).\r\n\r\nFeyrer, F., J. Hobbs, M. Baerwald, T. Sommer, Q. Yin, K. Clark, B. May and W. Bennett. Otolith microchemistry provides information complementary to microsatellite DNA for migratory fish. Trans. Amer. Fish. Soc. 136:469-476. (pdf)\r\n\r\nHull, J.M., D. Tufts, J. Topinka, B. May, and H. Ernest. Development of 19 microsatellite loci for Swainson\'s hawks (Buteo swainsoni) and other buteos. Molecular Ecology Notes 7:346-349. (pdf).\r\n\r\nLindley, S.T., R. Schick, E. Mora, P. Adams, J. Anderson, S. Greene, C. Hanson, B. May, D. McEwan, R. MacFarlane, C. Swanson, and J. Williams. Framework for Assessing Viability of Threatened and Endangered Chinook Salmon and Steelhead in the Sacramento-San Joaquin Basin . San Francisco Estuary and Watershed Science. Vol. 5, Issue 1 (1-26), Article 4. (pdf).\r\n\r\nMeredith, E. P., J.A. Rodzen, J. D. Banks, R. Schaefer, H. B. Ernest, T. R. Famula and B. P. May. Microsatellite analysis of three subspecies of Elk (Cervus elaphus) in California. Journal of Mammalogy 88(3):801-808. (pdf)\r\n\r\nPapa R., J.A. Israel, F. N. Marzan, and B. May. Assessment of genetic variation among reproductive ecotypes of Klamath River steelhead reveals differentiation associated with different run-timings. J. Appl. Icthy. (pdf)\r\n\r\nWilliamson, K. and B. May. Mitochondrial DNA Haplotype Diversity in Apparent XY Female Fall-Run and Spring-Run Chinook Salmon in California\'s Central Valley. Trans. Amer. Fish. Soc. 136: 1480-1486 (pdf).\r\n\r\n<strong>2006</strong>\r\n\r\nCordes, J.F., M.R. Stephens, M.A. Blumberg, and B. May. Identifying introgressive hybridization in native populations of California Golden Trout based on molecular markers. Trans. Am. Fish. Soc. 135:110-128. (pdf).\r\n\r\nIbarra, A.M., J.L. Petersen, T.R. Famula, and B. May. Characterization of 35 microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and their cross-species amplification in four other scallops of the Pectinidae family. Mol. Ecol. Notes. 6:153-156 (pdf).\r\n\r\nLindley, S.T., R.S. Schick, A. Agrawal, M. Goslin, T. Pearson, E. Mora, J. J. Anderson, B.P. May, S. Greene, C. Hanson, A. Low, D. McEwan, R.B. MacFarlane, C. Swanson and J.G. Williams. Historical population structure of Central Valley steelhead and its alteration by dams. SF Est. Wtshd. Sci. 4(1): Article 3 (pdf).\r\n\r\nSprowles, A.E., M.R. Stephens, N.W. Clipperton, and B.P. May. Fishing for SNPs: a targeted locus approach for Single Nucleotide Polymorphism discovery in rainbow trout. Trans. Am. Fish. Soc. 135:1698-1721. (pdf).\r\n\r\nTranah, G.J., and B. P. May. Patterns of Intra- and Interspecies Genetic Diversity in Klamath River Basin Suckers. Trans. Am Fish. Soc. 135:306316 (pdf).\r\n\r\nWelsh, A. and B. May. Development and standardization of disomic microsatellite markers for lake sturgeon genetic studies. J. Appl. Icthy. (pdf).\r\n\r\n<strong>2005</strong>\r\n\r\nBayard de Volo, S., RT Reynolds, JR Topinka, B. May and MF Antolin. Population genetic structure and the utility of muliti-locus genotyping for mark-recapture studies of northern goshawks (Accipiter gentilis). J. Raptor Res. 39:286-295. (pdf)\r\n\r\nCordes, J.F., D.L. Perkins, H.L.Kincaid, and B. May. Genetic analysis of fish genomes and populations: allozyme variation within and among Atlantic salmon from downeast rivers of Maine. J. Fish Biology. 67:104-117. (pdf)\r\n\r\nFloyd, C.H., D. Van Vuren, and B. May. Marmots on Great Basin mountaintops: using genetics to test an ecological paradigm. Ecology. 86: 2145-2153. (pdf)\r\n\r\nFlores Martnez, J.J., C. H. Floyd, L.G. Herrera, and B. May. Genetic variation and population size of the endangered fishing bat, Myotis vivesi, in Isla Partida. Pp. 187-192 in Contribuciones Mastozoolgicas en Homenaje a Bernardo Villa (V. Snchez-Cordero y R. A. Medelln, eds.). Instituto de Biologa, UNAM; Instituto de Ecologa, UNAM; CONABIO. Mxico. 706 pp.\r\n\r\nLeunda, P.M., R. Miranda, J. Madoz, S. Parmenter, Y. Chen, and B. May. Threatened fishes of the world: Siphateles bicolor snyderi (Miller, 1973) (Cyprinidae). Env. Biol. Fish. 73:109-110 (pdf)\r\n\r\nRodrigue, K.L., B.P. May, T.R. Famula, and M.E. Delany. Meiotic instability of chicken ultra-long telomeres and mapping of a 2.8 megabase array to the W-sex chromosome. Chrom. Res.13:581-591. (pdf)\r\n\r\nTeglas, M.B., B. May, P.R. Crosbie, M.R. Stephens, and W.M. Boyce. Genetic structure of the tick Ornithodoros coriaceus (Acarina: Argasidae) in California, Nevada and Oregon. J. Med. Entomol. 42:247-253.\r\n\r\nWilliamson, K. and B. May. Homogenization of fall-run chinook salmon gene pools in the Central Valley of California, USA. N. Am. J. Fish. Man. 25:9931009. (pdf)\r\n\r\nWilliamson, K. and B. May. Inheritance studies implicate a genetic mechanism for apparent sex-reversal in Chinook salmon. Trans. Am. Fish. Soc. 134:12531261. (pdf)','Publications 2005-2008','','inherit','closed','open','','129-revision-v1','','','2014-02-21 21:07:18','2014-02-21 21:07:18','',129,'http://localhost/~psalessi/Genetics_Lab/129-revision-v1/',0,'revision','',0),(140,1,'2014-02-21 21:09:51','2014-02-21 21:09:51','<strong>1999</strong>\r\n\r\nGavin, T.A., P.W. Sherman, E. Yensen, and B. May. Taxonomic status of Spermophilus brunneus and S. b. endemicus with comparison to other western spermophilus. J. Mam. 80:156-168.\r\n\r\nSeki, S, J.J. Agresti, , G.A.E. Gall, N. Taniguchi, and B. May. AFLP analysis of genetic diversity in three populations of ayu (Plecoglossus altivelis). Fish. Sci. 65:888-899.\r\n\r\n<strong>1998</strong>\r\n\r\nGavin, T.A., R.T. Reynolds, S.M. Joy, and B. May. Genetic evidence for low frequency of extra-pair fertilizations in Northern goshawks. Condor. 100:556-560 (<a title=\"Publications_4_files/1998 - Goshawks extra-pair.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1998%20-%20Goshawks%20extra-pair.pdf\">pdf</a>).\r\n\r\nMay, B. Starch gel electrophoresis of allozymes. In: Molecular Genetic Analysis of Populations: A Practical Approach. 2nd Ed. A.R. Hoelzel, ed. Oxford Univ. Press. Pp. 1-28 and 371-378.\r\n\r\nMay, B. Book Review. Molecular Systematics of Fishes, T.D. Kocher and C.A. Stepiens, eds. Trans. Am. Fish. Soc. 127:857-858.\r\n\r\nRodzen, J., J.J. Agresti, G. Tranah, and B. May. Agarose overlays allow simplified staining of polyacrylamide gels. Biotechniques. 25:584.\r\n\r\n<strong>1997</strong>\r\n\r\nMay, B., T.A. Gavin, P.W. Sherman, and T.M. Korves. Characterization of microsatellite loci in the Northern Idaho ground squirrel, Spermophilus brunneus Mol. Ecol. 6:399-400 (<a title=\"Publications_4_files/1997 - Spermophilus brunneus.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1997%20-%20Spermophilus%20brunneus.pdf\">pdf</a>).\r\n\r\nMay, B., C.C. Krueger, and H.L. Kincaid. Genetic variation at microsatellite loci in sturgeon: primer sequence homology in Acipenser and Scaphirhynchus. Can. J. Fish. &amp; Aquat. Sci. 54: 1542-1547 (<a title=\"Publications_4_files/1997 - May sturgeon 1997.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1997%20-%20May%20sturgeon%201997.pdf\">pdf</a>).\r\n\r\n<strong>1996</strong>\r\n\r\nLegge, J.T., R. Roush, B. May, R. DeSalle, and A. Vogler. Genetic criteria for establishing evolutionarily significant units in Cryan\'s buckmoth. Cons. Biol. 10(1):85-98 (<a title=\"Publications_4_files/1996 - Cryan\'s buckmoth.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1996%20-%20Cryan%27s%20buckmoth.pdf\">pdf</a>).\r\n\r\nMarsden, J.E., A. Spidle, and B. May. Review of genetic studies of Dreissena spp. Amer. Zool. 36:259-270 (<a title=\"Publications_4_files/1996 - marsden et al am zoo.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1996%20-%20marsden%20et%20al%20am%20zoo.pdf\">pdf</a>).\r\n\r\nMills, E.L., G. Rosenberg, A.P. Spidle, M. Ludyanskiy, Y. Pligin, and B. May. A review of the biology and ecology of the quagga mussel (Dreissena bugensis): a second species of freshwater dreissenid introduced to North America. Submitted to Amer. Zool. 36: 271-286.\r\n\r\n<strong>1995</strong>\r\n\r\nMarsden, J. E., A. Spidle, and B. May. Genetic similarity among zebra mussel poulations within North America and within Europe. Can. J. Fish. Aquat. Sci. 52:836-847 (<a title=\"Publications_4_files/1995 - marsden_et_al_zebra_mussel_structure[1].pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1995%20-%20marsden_et_al_zebra_mussel_structure%5B1%5D.pdf\">pdf</a>).\r\n\r\nPerkins, D.L., J. Fitzsimons, J.E. Marsden, C.C. Krueger, and B. May. Differences in reproduction among hatchery strains of lake trout at eight spawning areas in Lake Ontario: genetic evidence from mixed-stock analysis. J. Great Lakes Res. 21:364-374.\r\n\r\nSpidle, A.P., E.L. Mills, and B. May. Absence of naturally occurring hybridization between the quagga mussel (Dreissena bugensis) and the zebra mussel (D. polymorpha). Can. J. Zool. . 73:400-403 (<a title=\"Publications_4_files/1995 - spidle et al quagga zebe hyb.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1995%20-%20spidle%20et%20al%20quagga%20zebe%20hyb.pdf\">pdf</a>).\r\n\r\nSpidle, A.P., E.L. Mills, and B. May. Differential response to changes in salinity and temperature in the quagga and zebra mussels (Dreissena bugensis Andrusov and D. polymorpha Pallas). Can. J. Fish. Aquat. Sci. 52:2108-2119 (<a title=\"Publications_4_files/1995 - spidle et al dreissena physiology ocr.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1995%20-%20spidle%20et%20al%20dreissena%20physiology%20ocr.pdf\">pdf</a>).\r\n\r\n<strong>1994</strong>\r\n\r\nAiken, S.G., L.L. Consaul, A. Spidle, and B. May. Allozyme and morphological observations on Festuca hyperborea, compared with F. baffinensis and F. brachyphylla (Poaceae) from the Canadian Arctic. Nord. J. Bot. 14(2): 137-143.\r\n\r\nGrewe, P.M., C.C. Krueger, C.F. Aquadro, and B. May. Stability of allozyme and mitochondrial DNA markers among three year classes of lake trout propagated from Seneca Lake, New York. N. Amer. J. Fish. Man. 14:467-474 (<a title=\"Publications_4_files/1994 - GreweEtAl.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1994%20-%20GreweEtAl.pdf\">pdf</a>).\r\n\r\nGrewe, P.M., C.C. Krueger, J.E. Marsden, C.F. Aquadro, and B. May. Hatchery origins of naturally produced lake trout fry captured in Lake Ontario: temporal and spatial variability based on allozyme and mitochondrial DNA data. Trans. Am. Fish. Soc. 123:309-320.\r\n\r\nKrueger, C.C., D.L. Perkins, R.J. Everett, D.R. Schreiner, and B. May. Genetic variation in naturalized rainbow trout (steelhead) from Minnesota tributaries to Lake Superior. J. Great Lakes Res. 20:299-316.\r\n\r\nSpidle, A.P., J.E. Marsden, and B. May. Identification of the Great Lakes quagga mussel as Dreissena bugensis from the Dnieper River, Ukraine, on the basis of allozyme variation. Can. J. Fish. Aquat. Sci. 51:1485-1489 (<a title=\"Publications_4_files/1994 - spidle et al quagga id.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1994%20-%20spidle%20et%20al%20quagga%20id.pdf\">pdf</a>).\r\n\r\n<strong>1993</strong>\r\n\r\nGrewe, P.M., C.C. Krueger, C.F. Aquadro, E. Bermingham, H. L. Kincaid, and B. May. Mitochodrial DNA variation among lake trout (Salvelinus namaycush) strains stocked into Lake Ontario. Can J. Fish. Aquat. Sci. 50:2397-2403.\r\n\r\nKincaid, H. L. , C.C. Krueger, and B. May. Genetic variation among remnant groups of the Green Lake strain of lake trout: preservation of the remnant Green Lake gene pool. N. Am. J. Fish. Man. 13:318-325 (<a title=\"Publications_4_files/1993 - KincaidMay.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1993%20-%20KincaidMay.pdf\">pdf</a>).\r\n\r\nMarsden, J.E., C.C. Krueger, P.M. Grewe, H.L. Kincaid, and B. May. Genetic comparison of naturally-spawned and artificially-propagated Lake Ontario lake trout fry: evaluation of a stocking strategy for species rehabilitation. N. Am. J. Fish. Man. 13:304-317 (<a title=\"Publications_4_files/1993 - MarsdenEtAl.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1993%20-%20MarsdenEtAl.pdf\">pdf</a>).\r\n\r\nMay, B. and P. M. Grewe. Fate of maternal mtDNA following Cobalt60 inactivation of maternal nuclear DNA in unfertilized salmonid eggs. Genome 36:725-730.\r\n\r\nMay, B. and K. R. Johnson. Composite linkage map of salmonid fishes (Salvelinus, Salmo, and Oncorhynchus). In: Genetic Maps : Locus Maps of Complex Genomes. O\'Brien, S.J. (ed.) Cold Spring Harbor. 4:309-317.\r\n\r\nPerkins, D.L., C.C. Krueger, and B. May. Heritage brook trout in Northeastern USA: genetic variability within and among populations. Trans. Am Fish. Soc.122:515-532.\r\n\r\nRoyse, D.J. and B. May. Multilocus enzyme electrophoresis for the genetic analysis of edible mushrooms. In: Genetics and Breeding of Edible Mushrooms. Chang, S.T., J.A. Buswell, and P.G. Miles (ed.). pp. 225-247. Gordon and Breach Sci. Pub. Phil.\r\n\r\n<strong>Later Publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/\">2009-2013</a></li>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n</ul>\r\n<strong>Earlier Publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications 1993-1999','','publish','closed','open','','publications-1993-1999','','','2014-03-07 20:18:44','2014-03-07 20:18:44','',12,'http://localhost/~psalessi/Genetics_Lab/?page_id=140',0,'page','',0),(141,1,'2014-02-21 21:09:51','2014-02-21 21:09:51','<strong>1999</strong>\r\n\r\nGavin, T.A., P.W. Sherman, E. Yensen, and B. May. Taxonomic status of Spermophilus brunneus and S. b. endemicus with comparison to other western spermophilus. J. Mam. 80:156-168.\r\n\r\nSeki, S, J.J. Agresti, , G.A.E. Gall, N. Taniguchi, and B. May. AFLP analysis of genetic diversity in three populations of ayu (Plecoglossus altivelis). Fish. Sci. 65:888-899.\r\n\r\n<strong>1998</strong>\r\n\r\nGavin, T.A., R.T. Reynolds, S.M. Joy, and B. May. Genetic evidence for low frequency of extra-pair fertilizations in Northern goshawks. Condor. 100:556-560 (pdf).\r\n\r\nMay, B. Starch gel electrophoresis of allozymes. In: Molecular Genetic Analysis of Populations: A Practical Approach. 2nd Ed. A.R. Hoelzel, ed. Oxford Univ. Press. Pp. 1-28 and 371-378.\r\n\r\nMay, B. Book Review. Molecular Systematics of Fishes, T.D. Kocher and C.A. Stepiens, eds. Trans. Am. Fish. Soc. 127:857-858.\r\n\r\nRodzen, J., J.J. Agresti, G. Tranah, and B. May. Agarose overlays allow simplified staining of polyacrylamide gels. Biotechniques. 25:584.\r\n\r\n<strong>1997</strong>\r\n\r\nMay, B., T.A. Gavin, P.W. Sherman, and T.M. Korves. Characterization of microsatellite loci in the Northern Idaho ground squirrel, Spermophilus brunneus Mol. Ecol. 6:399-400 (pdf).\r\n\r\nMay, B., C.C. Krueger, and H.L. Kincaid. Genetic variation at microsatellite loci in sturgeon: primer sequence homology in Acipenser and Scaphirhynchus. Can. J. Fish. &amp; Aquat. Sci. 54: 1542-1547 (pdf).\r\n\r\n<strong>1996</strong>\r\n\r\nLegge, J.T., R. Roush, B. May, R. DeSalle, and A. Vogler. Genetic criteria for establishing evolutionarily significant units in Cryan\'s buckmoth. Cons. Biol. 10(1):85-98 (pdf).\r\n\r\nMarsden, J.E., A. Spidle, and B. May. Review of genetic studies of Dreissena spp. Amer. Zool. 36:259-270 (pdf).\r\n\r\nMills, E.L., G. Rosenberg, A.P. Spidle, M. Ludyanskiy, Y. Pligin, and B. May. A review of the biology and ecology of the quagga mussel (Dreissena bugensis): a second species of freshwater dreissenid introduced to North America. Submitted to Amer. Zool. 36: 271-286.\r\n\r\n<strong>1995</strong>\r\n\r\nMarsden, J. E., A. Spidle, and B. May. Genetic similarity among zebra mussel poulations within North America and within Europe. Can. J. Fish. Aquat. Sci. 52:836-847 (pdf).\r\n\r\nPerkins, D.L., J. Fitzsimons, J.E. Marsden, C.C. Krueger, and B. May. Differences in reproduction among hatchery strains of lake trout at eight spawning areas in Lake Ontario: genetic evidence from mixed-stock analysis. J. Great Lakes Res. 21:364-374.\r\n\r\nSpidle, A.P., E.L. Mills, and B. May. Absence of naturally occurring hybridization between the quagga mussel (Dreissena bugensis) and the zebra mussel (D. polymorpha). Can. J. Zool. . 73:400-403 (pdf).\r\n\r\nSpidle, A.P., E.L. Mills, and B. May. Differential response to changes in salinity and temperature in the quagga and zebra mussels (Dreissena bugensis Andrusov and D. polymorpha Pallas). Can. J. Fish. Aquat. Sci. 52:2108-2119 (pdf).\r\n\r\n<strong>1994</strong>\r\n\r\nAiken, S.G., L.L. Consaul, A. Spidle, and B. May. Allozyme and morphological observations on Festuca hyperborea, compared with F. baffinensis and F. brachyphylla (Poaceae) from the Canadian Arctic. Nord. J. Bot. 14(2): 137-143.\r\n\r\nGrewe, P.M., C.C. Krueger, C.F. Aquadro, and B. May. Stability of allozyme and mitochondrial DNA markers among three year classes of lake trout propagated from Seneca Lake, New York. N. Amer. J. Fish. Man. 14:467-474 (pdf).\r\n\r\nGrewe, P.M., C.C. Krueger, J.E. Marsden, C.F. Aquadro, and B. May. Hatchery origins of naturally produced lake trout fry captured in Lake Ontario: temporal and spatial variability based on allozyme and mitochondrial DNA data. Trans. Am. Fish. Soc. 123:309-320.\r\n\r\nKrueger, C.C., D.L. Perkins, R.J. Everett, D.R. Schreiner, and B. May. Genetic variation in naturalized rainbow trout (steelhead) from Minnesota tributaries to Lake Superior. J. Great Lakes Res. 20:299-316.\r\n\r\nSpidle, A.P., J.E. Marsden, and B. May. Identification of the Great Lakes quagga mussel as Dreissena bugensis from the Dnieper River, Ukraine, on the basis of allozyme variation. Can. J. Fish. Aquat. Sci. 51:1485-1489 (pdf).\r\n\r\n<strong>1993</strong>\r\n\r\nGrewe, P.M., C.C. Krueger, C.F. Aquadro, E. Bermingham, H. L. Kincaid, and B. May. Mitochodrial DNA variation among lake trout (Salvelinus namaycush) strains stocked into Lake Ontario. Can J. Fish. Aquat. Sci. 50:2397-2403.\r\n\r\nKincaid, H. L. , C.C. Krueger, and B. May. Genetic variation among remnant groups of the Green Lake strain of lake trout: preservation of the remnant Green Lake gene pool. N. Am. J. Fish. Man. 13:318-325 (pdf).\r\n\r\nMarsden, J.E., C.C. Krueger, P.M. Grewe, H.L. Kincaid, and B. May. Genetic comparison of naturally-spawned and artificially-propagated Lake Ontario lake trout fry: evaluation of a stocking strategy for species rehabilitation. N. Am. J. Fish. Man. 13:304-317 (pdf).\r\n\r\nMay, B. and P. M. Grewe. Fate of maternal mtDNA following Cobalt60 inactivation of maternal nuclear DNA in unfertilized salmonid eggs. Genome 36:725-730.\r\n\r\nMay, B. and K. R. Johnson. Composite linkage map of salmonid fishes (Salvelinus, Salmo, and Oncorhynchus). In: Genetic Maps : Locus Maps of Complex Genomes. O\'Brien, S.J. (ed.) Cold Spring Harbor. 4:309-317.\r\n\r\nPerkins, D.L., C.C. Krueger, and B. May. Heritage brook trout in Northeastern USA: genetic variability within and among populations. Trans. Am Fish. Soc.122:515-532.\r\n\r\nRoyse, D.J. and B. May. Multilocus enzyme electrophoresis for the genetic analysis of edible mushrooms. In: Genetics and Breeding of Edible Mushrooms. Chang, S.T., J.A. Buswell, and P.G. Miles (ed.). pp. 225-247. Gordon and Breach Sci. Pub. Phil.','Publications 1993-1999','','inherit','closed','open','','140-revision-v1','','','2014-02-21 21:09:51','2014-02-21 21:09:51','',140,'http://localhost/~psalessi/Genetics_Lab/140-revision-v1/',0,'revision','',0),(142,1,'2014-02-21 21:10:40','2014-02-21 21:10:40','<strong>1992</strong>\r\n\r\nMay, B. Starch gel electrophoresis of allozymes. In: Molecular Genetic Analysis of Populations: A Practical Approach. A.R. Hoelzel, ed. Oxford Univ. Press. pp. 1-27 and 271-280.\r\n\r\nMay, B. and J.E. Marsden. Genetic identification and implications of another invasive species of Dreissenid mussel in the Great Lakes. Can. J. Fish. Aqua. Sci. 49:1501-1506.\r\n\r\nStaub, J.E., L.D. Knerr, D.J.Holder, and B. May. Phylogenetic relationships among several African Cucumis species. Can. J. Botany. 70:509-517.\r\n\r\nSt. Leger, R.J. Allee, L.L, May. B, Staples, R.C and Roberts, D.W. World-wide distribution of genetic variation among isolates of Beauveria spp. Mycol. Res. 96:1007-1015.\r\n\r\nSt. Leger, R. J., B. May, L. Allee, D. C. Frank, R. C. Staples, and D. W. Roberts. Genetic differences in allozymes and in formation of infection structures among isolates of the Entomopathogenic fungus Metarhizium anisopliae. J. Inv. Path. 60:89-101.\r\n\r\n<strong>1991</strong>\r\n\r\n<strong></strong>Bowden, C.G., D.J. Royse, and B. May. Linkage relationships of allozyme encoding loci in shitake, Lentinula edodes. Genome 34:652-657.\r\n\r\nGavin, T.A., R.A. Howard, and B. May. Geographic variation in allozyme frequencies in red-winged blackbirds: the California conundrum. Auk 108:602-611 (<a title=\"Publications_5_files/1991 - Red-winged blackbirds.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_5_files/1991%20-%20Red-winged%20blackbirds.pdf\">pdf</a>).\r\n\r\nKrueger, C.C., and B. May. Ecological and Genetic Effects of Salmonid Introductions in North America. Can. J. Fish. &amp; Aq. Sci. 48:1-12.\r\n\r\nKukuk, P.F. and B. May. Colony dynamics in a primitively eusocial halictine bee, Lasioglossum (Dialictus) zephyrus, (Hymenoptera: halictidae). Insectes Sociaux 38:171-189.\r\n\r\n<strong>1990</strong>\r\n\r\nDixon, P. M., and B. May. Genetic diversity and population structure of a rare plant, Northern Monkshood (Aconitum noveboracense). In:Ecosystem Management: rare species and significant habitats. Proc. 15th Natural Areas Meeting. NY state museum bulletin No. 471. pp. 167-175.\r\n\r\nHajek, A.E., R.A. Humber, J.S. Elkinton, B. May, S.R. Walsh, and J.C. Silver. Allozyme and RFLP analyses confirm Entomophaga maimaiga responsible for 1989 epizootics in North American gypsy moth populations. PNAS. 87:6979-6982 (<a title=\"Publications_5_files/1990 - Entomophaga maimaiga.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_5_files/1990%20-%20Entomophaga%20maimaiga.pdf\">pdf</a>).\r\n\r\nKukuk, P.F. and B. May. Diploid males in a primitively eusocial bee, Lasioglossum (Dialictus) zephyrum (Hymenoptera: Halictidae). Evolution 44:1522-1528 (<a title=\"Publications_5_files/1990 - Lasioglossum (Dialictus) zephyrum.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_5_files/1990%20-%20Lasioglossum%20%28Dialictus%29%20zephyrum.pdf\">pdf</a>).\r\n\r\nMay, B. and C.C. Krueger. Use of allozyme data for population analysis. In: Electrophoretic and Isoelectric Focussing Techniques in Fisheries Management. D.H. Whitmore, (ed.). pp. 157-172. CRC Press.\r\n\r\nRoyse, D.J., and B. May. Interspecific allozyme variation among Morchella spp. and its inferences for systematics within the genus. Biochem. Syst. &amp; Ecol. 18:475-479 (<a title=\"Publications_5_files/1990 - Morchella spp.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_5_files/1990%20-%20Morchella%20spp.pdf\">pdf</a>).\r\n\r\n<strong>Later publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/\">2009-2013</a></li>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n</ul>','Publications 1990-1992','','publish','closed','open','','142-2','','','2014-03-07 20:20:59','2014-03-07 20:20:59','',12,'http://localhost/~psalessi/Genetics_Lab/?page_id=142',0,'page','',0),(143,1,'2014-02-21 21:10:40','2014-02-21 21:10:40','<strong>1992</strong>\r\n\r\nMay, B. Starch gel electrophoresis of allozymes. In: Molecular Genetic Analysis of Populations: A Practical Approach. A.R. Hoelzel, ed. Oxford Univ. Press. pp. 1-27 and 271-280.\r\n\r\nMay, B. and J.E. Marsden. Genetic identification and implications of another invasive species of Dreissenid mussel in the Great Lakes. Can. J. Fish. Aqua. Sci. 49:1501-1506.\r\n\r\nStaub, J.E., L.D. Knerr, D.J.Holder, and B. May. Phylogenetic relationships among several African Cucumis species. Can. J. Botany. 70:509-517.\r\n\r\nSt. Leger, R.J. Allee, L.L, May. B, Staples, R.C and Roberts, D.W. World-wide distribution of genetic variation among isolates of Beauveria spp. Mycol. Res. 96:1007-1015.\r\n\r\nSt. Leger, R. J., B. May, L. Allee, D. C. Frank, R. C. Staples, and D. W. Roberts. Genetic differences in allozymes and in formation of infection structures among isolates of the Entomopathogenic fungus Metarhizium anisopliae. J. Inv. Path. 60:89-101.\r\n\r\n<strong>1991</strong>\r\n\r\n<strong></strong>Bowden, C.G., D.J. Royse, and B. May. Linkage relationships of allozyme encoding loci in shitake, Lentinula edodes. Genome 34:652-657.\r\n\r\nGavin, T.A., R.A. Howard, and B. May. Geographic variation in allozyme frequencies in red-winged blackbirds: the California conundrum. Auk 108:602-611 (pdf).\r\n\r\nKrueger, C.C., and B. May. Ecological and Genetic Effects of Salmonid Introductions in North America. Can. J. Fish. &amp; Aq. Sci. 48:1-12.\r\n\r\nKukuk, P.F. and B. May. Colony dynamics in a primitively eusocial halictine bee, Lasioglossum (Dialictus) zephyrus, (Hymenoptera: halictidae). Insectes Sociaux 38:171-189.\r\n\r\n<strong>1990</strong>\r\n\r\nDixon, P. M., and B. May. Genetic diversity and population structure of a rare plant, Northern Monkshood (Aconitum noveboracense). In:Ecosystem Management: rare species and significant habitats. Proc. 15th Natural Areas Meeting. NY state museum bulletin No. 471. pp. 167-175.\r\n\r\nHajek, A.E., R.A. Humber, J.S. Elkinton, B. May, S.R. Walsh, and J.C. Silver. Allozyme and RFLP analyses confirm Entomophaga maimaiga responsible for 1989 epizootics in North American gypsy moth populations. PNAS. 87:6979-6982 (pdf).\r\n\r\nKukuk, P.F. and B. May. Diploid males in a primitively eusocial bee, Lasioglossum (Dialictus) zephyrum (Hymenoptera: Halictidae). Evolution 44:1522-1528 (pdf).\r\n\r\nMay, B. and C.C. Krueger. Use of allozyme data for population analysis. In: Electrophoretic and Isoelectric Focussing Techniques in Fisheries Management. D.H. Whitmore, (ed.). pp. 157-172. CRC Press.\r\n\r\nRoyse, D.J., and B. May. Interspecific allozyme variation among Morchella spp. and its inferences for systematics within the genus. Biochem. Syst. &amp; Ecol. 18:475-479 (pdf).','','','inherit','closed','open','','142-revision-v1','','','2014-02-21 21:10:40','2014-02-21 21:10:40','',142,'http://localhost/~psalessi/Genetics_Lab/142-revision-v1/',0,'revision','',0),(144,1,'2014-02-21 21:10:51','2014-02-21 21:10:51','<strong>1992</strong>\r\n\r\nMay, B. Starch gel electrophoresis of allozymes. In: Molecular Genetic Analysis of Populations: A Practical Approach. A.R. Hoelzel, ed. Oxford Univ. Press. pp. 1-27 and 271-280.\r\n\r\nMay, B. and J.E. Marsden. Genetic identification and implications of another invasive species of Dreissenid mussel in the Great Lakes. Can. J. Fish. Aqua. Sci. 49:1501-1506.\r\n\r\nStaub, J.E., L.D. Knerr, D.J.Holder, and B. May. Phylogenetic relationships among several African Cucumis species. Can. J. Botany. 70:509-517.\r\n\r\nSt. Leger, R.J. Allee, L.L, May. B, Staples, R.C and Roberts, D.W. World-wide distribution of genetic variation among isolates of Beauveria spp. Mycol. Res. 96:1007-1015.\r\n\r\nSt. Leger, R. J., B. May, L. Allee, D. C. Frank, R. C. Staples, and D. W. Roberts. Genetic differences in allozymes and in formation of infection structures among isolates of the Entomopathogenic fungus Metarhizium anisopliae. J. Inv. Path. 60:89-101.\r\n\r\n<strong>1991</strong>\r\n\r\n<strong></strong>Bowden, C.G., D.J. Royse, and B. May. Linkage relationships of allozyme encoding loci in shitake, Lentinula edodes. Genome 34:652-657.\r\n\r\nGavin, T.A., R.A. Howard, and B. May. Geographic variation in allozyme frequencies in red-winged blackbirds: the California conundrum. Auk 108:602-611 (pdf).\r\n\r\nKrueger, C.C., and B. May. Ecological and Genetic Effects of Salmonid Introductions in North America. Can. J. Fish. &amp; Aq. Sci. 48:1-12.\r\n\r\nKukuk, P.F. and B. May. Colony dynamics in a primitively eusocial halictine bee, Lasioglossum (Dialictus) zephyrus, (Hymenoptera: halictidae). Insectes Sociaux 38:171-189.\r\n\r\n<strong>1990</strong>\r\n\r\nDixon, P. M., and B. May. Genetic diversity and population structure of a rare plant, Northern Monkshood (Aconitum noveboracense). In:Ecosystem Management: rare species and significant habitats. Proc. 15th Natural Areas Meeting. NY state museum bulletin No. 471. pp. 167-175.\r\n\r\nHajek, A.E., R.A. Humber, J.S. Elkinton, B. May, S.R. Walsh, and J.C. Silver. Allozyme and RFLP analyses confirm Entomophaga maimaiga responsible for 1989 epizootics in North American gypsy moth populations. PNAS. 87:6979-6982 (pdf).\r\n\r\nKukuk, P.F. and B. May. Diploid males in a primitively eusocial bee, Lasioglossum (Dialictus) zephyrum (Hymenoptera: Halictidae). Evolution 44:1522-1528 (pdf).\r\n\r\nMay, B. and C.C. Krueger. Use of allozyme data for population analysis. In: Electrophoretic and Isoelectric Focussing Techniques in Fisheries Management. D.H. Whitmore, (ed.). pp. 157-172. CRC Press.\r\n\r\nRoyse, D.J., and B. May. Interspecific allozyme variation among Morchella spp. and its inferences for systematics within the genus. Biochem. Syst. &amp; Ecol. 18:475-479 (pdf).','Publications 1990-1992','','inherit','closed','open','','142-revision-v1','','','2014-02-21 21:10:51','2014-02-21 21:10:51','',142,'http://localhost/~psalessi/Genetics_Lab/142-revision-v1/',0,'revision','',0),(147,1,'2014-02-21 21:12:18','2014-02-21 21:12:18','<strong>2013</strong>\r\n\r\nChen, Y., S. Parmenter, and B. May. Genetic characterization of variation and management implications for the endangeredMohave tui chub. Cons. Gen. 14:11-20. (pdf)\r\n\r\nMeek, M. H., A. P. Wintzer, N. Shepherd, and B. May. Genetic diversity and reproductive mode in two non-native hydromedusae, Maeotias marginata and Moerisia sp., in the upper San Francisco Estuary, California. Biol. Invasions. 15(1): 199-212. (pdf)\r\n\r\nSchreier, A.D., B. Mahardja, and B. May. Hierarchical patterns of population structure in the endangered Fraser River white sturgeon, Acipenser transmontanus, and implications for conservation. Can. J. Fish. &amp; Aquat. Sci. 69:1968-1980. (pdf)\r\n\r\n<strong>2012</strong>\r\n\r\nBaerwald, M.R., B.M. Schreier, G. Schumer, and B. May. Detection of Threatened Delta Smelt in the Gut Contents of the Invasive Mississippi Silverside in the San Francisco Estuary Using TaqMan Assays. Trans. Am. Fish. Soc. 141:1600-1607. (pdf)\r\n\r\nMahardja, B., B. May, and M. Baerwald. Characterization of 36 additional microsatellite loci in splittail (Pogonichthys macrolepidotus) and cross-amplification in five other native Californian cyprinid species. Cons. Gen. Res. 4(4):917-921. (pdf)\r\n\r\nMeek, M., A. Wintzer, W. Wetzel, and B. May. Climate change likely to facilitate the invasion of the non-native hydroid, Cordylophora caspia, in the San Francisco Estuary, CA. PLos ONE. 7(10): e46373. doi:10.1371/journal.pone.0046373 (pdf)\r\n\r\nPetersen, J.L., M.R. Baerwald, J.M. Ibarra, and B. May. A first-generation linkage map of the Pacific lion-paw scallop: initial evidence of QTL for size traits and markers linked to orange shell color. Aquaculture. 350-353:200-209. (pdf)\r\n\r\nSchreier, A. D., J. Rodzen, S. Ireland, and B. May. Genetic techniques inform conservation aquaculture of the endangered Kootenai River white sturgeon, Acipenser transmontanus. Endang. Species Res. 16:65-75. (pdf)\r\n\r\nSukmanomon, S., W. Kamonrat, S. Poompuang, T. Nguyen, D. M. Bartley, B. May, U. Na-Nakorn. Genetic changes, intra- and interspecific introgression in farmed Nile tilapia in Thailand. Aquaculture 324-325:44-54. (pdf)\r\n\r\nVan Dam, A.R. and B. May. A new species of Dactylopius Costa (Dactylopius gracilipilus sp. nov.) (Hemiptera: Coccoidea: Dactylopiidae) from the Chihuahuan Desert, Texas, U.S.A. . Zootaxa 3573:33-39. (pdf)\r\n\r\n<strong>2011</strong>\r\n\r\nBaerwald, M., J. Petersen, R. Hedrick, G. Schisler, and B. May. A major effect quantitative trait locus for whirling disease resistance identified in rainbow trout (Oncorhynchus mykiss). Heredity. 106:920-926. (pdf)\r\n\r\nBaerwald, M., G. Schumer, B. Schreier, and B. May. TaqMan assays for the genetic identification of delta smelt (Hypomesus transpacificus) and wakasagi smelt (Hypomesus nipponensis). Mol. Ecol. Res. 11:784-785. (pdf)\r\n\r\nBaerwald, M.R., M. R. Stephens, K. S. Bork, M. H. Meek, K. Tomalty, and B. May. 2011. Spring-run Chinook salmon genetic management plan  San Joaquin River Restoration Program. 118 pgs. Scientific genetic management plan for the CA Dept. of Fish and Game. (pdf)\r\n\r\nFinger, A. J., E. C. Anderson, M. R. Stephens, and B. P. May. Application of a method for estimating effective population size and admixture using diagnostic SNPs: Implications for conservation of threatened Paiute cutthroat trout (Oncorhynchus clarkii seleniris) in Silver King Creek, California. Can. J. Fish. Aquat. Sci. 68:1369-1386 (pdf).\r\n\r\nFisch, K.M., J.M. Henderson, R.S. Burton, and B. May. Population genetics and conservation implications for the endangered delta smelt in the San Francisco Bay-Delta. Conservation Genetics. 12:1421-1434 (pdf).\r\n\r\nFloyd, C. H., D.H. Van Vuren, K.R. Crooks, K.L. Jones, D.K. Garcelon, N.M. Belfiore, J.W. Dragoo, and B. May. Genetic differentiation of island spotted skunks, Spilogale gracilis amphiala. J. Mam. 92(1):148-158 (pdf).\r\n\r\nMcEachern, M.B., D.H. Van Vuren, C.H. Floyd, B. May, and J.M. Eadie.Bottlenecks and rescue effects in a fluctuating population of golden-mantled ground squirrels (Spermophilus lateralis). Con. Gen. 12:285-296 (pdf).\r\n\r\nMejia , O., L. G. Herrera, B. May, R. A. Medellin,and J. J. Flores-Martinez. Effective population size dynamics of Myotis vivesi during the Pleistocene and Holocene climatic changes. Acta Chiropterologica. 13:33-40 (pdf).\r\n\r\nSchreier, A.D., D. Gille, B. Mahardja, and B. May. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon, Acipenser transmontanus. J. Appl. Ichthiol. 27:24-33 (pdf).\r\n\r\nWintzer, A.P., M.H. Meek, P.B. Moyle, and B. May. Ecological insights into the polyp stage of non-native hydrozoans in the San Francisco Estuary. Aquat. Ecol. 45:151161 (pdf).\r\n\r\n<strong>2010</strong>\r\n\r\nFloyd, C. H., J.J. Flores Martnez, , L.G. Herrera, O. Mejia, and B. May. Conserving the endangered Mexican fishing bat (Myotis vivesi): genetic variation indicates extensive gene flow among islands in the Gulf of California. Cons. Gen. 11:813-82 (pdf).\r\n\r\nIsrael, J.A. and B. May. Indirect estimates of breeding population size in the polyploid green sturgeon (Acipenser medirostris). Mol. Ecol. 19:1058-107 (pdf).\r\n\r\nIsrael, J.A. and B. May. Characterization and evaluation of polymorphic microsatellite markers in the anadromous fish Spirinchus thaleichthys. Cons. Gen. Res. online (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Nuclear and mtDNA lineage diversity in wild and cultured Pacific lion-paw scallop (Baja California Peninsula, Mexico). Mar. Biol. 157:27512767 (pdf).\r\n\r\n<strong>2009</strong>\r\n\r\nChen, Y., S. Reid, and B. May. Genetic relationships among tui chub populations in the Cow Head and surrounding basins of the Northwestern Great Basin. Cons. Gen. 10:101-114 (pdf).\r\n\r\nFinger, A.J., M.R. Stephens, N.W. Clipperton and B. May. Six diagnostic SNP markers for detecting introgression between cutthroat and rainbow trout. Mol. Ecol. Res. 9:759-763 (pdf).\r\n\r\nFisch, K., J.L. Petersen, M.R. Baerwald, and B.P. May. Characterization of 24 microsatellite loci in delta smelt, Hypomesus transpacificus, and their cross-species amplification in two other smelt species of the Osmeridae family. Mol. Ecol. Res. 9:405-408 (pdf).\r\n\r\nIsrael, J.A., K.J. Bando, E.C. Anderson, and B. May. Polyploid microsatellite data reveal stock complexity among estuarine North American green sturgeon (Acipenser medirostris). Can. J. Fish. Aqu. Sci. 66:1491-1504 (pdf).\r\n\r\nMeek, M., M. Baerwald, A. Wintzer, and B. May. Isolation and characterization of microsatellite loci in two non-native hydromedusae in the San Francisco Estuary: Maeotias marginata and Moerisia sp. Cons. Gen. Res. 1:205-208 (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Thirty-seven additional microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and cross-amplification in other Pectinids. Cons. Gen. Res. 1:101-105 (pdf).\r\n\r\nSimmons, R. E., P. Lavretsky, and B. May. Introgressive Hybridization of Redband Trout in the Upper McCloud River Watershed. Trans. Am. Fish. Soc. 139:201-213 (pdf).\r\n\r\nStephens, M.R., N.W. Clipperton, and B.P. May. Subspecies-informative TaqMan probes for evaluating introgression between native golden trout and introduced rainbow trout. Mol. Ecol. Res. 9:339-343 (pdf).\r\n\r\n<strong>Earlier publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications','','inherit','closed','open','','12-revision-v1','','','2014-02-21 21:12:18','2014-02-21 21:12:18','',12,'http://localhost/~psalessi/Genetics_Lab/12-revision-v1/',0,'revision','',0),(148,1,'2014-02-21 21:13:21','2014-02-21 21:13:21','<strong>2004</strong>\r\n\r\nBaerwald, M.R. and B. May. Characterization of microsatellite loci for five members of the minnow family Cyprinidae found in the Sacramento - San Joaquin Delta. Mol. Ecol. Notes. 4:385-390 (pdf).\r\n\r\nCordes, J.F., J.A. Israel and B. May. Conservation of Paiute cutthroat: the genetic legacy of population transplants in an endemic California salmonid. CA Fish and Game. 90-101-118 (pdf).\r\n\r\nFlores-Martnez, J.J., C. Floyd, L.G. Herrera, and B. May. Genetic variation and population size in the fishing bat, Myotis vivesi, in Isla Partida. In: Homenaje a Bernardo Villa, Eds., R. Medelln and V. Snchez-Cordero. Publ. Universidad Nacional Autnoma de Mxico. Pp. 185-190 (pdf).\r\n\r\nHogan, Z.S., P.B. Moyle, B. May, M.J. Vander Zanden and I.G. Baird. Imperiled Giants of the Mekong: Ecologists struggle to understandand protectSoutheast Asia\'s large migratory catfish. Am. Sci. 92:228-237b (pdf).\r\n\r\nIsrael, J.A., J.F. Cordes, M.A. Blumberg, and B. May. Geographic patterns of genetic differentiation among collections of green sturgeon. N. Am. J. Fish. Man. 24:922-931 (pdf).\r\n\r\nMoen, T., J.J. Agresti, A. Cnaani, H. Moses, T.R. Famula, G. Hulata, G.A.E. Gall, B. May. A genome scan of a four-way tilapia cross supports the existence of a quantitative trait locus for cold tolerance on linkage group 23. Aqua. Res. 35:893-904 (pdf).\r\n\r\nRodriguez, F., S. LaPatra, S. Williams, T. Famula and B. May. Genetic markers associated with resistance to infectious hematopoietic necrosis in rainbow and steelhead trout (Oncorhynchus mykiss) backcrosses. Aquaculture 241:93-115 (pdf).\r\n\r\nRodzen, J.A., T. Famula, and B. May. Estimation of parentage and relatedness in the polyploid white sturgeon Acipenser transmontanus using a dominant marker approach for duplicated microsatellite loci. Aquaculture 232:165-182 (pdf).\r\n\r\nSchwartz, R.S. and B. May. Characterization of microsatellite loci in Sacramento perch (Archoplites interruptus). Mol. Ecol. Notes. 4:694-697 (pdf).\r\n\r\nStadelmann, B., L.G. Herrera, J. Arroyo-Cabrales, J.J. Flores-Martnez, B.P. May, and M. Ruedi. Molecular systematics of the fishing bat Myotis (Pizonyz) vivesi. J. Mammo. 85:133139 (pdf).\r\n\r\nTopinka, J.R., A.J. Donovan, and B. May. Characterization of microsatellite loci in Kearneys bluestar (Amsonia kearneyana) and cross-amplification in other Amsonia species. Mol. Ecol. Notes. 4:710-712 (pdf).\r\n\r\nTopinka, J.R. and B. May. Development of polymorphic microsatellite loci in the northern goshawk (Accipiter gentilis atricapillus) and cross-amplification in other raptor species. Cons. Gen. 5:861-864 (pdf).\r\n\r\nTranah, G.J., D.E. Campton and B. May. Genetic evidence for hybridization of pallid and shovelnose sturgeon. J. Hered. 95:474-480 (pdf).\r\n\r\n<strong>2003</strong>\r\n\r\nBeyer, J. and B. May. A graph-theoretic approach to the partition of individuals into full-sib families. Mol. Ecol. 12:2243-2250 (pdf).\r\n\r\nErnest, H., W.M. Boyce, V. Bleich, B. May, S. Stiver, and S. Torres. Genetic structure of mountain lion (Puma concolor) populations in California. Cons. Gen. 3:353-366 (pdf).\r\n\r\nMay, B. Allozyme variation. In: Population Genetics: Principles and Applications for Fisheries Scientists, Ed. E. Hallerman. Amer. Fish. Soc. Pp. 23-36.\r\n\r\nMcQuown, E., C.C. Krueger, H.L. Kincaid, G.A.E. Gall, and B. May. Genetic comparison of lake sturgeon (Acipenser fulvescens) populations: differentiation based on allelic frequencies at seven microsatellite loci. J. Great Lakes Res. 29:3-13 (pdf).\r\n\r\nTranah, G.J., M.J. Bagley, J.J. Agresti, and B. May. Development of codominant markers for identifying species hybrids. Cons. Gen. 4:537-541 (pdf).\r\n\r\nWelsh, A.B., M. Blumberg, and B. May. Identification of microsatellite loci in lake sturgeon, Acipenser fulvescens, and their variability in green sturgeon, A. medirostris. Mol. Ecol. Notes 3:47-55 (pdf).\r\n\r\nWhitehead, A., S.L. Anderson, K.M Kuivila, J.L. Roach, and B. May. Genetic variation among interconnected populations of Catostomus occidentalis: implications for distinguishing impacts of contaminants from biogeographic structuring. Mol. Ecol. 12: 2817-2833 (pdf).\r\n\r\n<strong>2002</strong>\r\n\r\nBlankenship, S.M., B. May, and D. Hedgecock. Evolution of a perfect simple-sequence-repeat locus in the context of its flanking sequence. Mol. Biol. Evol. 19:1943-1951 (pdf).\r\n\r\nHogan, Z. S. and B. May. Twenty-seven new microsatellites for the migratory Asian catfish family Pangasiidae. Mol. Ecol. Notes. 2:38-41 (pdf).\r\n\r\nMcQuown, E., G.A.E. Gall, and B. May. Characterization and inheritance of six microsatellite loci in lake sturgeon (Acipenser fulvescens). Trans. Am. Fish. Soc. 131:299-307.\r\n\r\nMeredith, E.P. and B. May. Microsatellite loci in the Lahontan tui chub, Gila biocolor obesa, and their utililization in other chub species. Mol. Ecol. Notes. 2:156-158 (pdf).\r\n\r\nRodzen, J.A. and B. May. Inheritance of microsatellite loci in the white sturgeon (Acipenser transmontanus). Genome 45:1064-1076 (pdf).\r\n\r\nSmith, C.T., R.J. Nelson, S. Pollard, E. Rubridge, S.J. McKay, J. Rodzen, B. May, and B. Koop. Population genetic analysis of white sturgeon (Acipenser transmontanus) in the Fraser River. J. Appl. Icthy. 18:307-312 (pdf).\r\n\r\nWilliamson, K.S., J.F. Cordes, and B. May. Characterization of microsatellite loci in Chinook salmon (Oncorhynchus tshawytscha) and cross-species amplification in other salmonids. Mol. Ecol. Notes. 2:17-19 (pdf).\r\n\r\nWilliamson, K. and B. May. Incidence of phenotypic female chinook salmon (Oncorhynchus tshawytscha) positive for the Y-chromosome specific marker OtY1 in the Central Valley. J. Aquat. An. Health. 176-183 (pdf).\r\n\r\nZhu, B., F. Zhou, H. Cao, Z. Shao, N. Zhao, B. May, and J. Chang. Analysis of genetic variation in the Chinese sturgeon, Acipenser sinensis: estimating the contribution of artificially produced larvae in a wild population J. Appl. Icthy. 18:301-306 (pdf).\r\n\r\n<strong>2001</strong>\r\n\r\nBagley, M.J., S. Anderson, and B. May. Choice of methodology for assessing genetic impacts of environmental stressors: polymorphism and reproducibility of RAPD and AFLP fingerprints. Ecotoxicology 10:239-244 (pdf).\r\n\r\nJenneckens I., J.-N. Meyer, G. Hrstgen-Schwark, B. May, L. Debus, A. Ludwig. A fixed allele at microsatellite LS-39 exhibiting species-specificity for black caviar producer Acipenser stellatus. J. Appl. Icthyol. 17:39-42 (pdf).\r\n\r\nPyatskowit, J., C.C. Krueger, H.L. Kincaid, and B. May. Inheritance of microsatellite loci in the polyploid derivative lake sturgeon (Acipenser fulvescens). Genome 44:185-191 (pdf).\r\n\r\nTranah, G.J., J.J. Agresti, and B. May. New microsatellite loci for suckers (Catostomidae): primer homology in Catostomus, Chasmistes, and Deltistes. Mol. Ecol. Notes 1:55-60 (pdf).\r\n\r\nTranah, G.J., H.L. Kincaid, C.C. Krueger, D.E. Campton, B. May. Reproductive isolation in sympatric populations of pallid and shovelnose sturgeon. N. Am. J. Fish. Man. 21:367-373 (pdf).\r\n\r\n<strong>2000</strong>\r\n\r\n<span style=\"line-height: 1.714285714; font-size: 1rem;\">Agresti, J.J., S. Seki, A. Cnaani, S. Poompuang, E. M. Hallerman, N. Umiel, G. Hulata, G. A.E. Gall, and B. May. Breeding new strains of tilapia: development of an artificial center of origin and linkage map based on AFLP and microsatellite loci. Aquaculture. 185:43-56 (pdf).</span>\r\n\r\nBelfiore, N.M. and B. May. Variable microsatellite loci in red swamp crayfish, Procambarus clarkii, and their characterization in other crayfish taxa. Mol. Ecol. 12:2231-2234 (pdf).\r\n\r\nErnest, H.B., M.C.T. Penedo, B. May, M. Syvanen, and W.M. Boyce. Molecular tracking of mountain lions in the Yosemite Valley region in California: genetic analysis using microsatellites and fecal DNA. Mol. Ecol. 9: 433-441 (pdf).\r\n\r\nLudwig, A., B. May, L. Debus, and I, Jenneckens. Heteroplasmy in the mtDNA Control Region of Sturgeon (Acipenser, Huso and Scaphirhynchus) Genetics 156:1933-1947 (pdf).\r\n\r\nMcQuown, E.C., B.L. Sloss, R.J. Sheehan, J. Rodzen, G. Tranah, and B. May. Microsatellite analysis of genetic variation in sturgeon: new primer sequences for Scaphirynchus and Acipenser. Trans. Am. Fish. Soc. 129:1380-1388 (pdf).','Publications 2000-2004','','inherit','closed','open','','135-revision-v1','','','2014-02-21 21:13:21','2014-02-21 21:13:21','',135,'http://localhost/~psalessi/Genetics_Lab/135-revision-v1/',0,'revision','',0),(149,1,'2014-03-07 20:20:48','2014-03-07 20:20:48','<strong>1992</strong>\n\nMay, B. Starch gel electrophoresis of allozymes. In: Molecular Genetic Analysis of Populations: A Practical Approach. A.R. Hoelzel, ed. Oxford Univ. Press. pp. 1-27 and 271-280.\n\nMay, B. and J.E. Marsden. Genetic identification and implications of another invasive species of Dreissenid mussel in the Great Lakes. Can. J. Fish. Aqua. Sci. 49:1501-1506.\n\nStaub, J.E., L.D. Knerr, D.J.Holder, and B. May. Phylogenetic relationships among several African Cucumis species. Can. J. Botany. 70:509-517.\n\nSt. Leger, R.J. Allee, L.L, May. B, Staples, R.C and Roberts, D.W. World-wide distribution of genetic variation among isolates of Beauveria spp. Mycol. Res. 96:1007-1015.\n\nSt. Leger, R. J., B. May, L. Allee, D. C. Frank, R. C. Staples, and D. W. Roberts. Genetic differences in allozymes and in formation of infection structures among isolates of the Entomopathogenic fungus Metarhizium anisopliae. J. Inv. Path. 60:89-101.\n\n<strong>1991</strong>\n\n<strong></strong>Bowden, C.G., D.J. Royse, and B. May. Linkage relationships of allozyme encoding loci in shitake, Lentinula edodes. Genome 34:652-657.\n\nGavin, T.A., R.A. Howard, and B. May. Geographic variation in allozyme frequencies in red-winged blackbirds: the California conundrum. Auk 108:602-611 (<a title=\"Publications_5_files/1991 - Red-winged blackbirds.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_5_files/1991%20-%20Red-winged%20blackbirds.pdf\">pdf</a>).\n\nKrueger, C.C., and B. May. Ecological and Genetic Effects of Salmonid Introductions in North America. Can. J. Fish. &amp; Aq. Sci. 48:1-12.\n\nKukuk, P.F. and B. May. Colony dynamics in a primitively eusocial halictine bee, Lasioglossum (Dialictus) zephyrus, (Hymenoptera: halictidae). Insectes Sociaux 38:171-189.\n\n<strong>1990</strong>\n\nDixon, P. M., and B. May. Genetic diversity and population structure of a rare plant, Northern Monkshood (Aconitum noveboracense). In:Ecosystem Management: rare species and significant habitats. Proc. 15th Natural Areas Meeting. NY state museum bulletin No. 471. pp. 167-175.\n\nHajek, A.E., R.A. Humber, J.S. Elkinton, B. May, S.R. Walsh, and J.C. Silver. Allozyme and RFLP analyses confirm Entomophaga maimaiga responsible for 1989 epizootics in North American gypsy moth populations. PNAS. 87:6979-6982 (<a title=\"Publications_5_files/1990 - Entomophaga maimaiga.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_5_files/1990%20-%20Entomophaga%20maimaiga.pdf\">pdf</a>).\n\nKukuk, P.F. and B. May. Diploid males in a primitively eusocial bee, Lasioglossum (Dialictus) zephyrum (Hymenoptera: Halictidae). Evolution 44:1522-1528 (<a title=\"Publications_5_files/1990 - Lasioglossum (Dialictus) zephyrum.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_5_files/1990%20-%20Lasioglossum%20%28Dialictus%29%20zephyrum.pdf\">pdf</a>).\n\nMay, B. and C.C. Krueger. Use of allozyme data for population analysis. In: Electrophoretic and Isoelectric Focussing Techniques in Fisheries Management. D.H. Whitmore, (ed.). pp. 157-172. CRC Press.\n\nRoyse, D.J., and B. May. Interspecific allozyme variation among Morchella spp. and its inferences for systematics within the genus. Biochem. Syst. &amp; Ecol. 18:475-479 (pdf).\n\n<strong>Later publications:</strong>\n<ul>\n	<li><a title=\"Publications\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/\">2009-2013</a></li>\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\n</ul>','Publications 1990-1992','','inherit','closed','open','','142-autosave-v1','','','2014-03-07 20:20:48','2014-03-07 20:20:48','',142,'http://localhost/~psalessi/Genetics_Lab/142-autosave-v1/',0,'revision','',0),(150,1,'2014-02-21 21:15:29','2014-02-21 21:15:29','<strong>1992</strong>\r\n\r\nMay, B. Starch gel electrophoresis of allozymes. In: Molecular Genetic Analysis of Populations: A Practical Approach. A.R. Hoelzel, ed. Oxford Univ. Press. pp. 1-27 and 271-280.\r\n\r\nMay, B. and J.E. Marsden. Genetic identification and implications of another invasive species of Dreissenid mussel in the Great Lakes. Can. J. Fish. Aqua. Sci. 49:1501-1506.\r\n\r\nStaub, J.E., L.D. Knerr, D.J.Holder, and B. May. Phylogenetic relationships among several African Cucumis species. Can. J. Botany. 70:509-517.\r\n\r\nSt. Leger, R.J. Allee, L.L, May. B, Staples, R.C and Roberts, D.W. World-wide distribution of genetic variation among isolates of Beauveria spp. Mycol. Res. 96:1007-1015.\r\n\r\nSt. Leger, R. J., B. May, L. Allee, D. C. Frank, R. C. Staples, and D. W. Roberts. Genetic differences in allozymes and in formation of infection structures among isolates of the Entomopathogenic fungus Metarhizium anisopliae. J. Inv. Path. 60:89-101.\r\n\r\n<strong>1991</strong>\r\n\r\n<strong></strong>Bowden, C.G., D.J. Royse, and B. May. Linkage relationships of allozyme encoding loci in shitake, Lentinula edodes. Genome 34:652-657.\r\n\r\nGavin, T.A., R.A. Howard, and B. May. Geographic variation in allozyme frequencies in red-winged blackbirds: the California conundrum. Auk 108:602-611 (pdf).\r\n\r\nKrueger, C.C., and B. May. Ecological and Genetic Effects of Salmonid Introductions in North America. Can. J. Fish. &amp; Aq. Sci. 48:1-12.\r\n\r\nKukuk, P.F. and B. May. Colony dynamics in a primitively eusocial halictine bee, Lasioglossum (Dialictus) zephyrus, (Hymenoptera: halictidae). Insectes Sociaux 38:171-189.\r\n\r\n<strong>1990</strong>\r\n\r\nDixon, P. M., and B. May. Genetic diversity and population structure of a rare plant, Northern Monkshood (Aconitum noveboracense). In:Ecosystem Management: rare species and significant habitats. Proc. 15th Natural Areas Meeting. NY state museum bulletin No. 471. pp. 167-175.\r\n\r\nHajek, A.E., R.A. Humber, J.S. Elkinton, B. May, S.R. Walsh, and J.C. Silver. Allozyme and RFLP analyses confirm Entomophaga maimaiga responsible for 1989 epizootics in North American gypsy moth populations. PNAS. 87:6979-6982 (pdf).\r\n\r\nKukuk, P.F. and B. May. Diploid males in a primitively eusocial bee, Lasioglossum (Dialictus) zephyrum (Hymenoptera: Halictidae). Evolution 44:1522-1528 (pdf).\r\n\r\nMay, B. and C.C. Krueger. Use of allozyme data for population analysis. In: Electrophoretic and Isoelectric Focussing Techniques in Fisheries Management. D.H. Whitmore, (ed.). pp. 157-172. CRC Press.\r\n\r\nRoyse, D.J., and B. May. Interspecific allozyme variation among Morchella spp. and its inferences for systematics within the genus. Biochem. Syst. &amp; Ecol. 18:475-479 (pdf).\r\n\r\n<strong>Later publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/\">2009-2013</a></li>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n</ul>','Publications 1990-1992','','inherit','closed','open','','142-revision-v1','','','2014-02-21 21:15:29','2014-02-21 21:15:29','',142,'http://localhost/~psalessi/Genetics_Lab/142-revision-v1/',0,'revision','',0),(151,1,'2014-03-07 20:18:24','2014-03-07 20:18:24','<strong>1999</strong>\n\nGavin, T.A., P.W. Sherman, E. Yensen, and B. May. Taxonomic status of Spermophilus brunneus and S. b. endemicus with comparison to other western spermophilus. J. Mam. 80:156-168.\n\nSeki, S, J.J. Agresti, , G.A.E. Gall, N. Taniguchi, and B. May. AFLP analysis of genetic diversity in three populations of ayu (Plecoglossus altivelis). Fish. Sci. 65:888-899.\n\n<strong>1998</strong>\n\nGavin, T.A., R.T. Reynolds, S.M. Joy, and B. May. Genetic evidence for low frequency of extra-pair fertilizations in Northern goshawks. Condor. 100:556-560 (<a title=\"Publications_4_files/1998 - Goshawks extra-pair.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1998%20-%20Goshawks%20extra-pair.pdf\">pdf</a>).\n\nMay, B. Starch gel electrophoresis of allozymes. In: Molecular Genetic Analysis of Populations: A Practical Approach. 2nd Ed. A.R. Hoelzel, ed. Oxford Univ. Press. Pp. 1-28 and 371-378.\n\nMay, B. Book Review. Molecular Systematics of Fishes, T.D. Kocher and C.A. Stepiens, eds. Trans. Am. Fish. Soc. 127:857-858.\n\nRodzen, J., J.J. Agresti, G. Tranah, and B. May. Agarose overlays allow simplified staining of polyacrylamide gels. Biotechniques. 25:584.\n\n<strong>1997</strong>\n\nMay, B., T.A. Gavin, P.W. Sherman, and T.M. Korves. Characterization of microsatellite loci in the Northern Idaho ground squirrel, Spermophilus brunneus Mol. Ecol. 6:399-400 (<a title=\"Publications_4_files/1997 - Spermophilus brunneus.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1997%20-%20Spermophilus%20brunneus.pdf\">pdf</a>).\n\nMay, B., C.C. Krueger, and H.L. Kincaid. Genetic variation at microsatellite loci in sturgeon: primer sequence homology in Acipenser and Scaphirhynchus. Can. J. Fish. &amp; Aquat. Sci. 54: 1542-1547 (<a title=\"Publications_4_files/1997 - May sturgeon 1997.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1997%20-%20May%20sturgeon%201997.pdf\">pdf</a>).\n\n<strong>1996</strong>\n\nLegge, J.T., R. Roush, B. May, R. DeSalle, and A. Vogler. Genetic criteria for establishing evolutionarily significant units in Cryan\'s buckmoth. Cons. Biol. 10(1):85-98 (<a title=\"Publications_4_files/1996 - Cryan\'s buckmoth.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1996%20-%20Cryan%27s%20buckmoth.pdf\">pdf</a>).\n\nMarsden, J.E., A. Spidle, and B. May. Review of genetic studies of Dreissena spp. Amer. Zool. 36:259-270 (<a title=\"Publications_4_files/1996 - marsden et al am zoo.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1996%20-%20marsden%20et%20al%20am%20zoo.pdf\">pdf</a>).\n\nMills, E.L., G. Rosenberg, A.P. Spidle, M. Ludyanskiy, Y. Pligin, and B. May. A review of the biology and ecology of the quagga mussel (Dreissena bugensis): a second species of freshwater dreissenid introduced to North America. Submitted to Amer. Zool. 36: 271-286.\n\n<strong>1995</strong>\n\nMarsden, J. E., A. Spidle, and B. May. Genetic similarity among zebra mussel poulations within North America and within Europe. Can. J. Fish. Aquat. Sci. 52:836-847 (<a title=\"Publications_4_files/1995 - marsden_et_al_zebra_mussel_structure[1].pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1995%20-%20marsden_et_al_zebra_mussel_structure%5B1%5D.pdf\">pdf</a>).\n\nPerkins, D.L., J. Fitzsimons, J.E. Marsden, C.C. Krueger, and B. May. Differences in reproduction among hatchery strains of lake trout at eight spawning areas in Lake Ontario: genetic evidence from mixed-stock analysis. J. Great Lakes Res. 21:364-374.\n\nSpidle, A.P., E.L. Mills, and B. May. Absence of naturally occurring hybridization between the quagga mussel (Dreissena bugensis) and the zebra mussel (D. polymorpha). Can. J. Zool. . 73:400-403 (<a title=\"Publications_4_files/1995 - spidle et al quagga zebe hyb.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1995%20-%20spidle%20et%20al%20quagga%20zebe%20hyb.pdf\">pdf</a>).\n\nSpidle, A.P., E.L. Mills, and B. May. Differential response to changes in salinity and temperature in the quagga and zebra mussels (Dreissena bugensis Andrusov and D. polymorpha Pallas). Can. J. Fish. Aquat. Sci. 52:2108-2119 (<a title=\"Publications_4_files/1995 - spidle et al dreissena physiology ocr.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1995%20-%20spidle%20et%20al%20dreissena%20physiology%20ocr.pdf\">pdf</a>).\n\n<strong>1994</strong>\n\nAiken, S.G., L.L. Consaul, A. Spidle, and B. May. Allozyme and morphological observations on Festuca hyperborea, compared with F. baffinensis and F. brachyphylla (Poaceae) from the Canadian Arctic. Nord. J. Bot. 14(2): 137-143.\n\nGrewe, P.M., C.C. Krueger, C.F. Aquadro, and B. May. Stability of allozyme and mitochondrial DNA markers among three year classes of lake trout propagated from Seneca Lake, New York. N. Amer. J. Fish. Man. 14:467-474 (<a title=\"Publications_4_files/1994 - GreweEtAl.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1994%20-%20GreweEtAl.pdf\">pdf</a>).\n\nGrewe, P.M., C.C. Krueger, J.E. Marsden, C.F. Aquadro, and B. May. Hatchery origins of naturally produced lake trout fry captured in Lake Ontario: temporal and spatial variability based on allozyme and mitochondrial DNA data. Trans. Am. Fish. Soc. 123:309-320.\n\nKrueger, C.C., D.L. Perkins, R.J. Everett, D.R. Schreiner, and B. May. Genetic variation in naturalized rainbow trout (steelhead) from Minnesota tributaries to Lake Superior. J. Great Lakes Res. 20:299-316.\n\nSpidle, A.P., J.E. Marsden, and B. May. Identification of the Great Lakes quagga mussel as Dreissena bugensis from the Dnieper River, Ukraine, on the basis of allozyme variation. Can. J. Fish. Aquat. Sci. 51:1485-1489 (<a title=\"Publications_4_files/1994 - spidle et al quagga id.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1994%20-%20spidle%20et%20al%20quagga%20id.pdf\">pdf</a>).\n\n<strong>1993</strong>\n\nGrewe, P.M., C.C. Krueger, C.F. Aquadro, E. Bermingham, H. L. Kincaid, and B. May. Mitochodrial DNA variation among lake trout (Salvelinus namaycush) strains stocked into Lake Ontario. Can J. Fish. Aquat. Sci. 50:2397-2403.\n\nKincaid, H. L. , C.C. Krueger, and B. May. Genetic variation among remnant groups of the Green Lake strain of lake trout: preservation of the remnant Green Lake gene pool. N. Am. J. Fish. Man. 13:318-325 (<a title=\"Publications_4_files/1993 - KincaidMay.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1993%20-%20KincaidMay.pdf\">pdf</a>).\n\nMarsden, J.E., C.C. Krueger, P.M. Grewe, H.L. Kincaid, and B. May. Genetic comparison of naturally-spawned and artificially-propagated Lake Ontario lake trout fry: evaluation of a stocking strategy for species rehabilitation. N. Am. J. Fish. Man. 13:304-317 (pdf).\n\nMay, B. and P. M. Grewe. Fate of maternal mtDNA following Cobalt60 inactivation of maternal nuclear DNA in unfertilized salmonid eggs. Genome 36:725-730.\n\nMay, B. and K. R. Johnson. Composite linkage map of salmonid fishes (Salvelinus, Salmo, and Oncorhynchus). In: Genetic Maps : Locus Maps of Complex Genomes. O\'Brien, S.J. (ed.) Cold Spring Harbor. 4:309-317.\n\nPerkins, D.L., C.C. Krueger, and B. May. Heritage brook trout in Northeastern USA: genetic variability within and among populations. Trans. Am Fish. Soc.122:515-532.\n\nRoyse, D.J. and B. May. Multilocus enzyme electrophoresis for the genetic analysis of edible mushrooms. In: Genetics and Breeding of Edible Mushrooms. Chang, S.T., J.A. Buswell, and P.G. Miles (ed.). pp. 225-247. Gordon and Breach Sci. Pub. Phil.\n\n<strong>Later Publications:</strong>\n<ul>\n	<li><a title=\"Publications\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/\">2009-2013</a></li>\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\n</ul>\n<strong>Earlier Publications:</strong>\n<ul>\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\n</ul>','Publications 1993-1999','','inherit','closed','open','','140-autosave-v1','','','2014-03-07 20:18:24','2014-03-07 20:18:24','',140,'http://localhost/~psalessi/Genetics_Lab/140-autosave-v1/',0,'revision','',0),(152,1,'2014-02-21 21:16:38','2014-02-21 21:16:38','<strong>1999</strong>\r\n\r\nGavin, T.A., P.W. Sherman, E. Yensen, and B. May. Taxonomic status of Spermophilus brunneus and S. b. endemicus with comparison to other western spermophilus. J. Mam. 80:156-168.\r\n\r\nSeki, S, J.J. Agresti, , G.A.E. Gall, N. Taniguchi, and B. May. AFLP analysis of genetic diversity in three populations of ayu (Plecoglossus altivelis). Fish. Sci. 65:888-899.\r\n\r\n<strong>1998</strong>\r\n\r\nGavin, T.A., R.T. Reynolds, S.M. Joy, and B. May. Genetic evidence for low frequency of extra-pair fertilizations in Northern goshawks. Condor. 100:556-560 (pdf).\r\n\r\nMay, B. Starch gel electrophoresis of allozymes. In: Molecular Genetic Analysis of Populations: A Practical Approach. 2nd Ed. A.R. Hoelzel, ed. Oxford Univ. Press. Pp. 1-28 and 371-378.\r\n\r\nMay, B. Book Review. Molecular Systematics of Fishes, T.D. Kocher and C.A. Stepiens, eds. Trans. Am. Fish. Soc. 127:857-858.\r\n\r\nRodzen, J., J.J. Agresti, G. Tranah, and B. May. Agarose overlays allow simplified staining of polyacrylamide gels. Biotechniques. 25:584.\r\n\r\n<strong>1997</strong>\r\n\r\nMay, B., T.A. Gavin, P.W. Sherman, and T.M. Korves. Characterization of microsatellite loci in the Northern Idaho ground squirrel, Spermophilus brunneus Mol. Ecol. 6:399-400 (pdf).\r\n\r\nMay, B., C.C. Krueger, and H.L. Kincaid. Genetic variation at microsatellite loci in sturgeon: primer sequence homology in Acipenser and Scaphirhynchus. Can. J. Fish. &amp; Aquat. Sci. 54: 1542-1547 (pdf).\r\n\r\n<strong>1996</strong>\r\n\r\nLegge, J.T., R. Roush, B. May, R. DeSalle, and A. Vogler. Genetic criteria for establishing evolutionarily significant units in Cryan\'s buckmoth. Cons. Biol. 10(1):85-98 (pdf).\r\n\r\nMarsden, J.E., A. Spidle, and B. May. Review of genetic studies of Dreissena spp. Amer. Zool. 36:259-270 (pdf).\r\n\r\nMills, E.L., G. Rosenberg, A.P. Spidle, M. Ludyanskiy, Y. Pligin, and B. May. A review of the biology and ecology of the quagga mussel (Dreissena bugensis): a second species of freshwater dreissenid introduced to North America. Submitted to Amer. Zool. 36: 271-286.\r\n\r\n<strong>1995</strong>\r\n\r\nMarsden, J. E., A. Spidle, and B. May. Genetic similarity among zebra mussel poulations within North America and within Europe. Can. J. Fish. Aquat. Sci. 52:836-847 (pdf).\r\n\r\nPerkins, D.L., J. Fitzsimons, J.E. Marsden, C.C. Krueger, and B. May. Differences in reproduction among hatchery strains of lake trout at eight spawning areas in Lake Ontario: genetic evidence from mixed-stock analysis. J. Great Lakes Res. 21:364-374.\r\n\r\nSpidle, A.P., E.L. Mills, and B. May. Absence of naturally occurring hybridization between the quagga mussel (Dreissena bugensis) and the zebra mussel (D. polymorpha). Can. J. Zool. . 73:400-403 (pdf).\r\n\r\nSpidle, A.P., E.L. Mills, and B. May. Differential response to changes in salinity and temperature in the quagga and zebra mussels (Dreissena bugensis Andrusov and D. polymorpha Pallas). Can. J. Fish. Aquat. Sci. 52:2108-2119 (pdf).\r\n\r\n<strong>1994</strong>\r\n\r\nAiken, S.G., L.L. Consaul, A. Spidle, and B. May. Allozyme and morphological observations on Festuca hyperborea, compared with F. baffinensis and F. brachyphylla (Poaceae) from the Canadian Arctic. Nord. J. Bot. 14(2): 137-143.\r\n\r\nGrewe, P.M., C.C. Krueger, C.F. Aquadro, and B. May. Stability of allozyme and mitochondrial DNA markers among three year classes of lake trout propagated from Seneca Lake, New York. N. Amer. J. Fish. Man. 14:467-474 (pdf).\r\n\r\nGrewe, P.M., C.C. Krueger, J.E. Marsden, C.F. Aquadro, and B. May. Hatchery origins of naturally produced lake trout fry captured in Lake Ontario: temporal and spatial variability based on allozyme and mitochondrial DNA data. Trans. Am. Fish. Soc. 123:309-320.\r\n\r\nKrueger, C.C., D.L. Perkins, R.J. Everett, D.R. Schreiner, and B. May. Genetic variation in naturalized rainbow trout (steelhead) from Minnesota tributaries to Lake Superior. J. Great Lakes Res. 20:299-316.\r\n\r\nSpidle, A.P., J.E. Marsden, and B. May. Identification of the Great Lakes quagga mussel as Dreissena bugensis from the Dnieper River, Ukraine, on the basis of allozyme variation. Can. J. Fish. Aquat. Sci. 51:1485-1489 (pdf).\r\n\r\n<strong>1993</strong>\r\n\r\nGrewe, P.M., C.C. Krueger, C.F. Aquadro, E. Bermingham, H. L. Kincaid, and B. May. Mitochodrial DNA variation among lake trout (Salvelinus namaycush) strains stocked into Lake Ontario. Can J. Fish. Aquat. Sci. 50:2397-2403.\r\n\r\nKincaid, H. L. , C.C. Krueger, and B. May. Genetic variation among remnant groups of the Green Lake strain of lake trout: preservation of the remnant Green Lake gene pool. N. Am. J. Fish. Man. 13:318-325 (pdf).\r\n\r\nMarsden, J.E., C.C. Krueger, P.M. Grewe, H.L. Kincaid, and B. May. Genetic comparison of naturally-spawned and artificially-propagated Lake Ontario lake trout fry: evaluation of a stocking strategy for species rehabilitation. N. Am. J. Fish. Man. 13:304-317 (pdf).\r\n\r\nMay, B. and P. M. Grewe. Fate of maternal mtDNA following Cobalt60 inactivation of maternal nuclear DNA in unfertilized salmonid eggs. Genome 36:725-730.\r\n\r\nMay, B. and K. R. Johnson. Composite linkage map of salmonid fishes (Salvelinus, Salmo, and Oncorhynchus). In: Genetic Maps : Locus Maps of Complex Genomes. O\'Brien, S.J. (ed.) Cold Spring Harbor. 4:309-317.\r\n\r\nPerkins, D.L., C.C. Krueger, and B. May. Heritage brook trout in Northeastern USA: genetic variability within and among populations. Trans. Am Fish. Soc.122:515-532.\r\n\r\nRoyse, D.J. and B. May. Multilocus enzyme electrophoresis for the genetic analysis of edible mushrooms. In: Genetics and Breeding of Edible Mushrooms. Chang, S.T., J.A. Buswell, and P.G. Miles (ed.). pp. 225-247. Gordon and Breach Sci. Pub. Phil.\r\n\r\n<strong>Later Publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/\">2009-2013</a></li>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n</ul>\r\n<strong>Earlier Publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications 1993-1999','','inherit','closed','open','','140-revision-v1','','','2014-02-21 21:16:38','2014-02-21 21:16:38','',140,'http://localhost/~psalessi/Genetics_Lab/140-revision-v1/',0,'revision','',0),(153,1,'2014-02-21 21:17:42','2014-02-21 21:17:42','<strong>2004</strong>\r\n\r\nBaerwald, M.R. and B. May. Characterization of microsatellite loci for five members of the minnow family Cyprinidae found in the Sacramento - San Joaquin Delta. Mol. Ecol. Notes. 4:385-390 (pdf).\r\n\r\nCordes, J.F., J.A. Israel and B. May. Conservation of Paiute cutthroat: the genetic legacy of population transplants in an endemic California salmonid. CA Fish and Game. 90-101-118 (pdf).\r\n\r\nFlores-Martnez, J.J., C. Floyd, L.G. Herrera, and B. May. Genetic variation and population size in the fishing bat, Myotis vivesi, in Isla Partida. In: Homenaje a Bernardo Villa, Eds., R. Medelln and V. Snchez-Cordero. Publ. Universidad Nacional Autnoma de Mxico. Pp. 185-190 (pdf).\r\n\r\nHogan, Z.S., P.B. Moyle, B. May, M.J. Vander Zanden and I.G. Baird. Imperiled Giants of the Mekong: Ecologists struggle to understandand protectSoutheast Asia\'s large migratory catfish. Am. Sci. 92:228-237b (pdf).\r\n\r\nIsrael, J.A., J.F. Cordes, M.A. Blumberg, and B. May. Geographic patterns of genetic differentiation among collections of green sturgeon. N. Am. J. Fish. Man. 24:922-931 (pdf).\r\n\r\nMoen, T., J.J. Agresti, A. Cnaani, H. Moses, T.R. Famula, G. Hulata, G.A.E. Gall, B. May. A genome scan of a four-way tilapia cross supports the existence of a quantitative trait locus for cold tolerance on linkage group 23. Aqua. Res. 35:893-904 (pdf).\r\n\r\nRodriguez, F., S. LaPatra, S. Williams, T. Famula and B. May. Genetic markers associated with resistance to infectious hematopoietic necrosis in rainbow and steelhead trout (Oncorhynchus mykiss) backcrosses. Aquaculture 241:93-115 (pdf).\r\n\r\nRodzen, J.A., T. Famula, and B. May. Estimation of parentage and relatedness in the polyploid white sturgeon Acipenser transmontanus using a dominant marker approach for duplicated microsatellite loci. Aquaculture 232:165-182 (pdf).\r\n\r\nSchwartz, R.S. and B. May. Characterization of microsatellite loci in Sacramento perch (Archoplites interruptus). Mol. Ecol. Notes. 4:694-697 (pdf).\r\n\r\nStadelmann, B., L.G. Herrera, J. Arroyo-Cabrales, J.J. Flores-Martnez, B.P. May, and M. Ruedi. Molecular systematics of the fishing bat Myotis (Pizonyz) vivesi. J. Mammo. 85:133139 (pdf).\r\n\r\nTopinka, J.R., A.J. Donovan, and B. May. Characterization of microsatellite loci in Kearneys bluestar (Amsonia kearneyana) and cross-amplification in other Amsonia species. Mol. Ecol. Notes. 4:710-712 (pdf).\r\n\r\nTopinka, J.R. and B. May. Development of polymorphic microsatellite loci in the northern goshawk (Accipiter gentilis atricapillus) and cross-amplification in other raptor species. Cons. Gen. 5:861-864 (pdf).\r\n\r\nTranah, G.J., D.E. Campton and B. May. Genetic evidence for hybridization of pallid and shovelnose sturgeon. J. Hered. 95:474-480 (pdf).\r\n\r\n<strong>2003</strong>\r\n\r\nBeyer, J. and B. May. A graph-theoretic approach to the partition of individuals into full-sib families. Mol. Ecol. 12:2243-2250 (pdf).\r\n\r\nErnest, H., W.M. Boyce, V. Bleich, B. May, S. Stiver, and S. Torres. Genetic structure of mountain lion (Puma concolor) populations in California. Cons. Gen. 3:353-366 (pdf).\r\n\r\nMay, B. Allozyme variation. In: Population Genetics: Principles and Applications for Fisheries Scientists, Ed. E. Hallerman. Amer. Fish. Soc. Pp. 23-36.\r\n\r\nMcQuown, E., C.C. Krueger, H.L. Kincaid, G.A.E. Gall, and B. May. Genetic comparison of lake sturgeon (Acipenser fulvescens) populations: differentiation based on allelic frequencies at seven microsatellite loci. J. Great Lakes Res. 29:3-13 (pdf).\r\n\r\nTranah, G.J., M.J. Bagley, J.J. Agresti, and B. May. Development of codominant markers for identifying species hybrids. Cons. Gen. 4:537-541 (pdf).\r\n\r\nWelsh, A.B., M. Blumberg, and B. May. Identification of microsatellite loci in lake sturgeon, Acipenser fulvescens, and their variability in green sturgeon, A. medirostris. Mol. Ecol. Notes 3:47-55 (pdf).\r\n\r\nWhitehead, A., S.L. Anderson, K.M Kuivila, J.L. Roach, and B. May. Genetic variation among interconnected populations of Catostomus occidentalis: implications for distinguishing impacts of contaminants from biogeographic structuring. Mol. Ecol. 12: 2817-2833 (pdf).\r\n\r\n<strong>2002</strong>\r\n\r\nBlankenship, S.M., B. May, and D. Hedgecock. Evolution of a perfect simple-sequence-repeat locus in the context of its flanking sequence. Mol. Biol. Evol. 19:1943-1951 (pdf).\r\n\r\nHogan, Z. S. and B. May. Twenty-seven new microsatellites for the migratory Asian catfish family Pangasiidae. Mol. Ecol. Notes. 2:38-41 (pdf).\r\n\r\nMcQuown, E., G.A.E. Gall, and B. May. Characterization and inheritance of six microsatellite loci in lake sturgeon (Acipenser fulvescens). Trans. Am. Fish. Soc. 131:299-307.\r\n\r\nMeredith, E.P. and B. May. Microsatellite loci in the Lahontan tui chub, Gila biocolor obesa, and their utililization in other chub species. Mol. Ecol. Notes. 2:156-158 (pdf).\r\n\r\nRodzen, J.A. and B. May. Inheritance of microsatellite loci in the white sturgeon (Acipenser transmontanus). Genome 45:1064-1076 (pdf).\r\n\r\nSmith, C.T., R.J. Nelson, S. Pollard, E. Rubridge, S.J. McKay, J. Rodzen, B. May, and B. Koop. Population genetic analysis of white sturgeon (Acipenser transmontanus) in the Fraser River. J. Appl. Icthy. 18:307-312 (pdf).\r\n\r\nWilliamson, K.S., J.F. Cordes, and B. May. Characterization of microsatellite loci in Chinook salmon (Oncorhynchus tshawytscha) and cross-species amplification in other salmonids. Mol. Ecol. Notes. 2:17-19 (pdf).\r\n\r\nWilliamson, K. and B. May. Incidence of phenotypic female chinook salmon (Oncorhynchus tshawytscha) positive for the Y-chromosome specific marker OtY1 in the Central Valley. J. Aquat. An. Health. 176-183 (pdf).\r\n\r\nZhu, B., F. Zhou, H. Cao, Z. Shao, N. Zhao, B. May, and J. Chang. Analysis of genetic variation in the Chinese sturgeon, Acipenser sinensis: estimating the contribution of artificially produced larvae in a wild population J. Appl. Icthy. 18:301-306 (pdf).\r\n\r\n<strong>2001</strong>\r\n\r\nBagley, M.J., S. Anderson, and B. May. Choice of methodology for assessing genetic impacts of environmental stressors: polymorphism and reproducibility of RAPD and AFLP fingerprints. Ecotoxicology 10:239-244 (pdf).\r\n\r\nJenneckens I., J.-N. Meyer, G. Hrstgen-Schwark, B. May, L. Debus, A. Ludwig. A fixed allele at microsatellite LS-39 exhibiting species-specificity for black caviar producer Acipenser stellatus. J. Appl. Icthyol. 17:39-42 (pdf).\r\n\r\nPyatskowit, J., C.C. Krueger, H.L. Kincaid, and B. May. Inheritance of microsatellite loci in the polyploid derivative lake sturgeon (Acipenser fulvescens). Genome 44:185-191 (pdf).\r\n\r\nTranah, G.J., J.J. Agresti, and B. May. New microsatellite loci for suckers (Catostomidae): primer homology in Catostomus, Chasmistes, and Deltistes. Mol. Ecol. Notes 1:55-60 (pdf).\r\n\r\nTranah, G.J., H.L. Kincaid, C.C. Krueger, D.E. Campton, B. May. Reproductive isolation in sympatric populations of pallid and shovelnose sturgeon. N. Am. J. Fish. Man. 21:367-373 (pdf).\r\n\r\n<strong>2000</strong>\r\n\r\n<span style=\"line-height: 1.714285714; font-size: 1rem;\">Agresti, J.J., S. Seki, A. Cnaani, S. Poompuang, E. M. Hallerman, N. Umiel, G. Hulata, G. A.E. Gall, and B. May. Breeding new strains of tilapia: development of an artificial center of origin and linkage map based on AFLP and microsatellite loci. Aquaculture. 185:43-56 (pdf).</span>\r\n\r\nBelfiore, N.M. and B. May. Variable microsatellite loci in red swamp crayfish, Procambarus clarkii, and their characterization in other crayfish taxa. Mol. Ecol. 12:2231-2234 (pdf).\r\n\r\nErnest, H.B., M.C.T. Penedo, B. May, M. Syvanen, and W.M. Boyce. Molecular tracking of mountain lions in the Yosemite Valley region in California: genetic analysis using microsatellites and fecal DNA. Mol. Ecol. 9: 433-441 (pdf).\r\n\r\nLudwig, A., B. May, L. Debus, and I, Jenneckens. Heteroplasmy in the mtDNA Control Region of Sturgeon (Acipenser, Huso and Scaphirhynchus) Genetics 156:1933-1947 (pdf).\r\n\r\nMcQuown, E.C., B.L. Sloss, R.J. Sheehan, J. Rodzen, G. Tranah, and B. May. Microsatellite analysis of genetic variation in sturgeon: new primer sequences for Scaphirynchus and Acipenser. Trans. Am. Fish. Soc. 129:1380-1388 (pdf).\r\n\r\n<strong>Later Publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/\">2009-2013</a></li>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n</ul>\r\n<strong>Earlier Publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications 2000-2004','','inherit','closed','open','','135-revision-v1','','','2014-02-21 21:17:42','2014-02-21 21:17:42','',135,'http://localhost/~psalessi/Genetics_Lab/135-revision-v1/',0,'revision','',0),(154,1,'2014-02-21 21:18:19','2014-02-21 21:18:19','<strong>2008</strong>\r\n\r\nBaerwald, M., V. Bien, F. Feyrer, and B. May. Distribution of genetically differentiated splittail populations during nonspawning season. Trans. Am. Fish. Soc. 137:1335-1345 (pdf).\r\n\r\nBaerwald, M., A. Welsh, R. Hedrick, and B. May. Discovery of genes implicated in whirling disease infection and resistance in rainbow trout using genome-wide expression profiling. BMC Genomics 9:37 (pdf).\r\n\r\nBrk, K.S, A. Drauch, J. Israel, J. Pedroia, J. Rodzen, and B. May. Development of new microsatellite primers for green and white sturgeon. Cons. Gen. 9:973-979 (pdf)\r\n\r\nPetersen, J. L., A. M. Ibarra, J. L. Ramirez, and B. May. An induced mass spawn of the hermaphroditic lion-paw scallop: Genetic assignment of maternal and paternal parentage. J. Hered. 99:337-348 (pdf).\r\n\r\nSchwartz, R.S. and B. May. Genetic Evaluation of Isolated Populations for Use in Reintroductions Reveals Significant Genetic Bottlenecks in Potential Stocks of Sacramento Perch. Trans. Am. Fish. Soc. 137:1764-1777 (pdf).\r\n\r\nWelsh, A., T. Hill, H. Quinlan, C. Robinson, and B. May. Genetic assessment of lake sturgeon population structure in the Great Lakes. N. Am. J. Fish. Man. 28:572591 (pdf).\r\n\r\nWilliamson, K.S., R. Phillips, and B. May. Characterization of a chromosomal rearrangement responsible for producing \"apparent\" XY-female fall Chinook salmon in California. J. Hered. 99:483-490 (pdf).\r\n\r\n<strong>2007</strong>\r\n\r\nBaerwald, M.R., V. Bien, F. Feyrer, and B. May. Genetic analysis reveals two distinct Sacramento splittail (Pogonichthyes macrolepidotus) populations. Cons. Gen. 8:159-167. (pdf).\r\n\r\nChen, Y., S. Parmenter, and B. May. Introgression between Lahontan and endangered Owens tui chubs, and apparent discovery of a new tui chub in the Owens Valley, California. Conserv. Genet. 8:221-238. (pdf).\r\n\r\nFeyrer, F., J. Hobbs, M. Baerwald, T. Sommer, Q. Yin, K. Clark, B. May and W. Bennett. Otolith microchemistry provides information complementary to microsatellite DNA for migratory fish. Trans. Amer. Fish. Soc. 136:469-476. (pdf)\r\n\r\nHull, J.M., D. Tufts, J. Topinka, B. May, and H. Ernest. Development of 19 microsatellite loci for Swainson\'s hawks (Buteo swainsoni) and other buteos. Molecular Ecology Notes 7:346-349. (pdf).\r\n\r\nLindley, S.T., R. Schick, E. Mora, P. Adams, J. Anderson, S. Greene, C. Hanson, B. May, D. McEwan, R. MacFarlane, C. Swanson, and J. Williams. Framework for Assessing Viability of Threatened and Endangered Chinook Salmon and Steelhead in the Sacramento-San Joaquin Basin . San Francisco Estuary and Watershed Science. Vol. 5, Issue 1 (1-26), Article 4. (pdf).\r\n\r\nMeredith, E. P., J.A. Rodzen, J. D. Banks, R. Schaefer, H. B. Ernest, T. R. Famula and B. P. May. Microsatellite analysis of three subspecies of Elk (Cervus elaphus) in California. Journal of Mammalogy 88(3):801-808. (pdf)\r\n\r\nPapa R., J.A. Israel, F. N. Marzan, and B. May. Assessment of genetic variation among reproductive ecotypes of Klamath River steelhead reveals differentiation associated with different run-timings. J. Appl. Icthy. (pdf)\r\n\r\nWilliamson, K. and B. May. Mitochondrial DNA Haplotype Diversity in Apparent XY Female Fall-Run and Spring-Run Chinook Salmon in California\'s Central Valley. Trans. Amer. Fish. Soc. 136: 1480-1486 (pdf).\r\n\r\n<strong>2006</strong>\r\n\r\nCordes, J.F., M.R. Stephens, M.A. Blumberg, and B. May. Identifying introgressive hybridization in native populations of California Golden Trout based on molecular markers. Trans. Am. Fish. Soc. 135:110-128. (pdf).\r\n\r\nIbarra, A.M., J.L. Petersen, T.R. Famula, and B. May. Characterization of 35 microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and their cross-species amplification in four other scallops of the Pectinidae family. Mol. Ecol. Notes. 6:153-156 (pdf).\r\n\r\nLindley, S.T., R.S. Schick, A. Agrawal, M. Goslin, T. Pearson, E. Mora, J. J. Anderson, B.P. May, S. Greene, C. Hanson, A. Low, D. McEwan, R.B. MacFarlane, C. Swanson and J.G. Williams. Historical population structure of Central Valley steelhead and its alteration by dams. SF Est. Wtshd. Sci. 4(1): Article 3 (pdf).\r\n\r\nSprowles, A.E., M.R. Stephens, N.W. Clipperton, and B.P. May. Fishing for SNPs: a targeted locus approach for Single Nucleotide Polymorphism discovery in rainbow trout. Trans. Am. Fish. Soc. 135:1698-1721. (pdf).\r\n\r\nTranah, G.J., and B. P. May. Patterns of Intra- and Interspecies Genetic Diversity in Klamath River Basin Suckers. Trans. Am Fish. Soc. 135:306316 (pdf).\r\n\r\nWelsh, A. and B. May. Development and standardization of disomic microsatellite markers for lake sturgeon genetic studies. J. Appl. Icthy. (pdf).\r\n\r\n<strong>2005</strong>\r\n\r\nBayard de Volo, S., RT Reynolds, JR Topinka, B. May and MF Antolin. Population genetic structure and the utility of muliti-locus genotyping for mark-recapture studies of northern goshawks (Accipiter gentilis). J. Raptor Res. 39:286-295. (pdf)\r\n\r\nCordes, J.F., D.L. Perkins, H.L.Kincaid, and B. May. Genetic analysis of fish genomes and populations: allozyme variation within and among Atlantic salmon from downeast rivers of Maine. J. Fish Biology. 67:104-117. (pdf)\r\n\r\nFloyd, C.H., D. Van Vuren, and B. May. Marmots on Great Basin mountaintops: using genetics to test an ecological paradigm. Ecology. 86: 2145-2153. (pdf)\r\n\r\nFlores Martnez, J.J., C. H. Floyd, L.G. Herrera, and B. May. Genetic variation and population size of the endangered fishing bat, Myotis vivesi, in Isla Partida. Pp. 187-192 in Contribuciones Mastozoolgicas en Homenaje a Bernardo Villa (V. Snchez-Cordero y R. A. Medelln, eds.). Instituto de Biologa, UNAM; Instituto de Ecologa, UNAM; CONABIO. Mxico. 706 pp.\r\n\r\nLeunda, P.M., R. Miranda, J. Madoz, S. Parmenter, Y. Chen, and B. May. Threatened fishes of the world: Siphateles bicolor snyderi (Miller, 1973) (Cyprinidae). Env. Biol. Fish. 73:109-110 (pdf)\r\n\r\nRodrigue, K.L., B.P. May, T.R. Famula, and M.E. Delany. Meiotic instability of chicken ultra-long telomeres and mapping of a 2.8 megabase array to the W-sex chromosome. Chrom. Res.13:581-591. (pdf)\r\n\r\nTeglas, M.B., B. May, P.R. Crosbie, M.R. Stephens, and W.M. Boyce. Genetic structure of the tick Ornithodoros coriaceus (Acarina: Argasidae) in California, Nevada and Oregon. J. Med. Entomol. 42:247-253.\r\n\r\nWilliamson, K. and B. May. Homogenization of fall-run chinook salmon gene pools in the Central Valley of California, USA. N. Am. J. Fish. Man. 25:9931009. (pdf)\r\n\r\nWilliamson, K. and B. May. Inheritance studies implicate a genetic mechanism for apparent sex-reversal in Chinook salmon. Trans. Am. Fish. Soc. 134:12531261. (pdf)\r\n\r\n<strong>Later Publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/\">2009-2013</a></li>\r\n</ul>\r\n<strong>Earlier Publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications 2005-2008','','inherit','closed','open','','129-revision-v1','','','2014-02-21 21:18:19','2014-02-21 21:18:19','',129,'http://localhost/~psalessi/Genetics_Lab/129-revision-v1/',0,'revision','',0),(155,1,'2014-02-21 21:20:39','2014-02-21 21:20:39','<strong>2013</strong>\r\n\r\n<p>Chen, Y., S. Parmenter, and B. May. Genetic characterization of variation and management implications for the endangeredMohave tui chub. Cons. Gen. 14:11-20. (pdf) </p>\r\n\r\nMeek, M. H., A. P. Wintzer, N. Shepherd, and B. May. Genetic diversity and reproductive mode in two non-native hydromedusae, Maeotias marginata and Moerisia sp., in the upper San Francisco Estuary, California. Biol. Invasions. 15(1): 199-212. (pdf)\r\n\r\nSchreier, A.D., B. Mahardja, and B. May. Hierarchical patterns of population structure in the endangered Fraser River white sturgeon, Acipenser transmontanus, and implications for conservation. Can. J. Fish. &amp; Aquat. Sci. 69:1968-1980. (pdf)\r\n\r\n<strong>2012</strong>\r\n\r\nBaerwald, M.R., B.M. Schreier, G. Schumer, and B. May. Detection of Threatened Delta Smelt in the Gut Contents of the Invasive Mississippi Silverside in the San Francisco Estuary Using TaqMan Assays. Trans. Am. Fish. Soc. 141:1600-1607. (pdf)\r\n\r\nMahardja, B., B. May, and M. Baerwald. Characterization of 36 additional microsatellite loci in splittail (Pogonichthys macrolepidotus) and cross-amplification in five other native Californian cyprinid species. Cons. Gen. Res. 4(4):917-921. (pdf)\r\n\r\nMeek, M., A. Wintzer, W. Wetzel, and B. May. Climate change likely to facilitate the invasion of the non-native hydroid, Cordylophora caspia, in the San Francisco Estuary, CA. PLos ONE. 7(10): e46373. doi:10.1371/journal.pone.0046373 (pdf)\r\n\r\nPetersen, J.L., M.R. Baerwald, J.M. Ibarra, and B. May. A first-generation linkage map of the Pacific lion-paw scallop: initial evidence of QTL for size traits and markers linked to orange shell color. Aquaculture. 350-353:200-209. (pdf)\r\n\r\nSchreier, A. D., J. Rodzen, S. Ireland, and B. May. Genetic techniques inform conservation aquaculture of the endangered Kootenai River white sturgeon, Acipenser transmontanus. Endang. Species Res. 16:65-75. (pdf)\r\n\r\nSukmanomon, S., W. Kamonrat, S. Poompuang, T. Nguyen, D. M. Bartley, B. May, U. Na-Nakorn. Genetic changes, intra- and interspecific introgression in farmed Nile tilapia in Thailand. Aquaculture 324-325:44-54. (pdf)\r\n\r\nVan Dam, A.R. and B. May. A new species of Dactylopius Costa (Dactylopius gracilipilus sp. nov.) (Hemiptera: Coccoidea: Dactylopiidae) from the Chihuahuan Desert, Texas, U.S.A. . Zootaxa 3573:33-39. (pdf)\r\n\r\n<strong>2011</strong>\r\n\r\nBaerwald, M., J. Petersen, R. Hedrick, G. Schisler, and B. May. A major effect quantitative trait locus for whirling disease resistance identified in rainbow trout (Oncorhynchus mykiss). Heredity. 106:920-926. (pdf)\r\n\r\nBaerwald, M., G. Schumer, B. Schreier, and B. May. TaqMan assays for the genetic identification of delta smelt (Hypomesus transpacificus) and wakasagi smelt (Hypomesus nipponensis). Mol. Ecol. Res. 11:784-785. (pdf)\r\n\r\nBaerwald, M.R., M. R. Stephens, K. S. Bork, M. H. Meek, K. Tomalty, and B. May. 2011. Spring-run Chinook salmon genetic management plan  San Joaquin River Restoration Program. 118 pgs. Scientific genetic management plan for the CA Dept. of Fish and Game. (pdf)\r\n\r\nFinger, A. J., E. C. Anderson, M. R. Stephens, and B. P. May. Application of a method for estimating effective population size and admixture using diagnostic SNPs: Implications for conservation of threatened Paiute cutthroat trout (Oncorhynchus clarkii seleniris) in Silver King Creek, California. Can. J. Fish. Aquat. Sci. 68:1369-1386 (pdf).\r\n\r\nFisch, K.M., J.M. Henderson, R.S. Burton, and B. May. Population genetics and conservation implications for the endangered delta smelt in the San Francisco Bay-Delta. Conservation Genetics. 12:1421-1434 (pdf).\r\n\r\nFloyd, C. H., D.H. Van Vuren, K.R. Crooks, K.L. Jones, D.K. Garcelon, N.M. Belfiore, J.W. Dragoo, and B. May. Genetic differentiation of island spotted skunks, Spilogale gracilis amphiala. J. Mam. 92(1):148-158 (pdf).\r\n\r\nMcEachern, M.B., D.H. Van Vuren, C.H. Floyd, B. May, and J.M. Eadie.Bottlenecks and rescue effects in a fluctuating population of golden-mantled ground squirrels (Spermophilus lateralis). Con. Gen. 12:285-296 (pdf).\r\n\r\nMejia , O., L. G. Herrera, B. May, R. A. Medellin,and J. J. Flores-Martinez. Effective population size dynamics of Myotis vivesi during the Pleistocene and Holocene climatic changes. Acta Chiropterologica. 13:33-40 (pdf).\r\n\r\nSchreier, A.D., D. Gille, B. Mahardja, and B. May. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon, Acipenser transmontanus. J. Appl. Ichthiol. 27:24-33 (pdf).\r\n\r\nWintzer, A.P., M.H. Meek, P.B. Moyle, and B. May. Ecological insights into the polyp stage of non-native hydrozoans in the San Francisco Estuary. Aquat. Ecol. 45:151161 (pdf).\r\n\r\n<strong>2010</strong>\r\n\r\nFloyd, C. H., J.J. Flores Martnez, , L.G. Herrera, O. Mejia, and B. May. Conserving the endangered Mexican fishing bat (Myotis vivesi): genetic variation indicates extensive gene flow among islands in the Gulf of California. Cons. Gen. 11:813-82 (pdf).\r\n\r\nIsrael, J.A. and B. May. Indirect estimates of breeding population size in the polyploid green sturgeon (Acipenser medirostris). Mol. Ecol. 19:1058-107 (pdf).\r\n\r\nIsrael, J.A. and B. May. Characterization and evaluation of polymorphic microsatellite markers in the anadromous fish Spirinchus thaleichthys. Cons. Gen. Res. online (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Nuclear and mtDNA lineage diversity in wild and cultured Pacific lion-paw scallop (Baja California Peninsula, Mexico). Mar. Biol. 157:27512767 (pdf).\r\n\r\n<strong>2009</strong>\r\n\r\nChen, Y., S. Reid, and B. May. Genetic relationships among tui chub populations in the Cow Head and surrounding basins of the Northwestern Great Basin. Cons. Gen. 10:101-114 (pdf).\r\n\r\nFinger, A.J., M.R. Stephens, N.W. Clipperton and B. May. Six diagnostic SNP markers for detecting introgression between cutthroat and rainbow trout. Mol. Ecol. Res. 9:759-763 (pdf).\r\n\r\nFisch, K., J.L. Petersen, M.R. Baerwald, and B.P. May. Characterization of 24 microsatellite loci in delta smelt, Hypomesus transpacificus, and their cross-species amplification in two other smelt species of the Osmeridae family. Mol. Ecol. Res. 9:405-408 (pdf).\r\n\r\nIsrael, J.A., K.J. Bando, E.C. Anderson, and B. May. Polyploid microsatellite data reveal stock complexity among estuarine North American green sturgeon (Acipenser medirostris). Can. J. Fish. Aqu. Sci. 66:1491-1504 (pdf).\r\n\r\nMeek, M., M. Baerwald, A. Wintzer, and B. May. Isolation and characterization of microsatellite loci in two non-native hydromedusae in the San Francisco Estuary: Maeotias marginata and Moerisia sp. Cons. Gen. Res. 1:205-208 (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Thirty-seven additional microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and cross-amplification in other Pectinids. Cons. Gen. Res. 1:101-105 (pdf).\r\n\r\nSimmons, R. E., P. Lavretsky, and B. May. Introgressive Hybridization of Redband Trout in the Upper McCloud River Watershed. Trans. Am. Fish. Soc. 139:201-213 (pdf).\r\n\r\nStephens, M.R., N.W. Clipperton, and B.P. May. Subspecies-informative TaqMan probes for evaluating introgression between native golden trout and introduced rainbow trout. Mol. Ecol. Res. 9:339-343 (pdf).\r\n\r\n<strong>Earlier publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications','','inherit','closed','open','','12-revision-v1','','','2014-02-21 21:20:39','2014-02-21 21:20:39','',12,'http://localhost/~psalessi/Genetics_Lab/12-revision-v1/',0,'revision','',0),(156,1,'2014-02-21 21:21:02','2014-02-21 21:21:02','<strong>2013</strong>\r\n\r\n<p class=\"hangingindent\">Chen, Y., S. Parmenter, and B. May. Genetic characterization of variation and management implications for the endangeredMohave tui chub. Cons. Gen. 14:11-20. (pdf) </p>\r\n\r\nMeek, M. H., A. P. Wintzer, N. Shepherd, and B. May. Genetic diversity and reproductive mode in two non-native hydromedusae, Maeotias marginata and Moerisia sp., in the upper San Francisco Estuary, California. Biol. Invasions. 15(1): 199-212. (pdf)\r\n\r\nSchreier, A.D., B. Mahardja, and B. May. Hierarchical patterns of population structure in the endangered Fraser River white sturgeon, Acipenser transmontanus, and implications for conservation. Can. J. Fish. &amp; Aquat. Sci. 69:1968-1980. (pdf)\r\n\r\n<strong>2012</strong>\r\n\r\nBaerwald, M.R., B.M. Schreier, G. Schumer, and B. May. Detection of Threatened Delta Smelt in the Gut Contents of the Invasive Mississippi Silverside in the San Francisco Estuary Using TaqMan Assays. Trans. Am. Fish. Soc. 141:1600-1607. (pdf)\r\n\r\nMahardja, B., B. May, and M. Baerwald. Characterization of 36 additional microsatellite loci in splittail (Pogonichthys macrolepidotus) and cross-amplification in five other native Californian cyprinid species. Cons. Gen. Res. 4(4):917-921. (pdf)\r\n\r\nMeek, M., A. Wintzer, W. Wetzel, and B. May. Climate change likely to facilitate the invasion of the non-native hydroid, Cordylophora caspia, in the San Francisco Estuary, CA. PLos ONE. 7(10): e46373. doi:10.1371/journal.pone.0046373 (pdf)\r\n\r\nPetersen, J.L., M.R. Baerwald, J.M. Ibarra, and B. May. A first-generation linkage map of the Pacific lion-paw scallop: initial evidence of QTL for size traits and markers linked to orange shell color. Aquaculture. 350-353:200-209. (pdf)\r\n\r\nSchreier, A. D., J. Rodzen, S. Ireland, and B. May. Genetic techniques inform conservation aquaculture of the endangered Kootenai River white sturgeon, Acipenser transmontanus. Endang. Species Res. 16:65-75. (pdf)\r\n\r\nSukmanomon, S., W. Kamonrat, S. Poompuang, T. Nguyen, D. M. Bartley, B. May, U. Na-Nakorn. Genetic changes, intra- and interspecific introgression in farmed Nile tilapia in Thailand. Aquaculture 324-325:44-54. (pdf)\r\n\r\nVan Dam, A.R. and B. May. A new species of Dactylopius Costa (Dactylopius gracilipilus sp. nov.) (Hemiptera: Coccoidea: Dactylopiidae) from the Chihuahuan Desert, Texas, U.S.A. . Zootaxa 3573:33-39. (pdf)\r\n\r\n<strong>2011</strong>\r\n\r\nBaerwald, M., J. Petersen, R. Hedrick, G. Schisler, and B. May. A major effect quantitative trait locus for whirling disease resistance identified in rainbow trout (Oncorhynchus mykiss). Heredity. 106:920-926. (pdf)\r\n\r\nBaerwald, M., G. Schumer, B. Schreier, and B. May. TaqMan assays for the genetic identification of delta smelt (Hypomesus transpacificus) and wakasagi smelt (Hypomesus nipponensis). Mol. Ecol. Res. 11:784-785. (pdf)\r\n\r\nBaerwald, M.R., M. R. Stephens, K. S. Bork, M. H. Meek, K. Tomalty, and B. May. 2011. Spring-run Chinook salmon genetic management plan  San Joaquin River Restoration Program. 118 pgs. Scientific genetic management plan for the CA Dept. of Fish and Game. (pdf)\r\n\r\nFinger, A. J., E. C. Anderson, M. R. Stephens, and B. P. May. Application of a method for estimating effective population size and admixture using diagnostic SNPs: Implications for conservation of threatened Paiute cutthroat trout (Oncorhynchus clarkii seleniris) in Silver King Creek, California. Can. J. Fish. Aquat. Sci. 68:1369-1386 (pdf).\r\n\r\nFisch, K.M., J.M. Henderson, R.S. Burton, and B. May. Population genetics and conservation implications for the endangered delta smelt in the San Francisco Bay-Delta. Conservation Genetics. 12:1421-1434 (pdf).\r\n\r\nFloyd, C. H., D.H. Van Vuren, K.R. Crooks, K.L. Jones, D.K. Garcelon, N.M. Belfiore, J.W. Dragoo, and B. May. Genetic differentiation of island spotted skunks, Spilogale gracilis amphiala. J. Mam. 92(1):148-158 (pdf).\r\n\r\nMcEachern, M.B., D.H. Van Vuren, C.H. Floyd, B. May, and J.M. Eadie.Bottlenecks and rescue effects in a fluctuating population of golden-mantled ground squirrels (Spermophilus lateralis). Con. Gen. 12:285-296 (pdf).\r\n\r\nMejia , O., L. G. Herrera, B. May, R. A. Medellin,and J. J. Flores-Martinez. Effective population size dynamics of Myotis vivesi during the Pleistocene and Holocene climatic changes. Acta Chiropterologica. 13:33-40 (pdf).\r\n\r\nSchreier, A.D., D. Gille, B. Mahardja, and B. May. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon, Acipenser transmontanus. J. Appl. Ichthiol. 27:24-33 (pdf).\r\n\r\nWintzer, A.P., M.H. Meek, P.B. Moyle, and B. May. Ecological insights into the polyp stage of non-native hydrozoans in the San Francisco Estuary. Aquat. Ecol. 45:151161 (pdf).\r\n\r\n<strong>2010</strong>\r\n\r\nFloyd, C. H., J.J. Flores Martnez, , L.G. Herrera, O. Mejia, and B. May. Conserving the endangered Mexican fishing bat (Myotis vivesi): genetic variation indicates extensive gene flow among islands in the Gulf of California. Cons. Gen. 11:813-82 (pdf).\r\n\r\nIsrael, J.A. and B. May. Indirect estimates of breeding population size in the polyploid green sturgeon (Acipenser medirostris). Mol. Ecol. 19:1058-107 (pdf).\r\n\r\nIsrael, J.A. and B. May. Characterization and evaluation of polymorphic microsatellite markers in the anadromous fish Spirinchus thaleichthys. Cons. Gen. Res. online (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Nuclear and mtDNA lineage diversity in wild and cultured Pacific lion-paw scallop (Baja California Peninsula, Mexico). Mar. Biol. 157:27512767 (pdf).\r\n\r\n<strong>2009</strong>\r\n\r\nChen, Y., S. Reid, and B. May. Genetic relationships among tui chub populations in the Cow Head and surrounding basins of the Northwestern Great Basin. Cons. Gen. 10:101-114 (pdf).\r\n\r\nFinger, A.J., M.R. Stephens, N.W. Clipperton and B. May. Six diagnostic SNP markers for detecting introgression between cutthroat and rainbow trout. Mol. Ecol. Res. 9:759-763 (pdf).\r\n\r\nFisch, K., J.L. Petersen, M.R. Baerwald, and B.P. May. Characterization of 24 microsatellite loci in delta smelt, Hypomesus transpacificus, and their cross-species amplification in two other smelt species of the Osmeridae family. Mol. Ecol. Res. 9:405-408 (pdf).\r\n\r\nIsrael, J.A., K.J. Bando, E.C. Anderson, and B. May. Polyploid microsatellite data reveal stock complexity among estuarine North American green sturgeon (Acipenser medirostris). Can. J. Fish. Aqu. Sci. 66:1491-1504 (pdf).\r\n\r\nMeek, M., M. Baerwald, A. Wintzer, and B. May. Isolation and characterization of microsatellite loci in two non-native hydromedusae in the San Francisco Estuary: Maeotias marginata and Moerisia sp. Cons. Gen. Res. 1:205-208 (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Thirty-seven additional microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and cross-amplification in other Pectinids. Cons. Gen. Res. 1:101-105 (pdf).\r\n\r\nSimmons, R. E., P. Lavretsky, and B. May. Introgressive Hybridization of Redband Trout in the Upper McCloud River Watershed. Trans. Am. Fish. Soc. 139:201-213 (pdf).\r\n\r\nStephens, M.R., N.W. Clipperton, and B.P. May. Subspecies-informative TaqMan probes for evaluating introgression between native golden trout and introduced rainbow trout. Mol. Ecol. Res. 9:339-343 (pdf).\r\n\r\n<strong>Earlier publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications','','inherit','closed','open','','12-revision-v1','','','2014-02-21 21:21:02','2014-02-21 21:21:02','',12,'http://localhost/~psalessi/Genetics_Lab/12-revision-v1/',0,'revision','',0),(157,1,'2014-02-21 21:21:22','2014-02-21 21:21:22','<strong>2013</strong>\r\n\r\n<p class=\"hangingindent\">Chen, Y., S. Parmenter, and B. May. Genetic characterization of variation and management implications for the endangeredMohave tui chub. Cons. Gen. 14:11-20. (pdf) </p>\r\n\r\n<p class=\"hangingindent\">Meek, M. H., A. P. Wintzer, N. Shepherd, and B. May. Genetic diversity and reproductive mode in two non-native hydromedusae, Maeotias marginata and Moerisia sp., in the upper San Francisco Estuary, California. Biol. Invasions. 15(1): 199-212. (pdf)\r\n\r\n<p class=\"hangingindent\">Schreier, A.D., B. Mahardja, and B. May. Hierarchical patterns of population structure in the endangered Fraser River white sturgeon, Acipenser transmontanus, and implications for conservation. Can. J. Fish. &amp; Aquat. Sci. 69:1968-1980. (pdf)\r\n\r\n<strong>2012</strong>\r\n\r\nBaerwald, M.R., B.M. Schreier, G. Schumer, and B. May. Detection of Threatened Delta Smelt in the Gut Contents of the Invasive Mississippi Silverside in the San Francisco Estuary Using TaqMan Assays. Trans. Am. Fish. Soc. 141:1600-1607. (pdf)\r\n\r\nMahardja, B., B. May, and M. Baerwald. Characterization of 36 additional microsatellite loci in splittail (Pogonichthys macrolepidotus) and cross-amplification in five other native Californian cyprinid species. Cons. Gen. Res. 4(4):917-921. (pdf)\r\n\r\nMeek, M., A. Wintzer, W. Wetzel, and B. May. Climate change likely to facilitate the invasion of the non-native hydroid, Cordylophora caspia, in the San Francisco Estuary, CA. PLos ONE. 7(10): e46373. doi:10.1371/journal.pone.0046373 (pdf)\r\n\r\nPetersen, J.L., M.R. Baerwald, J.M. Ibarra, and B. May. A first-generation linkage map of the Pacific lion-paw scallop: initial evidence of QTL for size traits and markers linked to orange shell color. Aquaculture. 350-353:200-209. (pdf)\r\n\r\nSchreier, A. D., J. Rodzen, S. Ireland, and B. May. Genetic techniques inform conservation aquaculture of the endangered Kootenai River white sturgeon, Acipenser transmontanus. Endang. Species Res. 16:65-75. (pdf)\r\n\r\nSukmanomon, S., W. Kamonrat, S. Poompuang, T. Nguyen, D. M. Bartley, B. May, U. Na-Nakorn. Genetic changes, intra- and interspecific introgression in farmed Nile tilapia in Thailand. Aquaculture 324-325:44-54. (pdf)\r\n\r\nVan Dam, A.R. and B. May. A new species of Dactylopius Costa (Dactylopius gracilipilus sp. nov.) (Hemiptera: Coccoidea: Dactylopiidae) from the Chihuahuan Desert, Texas, U.S.A. . Zootaxa 3573:33-39. (pdf)\r\n\r\n<strong>2011</strong>\r\n\r\nBaerwald, M., J. Petersen, R. Hedrick, G. Schisler, and B. May. A major effect quantitative trait locus for whirling disease resistance identified in rainbow trout (Oncorhynchus mykiss). Heredity. 106:920-926. (pdf)\r\n\r\nBaerwald, M., G. Schumer, B. Schreier, and B. May. TaqMan assays for the genetic identification of delta smelt (Hypomesus transpacificus) and wakasagi smelt (Hypomesus nipponensis). Mol. Ecol. Res. 11:784-785. (pdf)\r\n\r\nBaerwald, M.R., M. R. Stephens, K. S. Bork, M. H. Meek, K. Tomalty, and B. May. 2011. Spring-run Chinook salmon genetic management plan  San Joaquin River Restoration Program. 118 pgs. Scientific genetic management plan for the CA Dept. of Fish and Game. (pdf)\r\n\r\nFinger, A. J., E. C. Anderson, M. R. Stephens, and B. P. May. Application of a method for estimating effective population size and admixture using diagnostic SNPs: Implications for conservation of threatened Paiute cutthroat trout (Oncorhynchus clarkii seleniris) in Silver King Creek, California. Can. J. Fish. Aquat. Sci. 68:1369-1386 (pdf).\r\n\r\nFisch, K.M., J.M. Henderson, R.S. Burton, and B. May. Population genetics and conservation implications for the endangered delta smelt in the San Francisco Bay-Delta. Conservation Genetics. 12:1421-1434 (pdf).\r\n\r\nFloyd, C. H., D.H. Van Vuren, K.R. Crooks, K.L. Jones, D.K. Garcelon, N.M. Belfiore, J.W. Dragoo, and B. May. Genetic differentiation of island spotted skunks, Spilogale gracilis amphiala. J. Mam. 92(1):148-158 (pdf).\r\n\r\nMcEachern, M.B., D.H. Van Vuren, C.H. Floyd, B. May, and J.M. Eadie.Bottlenecks and rescue effects in a fluctuating population of golden-mantled ground squirrels (Spermophilus lateralis). Con. Gen. 12:285-296 (pdf).\r\n\r\nMejia , O., L. G. Herrera, B. May, R. A. Medellin,and J. J. Flores-Martinez. Effective population size dynamics of Myotis vivesi during the Pleistocene and Holocene climatic changes. Acta Chiropterologica. 13:33-40 (pdf).\r\n\r\nSchreier, A.D., D. Gille, B. Mahardja, and B. May. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon, Acipenser transmontanus. J. Appl. Ichthiol. 27:24-33 (pdf).\r\n\r\nWintzer, A.P., M.H. Meek, P.B. Moyle, and B. May. Ecological insights into the polyp stage of non-native hydrozoans in the San Francisco Estuary. Aquat. Ecol. 45:151161 (pdf).\r\n\r\n<strong>2010</strong>\r\n\r\nFloyd, C. H., J.J. Flores Martnez, , L.G. Herrera, O. Mejia, and B. May. Conserving the endangered Mexican fishing bat (Myotis vivesi): genetic variation indicates extensive gene flow among islands in the Gulf of California. Cons. Gen. 11:813-82 (pdf).\r\n\r\nIsrael, J.A. and B. May. Indirect estimates of breeding population size in the polyploid green sturgeon (Acipenser medirostris). Mol. Ecol. 19:1058-107 (pdf).\r\n\r\nIsrael, J.A. and B. May. Characterization and evaluation of polymorphic microsatellite markers in the anadromous fish Spirinchus thaleichthys. Cons. Gen. Res. online (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Nuclear and mtDNA lineage diversity in wild and cultured Pacific lion-paw scallop (Baja California Peninsula, Mexico). Mar. Biol. 157:27512767 (pdf).\r\n\r\n<strong>2009</strong>\r\n\r\nChen, Y., S. Reid, and B. May. Genetic relationships among tui chub populations in the Cow Head and surrounding basins of the Northwestern Great Basin. Cons. Gen. 10:101-114 (pdf).\r\n\r\nFinger, A.J., M.R. Stephens, N.W. Clipperton and B. May. Six diagnostic SNP markers for detecting introgression between cutthroat and rainbow trout. Mol. Ecol. Res. 9:759-763 (pdf).\r\n\r\nFisch, K., J.L. Petersen, M.R. Baerwald, and B.P. May. Characterization of 24 microsatellite loci in delta smelt, Hypomesus transpacificus, and their cross-species amplification in two other smelt species of the Osmeridae family. Mol. Ecol. Res. 9:405-408 (pdf).\r\n\r\nIsrael, J.A., K.J. Bando, E.C. Anderson, and B. May. Polyploid microsatellite data reveal stock complexity among estuarine North American green sturgeon (Acipenser medirostris). Can. J. Fish. Aqu. Sci. 66:1491-1504 (pdf).\r\n\r\nMeek, M., M. Baerwald, A. Wintzer, and B. May. Isolation and characterization of microsatellite loci in two non-native hydromedusae in the San Francisco Estuary: Maeotias marginata and Moerisia sp. Cons. Gen. Res. 1:205-208 (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Thirty-seven additional microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and cross-amplification in other Pectinids. Cons. Gen. Res. 1:101-105 (pdf).\r\n\r\nSimmons, R. E., P. Lavretsky, and B. May. Introgressive Hybridization of Redband Trout in the Upper McCloud River Watershed. Trans. Am. Fish. Soc. 139:201-213 (pdf).\r\n\r\nStephens, M.R., N.W. Clipperton, and B.P. May. Subspecies-informative TaqMan probes for evaluating introgression between native golden trout and introduced rainbow trout. Mol. Ecol. Res. 9:339-343 (pdf).\r\n\r\n<strong>Earlier publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications','','inherit','closed','open','','12-revision-v1','','','2014-02-21 21:21:22','2014-02-21 21:21:22','',12,'http://localhost/~psalessi/Genetics_Lab/12-revision-v1/',0,'revision','',0),(158,1,'2014-02-21 21:22:27','2014-02-21 21:22:27','<strong>2013</strong>\r\n\r\n<div class=\"hangingindent\">\r\nChen, Y., S. Parmenter, and B. May. Genetic characterization of variation and management implications for the endangeredMohave tui chub. Cons. Gen. 14:11-20. (pdf)\r\n\r\nMeek, M. H., A. P. Wintzer, N. Shepherd, and B. May. Genetic diversity and reproductive mode in two non-native hydromedusae, Maeotias marginata and Moerisia sp., in the upper San Francisco Estuary, California. Biol. Invasions. 15(1): 199-212. (pdf)\r\n\r\nSchreier, A.D., B. Mahardja, and B. May. Hierarchical patterns of population structure in the endangered Fraser River white sturgeon, Acipenser transmontanus, and implications for conservation. Can. J. Fish. &amp; Aquat. Sci. 69:1968-1980. (pdf)\r\n\r\n<strong>2012</strong>\r\n\r\nBaerwald, M.R., B.M. Schreier, G. Schumer, and B. May. Detection of Threatened Delta Smelt in the Gut Contents of the Invasive Mississippi Silverside in the San Francisco Estuary Using TaqMan Assays. Trans. Am. Fish. Soc. 141:1600-1607. (pdf)\r\n\r\nMahardja, B., B. May, and M. Baerwald. Characterization of 36 additional microsatellite loci in splittail (Pogonichthys macrolepidotus) and cross-amplification in five other native Californian cyprinid species. Cons. Gen. Res. 4(4):917-921. (pdf)\r\n\r\nMeek, M., A. Wintzer, W. Wetzel, and B. May. Climate change likely to facilitate the invasion of the non-native hydroid, Cordylophora caspia, in the San Francisco Estuary, CA. PLos ONE. 7(10): e46373. doi:10.1371/journal.pone.0046373 (pdf)\r\n\r\nPetersen, J.L., M.R. Baerwald, J.M. Ibarra, and B. May. A first-generation linkage map of the Pacific lion-paw scallop: initial evidence of QTL for size traits and markers linked to orange shell color. Aquaculture. 350-353:200-209. (pdf)\r\n\r\nSchreier, A. D., J. Rodzen, S. Ireland, and B. May. Genetic techniques inform conservation aquaculture of the endangered Kootenai River white sturgeon, Acipenser transmontanus. Endang. Species Res. 16:65-75. (pdf)\r\n\r\nSukmanomon, S., W. Kamonrat, S. Poompuang, T. Nguyen, D. M. Bartley, B. May, U. Na-Nakorn. Genetic changes, intra- and interspecific introgression in farmed Nile tilapia in Thailand. Aquaculture 324-325:44-54. (pdf)\r\n\r\nVan Dam, A.R. and B. May. A new species of Dactylopius Costa (Dactylopius gracilipilus sp. nov.) (Hemiptera: Coccoidea: Dactylopiidae) from the Chihuahuan Desert, Texas, U.S.A. . Zootaxa 3573:33-39. (pdf)\r\n\r\n<strong>2011</strong>\r\n\r\nBaerwald, M., J. Petersen, R. Hedrick, G. Schisler, and B. May. A major effect quantitative trait locus for whirling disease resistance identified in rainbow trout (Oncorhynchus mykiss). Heredity. 106:920-926. (pdf)\r\n\r\nBaerwald, M., G. Schumer, B. Schreier, and B. May. TaqMan assays for the genetic identification of delta smelt (Hypomesus transpacificus) and wakasagi smelt (Hypomesus nipponensis). Mol. Ecol. Res. 11:784-785. (pdf)\r\n\r\nBaerwald, M.R., M. R. Stephens, K. S. Bork, M. H. Meek, K. Tomalty, and B. May. 2011. Spring-run Chinook salmon genetic management plan  San Joaquin River Restoration Program. 118 pgs. Scientific genetic management plan for the CA Dept. of Fish and Game. (pdf)\r\n\r\nFinger, A. J., E. C. Anderson, M. R. Stephens, and B. P. May. Application of a method for estimating effective population size and admixture using diagnostic SNPs: Implications for conservation of threatened Paiute cutthroat trout (Oncorhynchus clarkii seleniris) in Silver King Creek, California. Can. J. Fish. Aquat. Sci. 68:1369-1386 (pdf).\r\n\r\nFisch, K.M., J.M. Henderson, R.S. Burton, and B. May. Population genetics and conservation implications for the endangered delta smelt in the San Francisco Bay-Delta. Conservation Genetics. 12:1421-1434 (pdf).\r\n\r\nFloyd, C. H., D.H. Van Vuren, K.R. Crooks, K.L. Jones, D.K. Garcelon, N.M. Belfiore, J.W. Dragoo, and B. May. Genetic differentiation of island spotted skunks, Spilogale gracilis amphiala. J. Mam. 92(1):148-158 (pdf).\r\n\r\nMcEachern, M.B., D.H. Van Vuren, C.H. Floyd, B. May, and J.M. Eadie.Bottlenecks and rescue effects in a fluctuating population of golden-mantled ground squirrels (Spermophilus lateralis). Con. Gen. 12:285-296 (pdf).\r\n\r\nMejia , O., L. G. Herrera, B. May, R. A. Medellin,and J. J. Flores-Martinez. Effective population size dynamics of Myotis vivesi during the Pleistocene and Holocene climatic changes. Acta Chiropterologica. 13:33-40 (pdf).\r\n\r\nSchreier, A.D., D. Gille, B. Mahardja, and B. May. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon, Acipenser transmontanus. J. Appl. Ichthiol. 27:24-33 (pdf).\r\n\r\nWintzer, A.P., M.H. Meek, P.B. Moyle, and B. May. Ecological insights into the polyp stage of non-native hydrozoans in the San Francisco Estuary. Aquat. Ecol. 45:151161 (pdf).\r\n\r\n<strong>2010</strong>\r\n\r\nFloyd, C. H., J.J. Flores Martnez, , L.G. Herrera, O. Mejia, and B. May. Conserving the endangered Mexican fishing bat (Myotis vivesi): genetic variation indicates extensive gene flow among islands in the Gulf of California. Cons. Gen. 11:813-82 (pdf).\r\n\r\nIsrael, J.A. and B. May. Indirect estimates of breeding population size in the polyploid green sturgeon (Acipenser medirostris). Mol. Ecol. 19:1058-107 (pdf).\r\n\r\nIsrael, J.A. and B. May. Characterization and evaluation of polymorphic microsatellite markers in the anadromous fish Spirinchus thaleichthys. Cons. Gen. Res. online (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Nuclear and mtDNA lineage diversity in wild and cultured Pacific lion-paw scallop (Baja California Peninsula, Mexico). Mar. Biol. 157:27512767 (pdf).\r\n\r\n<strong>2009</strong>\r\n\r\nChen, Y., S. Reid, and B. May. Genetic relationships among tui chub populations in the Cow Head and surrounding basins of the Northwestern Great Basin. Cons. Gen. 10:101-114 (pdf).\r\n\r\nFinger, A.J., M.R. Stephens, N.W. Clipperton and B. May. Six diagnostic SNP markers for detecting introgression between cutthroat and rainbow trout. Mol. Ecol. Res. 9:759-763 (pdf).\r\n\r\nFisch, K., J.L. Petersen, M.R. Baerwald, and B.P. May. Characterization of 24 microsatellite loci in delta smelt, Hypomesus transpacificus, and their cross-species amplification in two other smelt species of the Osmeridae family. Mol. Ecol. Res. 9:405-408 (pdf).\r\n\r\nIsrael, J.A., K.J. Bando, E.C. Anderson, and B. May. Polyploid microsatellite data reveal stock complexity among estuarine North American green sturgeon (Acipenser medirostris). Can. J. Fish. Aqu. Sci. 66:1491-1504 (pdf).\r\n\r\nMeek, M., M. Baerwald, A. Wintzer, and B. May. Isolation and characterization of microsatellite loci in two non-native hydromedusae in the San Francisco Estuary: Maeotias marginata and Moerisia sp. Cons. Gen. Res. 1:205-208 (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Thirty-seven additional microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and cross-amplification in other Pectinids. Cons. Gen. Res. 1:101-105 (pdf).\r\n\r\nSimmons, R. E., P. Lavretsky, and B. May. Introgressive Hybridization of Redband Trout in the Upper McCloud River Watershed. Trans. Am. Fish. Soc. 139:201-213 (pdf).\r\n\r\nStephens, M.R., N.W. Clipperton, and B.P. May. Subspecies-informative TaqMan probes for evaluating introgression between native golden trout and introduced rainbow trout. Mol. Ecol. Res. 9:339-343 (pdf).\r\n\r\n</div>\r\n\r\n<strong>Earlier publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications','','inherit','closed','open','','12-revision-v1','','','2014-02-21 21:22:27','2014-02-21 21:22:27','',12,'http://localhost/~psalessi/Genetics_Lab/12-revision-v1/',0,'revision','',0),(159,1,'2014-02-21 21:26:57','2014-02-21 21:26:57','<strong>2013</strong>\r\n<div class=\"hanging_indent\">\r\n\r\nChen, Y., S. Parmenter, and B. May. Genetic characterization of variation and management implications for the endangeredMohave tui chub. Cons. Gen. 14:11-20. (pdf)\r\n\r\nMeek, M. H., A. P. Wintzer, N. Shepherd, and B. May. Genetic diversity and reproductive mode in two non-native hydromedusae, Maeotias marginata and Moerisia sp., in the upper San Francisco Estuary, California. Biol. Invasions. 15(1): 199-212. (pdf)\r\n\r\nSchreier, A.D., B. Mahardja, and B. May. Hierarchical patterns of population structure in the endangered Fraser River white sturgeon, Acipenser transmontanus, and implications for conservation. Can. J. Fish. &amp; Aquat. Sci. 69:1968-1980. (pdf)\r\n\r\n<strong>2012</strong>\r\n\r\nBaerwald, M.R., B.M. Schreier, G. Schumer, and B. May. Detection of Threatened Delta Smelt in the Gut Contents of the Invasive Mississippi Silverside in the San Francisco Estuary Using TaqMan Assays. Trans. Am. Fish. Soc. 141:1600-1607. (pdf)\r\n\r\nMahardja, B., B. May, and M. Baerwald. Characterization of 36 additional microsatellite loci in splittail (Pogonichthys macrolepidotus) and cross-amplification in five other native Californian cyprinid species. Cons. Gen. Res. 4(4):917-921. (pdf)\r\n\r\nMeek, M., A. Wintzer, W. Wetzel, and B. May. Climate change likely to facilitate the invasion of the non-native hydroid, Cordylophora caspia, in the San Francisco Estuary, CA. PLos ONE. 7(10): e46373. doi:10.1371/journal.pone.0046373 (pdf)\r\n\r\nPetersen, J.L., M.R. Baerwald, J.M. Ibarra, and B. May. A first-generation linkage map of the Pacific lion-paw scallop: initial evidence of QTL for size traits and markers linked to orange shell color. Aquaculture. 350-353:200-209. (pdf)\r\n\r\nSchreier, A. D., J. Rodzen, S. Ireland, and B. May. Genetic techniques inform conservation aquaculture of the endangered Kootenai River white sturgeon, Acipenser transmontanus. Endang. Species Res. 16:65-75. (pdf)\r\n\r\nSukmanomon, S., W. Kamonrat, S. Poompuang, T. Nguyen, D. M. Bartley, B. May, U. Na-Nakorn. Genetic changes, intra- and interspecific introgression in farmed Nile tilapia in Thailand. Aquaculture 324-325:44-54. (pdf)\r\n\r\nVan Dam, A.R. and B. May. A new species of Dactylopius Costa (Dactylopius gracilipilus sp. nov.) (Hemiptera: Coccoidea: Dactylopiidae) from the Chihuahuan Desert, Texas, U.S.A. . Zootaxa 3573:33-39. (pdf)\r\n\r\n<strong>2011</strong>\r\n\r\nBaerwald, M., J. Petersen, R. Hedrick, G. Schisler, and B. May. A major effect quantitative trait locus for whirling disease resistance identified in rainbow trout (Oncorhynchus mykiss). Heredity. 106:920-926. (pdf)\r\n\r\nBaerwald, M., G. Schumer, B. Schreier, and B. May. TaqMan assays for the genetic identification of delta smelt (Hypomesus transpacificus) and wakasagi smelt (Hypomesus nipponensis). Mol. Ecol. Res. 11:784-785. (pdf)\r\n\r\nBaerwald, M.R., M. R. Stephens, K. S. Bork, M. H. Meek, K. Tomalty, and B. May. 2011. Spring-run Chinook salmon genetic management plan  San Joaquin River Restoration Program. 118 pgs. Scientific genetic management plan for the CA Dept. of Fish and Game. (pdf)\r\n\r\nFinger, A. J., E. C. Anderson, M. R. Stephens, and B. P. May. Application of a method for estimating effective population size and admixture using diagnostic SNPs: Implications for conservation of threatened Paiute cutthroat trout (Oncorhynchus clarkii seleniris) in Silver King Creek, California. Can. J. Fish. Aquat. Sci. 68:1369-1386 (pdf).\r\n\r\nFisch, K.M., J.M. Henderson, R.S. Burton, and B. May. Population genetics and conservation implications for the endangered delta smelt in the San Francisco Bay-Delta. Conservation Genetics. 12:1421-1434 (pdf).\r\n\r\nFloyd, C. H., D.H. Van Vuren, K.R. Crooks, K.L. Jones, D.K. Garcelon, N.M. Belfiore, J.W. Dragoo, and B. May. Genetic differentiation of island spotted skunks, Spilogale gracilis amphiala. J. Mam. 92(1):148-158 (pdf).\r\n\r\nMcEachern, M.B., D.H. Van Vuren, C.H. Floyd, B. May, and J.M. Eadie.Bottlenecks and rescue effects in a fluctuating population of golden-mantled ground squirrels (Spermophilus lateralis). Con. Gen. 12:285-296 (pdf).\r\n\r\nMejia , O., L. G. Herrera, B. May, R. A. Medellin,and J. J. Flores-Martinez. Effective population size dynamics of Myotis vivesi during the Pleistocene and Holocene climatic changes. Acta Chiropterologica. 13:33-40 (pdf).\r\n\r\nSchreier, A.D., D. Gille, B. Mahardja, and B. May. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon, Acipenser transmontanus. J. Appl. Ichthiol. 27:24-33 (pdf).\r\n\r\nWintzer, A.P., M.H. Meek, P.B. Moyle, and B. May. Ecological insights into the polyp stage of non-native hydrozoans in the San Francisco Estuary. Aquat. Ecol. 45:151161 (pdf).\r\n\r\n<strong>2010</strong>\r\n\r\nFloyd, C. H., J.J. Flores Martnez, , L.G. Herrera, O. Mejia, and B. May. Conserving the endangered Mexican fishing bat (Myotis vivesi): genetic variation indicates extensive gene flow among islands in the Gulf of California. Cons. Gen. 11:813-82 (pdf).\r\n\r\nIsrael, J.A. and B. May. Indirect estimates of breeding population size in the polyploid green sturgeon (Acipenser medirostris). Mol. Ecol. 19:1058-107 (pdf).\r\n\r\nIsrael, J.A. and B. May. Characterization and evaluation of polymorphic microsatellite markers in the anadromous fish Spirinchus thaleichthys. Cons. Gen. Res. online (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Nuclear and mtDNA lineage diversity in wild and cultured Pacific lion-paw scallop (Baja California Peninsula, Mexico). Mar. Biol. 157:27512767 (pdf).\r\n\r\n<strong>2009</strong>\r\n\r\nChen, Y., S. Reid, and B. May. Genetic relationships among tui chub populations in the Cow Head and surrounding basins of the Northwestern Great Basin. Cons. Gen. 10:101-114 (pdf).\r\n\r\nFinger, A.J., M.R. Stephens, N.W. Clipperton and B. May. Six diagnostic SNP markers for detecting introgression between cutthroat and rainbow trout. Mol. Ecol. Res. 9:759-763 (pdf).\r\n\r\nFisch, K., J.L. Petersen, M.R. Baerwald, and B.P. May. Characterization of 24 microsatellite loci in delta smelt, Hypomesus transpacificus, and their cross-species amplification in two other smelt species of the Osmeridae family. Mol. Ecol. Res. 9:405-408 (pdf).\r\n\r\nIsrael, J.A., K.J. Bando, E.C. Anderson, and B. May. Polyploid microsatellite data reveal stock complexity among estuarine North American green sturgeon (Acipenser medirostris). Can. J. Fish. Aqu. Sci. 66:1491-1504 (pdf).\r\n\r\nMeek, M., M. Baerwald, A. Wintzer, and B. May. Isolation and characterization of microsatellite loci in two non-native hydromedusae in the San Francisco Estuary: Maeotias marginata and Moerisia sp. Cons. Gen. Res. 1:205-208 (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Thirty-seven additional microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and cross-amplification in other Pectinids. Cons. Gen. Res. 1:101-105 (pdf).\r\n\r\nSimmons, R. E., P. Lavretsky, and B. May. Introgressive Hybridization of Redband Trout in the Upper McCloud River Watershed. Trans. Am. Fish. Soc. 139:201-213 (pdf).\r\n\r\nStephens, M.R., N.W. Clipperton, and B.P. May. Subspecies-informative TaqMan probes for evaluating introgression between native golden trout and introduced rainbow trout. Mol. Ecol. Res. 9:339-343 (pdf).\r\n\r\n</div>\r\n<strong>Earlier publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications','','inherit','closed','open','','12-revision-v1','','','2014-02-21 21:26:57','2014-02-21 21:26:57','',12,'http://localhost/~psalessi/Genetics_Lab/12-revision-v1/',0,'revision','',0),(160,1,'2014-02-21 21:36:16','2014-02-21 21:36:16','','Partners Page Featured Image','','inherit','closed','open','','partners_header','','','2014-02-21 21:36:16','2014-02-21 21:36:16','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/partners_header.png',0,'attachment','image/png',0),(161,1,'2014-02-28 21:22:31','2014-02-28 21:22:31','<div class=\"partner_images\">\n<h2 style=\"text-align: center;\">Funders and Associates of the Genomic Variation Laboratory</h2>\n<p style=\"text-align: center;\"><a title=\"US Fish &amp; Wildlife Service\" href=\"http://www.fws.gov/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-193\" style=\"font-size: 1rem; line-height: 1;\" alt=\"U_S__Fish__and__Wildlife_Service\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/U_S__Fish__and__Wildlife_Service.jpg\" width=\"103\" height=\"124\" /></a><a title=\"California Department of Water Resources\" href=\"http://www.water.ca.gov/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-186\" style=\"font-size: 1rem; line-height: 1;\" alt=\"dwr-logobig_home\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/dwr-logobig_home.jpg\" width=\"125\" height=\"125\" /></a><a title=\"US Forest Service\" href=\"http://www.fs.fed.us/\" target=\"_blank\"><img class=\"alignnone size-medium wp-image-194\" style=\"font-size: 1rem; line-height: 1;\" alt=\"USForestServiceLogo\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/USForestServiceLogo.jpg\" width=\"108\" height=\"121\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" title=\"US Department of the Interior, Bureau of Land Management\" href=\"http://www.blm.gov/wo/st/en.html\" target=\"_blank\"><img class=\"alignnone size-full wp-image-171\" alt=\"BLMLogo\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/BLMLogo.jpg\" width=\"123\" height=\"123\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" title=\"California Department of Fish &amp; Game\" href=\"http://www.dfg.ca.gov/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-172\" alt=\"DFGshield\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/DFGshield.jpg\" width=\"102\" height=\"135\" /></a><span style=\"line-height: 1.714285714; font-size: 1rem;\"></span><a title=\"Cramer Fish Sciences\" href=\"http://www.fishsciences.net/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-184\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.jpg\" width=\"233\" height=\"46\" /></a><a title=\"Kern Community Foundation\" href=\"http://www.kernfoundation.org/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-195\" alt=\"logo\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/logo.jpg\" width=\"165\" height=\"57\" /></a><a title=\"Kootenai Tribe of Idaho\" href=\"http://www.kootenai.org/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-187\" style=\"font-size: 1rem; line-height: 1;\" alt=\"shapeimage_4\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/shapeimage_4.png\" width=\"167\" height=\"112\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_1.jpg\"><img class=\"alignnone size-full wp-image-173\" alt=\"droppedImage_1\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_1.jpg\" width=\"121\" height=\"121\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_2.jpg\"><img class=\"alignnone size-full wp-image-174\" alt=\"droppedImage_2\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_2.jpg\" width=\"188\" height=\"54\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_3.jpg\"><img class=\"alignnone size-full wp-image-175\" alt=\"droppedImage_3\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_3.jpg\" width=\"93\" height=\"93\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_4.jpg\"><img class=\"alignnone size-full wp-image-176\" alt=\"droppedImage_4\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_4.jpg\" width=\"124\" height=\"72\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_5.jpg\"><img class=\"alignnone size-full wp-image-177\" alt=\"droppedImage_5\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_5.jpg\" width=\"213\" height=\"68\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_6.jpg\"><img class=\"alignnone size-full wp-image-178\" alt=\"droppedImage_6\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_6.jpg\" width=\"86\" height=\"113\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_7.jpg\"><img class=\"alignnone size-full wp-image-179\" alt=\"droppedImage_7\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_7.jpg\" width=\"149\" height=\"102\" /></a><img class=\"alignnone size-full wp-image-181\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage_9\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_9.jpg\" width=\"119\" height=\"119\" /><img class=\"alignnone size-full wp-image-185\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.png\" width=\"97\" height=\"140\" /><img class=\"alignnone size-full wp-image-180\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage_8\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_8.jpg\" width=\"244\" height=\"70\" /><img class=\"alignnone size-full wp-image-182\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage_10\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_10.jpg\" width=\"157\" height=\"123\" /><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_11.jpg\"><img class=\"alignnone size-full wp-image-183\" alt=\"droppedImage_11\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_11.jpg\" width=\"192\" height=\"71\" /></a></p>\n\n</div>','Partners','','inherit','closed','open','','10-autosave-v1','','','2014-02-28 21:22:31','2014-02-28 21:22:31','',10,'http://localhost/~psalessi/Genetics_Lab/10-autosave-v1/',0,'revision','',0),(167,1,'2014-02-21 21:39:06','2014-02-21 21:39:06','<h2>Funders and Associates of the Genomic Variation Laboratory</h2>\r\n<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/ca_depart_of_fish_game.jpg\"><img class=\"size-full wp-image-162 alignnone\" alt=\"ca_depart_of_fish_game\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/ca_depart_of_fish_game.jpg\" width=\"102\" height=\"135\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/department_of_water_resources_ca.jpg\"><img class=\"size-full wp-image-163 alignnone\" alt=\"department_of_water_resources_ca\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/department_of_water_resources_ca.jpg\" width=\"125\" height=\"125\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/us_depart_of_interior.jpg\"><img class=\"size-full wp-image-164 alignnone\" alt=\"us_depart_of_interior\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/us_depart_of_interior.jpg\" width=\"123\" height=\"123\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/us_fish_wildlife_service.jpg\"><img class=\"size-full wp-image-165 alignnone\" alt=\"us_fish_wildlife_service\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/us_fish_wildlife_service.jpg\" width=\"103\" height=\"124\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/us_forest_service.jpg\"><img class=\"size-full wp-image-166 alignnone\" alt=\"us_forest_service\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/us_forest_service.jpg\" width=\"108\" height=\"121\" /></a>','Partners','','inherit','closed','open','','10-revision-v1','','','2014-02-21 21:39:06','2014-02-21 21:39:06','',10,'http://localhost/~psalessi/Genetics_Lab/10-revision-v1/',0,'revision','',0),(168,1,'2014-02-21 21:39:18','2014-02-21 21:39:18','<h2 style=\"text-align: center;\">Funders and Associates of the Genomic Variation Laboratory</h2>\r\n<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/ca_depart_of_fish_game.jpg\"><img class=\"size-full wp-image-162 alignnone\" alt=\"ca_depart_of_fish_game\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/ca_depart_of_fish_game.jpg\" width=\"102\" height=\"135\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/department_of_water_resources_ca.jpg\"><img class=\"size-full wp-image-163 alignnone\" alt=\"department_of_water_resources_ca\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/department_of_water_resources_ca.jpg\" width=\"125\" height=\"125\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/us_depart_of_interior.jpg\"><img class=\"size-full wp-image-164 alignnone\" alt=\"us_depart_of_interior\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/us_depart_of_interior.jpg\" width=\"123\" height=\"123\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/us_fish_wildlife_service.jpg\"><img class=\"size-full wp-image-165 alignnone\" alt=\"us_fish_wildlife_service\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/us_fish_wildlife_service.jpg\" width=\"103\" height=\"124\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/us_forest_service.jpg\"><img class=\"size-full wp-image-166 alignnone aligncenter\" alt=\"us_forest_service\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/us_forest_service.jpg\" width=\"108\" height=\"121\" /></a>','Partners','','inherit','closed','open','','10-revision-v1','','','2014-02-21 21:39:18','2014-02-21 21:39:18','',10,'http://localhost/~psalessi/Genetics_Lab/10-revision-v1/',0,'revision','',0),(169,1,'2014-02-21 21:39:51','2014-02-21 21:39:51','<h2 style=\"text-align: left;\">Funders and Associates of the Genomic Variation Laboratory</h2>\r\n<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/ca_depart_of_fish_game.jpg\"><img class=\"size-full wp-image-162 alignnone\" alt=\"ca_depart_of_fish_game\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/ca_depart_of_fish_game.jpg\" width=\"102\" height=\"135\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/department_of_water_resources_ca.jpg\"><img class=\"size-full wp-image-163 alignnone\" alt=\"department_of_water_resources_ca\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/department_of_water_resources_ca.jpg\" width=\"125\" height=\"125\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/us_depart_of_interior.jpg\"><img class=\"size-full wp-image-164 alignnone\" alt=\"us_depart_of_interior\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/us_depart_of_interior.jpg\" width=\"123\" height=\"123\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/us_fish_wildlife_service.jpg\"><img class=\"size-full wp-image-165 alignnone\" alt=\"us_fish_wildlife_service\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/us_fish_wildlife_service.jpg\" width=\"103\" height=\"124\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/us_forest_service.jpg\"><img class=\"size-full wp-image-166 alignnone\" alt=\"us_forest_service\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/us_forest_service.jpg\" width=\"108\" height=\"121\" /></a>','Partners','','inherit','closed','open','','10-revision-v1','','','2014-02-21 21:39:51','2014-02-21 21:39:51','',10,'http://localhost/~psalessi/Genetics_Lab/10-revision-v1/',0,'revision','',0),(171,1,'2014-02-21 21:42:13','2014-02-21 21:42:13','','Partners Page US Bureau of Land Management Logo Image','','inherit','closed','open','','blmlogo','','','2014-02-21 21:42:13','2014-02-21 21:42:13','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/BLMLogo.jpg',0,'attachment','image/jpeg',0),(172,1,'2014-02-21 21:42:13','2014-02-21 21:42:13','','Partners Page California Department of Fish and Game','','inherit','closed','open','','dfgshield','','','2014-02-21 21:42:13','2014-02-21 21:42:13','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/DFGshield.jpg',0,'attachment','image/jpeg',0),(173,1,'2014-02-21 21:42:14','2014-02-21 21:42:14','','Partners Page US Department of the Navy Logo Image','','inherit','closed','open','','droppedimage_1','','','2014-02-21 21:42:14','2014-02-21 21:42:14','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_1.jpg',0,'attachment','image/jpeg',0),(174,1,'2014-02-21 21:42:14','2014-02-21 21:42:14','','Partners Page UC Mexus Logo Image','','inherit','closed','open','','droppedimage_2','','','2014-02-21 21:42:14','2014-02-21 21:42:14','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_2.jpg',0,'attachment','image/jpeg',0),(175,1,'2014-02-21 21:42:14','2014-02-21 21:42:14','','Partners Page Delta Stewardship Council Logo Image','','inherit','closed','open','','droppedimage_3','','','2014-02-21 21:42:14','2014-02-21 21:42:14','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_3.jpg',0,'attachment','image/jpeg',0),(176,1,'2014-02-21 21:42:15','2014-02-21 21:42:15','','Partners Page Sea Grant Logo Image','','inherit','closed','open','','droppedimage_4','','','2014-02-21 21:42:15','2014-02-21 21:42:15','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_4.jpg',0,'attachment','image/jpeg',0),(177,1,'2014-02-21 21:42:15','2014-02-21 21:42:15','','Partners Page Pacific Rim Research Program Logo Image','','inherit','closed','open','','droppedimage_5','','','2014-02-21 21:42:15','2014-02-21 21:42:15','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_5.jpg',0,'attachment','image/jpeg',0),(178,1,'2014-02-21 21:42:15','2014-02-21 21:42:15','','Partners Page Oregon Fish and Wildlife Logo Image','','inherit','closed','open','','droppedimage_6','','','2014-02-21 21:42:15','2014-02-21 21:42:15','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_6.jpg',0,'attachment','image/jpeg',0),(179,1,'2014-02-21 21:42:16','2014-02-21 21:42:16','','Partners Page USDA Logo Image','','inherit','closed','open','','droppedimage_7','','','2014-02-21 21:42:16','2014-02-21 21:42:16','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_7.jpg',0,'attachment','image/jpeg',0),(180,1,'2014-02-21 21:42:16','2014-02-21 21:42:16','','Partners Page Centro De Investigaciones Biologicas Del Noroeste Logo Image','','inherit','closed','open','','droppedimage_8','','','2014-02-21 21:42:16','2014-02-21 21:42:16','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_8.jpg',0,'attachment','image/jpeg',0),(181,1,'2014-02-21 21:42:16','2014-02-21 21:42:16','','Partners Page US National Oceanic and Atmospheric Administration Logo Image','','inherit','closed','open','','droppedimage_9','','','2014-02-21 21:42:16','2014-02-21 21:42:16','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_9.jpg',0,'attachment','image/jpeg',0),(182,1,'2014-02-21 21:42:17','2014-02-21 21:42:17','','Partners Page Great Lakes Fishery Trust Logo Image','','inherit','closed','open','','droppedimage_10','','','2014-02-21 21:42:17','2014-02-21 21:42:17','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_10.jpg',0,'attachment','image/jpeg',0),(183,1,'2014-02-21 21:42:17','2014-02-21 21:42:17','','Partners Page Idaho Power Logo Image','','inherit','closed','open','','droppedimage_11','','','2014-02-21 21:42:17','2014-02-21 21:42:17','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_11.jpg',0,'attachment','image/jpeg',0),(184,1,'2014-02-21 21:42:17','2014-02-21 21:42:17','','Partners Page Cramer Fish Sciences Logo Image','','inherit','closed','open','','droppedimage','','','2014-02-21 21:42:17','2014-02-21 21:42:17','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.jpg',0,'attachment','image/jpeg',0),(185,1,'2014-02-21 21:42:18','2014-02-21 21:42:18','','Partners Page Nevada Division of Wildlife Logo Image','','inherit','closed','open','','droppedimage-2','','','2014-02-21 21:42:18','2014-02-21 21:42:18','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.png',0,'attachment','image/png',0),(186,1,'2014-02-21 21:42:18','2014-02-21 21:42:18','','Partners Page California Department of Water Resources Logo Image','','inherit','closed','open','','dwr-logobig_home','','','2014-02-21 21:42:18','2014-02-21 21:42:18','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/dwr-logobig_home.jpg',0,'attachment','image/jpeg',0),(187,1,'2014-02-21 21:42:18','2014-02-21 21:42:18','','Partners Page Kootenai Tribe of Idaho Logo Image','','inherit','closed','open','','shapeimage_4','','','2014-02-21 21:42:18','2014-02-21 21:42:18','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/shapeimage_4.png',0,'attachment','image/png',0),(188,1,'2014-02-21 21:43:11','2014-02-21 21:43:11','<h2 style=\"text-align: left;\">Funders and Associates of the Genomic Variation Laboratory</h2>\r\n<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/BLMLogo.jpg\"><img class=\"alignnone size-full wp-image-171\" alt=\"BLMLogo\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/BLMLogo.jpg\" width=\"123\" height=\"123\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/DFGshield.jpg\"><img class=\"alignnone size-full wp-image-172\" alt=\"DFGshield\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/DFGshield.jpg\" width=\"102\" height=\"135\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_1.jpg\"><img class=\"alignnone size-full wp-image-173\" alt=\"droppedImage_1\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_1.jpg\" width=\"121\" height=\"121\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_2.jpg\"><img class=\"alignnone size-full wp-image-174\" alt=\"droppedImage_2\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_2.jpg\" width=\"188\" height=\"54\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_3.jpg\"><img class=\"alignnone size-full wp-image-175\" alt=\"droppedImage_3\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_3.jpg\" width=\"93\" height=\"93\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_4.jpg\"><img class=\"alignnone size-full wp-image-176\" alt=\"droppedImage_4\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_4.jpg\" width=\"124\" height=\"72\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_5.jpg\"><img class=\"alignnone size-full wp-image-177\" alt=\"droppedImage_5\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_5.jpg\" width=\"213\" height=\"68\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_6.jpg\"><img class=\"alignnone size-full wp-image-178\" alt=\"droppedImage_6\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_6.jpg\" width=\"86\" height=\"113\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_7.jpg\"><img class=\"alignnone size-full wp-image-179\" alt=\"droppedImage_7\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_7.jpg\" width=\"149\" height=\"102\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_8.jpg\"><img class=\"alignnone size-full wp-image-180\" alt=\"droppedImage_8\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_8.jpg\" width=\"244\" height=\"70\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_9.jpg\"><img class=\"alignnone size-full wp-image-181\" alt=\"droppedImage_9\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_9.jpg\" width=\"119\" height=\"119\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_10.jpg\"><img class=\"alignnone size-full wp-image-182\" alt=\"droppedImage_10\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_10.jpg\" width=\"157\" height=\"123\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_11.jpg\"><img class=\"alignnone size-full wp-image-183\" alt=\"droppedImage_11\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_11.jpg\" width=\"192\" height=\"71\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.jpg\"><img class=\"alignnone size-full wp-image-184\" alt=\"droppedImage\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.jpg\" width=\"233\" height=\"46\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.png\"><img class=\"alignnone size-full wp-image-185\" alt=\"droppedImage\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.png\" width=\"97\" height=\"140\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/dwr-logobig_home.jpg\"><img class=\"alignnone size-full wp-image-186\" alt=\"dwr-logobig_home\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/dwr-logobig_home.jpg\" width=\"125\" height=\"125\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/shapeimage_4.png\"><img class=\"alignnone size-full wp-image-187\" alt=\"shapeimage_4\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/shapeimage_4.png\" width=\"167\" height=\"112\" /></a>','Partners','','inherit','closed','open','','10-revision-v1','','','2014-02-21 21:43:11','2014-02-21 21:43:11','',10,'http://localhost/~psalessi/Genetics_Lab/10-revision-v1/',0,'revision','',0),(189,1,'2014-02-21 21:43:54','2014-02-21 21:43:54','<h2 style=\"text-align: center;\">Funders and Associates of the Genomic Variation Laboratory</h2>\r\n<p style=\"text-align: center;\"><a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/BLMLogo.jpg\"><img class=\"alignnone size-full wp-image-171\" alt=\"BLMLogo\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/BLMLogo.jpg\" width=\"123\" height=\"123\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/DFGshield.jpg\"><img class=\"alignnone size-full wp-image-172\" alt=\"DFGshield\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/DFGshield.jpg\" width=\"102\" height=\"135\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_1.jpg\"><img class=\"alignnone size-full wp-image-173\" alt=\"droppedImage_1\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_1.jpg\" width=\"121\" height=\"121\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_2.jpg\"><img class=\"alignnone size-full wp-image-174\" alt=\"droppedImage_2\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_2.jpg\" width=\"188\" height=\"54\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_3.jpg\"><img class=\"alignnone size-full wp-image-175\" alt=\"droppedImage_3\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_3.jpg\" width=\"93\" height=\"93\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_4.jpg\"><img class=\"alignnone size-full wp-image-176\" alt=\"droppedImage_4\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_4.jpg\" width=\"124\" height=\"72\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_5.jpg\"><img class=\"alignnone size-full wp-image-177\" alt=\"droppedImage_5\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_5.jpg\" width=\"213\" height=\"68\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_6.jpg\"><img class=\"alignnone size-full wp-image-178\" alt=\"droppedImage_6\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_6.jpg\" width=\"86\" height=\"113\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_7.jpg\"><img class=\"alignnone size-full wp-image-179\" alt=\"droppedImage_7\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_7.jpg\" width=\"149\" height=\"102\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_8.jpg\"><img class=\"alignnone size-full wp-image-180\" alt=\"droppedImage_8\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_8.jpg\" width=\"244\" height=\"70\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_9.jpg\"><img class=\"alignnone size-full wp-image-181\" alt=\"droppedImage_9\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_9.jpg\" width=\"119\" height=\"119\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_10.jpg\"><img class=\"alignnone size-full wp-image-182\" alt=\"droppedImage_10\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_10.jpg\" width=\"157\" height=\"123\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_11.jpg\"><img class=\"alignnone size-full wp-image-183\" alt=\"droppedImage_11\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_11.jpg\" width=\"192\" height=\"71\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.jpg\"><img class=\"alignnone size-full wp-image-184\" alt=\"droppedImage\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.jpg\" width=\"233\" height=\"46\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.png\"><img class=\"alignnone size-full wp-image-185\" alt=\"droppedImage\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.png\" width=\"97\" height=\"140\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/dwr-logobig_home.jpg\"><img class=\"alignnone size-full wp-image-186\" alt=\"dwr-logobig_home\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/dwr-logobig_home.jpg\" width=\"125\" height=\"125\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/shapeimage_4.png\"><img class=\"alignnone size-full wp-image-187\" alt=\"shapeimage_4\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/shapeimage_4.png\" width=\"167\" height=\"112\" /></a></p>','Partners','','inherit','closed','open','','10-revision-v1','','','2014-02-21 21:43:54','2014-02-21 21:43:54','',10,'http://localhost/~psalessi/Genetics_Lab/10-revision-v1/',0,'revision','',0),(190,1,'2014-02-21 21:53:39','2014-02-21 21:53:39','<strong>2013</strong>\r\n\r\nChen, Y., S. Parmenter, and B. May. Genetic characterization of variation and management implications for the endangeredMohave tui chub. Cons. Gen. 14:11-20. (pdf)\r\n\r\nMeek, M. H., A. P. Wintzer, N. Shepherd, and B. May. Genetic diversity and reproductive mode in two non-native hydromedusae, Maeotias marginata and Moerisia sp., in the upper San Francisco Estuary, California. Biol. Invasions. 15(1): 199-212. (pdf)\r\n\r\nSchreier, A.D., B. Mahardja, and B. May. Hierarchical patterns of population structure in the endangered Fraser River white sturgeon, Acipenser transmontanus, and implications for conservation. Can. J. Fish. &amp; Aquat. Sci. 69:1968-1980. (pdf)\r\n\r\n<strong>2012</strong>\r\n\r\nBaerwald, M.R., B.M. Schreier, G. Schumer, and B. May. Detection of Threatened Delta Smelt in the Gut Contents of the Invasive Mississippi Silverside in the San Francisco Estuary Using TaqMan Assays. Trans. Am. Fish. Soc. 141:1600-1607. (pdf)\r\n\r\nMahardja, B., B. May, and M. Baerwald. Characterization of 36 additional microsatellite loci in splittail (Pogonichthys macrolepidotus) and cross-amplification in five other native Californian cyprinid species. Cons. Gen. Res. 4(4):917-921. (pdf)\r\n\r\nMeek, M., A. Wintzer, W. Wetzel, and B. May. Climate change likely to facilitate the invasion of the non-native hydroid, Cordylophora caspia, in the San Francisco Estuary, CA. PLos ONE. 7(10): e46373. doi:10.1371/journal.pone.0046373 (pdf)\r\n\r\nPetersen, J.L., M.R. Baerwald, J.M. Ibarra, and B. May. A first-generation linkage map of the Pacific lion-paw scallop: initial evidence of QTL for size traits and markers linked to orange shell color. Aquaculture. 350-353:200-209. (pdf)\r\n\r\nSchreier, A. D., J. Rodzen, S. Ireland, and B. May. Genetic techniques inform conservation aquaculture of the endangered Kootenai River white sturgeon, Acipenser transmontanus. Endang. Species Res. 16:65-75. (pdf)\r\n\r\nSukmanomon, S., W. Kamonrat, S. Poompuang, T. Nguyen, D. M. Bartley, B. May, U. Na-Nakorn. Genetic changes, intra- and interspecific introgression in farmed Nile tilapia in Thailand. Aquaculture 324-325:44-54. (pdf)\r\n\r\nVan Dam, A.R. and B. May. A new species of Dactylopius Costa (Dactylopius gracilipilus sp. nov.) (Hemiptera: Coccoidea: Dactylopiidae) from the Chihuahuan Desert, Texas, U.S.A. . Zootaxa 3573:33-39. (pdf)\r\n\r\n<strong>2011</strong>\r\n\r\nBaerwald, M., J. Petersen, R. Hedrick, G. Schisler, and B. May. A major effect quantitative trait locus for whirling disease resistance identified in rainbow trout (Oncorhynchus mykiss). Heredity. 106:920-926. (pdf)\r\n\r\nBaerwald, M., G. Schumer, B. Schreier, and B. May. TaqMan assays for the genetic identification of delta smelt (Hypomesus transpacificus) and wakasagi smelt (Hypomesus nipponensis). Mol. Ecol. Res. 11:784-785. (pdf)\r\n\r\nBaerwald, M.R., M. R. Stephens, K. S. Bork, M. H. Meek, K. Tomalty, and B. May. 2011. Spring-run Chinook salmon genetic management plan  San Joaquin River Restoration Program. 118 pgs. Scientific genetic management plan for the CA Dept. of Fish and Game. (pdf)\r\n\r\nFinger, A. J., E. C. Anderson, M. R. Stephens, and B. P. May. Application of a method for estimating effective population size and admixture using diagnostic SNPs: Implications for conservation of threatened Paiute cutthroat trout (Oncorhynchus clarkii seleniris) in Silver King Creek, California. Can. J. Fish. Aquat. Sci. 68:1369-1386 (pdf).\r\n\r\nFisch, K.M., J.M. Henderson, R.S. Burton, and B. May. Population genetics and conservation implications for the endangered delta smelt in the San Francisco Bay-Delta. Conservation Genetics. 12:1421-1434 (pdf).\r\n\r\nFloyd, C. H., D.H. Van Vuren, K.R. Crooks, K.L. Jones, D.K. Garcelon, N.M. Belfiore, J.W. Dragoo, and B. May. Genetic differentiation of island spotted skunks, Spilogale gracilis amphiala. J. Mam. 92(1):148-158 (pdf).\r\n\r\nMcEachern, M.B., D.H. Van Vuren, C.H. Floyd, B. May, and J.M. Eadie.Bottlenecks and rescue effects in a fluctuating population of golden-mantled ground squirrels (Spermophilus lateralis). Con. Gen. 12:285-296 (pdf).\r\n\r\nMejia , O., L. G. Herrera, B. May, R. A. Medellin,and J. J. Flores-Martinez. Effective population size dynamics of Myotis vivesi during the Pleistocene and Holocene climatic changes. Acta Chiropterologica. 13:33-40 (pdf).\r\n\r\nSchreier, A.D., D. Gille, B. Mahardja, and B. May. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon, Acipenser transmontanus. J. Appl. Ichthiol. 27:24-33 (pdf).\r\n\r\nWintzer, A.P., M.H. Meek, P.B. Moyle, and B. May. Ecological insights into the polyp stage of non-native hydrozoans in the San Francisco Estuary. Aquat. Ecol. 45:151161 (pdf).\r\n\r\n<strong>2010</strong>\r\n\r\nFloyd, C. H., J.J. Flores Martnez, , L.G. Herrera, O. Mejia, and B. May. Conserving the endangered Mexican fishing bat (Myotis vivesi): genetic variation indicates extensive gene flow among islands in the Gulf of California. Cons. Gen. 11:813-82 (pdf).\r\n\r\nIsrael, J.A. and B. May. Indirect estimates of breeding population size in the polyploid green sturgeon (Acipenser medirostris). Mol. Ecol. 19:1058-107 (pdf).\r\n\r\nIsrael, J.A. and B. May. Characterization and evaluation of polymorphic microsatellite markers in the anadromous fish Spirinchus thaleichthys. Cons. Gen. Res. online (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Nuclear and mtDNA lineage diversity in wild and cultured Pacific lion-paw scallop (Baja California Peninsula, Mexico). Mar. Biol. 157:27512767 (pdf).\r\n\r\n<strong>2009</strong>\r\n\r\nChen, Y., S. Reid, and B. May. Genetic relationships among tui chub populations in the Cow Head and surrounding basins of the Northwestern Great Basin. Cons. Gen. 10:101-114 (pdf).\r\n\r\nFinger, A.J., M.R. Stephens, N.W. Clipperton and B. May. Six diagnostic SNP markers for detecting introgression between cutthroat and rainbow trout. Mol. Ecol. Res. 9:759-763 (pdf).\r\n\r\nFisch, K., J.L. Petersen, M.R. Baerwald, and B.P. May. Characterization of 24 microsatellite loci in delta smelt, Hypomesus transpacificus, and their cross-species amplification in two other smelt species of the Osmeridae family. Mol. Ecol. Res. 9:405-408 (pdf).\r\n\r\nIsrael, J.A., K.J. Bando, E.C. Anderson, and B. May. Polyploid microsatellite data reveal stock complexity among estuarine North American green sturgeon (Acipenser medirostris). Can. J. Fish. Aqu. Sci. 66:1491-1504 (pdf).\r\n\r\nMeek, M., M. Baerwald, A. Wintzer, and B. May. Isolation and characterization of microsatellite loci in two non-native hydromedusae in the San Francisco Estuary: Maeotias marginata and Moerisia sp. Cons. Gen. Res. 1:205-208 (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Thirty-seven additional microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and cross-amplification in other Pectinids. Cons. Gen. Res. 1:101-105 (pdf).\r\n\r\nSimmons, R. E., P. Lavretsky, and B. May. Introgressive Hybridization of Redband Trout in the Upper McCloud River Watershed. Trans. Am. Fish. Soc. 139:201-213 (pdf).\r\n\r\nStephens, M.R., N.W. Clipperton, and B.P. May. Subspecies-informative TaqMan probes for evaluating introgression between native golden trout and introduced rainbow trout. Mol. Ecol. Res. 9:339-343 (pdf).\r\n\r\n<strong>Earlier publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications','','inherit','closed','open','','12-revision-v1','','','2014-02-21 21:53:39','2014-02-21 21:53:39','',12,'http://localhost/~psalessi/Genetics_Lab/12-revision-v1/',0,'revision','',0),(191,1,'2014-02-24 17:59:14','2014-02-24 17:59:14','<h2 style=\"text-align: center;\">Funders and Associates of the Genomic Variation Laboratory</h2>\r\n<div class=\"partner_images\">\r\n<p style=\"text-align: center;\"><a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/BLMLogo.jpg\"><img class=\"alignnone size-full wp-image-171\" alt=\"BLMLogo\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/BLMLogo.jpg\" width=\"123\" height=\"123\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/DFGshield.jpg\"><img class=\"alignnone size-full wp-image-172\" alt=\"DFGshield\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/DFGshield.jpg\" width=\"102\" height=\"135\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_1.jpg\"><img class=\"alignnone size-full wp-image-173\" alt=\"droppedImage_1\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_1.jpg\" width=\"121\" height=\"121\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_2.jpg\"><img class=\"alignnone size-full wp-image-174\" alt=\"droppedImage_2\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_2.jpg\" width=\"188\" height=\"54\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_3.jpg\"><img class=\"alignnone size-full wp-image-175\" alt=\"droppedImage_3\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_3.jpg\" width=\"93\" height=\"93\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_4.jpg\"><img class=\"alignnone size-full wp-image-176\" alt=\"droppedImage_4\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_4.jpg\" width=\"124\" height=\"72\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_5.jpg\"><img class=\"alignnone size-full wp-image-177\" alt=\"droppedImage_5\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_5.jpg\" width=\"213\" height=\"68\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_6.jpg\"><img class=\"alignnone size-full wp-image-178\" alt=\"droppedImage_6\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_6.jpg\" width=\"86\" height=\"113\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_7.jpg\"><img class=\"alignnone size-full wp-image-179\" alt=\"droppedImage_7\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_7.jpg\" width=\"149\" height=\"102\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_8.jpg\"><img class=\"alignnone size-full wp-image-180\" alt=\"droppedImage_8\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_8.jpg\" width=\"244\" height=\"70\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_9.jpg\"><img class=\"alignnone size-full wp-image-181\" alt=\"droppedImage_9\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_9.jpg\" width=\"119\" height=\"119\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_10.jpg\"><img class=\"alignnone size-full wp-image-182\" alt=\"droppedImage_10\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_10.jpg\" width=\"157\" height=\"123\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_11.jpg\"><img class=\"alignnone size-full wp-image-183\" alt=\"droppedImage_11\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_11.jpg\" width=\"192\" height=\"71\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.jpg\"><img class=\"alignnone size-full wp-image-184\" alt=\"droppedImage\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.jpg\" width=\"233\" height=\"46\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.png\"><img class=\"alignnone size-full wp-image-185\" alt=\"droppedImage\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.png\" width=\"97\" height=\"140\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/dwr-logobig_home.jpg\"><img class=\"alignnone size-full wp-image-186\" alt=\"dwr-logobig_home\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/dwr-logobig_home.jpg\" width=\"125\" height=\"125\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/shapeimage_4.png\"><img class=\"alignnone size-full wp-image-187\" alt=\"shapeimage_4\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/shapeimage_4.png\" width=\"167\" height=\"112\" /></a></p>\r\n</div>','Partners','','inherit','closed','open','','10-revision-v1','','','2014-02-24 17:59:14','2014-02-24 17:59:14','',10,'http://localhost/~psalessi/Genetics_Lab/10-revision-v1/',0,'revision','',0),(192,1,'2014-02-24 18:09:41','2014-02-24 18:09:41','<h2 style=\"text-align: center;\">Funders and Associates of the Genomic Variation Laboratory</h2>\r\n<p style=\"text-align: center;\"><a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/BLMLogo.jpg\"><img class=\"alignnone size-full wp-image-171\" alt=\"BLMLogo\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/BLMLogo.jpg\" width=\"123\" height=\"123\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/DFGshield.jpg\"><img class=\"alignnone size-full wp-image-172\" alt=\"DFGshield\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/DFGshield.jpg\" width=\"102\" height=\"135\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_1.jpg\"><img class=\"alignnone size-full wp-image-173\" alt=\"droppedImage_1\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_1.jpg\" width=\"121\" height=\"121\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_2.jpg\"><img class=\"alignnone size-full wp-image-174\" alt=\"droppedImage_2\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_2.jpg\" width=\"188\" height=\"54\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_3.jpg\"><img class=\"alignnone size-full wp-image-175\" alt=\"droppedImage_3\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_3.jpg\" width=\"93\" height=\"93\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_4.jpg\"><img class=\"alignnone size-full wp-image-176\" alt=\"droppedImage_4\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_4.jpg\" width=\"124\" height=\"72\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_5.jpg\"><img class=\"alignnone size-full wp-image-177\" alt=\"droppedImage_5\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_5.jpg\" width=\"213\" height=\"68\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_6.jpg\"><img class=\"alignnone size-full wp-image-178\" alt=\"droppedImage_6\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_6.jpg\" width=\"86\" height=\"113\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_7.jpg\"><img class=\"alignnone size-full wp-image-179\" alt=\"droppedImage_7\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_7.jpg\" width=\"149\" height=\"102\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_8.jpg\"><img class=\"alignnone size-full wp-image-180\" alt=\"droppedImage_8\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_8.jpg\" width=\"244\" height=\"70\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_9.jpg\"><img class=\"alignnone size-full wp-image-181\" alt=\"droppedImage_9\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_9.jpg\" width=\"119\" height=\"119\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_10.jpg\"><img class=\"alignnone size-full wp-image-182\" alt=\"droppedImage_10\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_10.jpg\" width=\"157\" height=\"123\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_11.jpg\"><img class=\"alignnone size-full wp-image-183\" alt=\"droppedImage_11\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_11.jpg\" width=\"192\" height=\"71\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.jpg\"><img class=\"alignnone size-full wp-image-184\" alt=\"droppedImage\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.jpg\" width=\"233\" height=\"46\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.png\"><img class=\"alignnone size-full wp-image-185\" alt=\"droppedImage\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.png\" width=\"97\" height=\"140\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/dwr-logobig_home.jpg\"><img class=\"alignnone size-full wp-image-186\" alt=\"dwr-logobig_home\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/dwr-logobig_home.jpg\" width=\"125\" height=\"125\" /></a> <a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/shapeimage_4.png\"><img class=\"alignnone size-full wp-image-187\" alt=\"shapeimage_4\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/shapeimage_4.png\" width=\"167\" height=\"112\" /></a></p>','Partners','','inherit','closed','open','','10-revision-v1','','','2014-02-24 18:09:41','2014-02-24 18:09:41','',10,'http://localhost/~psalessi/Genetics_Lab/10-revision-v1/',0,'revision','',0),(193,1,'2014-02-24 18:12:31','2014-02-24 18:12:31','','U_S__Fish__and__Wildlife_Service','','inherit','closed','open','','u_s__fish__and__wildlife_service','','','2014-02-24 18:12:31','2014-02-24 18:12:31','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/U_S__Fish__and__Wildlife_Service.jpg',0,'attachment','image/jpeg',0),(194,1,'2014-02-24 18:12:31','2014-02-24 18:12:31','','Partners Page US Forest Service Logo Image','','inherit','closed','open','','usforestservicelogo','','','2014-02-24 18:12:31','2014-02-24 18:12:31','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/USForestServiceLogo.jpg',0,'attachment','image/jpeg',0),(195,1,'2014-02-24 18:13:49','2014-02-24 18:13:49','','Partners Page Kern Community Foundation Logo Image','','inherit','closed','open','','logo','','','2014-02-24 18:13:49','2014-02-24 18:13:49','',10,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/logo.jpg',0,'attachment','image/jpeg',0),(196,1,'2014-02-24 18:15:18','2014-02-24 18:15:18','<h2 style=\"text-align: center;\">Funders and Associates of the Genomic Variation Laboratory</h2>\r\n<p style=\"text-align: center;\"><img class=\"alignnone size-full wp-image-193\" style=\"font-size: 1rem; line-height: 1;\" alt=\"U_S__Fish__and__Wildlife_Service\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/U_S__Fish__and__Wildlife_Service.jpg\" width=\"103\" height=\"124\" /><img class=\"alignnone size-full wp-image-186\" style=\"font-size: 1rem; line-height: 1;\" alt=\"dwr-logobig_home\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/dwr-logobig_home.jpg\" width=\"125\" height=\"125\" /><img class=\"alignnone size-medium wp-image-194\" style=\"font-size: 1rem; line-height: 1;\" alt=\"USForestServiceLogo\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/USForestServiceLogo.jpg\" width=\"108\" height=\"121\" /><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/BLMLogo.jpg\"><img class=\"alignnone size-full wp-image-171\" alt=\"BLMLogo\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/BLMLogo.jpg\" width=\"123\" height=\"123\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/DFGshield.jpg\"><img class=\"alignnone size-full wp-image-172\" alt=\"DFGshield\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/DFGshield.jpg\" width=\"102\" height=\"135\" /></a><span style=\"line-height: 1.714285714; font-size: 1rem;\"></span><img class=\"alignnone size-full wp-image-184\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.jpg\" width=\"233\" height=\"46\" /><a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/logo.jpg\"><img class=\"alignnone size-full wp-image-195\" alt=\"logo\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/logo.jpg\" width=\"165\" height=\"57\" /></a><img class=\"alignnone size-full wp-image-187\" style=\"font-size: 1rem; line-height: 1;\" alt=\"shapeimage_4\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/shapeimage_4.png\" width=\"167\" height=\"112\" /><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_1.jpg\"><img class=\"alignnone size-full wp-image-173\" alt=\"droppedImage_1\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_1.jpg\" width=\"121\" height=\"121\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_2.jpg\"><img class=\"alignnone size-full wp-image-174\" alt=\"droppedImage_2\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_2.jpg\" width=\"188\" height=\"54\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_3.jpg\"><img class=\"alignnone size-full wp-image-175\" alt=\"droppedImage_3\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_3.jpg\" width=\"93\" height=\"93\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_4.jpg\"><img class=\"alignnone size-full wp-image-176\" alt=\"droppedImage_4\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_4.jpg\" width=\"124\" height=\"72\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_5.jpg\"><img class=\"alignnone size-full wp-image-177\" alt=\"droppedImage_5\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_5.jpg\" width=\"213\" height=\"68\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_6.jpg\"><img class=\"alignnone size-full wp-image-178\" alt=\"droppedImage_6\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_6.jpg\" width=\"86\" height=\"113\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_7.jpg\"><img class=\"alignnone size-full wp-image-179\" alt=\"droppedImage_7\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_7.jpg\" width=\"149\" height=\"102\" /></a><img class=\"alignnone size-full wp-image-181\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage_9\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_9.jpg\" width=\"119\" height=\"119\" /><img class=\"alignnone size-full wp-image-185\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.png\" width=\"97\" height=\"140\" /><img class=\"alignnone size-full wp-image-180\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage_8\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_8.jpg\" width=\"244\" height=\"70\" /><img class=\"alignnone size-full wp-image-182\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage_10\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_10.jpg\" width=\"157\" height=\"123\" /><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_11.jpg\"><img class=\"alignnone size-full wp-image-183\" alt=\"droppedImage_11\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_11.jpg\" width=\"192\" height=\"71\" /></a></p>','Partners','','inherit','closed','open','','10-revision-v1','','','2014-02-24 18:15:18','2014-02-24 18:15:18','',10,'http://localhost/~psalessi/Genetics_Lab/10-revision-v1/',0,'revision','',0),(197,1,'2014-02-24 18:17:06','2014-02-24 18:17:06','<div class=\"partner_images\">\r\n\r\n<h2 style=\"text-align: center;\">Funders and Associates of the Genomic Variation Laboratory</h2>\r\n<p style=\"text-align: center;\"><img class=\"alignnone size-full wp-image-193\" style=\"font-size: 1rem; line-height: 1;\" alt=\"U_S__Fish__and__Wildlife_Service\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/U_S__Fish__and__Wildlife_Service.jpg\" width=\"103\" height=\"124\" /><img class=\"alignnone size-full wp-image-186\" style=\"font-size: 1rem; line-height: 1;\" alt=\"dwr-logobig_home\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/dwr-logobig_home.jpg\" width=\"125\" height=\"125\" /><img class=\"alignnone size-medium wp-image-194\" style=\"font-size: 1rem; line-height: 1;\" alt=\"USForestServiceLogo\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/USForestServiceLogo.jpg\" width=\"108\" height=\"121\" /><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/BLMLogo.jpg\"><img class=\"alignnone size-full wp-image-171\" alt=\"BLMLogo\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/BLMLogo.jpg\" width=\"123\" height=\"123\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/DFGshield.jpg\"><img class=\"alignnone size-full wp-image-172\" alt=\"DFGshield\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/DFGshield.jpg\" width=\"102\" height=\"135\" /></a><span style=\"line-height: 1.714285714; font-size: 1rem;\"></span><img class=\"alignnone size-full wp-image-184\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.jpg\" width=\"233\" height=\"46\" /><a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/logo.jpg\"><img class=\"alignnone size-full wp-image-195\" alt=\"logo\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/logo.jpg\" width=\"165\" height=\"57\" /></a><img class=\"alignnone size-full wp-image-187\" style=\"font-size: 1rem; line-height: 1;\" alt=\"shapeimage_4\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/shapeimage_4.png\" width=\"167\" height=\"112\" /><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_1.jpg\"><img class=\"alignnone size-full wp-image-173\" alt=\"droppedImage_1\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_1.jpg\" width=\"121\" height=\"121\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_2.jpg\"><img class=\"alignnone size-full wp-image-174\" alt=\"droppedImage_2\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_2.jpg\" width=\"188\" height=\"54\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_3.jpg\"><img class=\"alignnone size-full wp-image-175\" alt=\"droppedImage_3\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_3.jpg\" width=\"93\" height=\"93\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_4.jpg\"><img class=\"alignnone size-full wp-image-176\" alt=\"droppedImage_4\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_4.jpg\" width=\"124\" height=\"72\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_5.jpg\"><img class=\"alignnone size-full wp-image-177\" alt=\"droppedImage_5\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_5.jpg\" width=\"213\" height=\"68\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_6.jpg\"><img class=\"alignnone size-full wp-image-178\" alt=\"droppedImage_6\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_6.jpg\" width=\"86\" height=\"113\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_7.jpg\"><img class=\"alignnone size-full wp-image-179\" alt=\"droppedImage_7\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_7.jpg\" width=\"149\" height=\"102\" /></a><img class=\"alignnone size-full wp-image-181\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage_9\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_9.jpg\" width=\"119\" height=\"119\" /><img class=\"alignnone size-full wp-image-185\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.png\" width=\"97\" height=\"140\" /><img class=\"alignnone size-full wp-image-180\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage_8\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_8.jpg\" width=\"244\" height=\"70\" /><img class=\"alignnone size-full wp-image-182\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage_10\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_10.jpg\" width=\"157\" height=\"123\" /><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_11.jpg\"><img class=\"alignnone size-full wp-image-183\" alt=\"droppedImage_11\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_11.jpg\" width=\"192\" height=\"71\" /></a></p>\r\n\r\n</div>','Partners','','inherit','closed','open','','10-revision-v1','','','2014-02-24 18:17:06','2014-02-24 18:17:06','',10,'http://localhost/~psalessi/Genetics_Lab/10-revision-v1/',0,'revision','',0),(199,1,'2014-02-24 18:32:40','2014-02-24 18:32:40','','Ppl_People Category','','publish','closed','closed','','acf_ppl_people-category','','','2014-03-17 17:41:47','2014-03-17 17:41:47','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=acf&#038;p=199',0,'acf','',0),(202,1,'2014-02-24 21:37:59','2014-02-24 21:37:59','','Bernie May','','publish','closed','closed','','bernie-may','','','2014-03-12 17:29:41','2014-03-12 17:29:41','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=202',0,'people','',0),(204,1,'2014-02-24 21:37:54','2014-02-24 21:37:54','','Bernie May Featured Image','Hey this is a cool fish. Check me out','inherit','closed','open','','director_header','','','2014-02-24 21:37:54','2014-02-24 21:37:54','',202,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/director_header.png',0,'attachment','image/png',0),(205,1,'2014-02-24 21:38:48','2014-02-24 21:38:48','','Bernie May Profile Image','','inherit','closed','open','','director_profile','','','2014-02-24 21:38:48','2014-02-24 21:38:48','',202,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/director_profile.jpg',0,'attachment','image/jpeg',0),(208,1,'2014-02-28 18:34:57','2014-02-28 18:34:57','','Ppl_Alumni','','publish','closed','closed','','acf_ppl_alumni','','','2014-03-17 16:57:48','2014-03-17 16:57:48','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=acf&#038;p=208',0,'acf','',0),(217,1,'2014-02-28 20:45:59','2014-02-28 20:45:59','<strong>Buffers &amp; Stocks</strong>\r\n<ul>\r\n	<li><a title=\"98% Formamide Loading Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/98-formamide-loading-dye/\">Formamide Loading Dye</a></li>\r\n	<li><a title=\"Diluting 1Kb Plus Ladder with Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/diluting-1kb-plus-ladder-with-dye/\">Diluting 1kb Ladder/Dye</a></li>\r\n	<li><a title=\"Gelco Fluorescein Ladder\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/gelco-fluorescein-ladder/\">Gelco Fluorescein Ladder</a></li>\r\n	<li><a title=\"Proteinase K\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/proteinase-k/\">Proteinase K</a></li>\r\n	<li><a title=\"Glassware Cleaner\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/glassware-cleaner/\">Glassware Cleaner</a></li>\r\n	<li><a title=\"dNTPs (Promega)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/dntps-promega/\">dNTPs (Promega)</a></li>\r\n	<li><a title=\"1M Tris-Cl\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1m-tris-cl/\">1M Tris-Cl</a></li>\r\n	<li><a title=\".5M EDTA (pH 8.0)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/5m-edta-ph-8-0/\">0.5M EDTA (pH 8.0)</a></li>\r\n	<li><a title=\"1X TE (10mM Tris; 1mM EDTA)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1x-te-10mm-tris-1mm-edta/\">1X TE (10mM Tris; 1mM EDTA)</a></li>\r\n	<li><a title=\"1X TLE (10mM Tris; 0.1mM EDTA)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1x-tle-10mm-tris-0-1mm-edta/\">1X TLE (10mM Tris; 0.1 mM EDTA)</a></li>\r\n	<li><a title=\"10X TNE\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/10x-tne/\">10X TNE</a></li>\r\n	<li><a title=\"10X TBE One-Shots\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/10x-tbe-one-shots/\">10X TBE One-Shots</a></li>\r\n</ul>\r\n<strong>FujiFilm</strong>\r\n<ul>\r\n	<li><a title=\"FujiFilm DNA Quantification\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/fujifilm-dna-quantification/\">DNA quantification</a></li>\r\n</ul>\r\n<strong>ABI 3730xl Maintenance</strong>\r\n<ul>\r\n	<li><a title=\"3730xl General Maintenance\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/3730xl-general-maintenance/\">3730xl Maintenance</a></li>\r\n	<li><a title=\"Proteinase K Wash\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/proteinase-k-wash/\">Proteinase K Wash</a></li>\r\n</ul>\r\n<strong>Acrylamide gels</strong>\r\n<ul>\r\n	<li><a title=\"Acrylamide (Manual Gel)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/acrylamide-manual-gel/\">Acrylamide Gel</a></li>\r\n	<li><a title=\"Acrylamide Gel Pour\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/acrylamide-gel-pour/\">Pouring</a></li>\r\n	<li><a title=\"Running Acrylamide\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/running-acrylamide/\">Running</a></li>\r\n</ul>\r\n<strong>Fluidigm</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/FLUIDIGM EP1 PROTOCOL.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/FLUIDIGM%20EP1%20PROTOCOL.doc\">Fluidigm EP1 Protocol</a></li>\r\n</ul>\r\n<strong>Illumina</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/RAD-SEQ Protocol.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/RAD-SEQ%20Protocol.doc\">RAD-SEQ Protocol</a></li>\r\n</ul>\r\n<strong>Other</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/tissuesampleprot.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/tissuesampleprot.pdf\">Tissue Sampling</a></li>\r\n	<li><a title=\"Supplies\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/supplies/\">Ordering and Receiving Supplies</a></li>\r\n	<li><a title=\"PCR Tips and Tricks\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/pcr-tips-and-tricks/\">PCR Tips and Tricks</a></li>\r\n</ul>','Protocols and Procedures','','publish','closed','closed','','protocols-and-procedures','','','2014-03-10 18:35:46','2014-03-10 18:35:46','',16,'http://localhost/~psalessi/Genetics_Lab/?page_id=217',0,'page','',0),(218,1,'2014-02-28 20:45:59','2014-02-28 20:45:59','<strong>Buffers &amp; Stocks</strong>\r\n<ul>\r\n	<li><a title=\"FormamideDye.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/FormamideDye.html\">Formamide Loading Dye</a></li>\r\n	<li><a title=\"Ladder.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Ladder.html\">Diluting 1kb Ladder/Dye</a></li>\r\n	<li><a title=\"GelcoLadder.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/GelcoLadder.html\">Gelco Fluorescein Ladder</a></li>\r\n	<li><a title=\"Proteinase_K.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Proteinase_K.html\">Proteinase K</a></li>\r\n	<li><a title=\"Glassware_Cleaner.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Glassware_Cleaner.html\">Glassware Cleaner</a></li>\r\n	<li><a title=\"dNTPs.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/dNTPs.html\">dNTPs (Promega)</a></li>\r\n	<li><a title=\"Tris-Cl.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Tris-Cl.html\">1M Tris-Cl</a></li>\r\n	<li><a title=\"EDTA.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/EDTA.html\">0.5M EDTA (pH 8.0)</a></li>\r\n	<li><a title=\"1XTE.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/1XTE.html\">1X TE (10mM Tris; 1mM EDTA)</a></li>\r\n	<li><a title=\"1XTLE.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/1XTLE.html\">1X TLE (10mM Tris; 0.1 mM EDTA)</a></li>\r\n	<li><a title=\"10XTNE.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/10XTNE.html\">10X TNE</a></li>\r\n	<li><a title=\"10XTBE.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/10XTBE.html\">10XTBE One-Shots</a></li>\r\n</ul>\r\n<strong>FujiFilm</strong>\r\n<ul>\r\n	<li><a title=\"DNAquant.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/DNAquant.html\">DNA quantification</a></li>\r\n</ul>\r\nABI 3730xl Maintenance\r\n<ul>\r\n	<li><a title=\"3730Maintenance.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/3730Maintenance.html\">3730xl Maintenance</a></li>\r\n	<li><a title=\"ProtKwash.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/ProtKwash.html\">Proteinase K Wash</a></li>\r\n</ul>\r\n<strong>Acrylamide gels</strong>\r\n<ul>\r\n	<li><a title=\"Acrylamide.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Acrylamide.html\">Acrylamide Gel</a></li>\r\n	<li><a title=\"AcGelpour.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/AcGelpour.html\">Pouring</a></li>\r\n	<li><a title=\"AcGelRun.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/AcGelRun.html\">Running</a></li>\r\n</ul>\r\n<strong>Fluidigm</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/FLUIDIGM EP1 PROTOCOL.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/FLUIDIGM%20EP1%20PROTOCOL.doc\">Fluidigm EP1 Protocol</a></li>\r\n</ul>\r\n<strong>Illumina</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/RAD-SEQ Protocol.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/RAD-SEQ%20Protocol.doc\">RAD-SEQ Protocol</a></li>\r\n</ul>\r\n<strong>Other</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/tissuesampleprot.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/tissuesampleprot.pdf\">Tissue Sampling</a></li>\r\n	<li><a title=\"Supplies.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Supplies.html\">Ordering and Receiving Supplies</a></li>\r\n	<li><a title=\"Supplies.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Supplies.html\">PCR Tips and Tricks</a></li>\r\n</ul>','Protocols and Procedures','','inherit','closed','open','','217-revision-v1','','','2014-02-28 20:45:59','2014-02-28 20:45:59','',217,'http://localhost/~psalessi/Genetics_Lab/217-revision-v1/',0,'revision','',0),(219,1,'2014-02-28 20:46:43','2014-02-28 20:46:43','','Protocols Page Featured Image','','inherit','closed','open','','protocols_header','','','2014-02-28 20:46:43','2014-02-28 20:46:43','',217,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/protocols_header.png',0,'attachment','image/png',0),(220,1,'2014-02-28 20:49:11','2014-02-28 20:49:11','<strong>GVL</strong>\r\n\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/Pop_Genet_ECL290/ecl_290__spring_2004-1.htm\" href=\"http://genome-lab.ucdavis.edu/Links/Pop_Genet_ECL290/ecl_290__spring_2004-1.htm\">Population Genetic Data Analysis ECL290 Winter 2005</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/ECL 290 Winter 2007/index.htm\" href=\"http://genome-lab.ucdavis.edu/Links/ECL%20290%20Winter%202007/index.htm\">Population Genetics Software ECL290 Winter 2007</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/genomic_variation_laboratory-directions1.htm\" href=\"http://genome-lab.ucdavis.edu/Links/genomic_variation_laboratory-directions1.htm\">Directions to the GVL</a>\r\n<a title=\"http://www.aes.ucdavis.edu/publications/Outlook/spring2000/research.htm\" href=\"http://www.aes.ucdavis.edu/publications/Outlook/spring2000/research.htm\">Articles on the GVL</a>\r\n\r\n<strong>GVL Resources</strong>\r\n\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/onlinelib.htm\" href=\"http://genome-lab.ucdavis.edu/Links/onlinelib.htm\">On-line Journals</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/Jobs.htm\" href=\"http://genome-lab.ucdavis.edu/Links/Jobs.htm\">Jobs</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/research_grants.htm\" href=\"http://genome-lab.ucdavis.edu/Links/research_grants.htm\">Funding sources</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/newsgroups.htm\" href=\"http://genome-lab.ucdavis.edu/Links/newsgroups.htm\">Newsgroups</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/federal agencies.htm\" href=\"http://genome-lab.ucdavis.edu/Links/federal%20agencies.htm\">Federal Agencies</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/Mol bio and genetics.htm\" href=\"http://genome-lab.ucdavis.edu/Links/Mol%20bio%20and%20genetics.htm\">Molecular Biology and Genetic Sites</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/Vendors.htm\" href=\"http://genome-lab.ucdavis.edu/Links/Vendors.htm\">Science Vendors</a>\r\n\r\n<strong>Outside Links</strong>\r\n\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/Grad student grant-writing workshop handouts 2007.pdf\" href=\"http://genome-lab.ucdavis.edu/Links/Grad%20student%20grant-writing%20workshop%20handouts%202007.pdf\">How to write a successful grant proposal</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/Universities.htm\" href=\"http://genome-lab.ucdavis.edu/Links/Universities.htm\">Universities</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/Web Development.htm\" href=\"http://genome-lab.ucdavis.edu/Links/Web%20Development%20.htm\">Web Development Tools</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/State agencies.htm\" href=\"http://genome-lab.ucdavis.edu/Links/State%20agencies.htm\">State Agencies</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/davisinfo.htm\" href=\"http://genome-lab.ucdavis.edu/davisinfo.htm\">City of Davis Information</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/Glossary of Molecular Genetic Terms.htm\" href=\"http://genome-lab.ucdavis.edu/Links/Glossary%20of%20Molecular%20Genetic%20Terms.htm\">Glossary of Molecular Genetic Terms</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/Science sites.htm\" href=\"http://genome-lab.ucdavis.edu/Links/Science%20sites.htm\">Science Sites</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/other Protocols.htm\" href=\"http://genome-lab.ucdavis.edu/Links/other%20Protocols.htm\">Protocols</a>','Links','','publish','closed','open','','links','','','2014-02-28 21:59:34','2014-02-28 21:59:34','',16,'http://localhost/~psalessi/Genetics_Lab/?page_id=220',0,'page','',0),(221,1,'2014-02-28 20:49:11','2014-02-28 20:49:11','<strong>GVL</strong>\r\n\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/Pop_Genet_ECL290/ecl_290__spring_2004-1.htm\" href=\"http://genome-lab.ucdavis.edu/Links/Pop_Genet_ECL290/ecl_290__spring_2004-1.htm\">Population Genetic Data Analysis ECL290 Winter 2005</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/ECL 290 Winter 2007/index.htm\" href=\"http://genome-lab.ucdavis.edu/Links/ECL%20290%20Winter%202007/index.htm\">Population Genetics Software ECL290 Winter 2007</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/genomic_variation_laboratory-directions1.htm\" href=\"http://genome-lab.ucdavis.edu/Links/genomic_variation_laboratory-directions1.htm\">Directions to the GVL</a>\r\n<a title=\"http://www.aes.ucdavis.edu/publications/Outlook/spring2000/research.htm\" href=\"http://www.aes.ucdavis.edu/publications/Outlook/spring2000/research.htm\">Articles on the GVL</a>\r\n\r\n<strong>GVL Resources</strong>\r\n\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/onlinelib.htm\" href=\"http://genome-lab.ucdavis.edu/Links/onlinelib.htm\">On-line Journals</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/Jobs.htm\" href=\"http://genome-lab.ucdavis.edu/Links/Jobs.htm\">Jobs</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/research_grants.htm\" href=\"http://genome-lab.ucdavis.edu/Links/research_grants.htm\">Funding sources</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/newsgroups.htm\" href=\"http://genome-lab.ucdavis.edu/Links/newsgroups.htm\">Newsgroups</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/federal agencies.htm\" href=\"http://genome-lab.ucdavis.edu/Links/federal%20agencies.htm\">Federal Agencies</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/Mol bio and genetics.htm\" href=\"http://genome-lab.ucdavis.edu/Links/Mol%20bio%20and%20genetics.htm\">Molecular Biology and Genetic Sites</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/Vendors.htm\" href=\"http://genome-lab.ucdavis.edu/Links/Vendors.htm\">Science Vendors</a>\r\n\r\n<strong>Outside Links</strong>\r\n\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/Grad student grant-writing workshop handouts 2007.pdf\" href=\"http://genome-lab.ucdavis.edu/Links/Grad%20student%20grant-writing%20workshop%20handouts%202007.pdf\">How to write a successful grant proposal</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/Universities.htm\" href=\"http://genome-lab.ucdavis.edu/Links/Universities.htm\">Universities</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/Web Development.htm\" href=\"http://genome-lab.ucdavis.edu/Links/Web%20Development%20.htm\">Web Development Tools</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/State agencies.htm\" href=\"http://genome-lab.ucdavis.edu/Links/State%20agencies.htm\">State Agencies</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/davisinfo.htm\" href=\"http://genome-lab.ucdavis.edu/davisinfo.htm\">City of Davis Information</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/Glossary of Molecular Genetic Terms.htm\" href=\"http://genome-lab.ucdavis.edu/Links/Glossary%20of%20Molecular%20Genetic%20Terms.htm\">Glossary of Molecular Genetic Terms</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/Science sites.htm\" href=\"http://genome-lab.ucdavis.edu/Links/Science%20sites.htm\">Science Sites</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/Links/other Protocols.htm\" href=\"http://genome-lab.ucdavis.edu/Links/other%20Protocols.htm\">Protocols</a>','Links','','inherit','closed','open','','220-revision-v1','','','2014-02-28 20:49:11','2014-02-28 20:49:11','',220,'http://localhost/~psalessi/Genetics_Lab/220-revision-v1/',0,'revision','',0),(223,1,'2014-02-28 20:51:21','2014-02-28 20:51:21','<p style=\"text-align: left;\">2403 Meyer Hall, Department of Animal Science\r\nUniversity of California, DavisOne Shields Avenue<img class=\"size-full wp-image-226 alignright\" alt=\"GVL_logo_green\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_green.jpg\" width=\"177\" height=\"186\" /> Davis, CA 95616</p>\r\n<a href=\"http://localhost/~psalessi/Genetics_Lab/resources/directions/\">Directions</a>\r\n\r\nPhone: (530) 752-6351\r\nFax: (530) 752-0175','Contact Us','','trash','closed','open','','contact-us','','','2014-06-09 20:40:44','2014-06-09 20:40:44','',16,'http://localhost/~psalessi/Genetics_Lab/?page_id=223',0,'page','',0),(224,1,'2014-02-28 20:51:21','2014-02-28 20:51:21','2403 Meyer Hall, Department of Animal Science\r\nUniversity of California, Davis\r\nOne Shields Avenue\r\nDavis, CA 95616\r\n\r\n<a href=\"http://localhost/~psalessi/Genetics_Lab/directions/\">Directions</a>\r\n\r\nPhone: (530) 752-6351\r\nFax: (530) 752-0175','Contact Us','','inherit','closed','open','','223-revision-v1','','','2014-02-28 20:51:21','2014-02-28 20:51:21','',223,'http://localhost/~psalessi/Genetics_Lab/223-revision-v1/',0,'revision','',0),(225,1,'2014-02-28 20:52:13','2014-02-28 20:52:13','2403 Meyer Hall, Department of Animal Science\r\nUniversity of California, Davis\r\nOne Shields Avenue\r\nDavis, CA 95616\r\n\r\n<a href=\"http://localhost/~psalessi/Genetics_Lab/resources/directions/\">Directions</a>\r\n\r\nPhone: (530) 752-6351\r\nFax: (530) 752-0175','Contact Us','','inherit','closed','open','','223-revision-v1','','','2014-02-28 20:52:13','2014-02-28 20:52:13','',223,'http://localhost/~psalessi/Genetics_Lab/223-revision-v1/',0,'revision','',0),(226,1,'2014-02-28 20:52:20','2014-02-28 20:52:20','','Genomic Variation Laboratory Green Logo Image','','inherit','closed','open','','gvl_logo_green','','','2014-02-28 20:52:20','2014-02-28 20:52:20','',223,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_green.jpg',0,'attachment','image/jpeg',0),(227,1,'2014-02-28 20:52:32','2014-02-28 20:52:32','<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_green.jpg\"><img class=\"size-full wp-image-226 alignright\" alt=\"GVL_logo_green\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_green.jpg\" width=\"177\" height=\"186\" /></a>2403 Meyer Hall, Department of Animal Science\r\nUniversity of California, Davis\r\nOne Shields Avenue\r\nDavis, CA 95616\r\n\r\n<a href=\"http://localhost/~psalessi/Genetics_Lab/resources/directions/\">Directions</a>\r\n\r\nPhone: (530) 752-6351\r\nFax: (530) 752-0175','Contact Us','','inherit','closed','open','','223-revision-v1','','','2014-02-28 20:52:32','2014-02-28 20:52:32','',223,'http://localhost/~psalessi/Genetics_Lab/223-revision-v1/',0,'revision','',0),(228,1,'2014-02-28 20:53:09','2014-02-28 20:53:09','<p style=\"text-align: left;\">2403 Meyer Hall, Department of Animal Science\r\nUniversity of California, Davis\r\nOne Shields Avenue\r\nDavis, CA 95616</p>\r\n<a href=\"http://localhost/~psalessi/Genetics_Lab/resources/directions/\">Directions</a>\r\n\r\n<img class=\"size-full wp-image-226 aligncenter\" alt=\"GVL_logo_green\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_green.jpg\" width=\"177\" height=\"186\" />\r\n\r\nPhone: (530) 752-6351\r\nFax: (530) 752-0175','Contact Us','','inherit','closed','open','','223-revision-v1','','','2014-02-28 20:53:09','2014-02-28 20:53:09','',223,'http://localhost/~psalessi/Genetics_Lab/223-revision-v1/',0,'revision','',0),(229,1,'2014-02-28 20:53:38','2014-02-28 20:53:38','<p style=\"text-align: left;\">2403 Meyer Hall, Department of Animal Science\r\nUniversity of California, DavisOne Shields Avenue<img class=\"size-full wp-image-226 alignright\" alt=\"GVL_logo_green\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_green.jpg\" width=\"177\" height=\"186\" /> Davis, CA 95616</p>\r\n<a href=\"http://localhost/~psalessi/Genetics_Lab/resources/directions/\">Directions</a>\r\n\r\nPhone: (530) 752-6351\r\nFax: (530) 752-0175','Contact Us','','inherit','closed','open','','223-revision-v1','','','2014-02-28 20:53:38','2014-02-28 20:53:38','',223,'http://localhost/~psalessi/Genetics_Lab/223-revision-v1/',0,'revision','',0),(230,1,'2014-02-28 20:57:35','2014-02-28 20:57:35','<strong>From Sac. Metro Airport:</strong>\r\n\r\nGet on I-5 south from airport Blvd.\r\n\r\nTake I-80 West towards San Francisco.\r\n\r\n<strong>From I-80 east or west:</strong>\r\n\r\nExit at the \"U.C. Davis/Old Davis Road\" exit and travel North.\r\n\r\nFollow Old Davis Road and go straight through first stop sign.\r\n\r\nCross bridge and turn left at first intersection, La Rue\r\n\r\nTurn right on Bioletti Way, and left in the parking lot. There is a permit dispenser just as you enter from this side. Parking is $7 per day.\r\n\r\nMeyer Hall is on the corner of Bioletti and La Rue.\r\n\r\nThe GVL is on the second floor in Rm. 2403 in the north building of Meyer Hall.\r\n\r\n<iframe style=\"border: 0;\" class=\"aligncenter\" src=\"https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d6241.639725534423!2d-121.75234279909584!3d38.53792040586795!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x8085291ab1c780f9%3A0xc365a885cdb2e287!2sMeyer+Hall!5e0!3m2!1sen!2sus!4v1402333214210\" width=\"600\" height=\"450\" frameborder=\"0\"></iframe>','Directions','','trash','closed','open','','directions','','','2014-06-09 20:40:40','2014-06-09 20:40:40','',16,'http://localhost/~psalessi/Genetics_Lab/?page_id=230',0,'page','',0),(231,1,'2014-02-28 20:56:27','2014-02-28 20:56:27','','Directions Page Map of Airport to Lab Image','','inherit','closed','open','','gvl_map_freeway','','','2014-02-28 20:56:27','2014-02-28 20:56:27','',230,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_map_freeway.jpg',0,'attachment','image/jpeg',0),(232,1,'2014-02-28 20:56:27','2014-02-28 20:56:27','','Directions Page Map of Freeway to Lab Image','','inherit','closed','open','','gvl_map_meyer_hall','','','2014-02-28 20:56:27','2014-02-28 20:56:27','',230,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_map_meyer_hall.jpg',0,'attachment','image/jpeg',0),(233,1,'2014-02-28 20:57:35','2014-02-28 20:57:35','<strong>From Sac. Metro Airport:</strong>\r\n\r\nGet on I-5 south from airport Blvd.\r\n\r\nTake I-80 West towards San Francisco.\r\n\r\n<strong>From I-80 east or west:</strong>\r\n\r\nExit at the \"U.C. Davis/Old Davis Road\" exit and travel North.\r\n\r\nFollow Old Davis Road and go straight through first stop sign.\r\n\r\nCross bridge and turn left at first intersection, La Rue\r\n\r\nTurn right on Bioletti Way, and left in the parking lot. There is a permit dispenser just as you enter from this side. Parking is $7 per day.\r\n\r\nMeyer Hall is on the corner of Bioletti and La Rue.\r\n\r\nThe GVL is on the second floor in Rm. 2403 in the north building of Meyer Hall.\r\n<p style=\"text-align: center;\"><img class=\"size-full wp-image-231 aligncenter\" style=\"font-size: 1rem; line-height: 1;\" alt=\"gvl_map_freeway\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_map_freeway.jpg\" width=\"604\" height=\"455\" /><a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_map_meyer_hall.jpg\"><img class=\"size-full wp-image-232 aligncenter\" alt=\"gvl_map_meyer_hall\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_map_meyer_hall.jpg\" width=\"558\" height=\"497\" /></a></p>','Directions','','inherit','closed','open','','230-revision-v1','','','2014-02-28 20:57:35','2014-02-28 20:57:35','',230,'http://localhost/~psalessi/Genetics_Lab/230-revision-v1/',0,'revision','',0),(234,1,'2014-03-05 20:03:28','2014-03-05 20:03:28','<img class=\"gv_blue_logo\" alt=\"GVL_logo_blue\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_blue.png\" width=\"207\" height=\"217\" /><strong><a title=\"Protocols and Procedures\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/\">Protocol and Procedures</a></strong>\n\n<strong><a title=\"Links\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/links/\">Links</a></strong>\n\n<strong><a title=\"Contact Us\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/contact-us/\">Contact Us</a></strong>\n\n<strong><a title=\"Directions\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/directions/\">Directions</a></strong>','Resources','','inherit','closed','open','','16-autosave-v1','','','2014-03-05 20:03:28','2014-03-05 20:03:28','',16,'http://localhost/~psalessi/Genetics_Lab/16-autosave-v1/',0,'revision','',0),(235,1,'2014-02-28 21:03:25','2014-02-28 21:03:25','','Genomic Variation Laboratory Blue Logo Image','','inherit','closed','open','','gvl_logo_blue','','','2014-02-28 21:03:25','2014-02-28 21:03:25','',16,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_blue.png',0,'attachment','image/png',0),(236,1,'2014-02-28 21:03:47','2014-02-28 21:03:47','<img class=\"size-full wp-image-235 alignright\" alt=\"GVL_logo_blue\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_blue.png\" width=\"207\" height=\"217\" />\r\n<h2><a title=\"Protocols and Procedures\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/\">Protocol and Procedures</a></h2>\r\n<h2><a title=\"Links\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/links/\">Links</a></h2>\r\n<h2><a title=\"Contact Us\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/contact-us/\">Contact Us</a></h2>\r\n<h2><a title=\"Directions\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/directions/\">Directions</a></h2>','Resources','','inherit','closed','open','','16-revision-v1','','','2014-02-28 21:03:47','2014-02-28 21:03:47','',16,'http://localhost/~psalessi/Genetics_Lab/16-revision-v1/',0,'revision','',0),(237,1,'2014-02-28 21:05:00','2014-02-28 21:05:00','<img class=\"size-full wp-image-235 alignright\" alt=\"GVL_logo_blue\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_blue.png\" width=\"207\" height=\"217\" /><strong><a title=\"Protocols and Procedures\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/\">Protocol and Procedures</a></strong>\r\n\r\n<strong><a title=\"Links\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/links/\">Links</a></strong>\r\n\r\n<strong><a title=\"Contact Us\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/contact-us/\">Contact Us</a></strong>\r\n\r\n<strong><a title=\"Directions\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/directions/\">Directions</a></strong>','Resources','','inherit','closed','open','','16-revision-v1','','','2014-02-28 21:05:00','2014-02-28 21:05:00','',16,'http://localhost/~psalessi/Genetics_Lab/16-revision-v1/',0,'revision','',0),(238,1,'2014-02-28 21:14:15','2014-02-28 21:14:15','The Conservation Genetics Fund supports the Genomic Variation Laboratory (GVL) at UC Davis. Your donation supports both quality science that informs species conservation and the students that generate that science.\n\nThe Genomic Variation Laboratory works with wildlife managers to help answer questions essential for species conservation. For example, <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\">our work</a> helps define species and their phylogenetic relationships and informs the genetics of reintroductions. We study topics such as evolution, predation, aquaculture, population genetics structure, hybridization, and genetic management for reintroduction of salmonids.\n\nOur lab trains undergraduate and graduate <a title=\"People\" href=\"http://http://localhost/~psalessi/Genetics_Lab/people/\">students</a> in many different molecular biological techniques, preparing them for a wide variety of careers. Bernie May is very supportive of career development and encourages his students to attend conferences, participate in internships, and network with agency biologists. We also host visiting researchers from outside the university to work with us in our lab.\n\n[caption id=\"attachment_239\" align=\"aligncenter\" width=\"240\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\"><img class=\"size-full wp-image-239\" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\n<h2 style=\"text-align: center;\">Ways to Make A Donation:</h2>\n<ol>\n	<li>Online:Donate through the secure UC Davis donation website <a title=\"Giving to UC Davis\" href=\"https://awc.ucdavis.edu/MakeAGift.aspx?alloccat=130202120\" target=\"_blank\">HERE</a>.</li>\n	<li>By Mail:<span style=\"line-height: 1.714285714; font-size: 1rem;\">Mail your check made out to University of California Regents to</span>Bernie May, Meyer Hall, 1 Shields Avenue, Davis, CA 95616. Please <a title=\"E-Mail Bernie May\" href=\"mailto:bpmay@ucdavis.edu?subject=Mail%20Donation\" target=\"_blank\">email Bernie May</a> to let him know the check is coming.</li>\n	<li>Learn more about making a donation:<span style=\"line-height: 1.714285714; font-size: 1rem;\">Contact Dr. Bernie May through email or phone (530) 754-8123</span></li>\n</ol>','Support Us','','inherit','closed','open','','14-autosave-v1','','','2014-02-28 21:14:15','2014-02-28 21:14:15','',14,'http://localhost/~psalessi/Genetics_Lab/14-autosave-v1/',0,'revision','',0),(239,1,'2014-02-28 21:09:48','2014-02-28 21:09:48','','Kern Community Foundation Grant Image','$89,045 Kern Community Foundation Grant','inherit','closed','open','','kern_community_foundation_grant','','','2014-02-28 21:09:48','2014-02-28 21:09:48','',14,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png',0,'attachment','image/png',0),(240,1,'2014-02-28 21:14:40','2014-02-28 21:14:40','The Conservation Genetics Fund supports the Genomic Variation Laboratory (GVL) at UC Davis. Your donation supports both quality science that informs species conservation and the students that generate that science.\r\n\r\nThe Genomic Variation Laboratory works with wildlife managers to help answer questions essential for species conservation. For example, <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">our work</a> helps define species and their phylogenetic relationships and informs the genetics of reintroductions. We study topics such as evolution, predation, aquaculture, population genetics structure, hybridization, and genetic management for reintroduction of salmonids.\r\n\r\nOur lab trains undergraduate and graduate <a title=\"People\" href=\"http://http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">students</a> in many different molecular biological techniques, preparing them for a wide variety of careers. Bernie May is very supportive of career development and encourages his students to attend conferences, participate in internships, and network with agency biologists. We also host visiting researchers from outside the university to work with us in our lab.\r\n\r\n[caption id=\"attachment_239\" align=\"aligncenter\" width=\"240\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\"><img class=\"size-full wp-image-239\" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n<h2 style=\"text-align: center;\">Ways to Make A Donation:</h2>\r\n<ol>\r\n	<li>Online:Donate through the secure UC Davis donation website <a title=\"Giving to UC Davis\" href=\"https://awc.ucdavis.edu/MakeAGift.aspx?alloccat=130202120\" target=\"_blank\">HERE</a>.</li>\r\n	<li>By Mail:<span style=\"line-height: 1.714285714; font-size: 1rem;\">Mail your check made out to University of California Regents to</span>Bernie May, Meyer Hall, 1 Shields Avenue, Davis, CA 95616. Please <a title=\"E-Mail Bernie May\" href=\"mailto:bpmay@ucdavis.edu?subject=Mail%20Donation\" target=\"_blank\">email Bernie May</a> to let him know the check is coming.</li>\r\n	<li>Learn more about making a donation:<span style=\"line-height: 1.714285714; font-size: 1rem;\">Contact Dr. Bernie May through <a title=\"E-Mail Bernie May\" href=\"mailto:bpmay@ucdavis.edu?subject=Mail%20Donation\" target=\"_blank\">email</a> or phone (530) 754-8123</span></li>\r\n</ol>','Support Us','','inherit','closed','open','','14-revision-v1','','','2014-02-28 21:14:40','2014-02-28 21:14:40','',14,'http://localhost/~psalessi/Genetics_Lab/14-revision-v1/',0,'revision','',0),(241,1,'2014-02-28 21:15:36','2014-02-28 21:15:36','The Conservation Genetics Fund supports the Genomic Variation Laboratory (GVL) at UC Davis. Your donation supports both quality science that informs species conservation and the students that generate that science.\r\n\r\nThe Genomic Variation Laboratory works with wildlife managers to help answer questions essential for species conservation. For example, <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">our work</a> helps define species and their phylogenetic relationships and informs the genetics of reintroductions. We study topics such as evolution, predation, aquaculture, population genetics structure, hybridization, and genetic management for reintroduction of salmonids.\r\n\r\nOur lab trains undergraduate and graduate <a title=\"People\" href=\"http://http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">students</a> in many different molecular biological techniques, preparing them for a wide variety of careers. Bernie May is very supportive of career development and encourages his students to attend conferences, participate in internships, and network with agency biologists. We also host visiting researchers from outside the university to work with us in our lab.\r\n\r\n[caption id=\"attachment_239\" align=\"aligncenter\" width=\"240\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\"><img class=\"size-full wp-image-239\" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n<h2 style=\"text-align: center;\">Ways to Make A Donation:</h2>\r\n<ol>\r\n	<li>Online:Donate through the secure UC Davis donation website <a title=\"Giving to UC Davis\" href=\"https://awc.ucdavis.edu/MakeAGift.aspx?alloccat=130202120\" target=\"_blank\">HERE</a>.</li>\r\n	<li>By Mail:<span style=\"line-height: 1.714285714; font-size: 1rem;\">Mail your check made out to University of California Regents to</span>Bernie May, Meyer Hall, 1 Shields Avenue, Davis, CA 95616. Please <a title=\"E-Mail Bernie May\" href=\"mailto:bpmay@ucdavis.edu?subject=Mail%20Donation\">email Bernie May</a> to let him know the check is coming.</li>\r\n	<li>Learn more about making a donation:<span style=\"line-height: 1.714285714; font-size: 1rem;\">Contact Dr. Bernie May through <a title=\"E-Mail Bernie May\" href=\"mailto:bpmay@ucdavis.edu?subject=Mail%20Donation\">email</a> or phone (530) 754-8123</span></li>\r\n</ol>','Support Us','','inherit','closed','open','','14-revision-v1','','','2014-02-28 21:15:36','2014-02-28 21:15:36','',14,'http://localhost/~psalessi/Genetics_Lab/14-revision-v1/',0,'revision','',0),(242,1,'2014-02-28 21:16:21','2014-02-28 21:16:21','The Conservation Genetics Fund supports the Genomic Variation Laboratory (GVL) at UC Davis. Your donation supports both quality science that informs species conservation and the students that generate that science.\r\n\r\nThe Genomic Variation Laboratory works with wildlife managers to help answer questions essential for species conservation. For example, <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">our work</a> helps define species and their phylogenetic relationships and informs the genetics of reintroductions. We study topics such as evolution, predation, aquaculture, population genetics structure, hybridization, and genetic management for reintroduction of salmonids.\r\n\r\nOur lab trains undergraduate and graduate <a title=\"People\" href=\"http://http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">students</a> in many different molecular biological techniques, preparing them for a wide variety of careers. Bernie May is very supportive of career development and encourages his students to attend conferences, participate in internships, and network with agency biologists. We also host visiting researchers from outside the university to work with us in our lab.\r\n\r\n[caption id=\"attachment_239\" align=\"aligncenter\" width=\"240\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\"><img class=\"size-full wp-image-239\" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n<h2 style=\"text-align: center;\">Ways to Make A Donation:</h2>\r\n<ol>\r\n	<li>Online:Donate through the secure UC Davis donation website <a title=\"Giving to UC Davis\" href=\"https://awc.ucdavis.edu/MakeAGift.aspx?alloccat=130202120\" target=\"_blank\">HERE</a>.</li>\r\n	<li>By Mail:Mail your check made out to University of California Regents toBernie May, Meyer Hall, 1 Shields Avenue, Davis, CA 95616. Please <a title=\"E-Mail Bernie May\" href=\"mailto:bpmay@ucdavis.edu?subject=Mail%20Donation\">email Bernie May</a> to let him know the check is coming.</li>\r\n	<li>Learn more about making a donation:Contact Dr. Bernie May through <a title=\"E-Mail Bernie May\" href=\"mailto:bpmay@ucdavis.edu?subject=Mail%20Donation\">email</a> or phone (530) 754-8123</li>\r\n</ol>','Support Us','','inherit','closed','open','','14-revision-v1','','','2014-02-28 21:16:21','2014-02-28 21:16:21','',14,'http://localhost/~psalessi/Genetics_Lab/14-revision-v1/',0,'revision','',0),(243,1,'2014-02-28 21:23:04','2014-02-28 21:23:04','<div class=\"partner_images\">\r\n<h2 style=\"text-align: center;\">Funders and Associates of the Genomic Variation Laboratory</h2>\r\n<p style=\"text-align: center;\"><a title=\"US Fish &amp; Wildlife Service\" href=\"http://www.fws.gov/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-193\" style=\"font-size: 1rem; line-height: 1;\" alt=\"U_S__Fish__and__Wildlife_Service\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/U_S__Fish__and__Wildlife_Service.jpg\" width=\"103\" height=\"124\" /></a><a title=\"California Department of Water Resources\" href=\"http://www.water.ca.gov/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-186\" style=\"font-size: 1rem; line-height: 1;\" alt=\"dwr-logobig_home\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/dwr-logobig_home.jpg\" width=\"125\" height=\"125\" /></a><a title=\"US Forest Service\" href=\"http://www.fs.fed.us/\" target=\"_blank\"><img class=\"alignnone size-medium wp-image-194\" style=\"font-size: 1rem; line-height: 1;\" alt=\"USForestServiceLogo\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/USForestServiceLogo.jpg\" width=\"108\" height=\"121\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" title=\"US Department of the Interior, Bureau of Land Management\" href=\"http://www.blm.gov/wo/st/en.html\" target=\"_blank\"><img class=\"alignnone size-full wp-image-171\" alt=\"BLMLogo\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/BLMLogo.jpg\" width=\"123\" height=\"123\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" title=\"California Department of Fish &amp; Game\" href=\"http://www.dfg.ca.gov/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-172\" alt=\"DFGshield\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/DFGshield.jpg\" width=\"102\" height=\"135\" /></a><span style=\"line-height: 1.714285714; font-size: 1rem;\"></span><a title=\"Cramer Fish Sciences\" href=\"http://www.fishsciences.net/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-184\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.jpg\" width=\"233\" height=\"46\" /></a><a title=\"Kern Community Foundation\" href=\"http://www.kernfoundation.org/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-195\" alt=\"logo\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/logo.jpg\" width=\"165\" height=\"57\" /></a><a title=\"Kootenai Tribe of Idaho\" href=\"http://www.kootenai.org/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-187\" style=\"font-size: 1rem; line-height: 1;\" alt=\"shapeimage_4\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/shapeimage_4.png\" width=\"167\" height=\"112\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_1.jpg\"><img class=\"alignnone size-full wp-image-173\" alt=\"droppedImage_1\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_1.jpg\" width=\"121\" height=\"121\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_2.jpg\"><img class=\"alignnone size-full wp-image-174\" alt=\"droppedImage_2\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_2.jpg\" width=\"188\" height=\"54\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_3.jpg\"><img class=\"alignnone size-full wp-image-175\" alt=\"droppedImage_3\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_3.jpg\" width=\"93\" height=\"93\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_4.jpg\"><img class=\"alignnone size-full wp-image-176\" alt=\"droppedImage_4\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_4.jpg\" width=\"124\" height=\"72\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_5.jpg\"><img class=\"alignnone size-full wp-image-177\" alt=\"droppedImage_5\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_5.jpg\" width=\"213\" height=\"68\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_6.jpg\"><img class=\"alignnone size-full wp-image-178\" alt=\"droppedImage_6\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_6.jpg\" width=\"86\" height=\"113\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_7.jpg\"><img class=\"alignnone size-full wp-image-179\" alt=\"droppedImage_7\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_7.jpg\" width=\"149\" height=\"102\" /></a><img class=\"alignnone size-full wp-image-181\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage_9\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_9.jpg\" width=\"119\" height=\"119\" /><img class=\"alignnone size-full wp-image-185\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.png\" width=\"97\" height=\"140\" /><img class=\"alignnone size-full wp-image-180\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage_8\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_8.jpg\" width=\"244\" height=\"70\" /><img class=\"alignnone size-full wp-image-182\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage_10\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_10.jpg\" width=\"157\" height=\"123\" /><a style=\"line-height: 1.714285714; font-size: 1rem;\" href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_11.jpg\"><img class=\"alignnone size-full wp-image-183\" alt=\"droppedImage_11\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_11.jpg\" width=\"192\" height=\"71\" /></a></p>\r\n\r\n</div>','Partners','','inherit','closed','open','','10-revision-v1','','','2014-02-28 21:23:04','2014-02-28 21:23:04','',10,'http://localhost/~psalessi/Genetics_Lab/10-revision-v1/',0,'revision','',0),(244,1,'2014-02-28 21:23:51','2014-02-28 21:23:51','<div class=\"partner_images\">\r\n<h2 style=\"text-align: center;\">Funders and Associates of the Genomic Variation Laboratory</h2>\r\n<p style=\"text-align: center;\"><a title=\"US Fish &amp; Wildlife Service\" href=\"http://www.fws.gov/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-193\" style=\"font-size: 1rem; line-height: 1;\" alt=\"U_S__Fish__and__Wildlife_Service\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/U_S__Fish__and__Wildlife_Service.jpg\" width=\"103\" height=\"124\" /></a><a title=\"California Department of Water Resources\" href=\"http://www.water.ca.gov/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-186\" style=\"font-size: 1rem; line-height: 1;\" alt=\"dwr-logobig_home\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/dwr-logobig_home.jpg\" width=\"125\" height=\"125\" /></a><a title=\"US Forest Service\" href=\"http://www.fs.fed.us/\" target=\"_blank\"><img class=\"alignnone size-medium wp-image-194\" style=\"font-size: 1rem; line-height: 1;\" alt=\"USForestServiceLogo\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/USForestServiceLogo.jpg\" width=\"108\" height=\"121\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" title=\"US Department of the Interior, Bureau of Land Management\" href=\"http://www.blm.gov/wo/st/en.html\" target=\"_blank\"><img class=\"alignnone size-full wp-image-171\" alt=\"BLMLogo\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/BLMLogo.jpg\" width=\"123\" height=\"123\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" title=\"California Department of Fish &amp; Game\" href=\"http://www.dfg.ca.gov/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-172\" alt=\"DFGshield\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/DFGshield.jpg\" width=\"102\" height=\"135\" /></a><span style=\"line-height: 1.714285714; font-size: 1rem;\"></span><a title=\"Cramer Fish Sciences\" href=\"http://www.fishsciences.net/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-184\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.jpg\" width=\"233\" height=\"46\" /></a><a title=\"Kern Community Foundation\" href=\"http://www.kernfoundation.org/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-195\" alt=\"logo\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/logo.jpg\" width=\"165\" height=\"57\" /></a><a title=\"Kootenai Tribe of Idaho\" href=\"http://www.kootenai.org/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-187\" style=\"font-size: 1rem; line-height: 1;\" alt=\"shapeimage_4\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/shapeimage_4.png\" width=\"167\" height=\"112\" /></a><span style=\"line-height: 1.714285714; font-size: 1rem;\"><img class=\"alignnone size-full wp-image-173\" alt=\"droppedImage_1\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_1.jpg\" width=\"121\" height=\"121\" /><img class=\"alignnone size-full wp-image-174\" alt=\"droppedImage_2\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_2.jpg\" width=\"188\" height=\"54\" /><img class=\"alignnone size-full wp-image-175\" alt=\"droppedImage_3\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_3.jpg\" width=\"93\" height=\"93\" /><img class=\"alignnone size-full wp-image-176\" alt=\"droppedImage_4\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_4.jpg\" width=\"124\" height=\"72\" /><img class=\"alignnone size-full wp-image-177\" alt=\"droppedImage_5\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_5.jpg\" width=\"213\" height=\"68\" /><img class=\"alignnone size-full wp-image-178\" alt=\"droppedImage_6\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_6.jpg\" width=\"86\" height=\"113\" /><img class=\"alignnone size-full wp-image-179\" alt=\"droppedImage_7\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_7.jpg\" width=\"149\" height=\"102\" /></span><img class=\"alignnone size-full wp-image-181\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage_9\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_9.jpg\" width=\"119\" height=\"119\" /><img class=\"alignnone size-full wp-image-185\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.png\" width=\"97\" height=\"140\" /><img class=\"alignnone size-full wp-image-180\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage_8\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_8.jpg\" width=\"244\" height=\"70\" /><img class=\"alignnone size-full wp-image-182\" style=\"font-size: 1rem; line-height: 1;\" alt=\"droppedImage_10\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_10.jpg\" width=\"157\" height=\"123\" /><span style=\"line-height: 1.714285714; font-size: 1rem;\"><img class=\"alignnone size-full wp-image-183\" alt=\"droppedImage_11\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_11.jpg\" width=\"192\" height=\"71\" /></span></p>\r\n\r\n</div>','Partners','','inherit','closed','open','','10-revision-v1','','','2014-02-28 21:23:51','2014-02-28 21:23:51','',10,'http://localhost/~psalessi/Genetics_Lab/10-revision-v1/',0,'revision','',0),(245,1,'2014-06-05 22:44:05','2014-06-05 22:44:05','Our work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\n\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\n\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<img class=\"size-full wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"240\" height=\"181\" /> $89,045 Kern Community Foundation Grant[/caption]\n\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.\n\n[caption id=\"attachment_247\" align=\"alignleft\" width=\"110\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\"size-full wp-image-247 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" alt=\"Learn About Our Projects\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\">Learn About Our Projects</a>[/caption]\n\n[caption id=\"attachment_246\" align=\"alignleft\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" alt=\"Support Us\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\">Support Us</a>[/caption]\n\n[caption id=\"attachment_248\" align=\"alignleft\" width=\"110\"]<a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\"><img class=\"size-full wp-image-248 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" width=\"110\" height=\"83\" /></a> <a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]','Home','','inherit','closed','open','','30-autosave-v1','','','2014-06-05 22:44:05','2014-06-05 22:44:05','',30,'http://localhost/~psalessi/Genetics_Lab/30-autosave-v1/',0,'revision','',0),(246,1,'2014-02-28 21:27:01','2014-02-28 21:27:01','','Home Page Support Us Image','<a target=\"_blank\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\" title=\"Support Us\">Support Us</a>','inherit','closed','open','','gvl_home_catfish','','','2014-02-28 21:27:01','2014-02-28 21:27:01','',30,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png',0,'attachment','image/png',0),(247,1,'2014-02-28 21:27:01','2014-02-28 21:27:01','','Home Page Projects Image','<a target=\"_blank\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" title=\"Projects\">Learn About Our Projects</a>','inherit','closed','open','','gvl_home_owl','','','2014-02-28 21:27:01','2014-02-28 21:27:01','',30,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png',0,'attachment','image/png',0),(248,1,'2014-02-28 21:27:02','2014-02-28 21:27:02','','Home Page Salmon Genetics Meeting 2012 Image','<a href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" title=\"Salmon Genetics Meeting 2012\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>','inherit','closed','open','','gvl_home_salmon','','','2014-02-28 21:27:02','2014-02-28 21:27:02','',30,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png',0,'attachment','image/png',0),(249,1,'2014-02-28 21:34:03','2014-02-28 21:34:03','[caption id=\"attachment_246\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246 \" alt=\"Support Us\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\" target=\"_blank\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_247\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\"><img class=\"size-full wp-image-247  \" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]\r\n\r\n[caption id=\"attachment_248\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\"><img class=\"size-full wp-image-248 \" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" width=\"110\" height=\"83\" /></a> <a title=\"Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about the\r\npopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\"><img class=\"size-full wp-image-239 \" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:34:03','2014-02-28 21:34:03','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(250,1,'2014-02-28 21:37:00','2014-02-28 21:37:00','Our work at the Genomic Variation Laboratory (GVL) addresses important questions about the\r\npopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\n[caption id=\"\" align=\"alignright\" width=\"110\"]<img class=\"size-full wp-image-247 \" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]\r\n\r\n<img class=\" wp-image-239    \" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /><span style=\"line-height: 1.714285714; font-size: 1rem;\"></span>Examples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:37:00','2014-02-28 21:37:00','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(251,1,'2014-02-28 21:38:05','2014-02-28 21:38:05','Our work at the Genomic Variation Laboratory (GVL) addresses important questions about the\r\n\r\n[caption id=\"attachment_247\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\"><img class=\"size-full wp-image-247\" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]\r\n\r\npopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:38:05','2014-02-28 21:38:05','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(252,1,'2014-02-28 21:38:38','2014-02-28 21:38:38','Our work at the Genomic Variation Laboratory (GVL) addresses important questions about the\r\npopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.[caption id=\"attachment_247\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\"><img class=\"size-full wp-image-247\" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:38:38','2014-02-28 21:38:38','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(253,1,'2014-02-28 21:39:18','2014-02-28 21:39:18','Our work at the Genomic Variation Laboratory (GVL) addresses important questions about the\r\npopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\n[caption id=\"attachment_247\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\"><img class=\"size-full wp-image-247\" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\n[caption id=\"attachment_246\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246\" alt=\"Support Us\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\" target=\"_blank\">Support Us</a>[/caption]\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:39:18','2014-02-28 21:39:18','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(254,1,'2014-02-28 21:39:38','2014-02-28 21:39:38','Our work at the Genomic Variation Laboratory (GVL) addresses important questions about the\r\npopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\n[caption id=\"attachment_247\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\"><img class=\"size-full wp-image-247\" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]\r\n[caption id=\"attachment_246\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246\" alt=\"Support Us\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\" target=\"_blank\">Support Us</a>[/caption]\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:39:38','2014-02-28 21:39:38','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(255,1,'2014-02-28 21:40:47','2014-02-28 21:40:47','Our work at the Genomic Variation Laboratory (GVL) addresses important questions about the\r\npopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\n[caption id=\"attachment_247\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\"><img class=\"size-full wp-image-247\" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246\" alt=\"Support Us\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\" target=\"_blank\">Support Us</a>[/caption]\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\n[caption id=\"attachment_248\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\"><img class=\"size-full wp-image-248\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" width=\"110\" height=\"83\" /></a> <a title=\"Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:40:47','2014-02-28 21:40:47','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(256,1,'2014-02-28 21:41:15','2014-02-28 21:41:15','Our work at the Genomic Variation Laboratory (GVL) addresses important questions about the\r\npopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\n[caption id=\"attachment_247\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\"><img class=\"size-full wp-image-247\" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246\" alt=\"Support Us\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\" target=\"_blank\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_248\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\"><img class=\"size-full wp-image-248\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" width=\"110\" height=\"83\" /></a> <a title=\"Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:41:15','2014-02-28 21:41:15','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(257,1,'2014-02-28 21:41:45','2014-02-28 21:41:45','[caption id=\"attachment_248\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\"><img class=\"size-full wp-image-248\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" width=\"110\" height=\"83\" /></a> <a title=\"Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]\r\n\r\n[caption id=\"attachment_247\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\"><img class=\"size-full wp-image-247\" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246\" alt=\"Support Us\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\" target=\"_blank\">Support Us</a>[/caption]\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about the\r\npopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:41:45','2014-02-28 21:41:45','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(258,1,'2014-02-28 21:42:07','2014-02-28 21:42:07','[caption id=\"attachment_248\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\"><img class=\"size-full wp-image-248\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" width=\"110\" height=\"83\" /></a> <a title=\"Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246\" alt=\"Support Us\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\" target=\"_blank\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_247\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\"><img class=\"size-full wp-image-247\" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about the\r\npopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:42:07','2014-02-28 21:42:07','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(259,1,'2014-02-28 21:42:27','2014-02-28 21:42:27','[caption id=\"attachment_248\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\"><img class=\"size-full wp-image-248\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" width=\"110\" height=\"83\" /></a> <a title=\"Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246\" alt=\"Support Us\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\" target=\"_blank\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_247\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\"><img class=\"size-full wp-image-247\" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about the\r\npopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\"><img class=\"size-full wp-image-239\" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:42:27','2014-02-28 21:42:27','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(260,1,'2014-02-28 21:42:50','2014-02-28 21:42:50','[caption id=\"attachment_248\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\"><img class=\"size-full wp-image-248\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" width=\"110\" height=\"83\" /></a> <a title=\"Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246\" alt=\"Support Us\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\" target=\"_blank\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_247\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\"><img class=\"size-full wp-image-247\" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about the\r\npopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\"><img class=\"size-full wp-image-239\" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:42:50','2014-02-28 21:42:50','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(261,1,'2014-02-28 21:43:26','2014-02-28 21:43:26','[caption id=\"attachment_248\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\"><img class=\"size-full wp-image-248\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" width=\"110\" height=\"83\" /></a> <a title=\"Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246\" alt=\"Support Us\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\" target=\"_blank\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_247\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\"><img class=\"size-full wp-image-247\" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]\r\n\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\"><img class=\"size-full wp-image-239\" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about the\r\npopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:43:26','2014-02-28 21:43:26','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(262,1,'2014-02-28 21:43:42','2014-02-28 21:43:42','[caption id=\"attachment_248\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\"><img class=\"size-full wp-image-248\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" width=\"110\" height=\"83\" /></a> <a title=\"Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246\" alt=\"Support Us\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\" target=\"_blank\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_247\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\"><img class=\"size-full wp-image-247\" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]\r\n<br/>\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\"><img class=\"size-full wp-image-239\" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about the\r\npopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:43:42','2014-02-28 21:43:42','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(263,1,'2014-02-28 21:44:02','2014-02-28 21:44:02','[caption id=\"attachment_248\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\"><img class=\"size-full wp-image-248\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" width=\"110\" height=\"83\" /></a> <a title=\"Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246\" alt=\"Support Us\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\" target=\"_blank\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_247\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\"><img class=\"size-full wp-image-247\" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about the\r\npopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\"><img class=\"size-full wp-image-239\" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:44:02','2014-02-28 21:44:02','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(264,1,'2014-02-28 21:44:33','2014-02-28 21:44:33','[caption id=\"attachment_248\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\"><img class=\"size-full wp-image-248\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" width=\"110\" height=\"83\" /></a> <a title=\"Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246\" alt=\"Support Us\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\" target=\"_blank\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_247\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\"><img class=\"size-full wp-image-247\" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about the\r\npopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\"><img class=\"size-full wp-image-239\" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:44:33','2014-02-28 21:44:33','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(265,1,'2014-02-28 21:46:57','2014-02-28 21:46:57','[caption id=\"attachment_248\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\"><img class=\"size-full wp-image-248\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" width=\"110\" height=\"83\" /></a> <a title=\"Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246\" alt=\"Support Us\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\" target=\"_blank\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_247\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\"><img class=\"size-full wp-image-247\" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]\r\n\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about the\r\npopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\"><img class=\"size-full wp-image-239\" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:46:57','2014-02-28 21:46:57','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(266,1,'2014-02-28 21:48:31','2014-02-28 21:48:31','[caption id=\"attachment_248\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\"><img class=\"size-full wp-image-248\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" width=\"110\" height=\"83\" /></a> <a title=\"Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246\" alt=\"Support Us\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\" target=\"_blank\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_247\" align=\"alignnone\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\"><img class=\"size-full wp-image-247\" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\"><img class=\"size-full wp-image-239\" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:48:31','2014-02-28 21:48:31','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(267,1,'2014-02-28 21:49:08','2014-02-28 21:49:08','Our work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\"><img class=\"size-full wp-image-239\" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:49:08','2014-02-28 21:49:08','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(268,1,'2014-02-28 21:49:36','2014-02-28 21:49:36','Our work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\"><img class=\"size-full wp-image-239\" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.\r\n\r\n[caption id=\"attachment_248\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\"><img class=\"size-full wp-image-248\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" width=\"110\" height=\"83\" /></a> <a title=\"Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246\" alt=\"Support Us\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\" target=\"_blank\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_247\" align=\"alignright\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\"><img class=\"size-full wp-image-247\" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:49:36','2014-02-28 21:49:36','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(269,1,'2014-02-28 21:50:00','2014-02-28 21:50:00','Our work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\"><img class=\"size-full wp-image-239\" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.\r\n\r\n[caption id=\"attachment_248\" align=\"alignleft\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\"><img class=\"size-full wp-image-248\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" width=\"110\" height=\"83\" /></a> <a title=\"Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignleft\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246\" alt=\"Support Us\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\" target=\"_blank\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_247\" align=\"alignleft\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\"><img class=\"size-full wp-image-247\" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:50:00','2014-02-28 21:50:00','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(270,1,'2014-02-28 21:50:47','2014-02-28 21:50:47','Our work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\"><img class=\"size-full wp-image-239\" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.\r\n\r\n[caption id=\"attachment_247\" align=\"alignleft\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\"><img class=\"size-full wp-image-247\" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">Learn About Our Projects</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignleft\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246\" alt=\"Support Us\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\" target=\"_blank\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_248\" align=\"alignleft\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\"><img class=\"size-full wp-image-248\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" width=\"110\" height=\"83\" /></a> <a title=\"Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]','Home','','inherit','closed','open','','30-revision-v1','','','2014-02-28 21:50:47','2014-02-28 21:50:47','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(271,1,'2014-02-28 21:52:56','2014-02-28 21:52:56','','Support Us Page Featured Image','','inherit','closed','open','','support_header','','','2014-02-28 21:52:56','2014-02-28 21:52:56','',14,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/support_header.png',0,'attachment','image/png',0),(272,1,'2014-02-28 21:53:46','2014-02-28 21:53:46','','Home Page Featured Image','','inherit','closed','open','','home_header','','','2014-02-28 21:53:46','2014-02-28 21:53:46','',30,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/home_header.png',0,'attachment','image/png',0),(273,1,'2014-02-28 21:58:54','2014-02-28 21:58:54','','Resources Page Featured Image','','inherit','closed','open','','resources_header','','','2014-02-28 21:58:54','2014-02-28 21:58:54','',16,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/resources_header.png',0,'attachment','image/png',0),(274,1,'2014-02-28 21:59:31','2014-02-28 21:59:31','','Links Page Featured Image','','inherit','closed','open','','links_header','','','2014-02-28 21:59:31','2014-02-28 21:59:31','',220,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/links_header.png',0,'attachment','image/png',0),(275,1,'2014-02-28 22:00:11','2014-02-28 22:00:11','','Contact Us Page Featured Image','','inherit','closed','open','','contact_header','','','2014-02-28 22:00:11','2014-02-28 22:00:11','',223,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/contact_header.png',0,'attachment','image/png',0),(276,1,'2014-02-28 22:00:56','2014-02-28 22:00:56','','Directions Page Featured Image','','inherit','closed','open','','directions_header','','','2014-02-28 22:00:56','2014-02-28 22:00:56','',230,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/directions_headr.png',0,'attachment','image/png',0),(277,1,'2014-03-03 17:45:48','2014-03-03 17:45:48','<p style=\"text-align: center;\"><a title=\"Past Alumni\" href=\"http://localhost/~psalessi/Genetics_Lab/alumni/past-alumni/\">Our past alumni are listed here!</a></p>','Alumni','','publish','closed','open','','alumni','','','2014-06-05 20:54:53','2014-06-05 20:54:53','',0,'http://localhost/~psalessi/Genetics_Lab/?page_id=277',0,'page','',0),(278,1,'2014-03-03 17:45:48','2014-03-03 17:45:48','','Alumni','','inherit','closed','open','','277-revision-v1','','','2014-03-03 17:45:48','2014-03-03 17:45:48','',277,'http://localhost/~psalessi/Genetics_Lab/277-revision-v1/',0,'revision','',0),(279,1,'2014-03-03 18:38:22','2014-03-03 18:38:22','There are our alumni','Alumni','','inherit','closed','open','','277-revision-v1','','','2014-03-03 18:38:22','2014-03-03 18:38:22','',277,'http://localhost/~psalessi/Genetics_Lab/277-revision-v1/',0,'revision','',0),(280,1,'2014-03-03 18:41:17','2014-03-03 18:41:17','','Alumni Page Featured Image','','inherit','closed','open','','alumni_header','','','2014-03-03 18:41:17','2014-03-03 18:41:17','',277,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/alumni_header.png',0,'attachment','image/png',0),(281,1,'2014-03-03 18:42:57','2014-03-03 18:42:57','','Alumni','','inherit','closed','open','','277-revision-v1','','','2014-03-03 18:42:57','2014-03-03 18:42:57','',277,'http://localhost/~psalessi/Genetics_Lab/277-revision-v1/',0,'revision','',0),(282,1,'2014-03-03 19:08:58','2014-03-03 19:08:58','<strong>2013</strong>\r\n\r\nChen, Y., S. Parmenter, and B. May. Genetic characterization of variation and management implications for the endangeredMohave tui chub. Cons. Gen. 14:11-20. (<a href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008%20-%20Baerwald_Distrib_splittail.pdf\">pdf</a>)\r\n\r\nMeek, M. H., A. P. Wintzer, N. Shepherd, and B. May. Genetic diversity and reproductive mode in two non-native hydromedusae, Maeotias marginata and Moerisia sp., in the upper San Francisco Estuary, California. Biol. Invasions. 15(1): 199-212. (pdf)\r\n\r\nSchreier, A.D., B. Mahardja, and B. May. Hierarchical patterns of population structure in the endangered Fraser River white sturgeon, Acipenser transmontanus, and implications for conservation. Can. J. Fish. &amp; Aquat. Sci. 69:1968-1980. (pdf)\r\n\r\n<strong>2012</strong>\r\n\r\nBaerwald, M.R., B.M. Schreier, G. Schumer, and B. May. Detection of Threatened Delta Smelt in the Gut Contents of the Invasive Mississippi Silverside in the San Francisco Estuary Using TaqMan Assays. Trans. Am. Fish. Soc. 141:1600-1607. (pdf)\r\n\r\nMahardja, B., B. May, and M. Baerwald. Characterization of 36 additional microsatellite loci in splittail (Pogonichthys macrolepidotus) and cross-amplification in five other native Californian cyprinid species. Cons. Gen. Res. 4(4):917-921. (pdf)\r\n\r\nMeek, M., A. Wintzer, W. Wetzel, and B. May. Climate change likely to facilitate the invasion of the non-native hydroid, Cordylophora caspia, in the San Francisco Estuary, CA. PLos ONE. 7(10): e46373. doi:10.1371/journal.pone.0046373 (pdf)\r\n\r\nPetersen, J.L., M.R. Baerwald, J.M. Ibarra, and B. May. A first-generation linkage map of the Pacific lion-paw scallop: initial evidence of QTL for size traits and markers linked to orange shell color. Aquaculture. 350-353:200-209. (pdf)\r\n\r\nSchreier, A. D., J. Rodzen, S. Ireland, and B. May. Genetic techniques inform conservation aquaculture of the endangered Kootenai River white sturgeon, Acipenser transmontanus. Endang. Species Res. 16:65-75. (pdf)\r\n\r\nSukmanomon, S., W. Kamonrat, S. Poompuang, T. Nguyen, D. M. Bartley, B. May, U. Na-Nakorn. Genetic changes, intra- and interspecific introgression in farmed Nile tilapia in Thailand. Aquaculture 324-325:44-54. (pdf)\r\n\r\nVan Dam, A.R. and B. May. A new species of Dactylopius Costa (Dactylopius gracilipilus sp. nov.) (Hemiptera: Coccoidea: Dactylopiidae) from the Chihuahuan Desert, Texas, U.S.A. . Zootaxa 3573:33-39. (pdf)\r\n\r\n<strong>2011</strong>\r\n\r\nBaerwald, M., J. Petersen, R. Hedrick, G. Schisler, and B. May. A major effect quantitative trait locus for whirling disease resistance identified in rainbow trout (Oncorhynchus mykiss). Heredity. 106:920-926. (pdf)\r\n\r\nBaerwald, M., G. Schumer, B. Schreier, and B. May. TaqMan assays for the genetic identification of delta smelt (Hypomesus transpacificus) and wakasagi smelt (Hypomesus nipponensis). Mol. Ecol. Res. 11:784-785. (pdf)\r\n\r\nBaerwald, M.R., M. R. Stephens, K. S. Bork, M. H. Meek, K. Tomalty, and B. May. 2011. Spring-run Chinook salmon genetic management plan  San Joaquin River Restoration Program. 118 pgs. Scientific genetic management plan for the CA Dept. of Fish and Game. (pdf)\r\n\r\nFinger, A. J., E. C. Anderson, M. R. Stephens, and B. P. May. Application of a method for estimating effective population size and admixture using diagnostic SNPs: Implications for conservation of threatened Paiute cutthroat trout (Oncorhynchus clarkii seleniris) in Silver King Creek, California. Can. J. Fish. Aquat. Sci. 68:1369-1386 (pdf).\r\n\r\nFisch, K.M., J.M. Henderson, R.S. Burton, and B. May. Population genetics and conservation implications for the endangered delta smelt in the San Francisco Bay-Delta. Conservation Genetics. 12:1421-1434 (pdf).\r\n\r\nFloyd, C. H., D.H. Van Vuren, K.R. Crooks, K.L. Jones, D.K. Garcelon, N.M. Belfiore, J.W. Dragoo, and B. May. Genetic differentiation of island spotted skunks, Spilogale gracilis amphiala. J. Mam. 92(1):148-158 (pdf).\r\n\r\nMcEachern, M.B., D.H. Van Vuren, C.H. Floyd, B. May, and J.M. Eadie.Bottlenecks and rescue effects in a fluctuating population of golden-mantled ground squirrels (Spermophilus lateralis). Con. Gen. 12:285-296 (pdf).\r\n\r\nMejia , O., L. G. Herrera, B. May, R. A. Medellin,and J. J. Flores-Martinez. Effective population size dynamics of Myotis vivesi during the Pleistocene and Holocene climatic changes. Acta Chiropterologica. 13:33-40 (pdf).\r\n\r\nSchreier, A.D., D. Gille, B. Mahardja, and B. May. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon, Acipenser transmontanus. J. Appl. Ichthiol. 27:24-33 (pdf).\r\n\r\nWintzer, A.P., M.H. Meek, P.B. Moyle, and B. May. Ecological insights into the polyp stage of non-native hydrozoans in the San Francisco Estuary. Aquat. Ecol. 45:151161 (pdf).\r\n\r\n<strong>2010</strong>\r\n\r\nFloyd, C. H., J.J. Flores Martnez, , L.G. Herrera, O. Mejia, and B. May. Conserving the endangered Mexican fishing bat (Myotis vivesi): genetic variation indicates extensive gene flow among islands in the Gulf of California. Cons. Gen. 11:813-82 (pdf).\r\n\r\nIsrael, J.A. and B. May. Indirect estimates of breeding population size in the polyploid green sturgeon (Acipenser medirostris). Mol. Ecol. 19:1058-107 (pdf).\r\n\r\nIsrael, J.A. and B. May. Characterization and evaluation of polymorphic microsatellite markers in the anadromous fish Spirinchus thaleichthys. Cons. Gen. Res. online (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Nuclear and mtDNA lineage diversity in wild and cultured Pacific lion-paw scallop (Baja California Peninsula, Mexico). Mar. Biol. 157:27512767 (pdf).\r\n\r\n<strong>2009</strong>\r\n\r\nChen, Y., S. Reid, and B. May. Genetic relationships among tui chub populations in the Cow Head and surrounding basins of the Northwestern Great Basin. Cons. Gen. 10:101-114 (pdf).\r\n\r\nFinger, A.J., M.R. Stephens, N.W. Clipperton and B. May. Six diagnostic SNP markers for detecting introgression between cutthroat and rainbow trout. Mol. Ecol. Res. 9:759-763 (pdf).\r\n\r\nFisch, K., J.L. Petersen, M.R. Baerwald, and B.P. May. Characterization of 24 microsatellite loci in delta smelt, Hypomesus transpacificus, and their cross-species amplification in two other smelt species of the Osmeridae family. Mol. Ecol. Res. 9:405-408 (pdf).\r\n\r\nIsrael, J.A., K.J. Bando, E.C. Anderson, and B. May. Polyploid microsatellite data reveal stock complexity among estuarine North American green sturgeon (Acipenser medirostris). Can. J. Fish. Aqu. Sci. 66:1491-1504 (pdf).\r\n\r\nMeek, M., M. Baerwald, A. Wintzer, and B. May. Isolation and characterization of microsatellite loci in two non-native hydromedusae in the San Francisco Estuary: Maeotias marginata and Moerisia sp. Cons. Gen. Res. 1:205-208 (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Thirty-seven additional microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and cross-amplification in other Pectinids. Cons. Gen. Res. 1:101-105 (pdf).\r\n\r\nSimmons, R. E., P. Lavretsky, and B. May. Introgressive Hybridization of Redband Trout in the Upper McCloud River Watershed. Trans. Am. Fish. Soc. 139:201-213 (pdf).\r\n\r\nStephens, M.R., N.W. Clipperton, and B.P. May. Subspecies-informative TaqMan probes for evaluating introgression between native golden trout and introduced rainbow trout. Mol. Ecol. Res. 9:339-343 (pdf).\r\n\r\n<strong>Earlier publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications','','inherit','closed','open','','12-revision-v1','','','2014-03-03 19:08:58','2014-03-03 19:08:58','',12,'http://localhost/~psalessi/Genetics_Lab/12-revision-v1/',0,'revision','',0),(288,1,'2014-03-05 19:36:21','2014-03-05 19:36:21','Our work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<img class=\"size-full wp-image-239\" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.\r\n\r\n[caption id=\"attachment_247\" align=\"alignleft\" width=\"110\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\"size-full wp-image-247 \" alt=\"Learn About Our Projects\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\">Learn About Our Projects</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignleft\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246 \" alt=\"Support Us\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_248\" align=\"alignleft\" width=\"110\"]<a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\"><img class=\"size-full wp-image-248 \" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" width=\"110\" height=\"83\" /></a> <a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]','Home','','inherit','closed','open','','30-revision-v1','','','2014-03-05 19:36:21','2014-03-05 19:36:21','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(289,1,'2014-03-05 19:38:15','2014-03-05 19:38:15','The Conservation Genetics Fund supports the Genomic Variation Laboratory (GVL) at UC Davis. Your donation supports both quality science that informs species conservation and the students that generate that science.\r\n\r\nThe Genomic Variation Laboratory works with wildlife managers to help answer questions essential for species conservation. For example, <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">our work</a> helps define species and their phylogenetic relationships and informs the genetics of reintroductions. We study topics such as evolution, predation, aquaculture, population genetics structure, hybridization, and genetic management for reintroduction of salmonids.\r\n\r\nOur lab trains undergraduate and graduate <a title=\"People\" href=\"http://http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">students</a> in many different molecular biological techniques, preparing them for a wide variety of careers. Bernie May is very supportive of career development and encourages his students to attend conferences, participate in internships, and network with agency biologists. We also host visiting researchers from outside the university to work with us in our lab.\r\n\r\n[caption id=\"attachment_239\" align=\"aligncenter\" width=\"240\"]<img class=\"size-full wp-image-239\" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /> $89,045 Kern Community Foundation Grant[/caption]\r\n<h2 style=\"text-align: center;\">Ways to Make a Donation:</h2>\r\n<ol>\r\n	<li>Online:Donate through the secure UC Davis donation website <a title=\"Giving to UC Davis\" href=\"https://awc.ucdavis.edu/MakeAGift.aspx?alloccat=130202120\" target=\"_blank\">HERE</a>.</li>\r\n	<li>By Mail:Mail your check made out to University of California Regents toBernie May, Meyer Hall, 1 Shields Avenue, Davis, CA 95616. Please <a title=\"E-Mail Bernie May\" href=\"mailto:bpmay@ucdavis.edu?subject=Mail%20Donation\">email Bernie May</a> to let him know the check is coming.</li>\r\n	<li>Learn more about making a donation:Contact Dr. Bernie May through <a title=\"E-Mail Bernie May\" href=\"mailto:bpmay@ucdavis.edu?subject=Mail%20Donation\">email</a> or phone (530) 754-8123</li>\r\n</ol>','Support Us','','inherit','closed','open','','14-revision-v1','','','2014-03-05 19:38:15','2014-03-05 19:38:15','',14,'http://localhost/~psalessi/Genetics_Lab/14-revision-v1/',0,'revision','',0),(290,1,'2014-03-05 19:38:43','2014-03-05 19:38:43','<strong>From Sac. Metro Airport:</strong>\r\n\r\nGet on I-5 south from airport Blvd.\r\n\r\nTake I-80 West towards San Francisco.\r\n\r\n<strong>From I-80 east or west:</strong>\r\n\r\nExit at the \"U.C. Davis/Old Davis Road\" exit and travel North.\r\n\r\nFollow Old Davis Road and go straight through first stop sign.\r\n\r\nCross bridge and turn left at first intersection, La Rue\r\n\r\nTurn right on Bioletti Way, and left in the parking lot. There is a permit dispenser just as you enter from this side. Parking is $7 per day.\r\n\r\nMeyer Hall is on the corner of Bioletti and La Rue.\r\n\r\nThe GVL is on the second floor in Rm. 2403 in the north building of Meyer Hall.\r\n<p style=\"text-align: center;\"><img class=\"size-full wp-image-231 aligncenter\" style=\"font-size: 1rem; line-height: 1;\" alt=\"gvl_map_freeway\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_map_freeway.jpg\" width=\"604\" height=\"455\" /><img class=\"size-full wp-image-232 aligncenter\" alt=\"gvl_map_meyer_hall\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_map_meyer_hall.jpg\" width=\"558\" height=\"497\" /></p>','Directions','','inherit','closed','open','','230-revision-v1','','','2014-03-05 19:38:43','2014-03-05 19:38:43','',230,'http://localhost/~psalessi/Genetics_Lab/230-revision-v1/',0,'revision','',0),(291,1,'2014-03-05 19:40:15','2014-03-05 19:40:15','The Conservation Genetics Fund supports the Genomic Variation Laboratory (GVL) at UC Davis. Your donation supports both quality science that informs species conservation and the students that generate that science.\r\n\r\nThe Genomic Variation Laboratory works with wildlife managers to help answer questions essential for species conservation. For example, <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">our work</a> helps define species and their phylogenetic relationships and informs the genetics of reintroductions. We study topics such as evolution, predation, aquaculture, population genetics structure, hybridization, and genetic management for reintroduction of salmonids.\r\n\r\nOur lab trains undergraduate and graduate <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">students</a> in many different molecular biological techniques, preparing them for a wide variety of careers. Bernie May is very supportive of career development and encourages his students to attend conferences, participate in internships, and network with agency biologists. We also host visiting researchers from outside the university to work with us in our lab.\r\n\r\n[caption id=\"attachment_239\" align=\"aligncenter\" width=\"240\"]<img class=\"size-full wp-image-239\" alt=\"$89,045 Kern Community Foundation Grant\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" width=\"240\" height=\"181\" /> $89,045 Kern Community Foundation Grant[/caption]\r\n<h2 style=\"text-align: center;\">Ways to Make a Donation:</h2>\r\n<ol>\r\n	<li>Online:Donate through the secure UC Davis donation website <a title=\"Giving to UC Davis\" href=\"https://awc.ucdavis.edu/MakeAGift.aspx?alloccat=130202120\" target=\"_blank\">HERE</a>.</li>\r\n	<li>By Mail:Mail your check made out to University of California Regents toBernie May, Meyer Hall, 1 Shields Avenue, Davis, CA 95616. Please <a title=\"E-Mail Bernie May\" href=\"mailto:bpmay@ucdavis.edu?subject=Mail%20Donation\">email Bernie May</a> to let him know the check is coming.</li>\r\n	<li>Learn more about making a donation:Contact Dr. Bernie May through <a title=\"E-Mail Bernie May\" href=\"mailto:bpmay@ucdavis.edu?subject=Mail%20Donation\">email</a> or phone (530) 754-8123</li>\r\n</ol>','Support Us','','inherit','closed','open','','14-revision-v1','','','2014-03-05 19:40:15','2014-03-05 19:40:15','',14,'http://localhost/~psalessi/Genetics_Lab/14-revision-v1/',0,'revision','',0),(292,1,'2014-03-05 20:02:25','2014-03-05 20:02:25','<img class=\"GVL_blue_logo\" alt=\"GVL_logo_blue\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_blue.png\" width=\"207\" height=\"217\" /><strong><a title=\"Protocols and Procedures\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/\">Protocol and Procedures</a></strong>\r\n\r\n<strong><a title=\"Links\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/links/\">Links</a></strong>\r\n\r\n<strong><a title=\"Contact Us\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/contact-us/\">Contact Us</a></strong>\r\n\r\n<strong><a title=\"Directions\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/directions/\">Directions</a></strong>','Resources','','inherit','closed','open','','16-revision-v1','','','2014-03-05 20:02:25','2014-03-05 20:02:25','',16,'http://localhost/~psalessi/Genetics_Lab/16-revision-v1/',0,'revision','',0),(293,1,'2014-03-05 20:03:43','2014-03-05 20:03:43','<img class=\"gvl_blue_logo\" alt=\"GVL_logo_blue\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_blue.png\" width=\"207\" height=\"217\" /><strong><a title=\"Protocols and Procedures\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/\">Protocol and Procedures</a></strong>\r\n\r\n<strong><a title=\"Links\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/links/\">Links</a></strong>\r\n\r\n<strong><a title=\"Contact Us\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/contact-us/\">Contact Us</a></strong>\r\n\r\n<strong><a title=\"Directions\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/directions/\">Directions</a></strong>','Resources','','inherit','closed','open','','16-revision-v1','','','2014-03-05 20:03:43','2014-03-05 20:03:43','',16,'http://localhost/~psalessi/Genetics_Lab/16-revision-v1/',0,'revision','',0),(294,1,'2014-03-05 20:03:57','2014-03-05 20:03:57','<img class=\"gvl_blue_logo alignright\" alt=\"GVL_logo_blue\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_blue.png\" width=\"207\" height=\"217\" /><strong><a title=\"Protocols and Procedures\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/\">Protocol and Procedures</a></strong>\r\n\r\n<strong><a title=\"Links\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/links/\">Links</a></strong>\r\n\r\n<strong><a title=\"Contact Us\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/contact-us/\">Contact Us</a></strong>\r\n\r\n<strong><a title=\"Directions\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/directions/\">Directions</a></strong>','Resources','','inherit','closed','open','','16-revision-v1','','','2014-03-05 20:03:57','2014-03-05 20:03:57','',16,'http://localhost/~psalessi/Genetics_Lab/16-revision-v1/',0,'revision','',0),(295,1,'2014-03-07 18:22:33','2014-03-07 18:22:33','<strong>2013</strong>\r\n\r\nChen, Y., S. Parmenter, and B. May. Genetic characterization of variation and management implications for the endangeredMohave tui chub. Cons. Gen. 14:11-20. (<a title=\"Genetic characterization and management of the endangered Mohave tui chub\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2013%20-%20Chen%20Mohave%20tui%20chub.pdf\">pdf</a>)\r\n\r\nMeek, M. H., A. P. Wintzer, N. Shepherd, and B. May. Genetic diversity and reproductive mode in two non-native hydromedusae, <em>Maeotias marginata</em> and <em>Moerisia</em> sp., in the upper San Francisco Estuary, California. Biol. Invasions. 15(1): 199-212. (<a title=\"Genetic diversity and reproductive mode in two non-native hydromedusae, Maeotias marginata and Moerisia sp., in the upper San Francisco Estuary, California\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Meek%20et%20al.pdf\">pdf</a>)\r\n\r\nSchreier, A.D., B. Mahardja, and B. May. Hierarchical patterns of population structure in the endangered Fraser River white sturgeon, Acipenser transmontanus, and implications for conservation. Can. J. Fish. &amp; Aquat. Sci. 69:1968-1980. (<a title=\"Hierarchical patterns of population structure in the endangered Fraser River white sturgeon (Acipenser transmontanus) and implications for conservation\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2013%20-%20Schreier%20Fraser%20river%20white%20sturgeon.pdf\">pdf</a>)\r\n\r\n<strong>2012</strong>\r\n\r\nBaerwald, M.R., B.M. Schreier, G. Schumer, and B. May. Detection of Threatened Delta Smelt in the Gut Contents of the Invasive Mississippi Silverside in the San Francisco Estuary Using TaqMan Assays. Trans. Am. Fish. Soc. 141:1600-1607. (pdf)\r\n\r\nMahardja, B., B. May, and M. Baerwald. Characterization of 36 additional microsatellite loci in splittail (Pogonichthys macrolepidotus) and cross-amplification in five other native Californian cyprinid species. Cons. Gen. Res. 4(4):917-921. (pdf)\r\n\r\nMeek, M., A. Wintzer, W. Wetzel, and B. May. Climate change likely to facilitate the invasion of the non-native hydroid, Cordylophora caspia, in the San Francisco Estuary, CA. PLos ONE. 7(10): e46373. doi:10.1371/journal.pone.0046373 (pdf)\r\n\r\nPetersen, J.L., M.R. Baerwald, J.M. Ibarra, and B. May. A first-generation linkage map of the Pacific lion-paw scallop: initial evidence of QTL for size traits and markers linked to orange shell color. Aquaculture. 350-353:200-209. (pdf)\r\n\r\nSchreier, A. D., J. Rodzen, S. Ireland, and B. May. Genetic techniques inform conservation aquaculture of the endangered Kootenai River white sturgeon, Acipenser transmontanus. Endang. Species Res. 16:65-75. (pdf)\r\n\r\nSukmanomon, S., W. Kamonrat, S. Poompuang, T. Nguyen, D. M. Bartley, B. May, U. Na-Nakorn. Genetic changes, intra- and interspecific introgression in farmed Nile tilapia in Thailand. Aquaculture 324-325:44-54. (pdf)\r\n\r\nVan Dam, A.R. and B. May. A new species of Dactylopius Costa (Dactylopius gracilipilus sp. nov.) (Hemiptera: Coccoidea: Dactylopiidae) from the Chihuahuan Desert, Texas, U.S.A. . Zootaxa 3573:33-39. (pdf)\r\n\r\n<strong>2011</strong>\r\n\r\nBaerwald, M., J. Petersen, R. Hedrick, G. Schisler, and B. May. A major effect quantitative trait locus for whirling disease resistance identified in rainbow trout (Oncorhynchus mykiss). Heredity. 106:920-926. (pdf)\r\n\r\nBaerwald, M., G. Schumer, B. Schreier, and B. May. TaqMan assays for the genetic identification of delta smelt (Hypomesus transpacificus) and wakasagi smelt (Hypomesus nipponensis). Mol. Ecol. Res. 11:784-785. (pdf)\r\n\r\nBaerwald, M.R., M. R. Stephens, K. S. Bork, M. H. Meek, K. Tomalty, and B. May. 2011. Spring-run Chinook salmon genetic management plan  San Joaquin River Restoration Program. 118 pgs. Scientific genetic management plan for the CA Dept. of Fish and Game. (pdf)\r\n\r\nFinger, A. J., E. C. Anderson, M. R. Stephens, and B. P. May. Application of a method for estimating effective population size and admixture using diagnostic SNPs: Implications for conservation of threatened Paiute cutthroat trout (Oncorhynchus clarkii seleniris) in Silver King Creek, California. Can. J. Fish. Aquat. Sci. 68:1369-1386 (pdf).\r\n\r\nFisch, K.M., J.M. Henderson, R.S. Burton, and B. May. Population genetics and conservation implications for the endangered delta smelt in the San Francisco Bay-Delta. Conservation Genetics. 12:1421-1434 (pdf).\r\n\r\nFloyd, C. H., D.H. Van Vuren, K.R. Crooks, K.L. Jones, D.K. Garcelon, N.M. Belfiore, J.W. Dragoo, and B. May. Genetic differentiation of island spotted skunks, Spilogale gracilis amphiala. J. Mam. 92(1):148-158 (pdf).\r\n\r\nMcEachern, M.B., D.H. Van Vuren, C.H. Floyd, B. May, and J.M. Eadie.Bottlenecks and rescue effects in a fluctuating population of golden-mantled ground squirrels (Spermophilus lateralis). Con. Gen. 12:285-296 (pdf).\r\n\r\nMejia , O., L. G. Herrera, B. May, R. A. Medellin,and J. J. Flores-Martinez. Effective population size dynamics of Myotis vivesi during the Pleistocene and Holocene climatic changes. Acta Chiropterologica. 13:33-40 (pdf).\r\n\r\nSchreier, A.D., D. Gille, B. Mahardja, and B. May. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon, Acipenser transmontanus. J. Appl. Ichthiol. 27:24-33 (pdf).\r\n\r\nWintzer, A.P., M.H. Meek, P.B. Moyle, and B. May. Ecological insights into the polyp stage of non-native hydrozoans in the San Francisco Estuary. Aquat. Ecol. 45:151161 (pdf).\r\n\r\n<strong>2010</strong>\r\n\r\nFloyd, C. H., J.J. Flores Martnez, , L.G. Herrera, O. Mejia, and B. May. Conserving the endangered Mexican fishing bat (Myotis vivesi): genetic variation indicates extensive gene flow among islands in the Gulf of California. Cons. Gen. 11:813-82 (pdf).\r\n\r\nIsrael, J.A. and B. May. Indirect estimates of breeding population size in the polyploid green sturgeon (Acipenser medirostris). Mol. Ecol. 19:1058-107 (pdf).\r\n\r\nIsrael, J.A. and B. May. Characterization and evaluation of polymorphic microsatellite markers in the anadromous fish Spirinchus thaleichthys. Cons. Gen. Res. online (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Nuclear and mtDNA lineage diversity in wild and cultured Pacific lion-paw scallop (Baja California Peninsula, Mexico). Mar. Biol. 157:27512767 (pdf).\r\n\r\n<strong>2009</strong>\r\n\r\nChen, Y., S. Reid, and B. May. Genetic relationships among tui chub populations in the Cow Head and surrounding basins of the Northwestern Great Basin. Cons. Gen. 10:101-114 (pdf).\r\n\r\nFinger, A.J., M.R. Stephens, N.W. Clipperton and B. May. Six diagnostic SNP markers for detecting introgression between cutthroat and rainbow trout. Mol. Ecol. Res. 9:759-763 (pdf).\r\n\r\nFisch, K., J.L. Petersen, M.R. Baerwald, and B.P. May. Characterization of 24 microsatellite loci in delta smelt, Hypomesus transpacificus, and their cross-species amplification in two other smelt species of the Osmeridae family. Mol. Ecol. Res. 9:405-408 (pdf).\r\n\r\nIsrael, J.A., K.J. Bando, E.C. Anderson, and B. May. Polyploid microsatellite data reveal stock complexity among estuarine North American green sturgeon (Acipenser medirostris). Can. J. Fish. Aqu. Sci. 66:1491-1504 (pdf).\r\n\r\nMeek, M., M. Baerwald, A. Wintzer, and B. May. Isolation and characterization of microsatellite loci in two non-native hydromedusae in the San Francisco Estuary: Maeotias marginata and Moerisia sp. Cons. Gen. Res. 1:205-208 (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Thirty-seven additional microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and cross-amplification in other Pectinids. Cons. Gen. Res. 1:101-105 (pdf).\r\n\r\nSimmons, R. E., P. Lavretsky, and B. May. Introgressive Hybridization of Redband Trout in the Upper McCloud River Watershed. Trans. Am. Fish. Soc. 139:201-213 (pdf).\r\n\r\nStephens, M.R., N.W. Clipperton, and B.P. May. Subspecies-informative TaqMan probes for evaluating introgression between native golden trout and introduced rainbow trout. Mol. Ecol. Res. 9:339-343 (pdf).\r\n\r\n<strong>Earlier publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications','','inherit','closed','open','','12-revision-v1','','','2014-03-07 18:22:33','2014-03-07 18:22:33','',12,'http://localhost/~psalessi/Genetics_Lab/12-revision-v1/',0,'revision','',0),(296,1,'2014-03-07 18:51:37','2014-03-07 18:51:37','<strong>2013</strong>\r\n\r\nChen, Y., S. Parmenter, and B. May. Genetic characterization of variation and management implications for the endangeredMohave tui chub. Cons. Gen. 14:11-20. (<a title=\"Publications_files/2013 - Chen Mohave tui chub.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2013%20-%20Chen%20Mohave%20tui%20chub.pdf\">pdf</a>)\r\n\r\nMeek, M. H., A. P. Wintzer, N. Shepherd, and B. May. Genetic diversity and reproductive mode in two non-native hydromedusae, <em>Maeotias marginata</em> and <em>Moerisia</em> sp., in the upper San Francisco Estuary, California. Biol. Invasions. 15(1): 199-212. (<a title=\"Publications_files/2012 Meek et al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Meek%20et%20al.pdf\">pdf</a>)\r\n\r\nSchreier, A.D., B. Mahardja, and B. May. Hierarchical patterns of population structure in the endangered Fraser River white sturgeon, Acipenser transmontanus, and implications for conservation. Can. J. Fish. &amp; Aquat. Sci. 69:1968-1980. (<a title=\"Publications_files/2013 - Schreier Fraser river white sturgeon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2013%20-%20Schreier%20Fraser%20river%20white%20sturgeon.pdf\">pdf</a>)\r\n\r\n<strong>2012</strong>\r\n\r\nBaerwald, M.R., B.M. Schreier, G. Schumer, and B. May. Detection of Threatened Delta Smelt in the Gut Contents of the Invasive Mississippi Silverside in the San Francisco Estuary Using TaqMan Assays. Trans. Am. Fish. Soc. 141:1600-1607. (<a title=\"Publications_files/2012 - Baerwald gut contents.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20-%20Baerwald%20gut%20contents.pdf\">pdf</a>)\r\n\r\nMahardja, B., B. May, and M. Baerwald. Characterization of 36 additional microsatellite loci in splittail (<em>Pogonichthys macrolepidotus</em>) and cross-amplification in five other native Californian cyprinid species. Cons. Gen. Res. 4(4):917-921. (<a title=\"Publications_files/2012 Meek et al - Climate change.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Mahardja%20et%20al.%20-%20Cyprinid%20Microsat%20note.pdf\">pdf</a>)\r\n\r\nMeek, M., A. Wintzer, W. Wetzel, and B. May. Climate change likely to facilitate the invasion of the non-native hydroid, Cordylophora caspia, in the San Francisco Estuary, CA. PLos ONE. 7(10): e46373. doi:10.1371/journal.pone.0046373 (<a title=\"Publications_files/2012 - Petersen lion paw scallop.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Meek%20et%20al%20-%20Climate%20change.pdf\">pdf</a>)\r\n\r\nPetersen, J.L., M.R. Baerwald, J.M. Ibarra, and B. May. A first-generation linkage map of the Pacific lion-paw scallop: initial evidence of QTL for size traits and markers linked to orange shell color. Aquaculture. 350-353:200-209. (<a title=\"Publications_files/2012 Schreier et al. - Kootenai.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_filesPublications_files/2012%20Schreier%20et%20al.%20-%20Kootenai.pdf\">pdf</a>)\r\n\r\nSchreier, A. D., J. Rodzen, S. Ireland, and B. May. Genetic techniques inform conservation aquaculture of the endangered Kootenai River white sturgeon, Acipenser transmontanus. Endang. Species Res. 16:65-75. (pdf)\r\n\r\nSukmanomon, S., W. Kamonrat, S. Poompuang, T. Nguyen, D. M. Bartley, B. May, U. Na-Nakorn. Genetic changes, intra- and interspecific introgression in farmed Nile tilapia in Thailand. Aquaculture 324-325:44-54. (pdf)\r\n\r\nVan Dam, A.R. and B. May. A new species of Dactylopius Costa (Dactylopius gracilipilus sp. nov.) (Hemiptera: Coccoidea: Dactylopiidae) from the Chihuahuan Desert, Texas, U.S.A. . Zootaxa 3573:33-39. (pdf)\r\n\r\n<strong>2011</strong>\r\n\r\nBaerwald, M., J. Petersen, R. Hedrick, G. Schisler, and B. May. A major effect quantitative trait locus for whirling disease resistance identified in rainbow trout (Oncorhynchus mykiss). Heredity. 106:920-926. (pdf)\r\n\r\nBaerwald, M., G. Schumer, B. Schreier, and B. May. TaqMan assays for the genetic identification of delta smelt (Hypomesus transpacificus) and wakasagi smelt (Hypomesus nipponensis). Mol. Ecol. Res. 11:784-785. (pdf)\r\n\r\nBaerwald, M.R., M. R. Stephens, K. S. Bork, M. H. Meek, K. Tomalty, and B. May. 2011. Spring-run Chinook salmon genetic management plan  San Joaquin River Restoration Program. 118 pgs. Scientific genetic management plan for the CA Dept. of Fish and Game. (pdf)\r\n\r\nFinger, A. J., E. C. Anderson, M. R. Stephens, and B. P. May. Application of a method for estimating effective population size and admixture using diagnostic SNPs: Implications for conservation of threatened Paiute cutthroat trout (Oncorhynchus clarkii seleniris) in Silver King Creek, California. Can. J. Fish. Aquat. Sci. 68:1369-1386 (pdf).\r\n\r\nFisch, K.M., J.M. Henderson, R.S. Burton, and B. May. Population genetics and conservation implications for the endangered delta smelt in the San Francisco Bay-Delta. Conservation Genetics. 12:1421-1434 (pdf).\r\n\r\nFloyd, C. H., D.H. Van Vuren, K.R. Crooks, K.L. Jones, D.K. Garcelon, N.M. Belfiore, J.W. Dragoo, and B. May. Genetic differentiation of island spotted skunks, Spilogale gracilis amphiala. J. Mam. 92(1):148-158 (pdf).\r\n\r\nMcEachern, M.B., D.H. Van Vuren, C.H. Floyd, B. May, and J.M. Eadie.Bottlenecks and rescue effects in a fluctuating population of golden-mantled ground squirrels (Spermophilus lateralis). Con. Gen. 12:285-296 (pdf).\r\n\r\nMejia , O., L. G. Herrera, B. May, R. A. Medellin,and J. J. Flores-Martinez. Effective population size dynamics of Myotis vivesi during the Pleistocene and Holocene climatic changes. Acta Chiropterologica. 13:33-40 (pdf).\r\n\r\nSchreier, A.D., D. Gille, B. Mahardja, and B. May. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon, Acipenser transmontanus. J. Appl. Ichthiol. 27:24-33 (pdf).\r\n\r\nWintzer, A.P., M.H. Meek, P.B. Moyle, and B. May. Ecological insights into the polyp stage of non-native hydrozoans in the San Francisco Estuary. Aquat. Ecol. 45:151161 (pdf).\r\n\r\n<strong>2010</strong>\r\n\r\nFloyd, C. H., J.J. Flores Martnez, , L.G. Herrera, O. Mejia, and B. May. Conserving the endangered Mexican fishing bat (Myotis vivesi): genetic variation indicates extensive gene flow among islands in the Gulf of California. Cons. Gen. 11:813-82 (pdf).\r\n\r\nIsrael, J.A. and B. May. Indirect estimates of breeding population size in the polyploid green sturgeon (Acipenser medirostris). Mol. Ecol. 19:1058-107 (pdf).\r\n\r\nIsrael, J.A. and B. May. Characterization and evaluation of polymorphic microsatellite markers in the anadromous fish Spirinchus thaleichthys. Cons. Gen. Res. online (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Nuclear and mtDNA lineage diversity in wild and cultured Pacific lion-paw scallop (Baja California Peninsula, Mexico). Mar. Biol. 157:27512767 (pdf).\r\n\r\n<strong>2009</strong>\r\n\r\nChen, Y., S. Reid, and B. May. Genetic relationships among tui chub populations in the Cow Head and surrounding basins of the Northwestern Great Basin. Cons. Gen. 10:101-114 (pdf).\r\n\r\nFinger, A.J., M.R. Stephens, N.W. Clipperton and B. May. Six diagnostic SNP markers for detecting introgression between cutthroat and rainbow trout. Mol. Ecol. Res. 9:759-763 (pdf).\r\n\r\nFisch, K., J.L. Petersen, M.R. Baerwald, and B.P. May. Characterization of 24 microsatellite loci in delta smelt, Hypomesus transpacificus, and their cross-species amplification in two other smelt species of the Osmeridae family. Mol. Ecol. Res. 9:405-408 (pdf).\r\n\r\nIsrael, J.A., K.J. Bando, E.C. Anderson, and B. May. Polyploid microsatellite data reveal stock complexity among estuarine North American green sturgeon (Acipenser medirostris). Can. J. Fish. Aqu. Sci. 66:1491-1504 (pdf).\r\n\r\nMeek, M., M. Baerwald, A. Wintzer, and B. May. Isolation and characterization of microsatellite loci in two non-native hydromedusae in the San Francisco Estuary: Maeotias marginata and Moerisia sp. Cons. Gen. Res. 1:205-208 (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Thirty-seven additional microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and cross-amplification in other Pectinids. Cons. Gen. Res. 1:101-105 (pdf).\r\n\r\nSimmons, R. E., P. Lavretsky, and B. May. Introgressive Hybridization of Redband Trout in the Upper McCloud River Watershed. Trans. Am. Fish. Soc. 139:201-213 (pdf).\r\n\r\nStephens, M.R., N.W. Clipperton, and B.P. May. Subspecies-informative TaqMan probes for evaluating introgression between native golden trout and introduced rainbow trout. Mol. Ecol. Res. 9:339-343 (pdf).\r\n\r\n<strong>Earlier publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications','','inherit','closed','open','','12-revision-v1','','','2014-03-07 18:51:37','2014-03-07 18:51:37','',12,'http://localhost/~psalessi/Genetics_Lab/12-revision-v1/',0,'revision','',0),(297,1,'2014-03-07 18:53:12','2014-03-07 18:53:12','<strong>2013</strong>\r\n\r\nChen, Y., S. Parmenter, and B. May. Genetic characterization of variation and management implications for the endangeredMohave tui chub. Cons. Gen. 14:11-20. (<a title=\"Publications_files/2013 - Chen Mohave tui chub.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2013%20-%20Chen%20Mohave%20tui%20chub.pdf\">pdf</a>)\r\n\r\nMeek, M. H., A. P. Wintzer, N. Shepherd, and B. May. Genetic diversity and reproductive mode in two non-native hydromedusae, <em>Maeotias marginata</em> and <em>Moerisia</em> sp., in the upper San Francisco Estuary, California. Biol. Invasions. 15(1): 199-212. (<a title=\"Publications_files/2012 Meek et al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Meek%20et%20al.pdf\">pdf</a>)\r\n\r\nSchreier, A.D., B. Mahardja, and B. May. Hierarchical patterns of population structure in the endangered Fraser River white sturgeon, Acipenser transmontanus, and implications for conservation. Can. J. Fish. &amp; Aquat. Sci. 69:1968-1980. (<a title=\"Publications_files/2013 - Schreier Fraser river white sturgeon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2013%20-%20Schreier%20Fraser%20river%20white%20sturgeon.pdf\">pdf</a>)\r\n\r\n<strong>2012</strong>\r\n\r\nBaerwald, M.R., B.M. Schreier, G. Schumer, and B. May. Detection of Threatened Delta Smelt in the Gut Contents of the Invasive Mississippi Silverside in the San Francisco Estuary Using TaqMan Assays. Trans. Am. Fish. Soc. 141:1600-1607. (<a title=\"Publications_files/2012 - Baerwald gut contents.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20-%20Baerwald%20gut%20contents.pdf\">pdf</a>)\r\n\r\nMahardja, B., B. May, and M. Baerwald. Characterization of 36 additional microsatellite loci in splittail (<em>Pogonichthys macrolepidotus</em>) and cross-amplification in five other native Californian cyprinid species. Cons. Gen. Res. 4(4):917-921. (<a title=\"Publications_files/2012 Meek et al - Climate change.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Mahardja%20et%20al.%20-%20Cyprinid%20Microsat%20note.pdf\">pdf</a>)\r\n\r\nMeek, M., A. Wintzer, W. Wetzel, and B. May. Climate change likely to facilitate the invasion of the non-native hydroid, Cordylophora caspia, in the San Francisco Estuary, CA. PLos ONE. 7(10): e46373. doi:10.1371/journal.pone.0046373 (<a title=\"Publications_files/2012 - Petersen lion paw scallop.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Meek%20et%20al%20-%20Climate%20change.pdf\">pdf</a>)\r\n\r\nPetersen, J.L., M.R. Baerwald, J.M. Ibarra, and B. May. A first-generation linkage map of the Pacific lion-paw scallop: initial evidence of QTL for size traits and markers linked to orange shell color. Aquaculture. 350-353:200-209. (<a title=\"Publications_files/2012 - Petersen lion paw scallop.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/Publications_files/2012%20-%20Petersen%20lion%20paw%20scallop.pdf\">pdf</a>)\r\n\r\nSchreier, A. D., J. Rodzen, S. Ireland, and B. May. Genetic techniques inform conservation aquaculture of the endangered Kootenai River white sturgeon, Acipenser transmontanus. Endang. Species Res. 16:65-75. (pdf)\r\n\r\nSukmanomon, S., W. Kamonrat, S. Poompuang, T. Nguyen, D. M. Bartley, B. May, U. Na-Nakorn. Genetic changes, intra- and interspecific introgression in farmed Nile tilapia in Thailand. Aquaculture 324-325:44-54. (pdf)\r\n\r\nVan Dam, A.R. and B. May. A new species of Dactylopius Costa (Dactylopius gracilipilus sp. nov.) (Hemiptera: Coccoidea: Dactylopiidae) from the Chihuahuan Desert, Texas, U.S.A. . Zootaxa 3573:33-39. (pdf)\r\n\r\n<strong>2011</strong>\r\n\r\nBaerwald, M., J. Petersen, R. Hedrick, G. Schisler, and B. May. A major effect quantitative trait locus for whirling disease resistance identified in rainbow trout (Oncorhynchus mykiss). Heredity. 106:920-926. (pdf)\r\n\r\nBaerwald, M., G. Schumer, B. Schreier, and B. May. TaqMan assays for the genetic identification of delta smelt (Hypomesus transpacificus) and wakasagi smelt (Hypomesus nipponensis). Mol. Ecol. Res. 11:784-785. (pdf)\r\n\r\nBaerwald, M.R., M. R. Stephens, K. S. Bork, M. H. Meek, K. Tomalty, and B. May. 2011. Spring-run Chinook salmon genetic management plan  San Joaquin River Restoration Program. 118 pgs. Scientific genetic management plan for the CA Dept. of Fish and Game. (pdf)\r\n\r\nFinger, A. J., E. C. Anderson, M. R. Stephens, and B. P. May. Application of a method for estimating effective population size and admixture using diagnostic SNPs: Implications for conservation of threatened Paiute cutthroat trout (Oncorhynchus clarkii seleniris) in Silver King Creek, California. Can. J. Fish. Aquat. Sci. 68:1369-1386 (pdf).\r\n\r\nFisch, K.M., J.M. Henderson, R.S. Burton, and B. May. Population genetics and conservation implications for the endangered delta smelt in the San Francisco Bay-Delta. Conservation Genetics. 12:1421-1434 (pdf).\r\n\r\nFloyd, C. H., D.H. Van Vuren, K.R. Crooks, K.L. Jones, D.K. Garcelon, N.M. Belfiore, J.W. Dragoo, and B. May. Genetic differentiation of island spotted skunks, Spilogale gracilis amphiala. J. Mam. 92(1):148-158 (pdf).\r\n\r\nMcEachern, M.B., D.H. Van Vuren, C.H. Floyd, B. May, and J.M. Eadie.Bottlenecks and rescue effects in a fluctuating population of golden-mantled ground squirrels (Spermophilus lateralis). Con. Gen. 12:285-296 (pdf).\r\n\r\nMejia , O., L. G. Herrera, B. May, R. A. Medellin,and J. J. Flores-Martinez. Effective population size dynamics of Myotis vivesi during the Pleistocene and Holocene climatic changes. Acta Chiropterologica. 13:33-40 (pdf).\r\n\r\nSchreier, A.D., D. Gille, B. Mahardja, and B. May. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon, Acipenser transmontanus. J. Appl. Ichthiol. 27:24-33 (pdf).\r\n\r\nWintzer, A.P., M.H. Meek, P.B. Moyle, and B. May. Ecological insights into the polyp stage of non-native hydrozoans in the San Francisco Estuary. Aquat. Ecol. 45:151161 (pdf).\r\n\r\n<strong>2010</strong>\r\n\r\nFloyd, C. H., J.J. Flores Martnez, , L.G. Herrera, O. Mejia, and B. May. Conserving the endangered Mexican fishing bat (Myotis vivesi): genetic variation indicates extensive gene flow among islands in the Gulf of California. Cons. Gen. 11:813-82 (pdf).\r\n\r\nIsrael, J.A. and B. May. Indirect estimates of breeding population size in the polyploid green sturgeon (Acipenser medirostris). Mol. Ecol. 19:1058-107 (pdf).\r\n\r\nIsrael, J.A. and B. May. Characterization and evaluation of polymorphic microsatellite markers in the anadromous fish Spirinchus thaleichthys. Cons. Gen. Res. online (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Nuclear and mtDNA lineage diversity in wild and cultured Pacific lion-paw scallop (Baja California Peninsula, Mexico). Mar. Biol. 157:27512767 (pdf).\r\n\r\n<strong>2009</strong>\r\n\r\nChen, Y., S. Reid, and B. May. Genetic relationships among tui chub populations in the Cow Head and surrounding basins of the Northwestern Great Basin. Cons. Gen. 10:101-114 (pdf).\r\n\r\nFinger, A.J., M.R. Stephens, N.W. Clipperton and B. May. Six diagnostic SNP markers for detecting introgression between cutthroat and rainbow trout. Mol. Ecol. Res. 9:759-763 (pdf).\r\n\r\nFisch, K., J.L. Petersen, M.R. Baerwald, and B.P. May. Characterization of 24 microsatellite loci in delta smelt, Hypomesus transpacificus, and their cross-species amplification in two other smelt species of the Osmeridae family. Mol. Ecol. Res. 9:405-408 (pdf).\r\n\r\nIsrael, J.A., K.J. Bando, E.C. Anderson, and B. May. Polyploid microsatellite data reveal stock complexity among estuarine North American green sturgeon (Acipenser medirostris). Can. J. Fish. Aqu. Sci. 66:1491-1504 (pdf).\r\n\r\nMeek, M., M. Baerwald, A. Wintzer, and B. May. Isolation and characterization of microsatellite loci in two non-native hydromedusae in the San Francisco Estuary: Maeotias marginata and Moerisia sp. Cons. Gen. Res. 1:205-208 (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Thirty-seven additional microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and cross-amplification in other Pectinids. Cons. Gen. Res. 1:101-105 (pdf).\r\n\r\nSimmons, R. E., P. Lavretsky, and B. May. Introgressive Hybridization of Redband Trout in the Upper McCloud River Watershed. Trans. Am. Fish. Soc. 139:201-213 (pdf).\r\n\r\nStephens, M.R., N.W. Clipperton, and B.P. May. Subspecies-informative TaqMan probes for evaluating introgression between native golden trout and introduced rainbow trout. Mol. Ecol. Res. 9:339-343 (pdf).\r\n\r\n<strong>Earlier publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications','','inherit','closed','open','','12-revision-v1','','','2014-03-07 18:53:12','2014-03-07 18:53:12','',12,'http://localhost/~psalessi/Genetics_Lab/12-revision-v1/',0,'revision','',0),(298,1,'2014-03-07 18:54:06','2014-03-07 18:54:06','<strong>2013</strong>\r\n\r\nChen, Y., S. Parmenter, and B. May. Genetic characterization of variation and management implications for the endangeredMohave tui chub. Cons. Gen. 14:11-20. (<a title=\"Publications_files/2013 - Chen Mohave tui chub.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2013%20-%20Chen%20Mohave%20tui%20chub.pdf\">pdf</a>)\r\n\r\nMeek, M. H., A. P. Wintzer, N. Shepherd, and B. May. Genetic diversity and reproductive mode in two non-native hydromedusae, <em>Maeotias marginata</em> and <em>Moerisia</em> sp., in the upper San Francisco Estuary, California. Biol. Invasions. 15(1): 199-212. (<a title=\"Publications_files/2012 Meek et al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Meek%20et%20al.pdf\">pdf</a>)\r\n\r\nSchreier, A.D., B. Mahardja, and B. May. Hierarchical patterns of population structure in the endangered Fraser River white sturgeon, Acipenser transmontanus, and implications for conservation. Can. J. Fish. &amp; Aquat. Sci. 69:1968-1980. (<a title=\"Publications_files/2013 - Schreier Fraser river white sturgeon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2013%20-%20Schreier%20Fraser%20river%20white%20sturgeon.pdf\">pdf</a>)\r\n\r\n<strong>2012</strong>\r\n\r\nBaerwald, M.R., B.M. Schreier, G. Schumer, and B. May. Detection of Threatened Delta Smelt in the Gut Contents of the Invasive Mississippi Silverside in the San Francisco Estuary Using TaqMan Assays. Trans. Am. Fish. Soc. 141:1600-1607. (<a title=\"Publications_files/2012 - Baerwald gut contents.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20-%20Baerwald%20gut%20contents.pdf\">pdf</a>)\r\n\r\nMahardja, B., B. May, and M. Baerwald. Characterization of 36 additional microsatellite loci in splittail (<em>Pogonichthys macrolepidotus</em>) and cross-amplification in five other native Californian cyprinid species. Cons. Gen. Res. 4(4):917-921. (<a title=\"Publications_files/2012 Meek et al - Climate change.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Mahardja%20et%20al.%20-%20Cyprinid%20Microsat%20note.pdf\">pdf</a>)\r\n\r\nMeek, M., A. Wintzer, W. Wetzel, and B. May. Climate change likely to facilitate the invasion of the non-native hydroid, Cordylophora caspia, in the San Francisco Estuary, CA. PLos ONE. 7(10): e46373. doi:10.1371/journal.pone.0046373 (<a title=\"Publications_files/2012 - Petersen lion paw scallop.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Meek%20et%20al%20-%20Climate%20change.pdf\">pdf</a>)\r\n\r\nPetersen, J.L., M.R. Baerwald, J.M. Ibarra, and B. May. A first-generation linkage map of the Pacific lion-paw scallop: initial evidence of QTL for size traits and markers linked to orange shell color. Aquaculture. 350-353:200-209. (<a title=\"Publications_files/2012 - Petersen lion paw scallop.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20-%20Petersen%20lion%20paw%20scallop.pdf\">pdf</a>)\r\n\r\nSchreier, A. D., J. Rodzen, S. Ireland, and B. May. Genetic techniques inform conservation aquaculture of the endangered Kootenai River white sturgeon, Acipenser transmontanus. Endang. Species Res. 16:65-75. (pdf)\r\n\r\nSukmanomon, S., W. Kamonrat, S. Poompuang, T. Nguyen, D. M. Bartley, B. May, U. Na-Nakorn. Genetic changes, intra- and interspecific introgression in farmed Nile tilapia in Thailand. Aquaculture 324-325:44-54. (pdf)\r\n\r\nVan Dam, A.R. and B. May. A new species of Dactylopius Costa (Dactylopius gracilipilus sp. nov.) (Hemiptera: Coccoidea: Dactylopiidae) from the Chihuahuan Desert, Texas, U.S.A. . Zootaxa 3573:33-39. (pdf)\r\n\r\n<strong>2011</strong>\r\n\r\nBaerwald, M., J. Petersen, R. Hedrick, G. Schisler, and B. May. A major effect quantitative trait locus for whirling disease resistance identified in rainbow trout (Oncorhynchus mykiss). Heredity. 106:920-926. (pdf)\r\n\r\nBaerwald, M., G. Schumer, B. Schreier, and B. May. TaqMan assays for the genetic identification of delta smelt (Hypomesus transpacificus) and wakasagi smelt (Hypomesus nipponensis). Mol. Ecol. Res. 11:784-785. (pdf)\r\n\r\nBaerwald, M.R., M. R. Stephens, K. S. Bork, M. H. Meek, K. Tomalty, and B. May. 2011. Spring-run Chinook salmon genetic management plan  San Joaquin River Restoration Program. 118 pgs. Scientific genetic management plan for the CA Dept. of Fish and Game. (pdf)\r\n\r\nFinger, A. J., E. C. Anderson, M. R. Stephens, and B. P. May. Application of a method for estimating effective population size and admixture using diagnostic SNPs: Implications for conservation of threatened Paiute cutthroat trout (Oncorhynchus clarkii seleniris) in Silver King Creek, California. Can. J. Fish. Aquat. Sci. 68:1369-1386 (pdf).\r\n\r\nFisch, K.M., J.M. Henderson, R.S. Burton, and B. May. Population genetics and conservation implications for the endangered delta smelt in the San Francisco Bay-Delta. Conservation Genetics. 12:1421-1434 (pdf).\r\n\r\nFloyd, C. H., D.H. Van Vuren, K.R. Crooks, K.L. Jones, D.K. Garcelon, N.M. Belfiore, J.W. Dragoo, and B. May. Genetic differentiation of island spotted skunks, Spilogale gracilis amphiala. J. Mam. 92(1):148-158 (pdf).\r\n\r\nMcEachern, M.B., D.H. Van Vuren, C.H. Floyd, B. May, and J.M. Eadie.Bottlenecks and rescue effects in a fluctuating population of golden-mantled ground squirrels (Spermophilus lateralis). Con. Gen. 12:285-296 (pdf).\r\n\r\nMejia , O., L. G. Herrera, B. May, R. A. Medellin,and J. J. Flores-Martinez. Effective population size dynamics of Myotis vivesi during the Pleistocene and Holocene climatic changes. Acta Chiropterologica. 13:33-40 (pdf).\r\n\r\nSchreier, A.D., D. Gille, B. Mahardja, and B. May. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon, Acipenser transmontanus. J. Appl. Ichthiol. 27:24-33 (pdf).\r\n\r\nWintzer, A.P., M.H. Meek, P.B. Moyle, and B. May. Ecological insights into the polyp stage of non-native hydrozoans in the San Francisco Estuary. Aquat. Ecol. 45:151161 (pdf).\r\n\r\n<strong>2010</strong>\r\n\r\nFloyd, C. H., J.J. Flores Martnez, , L.G. Herrera, O. Mejia, and B. May. Conserving the endangered Mexican fishing bat (Myotis vivesi): genetic variation indicates extensive gene flow among islands in the Gulf of California. Cons. Gen. 11:813-82 (pdf).\r\n\r\nIsrael, J.A. and B. May. Indirect estimates of breeding population size in the polyploid green sturgeon (Acipenser medirostris). Mol. Ecol. 19:1058-107 (pdf).\r\n\r\nIsrael, J.A. and B. May. Characterization and evaluation of polymorphic microsatellite markers in the anadromous fish Spirinchus thaleichthys. Cons. Gen. Res. online (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Nuclear and mtDNA lineage diversity in wild and cultured Pacific lion-paw scallop (Baja California Peninsula, Mexico). Mar. Biol. 157:27512767 (pdf).\r\n\r\n<strong>2009</strong>\r\n\r\nChen, Y., S. Reid, and B. May. Genetic relationships among tui chub populations in the Cow Head and surrounding basins of the Northwestern Great Basin. Cons. Gen. 10:101-114 (pdf).\r\n\r\nFinger, A.J., M.R. Stephens, N.W. Clipperton and B. May. Six diagnostic SNP markers for detecting introgression between cutthroat and rainbow trout. Mol. Ecol. Res. 9:759-763 (pdf).\r\n\r\nFisch, K., J.L. Petersen, M.R. Baerwald, and B.P. May. Characterization of 24 microsatellite loci in delta smelt, Hypomesus transpacificus, and their cross-species amplification in two other smelt species of the Osmeridae family. Mol. Ecol. Res. 9:405-408 (pdf).\r\n\r\nIsrael, J.A., K.J. Bando, E.C. Anderson, and B. May. Polyploid microsatellite data reveal stock complexity among estuarine North American green sturgeon (Acipenser medirostris). Can. J. Fish. Aqu. Sci. 66:1491-1504 (pdf).\r\n\r\nMeek, M., M. Baerwald, A. Wintzer, and B. May. Isolation and characterization of microsatellite loci in two non-native hydromedusae in the San Francisco Estuary: Maeotias marginata and Moerisia sp. Cons. Gen. Res. 1:205-208 (pdf).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Thirty-seven additional microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and cross-amplification in other Pectinids. Cons. Gen. Res. 1:101-105 (pdf).\r\n\r\nSimmons, R. E., P. Lavretsky, and B. May. Introgressive Hybridization of Redband Trout in the Upper McCloud River Watershed. Trans. Am. Fish. Soc. 139:201-213 (pdf).\r\n\r\nStephens, M.R., N.W. Clipperton, and B.P. May. Subspecies-informative TaqMan probes for evaluating introgression between native golden trout and introduced rainbow trout. Mol. Ecol. Res. 9:339-343 (pdf).\r\n\r\n<strong>Earlier publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications','','inherit','closed','open','','12-revision-v1','','','2014-03-07 18:54:06','2014-03-07 18:54:06','',12,'http://localhost/~psalessi/Genetics_Lab/12-revision-v1/',0,'revision','',0),(299,1,'2014-03-07 19:05:51','2014-03-07 19:05:51','<strong>2013</strong>\r\n\r\nChen, Y., S. Parmenter, and B. May. Genetic characterization of variation and management implications for the endangeredMohave tui chub. Cons. Gen. 14:11-20. (<a title=\"Publications_files/2013 - Chen Mohave tui chub.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2013%20-%20Chen%20Mohave%20tui%20chub.pdf\">pdf</a>)\r\n\r\nMeek, M. H., A. P. Wintzer, N. Shepherd, and B. May. Genetic diversity and reproductive mode in two non-native hydromedusae, <em>Maeotias marginata</em> and <em>Moerisia</em> sp., in the upper San Francisco Estuary, California. Biol. Invasions. 15(1): 199-212. (<a title=\"Publications_files/2012 Meek et al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Meek%20et%20al.pdf\">pdf</a>)\r\n\r\nSchreier, A.D., B. Mahardja, and B. May. Hierarchical patterns of population structure in the endangered Fraser River white sturgeon, Acipenser transmontanus, and implications for conservation. Can. J. Fish. &amp; Aquat. Sci. 69:1968-1980. (<a title=\"Publications_files/2013 - Schreier Fraser river white sturgeon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2013%20-%20Schreier%20Fraser%20river%20white%20sturgeon.pdf\">pdf</a>)\r\n\r\n<strong>2012</strong>\r\n\r\nBaerwald, M.R., B.M. Schreier, G. Schumer, and B. May. Detection of Threatened Delta Smelt in the Gut Contents of the Invasive Mississippi Silverside in the San Francisco Estuary Using TaqMan Assays. Trans. Am. Fish. Soc. 141:1600-1607. (<a title=\"Publications_files/2012 - Baerwald gut contents.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20-%20Baerwald%20gut%20contents.pdf\">pdf</a>)\r\n\r\nMahardja, B., B. May, and M. Baerwald. Characterization of 36 additional microsatellite loci in splittail (<em>Pogonichthys macrolepidotus</em>) and cross-amplification in five other native Californian cyprinid species. Cons. Gen. Res. 4(4):917-921. (<a title=\"Publications_files/2012 Meek et al - Climate change.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Mahardja%20et%20al.%20-%20Cyprinid%20Microsat%20note.pdf\">pdf</a>)\r\n\r\nMeek, M., A. Wintzer, W. Wetzel, and B. May. Climate change likely to facilitate the invasion of the non-native hydroid, Cordylophora caspia, in the San Francisco Estuary, CA. PLos ONE. 7(10): e46373. doi:10.1371/journal.pone.0046373 (<a title=\"Publications_files/2012 - Petersen lion paw scallop.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Meek%20et%20al%20-%20Climate%20change.pdf\">pdf</a>)\r\n\r\nPetersen, J.L., M.R. Baerwald, J.M. Ibarra, and B. May. A first-generation linkage map of the Pacific lion-paw scallop: initial evidence of QTL for size traits and markers linked to orange shell color. Aquaculture. 350-353:200-209. (<a title=\"Publications_files/2012 - Petersen lion paw scallop.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20-%20Petersen%20lion%20paw%20scallop.pdf\">pdf</a>)\r\n\r\nSchreier, A. D., J. Rodzen, S. Ireland, and B. May. Genetic techniques inform conservation aquaculture of the endangered Kootenai River white sturgeon, Acipenser transmontanus. Endang. Species Res. 16:65-75. (<a title=\"Publications_files/2012 Schreier et al. - Kootenai.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Schreier%20et%20al.%20-%20Kootenai.pdf \">pdf</a>)\r\n\r\nSukmanomon, S., W. Kamonrat, S. Poompuang, T. Nguyen, D. M. Bartley, B. May, U. Na-Nakorn. Genetic changes, intra- and interspecific introgression in farmed Nile tilapia in Thailand. Aquaculture 324-325:44-54. (<a title=\"Publications_files/2012 - Sukmanomon Tilapia.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20-%20Sukmanomon%20%20Tilapia.pdf\">pdf</a>)\r\n\r\nVan Dam, A.R. and B. May. A new species of Dactylopius Costa (Dactylopius gracilipilus sp. nov.) (Hemiptera: Coccoidea: Dactylopiidae) from the Chihuahuan Desert, Texas, U.S.A. . Zootaxa 3573:33-39. (<a title=\"Publications_files/2012 - Van Dam new species.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20-%20Van%20Dam%20new%20species.pdf\">pdf</a>)\r\n\r\n<strong>2011</strong>\r\n\r\nBaerwald, M., J. Petersen, R. Hedrick, G. Schisler, and B. May. A major effect quantitative trait locus for whirling disease resistance identified in rainbow trout (Oncorhynchus mykiss). Heredity. 106:920-926. (<a title=\"Publications_files/2011 - Baerwald et al. - QTL whirling.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20-%20Baerwald%20et%20al.%20-%20QTL%20whirling.pdf\">pdf</a>)\r\n\r\nBaerwald, M., G. Schumer, B. Schreier, and B. May. TaqMan assays for the genetic identification of delta smelt (Hypomesus transpacificus) and wakasagi smelt (Hypomesus nipponensis). Mol. Ecol. Res. 11:784-785. (<a title=\"Publications_files/2011 - Baerwald TaqMan assays.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20-%20Baerwald%20TaqMan%20assays.pdf\">pdf</a>)\r\n\r\nBaerwald, M.R., M. R. Stephens, K. S. Bork, M. H. Meek, K. Tomalty, and B. May. 2011. Spring-run Chinook salmon genetic management plan  San Joaquin River Restoration Program. 118 pgs. Scientific genetic management plan for the CA Dept. of Fish and Game. (<a title=\"Publications_files/GMP_SpringRunChinook_SJR_UCD_GVL_FINAL.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/GMP_SpringRunChinook_SJR_UCD_GVL_FINAL.pdf\">pdf</a>)\r\n\r\nFinger, A. J., E. C. Anderson, M. R. Stephens, and B. P. May. Application of a method for estimating effective population size and admixture using diagnostic SNPs: Implications for conservation of threatened Paiute cutthroat trout (Oncorhynchus clarkii seleniris) in Silver King Creek, California. Can. J. Fish. Aquat. Sci. 68:1369-1386 (<a title=\"Publications_files/2011 - Finger et al. - Effective pop size of Paiute cutthroat.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20-%20Finger%20et%20al.%20-%20Effective%20pop%20size%20of%20Paiute%20cutthroat.pdf\">pdf</a>).\r\n\r\nFisch, K.M., J.M. Henderson, R.S. Burton, and B. May. Population genetics and conservation implications for the endangered delta smelt in the San Francisco Bay-Delta. Conservation Genetics. 12:1421-1434 (<a title=\"Publications_files/2011 Fisch et al. Delta smelt pop gen.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20Fisch%20et%20al.%20Delta%20smelt%20pop%20gen.pdf\">pdf</a>).\r\n\r\nFloyd, C. H., D.H. Van Vuren, K.R. Crooks, K.L. Jones, D.K. Garcelon, N.M. Belfiore, J.W. Dragoo, and B. May. Genetic differentiation of island spotted skunks, Spilogale gracilis amphiala. J. Mam. 92(1):148-158 (<a title=\"Publications_files/2011 Floyd et al. skunks.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20Floyd%20et%20al.%20skunks.pdf\">pdf</a>).\r\n\r\nMcEachern, M.B., D.H. Van Vuren, C.H. Floyd, B. May, and J.M. Eadie.Bottlenecks and rescue effects in a fluctuating population of golden-mantled ground squirrels (Spermophilus lateralis). Con. Gen. 12:285-296 (<a title=\"Publications_files/2011 McEachern ground squirrels.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20McEachern%20ground%20squirrels.pdf\">pdf</a>).\r\n\r\nMejia , O., L. G. Herrera, B. May, R. A. Medellin,and J. J. Flores-Martinez. Effective population size dynamics of Myotis vivesi during the Pleistocene and Holocene climatic changes. Acta Chiropterologica. 13:33-40 (<a title=\"Publications_files/2011 Mejia et al .pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20Mejia%20et%20al%20.pdf\">pdf</a>).\r\n\r\nSchreier, A.D., D. Gille, B. Mahardja, and B. May. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon, Acipenser transmontanus. J. Appl. Ichthiol. 27:24-33 (<a title=\"Publications_files/2011 Schreier et al. - Octoploid origin sturgeon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20Schreier%20et%20al.%20-%20Octoploid%20origin%20sturgeon.pdf\">pdf</a>).\r\n\r\nWintzer, A.P., M.H. Meek, P.B. Moyle, and B. May. Ecological insights into the polyp stage of non-native hydrozoans in the San Francisco Estuary. Aquat. Ecol. 45:151161 (<a title=\"Publications_files/2011 Wintzer - Aquatic Ecology.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20Wintzer%20-%20Aquatic%20Ecology.pdf\">pdf</a>).\r\n\r\n<strong>2010</strong>\r\n\r\nFloyd, C. H., J.J. Flores Martnez, , L.G. Herrera, O. Mejia, and B. May. Conserving the endangered Mexican fishing bat (Myotis vivesi): genetic variation indicates extensive gene flow among islands in the Gulf of California. Cons. Gen. 11:813-82 (<a title=\"Publications_files/2010 Floyd et al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2010%20Floyd%20et%20al.pdf\">pdf</a>).\r\n\r\nIsrael, J.A. and B. May. Indirect estimates of breeding population size in the polyploid green sturgeon (Acipenser medirostris). Mol. Ecol. 19:1058-107 (<a title=\"Publications_files/2010 Israel Mol. Ecol..pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2010%20Israel%20Mol.%20Ecol..pdf\">pdf</a>).\r\n\r\nIsrael, J.A. and B. May. Characterization and evaluation of polymorphic microsatellite markers in the anadromous fish Spirinchus thaleichthys. Cons. Gen. Res. online (<a title=\"Publications_files/2010 Israel &amp; May - Spirinchus microsat.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2010%20Israel%20%26%20May%20-%20Spirinchus%20microsat.pdf\">pdf</a>).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Nuclear and mtDNA lineage diversity in wild and cultured Pacific lion-paw scallop (Baja California Peninsula, Mexico). Mar. Biol. 157:27512767 (<a title=\"Publications_files/2010 Petersen et al. wild and cultured.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2010%20Petersen%20et%20al.%20wild%20and%20cultured.pdf\">pdf</a>).\r\n\r\n<strong>2009</strong>\r\n\r\nChen, Y., S. Reid, and B. May. Genetic relationships among tui chub populations in the Cow Head and surrounding basins of the Northwestern Great Basin. Cons. Gen. 10:101-114 (<a title=\"Publications_files/2009_Chen_great_basin_tui.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009_Chen_great_basin_tui.pdf\">pdf</a>).\r\n\r\nFinger, A.J., M.R. Stephens, N.W. Clipperton and B. May. Six diagnostic SNP markers for detecting introgression between cutthroat and rainbow trout. Mol. Ecol. Res. 9:759-763 (<a title=\"Publications_files/2009 - Finger_six_SNPs.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009%20-%20Finger_six_SNPs.pdf\">pdf</a>).\r\n\r\nFisch, K., J.L. Petersen, M.R. Baerwald, and B.P. May. Characterization of 24 microsatellite loci in delta smelt, Hypomesus transpacificus, and their cross-species amplification in two other smelt species of the Osmeridae family. Mol. Ecol. Res. 9:405-408 (<a title=\"Publications_files/2009_Fisch_primer_smelt.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009_Fisch_primer_smelt.pdf\">pdf</a>).\r\n\r\nIsrael, J.A., K.J. Bando, E.C. Anderson, and B. May. Polyploid microsatellite data reveal stock complexity among estuarine North American green sturgeon (Acipenser medirostris). Can. J. Fish. Aqu. Sci. 66:1491-1504 (<a title=\"Publications_files/2009 Israel green sturgeon stock.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009%20Israel%20green%20sturgeon%20stock.pdf\">pdf</a>).\r\n\r\nMeek, M., M. Baerwald, A. Wintzer, and B. May. Isolation and characterization of microsatellite loci in two non-native hydromedusae in the San Francisco Estuary: Maeotias marginata and Moerisia sp. Cons. Gen. Res. 1:205-208 (<a title=\"Publications_files/2009 - Meek et al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009%20-%20Meek%20et%20al.pdf\">pdf</a>).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Thirty-seven additional microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and cross-amplification in other Pectinids. Cons. Gen. Res. 1:101-105 (<a title=\"Publications_files/Petersen et al. 2009 - microsatellite.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/Petersen%20et%20al.%202009%20-%20microsatellite.pdf\">pdf</a>).\r\n\r\nSimmons, R. E., P. Lavretsky, and B. May. Introgressive Hybridization of Redband Trout in the Upper McCloud River Watershed. Trans. Am. Fish. Soc. 139:201-213 (<a title=\"Publications_files/2009 - Simmons_redband_introgression.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009%20-%20Simmons_redband_introgression.pdf\">pdf</a>).\r\n\r\nStephens, M.R., N.W. Clipperton, and B.P. May. Subspecies-informative TaqMan probes for evaluating introgression between native golden trout and introduced rainbow trout. Mol. Ecol. Res. 9:339-343 (<a title=\"Publications_files/2009_Stephens_SNP_primers.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009_Stephens_SNP_primers.pdf\">pdf</a>).\r\n\r\n<strong>Earlier publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications','','inherit','closed','open','','12-revision-v1','','','2014-03-07 19:05:51','2014-03-07 19:05:51','',12,'http://localhost/~psalessi/Genetics_Lab/12-revision-v1/',0,'revision','',0),(300,1,'2014-03-07 19:09:18','2014-03-07 19:09:18','<strong>2013</strong>\r\n\r\nChen, Y., S. Parmenter, and B. May. Genetic characterization of variation and management implications for the endangeredMohave tui chub. Cons. Gen. 14:11-20. (<a title=\"Publications_files/2013 - Chen Mohave tui chub.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2013%20-%20Chen%20Mohave%20tui%20chub.pdf\">pdf</a>)\r\n\r\nMeek, M. H., A. P. Wintzer, N. Shepherd, and B. May. Genetic diversity and reproductive mode in two non-native hydromedusae, <em>Maeotias marginata</em> and <em>Moerisia</em> sp., in the upper San Francisco Estuary, California. Biol. Invasions. 15(1): 199-212. (<a title=\"Publications_files/2012 Meek et al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Meek%20et%20al.pdf\">pdf</a>)\r\n\r\nSchreier, A.D., B. Mahardja, and B. May. Hierarchical patterns of population structure in the endangered Fraser River white sturgeon, Acipenser transmontanus, and implications for conservation. Can. J. Fish. &amp; Aquat. Sci. 69:1968-1980. (<a title=\"Publications_files/2013 - Schreier Fraser river white sturgeon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2013%20-%20Schreier%20Fraser%20river%20white%20sturgeon.pdf\">pdf</a>)\r\n\r\n<strong>2012</strong>\r\n\r\nBaerwald, M.R., B.M. Schreier, G. Schumer, and B. May. Detection of Threatened Delta Smelt in the Gut Contents of the Invasive Mississippi Silverside in the San Francisco Estuary Using TaqMan Assays. Trans. Am. Fish. Soc. 141:1600-1607. (<a title=\"Publications_files/2012 - Baerwald gut contents.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20-%20Baerwald%20gut%20contents.pdf\">pdf</a>)\r\n\r\nMahardja, B., B. May, and M. Baerwald. Characterization of 36 additional microsatellite loci in splittail (<em>Pogonichthys macrolepidotus</em>) and cross-amplification in five other native Californian cyprinid species. Cons. Gen. Res. 4(4):917-921. (<a title=\"Publications_files/2012 Meek et al - Climate change.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Mahardja%20et%20al.%20-%20Cyprinid%20Microsat%20note.pdf\">pdf</a>)\r\n\r\nMeek, M., A. Wintzer, W. Wetzel, and B. May. Climate change likely to facilitate the invasion of the non-native hydroid, Cordylophora caspia, in the San Francisco Estuary, CA. PLos ONE. 7(10): e46373. doi:10.1371/journal.pone.0046373 (<a title=\"Publications_files/2012 - Petersen lion paw scallop.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Meek%20et%20al%20-%20Climate%20change.pdf\">pdf</a>)\r\n\r\nPetersen, J.L., M.R. Baerwald, J.M. Ibarra, and B. May. A first-generation linkage map of the Pacific lion-paw scallop: initial evidence of QTL for size traits and markers linked to orange shell color. Aquaculture. 350-353:200-209. (<a title=\"Publications_files/2012 - Petersen lion paw scallop.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20-%20Petersen%20lion%20paw%20scallop.pdf\">pdf</a>)\r\n\r\nSchreier, A. D., J. Rodzen, S. Ireland, and B. May. Genetic techniques inform conservation aquaculture of the endangered Kootenai River white sturgeon, <em>Acipenser transmontanus</em>. Endang. Species Res. 16:65-75. (<a title=\"Publications_files/2012 Schreier et al. - Kootenai.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20Schreier%20et%20al.%20-%20Kootenai.pdf \">pdf</a>)\r\n\r\nSukmanomon, S., W. Kamonrat, S. Poompuang, T. Nguyen, D. M. Bartley, B. May, U. Na-Nakorn. Genetic changes, intra- and interspecific introgression in farmed Nile tilapia in Thailand. Aquaculture 324-325:44-54. (<a title=\"Publications_files/2012 - Sukmanomon Tilapia.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20-%20Sukmanomon%20%20Tilapia.pdf\">pdf</a>)\r\n\r\nVan Dam, A.R. and B. May. A new species of Dactylopius Costa (<em>Dactylopius gracilipilus</em> sp. nov.) (Hemiptera: Coccoidea: Dactylopiidae) from the Chihuahuan Desert, Texas, U.S.A. . Zootaxa 3573:33-39. (<a title=\"Publications_files/2012 - Van Dam new species.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2012%20-%20Van%20Dam%20new%20species.pdf\">pdf</a>)\r\n\r\n<strong>2011</strong>\r\n\r\nBaerwald, M., J. Petersen, R. Hedrick, G. Schisler, and B. May. A major effect quantitative trait locus for whirling disease resistance identified in rainbow trout (<em>Oncorhynchus mykiss</em>). Heredity. 106:920-926. (<a title=\"Publications_files/2011 - Baerwald et al. - QTL whirling.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20-%20Baerwald%20et%20al.%20-%20QTL%20whirling.pdf\">pdf</a>)\r\n\r\nBaerwald, M., G. Schumer, B. Schreier, and B. May. TaqMan assays for the genetic identification of delta smelt (Hypomesus transpacificus) and wakasagi smelt (<em>Hypomesus nipponensis</em>). Mol. Ecol. Res. 11:784-785. (<a title=\"Publications_files/2011 - Baerwald TaqMan assays.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20-%20Baerwald%20TaqMan%20assays.pdf\">pdf</a>)\r\n\r\nBaerwald, M.R., M. R. Stephens, K. S. Bork, M. H. Meek, K. Tomalty, and B. May. 2011. Spring-run Chinook salmon genetic management plan  San Joaquin River Restoration Program. 118 pgs. Scientific genetic management plan for the CA Dept. of Fish and Game. (<a title=\"Publications_files/GMP_SpringRunChinook_SJR_UCD_GVL_FINAL.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/GMP_SpringRunChinook_SJR_UCD_GVL_FINAL.pdf\">pdf</a>)\r\n\r\nFinger, A. J., E. C. Anderson, M. R. Stephens, and B. P. May. Application of a method for estimating effective population size and admixture using diagnostic SNPs: Implications for conservation of threatened Paiute cutthroat trout (<em>Oncorhynchus clarkii seleniris</em>) in Silver King Creek, California. Can. J. Fish. Aquat. Sci. 68:1369-1386 (<a title=\"Publications_files/2011 - Finger et al. - Effective pop size of Paiute cutthroat.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20-%20Finger%20et%20al.%20-%20Effective%20pop%20size%20of%20Paiute%20cutthroat.pdf\">pdf</a>).\r\n\r\nFisch, K.M., J.M. Henderson, R.S. Burton, and B. May. Population genetics and conservation implications for the endangered delta smelt in the San Francisco Bay-Delta. Conservation Genetics. 12:1421-1434 (<a title=\"Publications_files/2011 Fisch et al. Delta smelt pop gen.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20Fisch%20et%20al.%20Delta%20smelt%20pop%20gen.pdf\">pdf</a>).\r\n\r\nFloyd, C. H., D.H. Van Vuren, K.R. Crooks, K.L. Jones, D.K. Garcelon, N.M. Belfiore, J.W. Dragoo, and B. May. Genetic differentiation of island spotted skunks, <em>Spilogale gracilis amphiala.</em> J. Mam. 92(1):148-158 (<a title=\"Publications_files/2011 Floyd et al. skunks.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20Floyd%20et%20al.%20skunks.pdf\">pdf</a>).\r\n\r\nMcEachern, M.B., D.H. Van Vuren, C.H. Floyd, B. May, and J.M. Eadie.Bottlenecks and rescue effects in a fluctuating population of golden-mantled ground squirrels (<em>Spermophilus lateralis</em>). Con. Gen. 12:285-296 (<a title=\"Publications_files/2011 McEachern ground squirrels.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20McEachern%20ground%20squirrels.pdf\">pdf</a>).\r\n\r\nMejia , O., L. G. Herrera, B. May, R. A. Medellin,and J. J. Flores-Martinez. Effective population size dynamics of Myotis vivesi during the Pleistocene and Holocene climatic changes. Acta Chiropterologica. 13:33-40 (<a title=\"Publications_files/2011 Mejia et al .pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20Mejia%20et%20al%20.pdf\">pdf</a>).\r\n\r\nSchreier, A.D., D. Gille, B. Mahardja, and B. May. Neutral markers confirm the octoploid origin and reveal spontaneous polyploidy in white sturgeon,<em> Acipenser transmontanus</em>. J. Appl. Ichthiol. 27:24-33 (<a title=\"Publications_files/2011 Schreier et al. - Octoploid origin sturgeon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20Schreier%20et%20al.%20-%20Octoploid%20origin%20sturgeon.pdf\">pdf</a>).\r\n\r\nWintzer, A.P., M.H. Meek, P.B. Moyle, and B. May. Ecological insights into the polyp stage of non-native hydrozoans in the San Francisco Estuary. Aquat. Ecol. 45:151161 (<a title=\"Publications_files/2011 Wintzer - Aquatic Ecology.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2011%20Wintzer%20-%20Aquatic%20Ecology.pdf\">pdf</a>).\r\n\r\n<strong>2010</strong>\r\n\r\nFloyd, C. H., J.J. Flores Martnez, , L.G. Herrera, O. Mejia, and B. May. Conserving the endangered Mexican fishing bat (<em>Myotis vivesi</em>): genetic variation indicates extensive gene flow among islands in the Gulf of California. Cons. Gen. 11:813-82 (<a title=\"Publications_files/2010 Floyd et al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2010%20Floyd%20et%20al.pdf\">pdf</a>).\r\n\r\nIsrael, J.A. and B. May. Indirect estimates of breeding population size in the polyploid green sturgeon (<em>Acipenser medirostris</em>). Mol. Ecol. 19:1058-107 (<a title=\"Publications_files/2010 Israel Mol. Ecol..pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2010%20Israel%20Mol.%20Ecol..pdf\">pdf</a>).\r\n\r\nIsrael, J.A. and B. May. Characterization and evaluation of polymorphic microsatellite markers in the anadromous fish <em>Spirinchus thaleichthys</em>. Cons. Gen. Res. online (<a title=\"Publications_files/2010 Israel &amp; May - Spirinchus microsat.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2010%20Israel%20%26%20May%20-%20Spirinchus%20microsat.pdf\">pdf</a>).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Nuclear and mtDNA lineage diversity in wild and cultured Pacific lion-paw scallop (Baja California Peninsula, Mexico). Mar. Biol. 157:27512767 (<a title=\"Publications_files/2010 Petersen et al. wild and cultured.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2010%20Petersen%20et%20al.%20wild%20and%20cultured.pdf\">pdf</a>).\r\n\r\n<strong>2009</strong>\r\n\r\nChen, Y., S. Reid, and B. May. Genetic relationships among tui chub populations in the Cow Head and surrounding basins of the Northwestern Great Basin. Cons. Gen. 10:101-114 (<a title=\"Publications_files/2009_Chen_great_basin_tui.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009_Chen_great_basin_tui.pdf\">pdf</a>).\r\n\r\nFinger, A.J., M.R. Stephens, N.W. Clipperton and B. May. Six diagnostic SNP markers for detecting introgression between cutthroat and rainbow trout. Mol. Ecol. Res. 9:759-763 (<a title=\"Publications_files/2009 - Finger_six_SNPs.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009%20-%20Finger_six_SNPs.pdf\">pdf</a>).\r\n\r\nFisch, K., J.L. Petersen, M.R. Baerwald, and B.P. May. Characterization of 24 microsatellite loci in delta smelt, <em>Hypomesus transpacificus</em>, and their cross-species amplification in two other smelt species of the Osmeridae family. Mol. Ecol. Res. 9:405-408 (<a title=\"Publications_files/2009_Fisch_primer_smelt.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009_Fisch_primer_smelt.pdf\">pdf</a>).\r\n\r\nIsrael, J.A., K.J. Bando, E.C. Anderson, and B. May. Polyploid microsatellite data reveal stock complexity among estuarine North American green sturgeon (<em>Acipenser medirostris</em>). Can. J. Fish. Aqu. Sci. 66:1491-1504 (<a title=\"Publications_files/2009 Israel green sturgeon stock.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009%20Israel%20green%20sturgeon%20stock.pdf\">pdf</a>).\r\n\r\nMeek, M., M. Baerwald, A. Wintzer, and B. May. Isolation and characterization of microsatellite loci in two non-native hydromedusae in the San Francisco Estuary: <em>Maeotias marginata</em> and <em>Moerisia</em> sp. Cons. Gen. Res. 1:205-208 (<a title=\"Publications_files/2009 - Meek et al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009%20-%20Meek%20et%20al.pdf\">pdf</a>).\r\n\r\nPetersen, J.L., J.M. Ibarra, and B. May. Thirty-seven additional microsatellite loci in the Pacific lion-paw scallop (<em>Nodipecten subnodosus</em>) and cross-amplification in other Pectinids. Cons. Gen. Res. 1:101-105 (<a title=\"Publications_files/Petersen et al. 2009 - microsatellite.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/Petersen%20et%20al.%202009%20-%20microsatellite.pdf\">pdf</a>).\r\n\r\nSimmons, R. E., P. Lavretsky, and B. May. Introgressive Hybridization of Redband Trout in the Upper McCloud River Watershed. Trans. Am. Fish. Soc. 139:201-213 (<a title=\"Publications_files/2009 - Simmons_redband_introgression.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009%20-%20Simmons_redband_introgression.pdf\">pdf</a>).\r\n\r\nStephens, M.R., N.W. Clipperton, and B.P. May. Subspecies-informative TaqMan probes for evaluating introgression between native golden trout and introduced rainbow trout. Mol. Ecol. Res. 9:339-343 (<a title=\"Publications_files/2009_Stephens_SNP_primers.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_files/2009_Stephens_SNP_primers.pdf\">pdf</a>).\r\n\r\n<strong>Earlier publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications','','inherit','closed','open','','12-revision-v1','','','2014-03-07 19:09:18','2014-03-07 19:09:18','',12,'http://localhost/~psalessi/Genetics_Lab/12-revision-v1/',0,'revision','',0),(301,1,'2014-03-07 19:54:53','2014-03-07 19:54:53','<strong>2008</strong>\n\nBaerwald, M., V. Bien, F. Feyrer, and B. May. Distribution of genetically differentiated splittail populations during nonspawning season. Trans. Am. Fish. Soc. 137:1335-1345 (<a title=\"Publications_2_files/2008 - Baerwald_Distrib_splittail.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008%20-%20Baerwald_Distrib_splittail.pdf\">pdf</a>).\n\nBaerwald, M., A. Welsh, R. Hedrick, and B. May. Discovery of genes implicated in whirling disease infection and resistance in rainbow trout using genome-wide expression profiling. BMC Genomics 9:37 (<a title=\"Publications_2_files/2008_Baerwald_microarray.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008_Baerwald_microarray.pdf\">pdf</a>).\n\nBrk, K.S, A. Drauch, J. Israel, J. Pedroia, J. Rodzen, and B. May. Development of new microsatellite primers for green and white sturgeon. Cons. Gen. 9:973-979 (<a title=\"Publications_2_files/2008_KBork_Primer_Note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008_KBork_Primer_Note.pdf\">pdf</a>)\n\nPetersen, J. L., A. M. Ibarra, J. L. Ramirez, and B. May. An induced mass spawn of the hermaphroditic lion-paw scallop: Genetic assignment of maternal and paternal parentage. J. Hered. 99:337-348 (<a title=\"Publications_2_files/2008 - Petersen et al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008%20-%20Petersen%20et%20al.pdf\">pdf</a>).\n\nSchwartz, R.S. and B. May. Genetic Evaluation of Isolated Populations for Use in Reintroductions Reveals Significant Genetic Bottlenecks in Potential Stocks of Sacramento Perch. Trans. Am. Fish. Soc. 137:1764-1777 (<a title=\"Publications_2_files/2008_Schwartz_-_sac_perch.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008_Schwartz_-_sac_perch.pdf\">pdf</a>).\n\nWelsh, A., T. Hill, H. Quinlan, C. Robinson, and B. May. Genetic assessment of lake sturgeon population structure in the Great Lakes. N. Am. J. Fish. Man. 28:572591 (<a title=\"Publications_2_files/2008_Welsh_et_al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008_Welsh_et_al.pdf\">pdf</a>).\n\nWilliamson, K.S., R. Phillips, and B. May. Characterization of a chromosomal rearrangement responsible for producing \"apparent\" XY-female fall Chinook salmon in California. J. Hered. 99:483-490 (<a title=\"Publications_2_files/2008_Williamson_Heredity.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008_Williamson_Heredity.pdf\">pdf</a>).\n\n<strong>2007</strong>\n\nBaerwald, M.R., V. Bien, F. Feyrer, and B. May. Genetic analysis reveals two distinct Sacramento splittail (<em>Pogonichthyes macrolepidotus</em>) populations. Cons. Gen. 8:159-167. (<a title=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007_Baerwald_Genetic%20analysis%20splittail.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007_Baerwald_Genetic%20analysis%20splittail.pdf\">pdf</a>).\n\nChen, Y., S. Parmenter, and B. May. Introgression between Lahontan and endangered Owens tui chubs, and apparent discovery of a new tui chub in the Owens Valley, California. Conserv. Genet. 8:221-238. (<a title=\"Publications_2_files/2007_Chen_Owens_tui.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007_Chen_Owens_tui.pdf\">pdf</a>).\n\nFeyrer, F., J. Hobbs, M. Baerwald, T. Sommer, Q. Yin, K. Clark, B. May and W. Bennett. Otolith microchemistry provides information complementary to microsatellite DNA for migratory fish. Trans. Amer. Fish. Soc. 136:469-476. (<a title=\"Publications_2_files/2007 Feyrer_otoliths.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007%20Feyrer_otoliths.pdf\">pdf</a>)\n\nHull, J.M., D. Tufts, J. Topinka, B. May, and H. Ernest. Development of 19 microsatellite loci for Swainson\'s hawks (<em>Buteo swainsoni</em>) and other buteos. Molecular Ecology Notes 7:346-349. (<a title=\"Publications_2_files/2008_Hull_Swainsons.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008_Hull_Swainsons.pdf\">pdf</a>).\n\nLindley, S.T., R. Schick, E. Mora, P. Adams, J. Anderson, S. Greene, C. Hanson, B. May, D. McEwan, R. MacFarlane, C. Swanson, and J. Williams. Framework for Assessing Viability of Threatened and Endangered Chinook Salmon and Steelhead in the Sacramento-San Joaquin Basin . San Francisco Estuary and Watershed Science. Vol. 5, Issue 1 (1-26), Article 4. (<a title=\"Publications_2_files/2007_Lindley_viability.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007_Lindley_viability.pdf\">pdf</a>).\n\nMeredith, E. P., J.A. Rodzen, J. D. Banks, R. Schaefer, H. B. Ernest, T. R. Famula and B. P. May. Microsatellite analysis of three subspecies of Elk (<em>Cervus elaphus</em>) in California. Journal of Mammalogy 88(3):801-808. (<a title=\"Publications_2_files/2007 Meredith-elk.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007%20Meredith-elk.pdf\">pdf</a>)\n\nPapa R., J.A. Israel, F. N. Marzan, and B. May. Assessment of genetic variation among reproductive ecotypes of Klamath River steelhead reveals differentiation associated with different run-timings. J. Appl. Icthy. (<a title=\"Publications_2_files/2007 Papa_steelhead.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007%20Papa_steelhead.pdf\">pdf</a>)\n\nWilliamson, K. and B. May. Mitochondrial DNA Haplotype Diversity in Apparent XY Female Fall-Run and Spring-Run Chinook Salmon in California\'s Central Valley. Trans. Amer. Fish. Soc. 136: 1480-1486 (<a title=\"Publications_2_files/2007_Williamson_-_mtDNA_chinook.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007_Williamson_-_mtDNA_chinook.pdf\">pdf</a>).\n\n<strong>2006</strong>\n\nCordes, J.F., M.R. Stephens, M.A. Blumberg, and B. May. Identifying introgressive hybridization in native populations of California Golden Trout based on molecular markers. Trans. Am. Fish. Soc. 135:110-128. (<a title=\"Publications_2_files/2006_Cordes_etal.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2006_Cordes_etal.pdf\">pdf</a>).\n\nIbarra, A.M., J.L. Petersen, T.R. Famula, and B. May. Characterization of 35 microsatellite loci in the Pacific lion-paw scallop (<em>Nodipecten subnodosus</em>) and their cross-species amplification in four other scallops of the Pectinidae family. Mol. Ecol. Notes. 6:153-156 (<a title=\"Publications_2_files/2006 Ibarra_scallop_primers.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2006%20Ibarra_scallop_primers.pdf\">pdf</a>).\n\nLindley, S.T., R.S. Schick, A. Agrawal, M. Goslin, T. Pearson, E. Mora, J. J. Anderson, B.P. May, S. Greene, C. Hanson, A. Low, D. McEwan, R.B. MacFarlane, C. Swanson and J.G. Williams. Historical population structure of Central Valley steelhead and its alteration by dams. SF Est. Wtshd. Sci. 4(1): Article 3 (<a title=\"Publications_2_files/2006 Lindley___Steelhead.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2006%20Lindley___Steelhead.pdf\">pdf</a>).\n\nSprowles, A.E., M.R. Stephens, N.W. Clipperton, and B.P. May. Fishing for SNPs: a targeted locus approach for Single Nucleotide Polymorphism discovery in rainbow trout. Trans. Am. Fish. Soc. 135:1698-1721. (<a title=\"Publications_2_files/2006_Sprowles_SNPs.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2006_Sprowles_SNPs.pdf\">pdf</a>).\n\nTranah, G.J., and B. P. May. Patterns of Intra- and Interspecies Genetic Diversity in Klamath River Basin Suckers. Trans. Am Fish. Soc. 135:306316 (<a title=\"Publications_2_files/2006 Tranah_May.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2006%20Tranah_May.pdf\">pdf</a>).\n\nWelsh, A. and B. May. Development and standardization of disomic microsatellite markers for lake sturgeon genetic studies. J. Appl. Icthy. (<a title=\"Publications_2_files/2006 welsh standardization.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2006%20welsh%20standardization.pdf\">pdf</a>).\n\n<strong>2005</strong>\n\nBayard de Volo, S., RT Reynolds, JR Topinka, B. May and MF Antolin. Population genetic structure and the utility of muliti-locus genotyping for mark-recapture studies of northern goshawks (<em>Accipiter gentilis</em>). J. Raptor Res. 39:286-295. (<a title=\"Publications_2_files/2005 - Bayard_de_volo_etal.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20-%20Bayard_de_volo_etal.pdf\">pdf</a>)\n\nCordes, J.F., D.L. Perkins, H.L.Kincaid, and B. May. Genetic analysis of fish genomes and populations: allozyme variation within and among Atlantic salmon from downeast rivers of Maine. J. Fish Biology. 67:104-117. (<a title=\"Publications_2_files/2005 - Cordes Atlantic salmon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20-%20Cordes%20Atlantic%20salmon.pdf\">pdf</a>)\n\nFloyd, C.H., D. Van Vuren, and B. May. Marmots on Great Basin mountaintops: using genetics to test an ecological paradigm. Ecology. 86: 2145-2153. (<a title=\"Publications_2_files/2005 - Floyd marmot.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20-%20Floyd%20marmot.pdf\">pdf</a>)\n\nFlores Martnez, J.J., C. H. Floyd, L.G. Herrera, and B. May. Genetic variation and population size of the endangered fishing bat, Myotis vivesi, in Isla Partida. Pp. 187-192 in Contribuciones Mastozoolgicas en Homenaje a Bernardo Villa (V. Snchez-Cordero y R. A. Medelln, eds.). Instituto de Biologa, UNAM; Instituto de Ecologa, UNAM; CONABIO. Mxico. 706 pp.\n\nLeunda, P.M., R. Miranda, J. Madoz, S. Parmenter, Y. Chen, and B. May. Threatened fishes of the world: Siphateles bicolor snyderi (Miller, 1973) (Cyprinidae). Env. Biol. Fish. 73:109-110 (<a title=\"Publications_2_files/2005 Leunda etal.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20Leunda%20etal.pdf\">pdf</a>)\n\nRodrigue, K.L., B.P. May, T.R. Famula, and M.E. Delany. Meiotic instability of chicken ultra-long telomeres and mapping of a 2.8 megabase array to the W-sex chromosome. Chrom. Res.13:581-591. (<a title=\"Publications_2_files/2005 - Rodrigue___chicken.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20-%20Rodrigue___chicken.pdf\">pdf</a>)\n\nTeglas, M.B., B. May, P.R. Crosbie, M.R. Stephens, and W.M. Boyce. Genetic structure of the tick Ornithodoros coriaceus (Acarina: Argasidae) in California, Nevada and Oregon. J. Med. Entomol. 42:247-253.\n\nWilliamson, K. and B. May. Homogenization of fall-run chinook salmon gene pools in the Central Valley of California, USA. N. Am. J. Fish. Man. 25:9931009. (<a title=\"Publications_2_files/2005 - Williamson chinook homogenize.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20-%20Williamson%20chinook%20homogenize.pdf\">pdf</a>)\n\nWilliamson, K. and B. May. Inheritance studies implicate a genetic mechanism for apparent sex-reversal in Chinook salmon. Trans. Am. Fish. Soc. 134:12531261. (<a title=\"Publications_2_files/2005 - Williamson sex.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20-%20Williamson%20sex.pdf\">pdf</a>)\n\n<strong>Later Publications:</strong>\n<ul>\n	<li><a title=\"Publications\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/\">2009-2013</a></li>\n</ul>\n<strong>Earlier Publications:</strong>\n<ul>\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\n</ul>','Publications 2005-2008','','inherit','closed','open','','129-autosave-v1','','','2014-03-07 19:54:53','2014-03-07 19:54:53','',129,'http://localhost/~psalessi/Genetics_Lab/129-autosave-v1/',0,'revision','',0),(302,1,'2014-03-07 19:55:32','2014-03-07 19:55:32','<strong>2008</strong>\r\n\r\nBaerwald, M., V. Bien, F. Feyrer, and B. May. Distribution of genetically differentiated splittail populations during nonspawning season. Trans. Am. Fish. Soc. 137:1335-1345 (<a title=\"Publications_2_files/2008 - Baerwald_Distrib_splittail.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008%20-%20Baerwald_Distrib_splittail.pdf\">pdf</a>).\r\n\r\nBaerwald, M., A. Welsh, R. Hedrick, and B. May. Discovery of genes implicated in whirling disease infection and resistance in rainbow trout using genome-wide expression profiling. BMC Genomics 9:37 (<a title=\"Publications_2_files/2008_Baerwald_microarray.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008_Baerwald_microarray.pdf\">pdf</a>).\r\n\r\nBrk, K.S, A. Drauch, J. Israel, J. Pedroia, J. Rodzen, and B. May. Development of new microsatellite primers for green and white sturgeon. Cons. Gen. 9:973-979 (<a title=\"Publications_2_files/2008_KBork_Primer_Note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008_KBork_Primer_Note.pdf\">pdf</a>)\r\n\r\nPetersen, J. L., A. M. Ibarra, J. L. Ramirez, and B. May. An induced mass spawn of the hermaphroditic lion-paw scallop: Genetic assignment of maternal and paternal parentage. J. Hered. 99:337-348 (<a title=\"Publications_2_files/2008 - Petersen et al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008%20-%20Petersen%20et%20al.pdf\">pdf</a>).\r\n\r\nSchwartz, R.S. and B. May. Genetic Evaluation of Isolated Populations for Use in Reintroductions Reveals Significant Genetic Bottlenecks in Potential Stocks of Sacramento Perch. Trans. Am. Fish. Soc. 137:1764-1777 (<a title=\"Publications_2_files/2008_Schwartz_-_sac_perch.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008_Schwartz_-_sac_perch.pdf\">pdf</a>).\r\n\r\nWelsh, A., T. Hill, H. Quinlan, C. Robinson, and B. May. Genetic assessment of lake sturgeon population structure in the Great Lakes. N. Am. J. Fish. Man. 28:572591 (<a title=\"Publications_2_files/2008_Welsh_et_al.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008_Welsh_et_al.pdf\">pdf</a>).\r\n\r\nWilliamson, K.S., R. Phillips, and B. May. Characterization of a chromosomal rearrangement responsible for producing \"apparent\" XY-female fall Chinook salmon in California. J. Hered. 99:483-490 (<a title=\"Publications_2_files/2008_Williamson_Heredity.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008_Williamson_Heredity.pdf\">pdf</a>).\r\n\r\n<strong>2007</strong>\r\n\r\nBaerwald, M.R., V. Bien, F. Feyrer, and B. May. Genetic analysis reveals two distinct Sacramento splittail (<em>Pogonichthyes macrolepidotus</em>) populations. Cons. Gen. 8:159-167. (<a title=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007_Baerwald_Genetic%20analysis%20splittail.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007_Baerwald_Genetic%20analysis%20splittail.pdf\">pdf</a>).\r\n\r\nChen, Y., S. Parmenter, and B. May. Introgression between Lahontan and endangered Owens tui chubs, and apparent discovery of a new tui chub in the Owens Valley, California. Conserv. Genet. 8:221-238. (<a title=\"Publications_2_files/2007_Chen_Owens_tui.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007_Chen_Owens_tui.pdf\">pdf</a>).\r\n\r\nFeyrer, F., J. Hobbs, M. Baerwald, T. Sommer, Q. Yin, K. Clark, B. May and W. Bennett. Otolith microchemistry provides information complementary to microsatellite DNA for migratory fish. Trans. Amer. Fish. Soc. 136:469-476. (<a title=\"Publications_2_files/2007 Feyrer_otoliths.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007%20Feyrer_otoliths.pdf\">pdf</a>)\r\n\r\nHull, J.M., D. Tufts, J. Topinka, B. May, and H. Ernest. Development of 19 microsatellite loci for Swainson\'s hawks (<em>Buteo swainsoni</em>) and other buteos. Molecular Ecology Notes 7:346-349. (<a title=\"Publications_2_files/2008_Hull_Swainsons.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2008_Hull_Swainsons.pdf\">pdf</a>).\r\n\r\nLindley, S.T., R. Schick, E. Mora, P. Adams, J. Anderson, S. Greene, C. Hanson, B. May, D. McEwan, R. MacFarlane, C. Swanson, and J. Williams. Framework for Assessing Viability of Threatened and Endangered Chinook Salmon and Steelhead in the Sacramento-San Joaquin Basin . San Francisco Estuary and Watershed Science. Vol. 5, Issue 1 (1-26), Article 4. (<a title=\"Publications_2_files/2007_Lindley_viability.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007_Lindley_viability.pdf\">pdf</a>).\r\n\r\nMeredith, E. P., J.A. Rodzen, J. D. Banks, R. Schaefer, H. B. Ernest, T. R. Famula and B. P. May. Microsatellite analysis of three subspecies of Elk (<em>Cervus elaphus</em>) in California. Journal of Mammalogy 88(3):801-808. (<a title=\"Publications_2_files/2007 Meredith-elk.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007%20Meredith-elk.pdf\">pdf</a>)\r\n\r\nPapa R., J.A. Israel, F. N. Marzan, and B. May. Assessment of genetic variation among reproductive ecotypes of Klamath River steelhead reveals differentiation associated with different run-timings. J. Appl. Icthy. (<a title=\"Publications_2_files/2007 Papa_steelhead.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007%20Papa_steelhead.pdf\">pdf</a>)\r\n\r\nWilliamson, K. and B. May. Mitochondrial DNA Haplotype Diversity in Apparent XY Female Fall-Run and Spring-Run Chinook Salmon in California\'s Central Valley. Trans. Amer. Fish. Soc. 136: 1480-1486 (<a title=\"Publications_2_files/2007_Williamson_-_mtDNA_chinook.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2007_Williamson_-_mtDNA_chinook.pdf\">pdf</a>).\r\n\r\n<strong>2006</strong>\r\n\r\nCordes, J.F., M.R. Stephens, M.A. Blumberg, and B. May. Identifying introgressive hybridization in native populations of California Golden Trout based on molecular markers. Trans. Am. Fish. Soc. 135:110-128. (<a title=\"Publications_2_files/2006_Cordes_etal.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2006_Cordes_etal.pdf\">pdf</a>).\r\n\r\nIbarra, A.M., J.L. Petersen, T.R. Famula, and B. May. Characterization of 35 microsatellite loci in the Pacific lion-paw scallop (Nodipecten subnodosus) and their cross-species amplification in four other scallops of the Pectinidae family. Mol. Ecol. Notes. 6:153-156 (<a title=\"Publications_2_files/2006 Ibarra_scallop_primers.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2006%20Ibarra_scallop_primers.pdf\">pdf</a>).\r\n\r\nLindley, S.T., R.S. Schick, A. Agrawal, M. Goslin, T. Pearson, E. Mora, J. J. Anderson, B.P. May, S. Greene, C. Hanson, A. Low, D. McEwan, R.B. MacFarlane, C. Swanson and J.G. Williams. Historical population structure of Central Valley steelhead and its alteration by dams. SF Est. Wtshd. Sci. 4(1): Article 3 (<a title=\"Publications_2_files/2006 Lindley___Steelhead.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2006%20Lindley___Steelhead.pdf\">pdf</a>).\r\n\r\nSprowles, A.E., M.R. Stephens, N.W. Clipperton, and B.P. May. Fishing for SNPs: a targeted locus approach for Single Nucleotide Polymorphism discovery in rainbow trout. Trans. Am. Fish. Soc. 135:1698-1721. (<a title=\"Publications_2_files/2006_Sprowles_SNPs.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2006_Sprowles_SNPs.pdf\">pdf</a>).\r\n\r\nTranah, G.J., and B. P. May. Patterns of Intra- and Interspecies Genetic Diversity in Klamath River Basin Suckers. Trans. Am Fish. Soc. 135:306316 (<a title=\"Publications_2_files/2006 Tranah_May.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2006%20Tranah_May.pdf\">pdf</a>).\r\n\r\nWelsh, A. and B. May. Development and standardization of disomic microsatellite markers for lake sturgeon genetic studies. J. Appl. Icthy. (<a title=\"Publications_2_files/2006 welsh standardization.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2006%20welsh%20standardization.pdf\">pdf</a>).\r\n\r\n<strong>2005</strong>\r\n\r\nBayard de Volo, S., RT Reynolds, JR Topinka, B. May and MF Antolin. Population genetic structure and the utility of muliti-locus genotyping for mark-recapture studies of northern goshawks (Accipiter gentilis). J. Raptor Res. 39:286-295. (<a title=\"Publications_2_files/2005 - Bayard_de_volo_etal.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20-%20Bayard_de_volo_etal.pdf\">pdf</a>)\r\n\r\nCordes, J.F., D.L. Perkins, H.L.Kincaid, and B. May. Genetic analysis of fish genomes and populations: allozyme variation within and among Atlantic salmon from downeast rivers of Maine. J. Fish Biology. 67:104-117. (<a title=\"Publications_2_files/2005 - Cordes Atlantic salmon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20-%20Cordes%20Atlantic%20salmon.pdf\">pdf</a>)\r\n\r\nFloyd, C.H., D. Van Vuren, and B. May. Marmots on Great Basin mountaintops: using genetics to test an ecological paradigm. Ecology. 86: 2145-2153. (<a title=\"Publications_2_files/2005 - Floyd marmot.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20-%20Floyd%20marmot.pdf\">pdf</a>)\r\n\r\nFlores Martnez, J.J., C. H. Floyd, L.G. Herrera, and B. May. Genetic variation and population size of the endangered fishing bat, Myotis vivesi, in Isla Partida. Pp. 187-192 in Contribuciones Mastozoolgicas en Homenaje a Bernardo Villa (V. Snchez-Cordero y R. A. Medelln, eds.). Instituto de Biologa, UNAM; Instituto de Ecologa, UNAM; CONABIO. Mxico. 706 pp.\r\n\r\nLeunda, P.M., R. Miranda, J. Madoz, S. Parmenter, Y. Chen, and B. May. Threatened fishes of the world: Siphateles bicolor snyderi (Miller, 1973) (Cyprinidae). Env. Biol. Fish. 73:109-110 (<a title=\"Publications_2_files/2005 Leunda etal.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20Leunda%20etal.pdf\">pdf</a>)\r\n\r\nRodrigue, K.L., B.P. May, T.R. Famula, and M.E. Delany. Meiotic instability of chicken ultra-long telomeres and mapping of a 2.8 megabase array to the W-sex chromosome. Chrom. Res.13:581-591. (<a title=\"Publications_2_files/2005 - Rodrigue___chicken.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20-%20Rodrigue___chicken.pdf\">pdf</a>)\r\n\r\nTeglas, M.B., B. May, P.R. Crosbie, M.R. Stephens, and W.M. Boyce. Genetic structure of the tick Ornithodoros coriaceus (Acarina: Argasidae) in California, Nevada and Oregon. J. Med. Entomol. 42:247-253.\r\n\r\nWilliamson, K. and B. May. Homogenization of fall-run chinook salmon gene pools in the Central Valley of California, USA. N. Am. J. Fish. Man. 25:9931009. (<a title=\"Publications_2_files/2005 - Williamson chinook homogenize.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20-%20Williamson%20chinook%20homogenize.pdf\">pdf</a>)\r\n\r\nWilliamson, K. and B. May. Inheritance studies implicate a genetic mechanism for apparent sex-reversal in Chinook salmon. Trans. Am. Fish. Soc. 134:12531261. (<a title=\"Publications_2_files/2005 - Williamson sex.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_2_files/2005%20-%20Williamson%20sex.pdf\">pdf</a>)\r\n\r\n<strong>Later Publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/\">2009-2013</a></li>\r\n</ul>\r\n<strong>Earlier Publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications 2005-2008','','inherit','closed','open','','129-revision-v1','','','2014-03-07 19:55:32','2014-03-07 19:55:32','',129,'http://localhost/~psalessi/Genetics_Lab/129-revision-v1/',0,'revision','',0),(303,1,'2014-03-07 20:11:38','2014-03-07 20:11:38','<strong>2004</strong>\n\nBaerwald, M.R. and B. May. Characterization of microsatellite loci for five members of the minnow family Cyprinidae found in the Sacramento - San Joaquin Delta. Mol. Ecol. Notes. 4:385-390 (<a title=\"Publications_3_files/2004 - baerwald primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20baerwald%20primer%20note.pdf\">pdf</a>).\n\nCordes, J.F., J.A. Israel and B. May. Conservation of Paiute cutthroat: the genetic legacy of population transplants in an endemic California salmonid. CA Fish and Game. 90-101-118 (<a title=\"Publications_3_files/2004 - Cordes cutthroat.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Cordes%20cutthroat.pdf\">pdf</a>).\n\nFlores-Martnez, J.J., C. Floyd, L.G. Herrera, and B. May. Genetic variation and population size in the fishing bat, Myotis vivesi, in Isla Partida. In: Homenaje a Bernardo Villa, Eds., R. Medelln and V. Snchez-Cordero. Publ. Universidad Nacional Autnoma de Mxico. Pp. 185-190 (<a title=\"Publications_3_files/2004_fishing bats.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004_fishing%20bats.pdf\">pdf</a>).\n\nHogan, Z.S., P.B. Moyle, B. May, M.J. Vander Zanden and I.G. Baird. Imperiled Giants of the Mekong: Ecologists struggle to understandand protectSoutheast Asia\'s large migratory catfish. Am. Sci. 92:228-237b (<a title=\"Publications_3_files/2004 - Hogan Am Sci.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Hogan%20Am%20Sci.pdf\">pdf</a>).\n\nIsrael, J.A., J.F. Cordes, M.A. Blumberg, and B. May. Geographic patterns of genetic differentiation among collections of green sturgeon. N. Am. J. Fish. Man. 24:922-931 (<a title=\"Publications_3_files/2004 - Israel green sturgeon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Israel%20green%20sturgeon.pdf\">pdf</a>).\n\nMoen, T., J.J. Agresti, A. Cnaani, H. Moses, T.R. Famula, G. Hulata, G.A.E. Gall, B. May. A genome scan of a four-way tilapia cross supports the existence of a quantitative trait locus for cold tolerance on linkage group 23. Aqua. Res. 35:893-904 (<a title=\"Publications_3_files/2004 - Moen etal.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Moen%20etal.pdf\">pdf</a>).\n\nRodriguez, F., S. LaPatra, S. Williams, T. Famula and B. May. Genetic markers associated with resistance to infectious hematopoietic necrosis in rainbow and steelhead trout (Oncorhynchus mykiss) backcrosses. Aquaculture 241:93-115 (<a title=\"Publications_3_files/2004 - Rodriguez_et_al[1].pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Rodriguez_et_al%5B1%5D.pdf\">pdf</a>).\n\nRodzen, J.A., T. Famula, and B. May. Estimation of parentage and relatedness in the polyploid white sturgeon Acipenser transmontanus using a dominant marker approach for duplicated microsatellite loci. Aquaculture 232:165-182 (<a title=\"Publications_3_files/2004 - Rodzen Aquaculture paper.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Rodzen%20Aquaculture%20paper.pdf\">pdf</a>).\n\nSchwartz, R.S. and B. May. Characterization of microsatellite loci in Sacramento perch (Archoplites interruptus). Mol. Ecol. Notes. 4:694-697 (<a title=\"Publications_3_files/2004 - Schwartz primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Schwartz%20primer%20note.pdf\">pdf</a>).\n\nStadelmann, B., L.G. Herrera, J. Arroyo-Cabrales, J.J. Flores-Martnez, B.P. May, and M. Ruedi. Molecular systematics of the fishing bat Myotis (Pizonyz) vivesi. J. Mammo. 85:133139 (<a title=\"Publications_3_files/2004 - Stadelmann.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Stadelmann.pdf\">pdf</a>).\n\nTopinka, J.R., A.J. Donovan, and B. May. Characterization of microsatellite loci in Kearneys bluestar (Amsonia kearneyana) and cross-amplification in other Amsonia species. Mol. Ecol. Notes. 4:710-712 (<a title=\"Publications_3_files/2004 - Topinka Amsonia.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Topinka%20Amsonia.pdf\">pdf</a>).\n\nTopinka, J.R. and B. May. Development of polymorphic microsatellite loci in the northern goshawk (Accipiter gentilis atricapillus) and cross-amplification in other raptor species. Cons. Gen. 5:861-864 (<a title=\"Publications_3_files/2004 - Topinka goshawk.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Topinka%20goshawk.pdf\">pdf</a>).\n\nTranah, G.J., D.E. Campton and B. May. Genetic evidence for hybridization of pallid and shovelnose sturgeon. J. Hered. 95:474-480 (<a title=\"Publications_3_files/2004 - Tranah J Heredity.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Tranah%20J%20Heredity.pdf\">pdf</a>).\n\n<strong>2003</strong>\n\nBeyer, J. and B. May. A graph-theoretic approach to the partition of individuals into full-sib families. Mol. Ecol. 12:2243-2250 (<a title=\"Publications_3_files/2003 - Beyer-Jen MS.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2003%20-%20Beyer-Jen%20MS.pdf\">pdf</a>).\n\nErnest, H., W.M. Boyce, V. Bleich, B. May, S. Stiver, and S. Torres. Genetic structure of mountain lion (Puma concolor) populations in California. Cons. Gen. 3:353-366 (<a title=\"Publications_3_files/2003 - Ernest.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2003%20-%20Ernest.pdf\">pdf</a>).\n\nMay, B. Allozyme variation. In: Population Genetics: Principles and Applications for Fisheries Scientists, Ed. E. Hallerman. Amer. Fish. Soc. Pp. 23-36.\n\nMcQuown, E., C.C. Krueger, H.L. Kincaid, G.A.E. Gall, and B. May. Genetic comparison of lake sturgeon (Acipenser fulvescens) populations: differentiation based on allelic frequencies at seven microsatellite loci. J. Great Lakes Res. 29:3-13 (<a title=\"Publications_3_files/2003 McQuown_JGLR.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2003%20McQuown_JGLR.pdf\">pdf</a>).\n\nTranah, G.J., M.J. Bagley, J.J. Agresti, and B. May. Development of codominant markers for identifying species hybrids. Cons. Gen. 4:537-541 (<a title=\"Publications_3_files/2003 - Tranah codominant markers1.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2003%20-%20Tranah%20codominant%20markers1.pdf\">pdf</a>).\n\nWelsh, A.B., M. Blumberg, and B. May. Identification of microsatellite loci in lake sturgeon, Acipenser fulvescens, and their variability in green sturgeon, A. medirostris. Mol. Ecol. Notes 3:47-55 (<a title=\"Publications_3_files/2003 - Welsh primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2003%20-%20Welsh%20%20primer%20note.pdf\">pdf</a>).\n\nWhitehead, A., S.L. Anderson, K.M Kuivila, J.L. Roach, and B. May. Genetic variation among interconnected populations of Catostomus occidentalis: implications for distinguishing impacts of contaminants from biogeographic structuring. Mol. Ecol. 12: 2817-2833 (<a title=\"Publications_3_files/2003 - Whitehead etal 1.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2003%20-%20Whitehead%20etal%201.pdf\">pdf</a>).\n\n<strong>2002</strong>\n\nBlankenship, S.M., B. May, and D. Hedgecock. Evolution of a perfect simple-sequence-repeat locus in the context of its flanking sequence. Mol. Biol. Evol. 19:1943-1951 (<a title=\"Publications_3_files/2002 Blankenship, May.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20Blankenship,%20May.pdf\">pdf</a>).\n\nHogan, Z. S. and B. May. Twenty-seven new microsatellites for the migratory Asian catfish family Pangasiidae. Mol. Ecol. Notes. 2:38-41 (<a title=\"Publications_3_files/2002 - Hogan primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Hogan%20primer%20note.pdf\">pdf</a>).\n\nMcQuown, E., G.A.E. Gall, and B. May. Characterization and inheritance of six microsatellite loci in lake sturgeon (Acipenser fulvescens). Trans. Am. Fish. Soc. 131:299-307.\n\nMeredith, E.P. and B. May. Microsatellite loci in the Lahontan tui chub, Gila biocolor obesa, and their utililization in other chub species. Mol. Ecol. Notes. 2:156-158 (<a title=\"Publications_3_files/2002 - Meredith primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Meredith%20primer%20note.pdf\">pdf</a>).\n\nRodzen, J.A. and B. May. Inheritance of microsatellite loci in the white sturgeon (Acipenser transmontanus). Genome 45:1064-1076 (<a title=\"Publications_3_files/2002 - Rodzen and May Genome.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Rodzen%20and%20May%20Genome.pdf\">pdf</a>).\n\nSmith, C.T., R.J. Nelson, S. Pollard, E. Rubridge, S.J. McKay, J. Rodzen, B. May, and B. Koop. Population genetic analysis of white sturgeon (Acipenser transmontanus) in the Fraser River. J. Appl. Icthy. 18:307-312 (<a title=\"Publications_3_files/2002 - Acipenser transmontanus.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Acipenser%20transmontanus.pdf\">pdf</a>).\n\nWilliamson, K.S., J.F. Cordes, and B. May. Characterization of microsatellite loci in Chinook salmon (Oncorhynchus tshawytscha) and cross-species amplification in other salmonids. Mol. Ecol. Notes. 2:17-19 (<a title=\"Publications_3_files/2002 - Williamson primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Williamson%20primer%20note.pdf\">pdf</a>).\n\nWilliamson, K. and B. May. Incidence of phenotypic female chinook salmon (Oncorhynchus tshawytscha) positive for the Y-chromosome specific marker OtY1 in the Central Valley. J. Aquat. An. Health. 176-183 (<a title=\"Publications_3_files/2002 - Williamson_and_May_JAAH14.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Williamson_and_May_JAAH14.pdf\">pdf</a>).\n\nZhu, B., F. Zhou, H. Cao, Z. Shao, N. Zhao, B. May, and J. Chang. Analysis of genetic variation in the Chinese sturgeon, Acipenser sinensis: estimating the contribution of artificially produced larvae in a wild population J. Appl. Icthy. 18:301-306 (<a title=\"Publications_3_files/2002 - Acipenser sinensis.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Acipenser%20sinensis.pdf\">pdf</a>).\n\n<strong>2001</strong>\n\nBagley, M.J., S. Anderson, and B. May. Choice of methodology for assessing genetic impacts of environmental stressors: polymorphism and reproducibility of RAPD and AFLP fingerprints. Ecotoxicology 10:239-244 (<a title=\"Publications_3_files/2001 - RAPD and AFLP Fingerprints.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2001%20-%20RAPD%20and%20AFLP%20Fingerprints.pdf\">pdf</a>).\n\nJenneckens I., J.-N. Meyer, G. Hrstgen-Schwark, B. May, L. Debus, A. Ludwig. A fixed allele at microsatellite LS-39 exhibiting species-specificity for black caviar producer Acipenser stellatus. J. Appl. Icthyol. 17:39-42 (<a title=\"Publications_3_files/2001 - Acipenser stellatus.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2001%20-%20Acipenser%20stellatus.pdf\">pdf</a>).\n\nPyatskowit, J., C.C. Krueger, H.L. Kincaid, and B. May. Inheritance of microsatellite loci in the polyploid derivative lake sturgeon (Acipenser fulvescens). Genome 44:185-191 (<a title=\"Publications_3_files/2001 - pyatskowit.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2001%20-%20pyatskowit.pdf\">pdf</a>).\n\nTranah, G.J., J.J. Agresti, and B. May. New microsatellite loci for suckers (Catostomidae): primer homology in Catostomus, Chasmistes, and Deltistes. Mol. Ecol. Notes 1:55-60 (<a title=\"Publications_3_files/2001 - Tranah primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2001%20-%20Tranah%20primer%20note.pdf\">pdf</a>).\n\nTranah, G.J., H.L. Kincaid, C.C. Krueger, D.E. Campton, B. May. Reproductive isolation in sympatric populations of pallid and shovelnose sturgeon. N. Am. J. Fish. Man. 21:367-373 (<a title=\"Publications_3_files/2001 - Tranah_sturgeonEtAl_NAJFM.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2001%20-%20Tranah_sturgeonEtAl_NAJFM.pdf\">pdf</a>).\n\n<strong>2000</strong>\n\n<span style=\"line-height: 1.714285714; font-size: 1rem;\">Agresti, J.J., S. Seki, A. Cnaani, S. Poompuang, E. M. Hallerman, N. Umiel, G. Hulata, G. A.E. Gall, and B. May. Breeding new strains of tilapia: development of an artificial center of origin and linkage map based on AFLP and microsatellite loci. Aquaculture. 185:43-56 (<a title=\"Publications_3_files/2000 - Tilapia.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2000%20-%20Tilapia.pdf\">pdf</a>).</span>\n\nBelfiore, N.M. and B. May. Variable microsatellite loci in red swamp crayfish, Procambarus clarkii, and their characterization in other crayfish taxa. Mol. Ecol. 12:2231-2234 (<a title=\"Publications_3_files/2000 - Belfiore_red swamp crayfish.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2000%20-%20Belfiore_red%20swamp%20crayfish.pdf\">pdf</a>).\n\nErnest, H.B., M.C.T. Penedo, B. May, M. Syvanen, and W.M. Boyce. Molecular tracking of mountain lions in the Yosemite Valley region in California: genetic analysis using microsatellites and fecal DNA. Mol. Ecol. 9: 433-441 (pdf).\n\nLudwig, A., B. May, L. Debus, and I, Jenneckens. Heteroplasmy in the mtDNA Control Region of Sturgeon (Acipenser, Huso and Scaphirhynchus) Genetics 156:1933-1947 (pdf).\n\nMcQuown, E.C., B.L. Sloss, R.J. Sheehan, J. Rodzen, G. Tranah, and B. May. Microsatellite analysis of genetic variation in sturgeon: new primer sequences for Scaphirynchus and Acipenser. Trans. Am. Fish. Soc. 129:1380-1388 (pdf).\n\n<strong>Later Publications:</strong>\n<ul>\n	<li><a title=\"Publications\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/\">2009-2013</a></li>\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\n</ul>\n<strong>Earlier Publications:</strong>\n<ul>\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\n</ul>','Publications 2000-2004','','inherit','closed','open','','135-autosave-v1','','','2014-03-07 20:11:38','2014-03-07 20:11:38','',135,'http://localhost/~psalessi/Genetics_Lab/135-autosave-v1/',0,'revision','',0),(304,1,'2014-03-07 20:12:27','2014-03-07 20:12:27','<strong>2004</strong>\r\n\r\nBaerwald, M.R. and B. May. Characterization of microsatellite loci for five members of the minnow family Cyprinidae found in the Sacramento - San Joaquin Delta. Mol. Ecol. Notes. 4:385-390 (<a title=\"Publications_3_files/2004 - baerwald primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20baerwald%20primer%20note.pdf\">pdf</a>).\r\n\r\nCordes, J.F., J.A. Israel and B. May. Conservation of Paiute cutthroat: the genetic legacy of population transplants in an endemic California salmonid. CA Fish and Game. 90-101-118 (<a title=\"Publications_3_files/2004 - Cordes cutthroat.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Cordes%20cutthroat.pdf\">pdf</a>).\r\n\r\nFlores-Martnez, J.J., C. Floyd, L.G. Herrera, and B. May. Genetic variation and population size in the fishing bat, Myotis vivesi, in Isla Partida. In: Homenaje a Bernardo Villa, Eds., R. Medelln and V. Snchez-Cordero. Publ. Universidad Nacional Autnoma de Mxico. Pp. 185-190 (<a title=\"Publications_3_files/2004_fishing bats.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004_fishing%20bats.pdf\">pdf</a>).\r\n\r\nHogan, Z.S., P.B. Moyle, B. May, M.J. Vander Zanden and I.G. Baird. Imperiled Giants of the Mekong: Ecologists struggle to understandand protectSoutheast Asia\'s large migratory catfish. Am. Sci. 92:228-237b (<a title=\"Publications_3_files/2004 - Hogan Am Sci.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Hogan%20Am%20Sci.pdf\">pdf</a>).\r\n\r\nIsrael, J.A., J.F. Cordes, M.A. Blumberg, and B. May. Geographic patterns of genetic differentiation among collections of green sturgeon. N. Am. J. Fish. Man. 24:922-931 (<a title=\"Publications_3_files/2004 - Israel green sturgeon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Israel%20green%20sturgeon.pdf\">pdf</a>).\r\n\r\nMoen, T., J.J. Agresti, A. Cnaani, H. Moses, T.R. Famula, G. Hulata, G.A.E. Gall, B. May. A genome scan of a four-way tilapia cross supports the existence of a quantitative trait locus for cold tolerance on linkage group 23. Aqua. Res. 35:893-904 (<a title=\"Publications_3_files/2004 - Moen etal.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Moen%20etal.pdf\">pdf</a>).\r\n\r\nRodriguez, F., S. LaPatra, S. Williams, T. Famula and B. May. Genetic markers associated with resistance to infectious hematopoietic necrosis in rainbow and steelhead trout (Oncorhynchus mykiss) backcrosses. Aquaculture 241:93-115 (<a title=\"Publications_3_files/2004 - Rodriguez_et_al[1].pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Rodriguez_et_al%5B1%5D.pdf\">pdf</a>).\r\n\r\nRodzen, J.A., T. Famula, and B. May. Estimation of parentage and relatedness in the polyploid white sturgeon Acipenser transmontanus using a dominant marker approach for duplicated microsatellite loci. Aquaculture 232:165-182 (<a title=\"Publications_3_files/2004 - Rodzen Aquaculture paper.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Rodzen%20Aquaculture%20paper.pdf\">pdf</a>).\r\n\r\nSchwartz, R.S. and B. May. Characterization of microsatellite loci in Sacramento perch (Archoplites interruptus). Mol. Ecol. Notes. 4:694-697 (<a title=\"Publications_3_files/2004 - Schwartz primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Schwartz%20primer%20note.pdf\">pdf</a>).\r\n\r\nStadelmann, B., L.G. Herrera, J. Arroyo-Cabrales, J.J. Flores-Martnez, B.P. May, and M. Ruedi. Molecular systematics of the fishing bat Myotis (Pizonyz) vivesi. J. Mammo. 85:133139 (<a title=\"Publications_3_files/2004 - Stadelmann.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Stadelmann.pdf\">pdf</a>).\r\n\r\nTopinka, J.R., A.J. Donovan, and B. May. Characterization of microsatellite loci in Kearneys bluestar (Amsonia kearneyana) and cross-amplification in other Amsonia species. Mol. Ecol. Notes. 4:710-712 (<a title=\"Publications_3_files/2004 - Topinka Amsonia.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Topinka%20Amsonia.pdf\">pdf</a>).\r\n\r\nTopinka, J.R. and B. May. Development of polymorphic microsatellite loci in the northern goshawk (Accipiter gentilis atricapillus) and cross-amplification in other raptor species. Cons. Gen. 5:861-864 (<a title=\"Publications_3_files/2004 - Topinka goshawk.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Topinka%20goshawk.pdf\">pdf</a>).\r\n\r\nTranah, G.J., D.E. Campton and B. May. Genetic evidence for hybridization of pallid and shovelnose sturgeon. J. Hered. 95:474-480 (<a title=\"Publications_3_files/2004 - Tranah J Heredity.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2004%20-%20Tranah%20J%20Heredity.pdf\">pdf</a>).\r\n\r\n<strong>2003</strong>\r\n\r\nBeyer, J. and B. May. A graph-theoretic approach to the partition of individuals into full-sib families. Mol. Ecol. 12:2243-2250 (<a title=\"Publications_3_files/2003 - Beyer-Jen MS.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2003%20-%20Beyer-Jen%20MS.pdf\">pdf</a>).\r\n\r\nErnest, H., W.M. Boyce, V. Bleich, B. May, S. Stiver, and S. Torres. Genetic structure of mountain lion (Puma concolor) populations in California. Cons. Gen. 3:353-366 (<a title=\"Publications_3_files/2003 - Ernest.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2003%20-%20Ernest.pdf\">pdf</a>).\r\n\r\nMay, B. Allozyme variation. In: Population Genetics: Principles and Applications for Fisheries Scientists, Ed. E. Hallerman. Amer. Fish. Soc. Pp. 23-36.\r\n\r\nMcQuown, E., C.C. Krueger, H.L. Kincaid, G.A.E. Gall, and B. May. Genetic comparison of lake sturgeon (Acipenser fulvescens) populations: differentiation based on allelic frequencies at seven microsatellite loci. J. Great Lakes Res. 29:3-13 (<a title=\"Publications_3_files/2003 McQuown_JGLR.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2003%20McQuown_JGLR.pdf\">pdf</a>).\r\n\r\nTranah, G.J., M.J. Bagley, J.J. Agresti, and B. May. Development of codominant markers for identifying species hybrids. Cons. Gen. 4:537-541 (<a title=\"Publications_3_files/2003 - Tranah codominant markers1.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2003%20-%20Tranah%20codominant%20markers1.pdf\">pdf</a>).\r\n\r\nWelsh, A.B., M. Blumberg, and B. May. Identification of microsatellite loci in lake sturgeon, Acipenser fulvescens, and their variability in green sturgeon, A. medirostris. Mol. Ecol. Notes 3:47-55 (<a title=\"Publications_3_files/2003 - Welsh primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2003%20-%20Welsh%20%20primer%20note.pdf\">pdf</a>).\r\n\r\nWhitehead, A., S.L. Anderson, K.M Kuivila, J.L. Roach, and B. May. Genetic variation among interconnected populations of Catostomus occidentalis: implications for distinguishing impacts of contaminants from biogeographic structuring. Mol. Ecol. 12: 2817-2833 (<a title=\"Publications_3_files/2003 - Whitehead etal 1.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2003%20-%20Whitehead%20etal%201.pdf\">pdf</a>).\r\n\r\n<strong>2002</strong>\r\n\r\nBlankenship, S.M., B. May, and D. Hedgecock. Evolution of a perfect simple-sequence-repeat locus in the context of its flanking sequence. Mol. Biol. Evol. 19:1943-1951 (<a title=\"Publications_3_files/2002 Blankenship, May.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20Blankenship,%20May.pdf\">pdf</a>).\r\n\r\nHogan, Z. S. and B. May. Twenty-seven new microsatellites for the migratory Asian catfish family Pangasiidae. Mol. Ecol. Notes. 2:38-41 (<a title=\"Publications_3_files/2002 - Hogan primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Hogan%20primer%20note.pdf\">pdf</a>).\r\n\r\nMcQuown, E., G.A.E. Gall, and B. May. Characterization and inheritance of six microsatellite loci in lake sturgeon (Acipenser fulvescens). Trans. Am. Fish. Soc. 131:299-307.\r\n\r\nMeredith, E.P. and B. May. Microsatellite loci in the Lahontan tui chub, Gila biocolor obesa, and their utililization in other chub species. Mol. Ecol. Notes. 2:156-158 (<a title=\"Publications_3_files/2002 - Meredith primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Meredith%20primer%20note.pdf\">pdf</a>).\r\n\r\nRodzen, J.A. and B. May. Inheritance of microsatellite loci in the white sturgeon (Acipenser transmontanus). Genome 45:1064-1076 (<a title=\"Publications_3_files/2002 - Rodzen and May Genome.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Rodzen%20and%20May%20Genome.pdf\">pdf</a>).\r\n\r\nSmith, C.T., R.J. Nelson, S. Pollard, E. Rubridge, S.J. McKay, J. Rodzen, B. May, and B. Koop. Population genetic analysis of white sturgeon (Acipenser transmontanus) in the Fraser River. J. Appl. Icthy. 18:307-312 (<a title=\"Publications_3_files/2002 - Acipenser transmontanus.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Acipenser%20transmontanus.pdf\">pdf</a>).\r\n\r\nWilliamson, K.S., J.F. Cordes, and B. May. Characterization of microsatellite loci in Chinook salmon (Oncorhynchus tshawytscha) and cross-species amplification in other salmonids. Mol. Ecol. Notes. 2:17-19 (<a title=\"Publications_3_files/2002 - Williamson primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Williamson%20primer%20note.pdf\">pdf</a>).\r\n\r\nWilliamson, K. and B. May. Incidence of phenotypic female chinook salmon (Oncorhynchus tshawytscha) positive for the Y-chromosome specific marker OtY1 in the Central Valley. J. Aquat. An. Health. 176-183 (<a title=\"Publications_3_files/2002 - Williamson_and_May_JAAH14.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Williamson_and_May_JAAH14.pdf\">pdf</a>).\r\n\r\nZhu, B., F. Zhou, H. Cao, Z. Shao, N. Zhao, B. May, and J. Chang. Analysis of genetic variation in the Chinese sturgeon, Acipenser sinensis: estimating the contribution of artificially produced larvae in a wild population J. Appl. Icthy. 18:301-306 (<a title=\"Publications_3_files/2002 - Acipenser sinensis.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2002%20-%20Acipenser%20sinensis.pdf\">pdf</a>).\r\n\r\n<strong>2001</strong>\r\n\r\nBagley, M.J., S. Anderson, and B. May. Choice of methodology for assessing genetic impacts of environmental stressors: polymorphism and reproducibility of RAPD and AFLP fingerprints. Ecotoxicology 10:239-244 (<a title=\"Publications_3_files/2001 - RAPD and AFLP Fingerprints.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2001%20-%20RAPD%20and%20AFLP%20Fingerprints.pdf\">pdf</a>).\r\n\r\nJenneckens I., J.-N. Meyer, G. Hrstgen-Schwark, B. May, L. Debus, A. Ludwig. A fixed allele at microsatellite LS-39 exhibiting species-specificity for black caviar producer Acipenser stellatus. J. Appl. Icthyol. 17:39-42 (<a title=\"Publications_3_files/2001 - Acipenser stellatus.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2001%20-%20Acipenser%20stellatus.pdf\">pdf</a>).\r\n\r\nPyatskowit, J., C.C. Krueger, H.L. Kincaid, and B. May. Inheritance of microsatellite loci in the polyploid derivative lake sturgeon (Acipenser fulvescens). Genome 44:185-191 (<a title=\"Publications_3_files/2001 - pyatskowit.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2001%20-%20pyatskowit.pdf\">pdf</a>).\r\n\r\nTranah, G.J., J.J. Agresti, and B. May. New microsatellite loci for suckers (Catostomidae): primer homology in Catostomus, Chasmistes, and Deltistes. Mol. Ecol. Notes 1:55-60 (<a title=\"Publications_3_files/2001 - Tranah primer note.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2001%20-%20Tranah%20primer%20note.pdf\">pdf</a>).\r\n\r\nTranah, G.J., H.L. Kincaid, C.C. Krueger, D.E. Campton, B. May. Reproductive isolation in sympatric populations of pallid and shovelnose sturgeon. N. Am. J. Fish. Man. 21:367-373 (<a title=\"Publications_3_files/2001 - Tranah_sturgeonEtAl_NAJFM.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2001%20-%20Tranah_sturgeonEtAl_NAJFM.pdf\">pdf</a>).\r\n\r\n<strong>2000</strong>\r\n\r\n<span style=\"line-height: 1.714285714; font-size: 1rem;\">Agresti, J.J., S. Seki, A. Cnaani, S. Poompuang, E. M. Hallerman, N. Umiel, G. Hulata, G. A.E. Gall, and B. May. Breeding new strains of tilapia: development of an artificial center of origin and linkage map based on AFLP and microsatellite loci. Aquaculture. 185:43-56 (<a title=\"Publications_3_files/2000 - Tilapia.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2000%20-%20Tilapia.pdf\">pdf</a>).</span>\r\n\r\nBelfiore, N.M. and B. May. Variable microsatellite loci in red swamp crayfish, Procambarus clarkii, and their characterization in other crayfish taxa. Mol. Ecol. 12:2231-2234 (<a title=\"Publications_3_files/2000 - Belfiore_red swamp crayfish.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2000%20-%20Belfiore_red%20swamp%20crayfish.pdf\">pdf</a>).\r\n\r\nErnest, H.B., M.C.T. Penedo, B. May, M. Syvanen, and W.M. Boyce. Molecular tracking of mountain lions in the Yosemite Valley region in California: genetic analysis using microsatellites and fecal DNA. Mol. Ecol. 9: 433-441 (<a title=\"Publications_3_files/2000 - Ernest_Mountain Lions.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2000%20-%20Ernest_Mountain%20Lions.pdf\">pdf</a>).\r\n\r\nLudwig, A., B. May, L. Debus, and I, Jenneckens. Heteroplasmy in the mtDNA Control Region of Sturgeon (Acipenser, Huso and Scaphirhynchus) Genetics 156:1933-1947 (<a title=\"Publications_3_files/2000 - Ludwig_sturgeon.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2000%20-%20Ludwig_sturgeon.pdf\">pdf</a>).\r\n\r\nMcQuown, E.C., B.L. Sloss, R.J. Sheehan, J. Rodzen, G. Tranah, and B. May. Microsatellite analysis of genetic variation in sturgeon: new primer sequences for Scaphirynchus and Acipenser. Trans. Am. Fish. Soc. 129:1380-1388 (<a title=\"Publications_3_files/2000 - McQuowntafs1380.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_3_files/2000%20-%20McQuowntafs1380.pdf\">pdf</a>).\r\n\r\n<strong>Later Publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/\">2009-2013</a></li>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n</ul>\r\n<strong>Earlier Publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications 2000-2004','','inherit','closed','open','','135-revision-v1','','','2014-03-07 20:12:27','2014-03-07 20:12:27','',135,'http://localhost/~psalessi/Genetics_Lab/135-revision-v1/',0,'revision','',0),(305,1,'2014-03-07 20:18:44','2014-03-07 20:18:44','<strong>1999</strong>\r\n\r\nGavin, T.A., P.W. Sherman, E. Yensen, and B. May. Taxonomic status of Spermophilus brunneus and S. b. endemicus with comparison to other western spermophilus. J. Mam. 80:156-168.\r\n\r\nSeki, S, J.J. Agresti, , G.A.E. Gall, N. Taniguchi, and B. May. AFLP analysis of genetic diversity in three populations of ayu (Plecoglossus altivelis). Fish. Sci. 65:888-899.\r\n\r\n<strong>1998</strong>\r\n\r\nGavin, T.A., R.T. Reynolds, S.M. Joy, and B. May. Genetic evidence for low frequency of extra-pair fertilizations in Northern goshawks. Condor. 100:556-560 (<a title=\"Publications_4_files/1998 - Goshawks extra-pair.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1998%20-%20Goshawks%20extra-pair.pdf\">pdf</a>).\r\n\r\nMay, B. Starch gel electrophoresis of allozymes. In: Molecular Genetic Analysis of Populations: A Practical Approach. 2nd Ed. A.R. Hoelzel, ed. Oxford Univ. Press. Pp. 1-28 and 371-378.\r\n\r\nMay, B. Book Review. Molecular Systematics of Fishes, T.D. Kocher and C.A. Stepiens, eds. Trans. Am. Fish. Soc. 127:857-858.\r\n\r\nRodzen, J., J.J. Agresti, G. Tranah, and B. May. Agarose overlays allow simplified staining of polyacrylamide gels. Biotechniques. 25:584.\r\n\r\n<strong>1997</strong>\r\n\r\nMay, B., T.A. Gavin, P.W. Sherman, and T.M. Korves. Characterization of microsatellite loci in the Northern Idaho ground squirrel, Spermophilus brunneus Mol. Ecol. 6:399-400 (<a title=\"Publications_4_files/1997 - Spermophilus brunneus.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1997%20-%20Spermophilus%20brunneus.pdf\">pdf</a>).\r\n\r\nMay, B., C.C. Krueger, and H.L. Kincaid. Genetic variation at microsatellite loci in sturgeon: primer sequence homology in Acipenser and Scaphirhynchus. Can. J. Fish. &amp; Aquat. Sci. 54: 1542-1547 (<a title=\"Publications_4_files/1997 - May sturgeon 1997.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1997%20-%20May%20sturgeon%201997.pdf\">pdf</a>).\r\n\r\n<strong>1996</strong>\r\n\r\nLegge, J.T., R. Roush, B. May, R. DeSalle, and A. Vogler. Genetic criteria for establishing evolutionarily significant units in Cryan\'s buckmoth. Cons. Biol. 10(1):85-98 (<a title=\"Publications_4_files/1996 - Cryan\'s buckmoth.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1996%20-%20Cryan%27s%20buckmoth.pdf\">pdf</a>).\r\n\r\nMarsden, J.E., A. Spidle, and B. May. Review of genetic studies of Dreissena spp. Amer. Zool. 36:259-270 (<a title=\"Publications_4_files/1996 - marsden et al am zoo.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1996%20-%20marsden%20et%20al%20am%20zoo.pdf\">pdf</a>).\r\n\r\nMills, E.L., G. Rosenberg, A.P. Spidle, M. Ludyanskiy, Y. Pligin, and B. May. A review of the biology and ecology of the quagga mussel (Dreissena bugensis): a second species of freshwater dreissenid introduced to North America. Submitted to Amer. Zool. 36: 271-286.\r\n\r\n<strong>1995</strong>\r\n\r\nMarsden, J. E., A. Spidle, and B. May. Genetic similarity among zebra mussel poulations within North America and within Europe. Can. J. Fish. Aquat. Sci. 52:836-847 (<a title=\"Publications_4_files/1995 - marsden_et_al_zebra_mussel_structure[1].pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1995%20-%20marsden_et_al_zebra_mussel_structure%5B1%5D.pdf\">pdf</a>).\r\n\r\nPerkins, D.L., J. Fitzsimons, J.E. Marsden, C.C. Krueger, and B. May. Differences in reproduction among hatchery strains of lake trout at eight spawning areas in Lake Ontario: genetic evidence from mixed-stock analysis. J. Great Lakes Res. 21:364-374.\r\n\r\nSpidle, A.P., E.L. Mills, and B. May. Absence of naturally occurring hybridization between the quagga mussel (Dreissena bugensis) and the zebra mussel (D. polymorpha). Can. J. Zool. . 73:400-403 (<a title=\"Publications_4_files/1995 - spidle et al quagga zebe hyb.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1995%20-%20spidle%20et%20al%20quagga%20zebe%20hyb.pdf\">pdf</a>).\r\n\r\nSpidle, A.P., E.L. Mills, and B. May. Differential response to changes in salinity and temperature in the quagga and zebra mussels (Dreissena bugensis Andrusov and D. polymorpha Pallas). Can. J. Fish. Aquat. Sci. 52:2108-2119 (<a title=\"Publications_4_files/1995 - spidle et al dreissena physiology ocr.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1995%20-%20spidle%20et%20al%20dreissena%20physiology%20ocr.pdf\">pdf</a>).\r\n\r\n<strong>1994</strong>\r\n\r\nAiken, S.G., L.L. Consaul, A. Spidle, and B. May. Allozyme and morphological observations on Festuca hyperborea, compared with F. baffinensis and F. brachyphylla (Poaceae) from the Canadian Arctic. Nord. J. Bot. 14(2): 137-143.\r\n\r\nGrewe, P.M., C.C. Krueger, C.F. Aquadro, and B. May. Stability of allozyme and mitochondrial DNA markers among three year classes of lake trout propagated from Seneca Lake, New York. N. Amer. J. Fish. Man. 14:467-474 (<a title=\"Publications_4_files/1994 - GreweEtAl.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1994%20-%20GreweEtAl.pdf\">pdf</a>).\r\n\r\nGrewe, P.M., C.C. Krueger, J.E. Marsden, C.F. Aquadro, and B. May. Hatchery origins of naturally produced lake trout fry captured in Lake Ontario: temporal and spatial variability based on allozyme and mitochondrial DNA data. Trans. Am. Fish. Soc. 123:309-320.\r\n\r\nKrueger, C.C., D.L. Perkins, R.J. Everett, D.R. Schreiner, and B. May. Genetic variation in naturalized rainbow trout (steelhead) from Minnesota tributaries to Lake Superior. J. Great Lakes Res. 20:299-316.\r\n\r\nSpidle, A.P., J.E. Marsden, and B. May. Identification of the Great Lakes quagga mussel as Dreissena bugensis from the Dnieper River, Ukraine, on the basis of allozyme variation. Can. J. Fish. Aquat. Sci. 51:1485-1489 (<a title=\"Publications_4_files/1994 - spidle et al quagga id.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1994%20-%20spidle%20et%20al%20quagga%20id.pdf\">pdf</a>).\r\n\r\n<strong>1993</strong>\r\n\r\nGrewe, P.M., C.C. Krueger, C.F. Aquadro, E. Bermingham, H. L. Kincaid, and B. May. Mitochodrial DNA variation among lake trout (Salvelinus namaycush) strains stocked into Lake Ontario. Can J. Fish. Aquat. Sci. 50:2397-2403.\r\n\r\nKincaid, H. L. , C.C. Krueger, and B. May. Genetic variation among remnant groups of the Green Lake strain of lake trout: preservation of the remnant Green Lake gene pool. N. Am. J. Fish. Man. 13:318-325 (<a title=\"Publications_4_files/1993 - KincaidMay.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1993%20-%20KincaidMay.pdf\">pdf</a>).\r\n\r\nMarsden, J.E., C.C. Krueger, P.M. Grewe, H.L. Kincaid, and B. May. Genetic comparison of naturally-spawned and artificially-propagated Lake Ontario lake trout fry: evaluation of a stocking strategy for species rehabilitation. N. Am. J. Fish. Man. 13:304-317 (<a title=\"Publications_4_files/1993 - MarsdenEtAl.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_4_files/1993%20-%20MarsdenEtAl.pdf\">pdf</a>).\r\n\r\nMay, B. and P. M. Grewe. Fate of maternal mtDNA following Cobalt60 inactivation of maternal nuclear DNA in unfertilized salmonid eggs. Genome 36:725-730.\r\n\r\nMay, B. and K. R. Johnson. Composite linkage map of salmonid fishes (Salvelinus, Salmo, and Oncorhynchus). In: Genetic Maps : Locus Maps of Complex Genomes. O\'Brien, S.J. (ed.) Cold Spring Harbor. 4:309-317.\r\n\r\nPerkins, D.L., C.C. Krueger, and B. May. Heritage brook trout in Northeastern USA: genetic variability within and among populations. Trans. Am Fish. Soc.122:515-532.\r\n\r\nRoyse, D.J. and B. May. Multilocus enzyme electrophoresis for the genetic analysis of edible mushrooms. In: Genetics and Breeding of Edible Mushrooms. Chang, S.T., J.A. Buswell, and P.G. Miles (ed.). pp. 225-247. Gordon and Breach Sci. Pub. Phil.\r\n\r\n<strong>Later Publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/\">2009-2013</a></li>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n</ul>\r\n<strong>Earlier Publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications 1990-1992\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/142-2/\">1990-1992</a></li>\r\n</ul>','Publications 1993-1999','','inherit','closed','open','','140-revision-v1','','','2014-03-07 20:18:44','2014-03-07 20:18:44','',140,'http://localhost/~psalessi/Genetics_Lab/140-revision-v1/',0,'revision','',0),(306,1,'2014-03-07 20:20:59','2014-03-07 20:20:59','<strong>1992</strong>\r\n\r\nMay, B. Starch gel electrophoresis of allozymes. In: Molecular Genetic Analysis of Populations: A Practical Approach. A.R. Hoelzel, ed. Oxford Univ. Press. pp. 1-27 and 271-280.\r\n\r\nMay, B. and J.E. Marsden. Genetic identification and implications of another invasive species of Dreissenid mussel in the Great Lakes. Can. J. Fish. Aqua. Sci. 49:1501-1506.\r\n\r\nStaub, J.E., L.D. Knerr, D.J.Holder, and B. May. Phylogenetic relationships among several African Cucumis species. Can. J. Botany. 70:509-517.\r\n\r\nSt. Leger, R.J. Allee, L.L, May. B, Staples, R.C and Roberts, D.W. World-wide distribution of genetic variation among isolates of Beauveria spp. Mycol. Res. 96:1007-1015.\r\n\r\nSt. Leger, R. J., B. May, L. Allee, D. C. Frank, R. C. Staples, and D. W. Roberts. Genetic differences in allozymes and in formation of infection structures among isolates of the Entomopathogenic fungus Metarhizium anisopliae. J. Inv. Path. 60:89-101.\r\n\r\n<strong>1991</strong>\r\n\r\n<strong></strong>Bowden, C.G., D.J. Royse, and B. May. Linkage relationships of allozyme encoding loci in shitake, Lentinula edodes. Genome 34:652-657.\r\n\r\nGavin, T.A., R.A. Howard, and B. May. Geographic variation in allozyme frequencies in red-winged blackbirds: the California conundrum. Auk 108:602-611 (<a title=\"Publications_5_files/1991 - Red-winged blackbirds.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_5_files/1991%20-%20Red-winged%20blackbirds.pdf\">pdf</a>).\r\n\r\nKrueger, C.C., and B. May. Ecological and Genetic Effects of Salmonid Introductions in North America. Can. J. Fish. &amp; Aq. Sci. 48:1-12.\r\n\r\nKukuk, P.F. and B. May. Colony dynamics in a primitively eusocial halictine bee, Lasioglossum (Dialictus) zephyrus, (Hymenoptera: halictidae). Insectes Sociaux 38:171-189.\r\n\r\n<strong>1990</strong>\r\n\r\nDixon, P. M., and B. May. Genetic diversity and population structure of a rare plant, Northern Monkshood (Aconitum noveboracense). In:Ecosystem Management: rare species and significant habitats. Proc. 15th Natural Areas Meeting. NY state museum bulletin No. 471. pp. 167-175.\r\n\r\nHajek, A.E., R.A. Humber, J.S. Elkinton, B. May, S.R. Walsh, and J.C. Silver. Allozyme and RFLP analyses confirm Entomophaga maimaiga responsible for 1989 epizootics in North American gypsy moth populations. PNAS. 87:6979-6982 (<a title=\"Publications_5_files/1990 - Entomophaga maimaiga.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_5_files/1990%20-%20Entomophaga%20maimaiga.pdf\">pdf</a>).\r\n\r\nKukuk, P.F. and B. May. Diploid males in a primitively eusocial bee, Lasioglossum (Dialictus) zephyrum (Hymenoptera: Halictidae). Evolution 44:1522-1528 (<a title=\"Publications_5_files/1990 - Lasioglossum (Dialictus) zephyrum.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_5_files/1990%20-%20Lasioglossum%20%28Dialictus%29%20zephyrum.pdf\">pdf</a>).\r\n\r\nMay, B. and C.C. Krueger. Use of allozyme data for population analysis. In: Electrophoretic and Isoelectric Focussing Techniques in Fisheries Management. D.H. Whitmore, (ed.). pp. 157-172. CRC Press.\r\n\r\nRoyse, D.J., and B. May. Interspecific allozyme variation among Morchella spp. and its inferences for systematics within the genus. Biochem. Syst. &amp; Ecol. 18:475-479 (<a title=\"Publications_5_files/1990 - Morchella spp.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Publications_5_files/1990%20-%20Morchella%20spp.pdf\">pdf</a>).\r\n\r\n<strong>Later publications:</strong>\r\n<ul>\r\n	<li><a title=\"Publications\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/\">2009-2013</a></li>\r\n	<li><a title=\"Publications 2005-2008\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2005-2008/\">2005-2008</a></li>\r\n	<li><a title=\"Publications 2000-2004\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-2000-2004/\">2000-2004</a></li>\r\n	<li><a title=\"Publications 1993-1999\" href=\"http://localhost/~psalessi/Genetics_Lab/publications/publications-1993-1999/\">1993-1999</a></li>\r\n</ul>','Publications 1990-1992','','inherit','closed','open','','142-revision-v1','','','2014-03-07 20:20:59','2014-03-07 20:20:59','',142,'http://localhost/~psalessi/Genetics_Lab/142-revision-v1/',0,'revision','',0),(307,1,'2014-03-07 20:55:48','2014-03-07 20:55:48','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\"><strong>For denaturing polyacrylamide gels</strong></span></h3>\r\n<strong>Amounts:</strong>\r\n\r\nFormamide: 200 ml\r\n\r\nBromophenol Blue: 10 mg\r\n\r\nXylene Cyanol FF: 10 mg\r\n\r\nEDTA: 4 ml of 0.5 M pH 8.0 stock\r\n<ul>\r\n	<li>Transfer into 50 mL conical tubes and store at -20</li>\r\n	<li>When supply in 4 runs low, thaw one tube and add to 4 beaker</li>\r\n</ul>\r\n&nbsp;','98% Formamide Loading Dye','','publish','closed','open','','98-formamide-loading-dye','','','2014-03-07 21:14:17','2014-03-07 21:14:17','',217,'http://localhost/~psalessi/Genetics_Lab/?page_id=307',0,'page','',0),(308,1,'2014-03-07 20:55:48','2014-03-07 20:55:48','<h3><strong>For denaturing polyacrylamide gels:</strong></h3>\r\n<strong>Amounts:</strong>\r\n\r\nFormamide: 200 ml\r\n\r\nBromophenol Blue: 10 mg\r\n\r\nXylene Cyanol FF: 10 mg\r\n\r\nEDTA: 4 ml of 0.5 M pH 8.0 stock\r\n<ul>\r\n	<li>Transfer into 50 mL conical tubes and store at -20</li>\r\n	<li>When supply in 4 runs low, thaw one tube and add to 4 beaker</li>\r\n</ul>\r\n&nbsp;','98% Formamide Loading Dye','','inherit','closed','open','','307-revision-v1','','','2014-03-07 20:55:48','2014-03-07 20:55:48','',307,'http://localhost/~psalessi/Genetics_Lab/307-revision-v1/',0,'revision','',0),(309,1,'2014-03-07 20:58:28','2014-03-07 20:58:28','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">0.05ug/ul for agarose gels</span></h3>\r\n<strong>Amount:</strong>\r\n\r\nLadder stock (1ug/ul): 10ul\r\n\r\nLoading dye (BPB/XC found in fridge): 40ul\r\n\r\nNanopure water: 150ul\r\n<ul>\r\n	<li>This makes 200ul - aliquot into two tubes</li>\r\n	<li>Use 1ul in a typical agarose gel</li>\r\n</ul>','Diluting 1Kb Plus Ladder with Dye','','publish','closed','open','','diluting-1kb-plus-ladder-with-dye','','','2014-03-07 21:38:16','2014-03-07 21:38:16','',217,'http://localhost/~psalessi/Genetics_Lab/?page_id=309',0,'page','',0),(310,1,'2014-03-07 20:58:28','2014-03-07 20:58:28','<h3>0.05ug/ul for agarose gels:</h3>\r\n<strong>Amount:</strong>\r\n\r\nLadder stock (1ug/ul): 10ul\r\n\r\nLoading dye (BPB/XC found in fridge): 40ul\r\n\r\nNanopure water: 150ul\r\n<ul>\r\n	<li>This makes 200ul - aliquot into two tubes</li>\r\n	<li>Use 1ul in a typical agarose gel</li>\r\n</ul>\r\n&nbsp;','Diluting 1Kb Plus Ladder with Dye','','inherit','closed','open','','309-revision-v1','','','2014-03-07 20:58:28','2014-03-07 20:58:28','',309,'http://localhost/~psalessi/Genetics_Lab/309-revision-v1/',0,'revision','',0),(311,1,'2014-03-07 20:59:43','2014-03-07 20:59:43','<strong>Buffers &amp; Stocks</strong>\r\n<ul>\r\n	<li><a title=\"98% Formamide Loading Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/98-formamide-loading-dye/\">Formamide Loading Dye</a></li>\r\n	<li><a title=\"Diluting 1Kb Plus Ladder with Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/diluting-1kb-plus-ladder-with-dye/\">Diluting 1kb Ladder/Dye</a></li>\r\n	<li><a title=\"GelcoLadder.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/GelcoLadder.html\">Gelco Fluorescein Ladder</a></li>\r\n	<li><a title=\"Proteinase_K.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Proteinase_K.html\">Proteinase K</a></li>\r\n	<li><a title=\"Glassware_Cleaner.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Glassware_Cleaner.html\">Glassware Cleaner</a></li>\r\n	<li><a title=\"dNTPs.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/dNTPs.html\">dNTPs (Promega)</a></li>\r\n	<li><a title=\"Tris-Cl.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Tris-Cl.html\">1M Tris-Cl</a></li>\r\n	<li><a title=\"EDTA.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/EDTA.html\">0.5M EDTA (pH 8.0)</a></li>\r\n	<li><a title=\"1XTE.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/1XTE.html\">1X TE (10mM Tris; 1mM EDTA)</a></li>\r\n	<li><a title=\"1XTLE.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/1XTLE.html\">1X TLE (10mM Tris; 0.1 mM EDTA)</a></li>\r\n	<li><a title=\"10XTNE.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/10XTNE.html\">10X TNE</a></li>\r\n	<li><a title=\"10XTBE.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/10XTBE.html\">10XTBE One-Shots</a></li>\r\n</ul>\r\n<strong>FujiFilm</strong>\r\n<ul>\r\n	<li><a title=\"DNAquant.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/DNAquant.html\">DNA quantification</a></li>\r\n</ul>\r\nABI 3730xl Maintenance\r\n<ul>\r\n	<li><a title=\"3730Maintenance.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/3730Maintenance.html\">3730xl Maintenance</a></li>\r\n	<li><a title=\"ProtKwash.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/ProtKwash.html\">Proteinase K Wash</a></li>\r\n</ul>\r\n<strong>Acrylamide gels</strong>\r\n<ul>\r\n	<li><a title=\"Acrylamide.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Acrylamide.html\">Acrylamide Gel</a></li>\r\n	<li><a title=\"AcGelpour.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/AcGelpour.html\">Pouring</a></li>\r\n	<li><a title=\"AcGelRun.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/AcGelRun.html\">Running</a></li>\r\n</ul>\r\n<strong>Fluidigm</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/FLUIDIGM EP1 PROTOCOL.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/FLUIDIGM%20EP1%20PROTOCOL.doc\">Fluidigm EP1 Protocol</a></li>\r\n</ul>\r\n<strong>Illumina</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/RAD-SEQ Protocol.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/RAD-SEQ%20Protocol.doc\">RAD-SEQ Protocol</a></li>\r\n</ul>\r\n<strong>Other</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/tissuesampleprot.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/tissuesampleprot.pdf\">Tissue Sampling</a></li>\r\n	<li><a title=\"Supplies.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Supplies.html\">Ordering and Receiving Supplies</a></li>\r\n	<li><a title=\"Supplies.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Supplies.html\">PCR Tips and Tricks</a></li>\r\n</ul>','Protocols and Procedures','','inherit','closed','open','','217-revision-v1','','','2014-03-07 20:59:43','2014-03-07 20:59:43','',217,'http://localhost/~psalessi/Genetics_Lab/217-revision-v1/',0,'revision','',0),(312,1,'2014-03-07 21:02:06','2014-03-07 21:02:06','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Dilution of Gel Company Fluorescein Ladder : 50 to 400 bp for Manual Acrylamide Gels</span></h3>\r\nGel Company Ladder Catalog Number: MFK400-MD\r\n\r\nTo make 350 ul total (aliquotted into 10 small tubes):\r\n\r\nIn a tube, add\r\n<ul>\r\n	<li>30 ul H<sub>2</sub>O</li>\r\n	<li>10 ul ladder</li>\r\n	<li>300 ul</li>\r\n</ul>\r\nMix and aliquot into 10 small tubes\r\n\r\nThis dilution is adequate to be visible without agarose-overlay staining when loading 1 ul of diluted ladder to a well. If you are going to use an agarose-overlay, you can dilute this dilution 1:1 with more loading dye.','Gelco Fluorescein Ladder','','publish','closed','open','','gelco-fluorescein-ladder','','','2014-03-07 21:19:06','2014-03-07 21:19:06','',217,'http://localhost/~psalessi/Genetics_Lab/?page_id=312',0,'page','',0),(313,1,'2014-03-07 21:02:06','2014-03-07 21:02:06','<h3><span style=\"text-decoration: underline;\">Dilution of Gel Company Fluorescein Ladder : 50 to 400 bp for Manual Acrylamide Gels</span></h3>\r\nGel Company Ladder Catalog Number: MFK400-MD\r\n\r\nTo make 350 ul total (aliquotted into 10 small tubes):\r\n\r\nIn a tube, add\r\n<ul>\r\n	<li>30 ul H2O</li>\r\n	<li>10 ul ladder</li>\r\n	<li>300 ul</li>\r\n</ul>\r\nMix and aliquot into 10 small tubes\r\n\r\nThis dilution is adequate to be visible without agarose-overlay staining when loading 1 ul of diluted ladder to a well. If you are going to use an agarose-overlay, you can dilute this dilution 1:1 with more loading dye.','Gelco Fluorescein Ladder','','inherit','closed','open','','312-revision-v1','','','2014-03-07 21:02:06','2014-03-07 21:02:06','',312,'http://localhost/~psalessi/Genetics_Lab/312-revision-v1/',0,'revision','',0),(314,1,'2014-03-07 21:02:51','2014-03-07 21:02:51','<h3><span style=\"text-decoration: underline;\">0.05ug/ul for agarose gels</span></h3>\r\n<strong>Amount:</strong>\r\n\r\nLadder stock (1ug/ul): 10ul\r\n\r\nLoading dye (BPB/XC found in fridge): 40ul\r\n\r\nNanopure water: 150ul\r\n<ul>\r\n	<li>This makes 200ul - aliquot into two tubes</li>\r\n	<li>Use 1ul in a typical agarose gel</li>\r\n</ul>\r\n&nbsp;','Diluting 1Kb Plus Ladder with Dye','','inherit','closed','open','','309-revision-v1','','','2014-03-07 21:02:51','2014-03-07 21:02:51','',309,'http://localhost/~psalessi/Genetics_Lab/309-revision-v1/',0,'revision','',0),(315,1,'2014-03-07 21:03:10','2014-03-07 21:03:10','<h3><span style=\"text-decoration: underline;\"><strong>For denaturing polyacrylamide gels</strong></span></h3>\r\n<strong>Amounts:</strong>\r\n\r\nFormamide: 200 ml\r\n\r\nBromophenol Blue: 10 mg\r\n\r\nXylene Cyanol FF: 10 mg\r\n\r\nEDTA: 4 ml of 0.5 M pH 8.0 stock\r\n<ul>\r\n	<li>Transfer into 50 mL conical tubes and store at -20</li>\r\n	<li>When supply in 4 runs low, thaw one tube and add to 4 beaker</li>\r\n</ul>\r\n&nbsp;','98% Formamide Loading Dye','','inherit','closed','open','','307-revision-v1','','','2014-03-07 21:03:10','2014-03-07 21:03:10','',307,'http://localhost/~psalessi/Genetics_Lab/307-revision-v1/',0,'revision','',0),(316,1,'2014-03-07 21:07:00','2014-03-07 21:07:00','<strong>For 100ml of storage buffer:</strong>\r\n<ul>\r\n	<li>50 ml Glycerol</li>\r\n	<li>1 ml 1M Tris-HCl pH 7.5</li>\r\n	<li>0.29g CaCl<sub>2</sub></li>\r\n	<li>ddH20 to 100ml</li>\r\n</ul>\r\n<strong>For 10 ml of Proteinase K stock solution:</strong>\r\n<ul>\r\n	<li>Put 10ml of storage buffer in a 15ml screw-cap tube</li>\r\n	<li>Add 100mg of Proteinase K</li>\r\n	<li>Mix well by inverting until crystals are dissolved</li>\r\n	<li>Put 1ml aliquots into 1.5ml microfuge tubes</li>\r\n</ul>\r\nStore at -20C','Proteinase K','','publish','closed','open','','proteinase-k','','','2014-03-07 21:19:41','2014-03-07 21:19:41','',217,'http://localhost/~psalessi/Genetics_Lab/?page_id=316',0,'page','',0),(317,1,'2014-03-07 21:07:00','2014-03-07 21:07:00','<strong>For 100ml of storage buffer:</strong>\r\n<ul>\r\n	<li>50 ml Glycerol</li>\r\n	<li>1 ml 1M Tris-HCl pH 7.5</li>\r\n	<li>0.29g CaCl2</li>\r\n	<li>ddH20 to 100ml</li>\r\n</ul>\r\n<strong>For 10 ml of Proteinase K stock solution:</strong>\r\n<ul>\r\n	<li>Put 10ml of storage buffer in a 15ml screw-cap tube</li>\r\n	<li>Add 100mg of Proteinase K</li>\r\n	<li>Mix well by inverting until crystals are dissolved</li>\r\n	<li>Put 1ml aliquots into 1.5ml microfuge tubes</li>\r\n</ul>\r\nStore at -20C','Proteinase K','','inherit','closed','open','','316-revision-v1','','','2014-03-07 21:07:00','2014-03-07 21:07:00','',316,'http://localhost/~psalessi/Genetics_Lab/316-revision-v1/',0,'revision','',0),(318,1,'2014-03-07 21:08:00','2014-03-07 21:08:00','<strong>Buffers &amp; Stocks</strong>\r\n<ul>\r\n	<li><a title=\"98% Formamide Loading Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/98-formamide-loading-dye/\">Formamide Loading Dye</a></li>\r\n	<li><a title=\"Diluting 1Kb Plus Ladder with Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/diluting-1kb-plus-ladder-with-dye/\">Diluting 1kb Ladder/Dye</a></li>\r\n	<li><a title=\"Gelco Fluorescein Ladder\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/gelco-fluorescein-ladder/\">Gelco Fluorescein Ladder</a></li>\r\n	<li><a title=\"Proteinase K\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/proteinase-k/\">Proteinase K</a></li>\r\n	<li><a title=\"Glassware_Cleaner.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Glassware_Cleaner.html\">Glassware Cleaner</a></li>\r\n	<li><a title=\"dNTPs.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/dNTPs.html\">dNTPs (Promega)</a></li>\r\n	<li><a title=\"Tris-Cl.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Tris-Cl.html\">1M Tris-Cl</a></li>\r\n	<li><a title=\"EDTA.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/EDTA.html\">0.5M EDTA (pH 8.0)</a></li>\r\n	<li><a title=\"1XTE.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/1XTE.html\">1X TE (10mM Tris; 1mM EDTA)</a></li>\r\n	<li><a title=\"1XTLE.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/1XTLE.html\">1X TLE (10mM Tris; 0.1 mM EDTA)</a></li>\r\n	<li><a title=\"10XTNE.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/10XTNE.html\">10X TNE</a></li>\r\n	<li><a title=\"10XTBE.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/10XTBE.html\">10XTBE One-Shots</a></li>\r\n</ul>\r\n<strong>FujiFilm</strong>\r\n<ul>\r\n	<li><a title=\"DNAquant.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/DNAquant.html\">DNA quantification</a></li>\r\n</ul>\r\nABI 3730xl Maintenance\r\n<ul>\r\n	<li><a title=\"3730Maintenance.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/3730Maintenance.html\">3730xl Maintenance</a></li>\r\n	<li><a title=\"ProtKwash.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/ProtKwash.html\">Proteinase K Wash</a></li>\r\n</ul>\r\n<strong>Acrylamide gels</strong>\r\n<ul>\r\n	<li><a title=\"Acrylamide.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Acrylamide.html\">Acrylamide Gel</a></li>\r\n	<li><a title=\"AcGelpour.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/AcGelpour.html\">Pouring</a></li>\r\n	<li><a title=\"AcGelRun.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/AcGelRun.html\">Running</a></li>\r\n</ul>\r\n<strong>Fluidigm</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/FLUIDIGM EP1 PROTOCOL.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/FLUIDIGM%20EP1%20PROTOCOL.doc\">Fluidigm EP1 Protocol</a></li>\r\n</ul>\r\n<strong>Illumina</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/RAD-SEQ Protocol.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/RAD-SEQ%20Protocol.doc\">RAD-SEQ Protocol</a></li>\r\n</ul>\r\n<strong>Other</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/tissuesampleprot.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/tissuesampleprot.pdf\">Tissue Sampling</a></li>\r\n	<li><a title=\"Supplies.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Supplies.html\">Ordering and Receiving Supplies</a></li>\r\n	<li><a title=\"Supplies.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Supplies.html\">PCR Tips and Tricks</a></li>\r\n</ul>','Protocols and Procedures','','inherit','closed','open','','217-revision-v1','','','2014-03-07 21:08:00','2014-03-07 21:08:00','',217,'http://localhost/~psalessi/Genetics_Lab/217-revision-v1/',0,'revision','',0),(319,1,'2014-03-07 21:11:52','2014-03-07 21:11:52','The following product can be made from common laboratory chemicals.\r\n\r\n<strong>Alcoholic Sodium (Potassium) Hydroxide1</strong>\r\n<ul>\r\n	<li>Dissolve 120 g of sodium (or potassium) hydroxide in 120 ml water. Allow to cool.</li>\r\n	<li>Dilute to one liter with 95% ethyl alcohol. This solution acts as a general purpose cleaner.</li>\r\n</ul>\r\nREFERENCES 1. Gordon and Ford. 1972. The Chemists\' Companion: A Handbook of Practical Data, Techniques and References, pp 428-429. John Wiley &amp; Sons. Rev. 04/97','Glassware Cleaner','','publish','closed','open','','glassware-cleaner','','','2014-03-07 21:11:52','2014-03-07 21:11:52','',217,'http://localhost/~psalessi/Genetics_Lab/?page_id=319',0,'page','',0),(320,1,'2014-03-07 21:11:52','2014-03-07 21:11:52','The following product can be made from common laboratory chemicals.\r\n\r\n<strong>Alcoholic Sodium (Potassium) Hydroxide1</strong>\r\n<ul>\r\n	<li>Dissolve 120 g of sodium (or potassium) hydroxide in 120 ml water. Allow to cool.</li>\r\n	<li>Dilute to one liter with 95% ethyl alcohol. This solution acts as a general purpose cleaner.</li>\r\n</ul>\r\nREFERENCES 1. Gordon and Ford. 1972. The Chemists\' Companion: A Handbook of Practical Data, Techniques and References, pp 428-429. John Wiley &amp; Sons. Rev. 04/97','Glassware Cleaner','','inherit','closed','open','','319-revision-v1','','','2014-03-07 21:11:52','2014-03-07 21:11:52','',319,'http://localhost/~psalessi/Genetics_Lab/319-revision-v1/',0,'revision','',0),(321,1,'2014-03-07 21:14:00','2014-03-07 21:14:00','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Making dNTPs (Promega)</span></h3>\r\n<strong>NOTES:</strong>\r\n\r\nStock is kept on top shelf of -80 freezer in the main lab\r\n\r\nStock dNTP concentrations are 100mM each (dATP, dCTP, dGTP, dTTP)\r\n\r\n<strong>DIRECTIONS:</strong>\r\n<ul>\r\n	<li>Pipette 250 ul of each dNTP into a 30ml conical tube</li>\r\n	<li>Add 9000 ul of PCR water(Makes 10 ml total of 10uM dNTP, or 2.5uM each dNTP)</li>\r\n	<li>Mix</li>\r\n	<li>Aliquot 500 ul of mixture into twenty 1.5ml tubes</li>\r\n</ul>','dNTPs (Promega)','','publish','closed','open','','dntps-promega','','','2014-03-07 21:35:27','2014-03-07 21:35:27','',217,'http://localhost/~psalessi/Genetics_Lab/?page_id=321',0,'page','',0),(322,1,'2014-03-07 21:14:00','2014-03-07 21:14:00','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Making dNTPs (Promega)</span></h3>\r\n<strong>NOTES:</strong>\r\n\r\nStock is kept on top shelf of -80 freezer in the main lab\r\n\r\nStock dNTP concentrations are 100mM each (dATP, dCTP, dGTP, dTTP)\r\n\r\n<strong>DIRECTIONS:</strong>\r\n<ul>\r\n	<li>Pipette 250 ul of each dNTP into a 30ml conical tube</li>\r\n	<li>Add 9000 ul of PCR water(Makes 10 ml total of 10uM dNTP, or 2.5uM each dNTP)</li>\r\n	<li>Mix</li>\r\n	<li>Aliquot 500 ul of mixture into twenty 1.5ml tubes</li>\r\n</ul>','dNTPs (Promega)','','inherit','closed','open','','321-revision-v1','','','2014-03-07 21:14:00','2014-03-07 21:14:00','',321,'http://localhost/~psalessi/Genetics_Lab/321-revision-v1/',0,'revision','',0),(323,1,'2014-03-07 21:14:17','2014-03-07 21:14:17','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\"><strong>For denaturing polyacrylamide gels</strong></span></h3>\r\n<strong>Amounts:</strong>\r\n\r\nFormamide: 200 ml\r\n\r\nBromophenol Blue: 10 mg\r\n\r\nXylene Cyanol FF: 10 mg\r\n\r\nEDTA: 4 ml of 0.5 M pH 8.0 stock\r\n<ul>\r\n	<li>Transfer into 50 mL conical tubes and store at -20</li>\r\n	<li>When supply in 4 runs low, thaw one tube and add to 4 beaker</li>\r\n</ul>\r\n&nbsp;','98% Formamide Loading Dye','','inherit','closed','open','','307-revision-v1','','','2014-03-07 21:14:17','2014-03-07 21:14:17','',307,'http://localhost/~psalessi/Genetics_Lab/307-revision-v1/',0,'revision','',0),(324,1,'2014-03-07 21:14:36','2014-03-07 21:14:36','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">0.05ug/ul for agarose gels</span></h3>\r\n<strong>Amount:</strong>\r\n\r\nLadder stock (1ug/ul): 10ul\r\n\r\nLoading dye (BPB/XC found in fridge): 40ul\r\n\r\nNanopure water: 150ul\r\n<ul>\r\n	<li>This makes 200ul - aliquot into two tubes</li>\r\n	<li>Use 1ul in a typical agarose gel</li>\r\n</ul>\r\n&nbsp;','Diluting 1Kb Plus Ladder with Dye','','inherit','closed','open','','309-revision-v1','','','2014-03-07 21:14:36','2014-03-07 21:14:36','',309,'http://localhost/~psalessi/Genetics_Lab/309-revision-v1/',0,'revision','',0),(325,1,'2014-03-07 21:14:54','2014-03-07 21:14:54','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Dilution of Gel Company Fluorescein Ladder : 50 to 400 bp for Manual Acrylamide Gels</span></h3>\r\nGel Company Ladder Catalog Number: MFK400-MD\r\n\r\nTo make 350 ul total (aliquotted into 10 small tubes):\r\n\r\nIn a tube, add\r\n<ul>\r\n	<li>30 ul H2O</li>\r\n	<li>10 ul ladder</li>\r\n	<li>300 ul</li>\r\n</ul>\r\nMix and aliquot into 10 small tubes\r\n\r\nThis dilution is adequate to be visible without agarose-overlay staining when loading 1 ul of diluted ladder to a well. If you are going to use an agarose-overlay, you can dilute this dilution 1:1 with more loading dye.','Gelco Fluorescein Ladder','','inherit','closed','open','','312-revision-v1','','','2014-03-07 21:14:54','2014-03-07 21:14:54','',312,'http://localhost/~psalessi/Genetics_Lab/312-revision-v1/',0,'revision','',0),(326,1,'2014-03-07 21:17:04','2014-03-07 21:17:04','Dissolve 121.1 g of tris base (sigma # T-1503) in 700 ml of ddH<sub>2</sub>O.\r\n\r\nBring to desired pH (Usually 7.5 or 8.0) with concentrated HCl.\r\n\r\nBring volume to 1L with ddH<sub>2</sub>O.\r\n\r\nFilter with 0.5 micron filter.\r\n\r\nAutoclave.\r\n\r\nStore at room temperature.','1M Tris-Cl','','publish','closed','open','','1m-tris-cl','','','2014-03-07 21:17:17','2014-03-07 21:17:17','',217,'http://localhost/~psalessi/Genetics_Lab/?page_id=326',0,'page','',0),(327,1,'2014-03-07 21:17:04','2014-03-07 21:17:04','Dissolve 121.1 g of tris base (sigma # T-1503) in 700 ml of ddH<sub>2</sub>O.\r\n\r\nBring to desired pH (Usually 7.5 or 8.0) with concentrated HCl.\r\n\r\nBring volume to 1L with ddH<sub>2</sub>O.\r\n\r\nFilter with 0.5 micron filter.\r\n\r\nAutoclave.\r\n\r\nStore at room temperature.','1M Tris-Cl','','inherit','closed','open','','326-revision-v1','','','2014-03-07 21:17:04','2014-03-07 21:17:04','',326,'http://localhost/~psalessi/Genetics_Lab/326-revision-v1/',0,'revision','',0),(328,1,'2014-03-07 21:18:43','2014-03-07 21:18:43','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Dilution of Gel Company Fluorescein Ladder : 50 to 400 bp for Manual Acrylamide Gels</span></h3>\nGel Company Ladder Catalog Number: MFK400-MD\n\nTo make 350 ul total (aliquotted into 10 small tubes):\n\nIn a tube, add\n<ul>\n	<li>30 ul H<sub>2</sub>O</li>\n	<li>10 ul ladder</li>\n	<li>300 ul</li>\n</ul>\nMix and aliquot into 10 small tubes\n\nThis dilution is adequate to be visible without agarose-overlay staining when loading 1 ul of diluted ladder to a well. If you are going to use an agarose-overlay, you can dilute this dilution 1:1 with more loading dye.','Gelco Fluorescein Ladder','','inherit','closed','open','','312-autosave-v1','','','2014-03-07 21:18:43','2014-03-07 21:18:43','',312,'http://localhost/~psalessi/Genetics_Lab/312-autosave-v1/',0,'revision','',0),(329,1,'2014-03-07 21:18:46','2014-03-07 21:18:46','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Dilution of Gel Company Fluorescein Ladder : 50 to 400 bp for Manual Acrylamide Gels</span></h3>\r\nGel Company Ladder Catalog Number: MFK400-MD\r\n\r\nTo make 350 ul total (aliquotted into 10 small tubes):\r\n\r\nIn a tube, add\r\n<ul>\r\n	<li>30 ul H<sub>2</sub>O</li>\r\n	<li>10 ul ladder</li>\r\n	<li>300 ul</li>\r\n</ul>\r\nMix and aliquot into 10 small tubes\r\n\r\nThis dilution is adequate to be visible without agarose-overlay staining when loading 1 ul of diluted ladder to a well. If you are going to use an agarose-overlay, you can dilute this dilution 1:1 with more loading dye.','Gelco Fluorescein Ladder','','inherit','closed','open','','312-revision-v1','','','2014-03-07 21:18:46','2014-03-07 21:18:46','',312,'http://localhost/~psalessi/Genetics_Lab/312-revision-v1/',0,'revision','',0),(330,1,'2014-03-07 21:19:41','2014-03-07 21:19:41','<strong>For 100ml of storage buffer:</strong>\r\n<ul>\r\n	<li>50 ml Glycerol</li>\r\n	<li>1 ml 1M Tris-HCl pH 7.5</li>\r\n	<li>0.29g CaCl<sub>2</sub></li>\r\n	<li>ddH20 to 100ml</li>\r\n</ul>\r\n<strong>For 10 ml of Proteinase K stock solution:</strong>\r\n<ul>\r\n	<li>Put 10ml of storage buffer in a 15ml screw-cap tube</li>\r\n	<li>Add 100mg of Proteinase K</li>\r\n	<li>Mix well by inverting until crystals are dissolved</li>\r\n	<li>Put 1ml aliquots into 1.5ml microfuge tubes</li>\r\n</ul>\r\nStore at -20C','Proteinase K','','inherit','closed','open','','316-revision-v1','','','2014-03-07 21:19:41','2014-03-07 21:19:41','',316,'http://localhost/~psalessi/Genetics_Lab/316-revision-v1/',0,'revision','',0),(331,1,'2014-03-07 21:21:23','2014-03-07 21:21:23','Add 186.1 g EDTA (disodium, dihydrate, Sigma # E-5134) to 800 ml of ddH<sub>2</sub>0.\r\n\r\nAdd about 20g of NaOH pellets while stirring to bring the pH to 8.0.\r\n\r\nAdd the last few grams slowly to avoid overshooting the pH.\r\n\r\nNote that the EDTA won\'t completely dissolve until the pH is around 8.\r\n\r\nFilter with 0.5 micron filter and autoclave.','.5M EDTA (pH 8.0)','','publish','closed','open','','5m-edta-ph-8-0','','','2014-03-07 21:23:45','2014-03-07 21:23:45','',217,'http://localhost/~psalessi/Genetics_Lab/?page_id=331',0,'page','',0),(332,1,'2014-03-07 21:21:23','2014-03-07 21:21:23','Add 186.1 g EDTA (disodium, dihydrate, Sigma # E-5134) to 800 ml of ddH<sub>2</sub>0.\r\n\r\nAdd about 20g of NaOH pellets while stirring to bring the pH to 8.0.\r\n\r\nAdd the last few grams slowly to avoid overshooting the pH.\r\n\r\nNote that the EDTA won\'t completely dissolve until the pH is around 8.\r\n\r\nFilter with 0.5 micron filter and autoclave.','.5M EDTA (pH 8.0)','','inherit','closed','open','','331-revision-v1','','','2014-03-07 21:21:23','2014-03-07 21:21:23','',331,'http://localhost/~psalessi/Genetics_Lab/331-revision-v1/',0,'revision','',0),(333,1,'2014-03-07 21:25:06','2014-03-07 21:25:06','Add 10 mL of 1M Tris-HCl (pH 8.0)* and 2 mL of 0.5M EDTA (pH 8.0)* to 988 ml of ddH<sub>2</sub>0.\r\n\r\nFilter with 0.5 micron filter and autoclave.\r\n\r\n<strong>*1 M Tris-Cl</strong>\r\n\r\nDissolve 121.1 g of tris base (sigma # T-1503) in 700 ml of ddH<sub>2</sub>O. Bring to desired pH (Usually 7.5 or 8.0) with concentrated HCl. Bring volume to 1L with ddH<sub>2</sub>O. Filter with 0.5 micron filter. Autoclave. Store at room temperature.\r\n\r\n<strong>*0.5 M EDTA</strong>\r\n\r\nAdd 186.1 g EDTA (disodium, dihydrate, Sigma # E-5134) to 800 ml of ddH20. Add about 20g of NaOH pellets while stirring to bring the pH to 8.0. Add the last few grams slowly to avoid overshooting the pH. Note that the EDTA won\'t completely dissolve until the pH is around 8. Filter with 0.5 micron filter and autoclave.','1X TE (10mM Tris; 1mM EDTA)','','publish','closed','open','','1x-te-10mm-tris-1mm-edta','','','2014-03-07 21:36:16','2014-03-07 21:36:16','',217,'http://localhost/~psalessi/Genetics_Lab/?page_id=333',0,'page','',0),(334,1,'2014-03-07 21:25:06','2014-03-07 21:25:06','Add 10 mL of 1M Tris-HCl (pH 8.0)* and 2 mL of 0.5M EDTA (pH 8.0)* to 988 ml of ddH<sub>2</sub>0.\r\n\r\nFilter with 0.5 micron filter and autoclave.\r\n\r\n<strong>*1 M Tris-Cl</strong>\r\n\r\nDissolve 121.1 g of tris base (sigma # T-1503) in 700 ml of ddH2O. Bring to desired pH (Usually 7.5 or 8.0) with concentrated HCl. Bring volume to 1L with ddH2O. Filter with 0.5 micron filter. Autoclave. Store at room temperature.\r\n\r\n<strong>*0.5 M EDTA</strong>\r\n\r\nAdd 186.1 g EDTA (disodium, dihydrate, Sigma # E-5134) to 800 ml of ddH20. Add about 20g of NaOH pellets while stirring to bring the pH to 8.0. Add the last few grams slowly to avoid overshooting the pH. Note that the EDTA won\'t completely dissolve until the pH is around 8. Filter with 0.5 micron filter and autoclave.','1X TE (10mM Tris; 1mM EDTA)','','inherit','closed','open','','333-revision-v1','','','2014-03-07 21:25:06','2014-03-07 21:25:06','',333,'http://localhost/~psalessi/Genetics_Lab/333-revision-v1/',0,'revision','',0),(335,1,'2014-03-07 21:27:06','2014-03-07 21:27:06','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">1X Tris- \"low\" EDTA (TLE) pH 8.0</span></h3>\r\nAdd 10 mL of 1M Tris-HCl (pH 8.0)* and 0.2 mL of 0.5M EDTA (pH 8.0)* to 989.8 (990) ml of ddH<sub>2</sub>0.\r\n\r\nFilter with 0.5 micron filter and autoclave.\r\n\r\n<strong>*1 M Tris-Cl</strong>\r\n\r\nDissolve 121.1 g of tris base (sigma # T-1503) in 700 ml of ddH<sub>2</sub>O. Bring to desired pH (Usually 7.5 or 8.0) with concentrated HCl. Bring volume to 1L with ddH<sub>2</sub>O. Filter with 0.5 micron filter. Autoclave. Store at room temperature.\r\n\r\n<strong>*0.5 M EDTA</strong>\r\n\r\nAdd 186.1 g EDTA (disodium, dihydrate, Sigma # E-5134) to 800 ml of ddH<sub>2</sub>0. Add about 20g of NaOH pellets while stirring to bring the pH to 8.0. Add the last few grams slowly to avoid overshooting the pH. Note that the EDTA won\'t completely dissolve until the pH is around 8. Filter with 0.5 micron filter and autoclave.','1X TLE (10mM Tris; 0.1mM EDTA)','','publish','closed','open','','1x-tle-10mm-tris-0-1mm-edta','','','2014-03-07 21:36:41','2014-03-07 21:36:41','',217,'http://localhost/~psalessi/Genetics_Lab/?page_id=335',0,'page','',0),(336,1,'2014-03-07 21:27:06','2014-03-07 21:27:06','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">1X Tris- \"low\" EDTA (TLE) pH 8.0</span></h3>\r\nAdd 10 mL of 1M Tris-HCl (pH 8.0)* and 0.2 mL of 0.5M EDTA (pH 8.0)* to 989.8 (990) ml of ddH20.\r\n\r\nFilter with 0.5 micron filter and autoclave.\r\n\r\n<strong>*1 M Tris-Cl</strong>\r\n\r\nDissolve 121.1 g of tris base (sigma # T-1503) in 700 ml of ddH<sub>2</sub>O. Bring to desired pH (Usually 7.5 or 8.0) with concentrated HCl. Bring volume to 1L with ddH<sub>2</sub>O. Filter with 0.5 micron filter. Autoclave. Store at room temperature.\r\n\r\n<strong>*0.5 M EDTA</strong>\r\n\r\nAdd 186.1 g EDTA (disodium, dihydrate, Sigma # E-5134) to 800 ml of ddH<sub>2</sub>0. Add about 20g of NaOH pellets while stirring to bring the pH to 8.0. Add the last few grams slowly to avoid overshooting the pH. Note that the EDTA won\'t completely dissolve until the pH is around 8. Filter with 0.5 micron filter and autoclave.','1X TLE (10mM Tris; 0.1mM EDTA)','','inherit','closed','open','','335-revision-v1','','','2014-03-07 21:27:06','2014-03-07 21:27:06','',335,'http://localhost/~psalessi/Genetics_Lab/335-revision-v1/',0,'revision','',0),(337,1,'2014-03-07 21:29:07','2014-03-07 21:29:07','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">10X TNE Buffer (100mM Tris; 2.0M NaCl; 10mM EDTA; pH 7.4)</span></h3>\r\nTo 800ml ddH<sub>2</sub>O add:\r\n<ul>\r\n	<li>Trizma Base: 12.1g T-1503</li>\r\n	<li>disodium EDTA: 3.7g E-5134</li>\r\n	<li>NaCl: 116.8g</li>\r\n	<li>pH 7.4 with Conc. HCl</li>\r\n	<li>Bring to volume of 1L with ddH<sub>2</sub>O.</li>\r\n	<li>Filter and autoclave</li>\r\n</ul>\r\nStore at 4 C','10X TNE','','publish','closed','open','','10x-tne','','','2014-03-07 21:38:12','2014-03-07 21:38:12','',217,'http://localhost/~psalessi/Genetics_Lab/?page_id=337',0,'page','',0),(338,1,'2014-03-07 21:29:07','2014-03-07 21:29:07','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">10X TNE Buffer (100mM Tris; 2.0M NaCl; 10mM EDTA; pH 7.4)</span></h3>\r\nTo 800ml ddH2O add:\r\n<ul>\r\n	<li>Trizma Base: 12.1g T-1503</li>\r\n	<li>disodium EDTA: 3.7g E-5134</li>\r\n	<li>NaCl: 116.8g</li>\r\n	<li>pH 7.4 with Conc. HCl</li>\r\n	<li>Bring to volume of 1L with ddH2O.</li>\r\n	<li>Filter and autoclave</li>\r\n</ul>\r\nStore at 4 C','10X TNE','','inherit','closed','open','','337-revision-v1','','','2014-03-07 21:29:07','2014-03-07 21:29:07','',337,'http://localhost/~psalessi/Genetics_Lab/337-revision-v1/',0,'revision','',0),(339,1,'2014-03-07 21:29:20','2014-03-07 21:29:20','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">10X TNE Buffer (100mM Tris; 2.0M NaCl; 10mM EDTA; pH 7.4)</span></h3>\r\nTo 800ml ddH<sub>2</sub>O add:\r\n<ul>\r\n	<li>Trizma Base: 12.1g T-1503</li>\r\n	<li>disodium EDTA: 3.7g E-5134</li>\r\n	<li>NaCl: 116.8g</li>\r\n	<li>pH 7.4 with Conc. HCl</li>\r\n	<li>Bring to volume of 1L with ddH2O.</li>\r\n	<li>Filter and autoclave</li>\r\n</ul>\r\nStore at 4 C','10X TNE','','inherit','closed','open','','337-revision-v1','','','2014-03-07 21:29:20','2014-03-07 21:29:20','',337,'http://localhost/~psalessi/Genetics_Lab/337-revision-v1/',0,'revision','',0),(340,1,'2014-03-07 21:32:19','2014-03-07 21:32:19','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">10X TBE(1.1M Tris; 900mM Borate; 25mM EDTA; pH 8.3)</span></h3>\r\n<p style=\"text-align: center;\"><span style=\"color: #ff0000;\">****Note: Boric acid dissolves at a pH around 7. One may alternate between adding Trizma Base &amp; Boric acid. Dissolve the EDTA first.****</span></p>\r\nTo 2.5L H<sub>2</sub>O add:\r\n\r\ndisodium EDTA: 37.5g E-5134\r\nTrizma Base: 432g T-1503\r\nBoric Acid: 220g B-0252\r\n<ul>\r\n	<li>Dissolve and bring pH to 8.3</li>\r\n	<li>Dilute to 4L</li>\r\n	<li><span style=\"line-height: 1.714285714; font-size: 1rem;\">Filter into 5 \"One-Shot\" bottles of 800ml each and autoclave.</span></li>\r\n</ul>\r\n<strong>To make 1X TBE:</strong>\r\n<ul>\r\n	<li>Pour entire one-shot into 8L carboy</li>\r\n	<li>Fill to 8L mark with dH<sub>2</sub>O</li>\r\n</ul>','10X TBE One-Shots','','publish','closed','open','','10x-tbe-one-shots','','','2014-03-07 21:38:02','2014-03-07 21:38:02','',217,'http://localhost/~psalessi/Genetics_Lab/?page_id=340',0,'page','',0),(341,1,'2014-03-07 21:32:19','2014-03-07 21:32:19','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">10X TBE(1.1M Tris; 900mM Borate; 25mM EDTA; pH 8.3)</span></h3>\r\n<p style=\"text-align: center;\"><span style=\"color: #ff0000;\">****Note: Boric acid dissolves at a pH around 7. One may alternate between adding Trizma Base &amp; Boric acid. Dissolve the EDTA first.****</span></p>\r\nTo 2.5L H2O add:\r\n\r\ndisodium EDTA: 37.5g E-5134\r\nTrizma Base: 432g T-1503\r\nBoric Acid: 220g B-0252\r\n<ul>\r\n	<li>Dissolve and bring pH to 8.3</li>\r\n	<li>Dilute to 4L</li>\r\n	<li><span style=\"line-height: 1.714285714; font-size: 1rem;\">Filter into 5 \"One-Shot\" bottles of 800ml each and autoclave.</span></li>\r\n</ul>\r\n<strong>To make 1X TBE:</strong>\r\n<ul>\r\n	<li>Pour entire one-shot into 8L carboy</li>\r\n	<li>Fill to 8L mark with dH2O</li>\r\n</ul>','10X TBE One-Shots','','inherit','closed','open','','340-revision-v1','','','2014-03-07 21:32:19','2014-03-07 21:32:19','',340,'http://localhost/~psalessi/Genetics_Lab/340-revision-v1/',0,'revision','',0),(342,1,'2014-03-10 18:12:11','2014-03-10 18:12:11','<strong>Buffers &amp; Stocks</strong>\n<ul>\n	<li><a title=\"98% Formamide Loading Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/98-formamide-loading-dye/\">Formamide Loading Dye</a></li>\n	<li><a title=\"Diluting 1Kb Plus Ladder with Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/diluting-1kb-plus-ladder-with-dye/\">Diluting 1kb Ladder/Dye</a></li>\n	<li><a title=\"Gelco Fluorescein Ladder\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/gelco-fluorescein-ladder/\">Gelco Fluorescein Ladder</a></li>\n	<li><a title=\"Proteinase K\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/proteinase-k/\">Proteinase K</a></li>\n	<li><a title=\"Glassware Cleaner\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/glassware-cleaner/\">Glassware Cleaner</a></li>\n	<li><a title=\"dNTPs (Promega)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/dntps-promega/\">dNTPs (Promega)</a></li>\n	<li><a title=\"1M Tris-Cl\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1m-tris-cl/\">1M Tris-Cl</a></li>\n	<li><a title=\".5M EDTA (pH 8.0)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/5m-edta-ph-8-0/\">0.5M EDTA (pH 8.0)</a></li>\n	<li><a title=\"1X TE (10mM Tris; 1mM EDTA)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1x-te-10mm-tris-1mm-edta/\">1X TE (10mM Tris; 1mM EDTA)</a></li>\n	<li><a title=\"1X TLE (10mM Tris; 0.1mM EDTA)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1x-tle-10mm-tris-0-1mm-edta/\">1X TLE (10mM Tris; 0.1 mM EDTA)</a></li>\n	<li><a title=\"10X TNE\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/10x-tne/\">10X TNE</a></li>\n	<li><a title=\"10X TBE One-Shots\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/10x-tbe-one-shots/\">10X TBE One-Shots</a></li>\n</ul>\n<strong>FujiFilm</strong>\n<ul>\n	<li><a title=\"FujiFilm DNA Quantification\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/fujifilm-dna-quantification/\">DNA quantification</a></li>\n</ul>\n<strong>ABI 3730xl Maintenance</strong>\n<ul>\n	<li><a title=\"3730xl General Maintenance\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/3730xl-general-maintenance/\">3730xl Maintenance</a></li>\n	<li><a title=\"Proteinase K Wash\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/proteinase-k-wash/\">Proteinase K Wash</a></li>\n</ul>\n<strong>Acrylamide gels</strong>\n<ul>\n	<li><a title=\"Acrylamide (Manual Gel)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/acrylamide-manual-gel/\">Acrylamide Gel</a></li>\n	<li><a title=\"Acrylamide Gel Pour\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/acrylamide-gel-pour/\">Pouring</a></li>\n	<li><a title=\"Running Acrylamide\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/running-acrylamide/\">Running</a></li>\n</ul>\n<strong>Fluidigm</strong>\n<ul>\n	<li><a title=\"Protocols_files/FLUIDIGM EP1 PROTOCOL.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/FLUIDIGM%20EP1%20PROTOCOL.doc\">Fluidigm EP1 Protocol</a></li>\n</ul>\n<strong>Illumina</strong>\n<ul>\n	<li><a title=\"Protocols_files/RAD-SEQ Protocol.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/RAD-SEQ%20Protocol.doc\">RAD-SEQ Protocol</a></li>\n</ul>\n<strong>Other</strong>\n<ul>\n	<li><a title=\"Protocols_files/tissuesampleprot.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/tissuesampleprot.pdf\">Tissue Sampling</a></li>\n	<li><a title=\"Supplies\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/supplies/\">Ordering and Receiving Supplies</a></li>\n	<li><a title=\"Supplies.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Supplies.html\">PCR Tips and Tricks</a></li>\n</ul>','Protocols and Procedures','','inherit','closed','open','','217-autosave-v1','','','2014-03-10 18:12:11','2014-03-10 18:12:11','',217,'http://localhost/~psalessi/Genetics_Lab/217-autosave-v1/',0,'revision','',0),(343,1,'2014-03-07 21:33:41','2014-03-07 21:33:41','<strong>Buffers &amp; Stocks</strong>\r\n<ul>\r\n	<li><a title=\"98% Formamide Loading Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/98-formamide-loading-dye/\">Formamide Loading Dye</a></li>\r\n	<li><a title=\"Diluting 1Kb Plus Ladder with Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/diluting-1kb-plus-ladder-with-dye/\">Diluting 1kb Ladder/Dye</a></li>\r\n	<li><a title=\"Gelco Fluorescein Ladder\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/gelco-fluorescein-ladder/\">Gelco Fluorescein Ladder</a></li>\r\n	<li><a title=\"Proteinase K\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/proteinase-k/\">Proteinase K</a></li>\r\n	<li><a title=\"Glassware Cleaner\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/glassware-cleaner/\">Glassware Cleaner</a></li>\r\n	<li><a title=\"dNTPs (Promega)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/dntps-promega/\">dNTPs (Promega)</a></li>\r\n	<li><a title=\"1M Tris-Cl\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1m-tris-cl/\">1M Tris-Cl</a></li>\r\n	<li><a title=\".5M EDTA (pH 8.0)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/5m-edta-ph-8-0/\">0.5M EDTA (pH 8.0)</a></li>\r\n	<li><a title=\"1X TE (10mM Tris; 1mM EDTA)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1x-te-10mm-tris-1mm-edta/\">1X TE (10mM Tris; 1mM EDTA)</a></li>\r\n	<li><a title=\"1X TLE (10mM Tris; 0.1mM EDTA)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1x-tle-10mm-tris-0-1mm-edta/\">1X TLE (10mM Tris; 0.1 mM EDTA)</a></li>\r\n	<li><a title=\"10X TNE\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/10x-tne/\">10X TNE</a></li>\r\n	<li><a title=\"10X TBE One-Shots\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/10x-tbe-one-shots/\">10X TBE One-Shots</a></li>\r\n</ul>\r\n<strong>FujiFilm</strong>\r\n<ul>\r\n	<li><a title=\"DNAquant.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/DNAquant.html\">DNA quantification</a></li>\r\n</ul>\r\nABI 3730xl Maintenance\r\n<ul>\r\n	<li><a title=\"3730Maintenance.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/3730Maintenance.html\">3730xl Maintenance</a></li>\r\n	<li><a title=\"ProtKwash.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/ProtKwash.html\">Proteinase K Wash</a></li>\r\n</ul>\r\n<strong>Acrylamide gels</strong>\r\n<ul>\r\n	<li><a title=\"Acrylamide.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Acrylamide.html\">Acrylamide Gel</a></li>\r\n	<li><a title=\"AcGelpour.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/AcGelpour.html\">Pouring</a></li>\r\n	<li><a title=\"AcGelRun.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/AcGelRun.html\">Running</a></li>\r\n</ul>\r\n<strong>Fluidigm</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/FLUIDIGM EP1 PROTOCOL.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/FLUIDIGM%20EP1%20PROTOCOL.doc\">Fluidigm EP1 Protocol</a></li>\r\n</ul>\r\n<strong>Illumina</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/RAD-SEQ Protocol.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/RAD-SEQ%20Protocol.doc\">RAD-SEQ Protocol</a></li>\r\n</ul>\r\n<strong>Other</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/tissuesampleprot.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/tissuesampleprot.pdf\">Tissue Sampling</a></li>\r\n	<li><a title=\"Supplies.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Supplies.html\">Ordering and Receiving Supplies</a></li>\r\n	<li><a title=\"Supplies.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Supplies.html\">PCR Tips and Tricks</a></li>\r\n</ul>','Protocols and Procedures','','inherit','closed','open','','217-revision-v1','','','2014-03-07 21:33:41','2014-03-07 21:33:41','',217,'http://localhost/~psalessi/Genetics_Lab/217-revision-v1/',0,'revision','',0),(344,1,'2014-03-07 21:34:38','2014-03-07 21:34:38','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">10X TBE(1.1M Tris; 900mM Borate; 25mM EDTA; pH 8.3)</span></h3>\r\n<p style=\"text-align: center;\"><span style=\"color: #ff0000;\">****Note: Boric acid dissolves at a pH around 7. One may alternate between adding Trizma Base &amp; Boric acid. Dissolve the EDTA first.****</span></p>\r\nTo 2.5L H<sub>2</sub>O add:\r\n\r\ndisodium EDTA: 37.5g E-5134\r\nTrizma Base: 432g T-1503\r\nBoric Acid: 220g B-0252\r\n<ul>\r\n	<li>Dissolve and bring pH to 8.3</li>\r\n	<li>Dilute to 4L</li>\r\n	<li><span style=\"line-height: 1.714285714; font-size: 1rem;\">Filter into 5 \"One-Shot\" bottles of 800ml each and autoclave.</span></li>\r\n</ul>\r\n<strong>To make 1X TBE:</strong>\r\n<ul>\r\n	<li>Pour entire one-shot into 8L carboy</li>\r\n	<li>Fill to 8L mark with dH2O</li>\r\n</ul>','10X TBE One-Shots','','inherit','closed','open','','340-revision-v1','','','2014-03-07 21:34:38','2014-03-07 21:34:38','',340,'http://localhost/~psalessi/Genetics_Lab/340-revision-v1/',0,'revision','',0),(345,1,'2014-03-07 21:35:27','2014-03-07 21:35:27','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Making dNTPs (Promega)</span></h3>\r\n<strong>NOTES:</strong>\r\n\r\nStock is kept on top shelf of -80 freezer in the main lab\r\n\r\nStock dNTP concentrations are 100mM each (dATP, dCTP, dGTP, dTTP)\r\n\r\n<strong>DIRECTIONS:</strong>\r\n<ul>\r\n	<li>Pipette 250 ul of each dNTP into a 30ml conical tube</li>\r\n	<li>Add 9000 ul of PCR water(Makes 10 ml total of 10uM dNTP, or 2.5uM each dNTP)</li>\r\n	<li>Mix</li>\r\n	<li>Aliquot 500 ul of mixture into twenty 1.5ml tubes</li>\r\n</ul>','dNTPs (Promega)','','inherit','closed','open','','321-revision-v1','','','2014-03-07 21:35:27','2014-03-07 21:35:27','',321,'http://localhost/~psalessi/Genetics_Lab/321-revision-v1/',0,'revision','',0),(346,1,'2014-03-07 21:36:16','2014-03-07 21:36:16','Add 10 mL of 1M Tris-HCl (pH 8.0)* and 2 mL of 0.5M EDTA (pH 8.0)* to 988 ml of ddH<sub>2</sub>0.\r\n\r\nFilter with 0.5 micron filter and autoclave.\r\n\r\n<strong>*1 M Tris-Cl</strong>\r\n\r\nDissolve 121.1 g of tris base (sigma # T-1503) in 700 ml of ddH<sub>2</sub>O. Bring to desired pH (Usually 7.5 or 8.0) with concentrated HCl. Bring volume to 1L with ddH<sub>2</sub>O. Filter with 0.5 micron filter. Autoclave. Store at room temperature.\r\n\r\n<strong>*0.5 M EDTA</strong>\r\n\r\nAdd 186.1 g EDTA (disodium, dihydrate, Sigma # E-5134) to 800 ml of ddH20. Add about 20g of NaOH pellets while stirring to bring the pH to 8.0. Add the last few grams slowly to avoid overshooting the pH. Note that the EDTA won\'t completely dissolve until the pH is around 8. Filter with 0.5 micron filter and autoclave.','1X TE (10mM Tris; 1mM EDTA)','','inherit','closed','open','','333-revision-v1','','','2014-03-07 21:36:16','2014-03-07 21:36:16','',333,'http://localhost/~psalessi/Genetics_Lab/333-revision-v1/',0,'revision','',0),(347,1,'2014-03-07 21:36:41','2014-03-07 21:36:41','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">1X Tris- \"low\" EDTA (TLE) pH 8.0</span></h3>\r\nAdd 10 mL of 1M Tris-HCl (pH 8.0)* and 0.2 mL of 0.5M EDTA (pH 8.0)* to 989.8 (990) ml of ddH<sub>2</sub>0.\r\n\r\nFilter with 0.5 micron filter and autoclave.\r\n\r\n<strong>*1 M Tris-Cl</strong>\r\n\r\nDissolve 121.1 g of tris base (sigma # T-1503) in 700 ml of ddH<sub>2</sub>O. Bring to desired pH (Usually 7.5 or 8.0) with concentrated HCl. Bring volume to 1L with ddH<sub>2</sub>O. Filter with 0.5 micron filter. Autoclave. Store at room temperature.\r\n\r\n<strong>*0.5 M EDTA</strong>\r\n\r\nAdd 186.1 g EDTA (disodium, dihydrate, Sigma # E-5134) to 800 ml of ddH<sub>2</sub>0. Add about 20g of NaOH pellets while stirring to bring the pH to 8.0. Add the last few grams slowly to avoid overshooting the pH. Note that the EDTA won\'t completely dissolve until the pH is around 8. Filter with 0.5 micron filter and autoclave.','1X TLE (10mM Tris; 0.1mM EDTA)','','inherit','closed','open','','335-revision-v1','','','2014-03-07 21:36:41','2014-03-07 21:36:41','',335,'http://localhost/~psalessi/Genetics_Lab/335-revision-v1/',0,'revision','',0),(348,1,'2014-03-07 21:37:10','2014-03-07 21:37:10','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">10X TNE Buffer (100mM Tris; 2.0M NaCl; 10mM EDTA; pH 7.4)</span></h3>\r\nTo 800ml ddH<sub>2</sub>O add:\r\n<ul>\r\n	<li>Trizma Base: 12.1g T-1503</li>\r\n	<li>disodium EDTA: 3.7g E-5134</li>\r\n	<li>NaCl: 116.8g</li>\r\n	<li>pH 7.4 with Conc. HCl</li>\r\n	<li>Bring to volume of 1L with ddH<sub>2</sub>O.</li>\r\n	<li>Filter and autoclave</li>\r\n</ul>\r\nStore at 4 C','10X TNE','','inherit','closed','open','','337-revision-v1','','','2014-03-07 21:37:10','2014-03-07 21:37:10','',337,'http://localhost/~psalessi/Genetics_Lab/337-revision-v1/',0,'revision','',0),(349,1,'2014-03-07 21:37:30','2014-03-07 21:37:30','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">0.05ug/ul for agarose gels</span></h3>\r\n<strong>Amount:</strong>\r\n\r\nLadder stock (1ug/ul): 10ul\r\n\r\nLoading dye (BPB/XC found in fridge): 40ul\r\n\r\nNanopure water: 150ul\r\n<ul>\r\n	<li>This makes 200ul - aliquot into two tubes</li>\r\n	<li>Use 1ul in a typical agarose gel</li>\r\n</ul>','Diluting 1Kb Plus Ladder with Dye','','inherit','closed','open','','309-revision-v1','','','2014-03-07 21:37:30','2014-03-07 21:37:30','',309,'http://localhost/~psalessi/Genetics_Lab/309-revision-v1/',0,'revision','',0),(350,1,'2014-03-07 21:38:02','2014-03-07 21:38:02','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">10X TBE(1.1M Tris; 900mM Borate; 25mM EDTA; pH 8.3)</span></h3>\r\n<p style=\"text-align: center;\"><span style=\"color: #ff0000;\">****Note: Boric acid dissolves at a pH around 7. One may alternate between adding Trizma Base &amp; Boric acid. Dissolve the EDTA first.****</span></p>\r\nTo 2.5L H<sub>2</sub>O add:\r\n\r\ndisodium EDTA: 37.5g E-5134\r\nTrizma Base: 432g T-1503\r\nBoric Acid: 220g B-0252\r\n<ul>\r\n	<li>Dissolve and bring pH to 8.3</li>\r\n	<li>Dilute to 4L</li>\r\n	<li><span style=\"line-height: 1.714285714; font-size: 1rem;\">Filter into 5 \"One-Shot\" bottles of 800ml each and autoclave.</span></li>\r\n</ul>\r\n<strong>To make 1X TBE:</strong>\r\n<ul>\r\n	<li>Pour entire one-shot into 8L carboy</li>\r\n	<li>Fill to 8L mark with dH<sub>2</sub>O</li>\r\n</ul>','10X TBE One-Shots','','inherit','closed','open','','340-revision-v1','','','2014-03-07 21:38:02','2014-03-07 21:38:02','',340,'http://localhost/~psalessi/Genetics_Lab/340-revision-v1/',0,'revision','',0),(351,1,'2014-03-07 21:45:16','2014-03-07 21:45:16','<strong>Using FujiFilm for quantifying DNA plate</strong>\r\n<ol>\r\n	<li>Turn FujiFilm, then the computer on.</li>\r\n	<li>Login with regular AniSci password</li>\r\n	<li>Open Image Reader</li>\r\n	<li>Scan plate using 1Laser 1Image, 473nm</li>\r\n	<li>Open MultiGauge 3.0</li>\r\n	<li>Open the image file</li>\r\n	<li>If needed, go to Display and change the color intensity.</li>\r\n	<li>Go to Measure at the top of the window\r\n<ol>\r\n	<li>Click on Plate in the left tool bar\r\n<ol>\r\n	<li>Click on Setting</li>\r\n	<li>Change the diameter(d) to 5-6mm (the template circles should be within the wells using this measurement)</li>\r\n	<li>Click on the horizontal or vertical plate button and drag the template to your plate</li>\r\n	<li>Template is red when selected</li>\r\n</ol>\r\n</li>\r\n	<li>Set the well with the lowest standard as background by drawing a circle inside the well using the button under BG.\r\n<ol>\r\n	<li>The circle is purple when set as the background</li>\r\n</ol>\r\n</li>\r\n</ol>\r\n</li>\r\n	<li>Next to Calibration\r\n<ol>\r\n	<li>Change unit to ng</li>\r\n	<li>Click on the cursor button under Select STD band</li>\r\n	<li>Click on your standards. A box Band Attribute will show up.\r\n<ol>\r\n	<li>Choose Standard and input your standard concentration</li>\r\n	<li>Standard wells turn green</li>\r\n</ol>\r\n</li>\r\n	<li>Click on Calibration under Density Calib\r\n<ol>\r\n	<li>Select the curve function (ex: linear function)</li>\r\n	<li>Choose LAU-BG under the drop-down menu for Measured</li>\r\n</ol>\r\n</li>\r\n</ol>\r\n</li>\r\n	<li>Next to Analysis\r\n<ol>\r\n	<li>Result table should appear</li>\r\n	<li>Change the displayed information using the Table Option\r\n<ol>\r\n	<li>Refer to C-146 in the FujiFilm manual for the full definition.</li>\r\n</ol>\r\n</li>\r\n	<li>Change the view between All, Comparison, and Table at the bottom of the window.</li>\r\n	<li>Export as .txt file</li>\r\n</ol>\r\n</li>\r\n</ol>\r\n**Since the standard only goes up to 100ng/L, high concentration (ex: 1000ng) may be not as accurate.','FujiFilm DNA Quantification','','publish','closed','open','','fujifilm-dna-quantification','','','2014-03-07 21:55:11','2014-03-07 21:55:11','',217,'http://localhost/~psalessi/Genetics_Lab/?page_id=351',0,'page','',0),(352,1,'2014-03-07 21:45:16','2014-03-07 21:45:16','Using FujiFilm for quantifying DNA plate\r\n<ol>\r\n<ol>\r\n	<li>Turn FujiFilm, then the computer on.</li>\r\n	<li>Login with regular AniSci password</li>\r\n	<li>Open Image Reader</li>\r\n	<li>Scan plate using 1Laser 1Image, 473nm</li>\r\n	<li>Open MultiGauge 3.0</li>\r\n	<li>Open the image file</li>\r\n	<li>If needed, go to Display and change the color intensity.</li>\r\n	<li>Go to Measure at the top of the window\r\n<ol type=\"a\">\r\n	<li>Click on Plate in the left tool bar\r\n<ol type=\"i\">\r\n	<li>Click on Setting</li>\r\n	<li>Change the diameter(d) to 5-6mm (the template circles should be within the wells using this measurement)</li>\r\n	<li>Click on the horizontal or vertical plate button and drag the template to your plate</li>\r\n	<li>Template is red when selected</li>\r\n</ol>\r\n</li>\r\n	<li>Set the well with the lowest standard as background by drawing a circle inside the well using the button under BG.\r\n<ol type=\"i\">\r\n	<li>The circle is purple when set as the background</li>\r\n</ol>\r\n</li>\r\n</ol>\r\n</li>\r\n</ol>\r\n</ol>\r\n9. Next to Calibration\r\n\r\na. Change unit to ng\r\n\r\nb. Click on the cursor button under Select STD band\r\n\r\nc. Click on your standards. A box Band Attribute will show up.\r\n\r\ni. Choose Standard and input your standard concentration\r\n\r\nii. Standard wells turn green\r\n\r\nd. Click on Calibration under Density Calib\r\n\r\ni. Select the curve function (ex: linear function)\r\n\r\nii. Choose LAU-BG under the drop-down menu for Measured\r\n\r\n10. Next to Analysis\r\n\r\na. Result table should appear\r\n\r\nb. Change the displayed information using the Table Option\r\n\r\ni. Refer to C-146 in the FujiFilm manual for the full definition.\r\n\r\nc. Change the view between All, Comparison, and Table at the bottom of the window.\r\n\r\nd. Export as .txt file\r\n\r\n**Since the standard only goes up to 100ng/L, high concentration (ex: 1000ng) may be not as accurate.','FujiFilm DNA Quantification','','inherit','closed','open','','351-revision-v1','','','2014-03-07 21:45:16','2014-03-07 21:45:16','',351,'http://localhost/~psalessi/Genetics_Lab/351-revision-v1/',0,'revision','',0),(353,1,'2014-03-07 21:54:24','2014-03-07 21:54:24','Using FujiFilm for quantifying DNA plate\n<ol>\n	<li>Turn FujiFilm, then the computer on.</li>\n	<li>Login with regular AniSci password</li>\n	<li>Open Image Reader</li>\n	<li>Scan plate using 1Laser 1Image, 473nm</li>\n	<li>Open MultiGauge 3.0</li>\n	<li>Open the image file</li>\n	<li>If needed, go to Display and change the color intensity.</li>\n	<li>Go to Measure at the top of the window\n<ol>\n	<li>Click on Plate in the left tool bar\n<ol>\n	<li>Click on Setting</li>\n	<li>Change the diameter(d) to 5-6mm (the template circles should be within the wells using this measurement)</li>\n	<li>Click on the horizontal or vertical plate button and drag the template to your plate</li>\n	<li>Template is red when selected</li>\n</ol>\n</li>\n	<li>Set the well with the lowest standard as background by drawing a circle inside the well using the button under BG.\n<ol>\n	<li>The circle is purple when set as the background</li>\n</ol>\n\n</li>\n</ol>\n</li>\n<li>Next to Calibration\n<ol>\n<li>Change unit to ng</li>\n<li>Click on the cursor button under Select STD band</li>\n<li>Click on your standards. A box Band Attribute will show up.\n<ol>\n<li>Choose Standard and input your standard concentration</li>\n<li>Standard wells turn green</li>\n</ol>\n</li>\n<li>Click on Calibration under Density Calib\n<ol>\n<li>Select the curve function (ex: linear function)</li>\n<li>Choose LAU-BG under the drop-down menu for Measured</li>\n</ol>\n</li>\n</ol>\n</li>\n<li>Next to Analysis\n<ol>\n<li>Result table should appear</li>\n<li>Change the displayed information using the Table Option\n<ol>\n<li>Refer to C-146 in the FujiFilm manual for the full definition.</li>\n</ol>\n</li>\n<li>Change the view between All, Comparison, and Table at the bottom of the window.</li>\n\nd. Export as .txt file\n</ol>\n\n\n**Since the standard only goes up to 100ng/L, high concentration (ex: 1000ng) may be not as accurate.','FujiFilm DNA Quantification','','inherit','closed','open','','351-autosave-v1','','','2014-03-07 21:54:24','2014-03-07 21:54:24','',351,'http://localhost/~psalessi/Genetics_Lab/351-autosave-v1/',0,'revision','',0),(354,1,'2014-03-07 21:54:58','2014-03-07 21:54:58','<strong>Using FujiFilm for quantifying DNA plate</strong>\r\n<ol>\r\n	<li>Turn FujiFilm, then the computer on.</li>\r\n	<li>Login with regular AniSci password</li>\r\n	<li>Open Image Reader</li>\r\n	<li>Scan plate using 1Laser 1Image, 473nm</li>\r\n	<li>Open MultiGauge 3.0</li>\r\n	<li>Open the image file</li>\r\n	<li>If needed, go to Display and change the color intensity.</li>\r\n	<li>Go to Measure at the top of the window\r\n<ol>\r\n	<li>Click on Plate in the left tool bar\r\n<ol>\r\n	<li>Click on Setting</li>\r\n	<li>Change the diameter(d) to 5-6mm (the template circles should be within the wells using this measurement)</li>\r\n	<li>Click on the horizontal or vertical plate button and drag the template to your plate</li>\r\n	<li>Template is red when selected</li>\r\n</ol>\r\n</li>\r\n	<li>Set the well with the lowest standard as background by drawing a circle inside the well using the button under BG.\r\n<ol>\r\n	<li>The circle is purple when set as the background</li>\r\n</ol>\r\n</li>\r\n</ol>\r\n</li>\r\n	<li>Next to Calibration\r\n<ol>\r\n	<li>Change unit to ng</li>\r\n	<li>Click on the cursor button under Select STD band</li>\r\n	<li>Click on your standards. A box Band Attribute will show up.\r\n<ol>\r\n	<li>Choose Standard and input your standard concentration</li>\r\n	<li>Standard wells turn green</li>\r\n</ol>\r\n</li>\r\n	<li>Click on Calibration under Density Calib\r\n<ol>\r\n	<li>Select the curve function (ex: linear function)</li>\r\n	<li>Choose LAU-BG under the drop-down menu for Measured</li>\r\n</ol>\r\n</li>\r\n</ol>\r\n</li>\r\n	<li>Next to Analysis\r\n<ol>\r\n	<li>Result table should appear</li>\r\n	<li>Change the displayed information using the Table Option\r\n<ol>\r\n	<li>Refer to C-146 in the FujiFilm manual for the full definition.</li>\r\n</ol>\r\n</li>\r\n	<li>Change the view between All, Comparison, and Table at the bottom of the window.</li>\r\n	<li>Export as .txt file</li>\r\n</ol>\r\n</li>\r\n</ol>\r\n**Since the standard only goes up to 100ng/L, high concentration (ex: 1000ng) may be not as accurate.','FujiFilm DNA Quantification','','inherit','closed','open','','351-revision-v1','','','2014-03-07 21:54:58','2014-03-07 21:54:58','',351,'http://localhost/~psalessi/Genetics_Lab/351-revision-v1/',0,'revision','',0),(355,1,'2014-03-07 21:55:55','2014-03-07 21:55:55','<strong>Buffers &amp; Stocks</strong>\r\n<ul>\r\n	<li><a title=\"98% Formamide Loading Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/98-formamide-loading-dye/\">Formamide Loading Dye</a></li>\r\n	<li><a title=\"Diluting 1Kb Plus Ladder with Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/diluting-1kb-plus-ladder-with-dye/\">Diluting 1kb Ladder/Dye</a></li>\r\n	<li><a title=\"Gelco Fluorescein Ladder\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/gelco-fluorescein-ladder/\">Gelco Fluorescein Ladder</a></li>\r\n	<li><a title=\"Proteinase K\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/proteinase-k/\">Proteinase K</a></li>\r\n	<li><a title=\"Glassware Cleaner\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/glassware-cleaner/\">Glassware Cleaner</a></li>\r\n	<li><a title=\"dNTPs (Promega)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/dntps-promega/\">dNTPs (Promega)</a></li>\r\n	<li><a title=\"1M Tris-Cl\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1m-tris-cl/\">1M Tris-Cl</a></li>\r\n	<li><a title=\".5M EDTA (pH 8.0)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/5m-edta-ph-8-0/\">0.5M EDTA (pH 8.0)</a></li>\r\n	<li><a title=\"1X TE (10mM Tris; 1mM EDTA)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1x-te-10mm-tris-1mm-edta/\">1X TE (10mM Tris; 1mM EDTA)</a></li>\r\n	<li><a title=\"1X TLE (10mM Tris; 0.1mM EDTA)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1x-tle-10mm-tris-0-1mm-edta/\">1X TLE (10mM Tris; 0.1 mM EDTA)</a></li>\r\n	<li><a title=\"10X TNE\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/10x-tne/\">10X TNE</a></li>\r\n	<li><a title=\"10X TBE One-Shots\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/10x-tbe-one-shots/\">10X TBE One-Shots</a></li>\r\n</ul>\r\n<strong>FujiFilm</strong>\r\n<ul>\r\n	<li><a title=\"FujiFilm DNA Quantification\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/fujifilm-dna-quantification/\">DNA quantification</a></li>\r\n</ul>\r\nABI 3730xl Maintenance\r\n<ul>\r\n	<li><a title=\"3730Maintenance.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/3730Maintenance.html\">3730xl Maintenance</a></li>\r\n	<li><a title=\"ProtKwash.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/ProtKwash.html\">Proteinase K Wash</a></li>\r\n</ul>\r\n<strong>Acrylamide gels</strong>\r\n<ul>\r\n	<li><a title=\"Acrylamide.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Acrylamide.html\">Acrylamide Gel</a></li>\r\n	<li><a title=\"AcGelpour.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/AcGelpour.html\">Pouring</a></li>\r\n	<li><a title=\"AcGelRun.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/AcGelRun.html\">Running</a></li>\r\n</ul>\r\n<strong>Fluidigm</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/FLUIDIGM EP1 PROTOCOL.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/FLUIDIGM%20EP1%20PROTOCOL.doc\">Fluidigm EP1 Protocol</a></li>\r\n</ul>\r\n<strong>Illumina</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/RAD-SEQ Protocol.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/RAD-SEQ%20Protocol.doc\">RAD-SEQ Protocol</a></li>\r\n</ul>\r\n<strong>Other</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/tissuesampleprot.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/tissuesampleprot.pdf\">Tissue Sampling</a></li>\r\n	<li><a title=\"Supplies.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Supplies.html\">Ordering and Receiving Supplies</a></li>\r\n	<li><a title=\"Supplies.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Supplies.html\">PCR Tips and Tricks</a></li>\r\n</ul>','Protocols and Procedures','','inherit','closed','open','','217-revision-v1','','','2014-03-07 21:55:55','2014-03-07 21:55:55','',217,'http://localhost/~psalessi/Genetics_Lab/217-revision-v1/',0,'revision','',0),(356,1,'2014-03-07 21:56:50','2014-03-07 21:56:50','<strong>Buffers &amp; Stocks</strong>\r\n<ul>\r\n	<li><a title=\"98% Formamide Loading Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/98-formamide-loading-dye/\">Formamide Loading Dye</a></li>\r\n	<li><a title=\"Diluting 1Kb Plus Ladder with Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/diluting-1kb-plus-ladder-with-dye/\">Diluting 1kb Ladder/Dye</a></li>\r\n	<li><a title=\"Gelco Fluorescein Ladder\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/gelco-fluorescein-ladder/\">Gelco Fluorescein Ladder</a></li>\r\n	<li><a title=\"Proteinase K\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/proteinase-k/\">Proteinase K</a></li>\r\n	<li><a title=\"Glassware Cleaner\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/glassware-cleaner/\">Glassware Cleaner</a></li>\r\n	<li><a title=\"dNTPs (Promega)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/dntps-promega/\">dNTPs (Promega)</a></li>\r\n	<li><a title=\"1M Tris-Cl\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1m-tris-cl/\">1M Tris-Cl</a></li>\r\n	<li><a title=\".5M EDTA (pH 8.0)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/5m-edta-ph-8-0/\">0.5M EDTA (pH 8.0)</a></li>\r\n	<li><a title=\"1X TE (10mM Tris; 1mM EDTA)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1x-te-10mm-tris-1mm-edta/\">1X TE (10mM Tris; 1mM EDTA)</a></li>\r\n	<li><a title=\"1X TLE (10mM Tris; 0.1mM EDTA)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1x-tle-10mm-tris-0-1mm-edta/\">1X TLE (10mM Tris; 0.1 mM EDTA)</a></li>\r\n	<li><a title=\"10X TNE\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/10x-tne/\">10X TNE</a></li>\r\n	<li><a title=\"10X TBE One-Shots\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/10x-tbe-one-shots/\">10X TBE One-Shots</a></li>\r\n</ul>\r\n<strong>FujiFilm</strong>\r\n<ul>\r\n	<li><a title=\"FujiFilm DNA Quantification\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/fujifilm-dna-quantification/\">DNA quantification</a></li>\r\n</ul>\r\n<strong>ABI 3730xl Maintenance</strong>\r\n<ul>\r\n	<li><a title=\"3730Maintenance.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/3730Maintenance.html\">3730xl Maintenance</a></li>\r\n	<li><a title=\"ProtKwash.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/ProtKwash.html\">Proteinase K Wash</a></li>\r\n</ul>\r\n<strong>Acrylamide gels</strong>\r\n<ul>\r\n	<li><a title=\"Acrylamide.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Acrylamide.html\">Acrylamide Gel</a></li>\r\n	<li><a title=\"AcGelpour.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/AcGelpour.html\">Pouring</a></li>\r\n	<li><a title=\"AcGelRun.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/AcGelRun.html\">Running</a></li>\r\n</ul>\r\n<strong>Fluidigm</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/FLUIDIGM EP1 PROTOCOL.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/FLUIDIGM%20EP1%20PROTOCOL.doc\">Fluidigm EP1 Protocol</a></li>\r\n</ul>\r\n<strong>Illumina</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/RAD-SEQ Protocol.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/RAD-SEQ%20Protocol.doc\">RAD-SEQ Protocol</a></li>\r\n</ul>\r\n<strong>Other</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/tissuesampleprot.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/tissuesampleprot.pdf\">Tissue Sampling</a></li>\r\n	<li><a title=\"Supplies.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Supplies.html\">Ordering and Receiving Supplies</a></li>\r\n	<li><a title=\"Supplies.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Supplies.html\">PCR Tips and Tricks</a></li>\r\n</ul>','Protocols and Procedures','','inherit','closed','open','','217-revision-v1','','','2014-03-07 21:56:50','2014-03-07 21:56:50','',217,'http://localhost/~psalessi/Genetics_Lab/217-revision-v1/',0,'revision','',0),(357,1,'2014-03-07 21:59:10','2014-03-07 21:59:10','<ul>\r\n	<li>Check reservoir, buffer jar &amp; polymer level before each run\r\n<ul>\r\n	<li>Dilute buffer to 1x before use</li>\r\n	<li>Buffer should reach the red line of the jar</li>\r\n</ul>\r\n</li>\r\n	<li>Clean surfaces, check for bubbles, leaks, and capillaries daily</li>\r\n	<li>Change waste water, rinse water, and buffer chamber weekly\r\n<ul>\r\n	<li>Use DI H<sub>2</sub>O for wash and refill</li>\r\n	<li>Manual Control -&gt; Autosampler -&gt; Unload tray</li>\r\n</ul>\r\n</li>\r\n	<li>Flush water trap, replenish polymer weekly</li>\r\n	<li>Reboot system weekly</li>\r\n	<li>Run Water Wash Wizard monthly</li>\r\n</ul>','3730xl General Maintenance','','publish','closed','open','','3730xl-general-maintenance','','','2014-03-07 21:59:23','2014-03-07 21:59:23','',217,'http://localhost/~psalessi/Genetics_Lab/?page_id=357',0,'page','',0),(358,1,'2014-03-07 21:59:10','2014-03-07 21:59:10','<ul>\r\n	<li>Check reservoir, buffer jar &amp; polymer level before each run\r\n<ul>\r\n	<li>Dilute buffer to 1x before use</li>\r\n	<li>Buffer should reach the red line of the jar</li>\r\n</ul>\r\n</li>\r\n	<li>Clean surfaces, check for bubbles, leaks, and capillaries daily</li>\r\n	<li>Change waste water, rinse water, and buffer chamber weekly\r\n<ul>\r\n	<li>Use DI H<sub>2</sub>O for wash and refill</li>\r\n	<li>Manual Control -&gt; Autosampler -&gt; Unload tray</li>\r\n</ul>\r\n</li>\r\n	<li>Flush water trap, replenish polymer weekly</li>\r\n	<li>Reboot system weekly</li>\r\n	<li>Run Water Wash Wizard monthly</li>\r\n</ul>','3730xl General Maintenance','','inherit','closed','open','','357-revision-v1','','','2014-03-07 21:59:10','2014-03-07 21:59:10','',357,'http://localhost/~psalessi/Genetics_Lab/357-revision-v1/',0,'revision','',0),(359,1,'2014-03-07 22:01:20','2014-03-07 22:01:20','<strong>Buffers &amp; Stocks</strong>\r\n<ul>\r\n	<li><a title=\"98% Formamide Loading Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/98-formamide-loading-dye/\">Formamide Loading Dye</a></li>\r\n	<li><a title=\"Diluting 1Kb Plus Ladder with Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/diluting-1kb-plus-ladder-with-dye/\">Diluting 1kb Ladder/Dye</a></li>\r\n	<li><a title=\"Gelco Fluorescein Ladder\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/gelco-fluorescein-ladder/\">Gelco Fluorescein Ladder</a></li>\r\n	<li><a title=\"Proteinase K\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/proteinase-k/\">Proteinase K</a></li>\r\n	<li><a title=\"Glassware Cleaner\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/glassware-cleaner/\">Glassware Cleaner</a></li>\r\n	<li><a title=\"dNTPs (Promega)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/dntps-promega/\">dNTPs (Promega)</a></li>\r\n	<li><a title=\"1M Tris-Cl\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1m-tris-cl/\">1M Tris-Cl</a></li>\r\n	<li><a title=\".5M EDTA (pH 8.0)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/5m-edta-ph-8-0/\">0.5M EDTA (pH 8.0)</a></li>\r\n	<li><a title=\"1X TE (10mM Tris; 1mM EDTA)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1x-te-10mm-tris-1mm-edta/\">1X TE (10mM Tris; 1mM EDTA)</a></li>\r\n	<li><a title=\"1X TLE (10mM Tris; 0.1mM EDTA)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1x-tle-10mm-tris-0-1mm-edta/\">1X TLE (10mM Tris; 0.1 mM EDTA)</a></li>\r\n	<li><a title=\"10X TNE\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/10x-tne/\">10X TNE</a></li>\r\n	<li><a title=\"10X TBE One-Shots\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/10x-tbe-one-shots/\">10X TBE One-Shots</a></li>\r\n</ul>\r\n<strong>FujiFilm</strong>\r\n<ul>\r\n	<li><a title=\"FujiFilm DNA Quantification\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/fujifilm-dna-quantification/\">DNA quantification</a></li>\r\n</ul>\r\n<strong>ABI 3730xl Maintenance</strong>\r\n<ul>\r\n	<li><a title=\"3730xl General Maintenance\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/3730xl-general-maintenance/\">3730xl Maintenance</a></li>\r\n	<li><a title=\"ProtKwash.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/ProtKwash.html\">Proteinase K Wash</a></li>\r\n</ul>\r\n<strong>Acrylamide gels</strong>\r\n<ul>\r\n	<li><a title=\"Acrylamide.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Acrylamide.html\">Acrylamide Gel</a></li>\r\n	<li><a title=\"AcGelpour.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/AcGelpour.html\">Pouring</a></li>\r\n	<li><a title=\"AcGelRun.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/AcGelRun.html\">Running</a></li>\r\n</ul>\r\n<strong>Fluidigm</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/FLUIDIGM EP1 PROTOCOL.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/FLUIDIGM%20EP1%20PROTOCOL.doc\">Fluidigm EP1 Protocol</a></li>\r\n</ul>\r\n<strong>Illumina</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/RAD-SEQ Protocol.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/RAD-SEQ%20Protocol.doc\">RAD-SEQ Protocol</a></li>\r\n</ul>\r\n<strong>Other</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/tissuesampleprot.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/tissuesampleprot.pdf\">Tissue Sampling</a></li>\r\n	<li><a title=\"Supplies.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Supplies.html\">Ordering and Receiving Supplies</a></li>\r\n	<li><a title=\"Supplies.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Supplies.html\">PCR Tips and Tricks</a></li>\r\n</ul>','Protocols and Procedures','','inherit','closed','open','','217-revision-v1','','','2014-03-07 22:01:20','2014-03-07 22:01:20','',217,'http://localhost/~psalessi/Genetics_Lab/217-revision-v1/',0,'revision','',0),(361,1,'2014-03-10 17:29:51','2014-03-10 17:29:51','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Proteinase K Capillary Wash and Sapphire Block Wash Protocol</span></h3>\r\n<p style=\"text-align: center;\"><span style=\"color: #ff0000;\">WARNING: Polymer and Junk Polymer can act as a neurotoxin. Be sure to wear gloves when handling these materials.</span></p>\r\n<strong><span style=\"text-decoration: underline;\">Preparatory Steps</span></strong>\r\n<ol>\r\n	<li>ProK Solution:\r\n<ol>\r\n	<li>Check the fridge for previously made Proteinase K solution. It should be on the door in a small bottle marked Pro K, if the Pro K has only been used once before, go ahead and use it. If this is the second use, after youre done, pour it down the drain.</li>\r\n	<li>New Batch: 50uL of ProK + 25mL of Water</li>\r\n	<li>ProK solution can be found in the back freezer in the white box labeled Proteinase K. Only use the purified water in the bottles on the right side of the sink.</li>\r\n</ol>\r\n</li>\r\n	<li>Remove Junk polymer from refrigerator. Set aside to warm to room temperature.</li>\r\n	<li>Remove 28mL bottle of polymer from the refrigerator and a clean empty 7mL bottle from the back sink. Transfer polymer from the 28mL bottle into the 7mL bottle until the meniscus reaches the black line.</li>\r\n	<li>4.3730 buffer dilution: Using the red serological pipette, found in the Sephadex and Purification drawer, with a 25mL tip, add 15mL of 10x stock 3730 buffer, found on the door of the fridge, to a 250mL graduated cylinder. Add 135mL of HPLC water, or enough to reach the 150mL graduation.</li>\r\n	<li>Remove 3 clean trays and tops from the Sephadex and Purification drawer. One tray will be used for each of the following: buffer, waste, water.</li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Proteinase K Capillary Wash</span>\r\n<ol>\r\n	<li>Minimize collection window and enter My Computer on the desktop\r\n<ol>\r\n	<li>Enter Local Disk (C:)</li>\r\n	<li>Open Applied Biosystems folder</li>\r\n	<li>Open Firmware folder</li>\r\n	<li>Open 3730calib</li>\r\n	<li>Change Syringe Injection Speed from 200 to 100</li>\r\n	<li>Change Syringe Plunger Speed from 400 to 200</li>\r\n	<li>Save</li>\r\n</ol>\r\n</li>\r\n	<li>In the Service Console, press Stop All and wait for all symbols to turn from green squares to red circles. Restart the computer.</li>\r\n	<li>Windows will prompt your for username and password. Sign on as Administrator and enter password, and turn 3730 back on. Wait for the blinking yellow light to turn solid green.</li>\r\n	<li>Open Data Collection Software (Start, Programs, Applied Biosystems, Collection software) or use shortcut on desktop</li>\r\n	<li>Go to Manual Control in collection software.\r\n<ol>\r\n	<li>Send Defined Command for: Oven</li>\r\n	<li>Command Name: Set Oven State</li>\r\n	<li>Value: On, Send</li>\r\n	<li>Command Name: Set Oven Temperature</li>\r\n	<li>Value: 60.0, Send</li>\r\n</ol>\r\n</li>\r\n	<li>Pour Proteinase K solution that you have prepared into a clean, empty polymer jar.\r\n<ol>\r\n	<li>Remove polymer jar and replace with Proteinase K solution.</li>\r\n	<li>Transfer excess polymer into Junk Polymer bottle.</li>\r\n</ol>\r\n</li>\r\n	<li>Go to Spatial Run Scheduler in the collection software window.\r\n<ol>\r\n	<li>Find Spatial Protocols in lower right portion of window.</li>\r\n	<li>Protocol: SpatialFill_1</li>\r\n	<li>Click Start\r\n<ol>\r\n	<li>The Buffer tray will return to original position, the waste tray will rinse the capillaries, water will wash the capillaries and finally, the buffer tray returns to Active Position</li>\r\n</ol>\r\n</li>\r\n	<li>Click Stop when Collecting Data appears on the progress bar (If you are away from the machine for an extended period of time, DO NOT Accept the new spatialclick Reject and then continue with the next wash)</li>\r\n	<li>Repeat 14 more times for a total of 15 Washes (or more, if desired)</li>\r\n</ol>\r\n</li>\r\n	<li>Repeat step 7, except replace proteinase K solution with a polymer jar filled with warm HPLC grade water. This should be done for a total of 10 times to wash out any residual proteinase K solution in the capillaries. Return PK solution to the top, door shelf of the fridge, if you had to prepare the dilution. If not, pour the solution down the drain.</li>\r\n	<li><span style=\"color: #ff0000;\">Go to Manual Control and return the oven state to OFF by following steps 5 b thru c.</span></li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Sapphire Block Wash</span>\r\n<ol>\r\n	<li><span style=\"color: #ff0000;\">Remove anode jar from lower block</span>, empty the contents into the waste container and replace the jar back onto the lower block.</li>\r\n	<li>(If necessary, add additional HPLC water to polymer bottle)</li>\r\n	<li>Go to Instrument Status in the collection software</li>\r\n	<li>From the Wizards menu, select Water Wash Wizard\r\n<ol>\r\n	<li>Select Next</li>\r\n	<li>Select Water Washwait until wash is 100% complete.</li>\r\n	<li>Click Cancel. One wash is now complete.</li>\r\n	<li>Repeat these steps two more times for a total of three washes</li>\r\n</ol>\r\n</li>\r\n	<li>From the Wizards menu, select Change Polymer Wizard\r\n<ol>\r\n	<li>Make sure the polymer jar has enough volume of HPLC grade water and continue.</li>\r\n	<li>Select Same Lot, Next, Next</li>\r\n	<li>Select Prime, then Next</li>\r\n	<li>Select No Bubbles, then Next\r\n<ol>\r\n	<li>If you do see bubbles, or you are unsure if there are any, call over Sheryl, Kerry, or Shelley. You will use the Bubble Removal Wizard once you reach step 15. The bubble removal wizard requires a viscous substance, like polymer, so wait until step 15.</li>\r\n</ol>\r\n</li>\r\n	<li>Select No Bubbles, then Next</li>\r\n	<li>Select Fill Array, then Finish</li>\r\n	<li>Repeat these steps once more for a total of two washes</li>\r\n</ol>\r\n</li>\r\n	<li>Repeat step 14 with Junk Polymer bottle in place of HPLC filled jar.</li>\r\n	<li>Remove Junk Polymer jar and replace with Polymer and perform Change Polymer Wizard as in step 14.</li>\r\n	<li><span style=\"color: #ff0000;\">Return the Syringe Injection Speed from 100 to 200 and Syringe Plunger Speed from 200 to 400 repeating steps 1-4.</span></li>\r\n	<li>Press Tray so you can remove the old buffer tray and replace it with a clean, new tray. Pour all liquids from trays into the waste container. <span style=\"color: #ff0000;\">Add the appropriate solutions so the meniscus is at the black line: buffer in the buffer tray, and water in BOTH the water and waste trays.</span> Place lids and put on waste septa found in the Sephadex and Purification drawer.\r\n<ol>\r\n	<li>Dump the Waste tray into the waste container. The buffer and water trays can be dumped down the drain.</li>\r\n	<li>In a plastic tub soak all dirty trays in DI water.</li>\r\n</ol>\r\n</li>\r\n	<li><span style=\"color: #ff0000;\">Refill large anode jar with buffer, filling just below the red line. Make sure the anode is at least submersed in buffer. It is okay to be below the red line, just not above the line. Place the large anode jar onto the lower block. Make sure all buffer is wiped off of the top of the anode jar.</span></li>\r\n	<li>Press Tray again to return the buffer tray to the capillaries. Turn light off.</li>\r\n	<li>Restart the computer.</li>\r\n	<li>The machine is now ready.</li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Clean Up!</span>\r\n<ol>\r\n	<li>Wash all empty bottles, trays, graduated cylinders, and waste septa by soaking them in DI water in large plastic container.\r\n<ol>\r\n	<li>Remember to wear gloves as the residual polymer left in some of these containers can act as a neurotoxin</li>\r\n</ol>\r\n</li>\r\n	<li>Be sure to tightly close the waste container bottle after you are finished!</li>\r\n</ol>\r\nNote:This protocol is for a 48-cap array, so needs to do 2 flushes per every 1 on the 96 capillary','Proteinase K Wash','','publish','closed','open','','proteinase-k-wash','','','2014-03-10 17:54:53','2014-03-10 17:54:53','',217,'http://localhost/~psalessi/Genetics_Lab/?page_id=361',0,'page','',0),(362,1,'2014-03-10 17:29:51','2014-03-10 17:29:51','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Proteinase K Capillary Wash and Sapphire Block Wash Protocol</span></h3>\r\n<span style=\"color: #ff0000;\">WARNING: Polymer and Junk Polymer can act as a neurotoxin. Be sure to wear gloves when handling these materials.</span>\r\n\r\n<strong><span style=\"text-decoration: underline;\">Preparatory Steps</span></strong>\r\n<ol>\r\n	<li>ProK Solution:\r\n<ol>\r\n	<li>Check the fridge for previously made Proteinase K solution. It should be on the door in a small bottle marked Pro K, if the Pro K has only been used once before, go ahead and use it. If this is the second use, after youre done, pour it down the drain.</li>\r\n	<li>New Batch: 50uL of ProK + 25mL of Water</li>\r\n	<li>ProK solution can be found in the back freezer in the white box labeled Proteinase K. Only use the purified water in the bottles on the right side of the sink.</li>\r\n</ol>\r\n</li>\r\n	<li>Remove Junk polymer from refrigerator. Set aside to warm to room temperature.</li>\r\n	<li>Remove 28mL bottle of polymer from the refrigerator and a clean empty 7mL bottle from the back sink. Transfer polymer from the 28mL bottle into the 7mL bottle until the meniscus reaches the black line.</li>\r\n	<li>4.3730 buffer dilution: Using the red serological pipette, found in the Sephadex and Purification drawer, with a 25mL tip, add 15mL of 10x stock 3730 buffer, found on the door of the fridge, to a 250mL graduated cylinder. Add 135mL of HPLC water, or enough to reach the 150mL graduation.</li>\r\n	<li>Remove 3 clean trays and tops from the Sephadex and Purification drawer. One tray will be used for each of the following: buffer, waste, water.</li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Proteinase K Capillary Wash</span>\r\n<ol>\r\n	<li>Minimize collection window and enter My Computer on the desktop\r\n<ol>\r\n	<li>Enter Local Disk (C:)</li>\r\n	<li>Open Applied Biosystems folder</li>\r\n	<li>Open Firmware folder</li>\r\n	<li>Open 3730calib</li>\r\n	<li>Change Syringe Injection Speed from 200 to 100</li>\r\n	<li>Change Syringe Plunger Speed from 400 to 200</li>\r\n	<li>Save</li>\r\n</ol>\r\n</li>\r\n	<li>In the Service Console, press Stop All and wait for all symbols to turn from green squares to red circles. Restart the computer.</li>\r\n	<li>Windows will prompt your for username and password. Sign on as Administrator and enter password, and turn 3730 back on. Wait for the blinking yellow light to turn solid green.</li>\r\n	<li>Open Data Collection Software (Start, Programs, Applied Biosystems, Collection software) or use shortcut on desktop</li>\r\n	<li>Go to Manual Control in collection software.\r\n<ol>\r\n	<li>Send Defined Command for: Oven</li>\r\n	<li>Command Name: Set Oven State</li>\r\n	<li>Value: On, Send</li>\r\n	<li>Command Name: Set Oven Temperature</li>\r\n	<li>Value: 60.0, Send</li>\r\n</ol>\r\n</li>\r\n	<li>Pour Proteinase K solution that you have prepared into a clean, empty polymer jar.\r\n<ol>\r\n	<li>Remove polymer jar and replace with Proteinase K solution.</li>\r\n	<li>Transfer excess polymer into Junk Polymer bottle.</li>\r\n</ol>\r\n</li>\r\n	<li>Go to Spatial Run Scheduler in the collection software window.\r\n<ol>\r\n	<li>Find Spatial Protocols in lower right portion of window.</li>\r\n	<li>Protocol: SpatialFill_1</li>\r\n	<li>Click Start\r\n<ol>\r\n	<li>The Buffer tray will return to original position, the waste tray will rinse the capillaries, water will wash the capillaries and finally, the buffer tray returns to Active Position</li>\r\n</ol>\r\n</li>\r\n	<li>Click Stop when Collecting Data appears on the progress bar (If you are away from the machine for an extended period of time, DO NOT Accept the new spatialclick Reject and then continue with the next wash)</li>\r\n	<li>Repeat 14 more times for a total of 15 Washes (or more, if desired)</li>\r\n</ol>\r\n</li>\r\n	<li>Repeat step 7, except replace proteinase K solution with a polymer jar filled with warm HPLC grade water. This should be done for a total of 10 times to wash out any residual proteinase K solution in the capillaries. Return PK solution to the top, door shelf of the fridge, if you had to prepare the dilution. If not, pour the solution down the drain.</li>\r\n	<li><span style=\"color: #ff0000;\">Go to Manual Control and return the oven state to OFF by following steps 5 b thru c.</span></li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Sapphire Block Wash</span>\r\n<ol>\r\n	<li><span style=\"color: #ff0000;\">Remove anode jar from lower block</span>, empty the contents into the waste container and replace the jar back onto the lower block.</li>\r\n	<li>(If necessary, add additional HPLC water to polymer bottle)</li>\r\n	<li>Go to Instrument Status in the collection software</li>\r\n	<li>From the Wizards menu, select Water Wash Wizard\r\n<ol>\r\n	<li>Select Next</li>\r\n	<li>Select Water Washwait until wash is 100% complete.</li>\r\n	<li>Click Cancel. One wash is now complete.</li>\r\n	<li>Repeat these steps two more times for a total of three washes</li>\r\n</ol>\r\n</li>\r\n	<li>From the Wizards menu, select Change Polymer Wizard\r\n<ol>\r\n	<li>Make sure the polymer jar has enough volume of HPLC grade water and continue.</li>\r\n	<li>Select Same Lot, Next, Next</li>\r\n	<li>Select Prime, then Next</li>\r\n	<li>Select No Bubbles, then Next\r\n<ol>\r\n	<li>If you do see bubbles, or you are unsure if there are any, call over Sheryl, Kerry, or Shelley. You will use the Bubble Removal Wizard once you reach step 15. The bubble removal wizard requires a viscous substance, like polymer, so wait until step 15.</li>\r\n</ol>\r\n</li>\r\n	<li>Select No Bubbles, then Next</li>\r\n	<li>Select Fill Array, then Finish</li>\r\n	<li>Repeat these steps once more for a total of two washes</li>\r\n</ol>\r\n</li>\r\n	<li>Repeat step 14 with Junk Polymer bottle in place of HPLC filled jar.</li>\r\n	<li>Remove Junk Polymer jar and replace with Polymer and perform Change Polymer Wizard as in step 14.</li>\r\n	<li><span style=\"color: #ff0000;\">Return the Syringe Injection Speed from 100 to 200 and Syringe Plunger Speed from 200 to 400 repeating steps 1-4.</span></li>\r\n	<li>Press Tray so you can remove the old buffer tray and replace it with a clean, new tray. Pour all liquids from trays into the waste container. <span style=\"color: #ff0000;\">Add the appropriate solutions so the meniscus is at the black line: buffer in the buffer tray, and water in BOTH the water and waste trays.</span> Place lids and put on waste septa found in the Sephadex and Purification drawer.\r\n<ol>\r\n	<li>Dump the Waste tray into the waste container. The buffer and water trays can be dumped down the drain.</li>\r\n	<li>In a plastic tub soak all dirty trays in DI water.</li>\r\n</ol>\r\n</li>\r\n	<li><span style=\"color: #ff0000;\">Refill large anode jar with buffer, filling just below the red line. Make sure the anode is at least submersed in buffer. It is okay to be below the red line, just not above the line. Place the large anode jar onto the lower block. Make sure all buffer is wiped off of the top of the anode jar.</span></li>\r\n	<li>Press Tray again to return the buffer tray to the capillaries. Turn light off.</li>\r\n	<li>Restart the computer.</li>\r\n	<li>The machine is now ready.</li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Clean Up!</span>\r\n<ol>\r\n	<li>Wash all empty bottles, trays, graduated cylinders, and waste septa by soaking them in DI water in large plastic container.\r\n<ol>\r\n	<li>Remember to wear gloves as the residual polymer left in some of these containers can act as a neurotoxin</li>\r\n</ol>\r\n</li>\r\n	<li>Be sure to tightly close the waste container bottle after you are finished!</li>\r\n</ol>\r\nNote:This protocol is for a 48-cap array, so needs to do 2 flushes per every 1 on the 96 capillary','Proteinase K Wash','','inherit','closed','open','','361-revision-v1','','','2014-03-10 17:29:51','2014-03-10 17:29:51','',361,'http://localhost/~psalessi/Genetics_Lab/361-revision-v1/',0,'revision','',0),(363,1,'2014-03-10 17:33:25','2014-03-10 17:33:25','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">5% Acrylamide; 7.5M Urea</span></h3>\r\n<p style=\"text-align: center;\"><span style=\"color: #ff0000;\">Note: Urea and acrylamide are toxic, so wear gloves and avoid skin contamination</span></p>\r\n1X TBE: 250ml\r\n\r\n40% acrylamide: 62.5 mL (Biorad 40% acrylamide/bis solution 19:1 catalog#1610144)\r\n\r\nUrea: 227.29 g (Sigma U-1250/5kg)\r\n<ul>\r\n	<li>Mix acrylamide and TBE</li>\r\n	<li>Add urea and stir until it dissolves (at least 3 hours).</li>\r\n	<li>Note: Do not heat the solution to speed up the dissolution of urea since this will produce uric acid due to the breakdown of urea.</li>\r\n	<li>Bring to 500ml volume in graduated cylinder</li>\r\n	<li>Filter with 0.45um filter into 1L glass bottle</li>\r\n	<li>Transfer solution from glass bottles to brown plastic bottles</li>\r\n	<li>Store at 4C in the dark.</li>\r\n</ul>','Acrylamide (Manual Gel)','','publish','closed','open','','acrylamide-manual-gel','','','2014-03-10 17:55:16','2014-03-10 17:55:16','',217,'http://localhost/~psalessi/Genetics_Lab/?page_id=363',0,'page','',0),(364,1,'2014-03-10 17:33:25','2014-03-10 17:33:25','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">5% Acrylamide; 7.5M Urea</span></h3>\r\n<span style=\"color: #ff0000;\">Note: Urea and acrylamide are toxic, so wear gloves and avoid skin contamination</span>\r\n\r\n1X TBE: 250ml\r\n\r\n40% acrylamide: 62.5 mL (Biorad 40% acrylamide/bis solution 19:1 catalog#1610144)\r\n\r\nUrea: 227.29 g (Sigma U-1250/5kg)\r\n<ul>\r\n	<li>Mix acrylamide and TBE</li>\r\n	<li>Add urea and stir until it dissolves (at least 3 hours).</li>\r\n	<li>Note: Do not heat the solution to speed up the dissolution of urea since this will produce uric acid due to the breakdown of urea.</li>\r\n	<li>Bring to 500ml volume in graduated cylinder</li>\r\n	<li>Filter with 0.45um filter into 1L glass bottle</li>\r\n	<li>Transfer solution from glass bottles to brown plastic bottles</li>\r\n	<li>Store at 4C in the dark.</li>\r\n</ul>','Acrylamide (Manual Gel)','','inherit','closed','open','','363-revision-v1','','','2014-03-10 17:33:25','2014-03-10 17:33:25','',363,'http://localhost/~psalessi/Genetics_Lab/363-revision-v1/',0,'revision','',0),(365,1,'2014-03-10 17:47:34','2014-03-10 17:47:34','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Pouring Acrylamide Gels for Manual Run</span></h3>\r\n<span style=\"text-decoration: underline;\"><strong>Hazardous materials:</strong></span>\r\n\r\nAcrylamide: neurotoxin\r\n\r\nTEMED: corrosive\r\n\r\nFor complete information, refer to the MSDS for each substance.\r\n\r\n<span style=\"text-decoration: underline;\"><strong>PPE required:</strong></span>\r\n\r\nGloves: Nitrile\r\n\r\nEyewear\r\n\r\nLab Coat\r\n\r\nClosed toed shoes\r\n\r\n<span style=\"text-decoration: underline;\"><strong>Precautions to observe:</strong></span>\r\n\r\nMix polyacrylamide under hood\r\n\r\nAvoid contact with skin/eyes\r\n\r\n<span style=\"text-decoration: underline;\"><strong><span style=\"line-height: 1.714285714; font-size: 1rem;\">In case of contact:</span></strong></span>\r\n\r\nIf in contact with skin: remove clothing and use emergency shower or eye wash for 15 min\r\n\r\nCall EHS to clean up water in hallway\r\n\r\n<span style=\"text-decoration: underline;\"><strong><span style=\"line-height: 1.714285714; font-size: 1rem;\">Procedure:</span></strong></span>\r\n\r\nPrepare plates (use the gel-pouring bench on the west side of the main lab)\r\n<ol>\r\n	<li>Cut bench paper large enough to cover the surface that you will be using.</li>\r\n	<li>Choose 1 short and 1 long plate per gel</li>\r\n	<li>Set plates on white, plastic tube racks.</li>\r\n	<li>Clean the inside of both plates with 95% ethanol and a Kimwipe</li>\r\n	<li>Clean with isopropanol and a kimwipe.\r\n<ul>\r\n	<li>The plates should be completely free of smears or grease. After the isopropanol wash the glass will feel kind of slippery.</li>\r\n</ul>\r\n</li>\r\n	<li>Wipe a small amount of Rain-X on the inside of the short plate ONLY.</li>\r\n	<li>Place spacers on each (long) side of the long plate.</li>\r\n	<li>Place the clean, Rain-Xed side of the short plate on top of the spacers, facing the long plate.\r\n<ul>\r\n	<li>The bottom of the plates should be flush and spacers should be sandwiched inside the plates, aligned with the outer, long edges.</li>\r\n</ul>\r\n</li>\r\n	<li>Use 2-4 binding clips to clip and hold the plates together.</li>\r\n	<li>Apply a casting boot to the bottom and sides of the plates removing clips as necessary.\r\n<ul>\r\n	<li>Make sure the short plate is on top of the long plate and the hole to insert the syringe at the bottom of the boot is facing up.</li>\r\n</ul>\r\n</li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Preparation of polyacrylamide (must be done in the hood)</span>\r\n<ol>\r\n	<li>For each gel to be poured use the following:\r\n<ul>\r\n	<li>30ml 5% acrylamide (from 4 degree refrigerator)</li>\r\n	<li>90l TEMED (in the hood)</li>\r\n	<li>90l 10% APS (in 4 degree refrigerator)</li>\r\n</ul>\r\n</li>\r\n	<li>Swirl gently in beaker.</li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Pouring polyacrylamide into ge</span>l\r\n<ol>\r\n	<li>Carefully draw ~35ml of the polyacrylamide solution into a plastic syringe avoiding air bubbles.</li>\r\n	<li>Insert syringe into the hole in the bottom of the casting boot.</li>\r\n	<li>Slowly depress plunger, allowing acrylamide to flow in between the plates.</li>\r\n	<li>Proceed until the acrylamide fills the space between the plates and begins to run out at the top of the short plate.</li>\r\n	<li>Remove syringe.</li>\r\n	<li>Plug hole in casting boot with modified pipette tip.</li>\r\n	<li>Insert comb, blunt side down, into the space at between the plates at the top of the gel.</li>\r\n	<li>Use 1-3 binding clips to secure the comb and the top of the plates together.</li>\r\n	<li>Wait ~20 minutes to allow adequate polymerization.</li>\r\n</ol>\r\nNotes:\r\n<ul>\r\n	<li>If you have trouble and have to discard the gel, POLYMERIZE the acrylamide first, before disposal. NEVER pour unpolymerized acrylamide down the drain.</li>\r\n	<li>Dispose of used bench paper after each use.</li>\r\n	<li>If supplies are running low, be sure to notify the ordering or stocks personnel.</li>\r\n	<li>Polymerized gels can be stored for future use by placing dampened (with water) paper towels over the top/bottom ends and covering the whole gel in saran wrap to avoid dehydration. Remove the casing boot prior to wrapping and storage.</li>\r\n</ul>','Acrylamide Gel Pour','','publish','closed','open','','acrylamide-gel-pour','','','2014-03-10 17:47:34','2014-03-10 17:47:34','',217,'http://localhost/~psalessi/Genetics_Lab/?page_id=365',0,'page','',0),(366,1,'2014-03-10 17:47:34','2014-03-10 17:47:34','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Pouring Acrylamide Gels for Manual Run</span></h3>\r\n<span style=\"text-decoration: underline;\"><strong>Hazardous materials:</strong></span>\r\n\r\nAcrylamide: neurotoxin\r\n\r\nTEMED: corrosive\r\n\r\nFor complete information, refer to the MSDS for each substance.\r\n\r\n<span style=\"text-decoration: underline;\"><strong>PPE required:</strong></span>\r\n\r\nGloves: Nitrile\r\n\r\nEyewear\r\n\r\nLab Coat\r\n\r\nClosed toed shoes\r\n\r\n<span style=\"text-decoration: underline;\"><strong>Precautions to observe:</strong></span>\r\n\r\nMix polyacrylamide under hood\r\n\r\nAvoid contact with skin/eyes\r\n\r\n<span style=\"text-decoration: underline;\"><strong><span style=\"line-height: 1.714285714; font-size: 1rem;\">In case of contact:</span></strong></span>\r\n\r\nIf in contact with skin: remove clothing and use emergency shower or eye wash for 15 min\r\n\r\nCall EHS to clean up water in hallway\r\n\r\n<span style=\"text-decoration: underline;\"><strong><span style=\"line-height: 1.714285714; font-size: 1rem;\">Procedure:</span></strong></span>\r\n\r\nPrepare plates (use the gel-pouring bench on the west side of the main lab)\r\n<ol>\r\n	<li>Cut bench paper large enough to cover the surface that you will be using.</li>\r\n	<li>Choose 1 short and 1 long plate per gel</li>\r\n	<li>Set plates on white, plastic tube racks.</li>\r\n	<li>Clean the inside of both plates with 95% ethanol and a Kimwipe</li>\r\n	<li>Clean with isopropanol and a kimwipe.\r\n<ul>\r\n	<li>The plates should be completely free of smears or grease. After the isopropanol wash the glass will feel kind of slippery.</li>\r\n</ul>\r\n</li>\r\n	<li>Wipe a small amount of Rain-X on the inside of the short plate ONLY.</li>\r\n	<li>Place spacers on each (long) side of the long plate.</li>\r\n	<li>Place the clean, Rain-Xed side of the short plate on top of the spacers, facing the long plate.\r\n<ul>\r\n	<li>The bottom of the plates should be flush and spacers should be sandwiched inside the plates, aligned with the outer, long edges.</li>\r\n</ul>\r\n</li>\r\n	<li>Use 2-4 binding clips to clip and hold the plates together.</li>\r\n	<li>Apply a casting boot to the bottom and sides of the plates removing clips as necessary.\r\n<ul>\r\n	<li>Make sure the short plate is on top of the long plate and the hole to insert the syringe at the bottom of the boot is facing up.</li>\r\n</ul>\r\n</li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Preparation of polyacrylamide (must be done in the hood)</span>\r\n<ol>\r\n	<li>For each gel to be poured use the following:\r\n<ul>\r\n	<li>30ml 5% acrylamide (from 4 degree refrigerator)</li>\r\n	<li>90l TEMED (in the hood)</li>\r\n	<li>90l 10% APS (in 4 degree refrigerator)</li>\r\n</ul>\r\n</li>\r\n	<li>Swirl gently in beaker.</li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Pouring polyacrylamide into ge</span>l\r\n<ol>\r\n	<li>Carefully draw ~35ml of the polyacrylamide solution into a plastic syringe avoiding air bubbles.</li>\r\n	<li>Insert syringe into the hole in the bottom of the casting boot.</li>\r\n	<li>Slowly depress plunger, allowing acrylamide to flow in between the plates.</li>\r\n	<li>Proceed until the acrylamide fills the space between the plates and begins to run out at the top of the short plate.</li>\r\n	<li>Remove syringe.</li>\r\n	<li>Plug hole in casting boot with modified pipette tip.</li>\r\n	<li>Insert comb, blunt side down, into the space at between the plates at the top of the gel.</li>\r\n	<li>Use 1-3 binding clips to secure the comb and the top of the plates together.</li>\r\n	<li>Wait ~20 minutes to allow adequate polymerization.</li>\r\n</ol>\r\nNotes:\r\n<ul>\r\n	<li>If you have trouble and have to discard the gel, POLYMERIZE the acrylamide first, before disposal. NEVER pour unpolymerized acrylamide down the drain.</li>\r\n	<li>Dispose of used bench paper after each use.</li>\r\n	<li>If supplies are running low, be sure to notify the ordering or stocks personnel.</li>\r\n	<li>Polymerized gels can be stored for future use by placing dampened (with water) paper towels over the top/bottom ends and covering the whole gel in saran wrap to avoid dehydration. Remove the casing boot prior to wrapping and storage.</li>\r\n</ul>','Acrylamide Gel Pour','','inherit','closed','open','','365-revision-v1','','','2014-03-10 17:47:34','2014-03-10 17:47:34','',365,'http://localhost/~psalessi/Genetics_Lab/365-revision-v1/',0,'revision','',0),(367,1,'2014-03-10 17:51:46','2014-03-10 17:51:46','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Running Manual (not BaseStation) Acrylamide Gels</span></h3>\r\n<ol>\r\n	<li>Put bottom tray into gel rig.</li>\r\n	<li>Remove clamps and wrap.</li>\r\n	<li>Remove boot and wash acrylamide off glass with normal water.</li>\r\n	<li>With gentle water running take out comb.</li>\r\n	<li>Remove any extra acrylamide and dry glass.</li>\r\n	<li>Take 2 black gel spacers (little and sponge-like)</li>\r\n	<li>Put plates into gel rig with short plate in back and sponges on side right on top of the plates (to prevent leaks).</li>\r\n	<li>Lock plates in with side clamps of gel rig. Clamp down very tightly.</li>\r\n	<li>Close drain on side.</li>\r\n	<li>Add 1X TBE to top, filling just a little below back knob just above liquid.</li>\r\n	<li>Fill bottom tray until little knobs next to plate are just above the liquid.</li>\r\n	<li>Prerun gel for 20 minutes at 1900V, 50W.</li>\r\n	<li>For 10ul PCR reactions, add 1:1 loading dye (so 10ul of loading dye). Put the dye in a tray and use a multi-channel pipet. Pour the extra back into the dye container.</li>\r\n	<li>Heat plate for 3 minutes at 95C using Fisher hotplate and put weight on top.</li>\r\n	<li>The ladder (often use 400bp) is in 4C fridge and it needs to be heated to 95C for 2 minutes.</li>\r\n	<li>With a syringe, rinse the urea out of top of gel.</li>\r\n	<li>Put comb (teeth down) back into gel until teeth have entered acrylamide but not too deeply.</li>\r\n	<li>Add 3 ul of sample to each well using an 8 channel pipettor if possible.</li>\r\n	<li>Attach power leads and run at 1900V, 50W until dye front is near bottom.</li>\r\n	<li>Turn off power and open drain to remove buffer from top chamber.</li>\r\n	<li>Remove glass plates from gel rig.</li>\r\n	<li>Push spacer on right out a little bit and take a spatula to carefully open plate and remove smaller plate. Set it aside.</li>\r\n	<li>Put 0.8 grams of agarose into 30 ml nanopure water.</li>\r\n	<li>In a 50ml conical tube, add 25ml of nanopure water and 0.5ul of SYBR Green.</li>\r\n	<li>Pour SYBR Green/water solution into melted agarose flask.[alternatively add 50ml nanopure to 0.8g agarose and microwave; add 0.5ul of SYBR Green]\r\n<ul>\r\n	<li>Mix together and pour over gel. Smooth solution out over gel with hands quickly.</li>\r\n</ul>\r\n</li>\r\n	<li>Let it stain for ~10 minutes and then wipe plate with Kimwipe to remove any acrylamide from bottom and sides of plate so acrylamide wont fall in Flourimager.</li>\r\n	<li>Put into Flourimager with combs facing towards you.</li>\r\n	<li>In flourimager program: Templates -&gt; Acryl top -&gt; name scan something</li>\r\n	<li>After scanning, clean gel off plate with a paper towel.</li>\r\n	<li>Use Sigma cleaner and a sponge to clean plates.</li>\r\n	<li>Rinse 1X with regular water and 1X with dI water. Can reuse 1X TBE until it turns blue.</li>\r\n</ol>','Running Acrylamide','','publish','closed','open','','running-acrylamide','','','2014-03-10 17:51:59','2014-03-10 17:51:59','',217,'http://localhost/~psalessi/Genetics_Lab/?page_id=367',0,'page','',0),(368,1,'2014-03-10 17:51:46','2014-03-10 17:51:46','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Running Manual (not BaseStation) Acrylamide Gels</span></h3>\r\n<ol>\r\n	<li>Put bottom tray into gel rig.</li>\r\n	<li>Remove clamps and wrap.</li>\r\n	<li>Remove boot and wash acrylamide off glass with normal water.</li>\r\n	<li>With gentle water running take out comb.</li>\r\n	<li>Remove any extra acrylamide and dry glass.</li>\r\n	<li>Take 2 black gel spacers (little and sponge-like)</li>\r\n	<li>Put plates into gel rig with short plate in back and sponges on side right on top of the plates (to prevent leaks).</li>\r\n	<li>Lock plates in with side clamps of gel rig. Clamp down very tightly.</li>\r\n	<li>Close drain on side.</li>\r\n	<li>Add 1X TBE to top, filling just a little below back knob just above liquid.</li>\r\n	<li>Fill bottom tray until little knobs next to plate are just above the liquid.</li>\r\n	<li>Prerun gel for 20 minutes at 1900V, 50W.</li>\r\n	<li>For 10ul PCR reactions, add 1:1 loading dye (so 10ul of loading dye). Put the dye in a tray and use a multi-channel pipet. Pour the extra back into the dye container.</li>\r\n	<li>Heat plate for 3 minutes at 95C using Fisher hotplate and put weight on top.</li>\r\n	<li>The ladder (often use 400bp) is in 4C fridge and it needs to be heated to 95C for 2 minutes.</li>\r\n	<li>With a syringe, rinse the urea out of top of gel.</li>\r\n	<li>Put comb (teeth down) back into gel until teeth have entered acrylamide but not too deeply.</li>\r\n	<li>Add 3 ul of sample to each well using an 8 channel pipettor if possible.</li>\r\n	<li>Attach power leads and run at 1900V, 50W until dye front is near bottom.</li>\r\n	<li>Turn off power and open drain to remove buffer from top chamber.</li>\r\n	<li>Remove glass plates from gel rig.</li>\r\n	<li>Push spacer on right out a little bit and take a spatula to carefully open plate and remove smaller plate. Set it aside.</li>\r\n	<li>Put 0.8 grams of agarose into 30 ml nanopure water.</li>\r\n	<li>In a 50ml conical tube, add 25ml of nanopure water and 0.5ul of SYBR Green.</li>\r\n	<li>Pour SYBR Green/water solution into melted agarose flask.[alternatively add 50ml nanopure to 0.8g agarose and microwave; add 0.5ul of SYBR Green]\r\n<ul>\r\n	<li>Mix together and pour over gel. Smooth solution out over gel with hands quickly.</li>\r\n</ul>\r\n</li>\r\n	<li>Let it stain for ~10 minutes and then wipe plate with Kimwipe to remove any acrylamide from bottom and sides of plate so acrylamide wont fall in Flourimager.</li>\r\n	<li>Put into Flourimager with combs facing towards you.</li>\r\n	<li>In flourimager program: Templates -&gt; Acryl top -&gt; name scan something</li>\r\n	<li>After scanning, clean gel off plate with a paper towel.</li>\r\n	<li>Use Sigma cleaner and a sponge to clean plates.</li>\r\n	<li>Rinse 1X with regular water and 1X with dI water. Can reuse 1X TBE until it turns blue.</li>\r\n</ol>','Running Acrylamide','','inherit','closed','open','','367-revision-v1','','','2014-03-10 17:51:46','2014-03-10 17:51:46','',367,'http://localhost/~psalessi/Genetics_Lab/367-revision-v1/',0,'revision','',0),(369,1,'2014-03-10 17:51:59','2014-03-10 17:51:59','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Running Manual (not BaseStation) Acrylamide Gels</span></h3>\r\n<ol>\r\n	<li>Put bottom tray into gel rig.</li>\r\n	<li>Remove clamps and wrap.</li>\r\n	<li>Remove boot and wash acrylamide off glass with normal water.</li>\r\n	<li>With gentle water running take out comb.</li>\r\n	<li>Remove any extra acrylamide and dry glass.</li>\r\n	<li>Take 2 black gel spacers (little and sponge-like)</li>\r\n	<li>Put plates into gel rig with short plate in back and sponges on side right on top of the plates (to prevent leaks).</li>\r\n	<li>Lock plates in with side clamps of gel rig. Clamp down very tightly.</li>\r\n	<li>Close drain on side.</li>\r\n	<li>Add 1X TBE to top, filling just a little below back knob just above liquid.</li>\r\n	<li>Fill bottom tray until little knobs next to plate are just above the liquid.</li>\r\n	<li>Prerun gel for 20 minutes at 1900V, 50W.</li>\r\n	<li>For 10ul PCR reactions, add 1:1 loading dye (so 10ul of loading dye). Put the dye in a tray and use a multi-channel pipet. Pour the extra back into the dye container.</li>\r\n	<li>Heat plate for 3 minutes at 95C using Fisher hotplate and put weight on top.</li>\r\n	<li>The ladder (often use 400bp) is in 4C fridge and it needs to be heated to 95C for 2 minutes.</li>\r\n	<li>With a syringe, rinse the urea out of top of gel.</li>\r\n	<li>Put comb (teeth down) back into gel until teeth have entered acrylamide but not too deeply.</li>\r\n	<li>Add 3 ul of sample to each well using an 8 channel pipettor if possible.</li>\r\n	<li>Attach power leads and run at 1900V, 50W until dye front is near bottom.</li>\r\n	<li>Turn off power and open drain to remove buffer from top chamber.</li>\r\n	<li>Remove glass plates from gel rig.</li>\r\n	<li>Push spacer on right out a little bit and take a spatula to carefully open plate and remove smaller plate. Set it aside.</li>\r\n	<li>Put 0.8 grams of agarose into 30 ml nanopure water.</li>\r\n	<li>In a 50ml conical tube, add 25ml of nanopure water and 0.5ul of SYBR Green.</li>\r\n	<li>Pour SYBR Green/water solution into melted agarose flask.[alternatively add 50ml nanopure to 0.8g agarose and microwave; add 0.5ul of SYBR Green]\r\n<ul>\r\n	<li>Mix together and pour over gel. Smooth solution out over gel with hands quickly.</li>\r\n</ul>\r\n</li>\r\n	<li>Let it stain for ~10 minutes and then wipe plate with Kimwipe to remove any acrylamide from bottom and sides of plate so acrylamide wont fall in Flourimager.</li>\r\n	<li>Put into Flourimager with combs facing towards you.</li>\r\n	<li>In flourimager program: Templates -&gt; Acryl top -&gt; name scan something</li>\r\n	<li>After scanning, clean gel off plate with a paper towel.</li>\r\n	<li>Use Sigma cleaner and a sponge to clean plates.</li>\r\n	<li>Rinse 1X with regular water and 1X with dI water. Can reuse 1X TBE until it turns blue.</li>\r\n</ol>','Running Acrylamide','','inherit','closed','open','','367-revision-v1','','','2014-03-10 17:51:59','2014-03-10 17:51:59','',367,'http://localhost/~psalessi/Genetics_Lab/367-revision-v1/',0,'revision','',0),(370,1,'2014-03-10 17:52:46','2014-03-10 17:52:46','<strong>Buffers &amp; Stocks</strong>\r\n<ul>\r\n	<li><a title=\"98% Formamide Loading Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/98-formamide-loading-dye/\">Formamide Loading Dye</a></li>\r\n	<li><a title=\"Diluting 1Kb Plus Ladder with Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/diluting-1kb-plus-ladder-with-dye/\">Diluting 1kb Ladder/Dye</a></li>\r\n	<li><a title=\"Gelco Fluorescein Ladder\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/gelco-fluorescein-ladder/\">Gelco Fluorescein Ladder</a></li>\r\n	<li><a title=\"Proteinase K\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/proteinase-k/\">Proteinase K</a></li>\r\n	<li><a title=\"Glassware Cleaner\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/glassware-cleaner/\">Glassware Cleaner</a></li>\r\n	<li><a title=\"dNTPs (Promega)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/dntps-promega/\">dNTPs (Promega)</a></li>\r\n	<li><a title=\"1M Tris-Cl\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1m-tris-cl/\">1M Tris-Cl</a></li>\r\n	<li><a title=\".5M EDTA (pH 8.0)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/5m-edta-ph-8-0/\">0.5M EDTA (pH 8.0)</a></li>\r\n	<li><a title=\"1X TE (10mM Tris; 1mM EDTA)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1x-te-10mm-tris-1mm-edta/\">1X TE (10mM Tris; 1mM EDTA)</a></li>\r\n	<li><a title=\"1X TLE (10mM Tris; 0.1mM EDTA)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1x-tle-10mm-tris-0-1mm-edta/\">1X TLE (10mM Tris; 0.1 mM EDTA)</a></li>\r\n	<li><a title=\"10X TNE\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/10x-tne/\">10X TNE</a></li>\r\n	<li><a title=\"10X TBE One-Shots\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/10x-tbe-one-shots/\">10X TBE One-Shots</a></li>\r\n</ul>\r\n<strong>FujiFilm</strong>\r\n<ul>\r\n	<li><a title=\"FujiFilm DNA Quantification\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/fujifilm-dna-quantification/\">DNA quantification</a></li>\r\n</ul>\r\n<strong>ABI 3730xl Maintenance</strong>\r\n<ul>\r\n	<li><a title=\"3730xl General Maintenance\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/3730xl-general-maintenance/\">3730xl Maintenance</a></li>\r\n	<li><a title=\"ProtKwash.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/ProtKwash.html\">Proteinase K Wash</a></li>\r\n</ul>\r\n<strong>Acrylamide gels</strong>\r\n<ul>\r\n	<li><a title=\"Acrylamide (Manual Gel)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/acrylamide-manual-gel/\">Acrylamide Gel</a></li>\r\n	<li><a title=\"Acrylamide Gel Pour\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/acrylamide-gel-pour/\">Pouring</a></li>\r\n	<li><a title=\"Running Acrylamide\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/running-acrylamide/\">Running</a></li>\r\n</ul>\r\n<strong>Fluidigm</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/FLUIDIGM EP1 PROTOCOL.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/FLUIDIGM%20EP1%20PROTOCOL.doc\">Fluidigm EP1 Protocol</a></li>\r\n</ul>\r\n<strong>Illumina</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/RAD-SEQ Protocol.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/RAD-SEQ%20Protocol.doc\">RAD-SEQ Protocol</a></li>\r\n</ul>\r\n<strong>Other</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/tissuesampleprot.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/tissuesampleprot.pdf\">Tissue Sampling</a></li>\r\n	<li><a title=\"Supplies.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Supplies.html\">Ordering and Receiving Supplies</a></li>\r\n	<li><a title=\"Supplies.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Supplies.html\">PCR Tips and Tricks</a></li>\r\n</ul>','Protocols and Procedures','','inherit','closed','open','','217-revision-v1','','','2014-03-10 17:52:46','2014-03-10 17:52:46','',217,'http://localhost/~psalessi/Genetics_Lab/217-revision-v1/',0,'revision','',0),(371,1,'2014-03-10 17:53:27','2014-03-10 17:53:27','<strong>Buffers &amp; Stocks</strong>\r\n<ul>\r\n	<li><a title=\"98% Formamide Loading Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/98-formamide-loading-dye/\">Formamide Loading Dye</a></li>\r\n	<li><a title=\"Diluting 1Kb Plus Ladder with Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/diluting-1kb-plus-ladder-with-dye/\">Diluting 1kb Ladder/Dye</a></li>\r\n	<li><a title=\"Gelco Fluorescein Ladder\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/gelco-fluorescein-ladder/\">Gelco Fluorescein Ladder</a></li>\r\n	<li><a title=\"Proteinase K\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/proteinase-k/\">Proteinase K</a></li>\r\n	<li><a title=\"Glassware Cleaner\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/glassware-cleaner/\">Glassware Cleaner</a></li>\r\n	<li><a title=\"dNTPs (Promega)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/dntps-promega/\">dNTPs (Promega)</a></li>\r\n	<li><a title=\"1M Tris-Cl\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1m-tris-cl/\">1M Tris-Cl</a></li>\r\n	<li><a title=\".5M EDTA (pH 8.0)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/5m-edta-ph-8-0/\">0.5M EDTA (pH 8.0)</a></li>\r\n	<li><a title=\"1X TE (10mM Tris; 1mM EDTA)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1x-te-10mm-tris-1mm-edta/\">1X TE (10mM Tris; 1mM EDTA)</a></li>\r\n	<li><a title=\"1X TLE (10mM Tris; 0.1mM EDTA)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1x-tle-10mm-tris-0-1mm-edta/\">1X TLE (10mM Tris; 0.1 mM EDTA)</a></li>\r\n	<li><a title=\"10X TNE\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/10x-tne/\">10X TNE</a></li>\r\n	<li><a title=\"10X TBE One-Shots\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/10x-tbe-one-shots/\">10X TBE One-Shots</a></li>\r\n</ul>\r\n<strong>FujiFilm</strong>\r\n<ul>\r\n	<li><a title=\"FujiFilm DNA Quantification\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/fujifilm-dna-quantification/\">DNA quantification</a></li>\r\n</ul>\r\n<strong>ABI 3730xl Maintenance</strong>\r\n<ul>\r\n	<li><a title=\"3730xl General Maintenance\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/3730xl-general-maintenance/\">3730xl Maintenance</a></li>\r\n	<li><a title=\"Proteinase K Wash\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/proteinase-k-wash/\">Proteinase K Wash</a></li>\r\n</ul>\r\n<strong>Acrylamide gels</strong>\r\n<ul>\r\n	<li><a title=\"Acrylamide (Manual Gel)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/acrylamide-manual-gel/\">Acrylamide Gel</a></li>\r\n	<li><a title=\"Acrylamide Gel Pour\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/acrylamide-gel-pour/\">Pouring</a></li>\r\n	<li><a title=\"Running Acrylamide\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/running-acrylamide/\">Running</a></li>\r\n</ul>\r\n<strong>Fluidigm</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/FLUIDIGM EP1 PROTOCOL.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/FLUIDIGM%20EP1%20PROTOCOL.doc\">Fluidigm EP1 Protocol</a></li>\r\n</ul>\r\n<strong>Illumina</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/RAD-SEQ Protocol.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/RAD-SEQ%20Protocol.doc\">RAD-SEQ Protocol</a></li>\r\n</ul>\r\n<strong>Other</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/tissuesampleprot.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/tissuesampleprot.pdf\">Tissue Sampling</a></li>\r\n	<li><a title=\"Supplies.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Supplies.html\">Ordering and Receiving Supplies</a></li>\r\n	<li><a title=\"Supplies.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Supplies.html\">PCR Tips and Tricks</a></li>\r\n</ul>','Protocols and Procedures','','inherit','closed','open','','217-revision-v1','','','2014-03-10 17:53:27','2014-03-10 17:53:27','',217,'http://localhost/~psalessi/Genetics_Lab/217-revision-v1/',0,'revision','',0),(372,1,'2014-03-10 17:54:53','2014-03-10 17:54:53','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Proteinase K Capillary Wash and Sapphire Block Wash Protocol</span></h3>\r\n<p style=\"text-align: center;\"><span style=\"color: #ff0000;\">WARNING: Polymer and Junk Polymer can act as a neurotoxin. Be sure to wear gloves when handling these materials.</span></p>\r\n<strong><span style=\"text-decoration: underline;\">Preparatory Steps</span></strong>\r\n<ol>\r\n	<li>ProK Solution:\r\n<ol>\r\n	<li>Check the fridge for previously made Proteinase K solution. It should be on the door in a small bottle marked Pro K, if the Pro K has only been used once before, go ahead and use it. If this is the second use, after youre done, pour it down the drain.</li>\r\n	<li>New Batch: 50uL of ProK + 25mL of Water</li>\r\n	<li>ProK solution can be found in the back freezer in the white box labeled Proteinase K. Only use the purified water in the bottles on the right side of the sink.</li>\r\n</ol>\r\n</li>\r\n	<li>Remove Junk polymer from refrigerator. Set aside to warm to room temperature.</li>\r\n	<li>Remove 28mL bottle of polymer from the refrigerator and a clean empty 7mL bottle from the back sink. Transfer polymer from the 28mL bottle into the 7mL bottle until the meniscus reaches the black line.</li>\r\n	<li>4.3730 buffer dilution: Using the red serological pipette, found in the Sephadex and Purification drawer, with a 25mL tip, add 15mL of 10x stock 3730 buffer, found on the door of the fridge, to a 250mL graduated cylinder. Add 135mL of HPLC water, or enough to reach the 150mL graduation.</li>\r\n	<li>Remove 3 clean trays and tops from the Sephadex and Purification drawer. One tray will be used for each of the following: buffer, waste, water.</li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Proteinase K Capillary Wash</span>\r\n<ol>\r\n	<li>Minimize collection window and enter My Computer on the desktop\r\n<ol>\r\n	<li>Enter Local Disk (C:)</li>\r\n	<li>Open Applied Biosystems folder</li>\r\n	<li>Open Firmware folder</li>\r\n	<li>Open 3730calib</li>\r\n	<li>Change Syringe Injection Speed from 200 to 100</li>\r\n	<li>Change Syringe Plunger Speed from 400 to 200</li>\r\n	<li>Save</li>\r\n</ol>\r\n</li>\r\n	<li>In the Service Console, press Stop All and wait for all symbols to turn from green squares to red circles. Restart the computer.</li>\r\n	<li>Windows will prompt your for username and password. Sign on as Administrator and enter password, and turn 3730 back on. Wait for the blinking yellow light to turn solid green.</li>\r\n	<li>Open Data Collection Software (Start, Programs, Applied Biosystems, Collection software) or use shortcut on desktop</li>\r\n	<li>Go to Manual Control in collection software.\r\n<ol>\r\n	<li>Send Defined Command for: Oven</li>\r\n	<li>Command Name: Set Oven State</li>\r\n	<li>Value: On, Send</li>\r\n	<li>Command Name: Set Oven Temperature</li>\r\n	<li>Value: 60.0, Send</li>\r\n</ol>\r\n</li>\r\n	<li>Pour Proteinase K solution that you have prepared into a clean, empty polymer jar.\r\n<ol>\r\n	<li>Remove polymer jar and replace with Proteinase K solution.</li>\r\n	<li>Transfer excess polymer into Junk Polymer bottle.</li>\r\n</ol>\r\n</li>\r\n	<li>Go to Spatial Run Scheduler in the collection software window.\r\n<ol>\r\n	<li>Find Spatial Protocols in lower right portion of window.</li>\r\n	<li>Protocol: SpatialFill_1</li>\r\n	<li>Click Start\r\n<ol>\r\n	<li>The Buffer tray will return to original position, the waste tray will rinse the capillaries, water will wash the capillaries and finally, the buffer tray returns to Active Position</li>\r\n</ol>\r\n</li>\r\n	<li>Click Stop when Collecting Data appears on the progress bar (If you are away from the machine for an extended period of time, DO NOT Accept the new spatialclick Reject and then continue with the next wash)</li>\r\n	<li>Repeat 14 more times for a total of 15 Washes (or more, if desired)</li>\r\n</ol>\r\n</li>\r\n	<li>Repeat step 7, except replace proteinase K solution with a polymer jar filled with warm HPLC grade water. This should be done for a total of 10 times to wash out any residual proteinase K solution in the capillaries. Return PK solution to the top, door shelf of the fridge, if you had to prepare the dilution. If not, pour the solution down the drain.</li>\r\n	<li><span style=\"color: #ff0000;\">Go to Manual Control and return the oven state to OFF by following steps 5 b thru c.</span></li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Sapphire Block Wash</span>\r\n<ol>\r\n	<li><span style=\"color: #ff0000;\">Remove anode jar from lower block</span>, empty the contents into the waste container and replace the jar back onto the lower block.</li>\r\n	<li>(If necessary, add additional HPLC water to polymer bottle)</li>\r\n	<li>Go to Instrument Status in the collection software</li>\r\n	<li>From the Wizards menu, select Water Wash Wizard\r\n<ol>\r\n	<li>Select Next</li>\r\n	<li>Select Water Washwait until wash is 100% complete.</li>\r\n	<li>Click Cancel. One wash is now complete.</li>\r\n	<li>Repeat these steps two more times for a total of three washes</li>\r\n</ol>\r\n</li>\r\n	<li>From the Wizards menu, select Change Polymer Wizard\r\n<ol>\r\n	<li>Make sure the polymer jar has enough volume of HPLC grade water and continue.</li>\r\n	<li>Select Same Lot, Next, Next</li>\r\n	<li>Select Prime, then Next</li>\r\n	<li>Select No Bubbles, then Next\r\n<ol>\r\n	<li>If you do see bubbles, or you are unsure if there are any, call over Sheryl, Kerry, or Shelley. You will use the Bubble Removal Wizard once you reach step 15. The bubble removal wizard requires a viscous substance, like polymer, so wait until step 15.</li>\r\n</ol>\r\n</li>\r\n	<li>Select No Bubbles, then Next</li>\r\n	<li>Select Fill Array, then Finish</li>\r\n	<li>Repeat these steps once more for a total of two washes</li>\r\n</ol>\r\n</li>\r\n	<li>Repeat step 14 with Junk Polymer bottle in place of HPLC filled jar.</li>\r\n	<li>Remove Junk Polymer jar and replace with Polymer and perform Change Polymer Wizard as in step 14.</li>\r\n	<li><span style=\"color: #ff0000;\">Return the Syringe Injection Speed from 100 to 200 and Syringe Plunger Speed from 200 to 400 repeating steps 1-4.</span></li>\r\n	<li>Press Tray so you can remove the old buffer tray and replace it with a clean, new tray. Pour all liquids from trays into the waste container. <span style=\"color: #ff0000;\">Add the appropriate solutions so the meniscus is at the black line: buffer in the buffer tray, and water in BOTH the water and waste trays.</span> Place lids and put on waste septa found in the Sephadex and Purification drawer.\r\n<ol>\r\n	<li>Dump the Waste tray into the waste container. The buffer and water trays can be dumped down the drain.</li>\r\n	<li>In a plastic tub soak all dirty trays in DI water.</li>\r\n</ol>\r\n</li>\r\n	<li><span style=\"color: #ff0000;\">Refill large anode jar with buffer, filling just below the red line. Make sure the anode is at least submersed in buffer. It is okay to be below the red line, just not above the line. Place the large anode jar onto the lower block. Make sure all buffer is wiped off of the top of the anode jar.</span></li>\r\n	<li>Press Tray again to return the buffer tray to the capillaries. Turn light off.</li>\r\n	<li>Restart the computer.</li>\r\n	<li>The machine is now ready.</li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Clean Up!</span>\r\n<ol>\r\n	<li>Wash all empty bottles, trays, graduated cylinders, and waste septa by soaking them in DI water in large plastic container.\r\n<ol>\r\n	<li>Remember to wear gloves as the residual polymer left in some of these containers can act as a neurotoxin</li>\r\n</ol>\r\n</li>\r\n	<li>Be sure to tightly close the waste container bottle after you are finished!</li>\r\n</ol>\r\nNote:This protocol is for a 48-cap array, so needs to do 2 flushes per every 1 on the 96 capillary','Proteinase K Wash','','inherit','closed','open','','361-revision-v1','','','2014-03-10 17:54:53','2014-03-10 17:54:53','',361,'http://localhost/~psalessi/Genetics_Lab/361-revision-v1/',0,'revision','',0),(373,1,'2014-03-10 17:55:16','2014-03-10 17:55:16','<h3 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">5% Acrylamide; 7.5M Urea</span></h3>\r\n<p style=\"text-align: center;\"><span style=\"color: #ff0000;\">Note: Urea and acrylamide are toxic, so wear gloves and avoid skin contamination</span></p>\r\n1X TBE: 250ml\r\n\r\n40% acrylamide: 62.5 mL (Biorad 40% acrylamide/bis solution 19:1 catalog#1610144)\r\n\r\nUrea: 227.29 g (Sigma U-1250/5kg)\r\n<ul>\r\n	<li>Mix acrylamide and TBE</li>\r\n	<li>Add urea and stir until it dissolves (at least 3 hours).</li>\r\n	<li>Note: Do not heat the solution to speed up the dissolution of urea since this will produce uric acid due to the breakdown of urea.</li>\r\n	<li>Bring to 500ml volume in graduated cylinder</li>\r\n	<li>Filter with 0.45um filter into 1L glass bottle</li>\r\n	<li>Transfer solution from glass bottles to brown plastic bottles</li>\r\n	<li>Store at 4C in the dark.</li>\r\n</ul>','Acrylamide (Manual Gel)','','inherit','closed','open','','363-revision-v1','','','2014-03-10 17:55:16','2014-03-10 17:55:16','',363,'http://localhost/~psalessi/Genetics_Lab/363-revision-v1/',0,'revision','',0),(374,1,'2014-03-10 18:06:56','2014-03-10 18:06:56','<p style=\"text-align: center;\"><strong><span style=\"text-decoration: underline;\">INFORMATION FOR ORDERING AND RECEIVING SUPPLIES</span></strong></p>\r\n<span style=\"text-decoration: underline;\"><span style=\"line-height: 1.714285714; font-size: 1rem;\">Ordering Supplies</span></span>\r\n<ol>\r\n	<li>Recent orders can be found in the purchasing folders on the shelf in the main lab. The forms are kept alphabetically by manufacturer/company they were ordered from. Refer to the previous order forms for any special price quotes.\r\n<ul>\r\n	<li>We can purchase many of our office supplies and general lab supplies on campus from <strong>Vet Med Central Services</strong> or the <strong>Storehouse</strong>. Vet Med can be reached at 2-0157 and is located in Haring Hall. Directions to Vet Med are on genome-lab website under protocols. The Storehouse catalog can be found on the UC Davis website (<a title=\"UC Davis Homepage\" href=\"http://www.ucdavis.edu\">www.ucdavis.edu</a>) by searching under storehouse.</li>\r\n</ul>\r\n</li>\r\n	<li>Fill out a Purchase Order Request for the items that you would like to order.\r\n<ul>\r\n	<li>Completing the order (On-line or calling in yourself): Take the completed Purchase Order form to Norma Andrews in the Business Office. She will keep one copy and return the other copy to you with the PO number. Use this number when filling out the on-line billing information or when phoning in the order.</li>\r\n	<li>Or, take the completed Purchase Order form to Norma Andrews in the Business Office. She will phone in the order for you and return a copy of the Purchase Order to the lab.</li>\r\n	<li>NOTE: If several items are being ordered from the same company, place all items on one Purchase Order. Check with other lab members before you complete the order to see if others need items from that company before you complete the order.</li>\r\n</ul>\r\n</li>\r\n	<li>Keep track of all orders purchased on each separate account number in an Excel database. Currently, these account databases are in <strong>Lab 20/Lab Documents/John/Lab Accounts/Current</strong>\r\n<ul>\r\n	<li><strong>Example:</strong>\r\n<p style=\"text-align: center;\">Account 627Y Do Not Exceed $5000 1/24/06</p>\r\n<p style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Item</span> <span style=\"text-decoration: underline;\">Catalog</span> <span style=\"text-decoration: underline;\">Number</span> <span style=\"text-decoration: underline;\">Quantity</span> <span style=\"text-decoration: underline;\">Company</span> <span style=\"text-decoration: underline;\">Phone</span> <span style=\"text-decoration: underline;\">Number</span> <span style=\"text-decoration: underline;\">$</span> <span style=\"text-decoration: underline;\">Amount</span> <span style=\"text-decoration: underline;\">Contact</span> <span style=\"text-decoration: underline;\">Person</span> <span style=\"text-decoration: underline;\">Date</span> <span style=\"text-decoration: underline;\">Ordered</span></p>\r\n</li>\r\n</ul>\r\n</li>\r\n	<li>File the Purchase Order copy and place in Orders Not Yet Received folder in the Main Lab.</li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Receiving Supplies</span>\r\n<ol>\r\n	<li>If box is labeled <strong>Freeze Upon Arrival</strong> or <strong>Refrigerate Upon Arrival</strong>, make sure to unpack it immediately.</li>\r\n	<li>Label packaging slip with your initials (make sure that they are legible), the date, and the word received. Place the packaging slip in the slot labeled on the main shelves above the phone with the red label.</li>\r\n	<li>Remove the Purchase Order Request from the Orders Not Yet Received folder in the Main Lab and file it under the Company Name in the Purchasing notebooks (red notebooks) above the phone in the Main Lab.</li>\r\n</ol>','Supplies','','publish','closed','open','','supplies','','','2014-03-10 18:40:06','2014-03-10 18:40:06','',217,'http://localhost/~psalessi/Genetics_Lab/?page_id=374',0,'page','',0),(375,1,'2014-03-10 18:06:56','2014-03-10 18:06:56','<p style=\"text-align: center;\"><strong><span style=\"text-decoration: underline;\">INFORMATION FOR ORDERING AND RECEIVING SUPPLIES</span></strong></p>\r\n<span style=\"text-decoration: underline;\"><span style=\"line-height: 1.714285714; font-size: 1rem;\">Ordering Supplies</span></span>\r\n<ol>\r\n	<li>Recent orders can be found in the purchasing folders on the shelf in the main lab. The forms are kept alphabetically by manufacturer/company they were ordered from. Refer to the previous order forms for any special price quotes.\r\n<ul>\r\n	<li>We can purchase many of our office supplies and general lab supplies on campus from <strong>Vet Med Central Services</strong> or the <strong>Storehouse</strong>. Vet Med can be reached at 2-0157 and is located in Haring Hall. Directions to Vet Med are on genome-lab website under protocols. The Storehouse catalog can be found on the UC Davis website (www.ucdavis.edu) by searching under storehouse.</li>\r\n</ul>\r\n</li>\r\n	<li>Fill out a Purchase Order Request for the items that you would like to order.\r\n<ul>\r\n	<li>Completing the order (On-line or calling in yourself): Take the completed Purchase Order form to Norma Andrews in the Business Office. She will keep one copy and return the other copy to you with the PO number. Use this number when filling out the on-line billing information or when phoning in the order.</li>\r\n	<li>Or, take the completed Purchase Order form to Norma Andrews in the Business Office. She will phone in the order for you and return a copy of the Purchase Order to the lab.</li>\r\n	<li>NOTE: If several items are being ordered from the same company, place all items on one Purchase Order. Check with other lab members before you complete the order to see if others need items from that company before you complete the order.</li>\r\n</ul>\r\n</li>\r\n	<li>Keep track of all orders purchased on each separate account number in an Excel database. Currently, these account databases are in <strong>Lab 20/ Lab Documents / John / Lab Accounts / Current.</strong><strong>Example:</strong>\r\n<p style=\"text-align: center;\">Account 627Y Do Not Exceed $5000 1/24/06</p>\r\n<p style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Item</span> <span style=\"text-decoration: underline;\">Catalog</span> <span style=\"text-decoration: underline;\">Number</span> <span style=\"text-decoration: underline;\">Quantity</span> <span style=\"text-decoration: underline;\">Company</span> <span style=\"text-decoration: underline;\">Phone</span> <span style=\"text-decoration: underline;\">Number</span> <span style=\"text-decoration: underline;\">$</span> <span style=\"text-decoration: underline;\">Amount</span> <span style=\"text-decoration: underline;\">Contact</span> <span style=\"text-decoration: underline;\">Person</span> <span style=\"text-decoration: underline;\">Date</span> <span style=\"text-decoration: underline;\">Ordered</span></p>\r\n</li>\r\n	<li>File the Purchase Order copy and place in Orders Not Yet Received folder in the Main Lab.</li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Receiving Supplies</span>\r\n<ol>\r\n	<li>If box is labeled <strong>Freeze Upon Arrival</strong> or <strong>Refrigerate Upon Arrival</strong>, make sure to unpack it immediately.</li>\r\n	<li>Label packaging slip with your initials (make sure that they are legible), the date, and the word received. Place the packaging slip in the slot labeled on the main shelves above the phone with the red label.</li>\r\n	<li>Remove the Purchase Order Request from the Orders Not Yet Received folder in the Main Lab and file it under the Company Name in the Purchasing notebooks (red notebooks) above the phone in the Main Lab.</li>\r\n</ol>','Supplies','','inherit','closed','open','','374-revision-v1','','','2014-03-10 18:06:56','2014-03-10 18:06:56','',374,'http://localhost/~psalessi/Genetics_Lab/374-revision-v1/',0,'revision','',0),(376,1,'2014-03-10 18:40:04','2014-03-10 18:40:04','<p style=\"text-align: center;\"><strong><span style=\"text-decoration: underline;\">INFORMATION FOR ORDERING AND RECEIVING SUPPLIES</span></strong></p>\n<span style=\"text-decoration: underline;\"><span style=\"line-height: 1.714285714; font-size: 1rem;\">Ordering Supplies</span></span>\n<ol>\n	<li>Recent orders can be found in the purchasing folders on the shelf in the main lab. The forms are kept alphabetically by manufacturer/company they were ordered from. Refer to the previous order forms for any special price quotes.\n<ul>\n	<li>We can purchase many of our office supplies and general lab supplies on campus from <strong>Vet Med Central Services</strong> or the <strong>Storehouse</strong>. Vet Med can be reached at 2-0157 and is located in Haring Hall. Directions to Vet Med are on genome-lab website under protocols. The Storehouse catalog can be found on the UC Davis website (<a title=\"UC Davis Homepage\" href=\"http://www.ucdavis.edu\">www.ucdavis.edu</a>) by searching under storehouse.</li>\n</ul>\n</li>\n	<li>Fill out a Purchase Order Request for the items that you would like to order.\n<ul>\n	<li>Completing the order (On-line or calling in yourself): Take the completed Purchase Order form to Norma Andrews in the Business Office. She will keep one copy and return the other copy to you with the PO number. Use this number when filling out the on-line billing information or when phoning in the order.</li>\n	<li>Or, take the completed Purchase Order form to Norma Andrews in the Business Office. She will phone in the order for you and return a copy of the Purchase Order to the lab.</li>\n	<li>NOTE: If several items are being ordered from the same company, place all items on one Purchase Order. Check with other lab members before you complete the order to see if others need items from that company before you complete the order.</li>\n</ul>\n</li>\n	<li>Keep track of all orders purchased on each separate account number in an Excel database. Currently, these account databases are in <strong>Lab 20/Lab Documents/John/Lab Accounts/Current</strong>\n<ul>\n	<li><strong>Example:</strong>\n<p style=\"text-align: center;\">Account 627Y Do Not Exceed $5000 1/24/06</p>\n<p style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Item</span> <span style=\"text-decoration: underline;\">Catalog</span> <span style=\"text-decoration: underline;\">Number</span> <span style=\"text-decoration: underline;\">Quantity</span> <span style=\"text-decoration: underline;\">Company</span> <span style=\"text-decoration: underline;\">Phone</span> <span style=\"text-decoration: underline;\">Number</span> <span style=\"text-decoration: underline;\">$</span> <span style=\"text-decoration: underline;\">Amount</span> <span style=\"text-decoration: underline;\">Contact</span> <span style=\"text-decoration: underline;\">Person</span> <span style=\"text-decoration: underline;\">Date</span> <span style=\"text-decoration: underline;\">Ordered</span></p>\n</li>\n</ul>\n</li>\n	<li>File the Purchase Order copy and place in Orders Not Yet Received folder in the Main Lab.</li>\n</ol>\n<span style=\"text-decoration: underline;\">Receiving Supplies</span>\n<ol>\n	<li>If box is labeled <strong>Freeze Upon Arrival</strong> or <strong>Refrigerate Upon Arrival</strong>, make sure to unpack it immediately.</li>\n	<li>Label packaging slip with your initials (make sure that they are legible), the date, and the word received. Place the packaging slip in the slot labeled on the main shelves above the phone with the red label.</li>\n	<li>Remove the Purchase Order Request from the Orders Not Yet Received folder in the Main Lab and file it under the Company Name in the Purchasing notebooks (red notebooks) above the phone in the Main Lab.</li>\n</ol>','Supplies','','inherit','closed','open','','374-autosave-v1','','','2014-03-10 18:40:04','2014-03-10 18:40:04','',374,'http://localhost/~psalessi/Genetics_Lab/374-autosave-v1/',0,'revision','',0),(377,1,'2014-03-10 18:08:42','2014-03-10 18:08:42','<p style=\"text-align: center;\"><strong><span style=\"text-decoration: underline;\">INFORMATION FOR ORDERING AND RECEIVING SUPPLIES</span></strong></p>\r\n<span style=\"text-decoration: underline;\"><span style=\"line-height: 1.714285714; font-size: 1rem;\">Ordering Supplies</span></span>\r\n<ol>\r\n	<li>Recent orders can be found in the purchasing folders on the shelf in the main lab. The forms are kept alphabetically by manufacturer/company they were ordered from. Refer to the previous order forms for any special price quotes.\r\n<ul>\r\n	<li>We can purchase many of our office supplies and general lab supplies on campus from <strong>Vet Med Central Services</strong> or the <strong>Storehouse</strong>. Vet Med can be reached at 2-0157 and is located in Haring Hall. Directions to Vet Med are on genome-lab website under protocols. The Storehouse catalog can be found on the UC Davis website (www.ucdavis.edu) by searching under storehouse.</li>\r\n</ul>\r\n</li>\r\n	<li>Fill out a Purchase Order Request for the items that you would like to order.\r\n<ul>\r\n	<li>Completing the order (On-line or calling in yourself): Take the completed Purchase Order form to Norma Andrews in the Business Office. She will keep one copy and return the other copy to you with the PO number. Use this number when filling out the on-line billing information or when phoning in the order.</li>\r\n	<li>Or, take the completed Purchase Order form to Norma Andrews in the Business Office. She will phone in the order for you and return a copy of the Purchase Order to the lab.</li>\r\n	<li>NOTE: If several items are being ordered from the same company, place all items on one Purchase Order. Check with other lab members before you complete the order to see if others need items from that company before you complete the order.</li>\r\n</ul>\r\n</li>\r\n	<li>Keep track of all orders purchased on each separate account number in an Excel database. Currently, these account databases are in <strong>Lab 20/ Lab Documents / John / Lab Accounts / Current.</strong>\r\n&amp;nbsp\r\n<strong>Example:</strong>\r\n<p style=\"text-align: center;\">Account 627Y Do Not Exceed $5000 1/24/06</p>\r\n<p style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Item</span> <span style=\"text-decoration: underline;\">Catalog</span> <span style=\"text-decoration: underline;\">Number</span> <span style=\"text-decoration: underline;\">Quantity</span> <span style=\"text-decoration: underline;\">Company</span> <span style=\"text-decoration: underline;\">Phone</span> <span style=\"text-decoration: underline;\">Number</span> <span style=\"text-decoration: underline;\">$</span> <span style=\"text-decoration: underline;\">Amount</span> <span style=\"text-decoration: underline;\">Contact</span> <span style=\"text-decoration: underline;\">Person</span> <span style=\"text-decoration: underline;\">Date</span> <span style=\"text-decoration: underline;\">Ordered</span></p>\r\n</li>\r\n	<li>File the Purchase Order copy and place in Orders Not Yet Received folder in the Main Lab.</li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Receiving Supplies</span>\r\n<ol>\r\n	<li>If box is labeled <strong>Freeze Upon Arrival</strong> or <strong>Refrigerate Upon Arrival</strong>, make sure to unpack it immediately.</li>\r\n	<li>Label packaging slip with your initials (make sure that they are legible), the date, and the word received. Place the packaging slip in the slot labeled on the main shelves above the phone with the red label.</li>\r\n	<li>Remove the Purchase Order Request from the Orders Not Yet Received folder in the Main Lab and file it under the Company Name in the Purchasing notebooks (red notebooks) above the phone in the Main Lab.</li>\r\n</ol>','Supplies','','inherit','closed','open','','374-revision-v1','','','2014-03-10 18:08:42','2014-03-10 18:08:42','',374,'http://localhost/~psalessi/Genetics_Lab/374-revision-v1/',0,'revision','',0),(378,1,'2014-03-10 18:09:14','2014-03-10 18:09:14','<p style=\"text-align: center;\"><strong><span style=\"text-decoration: underline;\">INFORMATION FOR ORDERING AND RECEIVING SUPPLIES</span></strong></p>\r\n<span style=\"text-decoration: underline;\"><span style=\"line-height: 1.714285714; font-size: 1rem;\">Ordering Supplies</span></span>\r\n<ol>\r\n	<li>Recent orders can be found in the purchasing folders on the shelf in the main lab. The forms are kept alphabetically by manufacturer/company they were ordered from. Refer to the previous order forms for any special price quotes.\r\n<ul>\r\n	<li>We can purchase many of our office supplies and general lab supplies on campus from <strong>Vet Med Central Services</strong> or the <strong>Storehouse</strong>. Vet Med can be reached at 2-0157 and is located in Haring Hall. Directions to Vet Med are on genome-lab website under protocols. The Storehouse catalog can be found on the UC Davis website (www.ucdavis.edu) by searching under storehouse.</li>\r\n</ul>\r\n</li>\r\n	<li>Fill out a Purchase Order Request for the items that you would like to order.\r\n<ul>\r\n	<li>Completing the order (On-line or calling in yourself): Take the completed Purchase Order form to Norma Andrews in the Business Office. She will keep one copy and return the other copy to you with the PO number. Use this number when filling out the on-line billing information or when phoning in the order.</li>\r\n	<li>Or, take the completed Purchase Order form to Norma Andrews in the Business Office. She will phone in the order for you and return a copy of the Purchase Order to the lab.</li>\r\n	<li>NOTE: If several items are being ordered from the same company, place all items on one Purchase Order. Check with other lab members before you complete the order to see if others need items from that company before you complete the order.</li>\r\n</ul>\r\n</li>\r\n	<li>Keep track of all orders purchased on each separate account number in an Excel database. Currently, these account databases are in <strong>Lab 20/ Lab Documents / John / Lab Accounts / Current.</strong>\r\n&nbsp\r\n<strong>Example:</strong>\r\n<p style=\"text-align: center;\">Account 627Y Do Not Exceed $5000 1/24/06</p>\r\n<p style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Item</span> <span style=\"text-decoration: underline;\">Catalog</span> <span style=\"text-decoration: underline;\">Number</span> <span style=\"text-decoration: underline;\">Quantity</span> <span style=\"text-decoration: underline;\">Company</span> <span style=\"text-decoration: underline;\">Phone</span> <span style=\"text-decoration: underline;\">Number</span> <span style=\"text-decoration: underline;\">$</span> <span style=\"text-decoration: underline;\">Amount</span> <span style=\"text-decoration: underline;\">Contact</span> <span style=\"text-decoration: underline;\">Person</span> <span style=\"text-decoration: underline;\">Date</span> <span style=\"text-decoration: underline;\">Ordered</span></p>\r\n</li>\r\n	<li>File the Purchase Order copy and place in Orders Not Yet Received folder in the Main Lab.</li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Receiving Supplies</span>\r\n<ol>\r\n	<li>If box is labeled <strong>Freeze Upon Arrival</strong> or <strong>Refrigerate Upon Arrival</strong>, make sure to unpack it immediately.</li>\r\n	<li>Label packaging slip with your initials (make sure that they are legible), the date, and the word received. Place the packaging slip in the slot labeled on the main shelves above the phone with the red label.</li>\r\n	<li>Remove the Purchase Order Request from the Orders Not Yet Received folder in the Main Lab and file it under the Company Name in the Purchasing notebooks (red notebooks) above the phone in the Main Lab.</li>\r\n</ol>','Supplies','','inherit','closed','open','','374-revision-v1','','','2014-03-10 18:09:14','2014-03-10 18:09:14','',374,'http://localhost/~psalessi/Genetics_Lab/374-revision-v1/',0,'revision','',0),(379,1,'2014-03-10 18:09:30','2014-03-10 18:09:30','<p style=\"text-align: center;\"><strong><span style=\"text-decoration: underline;\">INFORMATION FOR ORDERING AND RECEIVING SUPPLIES</span></strong></p>\r\n<span style=\"text-decoration: underline;\"><span style=\"line-height: 1.714285714; font-size: 1rem;\">Ordering Supplies</span></span>\r\n<ol>\r\n	<li>Recent orders can be found in the purchasing folders on the shelf in the main lab. The forms are kept alphabetically by manufacturer/company they were ordered from. Refer to the previous order forms for any special price quotes.\r\n<ul>\r\n	<li>We can purchase many of our office supplies and general lab supplies on campus from <strong>Vet Med Central Services</strong> or the <strong>Storehouse</strong>. Vet Med can be reached at 2-0157 and is located in Haring Hall. Directions to Vet Med are on genome-lab website under protocols. The Storehouse catalog can be found on the UC Davis website (www.ucdavis.edu) by searching under storehouse.</li>\r\n</ul>\r\n</li>\r\n	<li>Fill out a Purchase Order Request for the items that you would like to order.\r\n<ul>\r\n	<li>Completing the order (On-line or calling in yourself): Take the completed Purchase Order form to Norma Andrews in the Business Office. She will keep one copy and return the other copy to you with the PO number. Use this number when filling out the on-line billing information or when phoning in the order.</li>\r\n	<li>Or, take the completed Purchase Order form to Norma Andrews in the Business Office. She will phone in the order for you and return a copy of the Purchase Order to the lab.</li>\r\n	<li>NOTE: If several items are being ordered from the same company, place all items on one Purchase Order. Check with other lab members before you complete the order to see if others need items from that company before you complete the order.</li>\r\n</ul>\r\n</li>\r\n	<li>Keep track of all orders purchased on each separate account number in an Excel database. Currently, these account databases are in <strong>Lab 20/ Lab Documents / John / Lab Accounts / Current.</strong>\r\n\r\n<strong>Example:</strong>\r\n<p style=\"text-align: center;\">Account 627Y Do Not Exceed $5000 1/24/06</p>\r\n<p style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Item</span> <span style=\"text-decoration: underline;\">Catalog</span> <span style=\"text-decoration: underline;\">Number</span> <span style=\"text-decoration: underline;\">Quantity</span> <span style=\"text-decoration: underline;\">Company</span> <span style=\"text-decoration: underline;\">Phone</span> <span style=\"text-decoration: underline;\">Number</span> <span style=\"text-decoration: underline;\">$</span> <span style=\"text-decoration: underline;\">Amount</span> <span style=\"text-decoration: underline;\">Contact</span> <span style=\"text-decoration: underline;\">Person</span> <span style=\"text-decoration: underline;\">Date</span> <span style=\"text-decoration: underline;\">Ordered</span></p>\r\n</li>\r\n	<li>File the Purchase Order copy and place in Orders Not Yet Received folder in the Main Lab.</li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Receiving Supplies</span>\r\n<ol>\r\n	<li>If box is labeled <strong>Freeze Upon Arrival</strong> or <strong>Refrigerate Upon Arrival</strong>, make sure to unpack it immediately.</li>\r\n	<li>Label packaging slip with your initials (make sure that they are legible), the date, and the word received. Place the packaging slip in the slot labeled on the main shelves above the phone with the red label.</li>\r\n	<li>Remove the Purchase Order Request from the Orders Not Yet Received folder in the Main Lab and file it under the Company Name in the Purchasing notebooks (red notebooks) above the phone in the Main Lab.</li>\r\n</ol>','Supplies','','inherit','closed','open','','374-revision-v1','','','2014-03-10 18:09:30','2014-03-10 18:09:30','',374,'http://localhost/~psalessi/Genetics_Lab/374-revision-v1/',0,'revision','',0),(380,1,'2014-03-10 18:10:37','2014-03-10 18:10:37','<p style=\"text-align: center;\"><strong><span style=\"text-decoration: underline;\">INFORMATION FOR ORDERING AND RECEIVING SUPPLIES</span></strong></p>\r\n<span style=\"text-decoration: underline;\"><span style=\"line-height: 1.714285714; font-size: 1rem;\">Ordering Supplies</span></span>\r\n<ol>\r\n	<li>Recent orders can be found in the purchasing folders on the shelf in the main lab. The forms are kept alphabetically by manufacturer/company they were ordered from. Refer to the previous order forms for any special price quotes.\r\n<ul>\r\n	<li>We can purchase many of our office supplies and general lab supplies on campus from <strong>Vet Med Central Services</strong> or the <strong>Storehouse</strong>. Vet Med can be reached at 2-0157 and is located in Haring Hall. Directions to Vet Med are on genome-lab website under protocols. The Storehouse catalog can be found on the UC Davis website (<a title=\"UC Davis Homepage\" href=\"http://www.ucdavis.edu\">www.ucdavis.edu</a>) by searching under storehouse.</li>\r\n</ul>\r\n</li>\r\n	<li>Fill out a Purchase Order Request for the items that you would like to order.\r\n<ul>\r\n	<li>Completing the order (On-line or calling in yourself): Take the completed Purchase Order form to Norma Andrews in the Business Office. She will keep one copy and return the other copy to you with the PO number. Use this number when filling out the on-line billing information or when phoning in the order.</li>\r\n	<li>Or, take the completed Purchase Order form to Norma Andrews in the Business Office. She will phone in the order for you and return a copy of the Purchase Order to the lab.</li>\r\n	<li>NOTE: If several items are being ordered from the same company, place all items on one Purchase Order. Check with other lab members before you complete the order to see if others need items from that company before you complete the order.</li>\r\n</ul>\r\n</li>\r\n	<li>Keep track of all orders purchased on each separate account number in an Excel database. Currently, these account databases are in <strong>Lab 20/ Lab Documents / John / Lab Accounts / Current.</strong><strong>Example:</strong>\r\n<p style=\"text-align: center;\">Account 627Y Do Not Exceed $5000 1/24/06</p>\r\n<p style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Item</span> <span style=\"text-decoration: underline;\">Catalog</span> <span style=\"text-decoration: underline;\">Number</span> <span style=\"text-decoration: underline;\">Quantity</span> <span style=\"text-decoration: underline;\">Company</span> <span style=\"text-decoration: underline;\">Phone</span> <span style=\"text-decoration: underline;\">Number</span> <span style=\"text-decoration: underline;\">$</span> <span style=\"text-decoration: underline;\">Amount</span> <span style=\"text-decoration: underline;\">Contact</span> <span style=\"text-decoration: underline;\">Person</span> <span style=\"text-decoration: underline;\">Date</span> <span style=\"text-decoration: underline;\">Ordered</span></p>\r\n</li>\r\n	<li>File the Purchase Order copy and place in Orders Not Yet Received folder in the Main Lab.</li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Receiving Supplies</span>\r\n<ol>\r\n	<li>If box is labeled <strong>Freeze Upon Arrival</strong> or <strong>Refrigerate Upon Arrival</strong>, make sure to unpack it immediately.</li>\r\n	<li>Label packaging slip with your initials (make sure that they are legible), the date, and the word received. Place the packaging slip in the slot labeled on the main shelves above the phone with the red label.</li>\r\n	<li>Remove the Purchase Order Request from the Orders Not Yet Received folder in the Main Lab and file it under the Company Name in the Purchasing notebooks (red notebooks) above the phone in the Main Lab.</li>\r\n</ol>','Supplies','','inherit','closed','open','','374-revision-v1','','','2014-03-10 18:10:37','2014-03-10 18:10:37','',374,'http://localhost/~psalessi/Genetics_Lab/374-revision-v1/',0,'revision','',0),(381,1,'2014-03-10 18:15:28','2014-03-10 18:15:28','<strong>Buffers &amp; Stocks</strong>\r\n<ul>\r\n	<li><a title=\"98% Formamide Loading Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/98-formamide-loading-dye/\">Formamide Loading Dye</a></li>\r\n	<li><a title=\"Diluting 1Kb Plus Ladder with Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/diluting-1kb-plus-ladder-with-dye/\">Diluting 1kb Ladder/Dye</a></li>\r\n	<li><a title=\"Gelco Fluorescein Ladder\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/gelco-fluorescein-ladder/\">Gelco Fluorescein Ladder</a></li>\r\n	<li><a title=\"Proteinase K\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/proteinase-k/\">Proteinase K</a></li>\r\n	<li><a title=\"Glassware Cleaner\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/glassware-cleaner/\">Glassware Cleaner</a></li>\r\n	<li><a title=\"dNTPs (Promega)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/dntps-promega/\">dNTPs (Promega)</a></li>\r\n	<li><a title=\"1M Tris-Cl\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1m-tris-cl/\">1M Tris-Cl</a></li>\r\n	<li><a title=\".5M EDTA (pH 8.0)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/5m-edta-ph-8-0/\">0.5M EDTA (pH 8.0)</a></li>\r\n	<li><a title=\"1X TE (10mM Tris; 1mM EDTA)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1x-te-10mm-tris-1mm-edta/\">1X TE (10mM Tris; 1mM EDTA)</a></li>\r\n	<li><a title=\"1X TLE (10mM Tris; 0.1mM EDTA)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1x-tle-10mm-tris-0-1mm-edta/\">1X TLE (10mM Tris; 0.1 mM EDTA)</a></li>\r\n	<li><a title=\"10X TNE\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/10x-tne/\">10X TNE</a></li>\r\n	<li><a title=\"10X TBE One-Shots\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/10x-tbe-one-shots/\">10X TBE One-Shots</a></li>\r\n</ul>\r\n<strong>FujiFilm</strong>\r\n<ul>\r\n	<li><a title=\"FujiFilm DNA Quantification\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/fujifilm-dna-quantification/\">DNA quantification</a></li>\r\n</ul>\r\n<strong>ABI 3730xl Maintenance</strong>\r\n<ul>\r\n	<li><a title=\"3730xl General Maintenance\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/3730xl-general-maintenance/\">3730xl Maintenance</a></li>\r\n	<li><a title=\"Proteinase K Wash\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/proteinase-k-wash/\">Proteinase K Wash</a></li>\r\n</ul>\r\n<strong>Acrylamide gels</strong>\r\n<ul>\r\n	<li><a title=\"Acrylamide (Manual Gel)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/acrylamide-manual-gel/\">Acrylamide Gel</a></li>\r\n	<li><a title=\"Acrylamide Gel Pour\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/acrylamide-gel-pour/\">Pouring</a></li>\r\n	<li><a title=\"Running Acrylamide\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/running-acrylamide/\">Running</a></li>\r\n</ul>\r\n<strong>Fluidigm</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/FLUIDIGM EP1 PROTOCOL.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/FLUIDIGM%20EP1%20PROTOCOL.doc\">Fluidigm EP1 Protocol</a></li>\r\n</ul>\r\n<strong>Illumina</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/RAD-SEQ Protocol.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/RAD-SEQ%20Protocol.doc\">RAD-SEQ Protocol</a></li>\r\n</ul>\r\n<strong>Other</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/tissuesampleprot.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/tissuesampleprot.pdf\">Tissue Sampling</a></li>\r\n	<li><a title=\"Supplies\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/supplies/\">Ordering and Receiving Supplies</a></li>\r\n	<li><a title=\"Supplies.html\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Supplies.html\">PCR Tips and Tricks</a></li>\r\n</ul>','Protocols and Procedures','','inherit','closed','open','','217-revision-v1','','','2014-03-10 18:15:28','2014-03-10 18:15:28','',217,'http://localhost/~psalessi/Genetics_Lab/217-revision-v1/',0,'revision','',0),(383,1,'2014-03-10 18:33:51','2014-03-10 18:33:51','<h2 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">General Guidelines</span></h2>\r\n<ul>\r\n	<li>Reagent concentrations - \"less is usually better\" (more specific)\r\n<ul>\r\n	<li>primers: final concentration 0.1-1.0 uM</li>\r\n	<li>MgCl<sub>2</sub>: final concentration 1.0-4.0 mM (depends on Taq used)</li>\r\n	<li>dNTPs: final concentration 0.2 mM each dNTP (depends on Taq used)\r\n<ul>\r\n	<li>note: sensitive to repeated freeze/thaws</li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</li>\r\n	<li>Vortex or finger-flick reagents to mix well before use</li>\r\n	<li>Primer design very important\r\n<ul>\r\n	<li>the higher the annealing temperature the better (TANN &gt; 50C)</li>\r\n	<li>primer pairs should have melting temps within 5C of each other</li>\r\n</ul>\r\n</li>\r\n	<li>Annealing temperature and step times are important</li>\r\n	<li>Titrations are a good idea\r\n<ul>\r\n	<li>most commonly temperature and MgCl<sub>2</sub></li>\r\n	<li>reactions often borderline resulting in inconsistent amplifications</li>\r\n	<li>use Eppendorf Mastercycler Gradient for titrations</li>\r\n	<li><a title=\"Titration on Mastercycler Gradient\" href=\"http://genome-lab.ucdavis.edu/Protocols/pcr_tips/titration_mastercycler.htm\" target=\"_blank\">titration on Mastercycler Gradient</a></li>\r\n	<li><a title=\"Sample Titration Gel\" href=\"http://genome-lab.ucdavis.edu/Protocols/pcr_tips/sample_titration_gel.htm\" target=\"_blank\">sample titration gel</a></li>\r\n</ul>\r\n</li>\r\n	<li>Hot starts improve reaction efficiency (fewer primer-dimers)\r\n<ul>\r\n	<li>manual: add Taq to tubes in thermalcycler at 94C (or MgCl<sub>2</sub> or dNTPs)</li>\r\n	<li>TaqStart Antibody (Clontech)</li>\r\n	<li>Faststart Taq (Roche)</li>\r\n	<li><a title=\"Hotstart Comparison Gel\" href=\"http://genome-lab.ucdavis.edu/Protocols/pcr_tips/hotstart_comparison.htm\" target=\"_blank\">hotstart comparison gel</a></li>\r\n</ul>\r\n</li>\r\n	<li>Always check program on thermalcycler</li>\r\n	<li>Run negative control(s) to check for contamination</li>\r\n	<li>Make a flow chart of what tried and in what order</li>\r\n	<li>Run a positive control (a sample known to amplify well)</li>\r\n	<li>Always run a ladder on gel (will indicate whether failed PCR or failed detection system)</li>\r\n	<li>Additives for fragments that are very long, G-C rich or prone to secondary structure\r\n<ul>\r\n	<li>glycerol, formamide, NMP - lower denaturing and annealing temperatures by a few degrees</li>\r\n	<li>DMSO decreases incidence of secondary structure</li>\r\n</ul>\r\n</li>\r\n	<li>For more tips see: <a title=\"Biowire Nucleus\" href=\"http://www.biowire.com/nucleus/nucleus_1_1.jsp\">http://www.biowire.com/nucleus/nucleus_1_1.jsp</a></li>\r\n</ul>\r\n<h2 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Troubleshooting</span></h2>\r\n<ul>\r\n	<li>Weak or no amplification:\r\n<ul>\r\n	<li>increase # cycles</li>\r\n	<li>increase time at step(s)</li>\r\n	<li>add final extension of 10 min</li>\r\n	<li>increase quantity of template</li>\r\n	<li>increase Taq concentration by 2X</li>\r\n	<li>try different Taq (e.g. Faststart)</li>\r\n	<li>increase MgCl<sub>2</sub>, primers, dNTPs</li>\r\n	<li>reamplify PCR product from previous reaction\r\n<ul>\r\n	<li><a title=\"PCR Reamplification Protocol\" href=\"http://genome-lab.ucdavis.edu/Protocols/pcr_tips/pcr_reamplification.htm\" target=\"_blank\">PCR reamplification protocol</a></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</li>\r\n	<li>Non-targeted bands:\r\n<ul>\r\n	<li>high weight bands: decrease Taq concentration</li>\r\n	<li>low weight bands: decrease MgCl<sub>2</sub> concentration</li>\r\n	<li>decrease # cycles</li>\r\n	<li>raise annealing temperature</li>\r\n	<li>decrease primer concentration</li>\r\n	<li>decrease dNTPs</li>\r\n</ul>\r\n</li>\r\n	<li>Possible inhibition\r\n<ul>\r\n	<li>dilute template 5:1 with 10mM Tris-HCl (pH 8.0) or ddH<sub>2</sub>O</li>\r\n	<li>add BSA - final concentration 1X</li>\r\n	<li>see <a title=\"Biowire Nucleus\" href=\"http://www.biowire.com/nucleus/nucleus_1_1.jsp\" target=\"_blank\">http://www.biowire.com/nucleus/nucleus_1_1.jsp</a></li>\r\n</ul>\r\n</li>\r\n	<li>+A problem\r\n<ul>\r\n	<li>add final extension of 45 min at 60C</li>\r\n</ul>\r\n</li>\r\n	<li>Smears above band\r\n<ul>\r\n	<li>decrease # cycles</li>\r\n</ul>\r\n</li>\r\n	<li>Carpet\r\n<ul>\r\n	<li>decrease concentration of reagent(s)</li>\r\n</ul>\r\n</li>\r\n</ul>','PCR Tips and Tricks','','publish','closed','open','','pcr-tips-and-tricks','','','2014-03-10 18:33:51','2014-03-10 18:33:51','',217,'http://localhost/~psalessi/Genetics_Lab/?page_id=383',0,'page','',0),(384,1,'2014-03-10 18:33:51','2014-03-10 18:33:51','<h2 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">General Guidelines</span></h2>\r\n<ul>\r\n	<li>Reagent concentrations - \"less is usually better\" (more specific)\r\n<ul>\r\n	<li>primers: final concentration 0.1-1.0 uM</li>\r\n	<li>MgCl<sub>2</sub>: final concentration 1.0-4.0 mM (depends on Taq used)</li>\r\n	<li>dNTPs: final concentration 0.2 mM each dNTP (depends on Taq used)\r\n<ul>\r\n	<li>note: sensitive to repeated freeze/thaws</li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</li>\r\n	<li>Vortex or finger-flick reagents to mix well before use</li>\r\n	<li>Primer design very important\r\n<ul>\r\n	<li>the higher the annealing temperature the better (TANN &gt; 50C)</li>\r\n	<li>primer pairs should have melting temps within 5C of each other</li>\r\n</ul>\r\n</li>\r\n	<li>Annealing temperature and step times are important</li>\r\n	<li>Titrations are a good idea\r\n<ul>\r\n	<li>most commonly temperature and MgCl<sub>2</sub></li>\r\n	<li>reactions often borderline resulting in inconsistent amplifications</li>\r\n	<li>use Eppendorf Mastercycler Gradient for titrations</li>\r\n	<li><a title=\"Titration on Mastercycler Gradient\" href=\"http://genome-lab.ucdavis.edu/Protocols/pcr_tips/titration_mastercycler.htm\" target=\"_blank\">titration on Mastercycler Gradient</a></li>\r\n	<li><a title=\"Sample Titration Gel\" href=\"http://genome-lab.ucdavis.edu/Protocols/pcr_tips/sample_titration_gel.htm\" target=\"_blank\">sample titration gel</a></li>\r\n</ul>\r\n</li>\r\n	<li>Hot starts improve reaction efficiency (fewer primer-dimers)\r\n<ul>\r\n	<li>manual: add Taq to tubes in thermalcycler at 94C (or MgCl<sub>2</sub> or dNTPs)</li>\r\n	<li>TaqStart Antibody (Clontech)</li>\r\n	<li>Faststart Taq (Roche)</li>\r\n	<li><a title=\"Hotstart Comparison Gel\" href=\"http://genome-lab.ucdavis.edu/Protocols/pcr_tips/hotstart_comparison.htm\" target=\"_blank\">hotstart comparison gel</a></li>\r\n</ul>\r\n</li>\r\n	<li>Always check program on thermalcycler</li>\r\n	<li>Run negative control(s) to check for contamination</li>\r\n	<li>Make a flow chart of what tried and in what order</li>\r\n	<li>Run a positive control (a sample known to amplify well)</li>\r\n	<li>Always run a ladder on gel (will indicate whether failed PCR or failed detection system)</li>\r\n	<li>Additives for fragments that are very long, G-C rich or prone to secondary structure\r\n<ul>\r\n	<li>glycerol, formamide, NMP - lower denaturing and annealing temperatures by a few degrees</li>\r\n	<li>DMSO decreases incidence of secondary structure</li>\r\n</ul>\r\n</li>\r\n	<li>For more tips see: <a title=\"Biowire Nucleus\" href=\"http://www.biowire.com/nucleus/nucleus_1_1.jsp\">http://www.biowire.com/nucleus/nucleus_1_1.jsp</a></li>\r\n</ul>\r\n<h2 style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Troubleshooting</span></h2>\r\n<ul>\r\n	<li>Weak or no amplification:\r\n<ul>\r\n	<li>increase # cycles</li>\r\n	<li>increase time at step(s)</li>\r\n	<li>add final extension of 10 min</li>\r\n	<li>increase quantity of template</li>\r\n	<li>increase Taq concentration by 2X</li>\r\n	<li>try different Taq (e.g. Faststart)</li>\r\n	<li>increase MgCl<sub>2</sub>, primers, dNTPs</li>\r\n	<li>reamplify PCR product from previous reaction\r\n<ul>\r\n	<li><a title=\"PCR Reamplification Protocol\" href=\"http://genome-lab.ucdavis.edu/Protocols/pcr_tips/pcr_reamplification.htm\" target=\"_blank\">PCR reamplification protocol</a></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</li>\r\n	<li>Non-targeted bands:\r\n<ul>\r\n	<li>high weight bands: decrease Taq concentration</li>\r\n	<li>low weight bands: decrease MgCl<sub>2</sub> concentration</li>\r\n	<li>decrease # cycles</li>\r\n	<li>raise annealing temperature</li>\r\n	<li>decrease primer concentration</li>\r\n	<li>decrease dNTPs</li>\r\n</ul>\r\n</li>\r\n	<li>Possible inhibition\r\n<ul>\r\n	<li>dilute template 5:1 with 10mM Tris-HCl (pH 8.0) or ddH<sub>2</sub>O</li>\r\n	<li>add BSA - final concentration 1X</li>\r\n	<li>see <a title=\"Biowire Nucleus\" href=\"http://www.biowire.com/nucleus/nucleus_1_1.jsp\" target=\"_blank\">http://www.biowire.com/nucleus/nucleus_1_1.jsp</a></li>\r\n</ul>\r\n</li>\r\n	<li>+A problem\r\n<ul>\r\n	<li>add final extension of 45 min at 60C</li>\r\n</ul>\r\n</li>\r\n	<li>Smears above band\r\n<ul>\r\n	<li>decrease # cycles</li>\r\n</ul>\r\n</li>\r\n	<li>Carpet\r\n<ul>\r\n	<li>decrease concentration of reagent(s)</li>\r\n</ul>\r\n</li>\r\n</ul>','PCR Tips and Tricks','','inherit','closed','open','','383-revision-v1','','','2014-03-10 18:33:51','2014-03-10 18:33:51','',383,'http://localhost/~psalessi/Genetics_Lab/383-revision-v1/',0,'revision','',0),(385,1,'2014-03-10 18:35:46','2014-03-10 18:35:46','<strong>Buffers &amp; Stocks</strong>\r\n<ul>\r\n	<li><a title=\"98% Formamide Loading Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/98-formamide-loading-dye/\">Formamide Loading Dye</a></li>\r\n	<li><a title=\"Diluting 1Kb Plus Ladder with Dye\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/diluting-1kb-plus-ladder-with-dye/\">Diluting 1kb Ladder/Dye</a></li>\r\n	<li><a title=\"Gelco Fluorescein Ladder\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/gelco-fluorescein-ladder/\">Gelco Fluorescein Ladder</a></li>\r\n	<li><a title=\"Proteinase K\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/proteinase-k/\">Proteinase K</a></li>\r\n	<li><a title=\"Glassware Cleaner\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/glassware-cleaner/\">Glassware Cleaner</a></li>\r\n	<li><a title=\"dNTPs (Promega)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/dntps-promega/\">dNTPs (Promega)</a></li>\r\n	<li><a title=\"1M Tris-Cl\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1m-tris-cl/\">1M Tris-Cl</a></li>\r\n	<li><a title=\".5M EDTA (pH 8.0)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/5m-edta-ph-8-0/\">0.5M EDTA (pH 8.0)</a></li>\r\n	<li><a title=\"1X TE (10mM Tris; 1mM EDTA)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1x-te-10mm-tris-1mm-edta/\">1X TE (10mM Tris; 1mM EDTA)</a></li>\r\n	<li><a title=\"1X TLE (10mM Tris; 0.1mM EDTA)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/1x-tle-10mm-tris-0-1mm-edta/\">1X TLE (10mM Tris; 0.1 mM EDTA)</a></li>\r\n	<li><a title=\"10X TNE\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/10x-tne/\">10X TNE</a></li>\r\n	<li><a title=\"10X TBE One-Shots\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/10x-tbe-one-shots/\">10X TBE One-Shots</a></li>\r\n</ul>\r\n<strong>FujiFilm</strong>\r\n<ul>\r\n	<li><a title=\"FujiFilm DNA Quantification\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/fujifilm-dna-quantification/\">DNA quantification</a></li>\r\n</ul>\r\n<strong>ABI 3730xl Maintenance</strong>\r\n<ul>\r\n	<li><a title=\"3730xl General Maintenance\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/3730xl-general-maintenance/\">3730xl Maintenance</a></li>\r\n	<li><a title=\"Proteinase K Wash\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/proteinase-k-wash/\">Proteinase K Wash</a></li>\r\n</ul>\r\n<strong>Acrylamide gels</strong>\r\n<ul>\r\n	<li><a title=\"Acrylamide (Manual Gel)\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/acrylamide-manual-gel/\">Acrylamide Gel</a></li>\r\n	<li><a title=\"Acrylamide Gel Pour\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/acrylamide-gel-pour/\">Pouring</a></li>\r\n	<li><a title=\"Running Acrylamide\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/running-acrylamide/\">Running</a></li>\r\n</ul>\r\n<strong>Fluidigm</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/FLUIDIGM EP1 PROTOCOL.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/FLUIDIGM%20EP1%20PROTOCOL.doc\">Fluidigm EP1 Protocol</a></li>\r\n</ul>\r\n<strong>Illumina</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/RAD-SEQ Protocol.doc\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/RAD-SEQ%20Protocol.doc\">RAD-SEQ Protocol</a></li>\r\n</ul>\r\n<strong>Other</strong>\r\n<ul>\r\n	<li><a title=\"Protocols_files/tissuesampleprot.pdf\" href=\"http://animalscience.ucdavis.edu/gvl/Conservation_Genetics_Fund/Protocols_files/tissuesampleprot.pdf\">Tissue Sampling</a></li>\r\n	<li><a title=\"Supplies\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/supplies/\">Ordering and Receiving Supplies</a></li>\r\n	<li><a title=\"PCR Tips and Tricks\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/pcr-tips-and-tricks/\">PCR Tips and Tricks</a></li>\r\n</ul>','Protocols and Procedures','','inherit','closed','open','','217-revision-v1','','','2014-03-10 18:35:46','2014-03-10 18:35:46','',217,'http://localhost/~psalessi/Genetics_Lab/217-revision-v1/',0,'revision','',0),(386,1,'2014-03-10 18:37:38','2014-03-10 18:37:38','<p style=\"text-align: center;\"><strong><span style=\"text-decoration: underline;\">INFORMATION FOR ORDERING AND RECEIVING SUPPLIES</span></strong></p>\r\n<span style=\"text-decoration: underline;\"><span style=\"line-height: 1.714285714; font-size: 1rem;\">Ordering Supplies</span></span>\r\n<ol>\r\n	<li>Recent orders can be found in the purchasing folders on the shelf in the main lab. The forms are kept alphabetically by manufacturer/company they were ordered from. Refer to the previous order forms for any special price quotes.\r\n<ul>\r\n	<li>We can purchase many of our office supplies and general lab supplies on campus from <strong>Vet Med Central Services</strong> or the <strong>Storehouse</strong>. Vet Med can be reached at 2-0157 and is located in Haring Hall. Directions to Vet Med are on genome-lab website under protocols. The Storehouse catalog can be found on the UC Davis website (<a title=\"UC Davis Homepage\" href=\"http://www.ucdavis.edu\">www.ucdavis.edu</a>) by searching under storehouse.</li>\r\n</ul>\r\n</li>\r\n	<li>Fill out a Purchase Order Request for the items that you would like to order.\r\n<ul>\r\n	<li>Completing the order (On-line or calling in yourself): Take the completed Purchase Order form to Norma Andrews in the Business Office. She will keep one copy and return the other copy to you with the PO number. Use this number when filling out the on-line billing information or when phoning in the order.</li>\r\n	<li>Or, take the completed Purchase Order form to Norma Andrews in the Business Office. She will phone in the order for you and return a copy of the Purchase Order to the lab.</li>\r\n	<li>NOTE: If several items are being ordered from the same company, place all items on one Purchase Order. Check with other lab members before you complete the order to see if others need items from that company before you complete the order.</li>\r\n</ul>\r\n</li>\r\n	<li>Keep track of all orders purchased on each separate account number in an Excel database. Currently, these account databases are in <strong>Lab 20/ Lab Documents / John / Lab Accounts / Current.\r\n</strong><strong>Example:</strong>\r\n<p style=\"text-align: center;\">Account 627Y Do Not Exceed $5000 1/24/06</p>\r\n<p style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Item</span> <span style=\"text-decoration: underline;\">Catalog</span> <span style=\"text-decoration: underline;\">Number</span> <span style=\"text-decoration: underline;\">Quantity</span> <span style=\"text-decoration: underline;\">Company</span> <span style=\"text-decoration: underline;\">Phone</span> <span style=\"text-decoration: underline;\">Number</span> <span style=\"text-decoration: underline;\">$</span> <span style=\"text-decoration: underline;\">Amount</span> <span style=\"text-decoration: underline;\">Contact</span> <span style=\"text-decoration: underline;\">Person</span> <span style=\"text-decoration: underline;\">Date</span> <span style=\"text-decoration: underline;\">Ordered</span></p>\r\n</li>\r\n	<li>File the Purchase Order copy and place in Orders Not Yet Received folder in the Main Lab.</li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Receiving Supplies</span>\r\n<ol>\r\n	<li>If box is labeled <strong>Freeze Upon Arrival</strong> or <strong>Refrigerate Upon Arrival</strong>, make sure to unpack it immediately.</li>\r\n	<li>Label packaging slip with your initials (make sure that they are legible), the date, and the word received. Place the packaging slip in the slot labeled on the main shelves above the phone with the red label.</li>\r\n	<li>Remove the Purchase Order Request from the Orders Not Yet Received folder in the Main Lab and file it under the Company Name in the Purchasing notebooks (red notebooks) above the phone in the Main Lab.</li>\r\n</ol>','Supplies','','inherit','closed','open','','374-revision-v1','','','2014-03-10 18:37:38','2014-03-10 18:37:38','',374,'http://localhost/~psalessi/Genetics_Lab/374-revision-v1/',0,'revision','',0),(387,1,'2014-03-10 18:39:03','2014-03-10 18:39:03','<p style=\"text-align: center;\"><strong><span style=\"text-decoration: underline;\">INFORMATION FOR ORDERING AND RECEIVING SUPPLIES</span></strong></p>\r\n<span style=\"text-decoration: underline;\"><span style=\"line-height: 1.714285714; font-size: 1rem;\">Ordering Supplies</span></span>\r\n<ol>\r\n	<li>Recent orders can be found in the purchasing folders on the shelf in the main lab. The forms are kept alphabetically by manufacturer/company they were ordered from. Refer to the previous order forms for any special price quotes.\r\n<ul>\r\n	<li>We can purchase many of our office supplies and general lab supplies on campus from <strong>Vet Med Central Services</strong> or the <strong>Storehouse</strong>. Vet Med can be reached at 2-0157 and is located in Haring Hall. Directions to Vet Med are on genome-lab website under protocols. The Storehouse catalog can be found on the UC Davis website (<a title=\"UC Davis Homepage\" href=\"http://www.ucdavis.edu\">www.ucdavis.edu</a>) by searching under storehouse.</li>\r\n</ul>\r\n</li>\r\n	<li>Fill out a Purchase Order Request for the items that you would like to order.\r\n<ul>\r\n	<li>Completing the order (On-line or calling in yourself): Take the completed Purchase Order form to Norma Andrews in the Business Office. She will keep one copy and return the other copy to you with the PO number. Use this number when filling out the on-line billing information or when phoning in the order.</li>\r\n	<li>Or, take the completed Purchase Order form to Norma Andrews in the Business Office. She will phone in the order for you and return a copy of the Purchase Order to the lab.</li>\r\n	<li>NOTE: If several items are being ordered from the same company, place all items on one Purchase Order. Check with other lab members before you complete the order to see if others need items from that company before you complete the order.</li>\r\n</ul>\r\n</li>\r\n	<li>Keep track of all orders purchased on each separate account number in an Excel database. Currently, these account databases are in <strong>Lab 20/Lab Documents/John/Lab Accounts/Current</strong>\r\n\r\n<strong>Example:</strong>\r\n<p style=\"text-align: center;\">Account 627Y Do Not Exceed $5000 1/24/06</p>\r\n<p style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Item</span> <span style=\"text-decoration: underline;\">Catalog</span> <span style=\"text-decoration: underline;\">Number</span> <span style=\"text-decoration: underline;\">Quantity</span> <span style=\"text-decoration: underline;\">Company</span> <span style=\"text-decoration: underline;\">Phone</span> <span style=\"text-decoration: underline;\">Number</span> <span style=\"text-decoration: underline;\">$</span> <span style=\"text-decoration: underline;\">Amount</span> <span style=\"text-decoration: underline;\">Contact</span> <span style=\"text-decoration: underline;\">Person</span> <span style=\"text-decoration: underline;\">Date</span> <span style=\"text-decoration: underline;\">Ordered</span></p>\r\n</li>\r\n	<li>File the Purchase Order copy and place in Orders Not Yet Received folder in the Main Lab.</li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Receiving Supplies</span>\r\n<ol>\r\n	<li>If box is labeled <strong>Freeze Upon Arrival</strong> or <strong>Refrigerate Upon Arrival</strong>, make sure to unpack it immediately.</li>\r\n	<li>Label packaging slip with your initials (make sure that they are legible), the date, and the word received. Place the packaging slip in the slot labeled on the main shelves above the phone with the red label.</li>\r\n	<li>Remove the Purchase Order Request from the Orders Not Yet Received folder in the Main Lab and file it under the Company Name in the Purchasing notebooks (red notebooks) above the phone in the Main Lab.</li>\r\n</ol>','Supplies','','inherit','closed','open','','374-revision-v1','','','2014-03-10 18:39:03','2014-03-10 18:39:03','',374,'http://localhost/~psalessi/Genetics_Lab/374-revision-v1/',0,'revision','',0),(388,1,'2014-03-10 18:40:06','2014-03-10 18:40:06','<p style=\"text-align: center;\"><strong><span style=\"text-decoration: underline;\">INFORMATION FOR ORDERING AND RECEIVING SUPPLIES</span></strong></p>\r\n<span style=\"text-decoration: underline;\"><span style=\"line-height: 1.714285714; font-size: 1rem;\">Ordering Supplies</span></span>\r\n<ol>\r\n	<li>Recent orders can be found in the purchasing folders on the shelf in the main lab. The forms are kept alphabetically by manufacturer/company they were ordered from. Refer to the previous order forms for any special price quotes.\r\n<ul>\r\n	<li>We can purchase many of our office supplies and general lab supplies on campus from <strong>Vet Med Central Services</strong> or the <strong>Storehouse</strong>. Vet Med can be reached at 2-0157 and is located in Haring Hall. Directions to Vet Med are on genome-lab website under protocols. The Storehouse catalog can be found on the UC Davis website (<a title=\"UC Davis Homepage\" href=\"http://www.ucdavis.edu\">www.ucdavis.edu</a>) by searching under storehouse.</li>\r\n</ul>\r\n</li>\r\n	<li>Fill out a Purchase Order Request for the items that you would like to order.\r\n<ul>\r\n	<li>Completing the order (On-line or calling in yourself): Take the completed Purchase Order form to Norma Andrews in the Business Office. She will keep one copy and return the other copy to you with the PO number. Use this number when filling out the on-line billing information or when phoning in the order.</li>\r\n	<li>Or, take the completed Purchase Order form to Norma Andrews in the Business Office. She will phone in the order for you and return a copy of the Purchase Order to the lab.</li>\r\n	<li>NOTE: If several items are being ordered from the same company, place all items on one Purchase Order. Check with other lab members before you complete the order to see if others need items from that company before you complete the order.</li>\r\n</ul>\r\n</li>\r\n	<li>Keep track of all orders purchased on each separate account number in an Excel database. Currently, these account databases are in <strong>Lab 20/Lab Documents/John/Lab Accounts/Current</strong>\r\n<ul>\r\n	<li><strong>Example:</strong>\r\n<p style=\"text-align: center;\">Account 627Y Do Not Exceed $5000 1/24/06</p>\r\n<p style=\"text-align: center;\"><span style=\"text-decoration: underline;\">Item</span> <span style=\"text-decoration: underline;\">Catalog</span> <span style=\"text-decoration: underline;\">Number</span> <span style=\"text-decoration: underline;\">Quantity</span> <span style=\"text-decoration: underline;\">Company</span> <span style=\"text-decoration: underline;\">Phone</span> <span style=\"text-decoration: underline;\">Number</span> <span style=\"text-decoration: underline;\">$</span> <span style=\"text-decoration: underline;\">Amount</span> <span style=\"text-decoration: underline;\">Contact</span> <span style=\"text-decoration: underline;\">Person</span> <span style=\"text-decoration: underline;\">Date</span> <span style=\"text-decoration: underline;\">Ordered</span></p>\r\n</li>\r\n</ul>\r\n</li>\r\n	<li>File the Purchase Order copy and place in Orders Not Yet Received folder in the Main Lab.</li>\r\n</ol>\r\n<span style=\"text-decoration: underline;\">Receiving Supplies</span>\r\n<ol>\r\n	<li>If box is labeled <strong>Freeze Upon Arrival</strong> or <strong>Refrigerate Upon Arrival</strong>, make sure to unpack it immediately.</li>\r\n	<li>Label packaging slip with your initials (make sure that they are legible), the date, and the word received. Place the packaging slip in the slot labeled on the main shelves above the phone with the red label.</li>\r\n	<li>Remove the Purchase Order Request from the Orders Not Yet Received folder in the Main Lab and file it under the Company Name in the Purchasing notebooks (red notebooks) above the phone in the Main Lab.</li>\r\n</ol>','Supplies','','inherit','closed','open','','374-revision-v1','','','2014-03-10 18:40:06','2014-03-10 18:40:06','',374,'http://localhost/~psalessi/Genetics_Lab/374-revision-v1/',0,'revision','',0),(389,1,'2014-03-10 18:45:56','2014-03-10 18:45:56','<p style=\"text-align: center;\">Our past Alumni are listed here!</p>','Alumni','','inherit','closed','open','','277-revision-v1','','','2014-03-10 18:45:56','2014-03-10 18:45:56','',277,'http://localhost/~psalessi/Genetics_Lab/277-revision-v1/',0,'revision','',0),(390,1,'2014-03-10 18:48:30','2014-03-10 18:48:30','<h3 style=\"text-align: center;\"><a title=\"http://genome-lab.ucdavis.edu/People/Alumni/JAgresti/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/JAgresti/default.htm\">Jeremy Agresti</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/MBagley/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/MBagley/default.htm\">Mark Bagley</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/TashaBelfiore/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/TashaBelfiore/default.htm\">Tasha Belfiore</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/JenBeyer/jenB\'s_page.html\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/JenBeyer/jenB\'s_page.html\">Jen Beyer</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/VickiNghiBien/Vicki_mainpage.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/VickiNghiBien/Vicki_mainpage.htm\">Vicki Nghi Bien</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/MarcBlumberg/Marc/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/MarcBlumberg/Marc/default.htm\">Marc Blumberg</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/YongjiuChen/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/YongjiuChen/default.htm\">Yongjiu Chen</a>\r\nCarol Conway\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/JanCordes/Jan\'s home page1.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/JanCordes/Jan\'s%20home%20page1.htm\">Jan Cordes</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/KatharineCoykendall/katherine_coykendall.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/KatharineCoykendall/katherine_coykendall.htm\">Katherine Coykendall</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/ChrisFloyd/chf-front.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/ChrisFloyd/chf-front.htm\">Chris Floyd</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/AdaFowler/Fowler.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/AdaFowler/Fowler.htm\">Ada Fowler</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/AnnaForlani/Anna1.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/AnnaForlani/Anna1.htm\">Anna Forlani</a>\r\n<a title=\"http://www.ibiologia.unam.mx/directorio/h/herrera_montalvo.htm\" href=\"http://www.ibiologia.unam.mx/directorio/h/herrera_montalvo.htm\">Gerardo Herrera</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/ZebHogan/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/ZebHogan/default.htm\">Zeb Hogan</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/EMcqown/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/EMcqown/default.htm\">Eve McQuown</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/ErinMeredith/Erin Meredith_mainpage.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/ErinMeredith/Erin%20Meredith_mainpage.htm\">Erin Meredith</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/ThomasMoen/ThomasMoen_mainpage.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/ThomasMoen/ThomasMoen_mainpage.htm\">Thomas Moen</a>\r\nRiccardo Papa\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/Jens_Web_Pages/Genomic Variation Laboratory-Jen\'s page.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/Jens_Web_Pages/Genomic%20Variation%20Laboratory-Jen\'s%20page.htm\">Jen Roach</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/JeffRodzen/rodzen.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/JeffRodzen/rodzen.htm\">Jeff Rodzen</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/FernandaRodriguez/Rodriguez.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/FernandaRodriguez/Rodriguez.htm\">Fernanda Rodriguez</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/GregTranah/Tranah.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/GregTranah/Tranah.htm\">Greg Tranah</a>\r\nKevin Williamson</h3>','Past Alumni','','publish','closed','open','','past-alumni','','','2014-03-10 18:48:51','2014-03-10 18:48:51','',277,'http://localhost/~psalessi/Genetics_Lab/?page_id=390',0,'page','',0),(391,1,'2014-03-10 18:47:47','2014-03-10 18:47:47','','Past Alumni Page Featured Image','','inherit','closed','open','','past_alumni_header','','','2014-03-10 18:47:47','2014-03-10 18:47:47','',390,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/past_alumni_header.png',0,'attachment','image/png',0),(392,1,'2014-03-10 18:48:30','2014-03-10 18:48:30','<h3><a title=\"http://genome-lab.ucdavis.edu/People/Alumni/JAgresti/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/JAgresti/default.htm\">Jeremy Agresti</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/MBagley/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/MBagley/default.htm\">Mark Bagley</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/TashaBelfiore/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/TashaBelfiore/default.htm\">Tasha Belfiore</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/JenBeyer/jenB\'s_page.html\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/JenBeyer/jenB\'s_page.html\">Jen Beyer</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/VickiNghiBien/Vicki_mainpage.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/VickiNghiBien/Vicki_mainpage.htm\">Vicki Nghi Bien</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/MarcBlumberg/Marc/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/MarcBlumberg/Marc/default.htm\">Marc Blumberg</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/YongjiuChen/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/YongjiuChen/default.htm\">Yongjiu Chen</a>\r\nCarol Conway\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/JanCordes/Jan\'s home page1.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/JanCordes/Jan\'s%20home%20page1.htm\">Jan Cordes</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/KatharineCoykendall/katherine_coykendall.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/KatharineCoykendall/katherine_coykendall.htm\">Katherine Coykendall</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/ChrisFloyd/chf-front.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/ChrisFloyd/chf-front.htm\">Chris Floyd</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/AdaFowler/Fowler.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/AdaFowler/Fowler.htm\">Ada Fowler</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/AnnaForlani/Anna1.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/AnnaForlani/Anna1.htm\">Anna Forlani</a>\r\n<a title=\"http://www.ibiologia.unam.mx/directorio/h/herrera_montalvo.htm\" href=\"http://www.ibiologia.unam.mx/directorio/h/herrera_montalvo.htm\">Gerardo Herrera</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/ZebHogan/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/ZebHogan/default.htm\">Zeb Hogan</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/EMcqown/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/EMcqown/default.htm\">Eve McQuown</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/ErinMeredith/Erin Meredith_mainpage.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/ErinMeredith/Erin%20Meredith_mainpage.htm\">Erin Meredith</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/ThomasMoen/ThomasMoen_mainpage.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/ThomasMoen/ThomasMoen_mainpage.htm\">Thomas Moen</a>\r\nRiccardo Papa\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/Jens_Web_Pages/Genomic Variation Laboratory-Jen\'s page.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/Jens_Web_Pages/Genomic%20Variation%20Laboratory-Jen\'s%20page.htm\">Jen Roach</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/JeffRodzen/rodzen.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/JeffRodzen/rodzen.htm\">Jeff Rodzen</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/FernandaRodriguez/Rodriguez.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/FernandaRodriguez/Rodriguez.htm\">Fernanda Rodriguez</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/GregTranah/Tranah.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/GregTranah/Tranah.htm\">Greg Tranah</a>\r\nKevin Williamson</h3>','Past Alumni','','inherit','closed','open','','390-revision-v1','','','2014-03-10 18:48:30','2014-03-10 18:48:30','',390,'http://localhost/~psalessi/Genetics_Lab/390-revision-v1/',0,'revision','',0),(393,1,'2014-03-10 18:48:51','2014-03-10 18:48:51','<h3 style=\"text-align: center;\"><a title=\"http://genome-lab.ucdavis.edu/People/Alumni/JAgresti/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/JAgresti/default.htm\">Jeremy Agresti</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/MBagley/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/MBagley/default.htm\">Mark Bagley</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/TashaBelfiore/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/TashaBelfiore/default.htm\">Tasha Belfiore</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/JenBeyer/jenB\'s_page.html\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/JenBeyer/jenB\'s_page.html\">Jen Beyer</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/VickiNghiBien/Vicki_mainpage.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/VickiNghiBien/Vicki_mainpage.htm\">Vicki Nghi Bien</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/MarcBlumberg/Marc/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/MarcBlumberg/Marc/default.htm\">Marc Blumberg</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/YongjiuChen/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/YongjiuChen/default.htm\">Yongjiu Chen</a>\r\nCarol Conway\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/JanCordes/Jan\'s home page1.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/JanCordes/Jan\'s%20home%20page1.htm\">Jan Cordes</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/KatharineCoykendall/katherine_coykendall.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/KatharineCoykendall/katherine_coykendall.htm\">Katherine Coykendall</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/ChrisFloyd/chf-front.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/ChrisFloyd/chf-front.htm\">Chris Floyd</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/AdaFowler/Fowler.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/AdaFowler/Fowler.htm\">Ada Fowler</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/AnnaForlani/Anna1.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/AnnaForlani/Anna1.htm\">Anna Forlani</a>\r\n<a title=\"http://www.ibiologia.unam.mx/directorio/h/herrera_montalvo.htm\" href=\"http://www.ibiologia.unam.mx/directorio/h/herrera_montalvo.htm\">Gerardo Herrera</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/ZebHogan/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/ZebHogan/default.htm\">Zeb Hogan</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/EMcqown/default.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/EMcqown/default.htm\">Eve McQuown</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/ErinMeredith/Erin Meredith_mainpage.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/ErinMeredith/Erin%20Meredith_mainpage.htm\">Erin Meredith</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/ThomasMoen/ThomasMoen_mainpage.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/ThomasMoen/ThomasMoen_mainpage.htm\">Thomas Moen</a>\r\nRiccardo Papa\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/Jens_Web_Pages/Genomic Variation Laboratory-Jen\'s page.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/Jens_Web_Pages/Genomic%20Variation%20Laboratory-Jen\'s%20page.htm\">Jen Roach</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/JeffRodzen/rodzen.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/JeffRodzen/rodzen.htm\">Jeff Rodzen</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/FernandaRodriguez/Rodriguez.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/FernandaRodriguez/Rodriguez.htm\">Fernanda Rodriguez</a>\r\n<a title=\"http://genome-lab.ucdavis.edu/People/Alumni/GregTranah/Tranah.htm\" href=\"http://genome-lab.ucdavis.edu/People/Alumni/GregTranah/Tranah.htm\">Greg Tranah</a>\r\nKevin Williamson</h3>','Past Alumni','','inherit','closed','open','','390-revision-v1','','','2014-03-10 18:48:51','2014-03-10 18:48:51','',390,'http://localhost/~psalessi/Genetics_Lab/390-revision-v1/',0,'revision','',0),(394,1,'2014-03-10 18:49:11','2014-03-10 18:49:11','<p style=\"text-align: center;\"><a title=\"Past Alumni\" href=\"http://localhost/~psalessi/Genetics_Lab/alumni/past-alumni/\">Our past Alumni are listed here!</a></p>','Alumni','','inherit','closed','open','','277-revision-v1','','','2014-03-10 18:49:11','2014-03-10 18:49:11','',277,'http://localhost/~psalessi/Genetics_Lab/277-revision-v1/',0,'revision','',0),(395,1,'2014-03-10 19:24:06','2014-03-10 19:24:06','','Little Kern Golden Trout (<em>Oncorhynchus mykiss whitei</em>)','','publish','closed','closed','','little-kern-golden-trout-oncorhynchus-mykiss-whitei','','','2014-03-17 17:02:22','2014-03-17 17:02:22','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=projects&#038;p=395',0,'projects','',0),(396,1,'2014-03-10 19:11:23','2014-03-10 19:11:23','','Little Kern Golden Trout Red Green Graph Image','Figure 2. STRUCTURE Bayesian analysis of Little Kern golden trout SPN data. Populations 1-5 are Little Kern broodstock populations, populations 6-23 and 25 are LKGT populations, and 26-30 are hatchery and wild rainbow trout. Each vertical bar represents an individual fish, with individuals grouped by population number (as in Table 4) on the x-axis. Bar colors represent golden trout (red) and rainbow trout (green), with the proportion of green indicating of the level of rainbow trout introgression in a given individual.','inherit','closed','open','','little_kern_golden_trout_graph_red_green','','','2014-03-10 19:11:23','2014-03-10 19:11:23','',395,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/little_kern_golden_trout_graph_red_green.jpg',0,'attachment','image/jpeg',0),(397,1,'2014-03-10 19:11:24','2014-03-10 19:11:24','','Little Kern Golden Trout Map Image','Figure 1. Distribution of Little Kern golden trout broodstock during restoration efforts. (From Gall 1997).','inherit','closed','open','','little_kern_golden_trout_map','','','2014-03-10 19:11:24','2014-03-10 19:11:24','',395,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/little_kern_golden_trout_map.jpg',0,'attachment','image/jpeg',0),(398,1,'2014-03-10 19:23:14','2014-03-10 19:23:14','','Little Kern Golden Trout Project Image','','inherit','closed','open','','little_kern_golden_trout_project_image','','','2014-03-10 19:23:14','2014-03-10 19:23:14','',395,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/little_kern_golden_trout_project_image.jpg',0,'attachment','image/jpeg',0),(400,1,'2014-03-10 19:32:58','2014-03-10 19:32:58','','Little Kern Project Page Featured Image','','inherit','closed','open','','little_kern_header','','','2014-03-10 19:32:58','2014-03-10 19:32:58','',395,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/little_kern_header.png',0,'attachment','image/png',0),(402,1,'2014-03-10 19:41:57','2014-03-10 19:41:57','','Mandi Finger','','publish','closed','closed','','mandi-finger','','','2014-03-12 17:40:40','2014-03-12 17:40:40','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=402',0,'people','',0),(403,1,'2014-03-10 19:41:14','2014-03-10 19:41:14','','Mandi Finger Profile Image','','inherit','closed','open','','mandi_finger_profile_image','','','2014-03-10 19:41:14','2014-03-10 19:41:14','',402,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/Mandi_Finger_profile_image.jpg',0,'attachment','image/jpeg',0),(405,1,'2014-03-10 19:56:35','2014-03-10 19:56:35','','Andrea Drauch Schreier','','publish','closed','closed','','andrea-drauch-schreier','','','2014-03-12 17:40:25','2014-03-12 17:40:25','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=405',0,'people','',0),(406,1,'2014-03-10 19:48:32','2014-03-10 19:48:32','','Andrea Schreier 2012 Curriculum Vitae PDF','','inherit','closed','open','','andrea_schreier_cv_2012','','','2014-03-10 19:48:32','2014-03-10 19:48:32','',405,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/andrea_schreier_cv_2012.pdf',0,'attachment','application/pdf',0),(407,1,'2014-03-10 19:48:33','2014-03-10 19:48:33','','Andrea Schreier Profile Image','','inherit','closed','open','','andrea_schreier_profile_image','','','2014-03-10 19:48:33','2014-03-10 19:48:33','',405,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/andrea_schreier_profile_image.jpg',0,'attachment','image/jpeg',0),(408,1,'2014-03-10 20:10:54','2014-03-10 20:10:54','','Melinda Baerwald','','publish','closed','closed','','melinda-baerwald','','','2014-03-17 18:31:09','2014-03-17 18:31:09','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=408',0,'people','',0),(409,1,'2014-03-10 20:01:43','2014-03-10 20:01:43','','Melinda Baerwald 2012 Curriculum Vitae PDF','','inherit','closed','open','','melinda_baerwald_2012_cv','','','2014-03-10 20:01:43','2014-03-10 20:01:43','',408,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/melinda_baerwald_2012_cv.pdf',0,'attachment','application/pdf',0),(410,1,'2014-03-10 20:01:44','2014-03-10 20:01:44','','Melinda Baerwald Profile Image','','inherit','closed','open','','melinda_baerwald_profile_image','','','2014-03-10 20:01:44','2014-03-10 20:01:44','',408,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/Melinda_Baerwald_profile_image.jpg',0,'attachment','image/jpeg',0),(411,1,'2014-03-10 20:25:34','2014-03-10 20:25:34','','Molly Stephens','','publish','closed','closed','','molly-stephens','','','2014-03-12 17:30:13','2014-03-12 17:30:13','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=411',0,'people','',0),(412,1,'2014-03-10 20:16:32','2014-03-10 20:16:32','','Molly Stephens 2012 Curriculum Vitae PDF','','inherit','closed','open','','molly_stephens_2012_cv','','','2014-03-10 20:16:32','2014-03-10 20:16:32','',411,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/molly_stephens_2012_cv.pdf',0,'attachment','application/pdf',0),(413,1,'2014-03-10 20:16:32','2014-03-10 20:16:32','','Molly Stephens Grassy Field Image','','inherit','closed','open','','molly_stephens_grassy_field_image','','','2014-03-10 20:16:32','2014-03-10 20:16:32','',411,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/molly_stephens_grassy_field_image.jpg',0,'attachment','image/jpeg',0),(414,1,'2014-03-10 20:16:33','2014-03-10 20:16:33','','Molly Stephens Profile Image','','inherit','closed','open','','molly_stephens_profile_image','','','2014-03-10 20:16:33','2014-03-10 20:16:33','',411,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/molly_stephens_profile_image.jpg',0,'attachment','image/jpeg',0),(417,1,'2014-03-10 20:38:23','2014-03-10 20:38:23','','Josh Hull','','publish','closed','closed','','josh-hull','','','2014-03-17 18:30:15','2014-03-17 18:30:15','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=417',0,'people','',0),(418,1,'2014-03-10 20:29:07','2014-03-10 20:29:07','','Josh Hull Featured Image','','inherit','closed','open','','josh_hull_header','','','2014-03-10 20:29:07','2014-03-10 20:29:07','',417,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/josh_hull_header.png',0,'attachment','image/png',0),(419,1,'2014-03-10 20:29:07','2014-03-10 20:29:07','','Josh Hull Profile Image','','inherit','closed','open','','josh_hull_profile_image','','','2014-03-10 20:29:07','2014-03-10 20:29:07','',417,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/josh_hull_profile_image.jpg',0,'attachment','image/jpeg',0),(420,1,'2014-03-10 20:45:38','2014-03-10 20:45:38','','Scott Brandl','','publish','closed','closed','','scott-brandl','','','2014-03-12 17:38:46','2014-03-12 17:38:46','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=420',0,'people','',0),(421,1,'2014-03-10 20:42:40','2014-03-10 20:42:40','','Scott Brandl Curriculum Vitae DOC','','inherit','closed','open','','cv-scott-brandl','','','2014-03-10 20:42:40','2014-03-10 20:42:40','',420,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/CV-Scott-Brandl.doc',0,'attachment','application/msword',0),(422,1,'2014-03-10 20:42:50','2014-03-10 20:42:50','','Scott Brandl Profile Image','','inherit','closed','open','','scott_brandl_profile_image','','','2014-03-10 20:42:50','2014-03-10 20:42:50','',420,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/scott_brandl_profile_image.jpg',0,'attachment','image/jpeg',0),(423,1,'2014-03-10 20:49:25','2014-03-10 20:49:25','','Bjorn Erickson','','publish','closed','closed','','bjorn-erickson','','','2014-03-12 17:38:34','2014-03-12 17:38:34','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=423',0,'people','',0),(424,1,'2014-03-10 20:47:05','2014-03-10 20:47:05','','Bjorn Erickson Curriculum Vitae DOC','','inherit','closed','open','','erickson_cv','','','2014-03-10 20:47:05','2014-03-10 20:47:05','',423,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/Erickson_CV.doc',0,'attachment','application/msword',0),(425,1,'2014-03-10 20:47:28','2014-03-10 20:47:28','','Bjorn Erickson Profile Image','','inherit','closed','open','','bjorn_erickson_profile_image','','','2014-03-10 20:47:28','2014-03-10 20:47:28','',423,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/Bjorn_erickson_profile_image.jpg',0,'attachment','image/jpeg',0),(426,1,'2014-03-12 17:04:29','2014-03-12 17:04:29','','Kristy Deiner','','publish','closed','closed','','kristy-deiner','','','2014-03-17 16:57:09','2014-03-17 16:57:09','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=426',0,'people','',0),(427,1,'2014-03-12 16:59:56','2014-03-12 16:59:56','','Kristy Deiner 2012 Curriculum Vitae PDF','','inherit','closed','open','','kristy_deiner_2012_cv','','','2014-03-12 16:59:56','2014-03-12 16:59:56','',426,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/kristy_deiner_2012_cv.pdf',0,'attachment','application/pdf',0),(428,1,'2014-03-12 16:59:56','2014-03-12 16:59:56','','Kristy Deiner Hiking A Mountain Image','','inherit','closed','open','','kristy_deiner_hiking_mountain_image','','','2014-03-12 16:59:56','2014-03-12 16:59:56','',426,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/kristy_deiner_hiking_mountain_image.jpg',0,'attachment','image/jpeg',0),(429,1,'2014-03-12 16:59:56','2014-03-12 16:59:56','','Kristy Deiner Profile Image','','inherit','closed','open','','kristy_deiner_profile_image','','','2014-03-12 16:59:56','2014-03-12 16:59:56','',426,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/kristy_deiner_profile_image.jpg',0,'attachment','image/jpeg',0),(430,1,'2014-03-12 17:18:20','2014-03-12 17:18:20','','Katharine Tomalty','','publish','closed','closed','','katharine-tomalty','','','2014-03-12 17:53:22','2014-03-12 17:53:22','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=430',0,'people','',0),(431,1,'2014-03-12 17:17:28','2014-03-12 17:17:28','','Alex Van Dam 2012 Curriculum Vitae DOC','','inherit','closed','open','','alex-van-dam-cv-i-24-12','','','2014-03-12 17:17:28','2014-03-12 17:17:28','',430,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/Alex-Van-Dam-CV-i-24-12.doc',0,'attachment','application/msword',0),(432,1,'2014-03-12 17:17:29','2014-03-12 17:17:29','','Alex Van Dam Profile Image','','inherit','closed','open','','alex_van_dam_profile_image','','','2014-03-12 17:17:29','2014-03-12 17:17:29','',430,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/alex_van_dam_profile_image.jpg',0,'attachment','image/jpeg',0),(434,1,'2014-03-12 17:17:29','2014-03-12 17:17:29','','Alexander Gaos Profile Image','','inherit','closed','open','','alexander_gaos_profile_image','','','2014-03-12 17:17:29','2014-03-12 17:17:29','',430,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/alexander_gaos_profile_image.jpg',0,'attachment','image/jpeg',0),(435,1,'2014-03-12 17:17:30','2014-03-12 17:17:30','','Joe Medley Profile Image','','inherit','closed','open','','joe_medley_profile_image','','','2014-03-12 17:17:30','2014-03-12 17:17:30','',430,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/joe_medley_profile_image.jpg',0,'attachment','image/jpeg',0),(436,1,'2014-03-12 17:17:30','2014-03-12 17:17:30','','Katharine Tomalty Profile Image','','inherit','closed','open','','katharine_tomalty_profile_image','','','2014-03-12 17:17:30','2014-03-12 17:17:30','',430,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/katharine_tomalty_profile_image.jpg',0,'attachment','image/jpeg',0),(437,1,'2014-03-12 17:19:31','2014-03-12 17:19:31','','Personal Information','','publish','closed','closed','','acf_personal-information','','','2014-03-12 17:25:17','2014-03-12 17:25:17','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=acf&#038;p=437',0,'acf','',0),(438,1,'2014-03-12 17:31:39','2014-03-12 17:31:39','','Project Excerpt','','publish','closed','closed','','acf_project-excerpt','','','2014-03-12 17:31:45','2014-03-12 17:31:45','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=acf&#038;p=438',0,'acf','',0),(439,1,'2014-03-12 17:33:31','2014-03-12 17:33:31','','Project Contact Information','','publish','closed','closed','','acf_project-contact-information','','','2014-03-12 17:33:54','2014-03-12 17:33:54','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=acf&#038;p=439',0,'acf','',0),(440,1,'2014-03-12 17:52:46','2014-03-12 17:52:46','','Alex R. Van Dam','','publish','closed','closed','','alex-r-van-dam','','','2014-03-12 17:52:46','2014-03-12 17:52:46','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=440',0,'people','',0),(441,1,'2014-03-12 17:58:33','2014-03-12 17:58:33','','Alexander R. Gaos','','publish','closed','closed','','alexander-r-gaos','','','2014-03-12 17:58:33','2014-03-12 17:58:33','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=441',0,'people','',0),(442,1,'2014-03-12 17:56:30','2014-03-12 17:56:30','','Alexander Gaos 2012 Curriculum Vitae PDF','','inherit','closed','open','','alexander_gaos_2012_cv','','','2014-03-12 17:56:30','2014-03-12 17:56:30','',441,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/alexander_gaos_2012_cv.pdf',0,'attachment','application/pdf',0),(443,1,'2014-03-12 18:01:23','2014-03-12 18:01:23','','Joe Medley','','publish','closed','closed','','joe-medley','','','2014-03-12 18:01:23','2014-03-12 18:01:23','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=443',0,'people','',0),(444,1,'2014-03-14 18:43:54','2014-03-14 18:43:54','','Alisha Goodbla','','publish','closed','closed','','alisha-goodbla','','','2014-03-14 18:44:30','2014-03-14 18:44:30','',0,'http://169.237.97.241/~psalessi/Genetics_Lab/?post_type=people&#038;p=444',0,'people','',0),(445,1,'2014-03-14 18:38:49','2014-03-14 18:38:49','','Alisha Goodbla 2012 Curriculum Vitae PDF','','inherit','closed','open','','alisha_goodbla_2012_cv','','','2014-03-14 18:38:49','2014-03-14 18:38:49','',444,'http://169.237.97.241/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/alisha_goodbla_2012_cv.pdf',0,'attachment','application/pdf',0),(446,1,'2014-03-14 18:38:49','2014-03-14 18:38:49','','Alisha Goodbla Profile Image','','inherit','closed','open','','alisha_goodbla_profile_image','','','2014-03-14 18:38:49','2014-03-14 18:38:49','',444,'http://169.237.97.241/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/alisha_goodbla_profile_image.jpg',0,'attachment','image/jpeg',0),(447,1,'2014-03-14 18:38:50','2014-03-14 18:38:50','','Antonia Wong Profile Image','','inherit','closed','open','','antonia_wong_profile_image','','','2014-03-14 18:38:50','2014-03-14 18:38:50','',444,'http://169.237.97.241/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/antonia_wong_profile_image.jpg',0,'attachment','image/jpeg',0),(448,1,'2014-03-14 18:38:50','2014-03-14 18:38:50','','Emily Ringelman Profile Image','','inherit','closed','open','','emily_ringelman_profile_image','','','2014-03-14 18:38:50','2014-03-14 18:38:50','',444,'http://169.237.97.241/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/emily_ringelman_profile_image.jpg',0,'attachment','image/jpeg',0),(449,1,'2014-03-14 18:47:46','2014-03-14 18:47:46','','Emily Ringelman','','publish','closed','closed','','emily-ringelman','','','2014-03-14 18:47:46','2014-03-14 18:47:46','',0,'http://169.237.97.241/~psalessi/Genetics_Lab/?post_type=people&#038;p=449',0,'people','',0),(450,1,'2014-03-14 18:52:04','2014-03-14 18:52:04','','Antonia Wong','','publish','closed','closed','','antonia-wong','','','2014-03-14 20:13:18','2014-03-14 20:13:18','',0,'http://169.237.97.241/~psalessi/Genetics_Lab/?post_type=people&#038;p=450',0,'people','',0),(452,1,'2014-03-14 18:50:52','2014-03-14 18:50:52','','Antonia Wong 2012 Curriculum Vitae DOC','','inherit','closed','open','','antonia_wong_2012_cv','','','2014-03-14 18:50:52','2014-03-14 18:50:52','',450,'http://169.237.97.241/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/antonia_wong_2012_cv.doc',0,'attachment','application/msword',0),(453,1,'2014-03-14 19:00:34','2014-03-14 19:00:34','','External Profile Page','','publish','closed','closed','','acf_external-profile-page','','','2014-03-14 19:05:47','2014-03-14 19:05:47','',0,'http://169.237.97.241/~psalessi/Genetics_Lab/?post_type=acf&#038;p=453',0,'acf','',0),(454,1,'2014-03-14 19:49:38','2014-03-14 19:49:38','','Louise Conrad','','publish','closed','closed','','454','','','2014-03-14 19:50:42','2014-03-14 19:50:42','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=454',0,'people','',0),(456,1,'2014-03-14 19:48:47','2014-03-14 19:48:47','','Louise Conrad Profile Image','','inherit','closed','open','','louise_conrad_profile_image','','','2014-03-14 19:48:47','2014-03-14 19:48:47','',454,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/louise_conrad_profile_image.jpg',0,'attachment','image/jpeg',0),(458,1,'2014-03-14 20:15:07','2014-03-14 20:15:07','','Melanie LaCava','','publish','closed','closed','','melanie-lacava','','','2014-03-14 20:15:07','2014-03-14 20:15:07','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=458',0,'people','',0),(459,1,'2014-03-14 20:16:10','2014-03-14 20:16:10','','Deepinder Sidhu','','publish','closed','closed','','deepinder-sidhu','','','2014-03-14 20:16:10','2014-03-14 20:16:10','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=459',0,'people','',0),(460,1,'2014-03-14 20:16:26','2014-03-14 20:16:26','','Jamie Yates','','publish','closed','closed','','jamie-yates','','','2014-03-14 20:16:26','2014-03-14 20:16:26','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=460',0,'people','',0),(461,1,'2014-03-14 20:16:38','2014-03-14 20:16:38','','Lucia Yee','','publish','closed','closed','','lucia-yee','','','2014-03-14 20:16:38','2014-03-14 20:16:38','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=461',0,'people','',0),(462,1,'2014-03-14 20:20:14','2014-03-14 20:20:14','','Katja Waldron','','publish','closed','closed','','katja-waldron','','','2014-03-14 20:20:14','2014-03-14 20:20:14','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=462',0,'people','',0),(463,1,'2014-03-14 20:20:27','2014-03-14 20:20:27','','Allison Weakley','','publish','closed','closed','','allison-weakley','','','2014-03-14 20:20:27','2014-03-14 20:20:27','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=463',0,'people','',0),(464,1,'2014-03-14 20:22:44','2014-03-14 20:22:44','','Gregg Schumer','','publish','closed','closed','','gregg-schumer','','','2014-03-14 20:22:44','2014-03-14 20:22:44','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=464',0,'people','',0),(465,1,'2014-03-14 20:22:12','2014-03-14 20:22:12','','Gregg Schumer Profile Image','','inherit','closed','open','','gregg_schumer_profile_image','','','2014-03-14 20:22:12','2014-03-14 20:22:12','',464,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/gregg_schumer_profile_image.jpg',0,'attachment','image/jpeg',0),(466,1,'2014-03-14 20:30:35','2014-03-14 20:30:35','','Kathleen Fisch','','publish','closed','closed','','kathleen-fisch','','','2014-03-14 20:30:35','2014-03-14 20:30:35','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=466',0,'people','',0),(467,1,'2014-03-14 20:25:13','2014-03-14 20:25:13','','Kathleen Fisch 2012 Curriculum Vitae PDF','','inherit','closed','open','','kathleen_fisch_2012_cv','','','2014-03-14 20:25:13','2014-03-14 20:25:13','',466,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/kathleen_fisch_2012_cv.pdf',0,'attachment','application/pdf',0),(468,1,'2014-03-14 20:25:14','2014-03-14 20:25:14','','Kathleen Fisch Profile Image','','inherit','closed','open','','kathleen_fisch_profile_image','','','2014-03-14 20:25:14','2014-03-14 20:25:14','',466,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/kathleen_fisch_profile_image.jpg',0,'attachment','image/jpeg',0),(470,1,'2014-03-14 20:50:28','2014-03-14 20:50:28','','Josh Israel','','publish','closed','closed','','josh-israel','','','2014-03-14 20:50:28','2014-03-14 20:50:28','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=470',0,'people','',0),(471,1,'2014-03-14 20:42:04','2014-03-14 20:42:04','','Josh Israel Profile Image','','inherit','closed','open','','josh_israel_profile_image','','','2014-03-14 20:42:04','2014-03-14 20:42:04','',470,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/josh_israel_profile_image.jpg',0,'attachment','image/jpeg',0),(472,1,'2014-03-14 20:52:53','2014-03-14 20:52:53','','Brian Schreier','','publish','closed','closed','','brian-schreier','','','2014-03-17 18:30:35','2014-03-17 18:30:35','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=472',0,'people','',0),(473,1,'2014-03-14 20:51:50','2014-03-14 20:51:50','','Brian Schreier Profile Image','','inherit','closed','open','','brian_schreier_profile_image','','','2014-03-14 20:51:50','2014-03-14 20:51:50','',472,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/brian_schreier_profile_image.jpg',0,'attachment','image/jpeg',0),(474,1,'2014-03-14 20:54:46','2014-03-14 20:54:46','','Scott Blankenship','','publish','closed','closed','','scott-blankenship','','','2014-03-14 20:54:46','2014-03-14 20:54:46','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=474',0,'people','',0),(475,1,'2014-03-14 20:53:53','2014-03-14 20:53:53','','Scott Blankenship Profile Image','','inherit','closed','open','','scott_blankenship_profile_image','','','2014-03-14 20:53:53','2014-03-14 20:53:53','',474,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/scott_blankenship_profile_image.jpg',0,'attachment','image/jpeg',0),(476,1,'2014-03-14 20:56:37','2014-03-14 20:56:37','','Jessica Petersen','','publish','closed','closed','','jessica-petersen','','','2014-03-14 20:56:37','2014-03-14 20:56:37','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=476',0,'people','',0),(477,1,'2014-03-14 20:56:02','2014-03-14 20:56:02','','Jessica Petersen Profile Image','','inherit','closed','open','','jessica_petersen_profile_image','','','2014-03-14 20:56:02','2014-03-14 20:56:02','',476,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/jessica_petersen_profile_image.png',0,'attachment','image/png',0),(478,1,'2014-03-14 20:57:57','2014-03-14 20:57:57','','Ryan Lew','','publish','closed','closed','','ryan-lew','','','2014-03-14 20:57:57','2014-03-14 20:57:57','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=478',0,'people','',0),(479,1,'2014-03-14 20:57:24','2014-03-14 20:57:24','','Ryan Lew Profile Image','','inherit','closed','open','','ryan_lew_profile_image','','','2014-03-14 20:57:24','2014-03-14 20:57:24','',478,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/ryan_lew_profile_image.png',0,'attachment','image/png',0),(480,1,'2014-03-17 16:21:54','2014-03-17 16:21:54','','Brian Mahardja','','publish','closed','closed','','brian-mahardja','','','2014-03-17 17:04:13','2014-03-17 17:04:13','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=480',0,'people','',0),(481,1,'2014-03-17 16:13:15','2014-03-17 16:13:15','','Brian Mahardja 2013 Curriculum Vitae PDF','','inherit','closed','open','','brian_mahardja_2013_cv','','','2014-03-17 16:13:15','2014-03-17 16:13:15','',480,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/brian_mahardja_2013_cv.pdf',0,'attachment','application/pdf',0),(482,1,'2014-03-17 16:13:16','2014-03-17 16:13:16','','Brian Mahardja Profile Image','','inherit','closed','open','','brian_mahardja_profile_image','','','2014-03-17 16:13:16','2014-03-17 16:13:16','',480,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/brian_mahardja_profile_image.jpg',0,'attachment','image/jpeg',0),(483,1,'2014-03-17 16:43:32','2014-03-17 16:43:32','','Daphne Gille','','publish','closed','closed','','daphne-gille','','','2014-03-17 19:00:57','2014-03-17 19:00:57','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=483',0,'people','',0),(484,1,'2014-03-17 16:35:53','2014-03-17 16:35:53','','Daphne Gille Curriculum Vitae','','inherit','closed','open','','daphne_gille_cv','','','2014-03-17 16:35:53','2014-03-17 16:35:53','',483,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/Daphne_Gille_cv.pdf',0,'attachment','application/pdf',0),(485,1,'2014-03-17 16:35:53','2014-03-17 16:35:53','','Daphne Gille Profile Image','','inherit','closed','open','','daphne_gille_profile_image','','','2014-03-17 16:35:53','2014-03-17 16:35:53','',483,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/daphne_gille_profile_image.jpg',0,'attachment','image/jpeg',0),(486,1,'2014-03-17 16:41:07','2014-03-17 16:41:07','','Daphne Gille Seal Image','','inherit','closed','open','','daphne_gille_seal_image','','','2014-03-17 16:41:07','2014-03-17 16:41:07','',483,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/daphne_gille_seal_image.jpg',0,'attachment','image/jpeg',0),(487,1,'2014-03-17 16:41:07','2014-03-17 16:41:07','','Daphne Gille Seal on Beach Image','','inherit','closed','open','','daphne_gille_seal_on_beach_image','','','2014-03-17 16:41:07','2014-03-17 16:41:07','',483,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/daphne_gille_seal_on_beach_image.jpg',0,'attachment','image/jpeg',0),(488,1,'2014-03-17 16:41:08','2014-03-17 16:41:08','','Daphne Gille Turtle on Beach Image','','inherit','closed','open','','daphne_gille_turtle_on_beach_image','','','2014-03-17 16:41:08','2014-03-17 16:41:08','',483,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/daphne_gille_turtle_on_beach_image.jpg',0,'attachment','image/jpeg',0),(489,1,'2014-03-17 16:55:26','2014-03-17 16:55:26','','K Jun Bando','','publish','closed','closed','','k-jun-bando','','','2014-03-17 17:44:47','2014-03-17 17:44:47','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=489',0,'people','',0),(490,1,'2014-03-17 16:51:46','2014-03-17 16:51:46','','K Jun Bando 2012 Curriculum Vitae','','inherit','closed','open','','k_jun_bando_2012_cv','','','2014-03-17 16:51:46','2014-03-17 16:51:46','',489,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/k_jun_bando_2012_cv.pdf',0,'attachment','application/pdf',0),(491,1,'2014-03-17 16:51:46','2014-03-17 16:51:46','','K Jun Bando Profile Image','','inherit','closed','open','','k_jun_bando_profile_image','','','2014-03-17 16:51:46','2014-03-17 16:51:46','',489,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/k_jun_bando_profile_image.jpg',0,'attachment','image/jpeg',0),(492,1,'2014-03-17 17:38:44','2014-03-17 17:38:44','','Karrigan S. Brk','','publish','closed','closed','','karrigan-s-bork','','','2014-03-17 17:44:07','2014-03-17 17:44:07','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=492',0,'people','',0),(493,1,'2014-03-17 17:35:39','2014-03-17 17:35:39','','Karrigan Bork Profile Image','','inherit','closed','open','','karrigan_bork_profile_image','','','2014-03-17 17:35:39','2014-03-17 17:35:39','',492,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/karrigan_bork_profile_image.png',0,'attachment','image/png',0),(494,1,'2014-03-17 17:35:39','2014-03-17 17:35:39','','Rachel Schwartz Profile Image','','inherit','closed','open','','rachel_schwartz_profile_image','','','2014-03-17 17:35:39','2014-03-17 17:35:39','',492,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/rachel_schwartz_profile_image.png',0,'attachment','image/png',0),(495,1,'2014-03-17 17:35:39','2014-03-17 17:35:39','','Rachel Simmons Profile Image','','inherit','closed','open','','rachel_simmons_profile_image','','','2014-03-17 17:35:39','2014-03-17 17:35:39','',492,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/rachel_simmons_profile_image.png',0,'attachment','image/png',0),(496,1,'2014-03-17 17:35:40','2014-03-17 17:35:40','','Rick Topinka Profile Image','','inherit','closed','open','','rick_topinka_profile_image','','','2014-03-17 17:35:40','2014-03-17 17:35:40','',492,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/rick_topinka_profile_image.png',0,'attachment','image/png',0),(497,1,'2014-03-17 17:35:40','2014-03-17 17:35:40','','Yonjiu Chen Profile Image','','inherit','closed','open','','yonjiu_chen_profile_image','','','2014-03-17 17:35:40','2014-03-17 17:35:40','',492,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/yonjiu_chen_profile_image.png',0,'attachment','image/png',0),(499,1,'2014-03-17 17:56:33','2014-03-17 17:56:33','','Rachel Schwartz','','publish','closed','closed','','rachel-schwartz','','','2014-03-17 17:56:33','2014-03-17 17:56:33','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=499',0,'people','',0),(500,1,'2014-03-17 17:57:20','2014-03-17 17:57:20','','Rachel Simmons','','publish','closed','closed','','rachel-simmons','','','2014-03-17 17:57:20','2014-03-17 17:57:20','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=500',0,'people','',0),(501,1,'2014-03-17 17:58:07','2014-03-17 17:58:07','','Rick Topinka','','publish','closed','closed','','rick-topinka','','','2014-03-17 18:29:22','2014-03-17 18:29:22','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=501',0,'people','',0),(502,1,'2014-03-17 17:59:23','2014-03-17 17:59:23','','Amy Welsh','','publish','closed','closed','','amy-welsh','','','2014-03-17 18:29:14','2014-03-17 18:29:14','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=502',0,'people','',0),(503,1,'2014-03-17 17:58:39','2014-03-17 17:58:39','','Amy Welsh Profile Image','','inherit','closed','open','','amy_welsh_profile_image','','','2014-03-17 17:58:39','2014-03-17 17:58:39','',502,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/amy_welsh_profile_image.png',0,'attachment','image/png',0),(504,1,'2014-03-17 18:00:01','2014-03-17 18:00:01','','Yonjiu Chen','','publish','closed','closed','','yonjiu-chen','','','2014-03-17 18:29:06','2014-03-17 18:29:06','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=people&#038;p=504',0,'people','',0),(505,1,'2014-03-17 18:38:41','2014-03-17 18:38:41','<p style=\"text-align: center;\"><a title=\"Past Alumni\" href=\"http://localhost/~psalessi/Genetics_Lab/alumni/past-alumni/\">Our past alumni are listed here!</a></p>','Alumni','','inherit','closed','open','','277-revision-v1','','','2014-03-17 18:38:41','2014-03-17 18:38:41','',277,'http://localhost/~psalessi/Genetics_Lab/277-revision-v1/',0,'revision','',0),(507,1,'2014-03-20 16:41:46','2014-03-20 16:41:46','','Brackish Water Hydrozoans','','publish','closed','closed','','brackish-water-hydrozoans','','','2014-03-20 17:19:55','2014-03-20 17:19:55','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=projects&#038;p=507',0,'projects','',0),(508,1,'2014-03-20 16:37:48','2014-03-20 16:37:48','','Brackish Water Hydro Jellyfish Single Image','','inherit','closed','open','','brackish_water_hydro_jellyfish_single_image','','','2014-03-20 16:37:48','2014-03-20 16:37:48','',507,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/brackish_water_hydro_jellyfish_single_image.jpg',0,'attachment','image/jpeg',0),(509,1,'2014-03-20 16:37:49','2014-03-20 16:37:49','','Brackish Water Hydros Jellyfish Two Image','','inherit','closed','open','','brackish_water_hydro_jellyfish_two_image','','','2014-03-20 16:37:49','2014-03-20 16:37:49','',507,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/brackish_water_hydro_jellyfish_two_image.jpg',0,'attachment','image/jpeg',0),(510,1,'2014-03-20 16:37:50','2014-03-20 16:37:50','','Brackish Water Hydros Project Image','','inherit','closed','open','','brackish_water_hydro_project_image','','','2014-03-20 16:37:50','2014-03-20 16:37:50','',507,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/brackish_water_hydro_project_image.jpg',0,'attachment','image/jpeg',0),(511,1,'2014-03-20 16:41:00','2014-03-20 16:41:00','','Brackish Water Hydros Featured Image','','inherit','closed','open','','brackish_water_hydros_header_image','','','2014-03-20 16:41:00','2014-03-20 16:41:00','',507,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/brackish_water_hydros_header_image.png',0,'attachment','image/png',0),(512,1,'2014-03-20 17:17:43','2014-03-20 17:17:43','','Genetic monitoring of the Kootenai River white sturgeon conservation aquaculture program','','publish','closed','closed','','genetic-monitoring-of-the-kootenai-river-white-sturgeon-conservation-aquaculture-program','','','2014-06-09 20:45:08','2014-06-09 20:45:08','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=projects&#038;p=512',0,'projects','',0),(513,1,'2014-03-20 17:13:59','2014-03-20 17:13:59','','Genetic Monitoring Sturgeon Project Image','','inherit','closed','open','','genetic_monitoring_sturgeon_project_image','','','2014-03-20 17:13:59','2014-03-20 17:13:59','',512,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/genetic_monitoring_sturgeon_project_image.jpg',0,'attachment','image/jpeg',0),(514,1,'2014-03-20 17:17:11','2014-03-20 17:17:11','','White Sturgeon Projects Featured Image','','inherit','closed','open','','white_sturgeo_header_image','','','2014-03-20 17:17:11','2014-03-20 17:17:11','',512,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/white_sturgeo_header_image.png',0,'attachment','image/png',0),(515,1,'2014-03-20 17:21:05','2014-03-20 17:21:05','','Examination of white sturgeon population structure across the species range','','publish','closed','closed','','examination-of-white-sturgeon-population-structure-across-the-species-range','','','2014-03-20 17:21:05','2014-03-20 17:21:05','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=projects&#038;p=515',0,'projects','',0),(516,1,'2014-03-20 17:31:34','2014-03-20 17:31:34','','Inference of ploidy level in white sturgeon','','publish','closed','closed','','inference-of-ploidy-level-in-white-sturgeon','','','2014-03-20 17:31:54','2014-03-20 17:31:54','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=projects&#038;p=516',0,'projects','',0),(517,1,'2014-03-20 17:27:59','2014-03-20 17:27:59','','Inference of Ploidy Project Image','','inherit','closed','open','','inference-_of_ploidy_many_sturgeons_image','','','2014-03-20 17:27:59','2014-03-20 17:27:59','',516,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/inference-_of_ploidy_many_sturgeons_image.jpg',0,'attachment','image/jpeg',0),(518,1,'2014-03-20 17:28:00','2014-03-20 17:28:00','','Inference of Ploidy Many Sturgeons Closeup Image','','inherit','closed','open','','inference_of_ploidy_many_sturgeons_closeup_image','','','2014-03-20 17:28:00','2014-03-20 17:28:00','',516,'http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/03/inference_of_ploidy_many_sturgeons_closeup_image.jpg',0,'attachment','image/jpeg',0),(519,1,'2014-06-05 19:05:58','0000-00-00 00:00:00','','Auto Draft','','auto-draft','closed','open','','','','','2014-06-05 19:05:58','0000-00-00 00:00:00','',0,'http://localhost/~psalessi/Genetics_Lab/?p=519',0,'post','',0),(521,1,'2014-06-05 19:15:09','2014-06-05 19:15:09','[lenslider id=\"9cc485fff2\"]\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<img class=\"size-full wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"240\" height=\"181\" /> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.\r\n\r\n[caption id=\"attachment_247\" align=\"alignleft\" width=\"110\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\"size-full wp-image-247 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" alt=\"Learn About Our Projects\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\">Learn About Our Projects</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignleft\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" alt=\"Support Us\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_248\" align=\"alignleft\" width=\"110\"]<a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\"><img class=\"size-full wp-image-248 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" width=\"110\" height=\"83\" /></a> <a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-05 19:15:09','2014-06-05 19:15:09','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(526,1,'2014-06-05 19:36:36','2014-06-05 19:36:36','','New Slider','','publish','closed','open','','new-slider','','','2014-06-05 22:11:58','2014-06-05 22:11:58','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=ml-slider&#038;p=526',0,'ml-slider','',0),(527,1,'2014-06-05 20:08:08','0000-00-00 00:00:00','','My SlideDeck','','auto-draft','closed','closed','','','','','2014-06-05 20:08:08','0000-00-00 00:00:00','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=slidedeck2&p=527',0,'slidedeck2','',0),(528,1,'2014-06-05 20:13:39','0000-00-00 00:00:00','','Auto Draft','','auto-draft','closed','open','','','','','2014-06-05 20:13:39','0000-00-00 00:00:00','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=soliloquy&p=528',0,'soliloquy','',0),(529,1,'2014-06-05 20:25:04','2014-06-05 20:25:04','','Hi my name is','','publish','closed','closed','','hi-my-name-is','','','2014-06-05 20:26:00','2014-06-05 20:26:00','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=ssp_slider&#038;p=529',0,'ssp_slider','',0),(530,1,'2014-06-05 20:25:19','2014-06-05 20:25:19','[slider id=\'529\' name=\'Hi my name is\']\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<img class=\"size-full wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"240\" height=\"181\" /> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.\r\n\r\n[caption id=\"attachment_247\" align=\"alignleft\" width=\"110\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\"size-full wp-image-247 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" alt=\"Learn About Our Projects\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\">Learn About Our Projects</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignleft\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" alt=\"Support Us\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_248\" align=\"alignleft\" width=\"110\"]<a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\"><img class=\"size-full wp-image-248 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" width=\"110\" height=\"83\" /></a> <a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-05 20:25:19','2014-06-05 20:25:19','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(540,1,'2014-06-05 20:49:55','2014-06-05 20:49:55','<span style=\"color: #63737f;\">[lenslider id=\'5659efb21d\']</span>\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<img class=\"size-full wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"240\" height=\"181\" /> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.\r\n\r\n[caption id=\"attachment_247\" align=\"alignleft\" width=\"110\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\"size-full wp-image-247 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" alt=\"Learn About Our Projects\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\">Learn About Our Projects</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignleft\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" alt=\"Support Us\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_248\" align=\"alignleft\" width=\"110\"]<a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\"><img class=\"size-full wp-image-248 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" width=\"110\" height=\"83\" /></a> <a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-05 20:49:55','2014-06-05 20:49:55','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(541,1,'2014-06-05 20:54:43','2014-06-05 20:54:43','<div class=\"partner_images\">\r\n<h2 style=\"text-align: center;\">Funders and Associates of the Genomic Variation Laboratory</h2>\r\n<p style=\"text-align: center;\"><a title=\"US Fish &amp; Wildlife Service\" href=\"http://www.fws.gov/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-193\" style=\"font-size: 1rem; line-height: 1;\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/U_S__Fish__and__Wildlife_Service.jpg\" alt=\"U_S__Fish__and__Wildlife_Service\" width=\"103\" height=\"124\" /></a><a title=\"California Department of Water Resources\" href=\"http://www.water.ca.gov/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-186\" style=\"font-size: 1rem; line-height: 1;\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/dwr-logobig_home.jpg\" alt=\"dwr-logobig_home\" width=\"125\" height=\"125\" /></a><a title=\"US Forest Service\" href=\"http://www.fs.fed.us/\" target=\"_blank\"><img class=\"alignnone size-medium wp-image-194\" style=\"font-size: 1rem; line-height: 1;\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/USForestServiceLogo.jpg\" alt=\"USForestServiceLogo\" width=\"108\" height=\"121\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" title=\"US Department of the Interior, Bureau of Land Management\" href=\"http://www.blm.gov/wo/st/en.html\" target=\"_blank\"><img class=\"alignnone size-full wp-image-171\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/BLMLogo.jpg\" alt=\"BLMLogo\" width=\"123\" height=\"123\" /></a><a style=\"line-height: 1.714285714; font-size: 1rem;\" title=\"California Department of Fish &amp; Game\" href=\"http://www.dfg.ca.gov/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-172\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/DFGshield.jpg\" alt=\"DFGshield\" width=\"102\" height=\"135\" /></a><span style=\"line-height: 1.714285714; font-size: 1rem;\"></span><a title=\"Cramer Fish Sciences\" href=\"http://www.fishsciences.net/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-184\" style=\"font-size: 1rem; line-height: 1;\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.jpg\" alt=\"droppedImage\" width=\"233\" height=\"46\" /></a><a title=\"Kern Community Foundation\" href=\"http://www.kernfoundation.org/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-195\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/logo.jpg\" alt=\"logo\" width=\"165\" height=\"57\" /></a><a title=\"Kootenai Tribe of Idaho\" href=\"http://www.kootenai.org/\" target=\"_blank\"><img class=\"alignnone size-full wp-image-187\" style=\"font-size: 1rem; line-height: 1;\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/shapeimage_4.png\" alt=\"shapeimage_4\" width=\"167\" height=\"112\" /></a><span style=\"line-height: 1.714285714; font-size: 1rem;\"><img class=\"alignnone size-full wp-image-173\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_1.jpg\" alt=\"droppedImage_1\" width=\"121\" height=\"121\" /><img class=\"alignnone size-full wp-image-174\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_2.jpg\" alt=\"droppedImage_2\" width=\"188\" height=\"54\" /><img class=\"alignnone size-full wp-image-175\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_3.jpg\" alt=\"droppedImage_3\" width=\"93\" height=\"93\" /><img class=\"alignnone size-full wp-image-176\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_4.jpg\" alt=\"droppedImage_4\" width=\"124\" height=\"72\" /><img class=\"alignnone size-full wp-image-177\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_5.jpg\" alt=\"droppedImage_5\" width=\"213\" height=\"68\" /><img class=\"alignnone size-full wp-image-178\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_6.jpg\" alt=\"droppedImage_6\" width=\"86\" height=\"113\" /><img class=\"alignnone size-full wp-image-179\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_7.jpg\" alt=\"droppedImage_7\" width=\"149\" height=\"102\" /></span><img class=\"alignnone size-full wp-image-181\" style=\"font-size: 1rem; line-height: 1;\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_9.jpg\" alt=\"droppedImage_9\" width=\"119\" height=\"119\" /><img class=\"alignnone size-full wp-image-185\" style=\"font-size: 1rem; line-height: 1;\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage.png\" alt=\"droppedImage\" width=\"97\" height=\"140\" /><img class=\"alignnone size-full wp-image-180\" style=\"font-size: 1rem; line-height: 1;\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_8.jpg\" alt=\"droppedImage_8\" width=\"244\" height=\"70\" /><img class=\"alignnone size-full wp-image-182\" style=\"font-size: 1rem; line-height: 1;\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_10.jpg\" alt=\"droppedImage_10\" width=\"157\" height=\"123\" /><span style=\"line-height: 1.714285714; font-size: 1rem;\"><img class=\"alignnone size-full wp-image-183\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/droppedImage_11.jpg\" alt=\"droppedImage_11\" width=\"192\" height=\"71\" /></span></p>\r\n\r\n</div>','Partners','','inherit','closed','open','','10-revision-v1','','','2014-06-05 20:54:43','2014-06-05 20:54:43','',10,'http://localhost/~psalessi/Genetics_Lab/10-revision-v1/',0,'revision','',0),(542,1,'2014-06-05 20:55:45','2014-06-05 20:55:45','[contentSlideshow]\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<img class=\"size-full wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"240\" height=\"181\" /> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.\r\n\r\n[caption id=\"attachment_247\" align=\"alignleft\" width=\"110\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\"size-full wp-image-247 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" alt=\"Learn About Our Projects\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\">Learn About Our Projects</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignleft\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" alt=\"Support Us\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_248\" align=\"alignleft\" width=\"110\"]<a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\"><img class=\"size-full wp-image-248 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" width=\"110\" height=\"83\" /></a> <a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-05 20:55:45','2014-06-05 20:55:45','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(543,1,'2014-06-05 21:21:36','2014-06-05 21:21:36','','Hmmmm','','publish','closed','closed','','hmmmm','','','2014-06-05 21:21:36','2014-06-05 21:21:36','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=resp_slider&#038;p=543',0,'resp_slider','',0),(544,1,'2014-06-05 21:22:21','2014-06-05 21:22:21','<span style=\"color: #555555;\">[responsive_slider id=\'543\' /]</span>\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<img class=\"size-full wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"240\" height=\"181\" /> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.\r\n\r\n[caption id=\"attachment_247\" align=\"alignleft\" width=\"110\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\"size-full wp-image-247 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" alt=\"Learn About Our Projects\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\">Learn About Our Projects</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignleft\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" alt=\"Support Us\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_248\" align=\"alignleft\" width=\"110\"]<a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\"><img class=\"size-full wp-image-248 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" width=\"110\" height=\"83\" /></a> <a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-05 21:22:21','2014-06-05 21:22:21','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(545,1,'2014-06-05 21:25:37','2014-06-05 21:25:37','[contentSlideshow]\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<img class=\"size-full wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"240\" height=\"181\" /> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.\r\n\r\n[caption id=\"attachment_247\" align=\"alignleft\" width=\"110\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\"size-full wp-image-247 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" alt=\"Learn About Our Projects\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\">Learn About Our Projects</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignleft\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" alt=\"Support Us\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_248\" align=\"alignleft\" width=\"110\"]<a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\"><img class=\"size-full wp-image-248 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" width=\"110\" height=\"83\" /></a> <a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-05 21:25:37','2014-06-05 21:25:37','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(546,1,'2014-06-05 22:19:30','0000-00-00 00:00:00','','Auto Draft','','auto-draft','closed','open','','','','','2014-06-05 22:19:30','0000-00-00 00:00:00','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=slide&p=546',0,'slide','',0),(547,1,'2014-06-05 22:22:16','2014-06-05 22:22:16','','Woohooo','','publish','closed','closed','','woohooo','','','2014-06-05 22:25:53','2014-06-05 22:25:53','',0,'http://localhost/~psalessi/Genetics_Lab/?post_type=cycloneslider&#038;p=547',0,'cycloneslider','',0),(548,1,'2014-06-05 22:22:34','2014-06-05 22:22:34','[cycloneslider id=\"woohooo\"]\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<img class=\"size-full wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"240\" height=\"181\" /> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.\r\n\r\n[caption id=\"attachment_247\" align=\"alignleft\" width=\"110\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\"size-full wp-image-247 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" alt=\"Learn About Our Projects\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\">Learn About Our Projects</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignleft\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" alt=\"Support Us\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_248\" align=\"alignleft\" width=\"110\"]<a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\"><img class=\"size-full wp-image-248 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" width=\"110\" height=\"83\" /></a> <a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-05 22:22:34','2014-06-05 22:22:34','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(549,1,'2014-06-05 22:32:52','2014-06-05 22:32:52','<span style=\"color: #000000;\">[tribulant_slideshow]</span>\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<img class=\"size-full wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"240\" height=\"181\" /> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.\r\n\r\n[caption id=\"attachment_247\" align=\"alignleft\" width=\"110\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\"size-full wp-image-247 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" alt=\"Learn About Our Projects\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\">Learn About Our Projects</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignleft\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" alt=\"Support Us\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_248\" align=\"alignleft\" width=\"110\"]<a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\"><img class=\"size-full wp-image-248 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" width=\"110\" height=\"83\" /></a> <a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-05 22:32:52','2014-06-05 22:32:52','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(550,1,'2014-06-05 22:44:07','2014-06-05 22:44:07','Our work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\n[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<img class=\"size-full wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"240\" height=\"181\" /> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.\r\n\r\n[caption id=\"attachment_247\" align=\"alignleft\" width=\"110\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\"size-full wp-image-247 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" alt=\"Learn About Our Projects\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\">Learn About Our Projects</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignleft\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" alt=\"Support Us\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_248\" align=\"alignleft\" width=\"110\"]<a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\"><img class=\"size-full wp-image-248 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" width=\"110\" height=\"83\" /></a> <a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-05 22:44:07','2014-06-05 22:44:07','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(551,1,'2014-06-05 22:47:04','2014-06-05 22:47:04','[caption id=\"attachment_239\" align=\"alignright\" width=\"240\"]<img class=\"size-full wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"240\" height=\"181\" /> $89,045 Kern Community Foundation Grant[/caption]\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.\r\n\r\n[caption id=\"attachment_247\" align=\"alignleft\" width=\"110\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\"size-full wp-image-247 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" alt=\"Learn About Our Projects\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\">Learn About Our Projects</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignleft\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" alt=\"Support Us\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_248\" align=\"alignleft\" width=\"110\"]<a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\"><img class=\"size-full wp-image-248 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" width=\"110\" height=\"83\" /></a> <a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-05 22:47:04','2014-06-05 22:47:04','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(552,1,'2014-06-05 22:47:11','2014-06-05 22:47:11','[caption id=\"attachment_239\" align=\"alignleft\" width=\"240\"]<img class=\"size-full wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"240\" height=\"181\" /> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.\r\n\r\n[caption id=\"attachment_247\" align=\"alignleft\" width=\"110\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\"size-full wp-image-247 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_owl.png\" alt=\"Learn About Our Projects\" width=\"110\" height=\"83\" /></a> <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\">Learn About Our Projects</a>[/caption]\r\n\r\n[caption id=\"attachment_246\" align=\"alignleft\" width=\"110\"]<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\"><img class=\"size-full wp-image-246 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_catfish.png\" alt=\"Support Us\" width=\"110\" height=\"83\" /></a> <a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\">Support Us</a>[/caption]\r\n\r\n[caption id=\"attachment_248\" align=\"alignleft\" width=\"110\"]<a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\"><img class=\"size-full wp-image-248 \" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/gvl_home_salmon.png\" alt=\"2012 Coastwide Salmon Genetics Meeting at UC Davis\" width=\"110\" height=\"83\" /></a> <a title=\"UC Davis Salmon Genetics Meeting 2012\" href=\"http://animalscience.ucdavis.edu/Coastwide/2012_Coastwide_Salmonid_Genetics_Meeting/Home.html\" target=\"_blank\">2012 Coastwide Salmon Genetics Meeting at UC Davis</a>[/caption]','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-05 22:47:11','2014-06-05 22:47:11','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(553,1,'2014-06-05 22:55:47','2014-06-05 22:55:47','[caption id=\"attachment_239\" align=\"alignleft\" width=\"240\"]<img class=\"size-full wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"240\" height=\"181\" /> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-05 22:55:47','2014-06-05 22:55:47','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(554,1,'2014-06-05 22:56:08','2014-06-05 22:56:08','[caption id=\"attachment_239\" align=\"alignleft\" width=\"240\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\"size-full wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-05 22:56:08','2014-06-05 22:56:08','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(555,1,'2014-06-09 16:30:24','2014-06-09 16:30:24','','UC Davis','','publish','closed','open','','uc-davis','','','2014-06-09 16:30:39','2014-06-09 16:30:39','',0,'http://localhost/~psalessi/Genetics_Lab/?p=555',1,'nav_menu_item','',0),(556,1,'2014-06-09 16:30:24','2014-06-09 16:30:24','','Department of Animal Science','','publish','closed','open','','department-of-animal-science','','','2014-06-09 16:30:39','2014-06-09 16:30:39','',0,'http://localhost/~psalessi/Genetics_Lab/?p=556',2,'nav_menu_item','',0),(557,1,'2014-06-09 16:58:56','2014-06-09 16:58:56','<strong>From Sac. Metro Airport:</strong>\r\n\r\nGet on I-5 south from airport Blvd.\r\n\r\nTake I-80 West towards San Francisco.\r\n\r\n<strong>From I-80 east or west:</strong>\r\n\r\nExit at the \"U.C. Davis/Old Davis Road\" exit and travel North.\r\n\r\nFollow Old Davis Road and go straight through first stop sign.\r\n\r\nCross bridge and turn left at first intersection, La Rue\r\n\r\nTurn right on Bioletti Way, and left in the parking lot. There is a permit dispenser just as you enter from this side. Parking is $7 per day.\r\n\r\nMeyer Hall is on the corner of Bioletti and La Rue.\r\n\r\nThe GVL is on the second floor in Rm. 2403 in the north building of Meyer Hall.\r\n\r\n<iframe src=\"https://www.google.com/maps/embed?pb=!1m22!1m12!1m3!1d6241.791190214867!2d-121.75414226310114!3d38.53617479063705!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!4m7!1i0!3e6!4m0!4m3!3m2!1d38.5347631!2d-121.7542419!5e0!3m2!1sen!2sus!4v1402333107571\" width=\"600\" height=\"450\" frameborder=\"0\" style=\"border:0\"></iframe>','Directions','','inherit','closed','open','','230-revision-v1','','','2014-06-09 16:58:56','2014-06-09 16:58:56','',230,'http://localhost/~psalessi/Genetics_Lab/230-revision-v1/',0,'revision','',0),(558,1,'2014-06-09 17:00:35','2014-06-09 17:00:35','<strong>From Sac. Metro Airport:</strong>\r\n\r\nGet on I-5 south from airport Blvd.\r\n\r\nTake I-80 West towards San Francisco.\r\n\r\n<strong>From I-80 east or west:</strong>\r\n\r\nExit at the \"U.C. Davis/Old Davis Road\" exit and travel North.\r\n\r\nFollow Old Davis Road and go straight through first stop sign.\r\n\r\nCross bridge and turn left at first intersection, La Rue\r\n\r\nTurn right on Bioletti Way, and left in the parking lot. There is a permit dispenser just as you enter from this side. Parking is $7 per day.\r\n\r\nMeyer Hall is on the corner of Bioletti and La Rue.\r\n\r\nThe GVL is on the second floor in Rm. 2403 in the north building of Meyer Hall.\r\n\r\n<iframe style=\"border: 0;\" src=\"https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d6241.639725534423!2d-121.75234279909584!3d38.53792040586795!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x8085291ab1c780f9%3A0xc365a885cdb2e287!2sMeyer+Hall!5e0!3m2!1sen!2sus!4v1402333214210\" width=\"600\" height=\"450\" frameborder=\"0\"></iframe>','Directions','','inherit','closed','open','','230-revision-v1','','','2014-06-09 17:00:35','2014-06-09 17:00:35','',230,'http://localhost/~psalessi/Genetics_Lab/230-revision-v1/',0,'revision','',0),(559,1,'2014-06-09 17:00:51','2014-06-09 17:00:51','<strong>From Sac. Metro Airport:</strong> Get on I-5 south from airport Blvd. Take I-80 West towards San Francisco. <strong>From I-80 east or west:</strong> Exit at the \"U.C. Davis/Old Davis Road\" exit and travel North. Follow Old Davis Road and go straight through first stop sign. Cross bridge and turn left at first intersection, La Rue Turn right on Bioletti Way, and left in the parking lot. There is a permit dispenser just as you enter from this side. Parking is $7 per day. Meyer Hall is on the corner of Bioletti and La Rue. The GVL is on the second floor in Rm. 2403 in the north building of Meyer Hall. <iframe style=\"border: 0;\" src=\"https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d6241.639725534423!2d-121.75234279909584!3d38.53792040586795!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x8085291ab1c780f9%3A0xc365a885cdb2e287!2sMeyer+Hall!5e0!3m2!1sen!2sus!4v1402333214210\" width=\"600\" height=\"450\" frameborder=\"0\"></iframe>','Directions','','inherit','closed','open','','230-revision-v1','','','2014-06-09 17:00:51','2014-06-09 17:00:51','',230,'http://localhost/~psalessi/Genetics_Lab/230-revision-v1/',0,'revision','',0),(560,1,'2014-06-09 17:02:02','2014-06-09 17:02:02','<strong>From Sac. Metro Airport:</strong>\r\n\r\nGet on I-5 south from airport Blvd.\r\n\r\nTake I-80 West towards San Francisco.\r\n\r\n<strong>From I-80 east or west:</strong>\r\n\r\nExit at the \"U.C. Davis/Old Davis Road\" exit and travel North.\r\n\r\nFollow Old Davis Road and go straight through first stop sign.\r\n\r\nCross bridge and turn left at first intersection, La Rue\r\n\r\nTurn right on Bioletti Way, and left in the parking lot. There is a permit dispenser just as you enter from this side. Parking is $7 per day.\r\n\r\nMeyer Hall is on the corner of Bioletti and La Rue.\r\n\r\nThe GVL is on the second floor in Rm. 2403 in the north building of Meyer Hall.\r\n\r\n<iframe style=\"border: 0;\" align=\"aligncenter\" src=\"https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d6241.639725534423!2d-121.75234279909584!3d38.53792040586795!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x8085291ab1c780f9%3A0xc365a885cdb2e287!2sMeyer+Hall!5e0!3m2!1sen!2sus!4v1402333214210\" width=\"600\" height=\"450\" frameborder=\"0\"></iframe>','Directions','','inherit','closed','open','','230-revision-v1','','','2014-06-09 17:02:02','2014-06-09 17:02:02','',230,'http://localhost/~psalessi/Genetics_Lab/230-revision-v1/',0,'revision','',0),(561,1,'2014-06-09 17:03:08','2014-06-09 17:03:08','<strong>From Sac. Metro Airport:</strong>\r\n\r\nGet on I-5 south from airport Blvd.\r\n\r\nTake I-80 West towards San Francisco.\r\n\r\n<strong>From I-80 east or west:</strong>\r\n\r\nExit at the \"U.C. Davis/Old Davis Road\" exit and travel North.\r\n\r\nFollow Old Davis Road and go straight through first stop sign.\r\n\r\nCross bridge and turn left at first intersection, La Rue\r\n\r\nTurn right on Bioletti Way, and left in the parking lot. There is a permit dispenser just as you enter from this side. Parking is $7 per day.\r\n\r\nMeyer Hall is on the corner of Bioletti and La Rue.\r\n\r\nThe GVL is on the second floor in Rm. 2403 in the north building of Meyer Hall.\r\n\r\n<iframe style=\"border: 0;\" class=\"aligncenter\" src=\"https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d6241.639725534423!2d-121.75234279909584!3d38.53792040586795!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x8085291ab1c780f9%3A0xc365a885cdb2e287!2sMeyer+Hall!5e0!3m2!1sen!2sus!4v1402333214210\" width=\"600\" height=\"450\" frameborder=\"0\"></iframe>','Directions','','inherit','closed','open','','230-revision-v1','','','2014-06-09 17:03:08','2014-06-09 17:03:08','',230,'http://localhost/~psalessi/Genetics_Lab/230-revision-v1/',0,'revision','',0),(562,1,'2014-06-09 19:16:02','2014-06-09 19:16:02','[caption id=\"attachment_239\" align=\"alignleft\" width=\"240\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\"size-full wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]Our work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-09 19:16:02','2014-06-09 19:16:02','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(563,1,'2014-06-09 20:27:28','2014-06-09 20:27:28','<img class=\"gvl_blue_logo alignright\" alt=\"GVL_logo_blue\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_blue.png\" width=\"207\" height=\"217\" /><strong><a title=\"Protocols and Procedures\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/\">Protocol and Procedures</a></strong>\r\n\r\n<strong><a title=\"Links\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/links/\">Links</a></strong>\r\n\r\n<strong><a title=\"Contact Us\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/contact-us/\">Contact Us</a></strong>\r\n<p style=\"text-align: left;\">2403 Meyer Hall, Department of Animal Science\r\nUniversity of California, DavisOne Shields Avenue Davis, CA 95616</p>\r\n<a href=\"http://localhost/~psalessi/Genetics_Lab/resources/directions/\">Directions</a>\r\n\r\nPhone: (530) 752-6351\r\nFax: (530) 752-0175\r\n\r\n<strong><a title=\"Directions\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/directions/\">Directions</a></strong>','Resources','','inherit','closed','open','','16-revision-v1','','','2014-06-09 20:27:28','2014-06-09 20:27:28','',16,'http://localhost/~psalessi/Genetics_Lab/16-revision-v1/',0,'revision','',0),(564,1,'2014-06-09 20:28:24','2014-06-09 20:28:24','<img class=\"gvl_blue_logo alignright\" alt=\"GVL_logo_blue\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_blue.png\" width=\"207\" height=\"217\" /><strong><a title=\"Protocols and Procedures\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/\">Protocol and Procedures</a></strong>\r\n\r\n<strong><a title=\"Links\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/links/\">Links</a></strong>\r\n\r\n<strong><a title=\"Contact Us\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/contact-us/\">Contact Us</a></strong>\r\n<p style=\"text-align: left;\">2403 Meyer Hall, Department of Animal Science\r\nUniversity of California, DavisOne Shields Avenue Davis, CA 95616</p>\r\n\r\nPhone: (530) 752-6351\r\nFax: (530) 752-0175\r\n\r\n<strong>From Sac. Metro Airport:</strong>\r\n\r\nGet on I-5 south from airport Blvd.\r\n\r\nTake I-80 West towards San Francisco.\r\n\r\n<strong>From I-80 east or west:</strong>\r\n\r\nExit at the \"U.C. Davis/Old Davis Road\" exit and travel North.\r\n\r\nFollow Old Davis Road and go straight through first stop sign.\r\n\r\nCross bridge and turn left at first intersection, La Rue\r\n\r\nTurn right on Bioletti Way, and left in the parking lot. There is a permit dispenser just as you enter from this side. Parking is $7 per day.\r\n\r\nMeyer Hall is on the corner of Bioletti and La Rue.\r\n\r\nThe GVL is on the second floor in Rm. 2403 in the north building of Meyer Hall.\r\n\r\n<iframe style=\"border: 0;\" class=\"aligncenter\" src=\"https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d6241.639725534423!2d-121.75234279909584!3d38.53792040586795!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x8085291ab1c780f9%3A0xc365a885cdb2e287!2sMeyer+Hall!5e0!3m2!1sen!2sus!4v1402333214210\" width=\"600\" height=\"450\" frameborder=\"0\"></iframe>','Resources','','inherit','closed','open','','16-revision-v1','','','2014-06-09 20:28:24','2014-06-09 20:28:24','',16,'http://localhost/~psalessi/Genetics_Lab/16-revision-v1/',0,'revision','',0),(565,1,'2014-06-09 20:29:36','2014-06-09 20:29:36','<img class=\"gvl_blue_logo alignright\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_blue.png\" alt=\"GVL_logo_blue\" width=\"207\" height=\"217\" /><strong><a title=\"Protocols and Procedures\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/\">Protocol and Procedures</a></strong>\r\n\r\n<strong><a title=\"Links\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/links/\">Links</a></strong>\r\n\r\n<strong><a title=\"Contact Us\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/contact-us/\">Contact Us</a></strong>\r\n<p style=\"text-align: left;\">2403 Meyer Hall, Department of Animal Science\r\nUniversity of California, DavisOne Shields Avenue Davis, CA 95616</p>\r\nPhone: (530) 752-6351\r\nFax: (530) 752-0175\r\n<h3>Directions:</h3>\r\n<h4><span style=\"text-decoration: underline;\">From Sac. Metro Airport</span></h4>\r\nGet on I-5 south from airport Blvd.\r\n\r\nTake I-80 West towards San Francisco.\r\n<h4><span style=\"text-decoration: underline;\"><strong>From I-80 east or west</strong></span></h4>\r\nExit at the \"U.C. Davis/Old Davis Road\" exit and travel North.\r\n\r\nFollow Old Davis Road and go straight through first stop sign.\r\n\r\nCross bridge and turn left at first intersection, La Rue\r\n\r\nTurn right on Bioletti Way, and left in the parking lot. There is a permit dispenser just as you enter from this side. Parking is $7 per day.\r\n\r\nMeyer Hall is on the corner of Bioletti and La Rue.\r\n\r\nThe GVL is on the second floor in Rm. 2403 in the north building of Meyer Hall.\r\n\r\n<iframe class=\"aligncenter\" style=\"border: 0;\" src=\"https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d6241.639725534423!2d-121.75234279909584!3d38.53792040586795!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x8085291ab1c780f9%3A0xc365a885cdb2e287!2sMeyer+Hall!5e0!3m2!1sen!2sus!4v1402333214210\" width=\"600\" height=\"450\" frameborder=\"0\"></iframe>','Resources','','inherit','closed','open','','16-revision-v1','','','2014-06-09 20:29:36','2014-06-09 20:29:36','',16,'http://localhost/~psalessi/Genetics_Lab/16-revision-v1/',0,'revision','',0),(566,1,'2014-06-09 20:30:38','2014-06-09 20:30:38','<img class=\"gvl_blue_logo alignright\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_blue.png\" alt=\"GVL_logo_blue\" width=\"207\" height=\"217\" /><strong><a title=\"Protocols and Procedures\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/\">Protocol and Procedures</a></strong>\r\n\r\n<strong><a title=\"Links\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/links/\">Links</a></strong>\r\n\r\n<h3>Contact Us</h3>\r\n<p style=\"text-align: left;\">2403 Meyer Hall, Department of Animal Science\r\nUniversity of California, DavisOne Shields Avenue Davis, CA 95616</p>\r\nPhone: (530) 752-6351\r\nFax: (530) 752-0175\r\n<h3>Directions</h3>\r\n<h4><span style=\"text-decoration: underline;\">From Sac. Metro Airport:</span></h4>\r\nGet on I-5 south from airport Blvd.\r\n\r\nTake I-80 West towards San Francisco.\r\n<h4><span style=\"text-decoration: underline;\"><strong>From I-80 east or west:</strong></span></h4>\r\nExit at the \"U.C. Davis/Old Davis Road\" exit and travel North.\r\n\r\nFollow Old Davis Road and go straight through first stop sign.\r\n\r\nCross bridge and turn left at first intersection, La Rue\r\n\r\nTurn right on Bioletti Way, and left in the parking lot. There is a permit dispenser just as you enter from this side. Parking is $7 per day.\r\n\r\nMeyer Hall is on the corner of Bioletti and La Rue.\r\n\r\nThe GVL is on the second floor in Rm. 2403 in the north building of Meyer Hall.\r\n\r\n<iframe class=\"aligncenter\" style=\"border: 0;\" src=\"https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d6241.639725534423!2d-121.75234279909584!3d38.53792040586795!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x8085291ab1c780f9%3A0xc365a885cdb2e287!2sMeyer+Hall!5e0!3m2!1sen!2sus!4v1402333214210\" width=\"600\" height=\"450\" frameborder=\"0\"></iframe>','Resources','','inherit','closed','open','','16-revision-v1','','','2014-06-09 20:30:38','2014-06-09 20:30:38','',16,'http://localhost/~psalessi/Genetics_Lab/16-revision-v1/',0,'revision','',0),(567,1,'2014-06-09 20:31:26','2014-06-09 20:31:26','<strong><a title=\"Protocols and Procedures\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/protocols-and-procedures/\">Protocol and Procedures</a></strong>\r\n\r\n<strong><a title=\"Links\" href=\"http://localhost/~psalessi/Genetics_Lab/resources/links/\">Links</a></strong>\r\n<h3>Contact Us<a href=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_blue.png\"><img class=\"alignright size-full wp-image-235\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/GVL_logo_blue.png\" alt=\"Genomic variation laboratory blue logo image\" width=\"207\" height=\"217\" /></a></h3>\r\n<p style=\"text-align: left;\">2403 Meyer Hall, Department of Animal Science\r\nUniversity of California, DavisOne Shields Avenue Davis, CA 95616</p>\r\nPhone: (530) 752-6351\r\nFax: (530) 752-0175\r\n<h3>Directions</h3>\r\n<h4><span style=\"text-decoration: underline;\">From Sac. Metro Airport:</span></h4>\r\nGet on I-5 south from airport Blvd.\r\n\r\nTake I-80 West towards San Francisco.\r\n<h4><span style=\"text-decoration: underline;\"><strong>From I-80 east or west:</strong></span></h4>\r\nExit at the \"U.C. Davis/Old Davis Road\" exit and travel North.\r\n\r\nFollow Old Davis Road and go straight through first stop sign.\r\n\r\nCross bridge and turn left at first intersection, La Rue\r\n\r\nTurn right on Bioletti Way, and left in the parking lot. There is a permit dispenser just as you enter from this side. Parking is $7 per day.\r\n\r\nMeyer Hall is on the corner of Bioletti and La Rue.\r\n\r\nThe GVL is on the second floor in Rm. 2403 in the north building of Meyer Hall.\r\n\r\n<iframe class=\"aligncenter\" style=\"border: 0;\" src=\"https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d6241.639725534423!2d-121.75234279909584!3d38.53792040586795!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x8085291ab1c780f9%3A0xc365a885cdb2e287!2sMeyer+Hall!5e0!3m2!1sen!2sus!4v1402333214210\" width=\"600\" height=\"450\" frameborder=\"0\"></iframe>','Resources','','inherit','closed','open','','16-revision-v1','','','2014-06-09 20:31:26','2014-06-09 20:31:26','',16,'http://localhost/~psalessi/Genetics_Lab/16-revision-v1/',0,'revision','',0),(568,1,'2014-06-09 20:47:40','2014-06-09 20:47:40','[caption id=\"attachment_239\" align=\"alignleft\" width=\"240\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\"size-full wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\n<p>Our work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-09 20:47:40','2014-06-09 20:47:40','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(569,1,'2014-06-09 20:48:05','2014-06-09 20:48:05','[caption id=\"attachment_239\" align=\"alignleft\" width=\"240\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\"size-full wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-09 20:48:05','2014-06-09 20:48:05','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(570,1,'2014-06-09 20:48:25','2014-06-09 20:48:25','[caption id=\"attachment_239\" align=\"alignleft\" width=\"240\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\"size-full wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\n<p>Our work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-09 20:48:25','2014-06-09 20:48:25','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(571,1,'2014-06-12 18:13:30','2014-06-12 18:13:30','[caption id=\"attachment_239\" align=\"alignleft\" width=\"240\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\"size-full wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"240\" height=\"181\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\n&nbsp;\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-12 18:13:30','2014-06-12 18:13:30','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(572,1,'2014-06-12 18:13:57','2014-06-12 18:13:57','[caption id=\"attachment_239\" align=\"alignleft\" width=\"270\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\" wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"270\" height=\"195\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-12 18:13:57','2014-06-12 18:13:57','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(573,1,'2014-06-12 19:13:51','2014-06-12 19:13:51','[infinite_slider]\r\n\r\n[caption id=\"attachment_239\" align=\"alignleft\" width=\"270\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\" wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"270\" height=\"195\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-12 19:13:51','2014-06-12 19:13:51','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(574,1,'2014-06-12 19:14:46','2014-06-12 19:14:46','[caption id=\"attachment_239\" align=\"alignleft\" width=\"270\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\" wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"270\" height=\"195\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-12 19:14:46','2014-06-12 19:14:46','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(575,1,'2014-06-12 19:59:14','2014-06-12 19:59:14','[caption id=\"attachment_239\" align=\"alignleft\" width=\"270\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\" wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"270\" height=\"195\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.\r\n[cycloneslider id=\"woohooo\"]','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-12 19:59:14','2014-06-12 19:59:14','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0),(576,1,'2014-06-12 20:27:23','2014-06-12 20:27:23','[caption id=\"attachment_239\" align=\"alignleft\" width=\"270\"]<a title=\"Support Us\" href=\"http://localhost/~psalessi/Genetics_Lab/support-us/\"><img class=\" wp-image-239\" src=\"http://localhost/~psalessi/Genetics_Lab/wp-content/uploads/2014/02/kern_community_foundation_grant.png\" alt=\"$89,045 Kern Community Foundation Grant\" width=\"270\" height=\"195\" /></a> $89,045 Kern Community Foundation Grant[/caption]\r\n\r\nOur work at the Genomic Variation Laboratory (GVL) addresses important questions about thepopulation genetics of threatened and endangered species. We use molecular tools such as microsatellites, single nucleotide polymorphisms, and high throughput sequencing to evaluate the processes that create and maintain biodiversity.\r\n\r\nExamples of current research <a title=\"Projects\" href=\"http://localhost/~psalessi/Genetics_Lab/projects/\" target=\"_blank\">projects</a> include: using molecular techniques to evaluate predation rates on Delta smelt, determining the genetic basis of disease resistance in rainbow trout, and investigating the ecology and evolution of various salmonid species. To address these pressing issues, we collaborate with numerous state and federal agencies, non-profits, and tribal organizations.\r\n\r\nThe GVL is also an active training ground for a large number of young <a title=\"People\" href=\"http://localhost/~psalessi/Genetics_Lab/people/\" target=\"_blank\">scientists</a> at various stages in their careers, from high school students to graduate students to project scientists.','Home','','inherit','closed','open','','30-revision-v1','','','2014-06-12 20:27:23','2014-06-12 20:27:23','',30,'http://localhost/~psalessi/Genetics_Lab/30-revision-v1/',0,'revision','',0);
/*!40000 ALTER TABLE `wp_posts` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_sds_slider`
--

DROP TABLE IF EXISTS `wp_sds_slider`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_sds_slider` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `cat_id` int(11) NOT NULL,
  `slider_title` varchar(50) NOT NULL,
  `slider_description` varchar(500) NOT NULL,
  `url` varchar(255) NOT NULL,
  `slider_img_path` varchar(255) NOT NULL,
  `status` char(1) NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_sds_slider`
--

LOCK TABLES `wp_sds_slider` WRITE;
/*!40000 ALTER TABLE `wp_sds_slider` DISABLE KEYS */;
/*!40000 ALTER TABLE `wp_sds_slider` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_sds_slider_cat`
--

DROP TABLE IF EXISTS `wp_sds_slider_cat`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_sds_slider_cat` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `cat_title` varchar(50) NOT NULL,
  `cat_slug` varchar(100) NOT NULL,
  `cat_description` varchar(500) NOT NULL,
  `status` char(1) NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_sds_slider_cat`
--

LOCK TABLES `wp_sds_slider_cat` WRITE;
/*!40000 ALTER TABLE `wp_sds_slider_cat` DISABLE KEYS */;
/*!40000 ALTER TABLE `wp_sds_slider_cat` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_smooth_slider`
--

DROP TABLE IF EXISTS `wp_smooth_slider`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_smooth_slider` (
  `id` int(5) NOT NULL AUTO_INCREMENT,
  `post_id` int(11) NOT NULL,
  `date` datetime NOT NULL,
  `slider_id` int(5) NOT NULL DEFAULT '1',
  `slide_order` int(5) NOT NULL DEFAULT '0',
  UNIQUE KEY `id` (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_smooth_slider`
--

LOCK TABLES `wp_smooth_slider` WRITE;
/*!40000 ALTER TABLE `wp_smooth_slider` DISABLE KEYS */;
/*!40000 ALTER TABLE `wp_smooth_slider` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_smooth_slider_meta`
--

DROP TABLE IF EXISTS `wp_smooth_slider_meta`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_smooth_slider_meta` (
  `slider_id` int(5) NOT NULL AUTO_INCREMENT,
  `slider_name` varchar(100) NOT NULL DEFAULT '',
  UNIQUE KEY `slider_id` (`slider_id`)
) ENGINE=InnoDB AUTO_INCREMENT=3 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_smooth_slider_meta`
--

LOCK TABLES `wp_smooth_slider_meta` WRITE;
/*!40000 ALTER TABLE `wp_smooth_slider_meta` DISABLE KEYS */;
INSERT INTO `wp_smooth_slider_meta` VALUES (1,'Smooth Slider'),(2,'Paul');
/*!40000 ALTER TABLE `wp_smooth_slider_meta` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_smooth_slider_postmeta`
--

DROP TABLE IF EXISTS `wp_smooth_slider_postmeta`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_smooth_slider_postmeta` (
  `post_id` int(11) NOT NULL,
  `slider_id` int(5) NOT NULL DEFAULT '1',
  UNIQUE KEY `post_id` (`post_id`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_smooth_slider_postmeta`
--

LOCK TABLES `wp_smooth_slider_postmeta` WRITE;
/*!40000 ALTER TABLE `wp_smooth_slider_postmeta` DISABLE KEYS */;
/*!40000 ALTER TABLE `wp_smooth_slider_postmeta` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_term_relationships`
--

DROP TABLE IF EXISTS `wp_term_relationships`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_term_relationships` (
  `object_id` bigint(20) unsigned NOT NULL DEFAULT '0',
  `term_taxonomy_id` bigint(20) unsigned NOT NULL DEFAULT '0',
  `term_order` int(11) NOT NULL DEFAULT '0',
  PRIMARY KEY (`object_id`,`term_taxonomy_id`),
  KEY `term_taxonomy_id` (`term_taxonomy_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_term_relationships`
--

LOCK TABLES `wp_term_relationships` WRITE;
/*!40000 ALTER TABLE `wp_term_relationships` DISABLE KEYS */;
INSERT INTO `wp_term_relationships` VALUES (20,3,0),(21,2,0),(22,2,0),(25,2,0),(26,2,0),(28,2,0),(80,1,0),(114,2,0),(115,2,0),(204,3,0),(555,4,0),(556,4,0);
/*!40000 ALTER TABLE `wp_term_relationships` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_term_taxonomy`
--

DROP TABLE IF EXISTS `wp_term_taxonomy`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_term_taxonomy` (
  `term_taxonomy_id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
  `term_id` bigint(20) unsigned NOT NULL DEFAULT '0',
  `taxonomy` varchar(32) NOT NULL DEFAULT '',
  `description` longtext NOT NULL,
  `parent` bigint(20) unsigned NOT NULL DEFAULT '0',
  `count` bigint(20) NOT NULL DEFAULT '0',
  PRIMARY KEY (`term_taxonomy_id`),
  UNIQUE KEY `term_id_taxonomy` (`term_id`,`taxonomy`),
  KEY `taxonomy` (`taxonomy`)
) ENGINE=InnoDB AUTO_INCREMENT=5 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_term_taxonomy`
--

LOCK TABLES `wp_term_taxonomy` WRITE;
/*!40000 ALTER TABLE `wp_term_taxonomy` DISABLE KEYS */;
INSERT INTO `wp_term_taxonomy` VALUES (1,1,'category','',0,0),(2,2,'nav_menu','',0,7),(3,3,'ml-slider','',0,1),(4,4,'nav_menu','',0,2);
/*!40000 ALTER TABLE `wp_term_taxonomy` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_terms`
--

DROP TABLE IF EXISTS `wp_terms`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_terms` (
  `term_id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
  `name` varchar(200) NOT NULL DEFAULT '',
  `slug` varchar(200) NOT NULL DEFAULT '',
  `term_group` bigint(10) NOT NULL DEFAULT '0',
  PRIMARY KEY (`term_id`),
  UNIQUE KEY `slug` (`slug`),
  KEY `name` (`name`)
) ENGINE=InnoDB AUTO_INCREMENT=5 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_terms`
--

LOCK TABLES `wp_terms` WRITE;
/*!40000 ALTER TABLE `wp_terms` DISABLE KEYS */;
INSERT INTO `wp_terms` VALUES (1,'Uncategorized','uncategorized',0),(2,'Main Menu','main-menu',0),(3,'526','526',0),(4,'Secondary Menu','secondary-menu',0);
/*!40000 ALTER TABLE `wp_terms` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_usermeta`
--

DROP TABLE IF EXISTS `wp_usermeta`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_usermeta` (
  `umeta_id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
  `user_id` bigint(20) unsigned NOT NULL DEFAULT '0',
  `meta_key` varchar(255) DEFAULT NULL,
  `meta_value` longtext,
  PRIMARY KEY (`umeta_id`),
  KEY `user_id` (`user_id`),
  KEY `meta_key` (`meta_key`)
) ENGINE=InnoDB AUTO_INCREMENT=66 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_usermeta`
--

LOCK TABLES `wp_usermeta` WRITE;
/*!40000 ALTER TABLE `wp_usermeta` DISABLE KEYS */;
INSERT INTO `wp_usermeta` VALUES (1,1,'first_name',''),(2,1,'last_name',''),(3,1,'nickname','psalessi'),(4,1,'description',''),(5,1,'rich_editing','true'),(6,1,'comment_shortcuts','false'),(7,1,'admin_color','fresh'),(8,1,'use_ssl','0'),(9,1,'show_admin_bar_front','true'),(10,1,'wp_capabilities','a:1:{s:13:\"administrator\";b:1;}'),(11,1,'wp_user_level','10'),(12,1,'dismissed_wp_pointers','wp330_toolbar,wp330_saving_widgets,wp340_choose_image_from_library,wp340_customize_current_theme_link,wp350_media,wp360_revisions,wp360_locks,slidedeck_installation-discount,wp390_widgets'),(13,1,'show_welcome_panel','0'),(14,1,'wp_user-settings','mfold=o&libraryContent=browse&editor=html&hidetb=1&ed_size=493&wplink=1&align=right&imgsize=full'),(15,1,'wp_user-settings-time','1402602888'),(16,1,'wp_dashboard_quick_press_last_post_id','519'),(17,1,'managenav-menuscolumnshidden','a:4:{i:0;s:11:\"link-target\";i:1;s:11:\"css-classes\";i:2;s:3:\"xfn\";i:3;s:11:\"description\";}'),(18,1,'metaboxhidden_nav-menus','a:3:{i:0;s:8:\"add-post\";i:1;s:12:\"add-post_tag\";i:2;s:15:\"add-post_format\";}'),(19,1,'nav_menu_recently_edited','2'),(20,1,'closedpostboxes_page','a:0:{}'),(21,1,'metaboxhidden_page','a:4:{i:0;s:10:\"postcustom\";i:1;s:16:\"commentstatusdiv\";i:2;s:7:\"slugdiv\";i:3;s:9:\"authordiv\";}'),(22,1,'closedpostboxes_people_profiles','a:5:{i:0;s:12:\"postimagediv\";i:1;s:14:\"education_meta\";i:2;s:23:\"research_interests_meta\";i:3;s:22:\"research_projects_meta\";i:4;s:17:\"publications_meta\";}'),(23,1,'metaboxhidden_people_profiles','a:3:{i:0;s:11:\"postexcerpt\";i:1;s:7:\"slugdiv\";i:2;s:9:\"authordiv\";}'),(24,1,'meta-box-order_people_profiles','a:4:{s:15:\"acf_after_title\";s:0:\"\";s:4:\"side\";s:36:\"postimagediv,acf_69,acf_72,submitdiv\";s:6:\"normal\";s:48:\"postexcerpt,slugdiv,personal_info_meta,authordiv\";s:8:\"advanced\";s:79:\"education_meta,research_interests_meta,research_projects_meta,publications_meta\";}'),(25,1,'screen_layout_people_profiles','2'),(26,1,'meta-box-order_people','a:4:{s:15:\"acf_after_title\";s:0:\"\";s:4:\"side\";s:59:\"acf_453,acf_69,acf_72,acf_208,postimagediv,submitdiv,acf_97\";s:6:\"normal\";s:123:\"acf_437,acf_439,acf_438,acf_100,acf_104,acf_103,acf_105,acf_121,acf_102,acf_101,acf_199,acf_93,acf_94,acf_95,acf_96,slugdiv\";s:8:\"advanced\";s:0:\"\";}'),(27,1,'screen_layout_people','2'),(28,1,'closedpostboxes_people','a:0:{}'),(29,1,'metaboxhidden_people','a:11:{i:0;s:6:\"acf_97\";i:1;s:7:\"acf_439\";i:2;s:7:\"acf_438\";i:3;s:7:\"acf_100\";i:4;s:7:\"acf_104\";i:5;s:7:\"acf_103\";i:6;s:7:\"acf_105\";i:7;s:7:\"acf_121\";i:8;s:7:\"acf_102\";i:9;s:7:\"acf_101\";i:10;s:7:\"slugdiv\";}'),(30,1,'closedpostboxes_acf','a:0:{}'),(31,1,'metaboxhidden_acf','a:1:{i:0;s:7:\"slugdiv\";}'),(32,1,'closedpostboxes_projects','a:0:{}'),(33,1,'metaboxhidden_projects','a:10:{i:0;s:6:\"acf_69\";i:1;s:6:\"acf_72\";i:2;s:7:\"acf_208\";i:3;s:7:\"acf_437\";i:4;s:7:\"acf_199\";i:5;s:6:\"acf_93\";i:6;s:6:\"acf_94\";i:7;s:6:\"acf_95\";i:8;s:6:\"acf_96\";i:9;s:7:\"slugdiv\";}'),(34,1,'meta-box-order_projects','a:4:{s:15:\"acf_after_title\";s:0:\"\";s:4:\"side\";s:51:\"submitdiv,acf_69,acf_97,acf_72,postimagediv,acf_208\";s:6:\"normal\";s:123:\"acf_437,acf_199,acf_438,acf_439,acf_121,acf_93,acf_100,acf_101,acf_102,acf_103,acf_104,acf_105,acf_94,acf_95,acf_96,slugdiv\";s:8:\"advanced\";s:0:\"\";}'),(35,1,'screen_layout_projects','2'),(36,1,'closedpostboxes_post','a:0:{}'),(37,1,'metaboxhidden_post','a:20:{i:0;s:6:\"acf_69\";i:1;s:6:\"acf_97\";i:2;s:6:\"acf_72\";i:3;s:6:\"acf_93\";i:4;s:7:\"acf_100\";i:5;s:7:\"acf_104\";i:6;s:7:\"acf_103\";i:7;s:7:\"acf_105\";i:8;s:7:\"acf_121\";i:9;s:7:\"acf_102\";i:10;s:6:\"acf_94\";i:11;s:6:\"acf_95\";i:12;s:7:\"acf_101\";i:13;s:6:\"acf_96\";i:14;s:11:\"postexcerpt\";i:15;s:13:\"trackbacksdiv\";i:16;s:10:\"postcustom\";i:17;s:16:\"commentstatusdiv\";i:18;s:7:\"slugdiv\";i:19;s:9:\"authordiv\";}'),(38,1,'meta-box-order_dashboard','a:4:{s:6:\"normal\";s:38:\"dashboard_right_now,dashboard_activity\";s:4:\"side\";s:21:\"dashboard_quick_press\";s:7:\"column3\";s:17:\"dashboard_primary\";s:7:\"column4\";s:0:\"\";}'),(39,2,'first_name',''),(40,2,'last_name',''),(41,2,'nickname','emschellhous'),(42,2,'description',''),(43,2,'rich_editing','true'),(44,2,'comment_shortcuts','false'),(45,2,'admin_color','fresh'),(46,2,'use_ssl','0'),(47,2,'show_admin_bar_front','true'),(48,2,'wp_capabilities','a:1:{s:13:\"administrator\";b:1;}'),(49,2,'wp_user_level','10'),(50,2,'dismissed_wp_pointers','wp330_toolbar,wp330_saving_widgets,wp340_choose_image_from_library,wp340_customize_current_theme_link,wp350_media,wp360_revisions,wp360_locks'),(61,1,'wp_easyrotator_activatetime','1402007257'),(62,1,'wp_easyrotator_lovebar_showtime','-1'),(63,1,'wp_easyrotator_lovebar_showurl','no'),(64,1,'wp_easyrotator_welcome_notice_hide','no'),(65,1,'wp_easyrotator_help_tooltips_postpagebtn_hide','no');
/*!40000 ALTER TABLE `wp_usermeta` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_users`
--

DROP TABLE IF EXISTS `wp_users`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_users` (
  `ID` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
  `user_login` varchar(60) NOT NULL DEFAULT '',
  `user_pass` varchar(64) NOT NULL DEFAULT '',
  `user_nicename` varchar(50) NOT NULL DEFAULT '',
  `user_email` varchar(100) NOT NULL DEFAULT '',
  `user_url` varchar(100) NOT NULL DEFAULT '',
  `user_registered` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `user_activation_key` varchar(60) NOT NULL DEFAULT '',
  `user_status` int(11) NOT NULL DEFAULT '0',
  `display_name` varchar(250) NOT NULL DEFAULT '',
  PRIMARY KEY (`ID`),
  KEY `user_login_key` (`user_login`),
  KEY `user_nicename` (`user_nicename`)
) ENGINE=InnoDB AUTO_INCREMENT=3 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_users`
--

LOCK TABLES `wp_users` WRITE;
/*!40000 ALTER TABLE `wp_users` DISABLE KEYS */;
INSERT INTO `wp_users` VALUES (1,'psalessi','$P$BDfYO2FT15MFaoGhNhbIOnU8p//DIm/','psalessi','pasalessi@ucdavis.edu','','2014-02-10 18:40:59','',0,'psalessi'),(2,'emschellhous','$P$BFHCFDrPxDxxpHLlJszkFjQQlj7lC60','emschellhous','emschellhous@ucdavis.edu','','2014-03-25 16:47:26','',0,'emschellhous');
/*!40000 ALTER TABLE `wp_users` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `wp_vslider`
--

DROP TABLE IF EXISTS `wp_vslider`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `wp_vslider` (
  `id` mediumint(9) NOT NULL AUTO_INCREMENT,
  `option_name` varchar(255) NOT NULL DEFAULT 'vslider_options',
  `active` tinyint(1) NOT NULL DEFAULT '0',
  PRIMARY KEY (`id`),
  UNIQUE KEY `option_name` (`option_name`)
) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `wp_vslider`
--

LOCK TABLES `wp_vslider` WRITE;
/*!40000 ALTER TABLE `wp_vslider` DISABLE KEYS */;
INSERT INTO `wp_vslider` VALUES (1,'vslider_options',1);
/*!40000 ALTER TABLE `wp_vslider` ENABLE KEYS */;
UNLOCK TABLES;
/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;

/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;

-- Dump completed on 2014-06-12 13:33:59
